{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "=begin\n",
    "\n",
    "Purpose: Create First Generation Output File From DesignStudio\n",
    "\n",
    "Use: CIViC_DesignStudio will pull existing variants from the CIViC Knowledgebase, iterate through all variants, and create an output that can be used to create IDT probes\n",
    "\n",
    "Inputs: 1) variants of interest = from the CIViC Probe DesignStudio Interface, any variants that are selected will be evaluated for probe design.\n",
    "\n",
    "Outputs: 1) CIViC_DesignStudio_coordinates.tsv = for each variant selected, we will provide the chromosome, start, stop, gene, variant and pipeline\n",
    "\n",
    "=end"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "false"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#!/usr/bin/env ruby\n",
    "\n",
    "require \"rubygems\"\n",
    "require \"json\"\n",
    "require \"net/http\"\n",
    "require \"uri\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "IOPub data rate exceeded.\n",
      "The notebook server will temporarily stop sending output\n",
      "to the client in order to avoid crashing it.\n",
      "To change this limit, set the config variable\n",
      "`--NotebookApp.iopub_data_rate_limit`.\n"
     ]
    }
   ],
   "source": [
    "#pull in CIViC API for DNA-based variants \n",
    "url_DNA = 'https://civic.genome.wustl.edu/api/panels/DNA-based/qualifying_variants?minimum_score=0'\n",
    "resp_DNA = Net::HTTP.get_response(URI.parse(url_DNA))\n",
    "variants_DNA = JSON.parse(resp_DNA.body)['records']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[[\"chrom\", \"start\", \"stop\", \"chrom2\", \"start2\", \"stop2\", \"gene\", \"variant\", \"pipeline\"]]"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#set list for DNA-based capture\n",
    "capture_DNA = []\n",
    "capture_DNA << [\"chrom\", \"start\", \"stop\", \"chrom2\", \"start2\", \"stop2\", \"gene\", \"variant\", \"pipeline\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[[\"chrom\", \"start\", \"stop\", \"chrom2\", \"start2\", \"stop2\", \"gene\", \"variant\", \"pipeline\"], [\"17\", 7571720, 7590856, \" \", \" \", \" \", \"TP53\", \"WILD TYPE\", \"DNA-based\"], [\"7\", 140434279, 140624564, \" \", \" \", \" \", \"BRAF\", \"WILD TYPE\", \"DNA-based\"], [\"14\", 106032614, 107288051, \"18\", 60794268, 60987019, \"BCL2\", \"IGH-BCL2\", \"DNA-based\"], [\"7\", 140434279, 140624564, \" \", \" \", \" \", \"BRAF\", \"V600E AMPLIFICATION\", \"DNA-based\"], [\"11\", 69455855, 69469242, \" \", \" \", \" \", \"CCND1\", \"AMPLIFICATION\", \"DNA-based\"], [\"3\", 71003844, 71633140, \" \", \" \", \" \", \"FOXP1\", \"AMPLIFICATION\", \"DNA-based\"], [\"2\", 61108709, 61149800, \" \", \" \", \" \", \"REL\", \"AMPLIFICATION\", \"DNA-based\"], [\"2\", 42396490, 42528380, \"2\", 29415640, 29446394, \"ALK\", \"EML4-ALK AMPLIFICATION\", \"DNA-based\"], [\"20\", 54944446, 54967393, \" \", \" \", \" \", \"AURKA\", \"AMPLIFICATION\", \"DNA-based\"], [\"19\", 30302805, 30315215, \" \", \" \", \" \", \"CCNE1\", \"AMPLIFICATION\", \"DNA-based\"], [\"7\", 55086794, 55279321, \" \", \" \", \" \", \"EGFR\", \"AMPLIFICATION\", \"DNA-based\"], [\"20\", 46130646, 46285621, \" \", \" \", \" \", \"NCOA3\", \"AMPLIFICATION\", \"DNA-based\"], [\"3\", 178866311, 178957881, \" \", \" \", \" \", \"PIK3CA\", \"AMPLIFICATION\", \"DNA-based\"], [\"5\", 1253282, 1295162, \" \", \" \", \" \", \"TERT\", \"AMPLIFICATION\", \"DNA-based\"], [\"9\", 135251008, 135282209, \" \", \" \", \" \", \"TTF1\", \"AMPLIFICATION\", \"DNA-based\"], [\"20\", 61867235, 61871859, \" \", \" \", \" \", \"BIRC7\", \"AMPLIFICATION\", \"DNA-based\"], [\"8\", 38268656, 38325363, \" \", \" \", \" \", \"FGFR1\", \"AMPLIFICATION\", \"DNA-based\"], [\"7\", 116312459, 116436396, \" \", \" \", \" \", \"MET\", \"AMPLIFICATION\", \"DNA-based\"], [\"2\", 16080686, 16087129, \" \", \" \", \" \", \"MYCN\", \"AMPLIFICATION\", \"DNA-based\"], [\"17\", 37856333, 37884915, \" \", \" \", \" \", \"ERBB2\", \"AMPLIFICATION\", \"DNA-based\"], [\"22\", 22108789, 22221919, \" \", \" \", \" \", \"MAPK1\", \"AMPLIFICATION\", \"DNA-based\"], [\"9\", 139388896, 139440314, \" \", \" \", \" \", \"NOTCH1\", \"AMPLIFICATION\", \"DNA-based\"], [\"11\", 77371041, 77532063, \" \", \" \", \" \", \"RSF1\", \"AMPLIFICATION\", \"DNA-based\"], [\"20\", 39657458, 39753127, \" \", \" \", \" \", \"TOP1\", \"AMPLIFICATION\", \"DNA-based\"], [\"18\", 657604, 673578, \" \", \" \", \" \", \"TYMS\", \"AMPLIFICATION\", \"DNA-based\"], [\"17\", 48712218, 48769613, \" \", \" \", \" \", \"ABCC3\", \"AMPLIFICATION\", \"DNA-based\"], [\"7\", 97481430, 97501854, \" \", \" \", \" \", \"ASNS\", \"AMPLIFICATION\", \"DNA-based\"], [\"7\", 116312459, 116436396, \" \", \" \", \" \", \"MET\", \"EXON 14 MUTATION + AMPLIFICATION\", \"DNA-based\"], [\"12\", 58141510, 58146304, \" \", \" \", \" \", \"CDK4\", \"AMPLIFICATION\", \"DNA-based\"], [\"5\", 38938021, 39074510, \" \", \" \", \" \", \"RICTOR\", \"AMPLIFICATION\", \"DNA-based\"], [\"4\", 55524085, 55606881, \" \", \" \", \" \", \"KIT\", \"AMPLIFICATION\", \"DNA-based\"], [\"3\", 12625100, 12705725, \" \", \" \", \" \", \"RAF1\", \"AMPLIFICATION\", \"DNA-based\"], [\"12\", 25362365, 25403737, \" \", \" \", \" \", \"KRAS\", \"AMPLIFICATION\", \"DNA-based\"], [\"10\", 123239371, 123357917, \" \", \" \", \" \", \"FGFR2\", \"AMPLIFICATION\", \"DNA-based\"], [\"11\", 69624992, 69633792, \" \", \" \", \" \", \"FGF3\", \"AMPLIFICATION\", \"DNA-based\"], [\"19\", 40736224, 40791302, \" \", \" \", \" \", \"AKT2\", \"AMPLIFICATION\", \"DNA-based\"], [\"4\", 55095264, 55164414, \" \", \" \", \" \", \"PDGFRA\", \"AMPLIFICATION\", \"DNA-based\"], [\"7\", 140434279, 140624564, \" \", \" \", \" \", \"BRAF\", \"AMPLIFICATION\", \"DNA-based\"], [\"7\", 50344378, 50472799, \" \", \" \", \" \", \"IKZF1\", \"DELETION\", \"DNA-based\"], [\"10\", 89622870, 89731687, \" \", \" \", \" \", \"PTEN\", \"DELETION\", \"DNA-based\"], [\"11\", 67351066, 67354131, \" \", \" \", \" \", \"GSTP1\", \"DELETION\", \"DNA-based\"], [\"9\", 22002902, 22009362, \" \", \" \", \" \", \"CDKN2B\", \"LOSS\", \"DNA-based\"], [\"18\", 48556583, 48611409, \" \", \" \", \" \", \"SMAD4\", \"DELETION\", \"DNA-based\"], [\"2\", 140988992, 142889270, \" \", \" \", \" \", \"LRP1B\", \"DELETION\", \"DNA-based\"], [\"22\", 24129150, 24176703, \" \", \" \", \" \", \"SMARCB1\", \"DELETION\", \"DNA-based\"], [\"3\", 10183696, 10183696, \" \", \" \", \" \", \"VHL\", \"E55E (c.165G>A)\", \"DNA-based\"], [\"7\", 55086794, 55279321, \" \", \" \", \" \", \"EGFR\", \"COPY NUMBER VARIATION\", \"DNA-based\"], [\"4\", 106067450, 106200973, \" \", \" \", \" \", \"TET2\", \"MUTATION\", \"DNA-based\"], [\"3\", 10182692, 10193904, \" \", \" \", \" \", \"VHL\", \"MUTATION\", \"DNA-based\"], [\"17\", 41196312, 41277387, \" \", \" \", \" \", \"BRCA1\", \"MUTATION\", \"DNA-based\"], [\"13\", 32889611, 32973347, \" \", \" \", \" \", \"BRCA2\", \"MUTATION\", \"DNA-based\"], [\"19\", 11071598, 11172958, \" \", \" \", \" \", \"SMARCA4\", \"MUTATION\", \"DNA-based\"], [\"9\", 98205262, 98270943, \" \", \" \", \" \", \"PTCH1\", \"MUTATION\", \"DNA-based\"], [\"7\", 140453136, 140481403, \" \", \" \", \" \", \"BRAF\", \"MUTATION\", \"DNA-based\"], [\"2\", 29415640, 30144432, \" \", \" \", \" \", \"ALK\", \"MUTATION\", \"DNA-based\"], [\"17\", 74732402, 74733113, \" \", \" \", \" \", \"SRSF2\", \"MUTATION\", \"DNA-based\"], [\"19\", 1205740, 1223074, \" \", \" \", \" \", \"STK11\", \"MUTATION\", \"DNA-based\"], [\"12\", 133202900, 133256623, \" \", \" \", \" \", \"POLE\", \"MUTATION\", \"DNA-based\"], [\"13\", 48877911, 49056122, \" \", \" \", \" \", \"RB1\", \"MUTATION\", \"DNA-based\"], [\"10\", 89717672, 89717672, \" \", \" \", \" \", \"PTEN\", \"R233*\", \"DNA-based\"], [\"16\", 2130300, 2130300, \" \", \" \", \" \", \"TSC2\", \"Q1178*\", \"DNA-based\"], [\"17\", 33433425, 33433425, \" \", \" \", \" \", \"RAD51D\", \"R186*\", \"DNA-based\"], [\"12\", 25398255, 25398255, \" \", \" \", \" \", \"KRAS\", \"Q22*\", \"DNA-based\"], [\"1\", 65321333, 65321333, \" \", \" \", \" \", \"JAK1\", \"Q503*\", \"DNA-based\"], [\"10\", 89692904, 89692904, \" \", \" \", \" \", \"PTEN\", \"R130*\", \"DNA-based\"], [\"1\", 8073509, 8073509, \" \", \" \", \" \", \"ERRFI1\", \"E384*\", \"DNA-based\"], [\"2\", 47656951, 47656951, \" \", \" \", \" \", \"MSH2\", \"R383*\", \"DNA-based\"], [\"14\", 75745477, 75748127, \" \", \" \", \" \", \"FOS\", \"TRUNCATING FUSION\", \"DNA-based\"], [\"3\", 37042536, 37042536, \" \", \" \", \" \", \"MLH1\", \"R100*\", \"DNA-based\"], [\"3\", 37048546, 37048546, \" \", \" \", \" \", \"MLH1\", \"Q149*\", \"DNA-based\"], [\"3\", 37053589, 37053589, \" \", \" \", \" \", \"MLH1\", \"R226*\", \"DNA-based\"], [\"2\", 47637380, 47637380, \" \", \" \", \" \", \"MSH2\", \"K172*\", \"DNA-based\"], [\"21\", 36171704, 36171704, \" \", \" \", \" \", \"RUNX1\", \"Y260*\", \"DNA-based\"], [\"3\", 10183748, 10183748, \" \", \" \", \" \", \"VHL\", \"Q73* (c.217C>T)\", \"DNA-based\"], [\"3\", 10188261, 10188261, \" \", \" \", \" \", \"VHL\", \"L135* (c.404T>A)\", \"DNA-based\"], [\"3\", 10183811, 10183811, \" \", \" \", \" \", \"VHL\", \"E94* (c.280G>T)\", \"DNA-based\"], [\"3\", 10191562, 10191562, \" \", \" \", \" \", \"VHL\", \"Y185* (c.555C>G)\", \"DNA-based\"], [\"3\", 10191590, 10191590, \" \", \" \", \" \", \"VHL\", \"Q195* (c.583C>T)\", \"DNA-based\"], [\"3\", 10183739, 10183739, \" \", \" \", \" \", \"VHL\", \"E70* (c.208G>T)\", \"DNA-based\"], [\"3\", 10191497, 10191497, \" \", \" \", \" \", \"VHL\", \"Q164* (c.490C>T)\", \"DNA-based\"], [\"3\", 10183795, 10183795, \" \", \" \", \" \", \"VHL\", \"W88* (c.264G>A)\", \"DNA-based\"], [\"3\", 10191593, 10191593, \" \", \" \", \" \", \"VHL\", \"K196* (c.586A>T)\", \"DNA-based\"], [\"3\", 10191532, 10191532, \" \", \" \", \" \", \"VHL\", \"Y175* (c.525C>G)\", \"DNA-based\"], [\"3\", 10188251, 10188251, \" \", \" \", \" \", \"VHL\", \"Q132* (c.394C>T)\", \"DNA-based\"], [\"5\", 1295161, 1295373, \" \", \" \", \" \", \"TERT\", \"PROMOTER MUTATION\", \"DNA-based\"], [\"5\", 1295228, 1295228, \" \", \" \", \" \", \"TERT\", \"C228T\", \"DNA-based\"], [\"14\", 65453063, 65453063, \" \", \" \", \" \", \"FNTB\", \"RS11623866\", \"DNA-based\"], [\"3\", 52581857, 52719852, \" \", \" \", \" \", \"PBRM1\", \"MUTATION\", \"DNA-based\"], [\"11\", 108093559, 108239826, \" \", \" \", \" \", \"ATM\", \"MUTATION\", \"DNA-based\"], [\"7\", 116312459, 116436396, \" \", \" \", \" \", \"MET\", \"MUTATION\", \"DNA-based\"], [\"16\", 68771128, 68869451, \" \", \" \", \" \", \"CDH1\", \"MUTATION\", \"DNA-based\"], [\"3\", 49412898, 49405977, \" \", \" \", \" \", \"RHOA\", \"MUTATION\", \"DNA-based\"], [\"7\", 151832010, 152133090, \" \", \" \", \" \", \"KMT2C\", \"MUTATION\", \"DNA-based\"], [\"9\", 5073770, 5073770, \" \", \" \", \" \", \"JAK2\", \"V617F\", \"DNA-based\"], [\"1\", 115247090, 115259515, \" \", \" \", \" \", \"NRAS\", \"MUTATION\", \"DNA-based\"], [\"1\", 11188519, 11188519, \" \", \" \", \" \", \"MTOR\", \"H1968Y\", \"DNA-based\"], [\"1\", 11184580, 11184580, \" \", \" \", \" \", \"MTOR\", \"P2213S\", \"DNA-based\"], [\"2\", 212295697, 212578308, \" \", \" \", \" \", \"ERBB4\", \"MUTATION\", \"DNA-based\"], [\"3\", 178866311, 178957881, \" \", \" \", \" \", \"PIK3CA\", \"MUTATION\", \"DNA-based\"], [\"22\", 22127164, 22127164, \" \", \" \", \" \", \"MAPK1\", \"E322K\", \"DNA-based\"], [\"4\", 55524085, 55606881, \" \", \" \", \" \", \"KIT\", \"MUTATION\", \"DNA-based\"], [\"3\", 38182641, 38182641, \" \", \" \", \" \", \"MYD88\", \"L265P\", \"DNA-based\"], [\"10\", 123239371, 123357917, \" \", \" \", \" \", \"FGFR2\", \"MUTATION\", \"DNA-based\"], [\"13\", 28577411, 28674729, \" \", \" \", \" \", \"FLT3\", \"MUTATION\", \"DNA-based\"], [\"10\", 123258034, 123258034, \" \", \" \", \" \", \"FGFR2\", \"N550K\", \"DNA-based\"], [\"5\", 67511548, 67597649, \" \", \" \", \" \", \"PIK3R1\", \"MUTATION\", \"DNA-based\"], [\"3\", 41240930, 41281936, \" \", \" \", \" \", \"CTNNB1\", \"ACTIVATING MUTATION\", \"DNA-based\"], [\"6\", 41902671, 41909586, \" \", \" \", \" \", \"CCND3\", \"LOSS\", \"DNA-based\"], [\"19\", 33790840, 33793470, \" \", \" \", \" \", \"CEBPA\", \"MUTATION\", \"DNA-based\"], [\"9\", 135766735, 135820008, \" \", \" \", \" \", \"TSC1\", \"LOSS-OF-FUNCTION\", \"DNA-based\"], [\"17\", 41196312, 41277387, \" \", \" \", \" \", \"BRCA1\", \"LOSS-OF-FUNCTION\", \"DNA-based\"], [\"13\", 32889611, 32973347, \" \", \" \", \" \", \"BRCA2\", \"LOSS-OF-FUNCTION\", \"DNA-based\"], [\"21\", 36160098, 36421641, \" \", \" \", \" \", \"RUNX1\", \"MUTATION\", \"DNA-based\"], [\"3\", 10182692, 10193904, \" \", \" \", \" \", \"VHL\", \"LOSS-OF-FUNCTION\", \"DNA-based\"], [\"12\", 49412758, 49449107, \" \", \" \", \" \", \"KMT2D\", \"LOSS-OF-FUNCTION\", \"DNA-based\"], [\"3\", 52435029, 52444366, \" \", \" \", \" \", \"BAP1\", \"MUTATION\", \"DNA-based\"], [\"10\", 89622870, 89731687, \" \", \" \", \" \", \"PTEN\", \"LOSS\", \"DNA-based\"], [\"18\", 48556583, 48611409, \" \", \" \", \" \", \"SMAD4\", \"MUTATION\", \"DNA-based\"], [\"19\", 11071598, 11172958, \" \", \" \", \" \", \"SMARCA4\", \"INACTIVATING MUTATION\", \"DNA-based\"], [\"22\", 29083732, 29137832, \" \", \" \", \" \", \"CHEK2\", \"LOSS-OF-FUNCTION\", \"DNA-based\"], [\"17\", 37618292, 37691399, \" \", \" \", \" \", \"CDK12\", \"LOSS-OF-FUNCTION\", \"DNA-based\"], [\"4\", 153245446, 153332714, \" \", \" \", \" \", \"FBXW7\", \"MUTATION\", \"DNA-based\"], [\"9\", 8314246, 10033790, \" \", \" \", \" \", \"PTPRD\", \"MUTATION\", \"DNA-based\"], [\"11\", 94152895, 94227074, \" \", \" \", \" \", \"MRE11\", \"LOSS\", \"DNA-based\"], [\"3\", 10182692, 10193904, \" \", \" \", \" \", \"VHL\", \"LOSS\", \"DNA-based\"], [\"19\", 1205740, 1228428, \" \", \" \", \" \", \"STK11\", \"LOSS\", \"DNA-based\"], [\"16\", 23614491, 23652631, \" \", \" \", \" \", \"PALB2\", \"BIALLELIC INACTIVATION\", \"DNA-based\"], [\"9\", 97861336, 98079984, \" \", \" \", \" \", \"FANCC\", \"LOSS-OF-FUNCTION\", \"DNA-based\"], [\"9\", 21967752, 21994623, \" \", \" \", \" \", \"CDKN2A\", \"LOSS\", \"DNA-based\"], [\"18\", 48556583, 48611409, \" \", \" \", \" \", \"SMAD4\", \"LOSS\", \"DNA-based\"], [\"7\", 140477859, 140477860, \" \", \" \", \" \", \"BRAF\", \"K483M\", \"DNA-based\"], [\"17\", 29421945, 29704695, \" \", \" \", \" \", \"NF1\", \"MUTATION\", \"DNA-based\"], [\"19\", 33790840, 33793470, \" \", \" \", \" \", \"CEBPA\", \"BIALLELIC INACTIVATION\", \"DNA-based\"], [\"9\", 139388896, 139440314, \" \", \" \", \" \", \"NOTCH1\", \"LOSS-OF-FUNCTION\", \"DNA-based\"], [\"13\", 48877911, 49056122, \" \", \" \", \" \", \"RB1\", \"LOSS-OF-FUNCTION\", \"DNA-based\"], [\"7\", 124462440, 124570035, \" \", \" \", \" \", \"POT1\", \"MUTATION\", \"DNA-based\"], [\"4\", 153242410, 153457253, \" \", \" \", \" \", \"FBXW7\", \"LOSS-OF-FUNCTION\", \"DNA-based\"], [\"6\", 106534195, 106557813, \" \", \" \", \" \", \"PRDM1\", \"MUTATION\", \"DNA-based\"], [\"22\", 38368307, 38380544, \" \", \" \", \" \", \"SOX10\", \"LOSS\", \"DNA-based\"], [\"22\", 29999547, 30094587, \" \", \" \", \" \", \"NF2\", \"LOSS\", \"DNA-based\"], [\"17\", 29421945, 29704695, \" \", \" \", \" \", \"NF1\", \"LOSS\", \"DNA-based\"], [\"22\", 42522501, 42526908, \" \", \" \", \" \", \"CYP2D6\", \"LOSS-OF-FUNCTION\", \"DNA-based\"], [\"21\", 36231792, 36231792, \" \", \" \", \" \", \"RUNX1\", \"D198Y\", \"DNA-based\"], [\"2\", 47630108, 47710367, \" \", \" \", \" \", \"MSH2\", \"LOSS\", \"DNA-based\"], [\"2\", 48010221, 48037240, \" \", \" \", \" \", \"MSH6\", \"LOSS\", \"DNA-based\"], [\"12\", 70910630, 71031220, \" \", \" \", \" \", \"PTPRB\", \"LOSS-OF-FUNCTION\", \"DNA-based\"], [\"19\", 33792961, 33793470, \" \", \" \", \" \", \"CEBPA\", \"N-TERMINAL FRAME SHIFT\", \"DNA-based\"], [\"9\", 98205262, 98270943, \" \", \" \", \" \", \"PTCH1\", \"LOH\", \"DNA-based\"], [\"19\", 1220489, 1220489, \" \", \" \", \" \", \"STK11\", \"D194E\", \"DNA-based\"], [\"17\", 37881011, 37881012, \" \", \" \", \" \", \"ERBB2\", \"P780INS\", \"DNA-based\"], [\"13\", 28608219, 28608351, \" \", \" \", \" \", \"FLT3\", \"ITD\", \"DNA-based\"], [\"4\", 55593582, 55593708, \" \", \" \", \" \", \"KIT\", \"INTERNAL DUPLICATION\", \"DNA-based\"], [\"12\", 25398281, 25398282, \" \", \" \", \" \", \"KRAS\", \"G13\", \"DNA-based\"], [\"2\", 29445271, 29445272, \" \", \" \", \" \", \"ALK\", \"EML4-ALK T1151INST\", \"DNA-based\"], [\"17\", 37880993, 37880994, \" \", \" \", \" \", \"ERBB2\", \"M774INSAYVM\", \"DNA-based\"], [\"7\", 55248991, 55249022, \" \", \" \", \" \", \"EGFR\", \"EXON 20 INSERTION\", \"DNA-based\"], [\"17\", 37880996, 37880997, \" \", \" \", \" \", \"ERBB2\", \"Y772_A775DUP\", \"DNA-based\"], [\"17\", 37881011, 37881012, \" \", \" \", \" \", \"ERBB2\", \"G778_P780DUP\", \"DNA-based\"], [\"3\", 10188232, 10188233, \" \", \" \", \" \", \"VHL\", \"D126FS (c.375_376insC)\", \"DNA-based\"], [\"5\", 1286516, 1286516, \" \", \" \", \" \", \"TERT\", \"RS2736100\", \"DNA-based\"], [\"12\", 69202580, 69202580, \" \", \" \", \" \", \"MDM2\", \"SNP309\", \"DNA-based\"], [\"1\", 68152388, 68152388, \" \", \" \", \" \", \"GADD45A\", \"rs681673\", \"DNA-based\"], [\"7\", 6022512, 6022512, \" \", \" \", \" \", \"PMS2\", \"K706FS*19\", \"DNA-based\"], [\"16\", 23646276, 23646276, \" \", \" \", \" \", \"PALB2\", \"L531FS*30\", \"DNA-based\"], [\"7\", 140453136, 140453137, \" \", \" \", \" \", \"BRAF\", \"V600\", \"DNA-based\"], [\"2\", 209113111, 209113113, \" \", \" \", \" \", \"IDH1\", \"R132\", \"DNA-based\"], [\"15\", 90631934, 90631934, \" \", \" \", \" \", \"IDH2\", \"R140\", \"DNA-based\"], [\"12\", 25398284, 25398285, \" \", \" \", \" \", \"KRAS\", \"G12\", \"DNA-based\"], [\"12\", 25398280, 25398285, \" \", \" \", \" \", \"KRAS\", \"G12/G13\", \"DNA-based\"], [\"17\", 7577535, 7577535, \" \", \" \", \" \", \"TP53\", \"R249\", \"DNA-based\"], [\"7\", 148504477, 148581370, \" \", \" \", \" \", \"EZH2\", \"MUTATION\", \"DNA-based\"], [\"7\", 148508726, 148508728, \" \", \" \", \" \", \"EZH2\", \"Y646\", \"DNA-based\"], [\"20\", 30946155, 31027122, \" \", \" \", \" \", \"ASXL1\", \"MUTATION\", \"DNA-based\"], [\"15\", 90631837, 90631839, \" \", \" \", \" \", \"IDH2\", \"R172\", \"DNA-based\"], [\"12\", 25380275, 25380277, \" \", \" \", \" \", \"KRAS\", \"Q61\", \"DNA-based\"], [\"9\", 139388896, 139440314, \" \", \" \", \" \", \"NOTCH1\", \"MUTATION\", \"DNA-based\"], [\"2\", 198254508, 198299815, \" \", \" \", \" \", \"SF3B1\", \"MUTATION\", \"DNA-based\"], [\"17\", 7571720, 7590856, \" \", \" \", \" \", \"TP53\", \"MUTATION\", \"DNA-based\"], [\"1\", 11166592, 11322564, \" \", \" \", \" \", \"MTOR\", \"MUTATION\", \"DNA-based\"], [\"12\", 25362365, 25403737, \" \", \" \", \" \", \"KRAS\", \"MUTATION\", \"DNA-based\"], [\"17\", 40465342, 40475372, \" \", \" \", \" \", \"STAT3\", \"SH2 DOMAIN MUTATION\", \"DNA-based\"], [\"17\", 37879863, 37882870, \" \", \" \", \" \", \"ERBB2\", \"KINASE DOMAIN MUTATION\", \"DNA-based\"], [\"13\", 28592642, 28592642, \" \", \" \", \" \", \"FLT3\", \"D835\", \"DNA-based\"], [\"9\", 80409487, 80409489, \" \", \" \", \" \", \"GNAQ\", \"Q209\", \"DNA-based\"], [\"10\", 89622870, 89731687, \" \", \" \", \" \", \"PTEN\", \"MUTATION\", \"DNA-based\"], [\"21\", 44513066, 44527697, \" \", \" \", \" \", \"U2AF1\", \"MUTATION\", \"DNA-based\"], [\"1\", 36932096, 36933444, \" \", \" \", \" \", \"CSF3R\", \"MUTATION\", \"DNA-based\"], [\"15\", 90626277, 90645736, \" \", \" \", \" \", \"IDH2\", \"MUTATION\", \"DNA-based\"], [\"1\", 115258744, 115258748, \" \", \" \", \" \", \"NRAS\", \"G12/G13\", \"DNA-based\"], [\"X\", 123176418, 123217308, \" \", \" \", \" \", \"STAG2\", \"MUTATION\", \"DNA-based\"], [\"7\", 55241707, 55241708, \" \", \" \", \" \", \"EGFR\", \"G719\", \"DNA-based\"], [\"7\", 55241710, 55241711, \" \", \" \", \" \", \"EGFR\", \"S720\", \"DNA-based\"], [\"9\", 133748283, 133748283, \" \", \" \", \" \", \"ABL1\", \"BCR-ABL T334I\", \"DNA-based\"], [\"9\", 133738363, 133738363, \" \", \" \", \" \", \"ABL1\", \"BCR-ABL E255K\", \"DNA-based\"], [\"14\", 105246551, 105246551, \" \", \" \", \" \", \"AKT1\", \"E17K\", \"DNA-based\"], [\"2\", 29445258, 29445258, \" \", \" \", \" \", \"ALK\", \"EML4-ALK C1156Y\", \"DNA-based\"], [\"2\", 29443631, 29443631, \" \", \" \", \" \", \"ALK\", \"EML4-ALK L1196M\", \"DNA-based\"], [\"2\", 29443695, 29443695, \" \", \" \", \" \", \"ALK\", \"F1174L\", \"DNA-based\"], [\"2\", 29432664, 29432664, \" \", \" \", \" \", \"ALK\", \"R1275Q\", \"DNA-based\"], [\"X\", 47426121, 47426121, \" \", \" \", \" \", \"ARAF\", \"S214C\", \"DNA-based\"], [\"7\", 140453135, 140453136, \" \", \" \", \" \", \"BRAF\", \"V600D\", \"DNA-based\"], [\"7\", 140453136, 140453136, \" \", \" \", \" \", \"BRAF\", \"V600E\", \"DNA-based\"], [\"7\", 140453135, 140453137, \" \", \" \", \" \", \"BRAF\", \"V600E+V600M\", \"DNA-based\"], [\"2\", 25457241, 25457243, \" \", \" \", \" \", \"DNMT3A\", \"R882\", \"DNA-based\"], [\"7\", 55259515, 55259515, \" \", \" \", \" \", \"EGFR\", \"L858R\", \"DNA-based\"], [\"7\", 55249071, 55249071, \" \", \" \", \" \", \"EGFR\", \"T790M\", \"DNA-based\"], [\"17\", 37880261, 37880261, \" \", \" \", \" \", \"ERBB2\", \"D769H\", \"DNA-based\"], [\"17\", 37880261, 37880261, \" \", \" \", \" \", \"ERBB2\", \"D769Y\", \"DNA-based\"], [\"17\", 37868205, 37868205, \" \", \" \", \" \", \"ERBB2\", \"G309A\", \"DNA-based\"], [\"17\", 37880220, 37880220, \" \", \" \", \" \", \"ERBB2\", \"L755S\", \"DNA-based\"], [\"17\", 37880220, 37880220, \" \", \" \", \" \", \"ERBB2\", \"L755W\", \"DNA-based\"], [\"17\", 37879658, 37879658, \" \", \" \", \" \", \"ERBB2\", \"R678Q\", \"DNA-based\"], [\"17\", 37881616, 37881616, \" \", \" \", \" \", \"ERBB2\", \"R896C\", \"DNA-based\"], [\"17\", 37881000, 37881000, \" \", \" \", \" \", \"ERBB2\", \"V777L\", \"DNA-based\"], [\"17\", 37881332, 37881332, \" \", \" \", \" \", \"ERBB2\", \"V842I\", \"DNA-based\"], [\"6\", 152419920, 152419921, \" \", \" \", \" \", \"ESR1\", \"L536Q\", \"DNA-based\"], [\"6\", 152419926, 152419926, \" \", \" \", \" \", \"ESR1\", \"D538G\", \"DNA-based\"], [\"6\", 152419923, 152419923, \" \", \" \", \" \", \"ESR1\", \"Y537C\", \"DNA-based\"], [\"6\", 152419922, 152419922, \" \", \" \", \" \", \"ESR1\", \"Y537N\", \"DNA-based\"], [\"6\", 152419923, 152419923, \" \", \" \", \" \", \"ESR1\", \"Y537S\", \"DNA-based\"], [\"2\", 209113113, 209113113, \" \", \" \", \" \", \"IDH1\", \"R132C\", \"DNA-based\"], [\"15\", 90631838, 90631838, \" \", \" \", \" \", \"IDH2\", \"R172K\", \"DNA-based\"], [\"4\", 55599321, 55599321, \" \", \" \", \" \", \"KIT\", \"D816V\", \"DNA-based\"], [\"4\", 55593661, 55593661, \" \", \" \", \" \", \"KIT\", \"L576P\", \"DNA-based\"], [\"4\", 55594258, 55594258, \" \", \" \", \" \", \"KIT\", \"V654A\", \"DNA-based\"], [\"12\", 25398208, 25398329, \" \", \" \", \" \", \"KRAS\", \"EXON 2 MUTATION\", \"DNA-based\"], [\"12\", 25398285, 25398285, \" \", \" \", \" \", \"KRAS\", \"G12C\", \"DNA-based\"], [\"12\", 25398284, 25398284, \" \", \" \", \" \", \"KRAS\", \"G12D\", \"DNA-based\"], [\"12\", 25398281, 25398281, \" \", \" \", \" \", \"KRAS\", \"G13D\", \"DNA-based\"], [\"15\", 66729162, 66729162, \" \", \" \", \" \", \"MAP2K1\", \"P124S\", \"DNA-based\"], [\"15\", 66727451, 66727451, \" \", \" \", \" \", \"MAP2K1\", \"Q56P\", \"DNA-based\"], [\"1\", 115258747, 115258748, \" \", \" \", \" \", \"NRAS\", \"G12\", \"DNA-based\"], [\"1\", 115258744, 115258744, \" \", \" \", \" \", \"NRAS\", \"G13D\", \"DNA-based\"], [\"1\", 115256528, 115256530, \" \", \" \", \" \", \"NRAS\", \"Q61\", \"DNA-based\"], [\"1\", 115256529, 115256529, \" \", \" \", \" \", \"NRAS\", \"Q61L\", \"DNA-based\"], [\"1\", 115256529, 115256529, \" \", \" \", \" \", \"NRAS\", \"Q61R\", \"DNA-based\"], [\"4\", 55152092, 55152093, \" \", \" \", \" \", \"PDGFRA\", \"D842I\", \"DNA-based\"], [\"4\", 55152093, 55152093, \" \", \" \", \" \", \"PDGFRA\", \"D842V\", \"DNA-based\"], [\"4\", 55152092, 55152092, \" \", \" \", \" \", \"PDGFRA\", \"D842Y\", \"DNA-based\"], [\"4\", 55152093, 55152097, \" \", \" \", \" \", \"PDGFRA\", \"DI842-843VM\", \"DNA-based\"], [\"3\", 178936082, 178936082, \" \", \" \", \" \", \"PIK3CA\", \"E542K\", \"DNA-based\"], [\"3\", 178936091, 178936091, \" \", \" \", \" \", \"PIK3CA\", \"E545K\", \"DNA-based\"], [\"3\", 178952085, 178952085, \" \", \" \", \" \", \"PIK3CA\", \"H1047R\", \"DNA-based\"], [\"10\", 43609950, 43609950, \" \", \" \", \" \", \"RET\", \"C634W\", \"DNA-based\"], [\"10\", 43617416, 43617416, \" \", \" \", \" \", \"RET\", \"M918T\", \"DNA-based\"], [\"2\", 198267359, 198267359, \" \", \" \", \" \", \"SF3B1\", \"K666N\", \"DNA-based\"], [\"17\", 7578406, 7578406, \" \", \" \", \" \", \"TP53\", \"R175H\", \"DNA-based\"], [\"17\", 7577538, 7577538, \" \", \" \", \" \", \"TP53\", \"R248Q\", \"DNA-based\"], [\"17\", 7577539, 7577539, \" \", \" \", \" \", \"TP53\", \"R248W\", \"DNA-based\"], [\"17\", 7577121, 7577121, \" \", \" \", \" \", \"TP53\", \"R273C\", \"DNA-based\"], [\"17\", 7577120, 7577120, \" \", \" \", \" \", \"TP53\", \"R273H\", \"DNA-based\"], [\"21\", 44514777, 44514777, \" \", \" \", \" \", \"U2AF1\", \"Q157P/R\", \"DNA-based\"], [\"21\", 44524456, 44524456, \" \", \" \", \" \", \"U2AF1\", \"S34Y/F\", \"DNA-based\"], [\"7\", 55241707, 55241707, \" \", \" \", \" \", \"EGFR\", \"G719S\", \"DNA-based\"], [\"9\", 139399219, 139399219, \" \", \" \", \" \", \"NOTCH1\", \"D1642H\", \"DNA-based\"], [\"9\", 139391212, 139391212, \" \", \" \", \" \", \"NOTCH1\", \"R2327W\", \"DNA-based\"], [\"1\", 162724415, 162724415, \" \", \" \", \" \", \"DDR2\", \"L63V\", \"DNA-based\"], [\"1\", 162729630, 162729630, \" \", \" \", \" \", \"DDR2\", \"L239R\", \"DNA-based\"], [\"1\", 162729671, 162729671, \" \", \" \", \" \", \"DDR2\", \"G253C\", \"DNA-based\"], [\"1\", 162741822, 162741822, \" \", \" \", \" \", \"DDR2\", \"G505S\", \"DNA-based\"], [\"1\", 162745497, 162745497, \" \", \" \", \" \", \"DDR2\", \"I638F\", \"DNA-based\"], [\"1\", 162748407, 162748407, \" \", \" \", \" \", \"DDR2\", \"G774V\", \"DNA-based\"], [\"1\", 162748390, 162748390, \" \", \" \", \" \", \"DDR2\", \"S768R\", \"DNA-based\"], [\"12\", 25398284, 25398284, \" \", \" \", \" \", \"KRAS\", \"G12A\", \"DNA-based\"], [\"X\", 100611165, 100611165, \" \", \" \", \" \", \"BTK\", \"C481S\", \"DNA-based\"], [\"14\", 105243048, 105243048, \" \", \" \", \" \", \"AKT1\", \"Q79K\", \"DNA-based\"], [\"2\", 29443613, 29443613, \" \", \" \", \" \", \"ALK\", \"ALK FUSION G1202R\", \"DNA-based\"], [\"2\", 29443600, 29443600, \" \", \" \", \" \", \"ALK\", \"EML4-ALK S1206Y\", \"DNA-based\"], [\"X\", 66943549, 66943549, \" \", \" \", \" \", \"AR\", \"F877L\", \"DNA-based\"], [\"X\", 66937370, 66937372, \" \", \" \", \" \", \"AR\", \"W742\", \"DNA-based\"], [\"4\", 1806099, 1806099, \" \", \" \", \" \", \"FGFR3\", \"Y375C\", \"DNA-based\"], [\"3\", 138665163, 138665163, \" \", \" \", \" \", \"FOXL2\", \"C134W\", \"DNA-based\"], [\"4\", 55593464, 55593464, \" \", \" \", \" \", \"KIT\", \"M541L\", \"DNA-based\"], [\"10\", 104852955, 104852955, \" \", \" \", \" \", \"NT5C2\", \"R367Q\", \"DNA-based\"], [\"10\", 104852980, 104852980, \" \", \" \", \" \", \"NT5C2\", \"K359Q\", \"DNA-based\"], [\"10\", 104850745, 104850745, \" \", \" \", \" \", \"NT5C2\", \"D407A\", \"DNA-based\"], [\"9\", 133748288, 133748288, \" \", \" \", \" \", \"ABL1\", \"BCR-ABL F317L\", \"DNA-based\"], [\"11\", 108218044, 108218044, \" \", \" \", \" \", \"ATM\", \"N2875H\", \"DNA-based\"], [\"4\", 55141036, 55141036, \" \", \" \", \" \", \"PDGFRA\", \"V561A\", \"DNA-based\"], [\"19\", 44057574, 44057574, \" \", \" \", \" \", \"XRCC1\", \"R194W\", \"DNA-based\"], [\"1\", 11856378, 11856378, \" \", \" \", \" \", \"MTHFR\", \"A222V\", \"DNA-based\"], [\"11\", 67352689, 67352689, \" \", \" \", \" \", \"GSTP1\", \"I105V\", \"DNA-based\"], [\"4\", 89052323, 89052323, \" \", \" \", \" \", \"ABCG2\", \"Q141K\", \"DNA-based\"], [\"19\", 44055726, 44055726, \" \", \" \", \" \", \"XRCC1\", \"Q399R\", \"DNA-based\"], [\"7\", 87160618, 87160618, \" \", \" \", \" \", \"ABCB1\", \"S893T\", \"DNA-based\"], [\"19\", 45854919, 45854919, \" \", \" \", \" \", \"ERCC2\", \"K751Q\", \"DNA-based\"], [\"11\", 534285, 534285, \" \", \" \", \" \", \"HRAS\", \"G13D\", \"DNA-based\"], [\"7\", 140453145, 140453145, \" \", \" \", \" \", \"BRAF\", \"L597R\", \"DNA-based\"], [\"3\", 178928226, 178928226, \" \", \" \", \" \", \"PIK3CA\", \"P471L\", \"DNA-based\"], [\"7\", 128849189, 128849189, \" \", \" \", \" \", \"SMO\", \"D473H\", \"DNA-based\"], [\"2\", 29445270, 29445270, \" \", \" \", \" \", \"ALK\", \"EML4-ALK L1152R\", \"DNA-based\"], [\"2\", 29432682, 29432682, \" \", \" \", \" \", \"ALK\", \"EML4-ALK G1269A\", \"DNA-based\"], [\"7\", 55241722, 55241722, \" \", \" \", \" \", \"EGFR\", \"G724S\", \"DNA-based\"], [\"12\", 25378561, 25378561, \" \", \" \", \" \", \"KRAS\", \"A146V\", \"DNA-based\"], [\"18\", 42531913, 42531913, \" \", \" \", \" \", \"SETBP1\", \"G870S\", \"DNA-based\"], [\"2\", 29443625, 29445258, \" \", \" \", \" \", \"ALK\", \"EML4-ALK C1156Y–L1198F\", \"DNA-based\"], [\"7\", 6426892, 6426892, \" \", \" \", \" \", \"RAC1\", \"P29S\", \"DNA-based\"], [\"16\", 69745145, 69745145, \" \", \" \", \" \", \"NQO1\", \"P187S\", \"DNA-based\"], [\"7\", 55249091, 55249091, \" \", \" \", \" \", \"EGFR\", \"C797S\", \"DNA-based\"], [\"2\", 209113112, 209113112, \" \", \" \", \" \", \"IDH1\", \"R132H\", \"DNA-based\"], [\"12\", 25398284, 25398284, \" \", \" \", \" \", \"KRAS\", \"G12V\", \"DNA-based\"], [\"1\", 115256530, 115256530, \" \", \" \", \" \", \"NRAS\", \"Q61K\", \"DNA-based\"], [\"7\", 55227926, 55227926, \" \", \" \", \" \", \"EGFR\", \"G465R\", \"DNA-based\"], [\"7\", 87160618, 87160618, \" \", \" \", \" \", \"ABCB1\", \"S893A/T\", \"DNA-based\"], [\"7\", 55228009, 55228009, \" \", \" \", \" \", \"EGFR\", \"S492R\", \"DNA-based\"], [\"7\", 55227884, 55227884, \" \", \" \", \" \", \"EGFR\", \"R451C\", \"DNA-based\"], [\"7\", 55227933, 55227933, \" \", \" \", \" \", \"EGFR\", \"K467T\", \"DNA-based\"], [\"1\", 161479745, 161479745, \" \", \" \", \" \", \"FCGR2A\", \"H167R\", \"DNA-based\"], [\"1\", 161514542, 161514542, \" \", \" \", \" \", \"FCGR3A\", \"F212V\", \"DNA-based\"], [\"1\", 161643798, 161643798, \" \", \" \", \" \", \"FCGR2B\", \"I232T\", \"DNA-based\"], [\"7\", 55242487, 55242487, \" \", \" \", \" \", \"EGFR\", \"P753S\", \"DNA-based\"], [\"15\", 74315213, 74315219, \" \", \" \", \" \", \"PML\", \"B2 DOMAIN MUTATION\", \"DNA-based\"], [\"15\", 74315213, 74315213, \" \", \" \", \" \", \"PML\", \"PML-RARA A216V\", \"DNA-based\"], [\"15\", 74315219, 74315219, \" \", \" \", \" \", \"PML\", \"PML-RARA L218P\", \"DNA-based\"], [\"6\", 117638347, 117638347, \" \", \" \", \" \", \"ROS1\", \"CD74-ROS1 G2032R\", \"DNA-based\"], [\"4\", 55955969, 55955969, \" \", \" \", \" \", \"KDR\", \"A1065T\", \"DNA-based\"], [\"4\", 55968180, 55968180, \" \", \" \", \" \", \"KDR\", \"D717V\", \"DNA-based\"], [\"1\", 11187094, 11187094, \" \", \" \", \" \", \"MTOR\", \"F2108L\", \"DNA-based\"], [\"17\", 37881404, 37881404, \" \", \" \", \" \", \"ERBB2\", \"L866M\", \"DNA-based\"], [\"17\", 37868208, 37868208, \" \", \" \", \" \", \"ERBB2\", \"S310F/Y\", \"DNA-based\"], [\"4\", 55961059, 55961059, \" \", \" \", \" \", \"KDR\", \"R961W\", \"DNA-based\"], [\"17\", 7579548, 7579548, \" \", \" \", \" \", \"TP53\", \"P47S\", \"DNA-based\"], [\"8\", 38274849, 38274849, \" \", \" \", \" \", \"FGFR1\", \"N546K\", \"DNA-based\"], [\"6\", 117638365, 117638365, \" \", \" \", \" \", \"ROS1\", \"CD74-ROS1 L2026M\", \"DNA-based\"], [\"6\", 117630062, 117630062, \" \", \" \", \" \", \"ROS1\", \"CD74-ROS1 L2155S\", \"DNA-based\"], [\"6\", 117631376, 117631376, \" \", \" \", \" \", \"ROS1\", \"CD74-ROS1 G2101A\", \"DNA-based\"], [\"2\", 29445213, 29445213, \" \", \" \", \" \", \"ALK\", \"ALK FUSION I1171\", \"DNA-based\"], [\"2\", 29443679, 29443679, \" \", \" \", \" \", \"ALK\", \"EML4-ALK  V1180L\", \"DNA-based\"], [\"12\", 25398285, 25398285, \" \", \" \", \" \", \"KRAS\", \"G12R\", \"DNA-based\"], [\"17\", 7579472, 7579472, \" \", \" \", \" \", \"TP53\", \"P72R\", \"DNA-based\"], [\"13\", 28624294, 28624294, \" \", \" \", \" \", \"FLT3\", \"T227M\", \"DNA-based\"], [\"1\", 11184573, 11184573, \" \", \" \", \" \", \"MTOR\", \"S2215Y\", \"DNA-based\"], [\"1\", 11217230, 11217230, \" \", \" \", \" \", \"MTOR\", \"C1483Y\", \"DNA-based\"], [\"1\", 11190804, 11190804, \" \", \" \", \" \", \"MTOR\", \"E1799K\", \"DNA-based\"], [\"19\", 49377242, 49377242, \" \", \" \", \" \", \"PPP1R15A\", \"RS557806\", \"DNA-based\"], [\"2\", 29436859, 29436859, \" \", \" \", \" \", \"ALK\", \"F1245C\", \"DNA-based\"], [\"2\", 29436859, 29436859, \" \", \" \", \" \", \"ALK\", \"ALK FUSION F1245C\", \"DNA-based\"], [\"2\", 29432682, 29432682, \" \", \" \", \" \", \"ALK\", \"ALK FUSION G1269A\", \"DNA-based\"], [\"12\", 58145431, 58145431, \" \", \" \", \" \", \"CDK4\", \"R24C\", \"DNA-based\"], [\"19\", 3118942, 3118942, \" \", \" \", \" \", \"GNA11\", \"Q209\", \"DNA-based\"], [\"7\", 55249005, 55249005, \" \", \" \", \" \", \"EGFR\", \"S768I\", \"DNA-based\"], [\"7\", 140453136, 140453137, \" \", \" \", \" \", \"BRAF\", \"V600K\", \"DNA-based\"], [\"2\", 198266834, 198266834, \" \", \" \", \" \", \"SF3B1\", \"K700E\", \"DNA-based\"], [\"19\", 7975157, 7975157, \" \", \" \", \" \", \"MAP2K7\", \"E116K\", \"DNA-based\"], [\"1\", 11174420, 11187857, \" \", \" \", \" \", \"MTOR\", \"E2014K + E2419K\", \"DNA-based\"], [\"4\", 55144547, 55144547, \" \", \" \", \" \", \"PDGFRA\", \"FIP1L1-PDGFRA T674I\", \"DNA-based\"], [\"7\", 140453154, 140453154, \" \", \" \", \" \", \"BRAF\", \"D594A\", \"DNA-based\"], [\"7\", 140453154, 140453154, \" \", \" \", \" \", \"BRAF\", \"D594V\", \"DNA-based\"], [\"7\", 140453145, 140453146, \" \", \" \", \" \", \"BRAF\", \"L597S\", \"DNA-based\"], [\"7\", 140453145, 140453145, \" \", \" \", \" \", \"BRAF\", \"L597Q\", \"DNA-based\"], [\"7\", 140453134, 140453134, \" \", \" \", \" \", \"BRAF\", \"K601E\", \"DNA-based\"], [\"7\", 140453146, 140453146, \" \", \" \", \" \", \"BRAF\", \"L597V\", \"DNA-based\"], [\"2\", 29445213, 29445213, \" \", \" \", \" \", \"ALK\", \"HIP1-ALK I1171N\", \"DNA-based\"], [\"2\", 29445213, 29445213, \" \", \" \", \" \", \"ALK\", \"EML4-ALK I1171S\", \"DNA-based\"], [\"9\", 80409488, 80409488, \" \", \" \", \" \", \"GNAQ\", \"Q209P\", \"DNA-based\"], [\"2\", 29443631, 29445258, \" \", \" \", \" \", \"ALK\", \"EML4-ALK C1156Y–L1196M\", \"DNA-based\"], [\"1\", 11177096, 11177096, \" \", \" \", \" \", \"MTOR\", \"M2327I\", \"DNA-based\"], [\"1\", 11187796, 11187796, \" \", \" \", \" \", \"MTOR\", \"A2034V\", \"DNA-based\"], [\"7\", 140453154, 140453154, \" \", \" \", \" \", \"BRAF\", \"D594G\", \"DNA-based\"], [\"13\", 28592642, 28592642, \" \", \" \", \" \", \"FLT3\", \"D835H\", \"DNA-based\"], [\"13\", 28592642, 28592642, \" \", \" \", \" \", \"FLT3\", \"D835H/Y\", \"DNA-based\"], [\"15\", 66729154, 66729154, \" \", \" \", \" \", \"MAP2K1\", \"C121S\", \"DNA-based\"], [\"4\", 1803568, 1803568, \" \", \" \", \" \", \"FGFR3\", \"S249C\", \"DNA-based\"], [\"3\", 10191605, 10191605, \" \", \" \", \" \", \"VHL\", \"R200W (c.598C>T)\", \"DNA-based\"], [\"7\", 116423407, 116423407, \" \", \" \", \" \", \"MET\", \"D1228N\", \"DNA-based\"], [\"2\", 202149589, 202149589, \" \", \" \", \" \", \"CASP8\", \"D302H\", \"DNA-based\"], [\"7\", 140477794, 140477794, \" \", \" \", \" \", \"BRAF\", \"L505H\", \"DNA-based\"], [\"13\", 32968854, 32968854, \" \", \" \", \" \", \"BRCA2\", \"D3095E\", \"DNA-based\"], [\"17\", 37868196, 37881332, \" \", \" \", \" \", \"ERBB2\", \"MUTATION\", \"DNA-based\"], [\"6\", 152415537, 152415537, \" \", \" \", \" \", \"ESR1\", \"S463P\", \"DNA-based\"], [\"12\", 111884608, 111884608, \" \", \" \", \" \", \"SH2B3\", \"RS3184504\", \"DNA-based\"], [\"12\", 56478854, 56478854, \" \", \" \", \" \", \"ERBB3\", \"V104M\", \"DNA-based\"], [\"12\", 56478851, 56478851, \" \", \" \", \" \", \"ERBB3\", \"R103G\", \"DNA-based\"], [\"12\", 56481922, 56481922, \" \", \" \", \" \", \"ERBB3\", \"G284R\", \"DNA-based\"], [\"7\", 55242511, 55242511, \" \", \" \", \" \", \"EGFR\", \"D761Y\", \"DNA-based\"], [\"9\", 135771933, 135771933, \" \", \" \", \" \", \"TSC1\", \"R1062W\", \"DNA-based\"], [\"7\", 55242500, 55242500, \" \", \" \", \" \", \"EGFR\", \"K757R\", \"DNA-based\"], [\"7\", 55242467, 55242467, \" \", \" \", \" \", \"EGFR\", \"E746G\", \"DNA-based\"], [\"3\", 37081718, 37081719, \" \", \" \", \" \", \"MLH1\", \"V534R\", \"DNA-based\"], [\"1\", 97981343, 97981343, \" \", \" \", \" \", \"DPYD\", \"DPYD*13 HOMOZYGOSITY\", \"DNA-based\"], [\"3\", 37045935, 37045935, \" \", \" \", \" \", \"MLH1\", \"T117M\", \"DNA-based\"], [\"3\", 37038139, 37038139, \" \", \" \", \" \", \"MLH1\", \"V49A\", \"DNA-based\"], [\"3\", 37035039, 37035039, \" \", \" \", \" \", \"MLH1\", \"M1L\", \"DNA-based\"], [\"3\", 37058999, 37058999, \" \", \" \", \" \", \"MLH1\", \"R265G\", \"DNA-based\"], [\"3\", 37081770, 37081770, \" \", \" \", \" \", \"MLH1\", \"N551T\", \"DNA-based\"], [\"3\", 37067359, 37067359, \" \", \" \", \" \", \"MLH1\", \"A424T\", \"DNA-based\"], [\"3\", 37038192, 37038192, \" \", \" \", \" \", \"MLH1\", \"G67R\", \"DNA-based\"], [\"3\", 37038196, 37038196, \" \", \" \", \" \", \"MLH1\", \"I68S\", \"DNA-based\"], [\"3\", 37038187, 37038187, \" \", \" \", \" \", \"MLH1\", \"G65D\", \"DNA-based\"], [\"3\", 37090447, 37090447, \" \", \" \", \" \", \"MLH1\", \"A681V\", \"DNA-based\"], [\"2\", 48018217, 48018217, \" \", \" \", \" \", \"MSH6\", \"P138T\", \"DNA-based\"], [\"2\", 48033421, 48033421, \" \", \" \", \" \", \"MSH6\", \"R1242H\", \"DNA-based\"], [\"3\", 37070334, 37070334, \" \", \" \", \" \", \"MLH1\", \"M490T\", \"DNA-based\"], [\"17\", 46805705, 46805705, \" \", \" \", \" \", \"HOXB13\", \"G84E\", \"DNA-based\"], [\"2\", 47690231, 47690231, \" \", \" \", \" \", \"MSH2\", \"E483G\", \"DNA-based\"], [\"2\", 48026176, 48026176, \" \", \" \", \" \", \"MSH6\", \"V352I\", \"DNA-based\"], [\"2\", 48033420, 48033420, \" \", \" \", \" \", \"MSH6\", \"R1242C\", \"DNA-based\"], [\"12\", 21331549, 21331549, \" \", \" \", \" \", \"SLCO1B1\", \"RS4149056\", \"DNA-based\"], [\"12\", 21329738, 21329738, \" \", \" \", \" \", \"SLCO1B1\", \"N130D\", \"DNA-based\"], [\"12\", 56491672, 56491672, \" \", \" \", \" \", \"ERBB3\", \"V855A\", \"DNA-based\"], [\"22\", 29121087, 29121087, \" \", \" \", \" \", \"CHEK2\", \"I157T\", \"DNA-based\"], [\"9\", 8521481, 8521481, \" \", \" \", \" \", \"PTPRD\", \"V253I\", \"DNA-based\"], [\"7\", 116423408, 116423408, \" \", \" \", \" \", \"MET\", \"D1228V\", \"DNA-based\"], [\"21\", 36259163, 36259163, \" \", \" \", \" \", \"RUNX1\", \"K83E\", \"DNA-based\"], [\"15\", 88476265, 88476265, \" \", \" \", \" \", \"NTRK3\", \"ETV6-NTRK3 G623R\", \"DNA-based\"], [\"21\", 36252962, 36252962, \" \", \" \", \" \", \"RUNX1\", \"A107P\", \"DNA-based\"], [\"5\", 176942767, 176942767, \" \", \" \", \" \", \"DDX41\", \"R164W\", \"DNA-based\"], [\"17\", 37880997, 37880998, \" \", \" \", \" \", \"ERBB2\", \"G776L\", \"DNA-based\"], [\"3\", 10188302, 10188302, \" \", \" \", \" \", \"VHL\", \"A149S (c.445G>T)\", \"DNA-based\"], [\"1\", 65311203, 65311203, \" \", \" \", \" \", \"JAK1\", \"S703I\", \"DNA-based\"], [\"X\", 47429340, 47429340, \" \", \" \", \" \", \"ARAF\", \"S490T\", \"DNA-based\"], [\"20\", 57484420, 57484420, \" \", \" \", \" \", \"GNAS\", \"R201C\", \"DNA-based\"], [\"10\", 89711899, 89711899, \" \", \" \", \" \", \"PTEN\", \"R173C\", \"DNA-based\"], [\"3\", 10183550, 10183550, \" \", \" \", \" \", \"VHL\", \"N7D (c.19A>G)\", \"DNA-based\"], [\"3\", 10183605, 10183605, \" \", \" \", \" \", \"VHL\", \"P25L (c.74C>T)\", \"DNA-based\"], [\"4\", 55141083, 55141083, \" \", \" \", \" \", \"PDGFRA\", \"P577S\", \"DNA-based\"], [\"4\", 55152090, 55152090, \" \", \" \", \" \", \"PDGFRA\", \"R841K\", \"DNA-based\"], [\"4\", 55152101, 55152101, \" \", \" \", \" \", \"PDGFRA\", \"H845Y\", \"DNA-based\"], [\"4\", 55152126, 55152126, \" \", \" \", \" \", \"PDGFRA\", \"G853D\", \"DNA-based\"], [\"X\", 100613633, 100613633, \" \", \" \", \" \", \"BTK\", \"T316A\", \"DNA-based\"], [\"17\", 37881392, 37881392, \" \", \" \", \" \", \"ERBB2\", \"T862A\", \"DNA-based\"], [\"17\", 37880989, 37880989, \" \", \" \", \" \", \"ERBB2\", \"V773A\", \"DNA-based\"], [\"17\", 37881378, 37881378, \" \", \" \", \" \", \"ERBB2\", \"N857S\", \"DNA-based\"], [\"17\", 37881440, 37881440, \" \", \" \", \" \", \"ERBB2\", \"H878Y\", \"DNA-based\"], [\"20\", 57428713, 57428713, \" \", \" \", \" \", \"GNAS\", \"T393C\", \"DNA-based\"], [\"2\", 209113112, 209113112, \" \", \" \", \" \", \"IDH1\", \"R132L\", \"DNA-based\"], [\"9\", 133748414, 133748414, \" \", \" \", \" \", \"ABL1\", \"F359V\", \"DNA-based\"], [\"12\", 25378562, 25378562, \" \", \" \", \" \", \"KRAS\", \"A146P\", \"DNA-based\"], [\"12\", 25378562, 25378562, \" \", \" \", \" \", \"KRAS\", \"A146T\", \"DNA-based\"], [\"12\", 25398285, 25398285, \" \", \" \", \" \", \"KRAS\", \"G12S\", \"DNA-based\"], [\"2\", 209113113, 209113113, \" \", \" \", \" \", \"IDH1\", \"R132S\", \"DNA-based\"], [\"4\", 55593610, 55593610, \" \", \" \", \" \", \"KIT\", \"V559D\", \"DNA-based\"], [\"4\", 55594221, 55594221, \" \", \" \", \" \", \"KIT\", \"K642E\", \"DNA-based\"], [\"4\", 55599320, 55599320, \" \", \" \", \" \", \"KIT\", \"D816H\", \"DNA-based\"], [\"4\", 55599332, 55599332, \" \", \" \", \" \", \"KIT\", \"D820Y\", \"DNA-based\"], [\"4\", 55599341, 55599341, \" \", \" \", \" \", \"KIT\", \"Y823D\", \"DNA-based\"], [\"7\", 140481402, 140481402, \" \", \" \", \" \", \"BRAF\", \"G469A\", \"DNA-based\"], [\"7\", 55221822, 55221822, \" \", \" \", \" \", \"EGFR\", \"A289V\", \"DNA-based\"], [\"9\", 133738342, 133738342, \" \", \" \", \" \", \"ABL1\", \"L248V\", \"DNA-based\"], [\"9\", 133738349, 133738349, \" \", \" \", \" \", \"ABL1\", \"G250E\", \"DNA-based\"], [\"9\", 133738356, 133738356, \" \", \" \", \" \", \"ABL1\", \"Q252H\", \"DNA-based\"], [\"9\", 133738358, 133738358, \" \", \" \", \" \", \"ABL1\", \"Y253F\", \"DNA-based\"], [\"9\", 133747520, 133747520, \" \", \" \", \" \", \"ABL1\", \"D276G\", \"DNA-based\"], [\"9\", 133748391, 133748391, \" \", \" \", \" \", \"ABL1\", \"M351T\", \"DNA-based\"], [\"9\", 133750356, 133750356, \" \", \" \", \" \", \"ABL1\", \"H396R\", \"DNA-based\"], [\"7\", 140481417, 140481417, \" \", \" \", \" \", \"BRAF\", \"G464V\", \"DNA-based\"], [\"9\", 133755488, 133755488, \" \", \" \", \" \", \"ABL1\", \"BCR-ABL F486S\", \"DNA-based\"], [\"9\", 133738364, 133738364, \" \", \" \", \" \", \"ABL1\", \"E255V\", \"DNA-based\"], [\"9\", 133747588, 133747588, \" \", \" \", \" \", \"ABL1\", \"V299L\", \"DNA-based\"], [\"9\", 133750361, 133750361, \" \", \" \", \" \", \"ABL1\", \"BCR-ABL G398R\", \"DNA-based\"], [\"10\", 43609070, 43609070, \" \", \" \", \" \", \"RET\", \"C609Y\", \"DNA-based\"], [\"4\", 55599340, 55599340, \" \", \" \", \" \", \"KIT\", \"N822K\", \"DNA-based\"], [\"4\", 55599299, 55599299, \" \", \" \", \" \", \"KIT\", \"C809G\", \"DNA-based\"], [\"4\", 55599333, 55599333, \" \", \" \", \" \", \"KIT\", \"D820A\", \"DNA-based\"], [\"4\", 55599333, 55599333, \" \", \" \", \" \", \"KIT\", \"D820G\", \"DNA-based\"], [\"4\", 55595519, 55595519, \" \", \" \", \" \", \"KIT\", \"T670I\", \"DNA-based\"], [\"6\", 117638347, 117638347, \" \", \" \", \" \", \"ROS1\", \"G2032R\", \"DNA-based\"], [\"15\", 66727455, 66727455, \" \", \" \", \" \", \"MAP2K1\", \"K57N\", \"DNA-based\"], [\"1\", 156846342, 156849107, \" \", \" \", \" \", \"NTRK1\", \"LMNA-NTRK1 G595R AND G667C\", \"DNA-based\"], [\"3\", 41266124, 41266124, \" \", \" \", \" \", \"CTNNB1\", \"T41A\", \"DNA-based\"], [\"3\", 41266137, 41266137, \" \", \" \", \" \", \"CTNNB1\", \"S45F\", \"DNA-based\"], [\"3\", 41266136, 41266136, \" \", \" \", \" \", \"CTNNB1\", \"S45P\", \"DNA-based\"], [\"13\", 28592641, 28592641, \" \", \" \", \" \", \"FLT3\", \"D835V\", \"DNA-based\"], [\"7\", 140453149, 140453149, \" \", \" \", \" \", \"BRAF\", \"G596R\", \"DNA-based\"], [\"3\", 10191507, 10191507, \" \", \" \", \" \", \"VHL\", \"R167Q (c.500G>A)\", \"DNA-based\"], [\"3\", 10183823, 10183823, \" \", \" \", \" \", \"VHL\", \"Y98H (c.292T>C)\", \"DNA-based\"], [\"3\", 10188302, 10188302, \" \", \" \", \" \", \"VHL\", \"A149T (c.445G>A)\", \"DNA-based\"], [\"3\", 10191477, 10191477, \" \", \" \", \" \", \"VHL\", \"T157I (c.470C>T)\", \"DNA-based\"], [\"3\", 10191506, 10191506, \" \", \" \", \" \", \"VHL\", \"R167W (c.499C>T)\", \"DNA-based\"], [\"3\", 10191540, 10191540, \" \", \" \", \" \", \"VHL\", \"L178P (c.533T>C)\", \"DNA-based\"], [\"3\", 10183725, 10183725, \" \", \" \", \" \", \"VHL\", \"S65W (c.194C>G)\", \"DNA-based\"], [\"3\", 10183763, 10183763, \" \", \" \", \" \", \"VHL\", \"N78H (c.232A>C)\", \"DNA-based\"], [\"3\", 10183764, 10183764, \" \", \" \", \" \", \"VHL\", \"N78S (c.233A>G)\", \"DNA-based\"], [\"3\", 10183764, 10183764, \" \", \" \", \" \", \"VHL\", \"N78T (c.233A>C)\", \"DNA-based\"], [\"3\", 10183770, 10183770, \" \", \" \", \" \", \"VHL\", \"S80I (c.239G>T)\", \"DNA-based\"], [\"3\", 10183788, 10183788, \" \", \" \", \" \", \"VHL\", \"P86L (c.257C>T)\", \"DNA-based\"], [\"3\", 10183793, 10183793, \" \", \" \", \" \", \"VHL\", \"W88R (c.262T>A)\", \"DNA-based\"], [\"3\", 10191491, 10191491, \" \", \" \", \" \", \"VHL\", \"C162R (c.484T>C)\", \"DNA-based\"], [\"3\", 10191492, 10191492, \" \", \" \", \" \", \"VHL\", \"C162F (c.485G>T)\", \"DNA-based\"], [\"3\", 10191493, 10191493, \" \", \" \", \" \", \"VHL\", \"C162W (c.486C>G)\", \"DNA-based\"], [\"3\", 10188318, 10188318, \" \", \" \", \" \", \"VHL\", \"P154L (c.461C>T)\", \"DNA-based\"], [\"3\", 10183706, 10183706, \" \", \" \", \" \", \"VHL\", \"P59R (c.175delC)\", \"DNA-based\"], [\"3\", 10183725, 10183725, \" \", \" \", \" \", \"VHL\", \"S65W (c.194C>G)\", \"DNA-based\"], [\"3\", 10183725, 10183725, \" \", \" \", \" \", \"VHL\", \"S65L (c.194C>T)\", \"DNA-based\"], [\"3\", 10183767, 10183767, \" \", \" \", \" \", \"VHL\", \"R79P (c.236G>C)\", \"DNA-based\"], [\"3\", 10183771, 10183771, \" \", \" \", \" \", \"VHL\", \"S80R (c.240T>G)\", \"DNA-based\"], [\"3\", 10183797, 10183797, \" \", \" \", \" \", \"VHL\", \"L89P (c.266T>C)\", \"DNA-based\"], [\"3\", 10183868, 10183868, \" \", \" \", \" \", \"VHL\", \"R113* (c.337C>T)\", \"DNA-based\"], [\"3\", 10183871, 10183871, \" \", \" \", \" \", \"VHL\", \"G114C (c.340G>T)\", \"DNA-based\"], [\"3\", 10188210, 10188210, \" \", \" \", \" \", \"VHL\", \"L118P (c.353T>C)\", \"DNA-based\"], [\"3\", 10188247, 10188248, \" \", \" \", \" \", \"VHL\", \"N131P (c.390_391delTA)\", \"DNA-based\"], [\"3\", 10188264, 10188264, \" \", \" \", \" \", \"VHL\", \"F136S (c.407T>C)\", \"DNA-based\"], [\"3\", 10191479, 10191479, \" \", \" \", \" \", \"VHL\", \"L158V (c.472C>G)\", \"DNA-based\"], [\"3\", 10191515, 10191515, \" \", \" \", \" \", \"VHL\", \"V170F (c.508G>T)\", \"DNA-based\"], [\"3\", 10191545, 10191545, \" \", \" \", \" \", \"VHL\", \"I180V (c.538A>G)\", \"DNA-based\"], [\"3\", 10191558, 10191558, \" \", \" \", \" \", \"VHL\", \"L184R (c.551T>G)\", \"DNA-based\"], [\"3\", 10188219, 10188219, \" \", \" \", \" \", \"VHL\", \"D121G (c.362A>G)\", \"DNA-based\"], [\"3\", 10188203, 10188203, \" \", \" \", \" \", \"VHL\", \"L116V (c.346C>G)\", \"DNA-based\"], [\"3\", 10191492, 10191492, \" \", \" \", \" \", \"VHL\", \"C162Y (c.485G>A)\", \"DNA-based\"], [\"3\", 10191530, 10191530, \" \", \" \", \" \", \"VHL\", \"Y175D (c.523T>G)\", \"DNA-based\"], [\"3\", 10183788, 10183788, \" \", \" \", \" \", \"VHL\", \"P86R (c.257C>G)\", \"DNA-based\"], [\"3\", 10188201, 10188201, \" \", \" \", \" \", \"VHL\", \"H115P (c.344A>C)\", \"DNA-based\"], [\"3\", 10188240, 10188240, \" \", \" \", \" \", \"VHL\", \"L128R (c.383T>G)\", \"DNA-based\"], [\"7\", 55249118, 55249118, \" \", \" \", \" \", \"EGFR\", \"K806E\", \"DNA-based\"], [\"3\", 10183787, 10183787, \" \", \" \", \" \", \"VHL\", \"P86S (c.256C>T)\", \"DNA-based\"], [\"3\", 10183724, 10183724, \" \", \" \", \" \", \"VHL\", \"S65P (c.193T>C)\", \"DNA-based\"], [\"3\", 10183745, 10183745, \" \", \" \", \" \", \"VHL\", \"S72P (c.214T>C)\", \"DNA-based\"], [\"3\", 10191471, 10191471, \" \", \" \", \" \", \"VHL\", \"V155E (c.464T>A)\", \"DNA-based\"], [\"3\", 10183752, 10183752, \" \", \" \", \" \", \"VHL\", \"V74G (c.221T>G)\", \"DNA-based\"], [\"3\", 10183785, 10183785, \" \", \" \", \" \", \"VHL\", \"L85P (c.254T>C)\", \"DNA-based\"], [\"4\", 55595501, 55595651, \" \", \" \", \" \", \"KIT\", \"EXON 14 MUTATION\", \"DNA-based\"], [\"5\", 170837531, 170837569, \" \", \" \", \" \", \"NPM1\", \"EXON 12 MUTATION\", \"DNA-based\"], [\"3\", 178951882, 178952495, \" \", \" \", \" \", \"PIK3CA\", \"EXON 21 MUTATION\", \"DNA-based\"], [\"3\", 178935998, 178936122, \" \", \" \", \" \", \"PIK3CA\", \"EXON 10 MUTATION\", \"DNA-based\"], [\"11\", 32417803, 32417953, \" \", \" \", \" \", \"WT1\", \"EXON 7 MUTATION\", \"DNA-based\"], [\"11\", 32413518, 32413610, \" \", \" \", \" \", \"WT1\", \"EXON 9 MUTATION\", \"DNA-based\"], [\"19\", 1205740, 1218499, \" \", \" \", \" \", \"STK11\", \"EXON 1-2 MUTATION\", \"DNA-based\"], [\"18\", 42531877, 42531918, \" \", \" \", \" \", \"SETBP1\", \"EXON 4 MUTATION\", \"DNA-based\"], [\"7\", 55237999, 55238735, \" \", \" \", \" \", \"EGFR\", \"EXON 18 OVEREXPRESSION\", \"DNA-based\"], [\"4\", 55592023, 55592216, \" \", \" \", \" \", \"KIT\", \"EXON 9 MUTATION\", \"DNA-based\"], [\"20\", 31021087, 31021720, \" \", \" \", \" \", \"ASXL1\", \"EXON 12 MUTATION\", \"DNA-based\"], [\"19\", 13054527, 13054727, \" \", \" \", \" \", \"CALR\", \"EXON 9 FRAMESHIFT\", \"DNA-based\"], [\"12\", 25360224, 25360224, \" \", \" \", \" \", \"KRAS\", \"RS61764370\", \"DNA-based\"], [\"4\", 55604724, 55606881, \" \", \" \", \" \", \"KIT\", \"3' UTR MUTATION\", \"DNA-based\"], [\"14\", 62213848, 62213848, \" \", \" \", \" \", \"HIF1A\", \"3' UTR Polymorphism\", \"DNA-based\"], [\"3\", 10183532, 10183548, \" \", \" \", \" \", \"VHL\", \"M1FS (c.1_17del17)\", \"DNA-based\"], [\"3\", 10183531, 10183551, \" \", \" \", \" \", \"VHL\", \"M1? (c.1-1_20del21)\", \"DNA-based\"], [\"3\", 10183534, 10183534, \" \", \" \", \" \", \"VHL\", \"M1? (c.3G>A)\", \"DNA-based\"], [\"7\", 55086794, 55279321, \" \", \" \", \" \", \"EGFR\", \"MUTATION\", \"DNA-based\"], [\"3\", 10191470, 10191470, \" \", \" \", \" \", \"VHL\", \"Splicing alteration (c.464-1G>A)\", \"DNA-based\"], [\"3\", 10191470, 10191470, \" \", \" \", \" \", \"VHL\", \"Splicing alteration (c.464-1G>T)\", \"DNA-based\"], [\"3\", 10191469, 10191469, \" \", \" \", \" \", \"VHL\", \"Splicing alteration (c.464-2A>T)\", \"DNA-based\"], [\"3\", 10191470, 10191470, \" \", \" \", \" \", \"VHL\", \"Splicing alteration (c.464-1G>C)\", \"DNA-based\"], [\"3\", 10191469, 10191469, \" \", \" \", \" \", \"VHL\", \"Splicing alteration (c.464-2A>G)\", \"DNA-based\"], [\"3\", 10188196, 10188196, \" \", \" \", \" \", \"VHL\", \"Splicing alteration (c.341-2A>C)\", \"DNA-based\"], [\"3\", 37092143, 37092143, \" \", \" \", \" \", \"MLH1\", \"*757L\", \"DNA-based\"], [\"3\", 10191649, 10191649, \" \", \" \", \" \", \"VHL\", \"*214W (c.642A>G)\", \"DNA-based\"], [\"13\", 103498198, 103498199, \" \", \" \", \" \", \"ERCC5\", \"RS751402\", \"DNA-based\"], [\"18\", 657604, 657742, \" \", \" \", \" \", \"TYMS\", \"5' TANDEM REPEAT\", \"DNA-based\"], [\"7\", 55087058, 55223523, \" \", \" \", \" \", \"EGFR\", \"VIII\", \"DNA-based\"], [\"2\", 47612305, 47614173, \" \", \" \", \" \", \"EPCAM\", \"3' EXON DELETION\", \"DNA-based\"], [\"4\", 55593582, 55593708, \" \", \" \", \" \", \"KIT\", \"EXON 11 MUTATION\", \"DNA-based\"], [\"17\", 37880991, 37880991, \" \", \" \", \" \", \"ERBB2\", \"M774DELINSWLV\", \"DNA-based\"], [\"17\", 37880219, 37880233, \" \", \" \", \" \", \"ERBB2\", \"DEL 755-759\", \"DNA-based\"], [\"4\", 55152095, 55152097, \" \", \" \", \" \", \"PDGFRA\", \"I843DEL\", \"DNA-based\"], [\"7\", 55242415, 55242513, \" \", \" \", \" \", \"EGFR\", \"EXON 19 DELETION\", \"DNA-based\"], [\"4\", 55593612, 55593614, \" \", \" \", \" \", \"KIT\", \"V560DEL\", \"DNA-based\"], [\"7\", 140477838, 140477855, \" \", \" \", \" \", \"BRAF\", \"DEL 485-490\", \"DNA-based\"], [\"2\", 29497965, 29754982, \" \", \" \", \" \", \"ALK\", \"DEL4-11\", \"DNA-based\"], [\"15\", 66727449, 66727463, \" \", \" \", \" \", \"MAP2K1\", \"Q56_V60del\", \"DNA-based\"], [\"3\", 37089130, 37089132, \" \", \" \", \" \", \"MLH1\", \"K618DEL\", \"DNA-based\"], [\"3\", 10183711, 10183711, \" \", \" \", \" \", \"VHL\", \"V62C (c.180delG)\", \"DNA-based\"], [\"3\", 10183758, 10183760, \" \", \" \", \" \", \"VHL\", \"F76del (c.227_229delTCT)\", \"DNA-based\"], [\"3\", 10183757, 10183759, \" \", \" \", \" \", \"VHL\", \"F76del (c.226_228delTTC)\", \"DNA-based\"], [\"3\", 10188302, 10188302, \" \", \" \", \" \", \"VHL\", \"A149fs (c.445delG)\", \"DNA-based\"], [\"12\", 69234988, 69234988, \" \", \" \", \" \", \"MDM2\", \"RS34886328\", \"DNA-based\"], [\"9\", 139390861, 139390861, \" \", \" \", \" \", \"NOTCH1\", \"V2444FS\", \"DNA-based\"], [\"11\", 94152895, 94227074, \" \", \" \", \" \", \"MRE11\", \"FRAMESHIFT MUTATION\", \"DNA-based\"], [\"11\", 102201730, 102207833, \" \", \" \", \" \", \"BIRC3\", \"TRUNCATING MUTATION\", \"DNA-based\"], [\"10\", 89720799, 89720802, \" \", \" \", \" \", \"PTEN\", \"V317FS\", \"DNA-based\"], [\"3\", 37090464, 37090464, \" \", \" \", \" \", \"MLH1\", \"R687FS\", \"DNA-based\"], [\"2\", 47641433, 47641433, \" \", \" \", \" \", \"MSH2\", \"V273FS\", \"DNA-based\"], [\"3\", 10183840, 10183841, \" \", \" \", \" \", \"VHL\", \"P103FS (c.309_310delTG)\", \"DNA-based\"], [\"3\", 10188241, 10188241, \" \", \" \", \" \", \"VHL\", \"L129fs (c.384delT)\", \"DNA-based\"], [\"3\", 10183794, 10183796, \" \", \" \", \" \", \"VHL\", \"W88fs (c.263_265delGGCinsTT)\", \"DNA-based\"], [\"3\", 10183713, 10183716, \" \", \" \", \" \", \"VHL\", \"P61fs (c.182_185delCCGT)\", \"DNA-based\"], [\"4\", 108603171, 108641608, \"7\", 140434279, 140487384, \"BRAF\", \"PAPSS1-BRAF\", \"DNA-based\"], [\"7\", 148524359, 148525831, \" \", \" \", \" \", \"EZH2\", \"INTRON 6 MUTATION\", \"DNA-based\"], [\"11\", 9599943, 9599943, \" \", \" \", \" \", \"WEE1\", \"RS3910384\", \"DNA-based\"], [\"2\", 25455845, 25565459, \" \", \" \", \" \", \"DNMT3A\", \"MUTATION\", \"DNA-based\"], [\"1\", 155870065, 155880706, \" \", \" \", \" \", \"RIT1\", \"MUTATION\", \"DNA-based\"], [\"X\", 66764465, 66950461, \" \", \" \", \" \", \"AR\", \"MUTATION\", \"DNA-based\"], [\"2\", 209101803, 209116275, \" \", \" \", \" \", \"IDH1\", \"MUTATION\", \"DNA-based\"], [\"22\", 29999547, 30094587, \" \", \" \", \" \", \"NF2\", \"MUTATION\", \"DNA-based\"], [\"X\", 39910501, 40036582, \" \", \" \", \" \", \"BCOR\", \"MUTATION\", \"DNA-based\"], [\"X\", 20156713, 20159754, \" \", \" \", \" \", \"EIF1AX\", \"MUTATION\", \"DNA-based\"], [\"9\", 135766735, 135820008, \" \", \" \", \" \", \"TSC1\", \"FRAMESHIFT TRUNCATION\", \"DNA-based\"], [\"9\", 139390649, 139390650, \" \", \" \", \" \", \"NOTCH1\", \"P2514FS\", \"DNA-based\"], [\"17\", 7573998, 7579714, \" \", \" \", \" \", \"TP53\", \"TRUNCATING MUTATION\", \"DNA-based\"], [\"11\", 108196840, 108196841, \" \", \" \", \" \", \"ATM\", \"V2288FS*1\", \"DNA-based\"], [\"3\", 142274740, 142274740, \" \", \" \", \" \", \"ATR\", \"I774FS\", \"DNA-based\"], [\"22\", 30050673, 30050675, \" \", \" \", \" \", \"NF2\", \"K159FS*16\", \"DNA-based\"], [\"17\", 41244645, 41244646, \" \", \" \", \" \", \"BRCA1\", \"P968FS\", \"DNA-based\"], [\"15\", 45003786, 45003789, \" \", \" \", \" \", \"B2M\", \"S14FS\", \"DNA-based\"], [\"13\", 49033946, 49033947, \" \", \" \", \" \", \"RB1\", \"M695FS*26\", \"DNA-based\"], [\"11\", 64572092, 64572092, \" \", \" \", \" \", \"MEN1\", \"FRAMESHIFT TRUNCATION\", \"DNA-based\"], [\"3\", 37035075, 37035075, \" \", \" \", \" \", \"MLH1\", \"E13FS*3\", \"DNA-based\"], [\"3\", 37089094, 37089095, \" \", \" \", \" \", \"MLH1\", \"G606FS*2\", \"DNA-based\"], [\"3\", 37067365, 37067365, \" \", \" \", \" \", \"MLH1\", \"Q426FS\", \"DNA-based\"], [\"3\", 37090406, 37090406, \" \", \" \", \" \", \"MLH1\", \"D667FS\", \"DNA-based\"], [\"3\", 37053551, 37053551, \" \", \" \", \" \", \"MLH1\", \"V213FS\", \"DNA-based\"], [\"3\", 37081726, 37081726, \" \", \" \", \" \", \"MLH1\", \"P536FS\", \"DNA-based\"], [\"3\", 10183694, 10183695, \" \", \" \", \" \", \"VHL\", \"E55FS (c.163_164delGA)\", \"DNA-based\"], [\"2\", 48027794, 48027795, \" \", \" \", \" \", \"MSH6\", \"I891FS\", \"DNA-based\"], [\"3\", 10183694, 10183694, \" \", \" \", \" \", \"VHL\", \"E55RfsTer11 (c.163delG)\", \"DNA-based\"], [\"3\", 10183695, 10183696, \" \", \" \", \" \", \"VHL\", \"A56FS (c.164_165insG)\", \"DNA-based\"], [\"22\", 29091857, 29091857, \" \", \" \", \" \", \"CHEK2\", \"1100DELC\", \"DNA-based\"], [\"3\", 10183695, 10183696, \" \", \" \", \" \", \"VHL\", \"A56FS (c.164_165insA)\", \"DNA-based\"], [\"21\", 36252913, 36252920, \" \", \" \", \" \", \"RUNX1\", \"T148HFSX9\", \"DNA-based\"], [\"3\", 10183693, 10183697, \" \", \" \", \" \", \"VHL\", \"M54IFS (c.162_166delGGAGG)\", \"DNA-based\"], [\"3\", 10183704, 10183705, \" \", \" \", \" \", \"VHL\", \"P59fs (c.173_174insC)\", \"DNA-based\"], [\"3\", 10183822, 10183841, \" \", \" \", \" \", \"VHL\", \"Y98FS (c.291_310del20)\", \"DNA-based\"], [\"3\", 10191533, 10191533, \" \", \" \", \" \", \"VHL\", \"Y175fs (c.526delA)\", \"DNA-based\"], [\"3\", 10183745, 10183745, \" \", \" \", \" \", \"VHL\", \"S72P (c.214delT)\", \"DNA-based\"], [\"3\", 10188224, 10188224, \" \", \" \", \" \", \"VHL\", \"G123FS (c.367delG)\", \"DNA-based\"], [\"3\", 10188261, 10188261, \" \", \" \", \" \", \"VHL\", \"L135fs (c.404delT)\", \"DNA-based\"], [\"3\", 10191535, 10191535, \" \", \" \", \" \", \"VHL\", \"R176fs (c.528delG)\", \"DNA-based\"], [\"3\", 10191486, 10191487, \" \", \" \", \" \", \"VHL\", \"E160FS (c.479_480delAG)\", \"DNA-based\"], [\"3\", 10191578, 10191578, \" \", \" \", \" \", \"VHL\", \"H191FS (c.571delC)\", \"DNA-based\"], [\"3\", 10191490, 10191490, \" \", \" \", \" \", \"VHL\", \"C162FS (c.483delA)\", \"DNA-based\"], [\"3\", 10183725, 10183725, \" \", \" \", \" \", \"VHL\", \"S65fs (c.194delC)\", \"DNA-based\"], [\"3\", 10191532, 10191532, \" \", \" \", \" \", \"VHL\", \"Y175fs (c.525delC)\", \"DNA-based\"], [\"3\", 10183868, 10183868, \" \", \" \", \" \", \"VHL\", \"R113fs (c.337delC)\", \"DNA-based\"], [\"3\", 10183822, 10183822, \" \", \" \", \" \", \"VHL\", \"Y98FS (c.291delC)\", \"DNA-based\"], [\"5\", 170837547, 170837548, \" \", \" \", \" \", \"NPM1\", \"W288FS\", \"DNA-based\"], [\"22\", 30051593, 30051594, \" \", \" \", \" \", \"NF2\", \"Y177FS\", \"DNA-based\"], [\"2\", 47630413, 47630414, \" \", \" \", \" \", \"MSH2\", \"E28FS\", \"DNA-based\"], [\"9\", 139391369, 139391370, \" \", \" \", \" \", \"NOTCH1\", \"S2275FS\", \"DNA-based\"], [\"13\", 28592642, 28592642, \" \", \" \", \" \", \"FLT3\", \"TKD MUTATION\", \"DNA-based\"], [\"7\", 55214299, 55214433, \" \", \" \", \" \", \"EGFR\", \"EXON 4 DELETION\", \"DNA-based\"], [\"2\", 141665446, 141777671, \" \", \" \", \" \", \"LRP1B\", \"EXON 12–22 DELETION\", \"DNA-based\"], [\"7\", 116411903, 116412043, \" \", \" \", \" \", \"MET\", \"EXON 14 SKIPPING MUTATION\", \"DNA-based\"], [\"3\", 10183532, 10191649, \" \", \" \", \" \", \"VHL\", \"EXON 1-3 DELETION\", \"DNA-based\"], [\"3\", 10183532, 10183871, \" \", \" \", \" \", \"VHL\", \"EXON 1 DELETION\", \"DNA-based\"], [\"3\", 10183532, 10188320, \" \", \" \", \" \", \"VHL\", \"EXON 1-2 DELETION\", \"DNA-based\"], [\"7\", 87138645, 87138645, \" \", \" \", \" \", \"ABCB1\", \"I1145I\", \"DNA-based\"], [\"10\", 131265545, 131265545, \" \", \" \", \" \", \"MGMT\", \"RS16906252\", \"DNA-based\"], [\"21\", 40191638, 40191638, \" \", \" \", \" \", \"ETS2\", \"RS461155\", \"DNA-based\"], [\"4\", 55602765, 55602765, \" \", \" \", \" \", \"KIT\", \"RS3733542\", \"DNA-based\"], [\"3\", 105439026, 105439026, \" \", \" \", \" \", \"CBLB\", \"RS2305035\", \"DNA-based\"], [\"3\", 37056036, 37056036, \" \", \" \", \" \", \"MLH1\", \"C.790+1G>A\", \"DNA-based\"], [\"1\", 97915614, 97915614, \" \", \" \", \" \", \"DPYD\", \"DPYD*2A HOMOZYGOSITY\", \"DNA-based\"], [\"9\", 21975017, 21975017, \" \", \" \", \" \", \"CDKN2A\", \"RS3814960\", \"DNA-based\"], [\"13\", 31722621, 31722637, \" \", \" \", \" \", \"HSPH1\", \"T17 DELETION\", \"DNA-based\"], [\"2\", 234668881, 234668882, \" \", \" \", \" \", \"UGT1A1\", \"UGT1A1*28\", \"DNA-based\"], [\"17\", 7577149, 7578443, \" \", \" \", \" \", \"TP53\", \"DNA BINDING DOMAIN MUTATION\", \"DNA-based\"], [\"2\", 111883195, 111886097, \" \", \" \", \" \", \"BCL2L11\", \"DELETION POLYMORPHISM\", \"DNA-based\"]]\n"
     ]
    }
   ],
   "source": [
    "## For variants listed in the DNA-based API, create bed-like files for capture design\n",
    "pipeline = 'DNA-based' #set pipeline\n",
    "blank = ' ' #set blanks\n",
    "\n",
    "#iterate through evidence items\n",
    "for item in variants_DNA\n",
    "  flag = false #set flag for coordinates; false = chr1,start1,stop1 only\n",
    "  chrom = item['coordinates']['chromosome'] #call chrom\n",
    "  start = item['coordinates']['start'] #call start\n",
    "  stop = item['coordinates']['stop']\n",
    "  if item['coordinates']['chromosome2'] and item['coordinates']['start2'] and item['coordinates']['stop2'] #determine if there is a second set of coordinates\n",
    "    chrom2 = item['coordinates']['chromosome2']  # call chrom2\n",
    "    start2 = item['coordinates']['start2']  # call start2\n",
    "    stop2 = item['coordinates']['stop2']  # call stop2\n",
    "    flag = true #set flag for coordinates; true = second set of coordinates available\n",
    "  end\n",
    "  gene = item['entrez_name']  #Call Gene name\n",
    "  variant = item['name'] #call variant\n",
    "  if flag #if there is two sets of coordinates\n",
    "    capture_DNA << [chrom, start, stop, chrom2, start2, stop2, gene, variant, pipeline] #append to list\n",
    "  end\n",
    "  if not flag #if there is only one set of coordinates\n",
    "    capture_DNA << [chrom, start, stop, blank, blank, blank, gene, variant, pipeline] #append to list\n",
    "  end\n",
    "end"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{\"id\"=>19, \"entrez_name\"=>\"CCND1\", \"entrez_id\"=>595, \"name\"=>\"EXPRESSION\", \"description\"=>\"CCND1 expression, and its prognositc impact, is still in dispute. Three experiments in non-small cell lung cancer have shown it to have no impact on survival, but three additional studies have shown it results in poorer prognosis. There is also some ambiguity in how the boundaries between expression and overexpression are defined.\", \"gene_id\"=>8, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>30.0, \"coordinates\"=>{\"chromosome\"=>\"11\", \"start\"=>69455855, \"stop\"=>69469242, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000227507.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/8/summary/variants/19/summary#variant\", \"evidence_items\"=>[{\"id\"=>355, \"name\"=>\"EID355\", \"description\"=>\"Three studies have associated CCND1 expression with poorer survival.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>210, \"name\"=>\"Cyclin D1 in non-small cell lung cancer: a key driver of malignant transformation.\", \"citation\"=>\"Gautschi et al., 2007, Lung Cancer\", \"pubmed_id\"=>\"17070615\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17070615\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>1}, \"journal\"=>\"Lung Cancer\", \"full_journal_title\"=>\"Lung cancer (Amsterdam, Netherlands)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>19, \"phenotypes\"=>[]}, {\"id\"=>342, \"name\"=>\"EID342\", \"description\"=>\"Three studies have found no significant survival impact for CCND1 in lung cancer.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>210, \"name\"=>\"Cyclin D1 in non-small cell lung cancer: a key driver of malignant transformation.\", \"citation\"=>\"Gautschi et al., 2007, Lung Cancer\", \"pubmed_id\"=>\"17070615\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17070615\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>1}, \"journal\"=>\"Lung Cancer\", \"full_journal_title\"=>\"Lung cancer (Amsterdam, Netherlands)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>19, \"phenotypes\"=>[]}]}, {\"id\"=>20, \"entrez_name\"=>\"CCND1\", \"entrez_id\"=>595, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.\", \"gene_id\"=>8, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>106.0, \"coordinates\"=>{\"chromosome\"=>\"11\", \"start\"=>69455855, \"stop\"=>69469242, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000227507.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/8/summary/variants/20/summary#variant\", \"evidence_items\"=>[{\"id\"=>1536, \"name\"=>\"EID1536\", \"description\"=>\"A pharmacodynamic study (NCT00420056) of 17 mantle cell lymphoma MCL patients was performed using the CDK4/6 inhibitor palbociclib (PD0332991). Patients were verified by IHC, FISH or molecular testing for the t(11:14) rearrangement  causing Cyclin D overexpression in MCL. 1 patient demonstrated complete response, and 2 showed partial response for an 18% overall response rate. 7 patients showed stable disease. Patient population was heavily pretreated.\", \"disease\"=>{\"id\"=>38, \"name\"=>\"Mantle Cell Lymphoma\", \"display_name\"=>\"Mantle Cell Lymphoma\", \"doid\"=>\"0050746\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050746\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1023, \"name\"=>\"Selective CDK4/6 inhibition with tumor responses by PD0332991 in patients with mantle cell lymphoma.\", \"citation\"=>\"Leonard et al., 2012, Blood\", \"pubmed_id\"=>\"22383795\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22383795\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>5, \"day\"=>17}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>20, \"phenotypes\"=>[]}, {\"id\"=>1563, \"name\"=>\"EID1563\", \"description\"=>\"In Mantle Cell Lymphoma (MCL), t(11:14)-induced overexpression of cyclin D1 was targeted by CDK4/6 inhibitor Palbociclib (PD0332991). Palbociclib caused Rb dephosphorylation in MCL cell lines and patient derived tumor cells. MCL cell lines overexpressing cyclin D1 showed growth inhibition with Palbociclib. The VAL DLBCL cell line was used as control to verify cyclin D1 overexpression but was not used as a control in Palbociclib inhibitor experiments. Instead, the investigators report that other publications report cell lines without cyclin D1 overexpression had higher IC50 values for palbociclib, suggesting overexpresssed cyclin D1 targetability in MCL.\", \"disease\"=>{\"id\"=>38, \"name\"=>\"Mantle Cell Lymphoma\", \"display_name\"=>\"Mantle Cell Lymphoma\", \"doid\"=>\"0050746\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050746\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}], \"rating\"=>1, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1032, \"name\"=>\"Mantle cell lymphoma cells express predominantly cyclin D1a isoform and are highly sensitive to selective inhibition of CDK4 kinase activity.\", \"citation\"=>\"Marzec et al., 2006, Blood\", \"pubmed_id\"=>\"16690963\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16690963\", \"open_access\"=>true, \"pmc_id\"=>\"PMC1895502\", \"publication_date\"=>{\"year\"=>2006, \"month\"=>9, \"day\"=>1}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>20, \"phenotypes\"=>[]}, {\"id\"=>856, \"name\"=>\"EID856\", \"description\"=>\"The authors performed a tissue microarray analysis of 167 postmenopausal breast cancers that had been randomized to treatment with or without tamoxifen.  ER +ve patients with high CCND1 tumor expression did not show any difference in survival between tamoxifen and no treatment.\", \"disease\"=>{\"id\"=>28, \"name\"=>\"Estrogen-receptor Positive Breast Cancer\", \"display_name\"=>\"Estrogen-receptor Positive Breast Cancer\", \"doid\"=>\"0060075\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060075\"}, \"drugs\"=>[{\"id\"=>53, \"name\"=>\"Tamoxifen\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>573, \"name\"=>\"Cyclin D1 overexpression is a negative predictive factor for tamoxifen response in postmenopausal breast cancer patients.\", \"citation\"=>\"Stendahl et al., 2004, Br. J. Cancer\", \"pubmed_id\"=>\"15138475\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15138475\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2409465\", \"publication_date\"=>{\"year\"=>2004, \"month\"=>5, \"day\"=>17}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>20, \"phenotypes\"=>[]}, {\"id\"=>358, \"name\"=>\"EID358\", \"description\"=>\"Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in mantle cell lymphoma.\", \"disease\"=>{\"id\"=>38, \"name\"=>\"Mantle Cell Lymphoma\", \"display_name\"=>\"Mantle Cell Lymphoma\", \"doid\"=>\"0050746\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050746\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>219, \"name\"=>\"Genetic and molecular pathogenesis of mantle cell lymphoma: perspectives for new targeted therapeutics.\", \"citation\"=>\"Jares et al., 2007, Nat. Rev. Cancer\", \"pubmed_id\"=>\"17891190\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17891190\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>10}, \"journal\"=>\"Nat. Rev. Cancer\", \"full_journal_title\"=>\"Nature reviews. Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>20, \"phenotypes\"=>[]}, {\"id\"=>357, \"name\"=>\"EID357\", \"description\"=>\"Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis in head and neck squamous cell carcinoma.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>218, \"name\"=>\"Molecular predictors of clinical outcome in patients with head and neck squamous cell carcinoma.\", \"citation\"=>\"Thomas et al., 2005, Int J Exp Pathol\", \"pubmed_id\"=>\"16309541\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16309541\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2517451\", \"publication_date\"=>{\"year\"=>2005, \"month\"=>12}, \"journal\"=>\"Int J Exp Pathol\", \"full_journal_title\"=>\"International journal of experimental pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>20, \"phenotypes\"=>[]}, {\"id\"=>356, \"name\"=>\"EID356\", \"description\"=>\"In ER-positive breast cancer patients, Cyclin D1 overexpression is associated with shorter overall survival and increased metastasis.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>157, \"name\"=>\"Cyclin D1 in breast cancer pathogenesis.\", \"citation\"=>\"Arnold et al., 2005, J. Clin. Oncol.\", \"pubmed_id\"=>\"15961768\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15961768\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>6, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>20, \"phenotypes\"=>[]}, {\"id\"=>221, \"name\"=>\"EID221\", \"description\"=>\"Cyclin D1 overexpression is associated with the ER-positive subtype of breast cancer.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>157, \"name\"=>\"Cyclin D1 in breast cancer pathogenesis.\", \"citation\"=>\"Arnold et al., 2005, J. Clin. Oncol.\", \"pubmed_id\"=>\"15961768\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15961768\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>6, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>20, \"phenotypes\"=>[]}]}, {\"id\"=>21, \"entrez_name\"=>\"CCND2\", \"entrez_id\"=>894, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.\", \"gene_id\"=>9, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>16.0, \"coordinates\"=>{\"chromosome\"=>\"12\", \"start\"=>4382938, \"stop\"=>4414516, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000261254.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/9/summary/variants/21/summary#variant\", \"evidence_items\"=>[{\"id\"=>359, \"name\"=>\"EID359\", \"description\"=>\"Cyclin D2 overexpression is associated with poor prognosis in gastric cancers.\", \"disease\"=>{\"id\"=>21, \"name\"=>\"Stomach Cancer\", \"display_name\"=>\"Stomach Cancer\", \"doid\"=>\"10534\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10534\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>220, \"name\"=>\"Cyclin D2, but not cyclin D1, overexpression closely correlates with gastric cancer progression and prognosis.\", \"citation\"=>\"Takano et al., 1999, J. Pathol.\", \"pubmed_id\"=>\"10547574\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/10547574\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1999, \"month\"=>10}, \"journal\"=>\"J. Pathol.\", \"full_journal_title\"=>\"The Journal of pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>21, \"phenotypes\"=>[]}, {\"id\"=>360, \"name\"=>\"EID360\", \"description\"=>\"Cyclin D2 overexpression is associated with the increased depth of cancer invasion in gastric cancers.\", \"disease\"=>{\"id\"=>21, \"name\"=>\"Stomach Cancer\", \"display_name\"=>\"Stomach Cancer\", \"doid\"=>\"10534\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10534\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>220, \"name\"=>\"Cyclin D2, but not cyclin D1, overexpression closely correlates with gastric cancer progression and prognosis.\", \"citation\"=>\"Takano et al., 1999, J. Pathol.\", \"pubmed_id\"=>\"10547574\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/10547574\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1999, \"month\"=>10}, \"journal\"=>\"J. Pathol.\", \"full_journal_title\"=>\"The Journal of pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>21, \"phenotypes\"=>[]}, {\"id\"=>361, \"name\"=>\"EID361\", \"description\"=>\"Cyclin D2 overexpression is associated with the presence pf lymph node metastasis in gastric cancers.\", \"disease\"=>{\"id\"=>21, \"name\"=>\"Stomach Cancer\", \"display_name\"=>\"Stomach Cancer\", \"doid\"=>\"10534\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10534\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>220, \"name\"=>\"Cyclin D2, but not cyclin D1, overexpression closely correlates with gastric cancer progression and prognosis.\", \"citation\"=>\"Takano et al., 1999, J. Pathol.\", \"pubmed_id\"=>\"10547574\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/10547574\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1999, \"month\"=>10}, \"journal\"=>\"J. Pathol.\", \"full_journal_title\"=>\"The Journal of pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>21, \"phenotypes\"=>[]}, {\"id\"=>362, \"name\"=>\"EID362\", \"description\"=>\"Cyclin D2 overexpression is associated with vascular invasion by cancer cells in gastric cancers.\", \"disease\"=>{\"id\"=>21, \"name\"=>\"Stomach Cancer\", \"display_name\"=>\"Stomach Cancer\", \"doid\"=>\"10534\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10534\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>220, \"name\"=>\"Cyclin D2, but not cyclin D1, overexpression closely correlates with gastric cancer progression and prognosis.\", \"citation\"=>\"Takano et al., 1999, J. Pathol.\", \"pubmed_id\"=>\"10547574\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/10547574\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1999, \"month\"=>10}, \"journal\"=>\"J. Pathol.\", \"full_journal_title\"=>\"The Journal of pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>21, \"phenotypes\"=>[]}]}, {\"id\"=>22, \"entrez_name\"=>\"CCND2\", \"entrez_id\"=>894, \"name\"=>\"PROMOTER DEMETHYLATION\", \"description\"=>\"Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are misregulated are widely variable, and range from genomic amplification to promoter methylation changes. While Cyclin D2 has only been found to be significantly misregulated in glioma, Cyclin D1 in particular seems to be a pan-cancer actor. Cyclin D misregulation has been shown to lead to poorer outcomes in a number of studies, and currently there are no FDA-approved targeted therapies.\", \"gene_id\"=>9, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"12\", \"start\"=>4381437, \"stop\"=>4382937, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000261254.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/9/summary/variants/22/summary#variant\", \"evidence_items\"=>[{\"id\"=>219, \"name\"=>\"EID219\", \"description\"=>\"CCND2 hypomethylation is seen to be more common in stage III and IV gastric tumors than stage I and II.\", \"disease\"=>{\"id\"=>21, \"name\"=>\"Stomach Cancer\", \"display_name\"=>\"Stomach Cancer\", \"doid\"=>\"10534\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10534\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Negative\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>155, \"name\"=>\"Promoter methylation of cyclin D2 gene in gastric carcinoma.\", \"citation\"=>\"Oshimo et al., 2003, Int. J. Oncol.\", \"pubmed_id\"=>\"14612939\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/14612939\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2003, \"month\"=>12}, \"journal\"=>\"Int. J. Oncol.\", \"full_journal_title\"=>\"International journal of oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>22, \"phenotypes\"=>[]}]}, {\"id\"=>24, \"entrez_name\"=>\"CCNE1\", \"entrez_id\"=>898, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"Cyclin E, while currenly not as widely implicated as its counterpart, cyclin D, has been implicated in various carcinomas, including breast, gastric, stomach and colorectal. High levels of cyclin E, either by gene amplification or overexpression, are correlated with later stage disease and have been shown to lead to poorer prognosis in gastic carcinoma. It has also been shown, in lung cancer specifically, that neoplastic cells with higher levels of the cyclin E/CDK2 complex are more radiosensitive than their lowly expressed counterparts.\", \"gene_id\"=>11, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>62.0, \"coordinates\"=>{\"chromosome\"=>\"19\", \"start\"=>30302805, \"stop\"=>30315215, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000262643.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/11/summary/variants/24/summary#variant\", \"evidence_items\"=>[{\"id\"=>365, \"name\"=>\"EID365\", \"description\"=>\"Elevated levels of CCNE1 mRNA has been shown to be correlated with increased depth of tumor invasion in gastic carcinoma.\", \"disease\"=>{\"id\"=>39, \"name\"=>\"Gastric Adenosquamous Carcinoma\", \"display_name\"=>\"Gastric Adenosquamous Carcinoma\", \"doid\"=>\"5635\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5635\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>222, \"name\"=>\"Cyclin E in human cancers.\", \"citation\"=>\"Donnellan et al., 1999, FASEB J.\", \"pubmed_id\"=>\"10224221\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/10224221\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1999, \"month\"=>5}, \"journal\"=>\"FASEB J.\", \"full_journal_title\"=>\"FASEB journal : official publication of the Federation of American Societies for Experimental Biology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>24, \"phenotypes\"=>[]}, {\"id\"=>364, \"name\"=>\"EID364\", \"description\"=>\"Elevated levels of CCNE1 mRNA has been shown to be correlated with advanced stage gastric carcinoma.\", \"disease\"=>{\"id\"=>39, \"name\"=>\"Gastric Adenosquamous Carcinoma\", \"display_name\"=>\"Gastric Adenosquamous Carcinoma\", \"doid\"=>\"5635\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5635\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>222, \"name\"=>\"Cyclin E in human cancers.\", \"citation\"=>\"Donnellan et al., 1999, FASEB J.\", \"pubmed_id\"=>\"10224221\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/10224221\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1999, \"month\"=>5}, \"journal\"=>\"FASEB J.\", \"full_journal_title\"=>\"FASEB journal : official publication of the Federation of American Societies for Experimental Biology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>24, \"phenotypes\"=>[]}, {\"id\"=>363, \"name\"=>\"EID363\", \"description\"=>\"Patients with high expression of total cyclin E and low-molecular weight cyclin E is strongly associated with poor prognosis in breast cancer, and the hazard ratio for these patients in 13.3 times higher than those with normal cyclin E levels.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>221, \"name\"=>\"Cyclin E and survival in patients with breast cancer.\", \"citation\"=>\"Keyomarsi et al., 2002, N. Engl. J. Med.\", \"pubmed_id\"=>\"12432043\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/12432043\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2002, \"month\"=>11, \"day\"=>14}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>24, \"phenotypes\"=>[]}, {\"id\"=>1734, \"name\"=>\"EID1734\", \"description\"=>\"Four ovarian cancer cell lines with elevated CCNE1 expression were 40x more sensitive to Cdk2 inhibitor SNS-032 than four cell lines without CCNE1 overexpression. SNS-032 greatly prolonged the survival of mice with CCNE1 overexpressing xenografts.\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>412, \"name\"=>\"BMS-387032 (SNS-032)\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1177, \"name\"=>\"Cyclin-dependent kinase 2 is an ideal target for ovary tumors with elevated cyclin E1 expression.\", \"citation\"=>\"Yang et al., 2015, Oncotarget\", \"pubmed_id\"=>\"26204491\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26204491\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4673230\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>8, \"day\"=>28}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>24, \"phenotypes\"=>[]}]}, {\"id\"=>25, \"entrez_name\"=>\"CDK4\", \"entrez_id\"=>1019, \"name\"=>\"EXPRESSION\", \"description\"=>\"CDK4, along with its partner CDK6, are key players in cell cycle progression. The complex has been implicated in a number of cancer types, and is the focus of therapeutic research and development. One targeted therapy for CDK inhibition is palbociclib, which may slow the growth of advanced stage breast cancers. It has also been shown, in mouse, that CDK inhibition may sensitize mutant PIK3CA tumors to PI3K inhibitors.\", \"gene_id\"=>13, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>8.0, \"coordinates\"=>{\"chromosome\"=>\"12\", \"start\"=>58141510, \"stop\"=>58146304, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000257904.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/13/summary/variants/25/summary#variant\", \"evidence_items\"=>[{\"id\"=>4872, \"name\"=>\"EID4872\", \"description\"=>\"A preclinical study to explore the efficacy of CDK4/6 inhibition with palbociclib, in a panel of sarcoma cell lines and sarcoma tumor xenografts (PDXs). The responsiveness of these cell lines correlated with their levels of CDK4 mRNA as well as in vivo PDX, but not against sarcomas displaying low levels of CDK4 and high levels of p16ink4a (CDKN2A).\", \"disease\"=>{\"id\"=>105, \"name\"=>\"Sarcoma\", \"display_name\"=>\"Sarcoma\", \"doid\"=>\"1115\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1115\"}, \"drugs\"=>[{\"id\"=>429, \"name\"=>\"Palbociclib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2168, \"name\"=>\"Efficacy of CDK4 inhibition against sarcomas depends on their levels of CDK4 and p16ink4 mRNA.\", \"citation\"=>\"Perez et al., 2015, Oncotarget\", \"pubmed_id\"=>\"26528855\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26528855\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4747352\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>12, \"day\"=>1}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>25, \"phenotypes\"=>[]}, {\"id\"=>345, \"name\"=>\"EID345\", \"description\"=>\"Palbociclib has been shown to dramatically improve progression-free survival by over 18 months in patients with ER+ Breast Cancer.\", \"disease\"=>{\"id\"=>28, \"name\"=>\"Estrogen-receptor Positive Breast Cancer\", \"display_name\"=>\"Estrogen-receptor Positive Breast Cancer\", \"doid\"=>\"0060075\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060075\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"rejected\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>212, \"name\"=>\"PD-0332991, a potent and selective inhibitor of cyclin-dependent kinase 4/6, demonstrates inhibition of proliferation in renal cell carcinoma at nanomolar concentrations and molecular markers predict for sensitivity.\", \"citation\"=>\"Logan et al., 2013, Anticancer Res.\", \"pubmed_id\"=>\"23898052\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23898052\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>8}, \"journal\"=>\"Anticancer Res.\", \"full_journal_title\"=>\"Anticancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>25, \"phenotypes\"=>[]}, {\"id\"=>264, \"name\"=>\"EID264\", \"description\"=>\"In mouse xenograft studies of PIK3CA mutant breast cancers, the combination of PI3K and CDK4/6 inhibitors overcomes intrinsic and adaptive resistance leading to tumor regressions.\", \"disease\"=>{\"id\"=>28, \"name\"=>\"Estrogen-receptor Positive Breast Cancer\", \"display_name\"=>\"Estrogen-receptor Positive Breast Cancer\", \"doid\"=>\"0060075\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060075\"}, \"drugs\"=>[{\"id\"=>339, \"name\"=>\"BYL719 (Alpelisib)\", \"pubchem_id\"=>\"\"}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>178, \"name\"=>\"CDK 4/6 inhibitors sensitize PIK3CA mutant breast cancer to PI3K inhibitors.\", \"citation\"=>\"Vora et al., 2014, Cancer Cell\", \"pubmed_id\"=>\"25002028\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25002028\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4155598\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>7, \"day\"=>14}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>25, \"phenotypes\"=>[]}]}, {\"id\"=>27, \"entrez_name\"=>\"CDKN2A\", \"entrez_id\"=>1029, \"name\"=>\"PROMOTER HYPERMETHYLATION\", \"description\"=>\"CDKN2A loss has been shown to be a significant event in a number of cancer types. One mechanism by which this can occur is by hypermethylation of the CDKN2A promoter region. While no targeted therapeutic has been engaged in clinical trials, the prognostic impact has been studied by a number of meta-analyses. The prognostic impact of promoter hypermethylation has been relatively ambiguous. Many studies have shown significant p-values suggesting poorer prognostic outcomes for patients with hypermethylation in colorectal, liver, and younger lung cancer patients. This being said, there is still research to be done before this becomes a widely-accepted prognostic indicator.\", \"gene_id\"=>14, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>21974827, \"stop\"=>21994591, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000498124.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/14/summary/variants/27/summary#variant\", \"evidence_items\"=>[{\"id\"=>367, \"name\"=>\"EID367\", \"description\"=>\"In NSCLC patients younger than 60 years at age of diagnosis, p16 (CDKN2A) promoter hypermethylation is associated with shorter time to recurrence and shorter overall survival.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>18, \"name\"=>\"CDKN2A (p16) promoter hypermethylation influences the outcome in young lung cancer patients.\", \"citation\"=>\"Bradly et al., 2012, Diagn. Mol. Pathol.\", \"pubmed_id\"=>\"23111194\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23111194\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>12}, \"journal\"=>\"Diagn. Mol. Pathol.\", \"full_journal_title\"=>\"Diagnostic molecular pathology : the American journal of surgical pathology, part B\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>27, \"phenotypes\"=>[]}]}, {\"id\"=>57, \"entrez_name\"=>\"GATA2\", \"entrez_id\"=>2624, \"name\"=>\"EXPRESSION\", \"description\"=>\"GATA2 misregulation has been observed in a number of hematologic malignancies, as well as non-small cell lung cancer. Treatment of over-active GATA2 pathways using the proteasome inhibitor bortezomib has shown dramatic tumor regression in lung cancer.\", \"gene_id\"=>25, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>4.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>128198270, \"stop\"=>128212028, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000341105.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/25/summary/variants/57/summary#variant\", \"evidence_items\"=>[{\"id\"=>301, \"name\"=>\"EID301\", \"description\"=>\"Targeting overactive GATA2-mediated pathways with inhibitors bortezomib and fasudil led to dramatic tumor regression.\", \"disease\"=>{\"id\"=>30, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>59, \"name\"=>\"Bortezomib\", \"pubchem_id\"=>nil}, {\"id\"=>60, \"name\"=>\"Fasudil\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>35, \"name\"=>\"The GATA2 transcriptional network is requisite for RAS oncogene-driven non-small cell lung cancer.\", \"citation\"=>\"Kumar et al., 2012, Cell\", \"pubmed_id\"=>\"22541434\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22541434\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>4, \"day\"=>27}, \"journal\"=>\"Cell\", \"full_journal_title\"=>\"Cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>57, \"phenotypes\"=>[]}]}, {\"id\"=>85, \"entrez_name\"=>\"MGMT\", \"entrez_id\"=>4255, \"name\"=>\"PROMOTER METHYLATION\", \"description\"=>\"MGMT promoter methylation has been observed to impact tumor progression in glioblastoma multiforme. In patients exhibiting promoter methylation, the akylating agent carmustine has shown efficacy. In patients lacking methylation, combining carmustine with the MGMT inhibitor O(6)-benzylguanine may be effective, but more experiments are required. Clinical trials have also shown selective sensitivity of promoter methylation-positive patients to temozolomide, making a case for wider methylation screening in GBM patients.\", \"gene_id\"=>34, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>136.0, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>131264495, \"stop\"=>131265656, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000306010.7\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/34/summary/variants/85/summary#variant\", \"evidence_items\"=>[{\"id\"=>490, \"name\"=>\"EID490\", \"description\"=>\"In a study of 152 anaplastic oligodendroglial tumors, 121 samples had MGMT promoter methylation. Those with promoter methylation had better progression-free and overall survival.\", \"disease\"=>{\"id\"=>241, \"name\"=>\"Anaplastic Oligodendroglioma\", \"display_name\"=>\"Anaplastic Oligodendroglioma\", \"doid\"=>\"3181\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3181\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>298, \"name\"=>\"MGMT promoter methylation is prognostic but not predictive for outcome to adjuvant PCV chemotherapy in anaplastic oligodendroglial tumors: a report from EORTC Brain Tumor Group Study 26951.\", \"citation\"=>\"van den Bent et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19901104\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19901104\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2793037\", \"publication_date\"=>{\"year\"=>2009, \"month\"=>12, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>85, \"phenotypes\"=>[]}, {\"id\"=>491, \"name\"=>\"EID491\", \"description\"=>\"In patients with glioblastoma, those with methylated MGMT promoters have better overall survival than those with unmethylated promoters.\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>299, \"name\"=>\"MGMT gene silencing and benefit from temozolomide in glioblastoma.\", \"citation\"=>\"Hegi et al., 2005, N. Engl. J. Med.\", \"pubmed_id\"=>\"15758010\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15758010\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>3, \"day\"=>10}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>85, \"phenotypes\"=>[]}, {\"id\"=>309, \"name\"=>\"EID309\", \"description\"=>\"In patients lacking MGMT promoter methylation, the use of an MGMT inhibitor, such as O(6)-benzylguanine, in combination with alkylating agents (Carmustine) may be useful in treating patients with initial resistance to alkylating agents.\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[{\"id\"=>65, \"name\"=>\"O(6)-benzylguanine\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"E\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>139, \"name\"=>\"Inactivation of the DNA-repair gene MGMT and the clinical response of gliomas to alkylating agents.\", \"citation\"=>\"Esteller et al., 2000, N. Engl. J. Med.\", \"pubmed_id\"=>\"11070098\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/11070098\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2000, \"month\"=>11, \"day\"=>9}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>85, \"phenotypes\"=>[]}, {\"id\"=>308, \"name\"=>\"EID308\", \"description\"=>\"MGMT promoter methylation has been shown to confer sensitivity to alkylating agents, such as carmustine, in glioblastoma cells.\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[{\"id\"=>64, \"name\"=>\"Carmustine\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>139, \"name\"=>\"Inactivation of the DNA-repair gene MGMT and the clinical response of gliomas to alkylating agents.\", \"citation\"=>\"Esteller et al., 2000, N. Engl. J. Med.\", \"pubmed_id\"=>\"11070098\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/11070098\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2000, \"month\"=>11, \"day\"=>9}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>85, \"phenotypes\"=>[]}, {\"id\"=>307, \"name\"=>\"EID307\", \"description\"=>\"In a randomized clinical trial, patients with MGMT promoter methyaltion benefitted from temozolomide. This benefit was also methylation status dependent, as those without methylation did not see increased survival.\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[{\"id\"=>11, \"name\"=>\"Temozolomide\", \"pubchem_id\"=>\"\"}], \"rating\"=>5, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>194, \"name\"=>\"MGMT gene silencing and benefit from temozolomide in glioblastoma.\", \"citation\"=>\"Hegi et al., 2005, N. Engl. J. Med.\", \"pubmed_id\"=>\"15758010\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15758010\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>3, \"day\"=>10}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>85, \"phenotypes\"=>[]}, {\"id\"=>172, \"name\"=>\"EID172\", \"description\"=>\"MGMT promoter methylation was associated with increased overall and disease-free survival as well as tumor regression in patients with gliomas.\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>139, \"name\"=>\"Inactivation of the DNA-repair gene MGMT and the clinical response of gliomas to alkylating agents.\", \"citation\"=>\"Esteller et al., 2000, N. Engl. J. Med.\", \"pubmed_id\"=>\"11070098\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/11070098\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2000, \"month\"=>11, \"day\"=>9}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>85, \"phenotypes\"=>[]}]}, {\"id\"=>152, \"entrez_name\"=>\"BCL2\", \"entrez_id\"=>596, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>59, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>40.0, \"coordinates\"=>{\"chromosome\"=>\"18\", \"start\"=>60790579, \"stop\"=>60987361, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000398117.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/59/summary/variants/152/summary#variant\", \"evidence_items\"=>[{\"id\"=>424, \"name\"=>\"EID424\", \"description\"=>\"In patients with Diffuse Large B-Cell Lymphoma, those with increased levels of BCL2 were shown to have decreased OS, DFS, and RFS.\", \"disease\"=>{\"id\"=>45, \"name\"=>\"Diffuse Large B-cell Lymphoma\", \"display_name\"=>\"Diffuse Large B-cell Lymphoma\", \"doid\"=>\"0050745\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050745\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>251, \"name\"=>\"Prognostic significance of Bcl-2 protein expression and Bcl-2 gene rearrangement in diffuse aggressive non-Hodgkin's lymphoma.\", \"citation\"=>\"Gascoyne et al., 1997, Blood\", \"pubmed_id\"=>\"9207459\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/9207459\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1997, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>152, \"phenotypes\"=>[]}, {\"id\"=>423, \"name\"=>\"EID423\", \"description\"=>\"In patients with Diffuse Large B-Cell Lymphoma, those with increased levels of BCL2 were shown to have decreased overall, disease-free, and relapse-free survival.\", \"disease\"=>{\"id\"=>45, \"name\"=>\"Diffuse Large B-cell Lymphoma\", \"display_name\"=>\"Diffuse Large B-cell Lymphoma\", \"doid\"=>\"0050745\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050745\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>251, \"name\"=>\"Prognostic significance of Bcl-2 protein expression and Bcl-2 gene rearrangement in diffuse aggressive non-Hodgkin's lymphoma.\", \"citation\"=>\"Gascoyne et al., 1997, Blood\", \"pubmed_id\"=>\"9207459\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/9207459\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1997, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>152, \"phenotypes\"=>[]}]}, {\"id\"=>156, \"entrez_name\"=>\"IGF2\", \"entrez_id\"=>3481, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>60, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>6.0, \"coordinates\"=>{\"chromosome\"=>\"11\", \"start\"=>2150348, \"stop\"=>2170833, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000300632.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/60/summary/variants/156/summary#variant\", \"evidence_items\"=>[{\"id\"=>414, \"name\"=>\"EID414\", \"description\"=>\"OSI-906, an IGF1R/INSR inhibitor, sensitizes cell lines to docetaxel and cabazitaxel, resulting in better overall survival in mouse xenografts of patient derived chemotherapy resistant castration-resistant prostate cancer cell lines.\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[{\"id\"=>72, \"name\"=>\"OSI-906 (Linsitinib)\", \"pubchem_id\"=>nil}, {\"id\"=>73, \"name\"=>\"Docetaxel\", \"pubchem_id\"=>nil}, {\"id\"=>74, \"name\"=>\"Cabazitaxel\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>242, \"name\"=>\"A targetable GATA2-IGF2 axis confers aggressiveness in lethal prostate cancer.\", \"citation\"=>\"Vidal et al., 2015, Cancer Cell\", \"pubmed_id\"=>\"25670080\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25670080\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4356948\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>2, \"day\"=>9}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>156, \"phenotypes\"=>[]}, {\"id\"=>412, \"name\"=>\"EID412\", \"description\"=>\"OSI-906, an IGF1R/INSR inhibitor, does not have a significant effect on chemotherapy resistant castration-resistant prostate cancer cell lines.\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[{\"id\"=>72, \"name\"=>\"OSI-906 (Linsitinib)\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"N/A\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>242, \"name\"=>\"A targetable GATA2-IGF2 axis confers aggressiveness in lethal prostate cancer.\", \"citation\"=>\"Vidal et al., 2015, Cancer Cell\", \"pubmed_id\"=>\"25670080\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25670080\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4356948\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>2, \"day\"=>9}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>156, \"phenotypes\"=>[]}]}, {\"id\"=>158, \"entrez_name\"=>\"AURKA\", \"entrez_id\"=>6790, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>61, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>30.0, \"coordinates\"=>{\"chromosome\"=>\"20\", \"start\"=>54944446, \"stop\"=>54967393, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000395913.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/61/summary/variants/158/summary#variant\", \"evidence_items\"=>[{\"id\"=>1959, \"name\"=>\"EID1959\", \"description\"=>\"Ectopic expression of Aurora-A decreases and depletion of Aurora-A enhances tamoxifen sensitivity in ERα-positive breast cancer. Elevated Aurora-A was significantly associated with the recurrence of ERα-positive tumours. Notably, Aurora-A inhibitor MLN8237, which is currently in clinical trial, synergizes with tamoxifen and overcomes tamoxifen resistance\", \"disease\"=>{\"id\"=>28, \"name\"=>\"Estrogen-receptor Positive Breast Cancer\", \"display_name\"=>\"Estrogen-receptor Positive Breast Cancer\", \"doid\"=>\"0060075\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060075\"}, \"drugs\"=>[{\"id\"=>53, \"name\"=>\"Tamoxifen\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1378, \"name\"=>\"Aurora-A is a determinant of tamoxifen sensitivity through phosphorylation of ERα in breast cancer.\", \"citation\"=>\"Zheng et al., 2014, Oncogene\", \"pubmed_id\"=>\"24166501\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24166501\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4002670\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>10, \"day\"=>16}, \"journal\"=>\"Oncogene\", \"full_journal_title\"=>\"Oncogene\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>158, \"phenotypes\"=>[]}, {\"id\"=>1653, \"name\"=>\"EID1653\", \"description\"=>\"A retrospective study examining the expression of AURKA in tumor samples of 78 esophageal squamous cell carcinoma patients found that positive expression of AURKA (>10% of cells by IHC) was correlated with increased clinical response to chemoradiation therapy consisting of cisplatin, 5-fluorouracil, and 40 gy radiation (P = 0.0003).\", \"disease\"=>{\"id\"=>149, \"name\"=>\"Esophagus Squamous Cell Carcinoma\", \"display_name\"=>\"Esophagus Squamous Cell Carcinoma\", \"doid\"=>\"3748\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3748\"}, \"drugs\"=>[{\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}, {\"id\"=>395, \"name\"=>\"5-fluorouracil\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1117, \"name\"=>\"Correlation of Aurora-A expression with the effect of chemoradiation therapy on esophageal squamous cell carcinoma.\", \"citation\"=>\"Tamotsu et al., 2015, BMC Cancer\", \"pubmed_id\"=>\"25924824\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25924824\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4423148\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>4, \"day\"=>29}, \"journal\"=>\"BMC Cancer\", \"full_journal_title\"=>\"BMC cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>158, \"phenotypes\"=>[]}, {\"id\"=>416, \"name\"=>\"EID416\", \"description\"=>\"In patients with ovarian carcinomas, those with Aurora kinase A expression (as measured by immunohistochemistry) had increased progression-free and overall survival compared to those without expression.\", \"disease\"=>{\"id\"=>47, \"name\"=>\"Ovarian Carcinoma\", \"display_name\"=>\"Ovarian Carcinoma\", \"doid\"=>\"4001\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4001\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>246, \"name\"=>\"Aurora kinases as prognostic biomarkers in ovarian carcinoma.\", \"citation\"=>\"Mendiola et al., 2009, Hum. Pathol.\", \"pubmed_id\"=>\"19157502\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19157502\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>5}, \"journal\"=>\"Hum. Pathol.\", \"full_journal_title\"=>\"Human pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>158, \"phenotypes\"=>[]}]}, {\"id\"=>179, \"entrez_name\"=>\"ATM\", \"entrez_id\"=>472, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>69, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>54.0, \"coordinates\"=>{\"chromosome\"=>\"11\", \"start\"=>108093559, \"stop\"=>108239826, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000278616.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/69/summary/variants/179/summary#variant\", \"evidence_items\"=>[{\"id\"=>5215, \"name\"=>\"EID5215\", \"description\"=>\"A randomized phase 2 trial investigating the efficiency of Olaparib plus Paclitaxel vs. Placebo plus Paclitaxel in recurrent or metastatic gastric cancer patients. 123 patients received treatment. Combination treatment did not significantly improve ORR (26,4% vs 19,1%) or PFS (3,91m vs 3,55 m) but OS (13,8 m vs. 8,3 m) in the overall cohort.\\n\\nA subcohort of ATM low expressing tumours (14%) experienced improvement of OS (not reached vs. 8,2 m) and a trend towards improved PFS (5,29 m vs 3,68 m).\", \"disease\"=>{\"id\"=>21, \"name\"=>\"Stomach Cancer\", \"display_name\"=>\"Stomach Cancer\", \"doid\"=>\"10534\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10534\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2209, \"name\"=>\"Randomized, Double-Blind Phase II Trial With Prospective Classification by ATM Protein Level to Evaluate the Efficacy and Tolerability of Olaparib Plus Paclitaxel in Patients With Recurrent or Metastatic Gastric Cancer.\", \"citation\"=>\"Bang et al., 2015, J. Clin. Oncol.\", \"pubmed_id\"=>\"26282658\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26282658\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>11, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>179, \"phenotypes\"=>[]}, {\"id\"=>1292, \"name\"=>\"EID1292\", \"description\"=>\"Absent expression of ATM, as assessed by IHC in 133 pancreatic ductal adenocarcinomas, was found to be associated with dismal prognosis (median survival, 9.6 months), whereas expression of ATM was associated with increased survival (17.7 months, P = 0.001). Furthermore loss of expression of nuclear phosphorylated Ser1981-ATM (IHC) was indicative of the worst prognosis (median survival, 10.8 vs 14.3 months [low expression] vs 31.1 months [high expression], P < 0.001).\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>888, \"name\"=>\"Ataxia-telangiectasia-mutated protein kinase levels stratify patients with pancreatic adenocarcinoma into prognostic subgroups with loss being a strong indicator of poor survival.\", \"citation\"=>\"Kamphues et al., 2015, Pancreas\", \"pubmed_id\"=>\"25423555\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25423555\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>3}, \"journal\"=>\"Pancreas\", \"full_journal_title\"=>\"Pancreas\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>179, \"phenotypes\"=>[]}, {\"id\"=>1564, \"name\"=>\"EID1564\", \"description\"=>\"43 B-CLL patients with LoH or reduced protein levels (<50% normal expression, radioimmunoassay, western blot) of ATM were found to have significantly shorter overall survival (35.6 vs. 97.3 months; P=0.003) when compared to 108 patients with normal expression levels.\", \"disease\"=>{\"id\"=>52, \"name\"=>\"Chronic Lymphocytic Leukemia\", \"display_name\"=>\"Chronic Lymphocytic Leukemia\", \"doid\"=>\"1040\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1040\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1033, \"name\"=>\"Deficiency of the ATM protein expression defines an aggressive subgroup of B-cell chronic lymphocytic leukemia.\", \"citation\"=>\"Starostik et al., 1998, Cancer Res.\", \"pubmed_id\"=>\"9788599\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/9788599\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1998, \"month\"=>10, \"day\"=>15}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>179, \"phenotypes\"=>[]}, {\"id\"=>878, \"name\"=>\"EID878\", \"description\"=>\"Low ATM protein expression led to sensitivity to PARP inhibitors in TP53 -ve gastric cancer cell lines (8 lines were used for various aspects of this study).\", \"disease\"=>{\"id\"=>147, \"name\"=>\"Gastric Adenocarcinoma\", \"display_name\"=>\"Gastric Adenocarcinoma\", \"doid\"=>\"3717\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3717\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>591, \"name\"=>\"Low ATM protein expression and depletion of p53 correlates with olaparib sensitivity in gastric cancer cell lines.\", \"citation\"=>\"Kubota et al., 2014, Cell Cycle\", \"pubmed_id\"=>\"24841718\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24841718\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4111704\", \"publication_date\"=>{\"year\"=>2014}, \"journal\"=>\"Cell Cycle\", \"full_journal_title\"=>\"Cell cycle (Georgetown, Tex.)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>179, \"phenotypes\"=>[]}, {\"id\"=>454, \"name\"=>\"EID454\", \"description\"=>\"ATM-deficient H1395 and HT144 cancer cell lines showed growth inhibition when treated with DNA PKcs inhibitors KU-0060648 and NU7441. These agents also increased the lifespan of mice transfused with ATM-deficient lymphoma cells when compared with etoposide or control.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>405, \"name\"=>\"KU-0060648\", \"pubchem_id\"=>nil}, {\"id\"=>406, \"name\"=>\"NU7441\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>270, \"name\"=>\"Therapeutic targeting of a robust non-oncogene addiction to PRKDC in ATM-defective tumors.\", \"citation\"=>\"Riabinska et al., 2013, Sci Transl Med\", \"pubmed_id\"=>\"23761041\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23761041\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>6, \"day\"=>12}, \"journal\"=>\"Sci Transl Med\", \"full_journal_title\"=>\"Science translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>179, \"phenotypes\"=>[]}]}, {\"id\"=>181, \"entrez_name\"=>\"AURKA\", \"entrez_id\"=>6790, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>61, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>38.0, \"coordinates\"=>{\"chromosome\"=>\"20\", \"start\"=>54944446, \"stop\"=>54967393, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000395913.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/61/summary/variants/181/summary#variant\", \"evidence_items\"=>[{\"id\"=>1650, \"name\"=>\"EID1650\", \"description\"=>\"A study of 41 patients with high-grade ovarian serous carcinoma found that high expression levels of AURKA correlated with poorer overall survival (P = 0.001; HR 0.14) and platinum therapy resistance (P < 0.001). Expression of AURKA was measured using immunohistochemical techniques on tumor samples.\", \"disease\"=>{\"id\"=>87, \"name\"=>\"Ovarian Serous Carcinoma\", \"display_name\"=>\"Ovarian Serous Carcinoma\", \"doid\"=>\"0050933\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050933\"}, \"drugs\"=>[{\"id\"=>103, \"name\"=>\"PLATINUM\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1114, \"name\"=>\"Aurora Kinase A expression predicts platinum-resistance and adverse outcome in high-grade serous ovarian carcinoma patients.\", \"citation\"=>\"Mignogna et al., 2016, J Ovarian Res\", \"pubmed_id\"=>\"27209210\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27209210\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4875597\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>5, \"day\"=>21}, \"journal\"=>\"J Ovarian Res\", \"full_journal_title\"=>\"Journal of ovarian research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>181, \"phenotypes\"=>[]}, {\"id\"=>871, \"name\"=>\"EID871\", \"description\"=>\"High levels of Aurora-A expression (retrospectively assessed by IHC in 102 NSCLC patients) predicted poorer overall survival and progression free survival in a population treated with surgery and cisplatin-based chemotherapy. In-vitro data confirmed these results and overexpression or knockdown of AURKA decreased or increased in-vitro sensitivity to cisplatin.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>585, \"name\"=>\"Aurora-A contributes to cisplatin resistance and lymphatic metastasis in non-small cell lung cancer and predicts poor prognosis.\", \"citation\"=>\"Xu et al., 2014, J Transl Med\", \"pubmed_id\"=>\"25082261\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25082261\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4237886\", \"publication_date\"=>{\"year\"=>2014}, \"journal\"=>\"J Transl Med\", \"full_journal_title\"=>\"Journal of translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>181, \"phenotypes\"=>[]}, {\"id\"=>457, \"name\"=>\"EID457\", \"description\"=>\"In the cervial andenocarcinoma cell line HeLa, when AURK-A was overexpressed, the cells were resistant to the apoptotic effects of Taxol.\", \"disease\"=>{\"id\"=>219, \"name\"=>\"Cervical Adenocarcinoma\", \"display_name\"=>\"Cervical Adenocarcinoma\", \"doid\"=>\"3702\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3702\"}, \"drugs\"=>[{\"id\"=>81, \"name\"=>\"Paclitaxel\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>273, \"name\"=>\"AURORA-A amplification overrides the mitotic spindle assembly checkpoint, inducing resistance to Taxol.\", \"citation\"=>\"Anand et al., 2003, Cancer Cell\", \"pubmed_id\"=>\"12559175\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/12559175\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2003, \"month\"=>1}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>181, \"phenotypes\"=>[]}]}, {\"id\"=>182, \"entrez_name\"=>\"BAP1\", \"entrez_id\"=>8314, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>70, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>52435029, \"stop\"=>52444366, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000460680.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/70/summary/variants/182/summary#variant\", \"evidence_items\"=>[{\"id\"=>458, \"name\"=>\"EID458\", \"description\"=>\"In patients with uveal melanoma, those with BAP1 expression (determined by immunohistochemical staining) had better survival.\", \"disease\"=>{\"id\"=>188, \"name\"=>\"Uveal Melanoma\", \"display_name\"=>\"Uveal Melanoma\", \"doid\"=>\"6039\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:6039\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>274, \"name\"=>\"Prognostic parameters in uveal melanoma and their association with BAP1 expression.\", \"citation\"=>\"van Essen et al., 2014, Br J Ophthalmol\", \"pubmed_id\"=>\"25147369\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25147369\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4966163\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>12}, \"journal\"=>\"Br J Ophthalmol\", \"full_journal_title\"=>\"The British journal of ophthalmology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>182, \"phenotypes\"=>[]}]}, {\"id\"=>193, \"entrez_name\"=>\"EGFR\", \"entrez_id\"=>1956, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>19, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>130.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>55086794, \"stop\"=>55279321, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000275493.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/19/summary/variants/193/summary#variant\", \"evidence_items\"=>[{\"id\"=>3797, \"name\"=>\"EID3797\", \"description\"=>\"A phase 3 study of 889 advanced non-small cell lung cancer patients (with non-progressive disease while on standard chemotherapy treatment), reported that patients harboring EGFR mutation (exon 19 deletion and L858R mutation; n= 493) who were treated with erlotinib had an improved progression free survival (44.6 weeks vs. 13 weeks, HR: 0.10, 95% CI 0.04-0.25, P<0.001) as compared to patients who received placebo.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>15, \"name\"=>\"Erlotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1938, \"name\"=>\"Prospective molecular marker analyses of EGFR and KRAS from a randomized, placebo-controlled study of erlotinib maintenance therapy in advanced non-small-cell lung cancer.\", \"citation\"=>\"Brugger et al., 2011, J. Clin. Oncol.\", \"pubmed_id\"=>\"21969500\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21969500\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>11, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>193, \"phenotypes\"=>[]}, {\"id\"=>5827, \"name\"=>\"EID5827\", \"description\"=>\"A total of 57 patients with HER1/EGFR positive non-small cell lung cancer were treated with erlotinib therapy. Two patients achieved a complete response, five had a partial response, and 22/57 patients (39%) had stable disease. Median duration of response was 19.7 weeks, median progression-free survival was 9 weeks (95% CI, 8-15 weeks), median overall survival was 8.4 months (95% CI, 4.8-13.9 months), and 1-year survival rate was 40% (95% CI, 28%-54%).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>15, \"name\"=>\"Erlotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2173, \"name\"=>\"Determinants of tumor response and survival with erlotinib in patients with non--small-cell lung cancer.\", \"citation\"=>\"Pérez-Soler et al., 2004, J. Clin. Oncol.\", \"pubmed_id\"=>\"15310767\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15310767\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2004, \"month\"=>8, \"day\"=>15}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"submitted\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>193, \"phenotypes\"=>[]}, {\"id\"=>784, \"name\"=>\"EID784\", \"description\"=>\"55 tumor samples were analyzed for EGFR expression using IHC.\\nPFS and OS was shorter for the EGFR high expressing group (18 pts) than for the low to moderate expression group (37 pts). PFS: 5.8 ± 0.5 vs. 11.0 ± 2.8 months, P = 0.007; OS: 9.7 ± 0.5 vs. 21.5 ± 1.5 months, P = 0.03.\", \"disease\"=>{\"id\"=>149, \"name\"=>\"Esophagus Squamous Cell Carcinoma\", \"display_name\"=>\"Esophagus Squamous Cell Carcinoma\", \"doid\"=>\"3748\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3748\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>522, \"name\"=>\"The relation of EGFR expression by immunohistochemical staining and clinical response of combination treatment of nimotuzumab and chemotherapy in esophageal squamous cell carcinoma.\", \"citation\"=>\"Jia et al., 2016, Clin Transl Oncol\", \"pubmed_id\"=>\"26459251\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26459251\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>6}, \"journal\"=>\"Clin Transl Oncol\", \"full_journal_title\"=>\"Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>193, \"phenotypes\"=>[]}, {\"id\"=>834, \"name\"=>\"EID834\", \"description\"=>\"Overexpression of EGFR in 55 samples of esophageal squamous cell carcinoma as assesesed by IHC, was related to poor OS and PFS  (PFS: 5.8 ± 0.5 vs. 11.0 ± 2.8 months, P = 0.007; OS: 9.7 ± 0.5 vs. 21.5 ± 1.5 months, P = 0.03).\", \"disease\"=>{\"id\"=>149, \"name\"=>\"Esophagus Squamous Cell Carcinoma\", \"display_name\"=>\"Esophagus Squamous Cell Carcinoma\", \"doid\"=>\"3748\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3748\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"rejected\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>522, \"name\"=>\"The relation of EGFR expression by immunohistochemical staining and clinical response of combination treatment of nimotuzumab and chemotherapy in esophageal squamous cell carcinoma.\", \"citation\"=>\"Jia et al., 2016, Clin Transl Oncol\", \"pubmed_id\"=>\"26459251\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26459251\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>6}, \"journal\"=>\"Clin Transl Oncol\", \"full_journal_title\"=>\"Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>193, \"phenotypes\"=>[]}, {\"id\"=>783, \"name\"=>\"EID783\", \"description\"=>\"55 tumor samples were analyzed for EGFR expression using IHC. The objective response rate to anti-EGFR antibody nimotuzumab did not differ significantly between EGFR high- (18 pts) and EGFR low to moderate groups (37 pts).\", \"disease\"=>{\"id\"=>149, \"name\"=>\"Esophagus Squamous Cell Carcinoma\", \"display_name\"=>\"Esophagus Squamous Cell Carcinoma\", \"doid\"=>\"3748\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3748\"}, \"drugs\"=>[{\"id\"=>188, \"name\"=>\"Nimotuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>522, \"name\"=>\"The relation of EGFR expression by immunohistochemical staining and clinical response of combination treatment of nimotuzumab and chemotherapy in esophageal squamous cell carcinoma.\", \"citation\"=>\"Jia et al., 2016, Clin Transl Oncol\", \"pubmed_id\"=>\"26459251\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26459251\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>6}, \"journal\"=>\"Clin Transl Oncol\", \"full_journal_title\"=>\"Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>193, \"phenotypes\"=>[]}, {\"id\"=>954, \"name\"=>\"EID954\", \"description\"=>\"82 patients treated with gefitinib were retrospectively analyzed for EGFR mRNA expression (rt-PCR in 64 patients). Patients with high EGFR mRNA expression (>5.01) had 43% response probability, whereas patients with low EGFR mRNA expression had 8% response probability (P = 0.006). PFS but not OS was also longer in patients with high EGFR mRNA expression.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>14, \"name\"=>\"Gefitinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>656, \"name\"=>\"Epidermal growth factor receptor messenger RNA expression, gene dosage, and gefitinib sensitivity in non-small cell lung cancer.\", \"citation\"=>\"Dziadziuszko et al., 2006, Clin. Cancer Res.\", \"pubmed_id\"=>\"16707605\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16707605\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2006, \"month\"=>5, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>193, \"phenotypes\"=>[]}, {\"id\"=>833, \"name\"=>\"EID833\", \"description\"=>\"Expression of EGFR in 55 samples of esophageal squamous cell carcinoma with IHC was not predictive of response to EGFR inhibitor nimotuzumab. EGFR overexpression was related to poor OS and PFS.\", \"disease\"=>{\"id\"=>149, \"name\"=>\"Esophagus Squamous Cell Carcinoma\", \"display_name\"=>\"Esophagus Squamous Cell Carcinoma\", \"doid\"=>\"3748\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3748\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"rejected\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>522, \"name\"=>\"The relation of EGFR expression by immunohistochemical staining and clinical response of combination treatment of nimotuzumab and chemotherapy in esophageal squamous cell carcinoma.\", \"citation\"=>\"Jia et al., 2016, Clin Transl Oncol\", \"pubmed_id\"=>\"26459251\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26459251\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>6}, \"journal\"=>\"Clin Transl Oncol\", \"full_journal_title\"=>\"Clinical & translational oncology : official publication of the Federation of Spanish Oncology Societies and of the National Cancer Institute of Mexico\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>193, \"phenotypes\"=>[]}, {\"id\"=>782, \"name\"=>\"EID782\", \"description\"=>\"Primary tumors of 91 patients with head and neck squamous cell carcinoma were analyzed for expression of EGFR and TGF-alpha using IHC. Expression of EGFR or TGF-alpha were associated with reduced disease-free and cause-specific survival.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>521, \"name\"=>\"Levels of TGF-alpha and EGFR protein in head and neck squamous cell carcinoma and patient survival.\", \"citation\"=>\"Rubin Grandis et al., 1998, J. Natl. Cancer Inst.\", \"pubmed_id\"=>\"9625170\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/9625170\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1998, \"month\"=>6, \"day\"=>3}, \"journal\"=>\"J. Natl. Cancer Inst.\", \"full_journal_title\"=>\"Journal of the National Cancer Institute\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>193, \"phenotypes\"=>[]}, {\"id\"=>693, \"name\"=>\"EID693\", \"description\"=>\"EGFR overexpression was correlated with longer overall survival in patients with advanced NSCLC treated with first line chemotherapy and cetuximab than those treated with first line chemotherapy alone.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>145, \"name\"=>\"EGFR Inhibitor\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>452, \"name\"=>\"EGFR expression as a predictor of survival for first-line chemotherapy plus cetuximab in patients with advanced non-small-cell lung cancer: analysis of data from the phase 3 FLEX study.\", \"citation\"=>\"Pirker et al., 2012, Lancet Oncol.\", \"pubmed_id\"=>\"22056021\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22056021\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>1}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>193, \"phenotypes\"=>[]}, {\"id\"=>737, \"name\"=>\"EID737\", \"description\"=>\"In this prospective study in 37 patients of whom 34 were treated with different EGFR inhibitors (gefitinib, erlotinib, afatinib), EGFR mutations and a higher relative expression of EGFR in comparison to MET expression were correlated with longer PFS.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>145, \"name\"=>\"EGFR Inhibitor\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>485, \"name\"=>\"Low EGFR/MET ratio is associated with resistance to EGFR inhibitors in non-small cell lung cancer.\", \"citation\"=>\"Park et al., 2015, Oncotarget\", \"pubmed_id\"=>\"26439803\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26439803\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4741578\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>10, \"day\"=>13}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>193, \"phenotypes\"=>[]}, {\"id\"=>692, \"name\"=>\"EID692\", \"description\"=>\"EGFR expression level was not predicitive of response to cetuximab-containing first line regimens in recurrent or metastatic head and neck squamous cell carcinoma and KRAS-wild type colorectal carcinoma.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>451, \"name\"=>\"Predictive value of epidermal growth factor receptor expression for first-line chemotherapy plus cetuximab in patients with head and neck and colorectal cancer: analysis of data from the EXTREME and CRYSTAL studies.\", \"citation\"=>\"Licitra et al., 2013, Eur. J. Cancer\", \"pubmed_id\"=>\"23265711\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23265711\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>4}, \"journal\"=>\"Eur. J. Cancer\", \"full_journal_title\"=>\"European journal of cancer (Oxford, England : 1990)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>193, \"phenotypes\"=>[]}, {\"id\"=>474, \"name\"=>\"EID474\", \"description\"=>\"In patients with glioblastoma multiforme, those with overespression of wild-type EGFR had shorter overall survival.\", \"disease\"=>{\"id\"=>231, \"name\"=>\"Supratentorial Glioblastoma Multiforme\", \"display_name\"=>\"Supratentorial Glioblastoma Multiforme\", \"doid\"=>\"3073\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3073\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>288, \"name\"=>\"Prognostic value of epidermal growth factor receptor in patients with glioblastoma multiforme.\", \"citation\"=>\"Shinojima et al., 2003, Cancer Res.\", \"pubmed_id\"=>\"14583498\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/14583498\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2003, \"month\"=>10, \"day\"=>15}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>193, \"phenotypes\"=>[]}]}, {\"id\"=>205, \"entrez_name\"=>\"NCOA3\", \"entrez_id\"=>8202, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>74, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"20\", \"start\"=>46130646, \"stop\"=>46285621, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000372004.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/74/summary/variants/205/summary#variant\", \"evidence_items\"=>[{\"id\"=>493, \"name\"=>\"EID493\", \"description\"=>\"High expression determined by immunohistochemistry of NCOA3 was found in 571 out of 1,836 breast cancer samples. Those with high expression had worse overall and disease-specific survival.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>300, \"name\"=>\"Prognostic relevance of AIB1 (NCoA3) amplification and overexpression in breast cancer.\", \"citation\"=>\"Burandt et al., 2013, Breast Cancer Res. Treat.\", \"pubmed_id\"=>\"23322234\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23322234\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>2}, \"journal\"=>\"Breast Cancer Res. Treat.\", \"full_journal_title\"=>\"Breast cancer research and treatment\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>205, \"phenotypes\"=>[]}]}, {\"id\"=>210, \"entrez_name\"=>\"PGR\", \"entrez_id\"=>5241, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>76, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>70.0, \"coordinates\"=>{\"chromosome\"=>\"11\", \"start\"=>100900355, \"stop\"=>101001255, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000325455.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/76/summary/variants/210/summary#variant\", \"evidence_items\"=>[{\"id\"=>500, \"name\"=>\"EID500\", \"description\"=>\"In patients with breast cancer, those with expression of PgR (determined by immunohistochemical staining) had better relapse-free survival than those without expression.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>306, \"name\"=>\"Benefit from adjuvant tamoxifen therapy in primary breast cancer patients according oestrogen receptor, progesterone receptor, EGF receptor and HER2 status.\", \"citation\"=>\"Dowsett et al., 2006, Ann. Oncol.\", \"pubmed_id\"=>\"16497822\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16497822\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2006, \"month\"=>5}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology / ESMO\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>210, \"phenotypes\"=>[]}, {\"id\"=>501, \"name\"=>\"EID501\", \"description\"=>\"In patients with ER-Positive breast cancer, expression of PgR does not effect the outcome of treatment with tamoxifen.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>53, \"name\"=>\"Tamoxifen\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"N/A\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>307, \"name\"=>\"Biomarkers for the clinical management of breast cancer: international perspective.\", \"citation\"=>\"Patani et al., 2013, Int. J. Cancer\", \"pubmed_id\"=>\"23280579\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23280579\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>7}, \"journal\"=>\"Int. J. Cancer\", \"full_journal_title\"=>\"International journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>210, \"phenotypes\"=>[]}, {\"id\"=>502, \"name\"=>\"EID502\", \"description\"=>\"In patients with breast cancer, those with low expression of PgR treated with tamoxifen had the same disease-free survial as those with high expression of PgR.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>53, \"name\"=>\"Tamoxifen\", \"pubchem_id\"=>nil}, {\"id\"=>82, \"name\"=>\"Exemestane\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"N/A\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>308, \"name\"=>\"Estrogen receptor and progesterone receptor as predictive biomarkers of response to endocrine therapy: a prospectively powered pathology study in the Tamoxifen and Exemestane Adjuvant Multinational trial.\", \"citation\"=>\"Bartlett et al., 2011, J. Clin. Oncol.\", \"pubmed_id\"=>\"21422407\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21422407\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3082973\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>4, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>210, \"phenotypes\"=>[]}, {\"id\"=>503, \"name\"=>\"EID503\", \"description\"=>\"In patients with breast cancer, those with low expression of PgR had worse disease-free survival than those with high expression.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>308, \"name\"=>\"Estrogen receptor and progesterone receptor as predictive biomarkers of response to endocrine therapy: a prospectively powered pathology study in the Tamoxifen and Exemestane Adjuvant Multinational trial.\", \"citation\"=>\"Bartlett et al., 2011, J. Clin. Oncol.\", \"pubmed_id\"=>\"21422407\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21422407\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3082973\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>4, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>210, \"phenotypes\"=>[]}]}, {\"id\"=>268, \"entrez_name\"=>\"FGFR1\", \"entrez_id\"=>2260, \"name\"=>\"EXPRESSION\", \"description\"=>\"FGFR1 high expression has been reported in 11.8% (178 patients) of oral squamous cell carcinoma patients, and was enriched in early stages (Freier et. al., 2007). Another study reports FGFR1 expression in 35% of primary HNSSC (353 patients) (Göke et. al., 2015). In both studies, FGFR1 gene amplification and expression do not correlate well with about 35% of samples with FGFR1 amplification overexpressing FGFR1 (Göke et. al., 2015). FGFR1 expression also correlated with sensitivity to FGFR1 inhibitors in preclinical models of lung cancer (Wynes et. al., 2014), HNSSC (Göke et. al., 2015) and urothelial carcinoma (Lamont et. al., 2011).\", \"gene_id\"=>1885, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>46.0, \"coordinates\"=>{\"chromosome\"=>\"8\", \"start\"=>38268656, \"stop\"=>38325363, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000425967.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1885/summary/variants/268/summary#variant\", \"evidence_items\"=>[{\"id\"=>2915, \"name\"=>\"EID2915\", \"description\"=>\"Targeting of FGFR1 by RNA interference and small-molecule inhibitors (PD173074, AZD4547, BGJ398) revealed that the requirement for FGFR1 signaling in STS cells is dictated by FGFR1 expression levels, and identified the MAPK-ERK1/2 axis as critical FGFR1 effector pathway.\", \"disease\"=>{\"id\"=>105, \"name\"=>\"Sarcoma\", \"display_name\"=>\"Sarcoma\", \"doid\"=>\"1115\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1115\"}, \"drugs\"=>[{\"id\"=>151, \"name\"=>\"BGJ398\", \"pubchem_id\"=>nil}, {\"id\"=>452, \"name\"=>\"AZD4547\", \"pubchem_id\"=>nil}, {\"id\"=>480, \"name\"=>\"FGF/VEGF Receptor Tyrosine Kinase Inhibitor, PD173074\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1678, \"name\"=>\"Targeting Fibroblast Growth Factor Receptor 1 for Treatment of Soft-Tissue Sarcoma.\", \"citation\"=>\"Chudasama et al., 2017, Clin. Cancer Res.\", \"pubmed_id\"=>\"27535980\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27535980\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>2, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>268, \"phenotypes\"=>[]}, {\"id\"=>970, \"name\"=>\"EID970\", \"description\"=>\"Expression of bFGF and FGFR1 were evaluated in 61 biopsy specimens of primary OSCC (IHC). Expression was evaluated in both the cancer cells and fibroblasts at the invasive front of each tumor. bFGF and FGFR1 were expressed in all tumor specimens, whereas expression of these proteins in cancer-associated fibroblasts was associated with lymph node metastasis (P < 0.05) and worse patient survival.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>670, \"name\"=>\"Correlation of basic fibroblast growth factor expression with the invasion and the prognosis of oral squamous cell carcinoma.\", \"citation\"=>\"Hase et al., 2006, J. Oral Pathol. Med.\", \"pubmed_id\"=>\"16454808\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16454808\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2006, \"month\"=>3}, \"journal\"=>\"J. Oral Pathol. Med.\", \"full_journal_title\"=>\"Journal of oral pathology & medicine : official publication of the International Association of Oral Pathologists and the American Academy of Oral Pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>268, \"phenotypes\"=>[]}, {\"id\"=>799, \"name\"=>\"EID799\", \"description\"=>\"FGFR1 mRNA expression, rather than copy number alterations, were predictive of response to FGFR inhibitor BGJ398 in cell lines and patient derived xenografts.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>151, \"name\"=>\"BGJ398\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>533, \"name\"=>\"FGFR1 Expression Levels Predict BGJ398 Sensitivity of FGFR1-Dependent Head and Neck Squamous Cell Cancers.\", \"citation\"=>\"Göke et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"26015511\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26015511\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4592392\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>10, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>268, \"phenotypes\"=>[]}, {\"id\"=>802, \"name\"=>\"EID802\", \"description\"=>\"FGFR1 expression was increased in urothelial carcinoma cell lines and patient tumours, independent of stage and grade. FGFR1 knockdown resulted in decreases proliferation and survival in some of the cell lines.\", \"disease\"=>{\"id\"=>18, \"name\"=>\"Bladder Carcinoma\", \"display_name\"=>\"Bladder Carcinoma\", \"doid\"=>\"4007\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4007\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"rejected\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>536, \"name\"=>\"Fibroblast growth factor receptor 1 promotes proliferation and survival via activation of the mitogen-activated protein kinase pathway in bladder cancer.\", \"citation\"=>\"Tomlinson et al., 2009, Cancer Res.\", \"pubmed_id\"=>\"19458078\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19458078\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2737316\", \"publication_date\"=>{\"year\"=>2009, \"month\"=>6, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>268, \"phenotypes\"=>[]}, {\"id\"=>793, \"name\"=>\"EID793\", \"description\"=>\"FGFR1 expression, was assessed by IHC in 1,086 invasive breast cancers. FGFR1 expression was mainly associated with luminal cancer subtypes. Multivariate Cox regression analysis showed FGFR1 to be an independent poor prognostic marker in luminal A cancers for disease free survival (hazard ratio = 3.341, p = 0.008). In luminal A cancers, FGFR1 expression was also significantly associated with higher pN (lymph node metastases; p = 0.023) and pT (unfavorable tumor characteristics; p = 0.003) stages and lymphovascular invasion (p = 0.010).\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>527, \"name\"=>\"FGFR1 is an adverse outcome indicator for luminal A breast cancers.\", \"citation\"=>\"Shi et al., 2016, Oncotarget\", \"pubmed_id\"=>\"26673008\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26673008\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4826266\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>1, \"day\"=>26}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>268, \"phenotypes\"=>[]}, {\"id\"=>800, \"name\"=>\"EID800\", \"description\"=>\"FGFR inhibitor BGJ398 was most effective in a gastric cancer cell line model with high expression of FGFR1 and FGFR2IIIc.\", \"disease\"=>{\"id\"=>147, \"name\"=>\"Gastric Adenocarcinoma\", \"display_name\"=>\"Gastric Adenocarcinoma\", \"doid\"=>\"3717\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3717\"}, \"drugs\"=>[{\"id\"=>151, \"name\"=>\"BGJ398\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>534, \"name\"=>\"Targeting Fibroblast Growth Factor Receptor (FGFR) with BGJ398 in a Gastric Cancer Model.\", \"citation\"=>\"Schmidt et al., 2015, Anticancer Res.\", \"pubmed_id\"=>\"26637881\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26637881\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>12}, \"journal\"=>\"Anticancer Res.\", \"full_journal_title\"=>\"Anticancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>268, \"phenotypes\"=>[]}, {\"id\"=>798, \"name\"=>\"EID798\", \"description\"=>\"This review identified 12 studies that addressed the prognostic role of FGFRs in HNSSC. Evidence was found for the expression of FGFR1 in cancer-associated fibroblasts, other results were conflicting. Significant risk of bias was reported for the included studies.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>532, \"name\"=>\"Fibroblast Growth Factor Receptor Family Members as Prognostic Biomarkers in Head and Neck Squamous Cell Carcinoma: A Systematic Review.\", \"citation\"=>\"Ipenburg et al., 2016, Target Oncol\", \"pubmed_id\"=>\"26115874\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26115874\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4759209\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>2}, \"journal\"=>\"Target Oncol\", \"full_journal_title\"=>\"Targeted oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>268, \"phenotypes\"=>[]}, {\"id\"=>681, \"name\"=>\"EID681\", \"description\"=>\"Preclinical evidence (in-vitro) for predictive value of FGFR1 mRNA and protein expression as well as mRNA expression of FGF2 and FGF9 for sensitivity to FGFR1 inhibition. The authors suggested that FGFR1 expression is a better predictive marker than FGFR1 amplification and proposes a connection between KRAS and PIK3CA mutations. This study used 58 cell lines, of which 14 exhibited ponatinib sensitivity. This sensitivity was correlated with FGFR1 mRNA and protein expression.\", \"disease\"=>{\"id\"=>23, \"name\"=>\"Lung Cancer\", \"display_name\"=>\"Lung Cancer\", \"doid\"=>\"1324\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1324\"}, \"drugs\"=>[{\"id\"=>55, \"name\"=>\"Ponatinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>435, \"name\"=>\"FGFR1 mRNA and protein expression, not gene copy number, predict FGFR TKI sensitivity across all lung cancer histologies.\", \"citation\"=>\"Wynes et al., 2014, Clin. Cancer Res.\", \"pubmed_id\"=>\"24771645\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24771645\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4062100\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>6, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>268, \"phenotypes\"=>[]}, {\"id\"=>896, \"name\"=>\"EID896\", \"description\"=>\"In vitro studies showed efficacy of FGFR inhibitor NVP-BGJ398 in cell lines exhibiting high FGFR1 protein. Three cell lines (SNU-C1, HDC9, HCT116) were analyzed. FGFR1 amplification (24/454, assessed by FISH) and/or activated pFGFR1 protein (18/99, assessed by mRNA in situ hybridization) was identified in primary CRC tissue samples.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>151, \"name\"=>\"BGJ398\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>604, \"name\"=>\"Fibroblast growth factor receptor 1 as a putative therapy target in colorectal cancer.\", \"citation\"=>\"Göke et al., 2013, Digestion\", \"pubmed_id\"=>\"24135816\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24135816\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4186657\", \"publication_date\"=>{\"year\"=>2013}, \"journal\"=>\"Digestion\", \"full_journal_title\"=>\"Digestion\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>268, \"phenotypes\"=>[]}, {\"id\"=>801, \"name\"=>\"EID801\", \"description\"=>\"One urothelial cell line (JMSU1) with high expression of FGFR1 was sensitive to treatment with FGFR inhibitors  (SU5402, TKI258 or PD173074).\", \"disease\"=>{\"id\"=>18, \"name\"=>\"Bladder Carcinoma\", \"display_name\"=>\"Bladder Carcinoma\", \"doid\"=>\"4007\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4007\"}, \"drugs\"=>[{\"id\"=>36, \"name\"=>\"PD173074\", \"pubchem_id\"=>nil}, {\"id\"=>152, \"name\"=>\"Dovitinib Dilactic Acid (TKI258 Dilactic Acid)\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>535, \"name\"=>\"Small molecule FGF receptor inhibitors block FGFR-dependent urothelial carcinoma growth in vitro and in vivo.\", \"citation\"=>\"Lamont et al., 2011, Br. J. Cancer\", \"pubmed_id\"=>\"21119661\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21119661\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3039817\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>1, \"day\"=>4}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>268, \"phenotypes\"=>[]}]}, {\"id\"=>272, \"entrez_name\"=>\"CDKN2A\", \"entrez_id\"=>1029, \"name\"=>\"p16 EXPRESSION\", \"description\"=>\"CDKN2A (p16) expression is a surrogate marker of HPV infection in oropharyngeal squamous cell carcinoma. CDKN2A and p53 are inactivated through HPV proteins E6 and E7 and therefore not altered in expression level in HPV-associated carcinomas. However, there is no perfect overlap between p16 expression and HPV status with some 10-12.5% of HPV-negative tumors expressing p16. (Seiwert, T JCO, 2014). p16 positivity is a well-established positive prognostic factor in squamous cell cancer of the head and neck in the primary and, as increasing evidence suggests, recurrent/metastatic disease settings. Predictive information from HPV status have not been established to date. A lack of benefit from EGFR-inhibitors has been proposed by some studies. Other studies were not able to reproduce this finding.\", \"gene_id\"=>14, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>180.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>21968055, \"stop\"=>21974865, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000498124.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/14/summary/variants/272/summary#variant\", \"evidence_items\"=>[{\"id\"=>4873, \"name\"=>\"EID4873\", \"description\"=>\"A preclinical study to explore the efficacy of CDK4/6 inhibition with palbociclib, in a panel of sarcoma cell lines and sarcoma tumor xenografts (PDXs). The responsiveness of these cell lines/PDX correlated with CDK4 mRNA levels but not in those showing high p16ink4a. High levels of CDKN2A predicted poorer efficacy.\", \"disease\"=>{\"id\"=>105, \"name\"=>\"Sarcoma\", \"display_name\"=>\"Sarcoma\", \"doid\"=>\"1115\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1115\"}, \"drugs\"=>[{\"id\"=>429, \"name\"=>\"Palbociclib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>nil, \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2168, \"name\"=>\"Efficacy of CDK4 inhibition against sarcomas depends on their levels of CDK4 and p16ink4 mRNA.\", \"citation\"=>\"Perez et al., 2015, Oncotarget\", \"pubmed_id\"=>\"26528855\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26528855\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4747352\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>12, \"day\"=>1}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>272, \"phenotypes\"=>[]}, {\"id\"=>1155, \"name\"=>\"EID1155\", \"description\"=>\"Phase III trial to compare afatinib or metothrexate in recurrent or metastatic head and neck squamous cell carcinoma. 483 patients were randomized to either afatinib (n=322) or methotrexate (n=161). 285 patients were assessed for p16 Status (IHC). In the post-hoc analysis, benefit of afatinib compared with methotrexate was more pronounced in p16-negative disease (HR 0·69 [95% CI 0·50–0·96], p=0.022) than in p16-positive disease (HR 0·95 [CI 0·51–1·75], p=0.67, interaction test p=0·32).\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>146, \"name\"=>\"Afatinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>795, \"name\"=>\"Afatinib versus methotrexate as second-line treatment in patients with recurrent or metastatic squamous-cell carcinoma of the head and neck progressing on or after platinum-based therapy (LUX-Head & Neck 1): an open-label, randomised phase 3 trial.\", \"citation\"=>\"Machiels et al., 2015, Lancet Oncol.\", \"pubmed_id\"=>\"25892145\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25892145\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>5}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>272, \"phenotypes\"=>[]}, {\"id\"=>1314, \"name\"=>\"EID1314\", \"description\"=>\"In a cohort of 73 patients with stage I or stage II NSCLC it was discovered that the CDKN2A protein (p16-INK4A) retrospectively evaluated via IHC was associated with adverse outcomes (p=0.004).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>904, \"name\"=>\"Altered p16(INK4) and RB1 Expressions Are Associated with Poor Prognosis in Patients with Nonsmall Cell Lung Cancer.\", \"citation\"=>\"Zhao et al., 2012, J Oncol\", \"pubmed_id\"=>\"22619677\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22619677\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3350860\", \"publication_date\"=>{\"year\"=>2012}, \"journal\"=>\"J Oncol\", \"full_journal_title\"=>\"Journal of oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>272, \"phenotypes\"=>[]}, {\"id\"=>758, \"name\"=>\"EID758\", \"description\"=>\"720 patients with stage III/IV oropharyngeal cancer were retrospectively analyzed for p16 and HPV status. p16 and HPV-positivity correlated well and both showed improved overall-survival compared to patients with HPV-negative/p16 negative tumors. HPV/p16 status was shown to be a strong and independent risk factor in oropharyngeal carcinoma.\", \"disease\"=>{\"id\"=>204, \"name\"=>\"Oropharynx Cancer\", \"display_name\"=>\"Oropharynx Cancer\", \"doid\"=>\"8557\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8557\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>501, \"name\"=>\"Human papillomavirus and survival of patients with oropharyngeal cancer.\", \"citation\"=>\"Ang et al., 2010, N. Engl. J. Med.\", \"pubmed_id\"=>\"20530316\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20530316\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2943767\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>272, \"phenotypes\"=>[]}, {\"id\"=>804, \"name\"=>\"EID804\", \"description\"=>\"Patients from three studies (RTOG 0129, 0234, and 0522; 85, 95 and 142 patients, respectively) were retrospectively analyzed for p16 (CDKN2A) expression (IHC) and HPV-high risk status (ISH). p16 expression in non-oropharyngeal HNSSC was also prognostic and associated with a better outcome. However, p16-positive oropharyngeal HNSSC still had better PFS and OS than p16-positive non-oropharyngeal HNSSC.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>538, \"name\"=>\"p16 protein expression and human papillomavirus status as prognostic biomarkers of nonoropharyngeal head and neck squamous cell carcinoma.\", \"citation\"=>\"Chung et al., 2014, J. Clin. Oncol.\", \"pubmed_id\"=>\"25267748\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25267748\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4251957\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>12, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>272, \"phenotypes\"=>[]}, {\"id\"=>697, \"name\"=>\"EID697\", \"description\"=>\"In this study response to EGFR-inhibition by cetuximab was independent of HPV-status, represented by expression of p16 (CDKN2A, an established biomarker of HPV infection). Expression of p16 was assayed in 69 HNSCC patients by IHC.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>456, \"name\"=>\"Impact of human papilloma virus infection on the response of head and neck cancers to anti-epidermal growth factor receptor antibody therapy.\", \"citation\"=>\"Pogorzelski et al., 2014, Cell Death Dis\", \"pubmed_id\"=>\"24577089\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24577089\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3944273\", \"publication_date\"=>{\"year\"=>2014}, \"journal\"=>\"Cell Death Dis\", \"full_journal_title\"=>\"Cell death & disease\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>272, \"phenotypes\"=>[]}, {\"id\"=>695, \"name\"=>\"EID695\", \"description\"=>\"p16 positivity, as a surrogate marker for HPV-associated HNSSC was a favorable prognostic marker in recurrent or metastatic carcinoma.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>454, \"name\"=>\"Prognostic significance of human papillomavirus in recurrent or metastatic head and neck cancer: an analysis of Eastern Cooperative Oncology Group trials.\", \"citation\"=>\"Argiris et al., 2014, Ann. Oncol.\", \"pubmed_id\"=>\"24799460\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24799460\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4071756\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>7}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology / ESMO\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>272, \"phenotypes\"=>[]}, {\"id\"=>696, \"name\"=>\"EID696\", \"description\"=>\"In this retrospective analysis of p16 status as surrogate marker for HPV in the SPECTRUM trial, p16 (CDKN2A) expression was not associated with improved response to EGFR-directed therapy (Panitumumab) in combination with chemotherapy compared to chemotherapy alone.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>28, \"name\"=>\"Panitumumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>455, \"name\"=>\"Cisplatin and fluorouracil with or without panitumumab in patients with recurrent or metastatic squamous-cell carcinoma of the head and neck (SPECTRUM): an open-label phase 3 randomised trial.\", \"citation\"=>\"Vermorken et al., 2013, Lancet Oncol.\", \"pubmed_id\"=>\"23746666\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23746666\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>7}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>272, \"phenotypes\"=>[]}, {\"id\"=>759, \"name\"=>\"EID759\", \"description\"=>\"This study shows that benefit of EGFR inhibition in addition to radiotherapy was independent of p16/HPV status. 182 patients were evaluated.\", \"disease\"=>{\"id\"=>204, \"name\"=>\"Oropharynx Cancer\", \"display_name\"=>\"Oropharynx Cancer\", \"doid\"=>\"8557\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8557\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>502, \"name\"=>\"Association of Human Papillomavirus and p16 Status With Outcomes in the IMCL-9815 Phase III Registration Trial for Patients With Locoregionally Advanced Oropharyngeal Squamous Cell Carcinoma of the Head and Neck Treated With Radiotherapy With or Without Cetuximab.\", \"citation\"=>\"Rosenthal et al., 2016, J. Clin. Oncol.\", \"pubmed_id\"=>\"26712222\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26712222\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>272, \"phenotypes\"=>[]}, {\"id\"=>694, \"name\"=>\"EID694\", \"description\"=>\"In this retrospective analysis of the SPECTRUM trial, p16 (CDKN2A) expression (and HPV-infection) was not predictive of response to EGFR-targeted therapy in combination with chemotherapy (n = 416 patient tumors).\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>145, \"name\"=>\"EGFR Inhibitor\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>453, \"name\"=>\"Impact of tumor HPV status on outcome in patients with recurrent and/or metastatic squamous cell carcinoma of the head and neck receiving chemotherapy with or without cetuximab: retrospective analysis of the phase III EXTREME trial.\", \"citation\"=>\"Vermorken et al., 2014, Ann. Oncol.\", \"pubmed_id\"=>\"24577117\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24577117\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3969553\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>4}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology / ESMO\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>272, \"phenotypes\"=>[]}, {\"id\"=>753, \"name\"=>\"EID753\", \"description\"=>\"p16-negative patients (HPV negative) have a longer overall survival under panitumumab/chemotherapy in comparison to chemotherapy alone.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>28, \"name\"=>\"Panitumumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>455, \"name\"=>\"Cisplatin and fluorouracil with or without panitumumab in patients with recurrent or metastatic squamous-cell carcinoma of the head and neck (SPECTRUM): an open-label phase 3 randomised trial.\", \"citation\"=>\"Vermorken et al., 2013, Lancet Oncol.\", \"pubmed_id\"=>\"23746666\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23746666\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>7}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>272, \"phenotypes\"=>[]}]}, {\"id\"=>276, \"entrez_name\"=>\"CD274\", \"entrez_id\"=>29126, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>11335, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>315.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>5450525, \"stop\"=>5470547, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000381577.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/11335/summary/variants/276/summary#variant\", \"evidence_items\"=>[{\"id\"=>5507, \"name\"=>\"EID5507\", \"description\"=>\"A meta analysis of 60 research studies involving 10,310 solid tumors indicated that CD274 over expression results in poor overall survival (HR = 1.58, P <.001) and poor disease free/progression free survival (DFS/PFS) (HR = 1.72, P =0 .001). When split into sub groups by cancer type overall survival  was poor for breast (HR = 1.98, P =0 .014), urothelial (HR = 2.24, P <0.001), renal (HR = 3.30,  P <0.001), and gastric cancers (HR = 1.56, P =0 .040). DFS/PFS was poor in patients with hepatocellular carcinoma (HR = 1.72,  P = 0.003), melanoma (HR = 3.39,  P <0.001), and renal carcinoma, (HR = 5.04,  P <0.001).\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2256, \"name\"=>\"Prognostic significance of PD-L1 in solid tumor: An updated meta-analysis.\", \"citation\"=>\"Wang et al., 2017, Medicine (Baltimore)\", \"pubmed_id\"=>\"28471952\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28471952\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5419898\", \"publication_date\"=>{\"year\"=>2017, \"month\"=>5}, \"journal\"=>\"Medicine (Baltimore)\", \"full_journal_title\"=>\"Medicine\", \"status\"=>\"fully curated\", \"is_review\"=>true, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>4856, \"name\"=>\"EID4856\", \"description\"=>\"PD-L1 is not only expressed in tumor cells (TCs) but also upregulated in tumor-infiltrating immune cells (TIICs). This meta-analysis evaluated the prognostic value of PD-L1 expression in TIICs in different types of cancers. 18 studies involving 3674 patients were included. PD-L1 positivity in TIICs related to a lower risk of death (HR = 0.784, p=0.047), and TIICs indicated a better prognosis especially in breast cancer patients (HR = 0.359, p=0.002).\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2155, \"name\"=>\"Prognostic value of PD-L1 expression in tumor infiltrating immune cells in cancers: A meta-analysis.\", \"citation\"=>\"Zhao et al., 2017, PLoS ONE\", \"pubmed_id\"=>\"28453554\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28453554\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5409185\", \"publication_date\"=>{\"year\"=>2017}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>4844, \"name\"=>\"EID4844\", \"description\"=>\"A meta-analysis of the role of PD-L1 expression in breast cancer found nine relevant studies with a total 8583 patients. PD-L1 expression was found to be associated with several poor prognostic indicators, such as ductal cancer, high tumor grade, ER negativity, PR negativity, HER2 positivity and aggressive molecular subtypes. PD-L1 overexpression had no significant impact on metastasis-free survival (HR 0.924, 95% CI = 0.747-1.141, p = 0.462), disease-free survival (HR 1.122, 95% CI = 0.878-1.434, p = 0.357) and overall specific survival (HR 0.837, 95% CI = 0.640-1.093, p = 0.191), but significantly correlated with shortened overall survival (HR 1.573, 95% CI = 1.010-2.451, p = 0.045).\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2148, \"name\"=>\"Prognostic Value of PD-L1 in Breast Cancer: A Meta-Analysis.\", \"citation\"=>\"Wang et al., 2017, Breast J\", \"pubmed_id\"=>\"28079291\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28079291\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>1, \"day\"=>12}, \"journal\"=>\"Breast J\", \"full_journal_title\"=>\"The breast journal\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>4855, \"name\"=>\"EID4855\", \"description\"=>\"In this meta-analysis, 5 studies totalling 2,546 cases were combined to assess the prognostic association of PD-L1 expression in breast cancer. The combined hazard ratio (HR) for PD-L1 overexpression was 1.76 (OS). The pooled odds ratios (ORs) indicated that PD-L1 expression was associated with positive lymph node metastasis, higher histological grades, estrogen receptor (ER)-negativity, and triple-negative breast cancer (TNBC).\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2154, \"name\"=>\"Expression of PD-L1 and prognosis in breast cancer: a meta-analysis.\", \"citation\"=>\"Zhang et al., 2017, Oncotarget\", \"pubmed_id\"=>\"28430626\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28430626\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>5, \"day\"=>9}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>4857, \"name\"=>\"EID4857\", \"description\"=>\"In this meta-analysis of 14 studies with 2857 patients, the immunotherapy response rate was higher among PD-L1-positive patients (RR: 2.19, 95% CI: 1.63-2.94). PD-L1 expression was also related to better progression-free survival (HR: 0.69, 95% CI: 0.57-0.85) and better overall survival (HR: 0.77, 95% CI: 0.67-0.89).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>137, \"name\"=>\"Nivolumab\", \"pubchem_id\"=>nil}, {\"id\"=>138, \"name\"=>\"Pembrolizumab\", \"pubchem_id\"=>nil}, {\"id\"=>261, \"name\"=>\"Atezolizumab\", \"pubchem_id\"=>nil}, {\"id\"=>296, \"name\"=>\"Avelumab\", \"pubchem_id\"=>nil}, {\"id\"=>530, \"name\"=>\"Durvalumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2156, \"name\"=>\"PD-L1 expression as a predictive biomarker in advanced non-small-cell lung cancer: updated survival data.\", \"citation\"=>\"Aguiar et al., 2017, Immunotherapy\", \"pubmed_id\"=>\"28472902\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28472902\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>5}, \"journal\"=>\"Immunotherapy\", \"full_journal_title\"=>\"Immunotherapy\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>3000, \"name\"=>\"EID3000\", \"description\"=>\"Radiation-resistant cell lines exhibited upregulation and increased activation of Axl, PI3 kinase signalling as well as increased expression of PD-L1. In 3 cohorts of HPV-negative head and neck squamous cell carcinoma patient samples evaluated with different methods [reverse phase protein array (N=68), mRNA expression array (N=97) or immunohistochemistry (N=114), respectively], increased expression of PD-L1 was also correlated with local failure following radiotherapy as compared to the PD-L1–low expression group (P = 0.01, 1.9 × 10−3, and 9 × 10−4, respectively).\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>360, \"name\"=>\"Radiation\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1728, \"name\"=>\"Integrative Analysis Identifies a Novel AXL-PI3 Kinase-PD-L1 Signaling Axis Associated with Radiation Resistance in Head and Neck Cancer.\", \"citation\"=>\"Skinner et al., 2017, Clin. Cancer Res.\", \"pubmed_id\"=>\"28476872\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28476872\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>5, \"day\"=>5}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>1733, \"name\"=>\"EID1733\", \"description\"=>\"This open-label, randomized, phase-3 clinical trial compared pembrolizumab to chemotherapy in treating NSCLC patients with PD-L1 expression on at least 50% of tumor cells and no sensitizing mutation of EGFR or translocation of ALK. Patients in the pembrolizumab group exhibited significantly better progression-free survival and overall survival compared to patients in the chemotherapy group.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>138, \"name\"=>\"Pembrolizumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1176, \"name\"=>\"Pembrolizumab versus Chemotherapy for PD-L1-Positive Non-Small-Cell Lung Cancer.\", \"citation\"=>\"Reck et al., 2016, N. Engl. J. Med.\", \"pubmed_id\"=>\"27718847\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27718847\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>11, \"day\"=>10}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>1615, \"name\"=>\"EID1615\", \"description\"=>\"A clinical trial of 86 patients evaluated concurrent (n=53) or sequential (n=33) treatment of ipilimumab (anti-CTLA4) and nivolumab (anti-PD1). In the concurrent regimen cohort, 65% of patients showed clinical activity with no correlation between PD-L1 expression and response (6 of 13 patients were PD-L1 positive versus 9 of 22 patients were PD-L1 negative). In the sequential regimen cohort, 20% of patients had an objective response with a potential correlation between PD-L1 expression and response (4 of 8 patients were PD-L1 positive versus 1 of 13 patients were PD-L1 negative).\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>137, \"name\"=>\"Nivolumab\", \"pubchem_id\"=>nil}, {\"id\"=>365, \"name\"=>\"Ipilimumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1068, \"name\"=>\"Nivolumab plus ipilimumab in advanced melanoma.\", \"citation\"=>\"Wolchok et al., 2013, N. Engl. J. Med.\", \"pubmed_id\"=>\"23724867\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23724867\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>7, \"day\"=>11}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>1578, \"name\"=>\"EID1578\", \"description\"=>\"Patients with Merkel Cell Carcinoma (n=26) were enrolled to receive pembrolizumab. Of these patients, 25 were evaluated for response rate to pembrolizumab and PD-L1 (CD274) expression.  The response rate to pembrolizumab was 56% (95%CI: 35-76%), however, there was no correlation between PD-L1 expression (cutoff >1%) on tumor cells or on infiltrating immune cells and clinical response to pembrolizumab (P=0.61).\", \"disease\"=>{\"id\"=>311, \"name\"=>\"Merkel Cell Carcinoma\", \"display_name\"=>\"Merkel Cell Carcinoma\", \"doid\"=>\"3965\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3965\"}, \"drugs\"=>[{\"id\"=>138, \"name\"=>\"Pembrolizumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1042, \"name\"=>\"PD-1 Blockade with Pembrolizumab in Advanced Merkel-Cell Carcinoma.\", \"citation\"=>\"Nghiem et al., 2016, N. Engl. J. Med.\", \"pubmed_id\"=>\"27093365\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27093365\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4927341\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>6, \"day\"=>30}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>1517, \"name\"=>\"EID1517\", \"description\"=>\"Trial NCT00730639 tested monoclonal antibody nivolumab against the PD-1 immune checkpoint receptor in pretreated patients with various cancer types. PD-L1 expression was tested in 42 patients by ICH from tumor FFPE biopsies. (18 melanoma, 10 NSCLC, 7 colorectal cancer, 5 renal cell cancer, 2 prostate cancer). 9 of 25 PD-L1 expressing patients showed objective response, and zero of 17 PD-L1 negative patients showed objective response (p=0.006). Biopsies were optional, and therefore not fully randomized.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>137, \"name\"=>\"Nivolumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1010, \"name\"=>\"Safety, activity, and immune correlates of anti-PD-1 antibody in cancer.\", \"citation\"=>\"Topalian et al., 2012, N. Engl. J. Med.\", \"pubmed_id\"=>\"22658127\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22658127\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3544539\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>6, \"day\"=>28}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>1274, \"name\"=>\"EID1274\", \"description\"=>\"Immunohistochemical analyses of PD-L1 protein was retrospectively performed on 251 archived FFPE surgical tissues (66 benign thyroid nodules and 185 papillary thyroid carcinomas PTC, E1L3N antibody). Patients with increased PD-L1 protein had significantly shorter median DFS (36 months [membranous staining] and 49 months [cytoplasmic staining] vs. 186 months [negative], p < 0.001 and p < 0.01, respectively).\", \"disease\"=>{\"id\"=>156, \"name\"=>\"Papillary Thyroid Carcinoma\", \"display_name\"=>\"Papillary Thyroid Carcinoma\", \"doid\"=>\"3969\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3969\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>880, \"name\"=>\"Programmed death-ligand 1 overexpression is a prognostic marker for aggressive papillary thyroid cancer and its variants.\", \"citation\"=>\"Chowdhury et al., 2016, Oncotarget\", \"pubmed_id\"=>\"27086918\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27086918\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>12}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>1275, \"name\"=>\"EID1275\", \"description\"=>\"465 gastric cancer samples and 15 corresponding liver metastases underwent IHC for PD1/PD-L1 protein expression. PD-L1 positivity in tumor cells was defined by at least 2% tumor cells with membranous positivity, PD-L1 positivity in immune cells by at least 10% PD-L1 positive immune cells. Patients with PD-L1 positive tumor cells had a significant better overall (p=0.028) and tumor specific survival (p=0.018). The same applied for patients with PD-L1 positive immune cells (p<0.001 each).\", \"disease\"=>{\"id\"=>1322, \"name\"=>\"Stomach Carcinoma\", \"display_name\"=>\"Stomach Carcinoma\", \"doid\"=>\"5517\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5517\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>881, \"name\"=>\"PD-L1 is an independent prognostic predictor in gastric cancer of Western patients.\", \"citation\"=>\"Böger et al., 2016, Oncotarget\", \"pubmed_id\"=>\"27009855\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27009855\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>26}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>1172, \"name\"=>\"EID1172\", \"description\"=>\"Exploratory biomarker analysis from a phase-II randomized trial comparing atezolizumab (PD-L1) and docetaxel in previously treated, advanced or metastatic non-small cell lung cancer. Baseline PD-L1 expression (IHC) was assessed in tumour cells (as percentage of PD-L1-expressing tumour cells TC3≥50%, TC2≥5% and <50%, TC1≥1% and <5%, and TC0<1%) and tumour-infiltrating immune cells (as percentage of tumour area: IC3≥10%, IC2≥5% and <10%, IC1≥1% and <5%, and IC0<1%). Increasing PD-L1 Expression was associated with improved overall survival (TC3 or IC3 HR 0·49 [0·22-1·07; p=0·068], TC2/3 or IC2/3 HR 0·54 [0·33-0·89; p=0·014], TC1/2/3 or IC1/2/3 HR 0·59 [0·40-0·85; p=0·005], TC0 and IC0 HR 1·04 [0·62-1·75; p=0·871]) but not progression free survival or overall response rate.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>261, \"name\"=>\"Atezolizumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>809, \"name\"=>\"Atezolizumab versus docetaxel for patients with previously treated non-small-cell lung cancer (POPLAR): a multicentre, open-label, phase 2 randomised controlled trial.\", \"citation\"=>\"Fehrenbacher et al., 2016, Lancet\", \"pubmed_id\"=>\"26970723\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26970723\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>30}, \"journal\"=>\"Lancet\", \"full_journal_title\"=>\"Lancet (London, England)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>1167, \"name\"=>\"EID1167\", \"description\"=>\"Meta-Analysis of PD-L1 (CD274) Expression Status in 914 patients (7 studies) and response to PD-1/PD-L1 monoclonal antibodies. Patients with PD-L1 positive tumors (PD-L1 tumor cell staining ≥1%), had a significantly higher ORR, compared to patients with PD-L1 negative tumors (OR: 2.44; 95% CIs: 1.61-3.68).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>137, \"name\"=>\"Nivolumab\", \"pubchem_id\"=>nil}, {\"id\"=>138, \"name\"=>\"Pembrolizumab\", \"pubchem_id\"=>nil}, {\"id\"=>261, \"name\"=>\"Atezolizumab\", \"pubchem_id\"=>nil}, {\"id\"=>296, \"name\"=>\"Avelumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>804, \"name\"=>\"PD-L1 expression as predictive biomarker in patients with NSCLC: a pooled analysis.\", \"citation\"=>\"Passiglia et al., 2016, Oncotarget\", \"pubmed_id\"=>\"26918451\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26918451\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>12}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>975, \"name\"=>\"EID975\", \"description\"=>\"74 patients with stage III NSCLC treated with concurrent chemoradiotherapy were retrospectively analyzed for PD-L1 expression and CD8+ tumour-infiltrating lymphocytes (TILs) density (IHC). CD8+ TIL density was an independent predictor of progression free survival (PFS) and overall survival (OS). Only when both TIL density and PD-L1 expression were both considered, was PD-L1 expression associated with survival. The subgroup, PD-L1+/CD8 low tumors had the shortest PFS (8.6 months, p = 0.02) and shortest, but not significantly different, OS (13.9 months, p = 0.11).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>674, \"name\"=>\"Predictive relevance of PD-L1 expression combined with CD8+ TIL density in stage III non-small cell lung cancer patients receiving concurrent chemoradiotherapy.\", \"citation\"=>\"Tokito et al., 2016, Eur. J. Cancer\", \"pubmed_id\"=>\"26771872\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26771872\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>3}, \"journal\"=>\"Eur. J. Cancer\", \"full_journal_title\"=>\"European journal of cancer (Oxford, England : 1990)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>743, \"name\"=>\"EID743\", \"description\"=>\"Based on a meta-analysis of 20 trials (1475 patients), PD-L1 expression was associated with a higher ORR rate in patients with NSCLC. Active drugs were Nivolumab and Pembrolizumab, PD-1 receptor targeting antibodies.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>137, \"name\"=>\"Nivolumab\", \"pubchem_id\"=>nil}, {\"id\"=>138, \"name\"=>\"Pembrolizumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>462, \"name\"=>\"Differential Activity of Nivolumab, Pembrolizumab and MPDL3280A according to the Tumor Expression of Programmed Death-Ligand-1 (PD-L1): Sensitivity Analysis of Trials in Melanoma, Lung and Genitourinary Cancers.\", \"citation\"=>\"Carbognin et al., 2015, PLoS ONE\", \"pubmed_id\"=>\"26086854\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26086854\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4472786\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>702, \"name\"=>\"EID702\", \"description\"=>\"In this study, PD-L1 expression (measured by IHC) was associated with better overall survival in the subgroup of Lung Squamous Cell Carcinomas (149 of 321 patients in the study).\", \"disease\"=>{\"id\"=>153, \"name\"=>\"Lung Squamous Cell Carcinoma\", \"display_name\"=>\"Lung Squamous Cell Carcinoma\", \"doid\"=>\"3907\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3907\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>460, \"name\"=>\"PD-1 and PD-L1 Expression in NSCLC Indicate a Favorable Prognosis in Defined Subgroups.\", \"citation\"=>\"Schmidt et al., 2015, PLoS ONE\", \"pubmed_id\"=>\"26313362\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26313362\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4552388\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>701, \"name\"=>\"EID701\", \"description\"=>\"This meta-analysis did not find a significant connection between expression of PD-L1 and overall survival in patients with NSCLC.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>459, \"name\"=>\"Prognostic value of programmed cell death-ligand 1 expression in patients with non-small-cell lung cancer: evidence from an updated meta-analysis.\", \"citation\"=>\"Zhong et al., 2015, Onco Targets Ther\", \"pubmed_id\"=>\"26664143\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26664143\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4671817\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"Onco Targets Ther\", \"full_journal_title\"=>\"OncoTargets and therapy\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>704, \"name\"=>\"EID704\", \"description\"=>\"Based on a meta-analysis of 20 trials (1475 patients), of which 7 studies (636 patients) included melanoma, PD-L1 expression was associated with a higher ORR rate in patients with melanoma. Active drugs were Nivolumab and Pembrolizumab, PD-1 receptor targeting antibodies.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>137, \"name\"=>\"Nivolumab\", \"pubchem_id\"=>nil}, {\"id\"=>138, \"name\"=>\"Pembrolizumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>462, \"name\"=>\"Differential Activity of Nivolumab, Pembrolizumab and MPDL3280A according to the Tumor Expression of Programmed Death-Ligand-1 (PD-L1): Sensitivity Analysis of Trials in Melanoma, Lung and Genitourinary Cancers.\", \"citation\"=>\"Carbognin et al., 2015, PLoS ONE\", \"pubmed_id\"=>\"26086854\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26086854\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4472786\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>703, \"name\"=>\"EID703\", \"description\"=>\"High expression of PD-L1 and PD-1 were associated with a favorable prognosis in patients with high grade serous ovarian carcinoma (n = 215 cases).\", \"disease\"=>{\"id\"=>87, \"name\"=>\"Ovarian Serous Carcinoma\", \"display_name\"=>\"Ovarian Serous Carcinoma\", \"doid\"=>\"0050933\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050933\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>461, \"name\"=>\"Prognostic impact of programmed cell death-1 (PD-1) and PD-ligand 1 (PD-L1) expression in cancer cells and tumor-infiltrating lymphocytes in ovarian high grade serous carcinoma.\", \"citation\"=>\"Darb-Esfahani et al., 2016, Oncotarget\", \"pubmed_id\"=>\"26625204\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26625204\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4811475\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>1, \"day\"=>12}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>276, \"phenotypes\"=>[]}, {\"id\"=>5811, \"name\"=>\"EID5811\", \"description\"=>\"PD-L1 expression in at least 50% of tumor cells correlated with improved efficacy of pembrolizumab.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>138, \"name\"=>\"Pembrolizumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2351, \"name\"=>\"Pembrolizumab for the treatment of non-small-cell lung cancer.\", \"citation\"=>\"Garon et al., 2015, N. Engl. J. Med.\", \"pubmed_id\"=>\"25891174\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25891174\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>5, \"day\"=>21}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[{\"nct_id\"=>\"NCT01295827\", \"name\"=>\"Study of Pembrolizumab (MK-3475) in Participants With Progressive Locally Advanced or Metastatic Carcinoma, Melanoma, or Non-small Cell Lung Carcinoma (P07990/MK-3475-001/KEYNOTE-001)\", \"description\"=>\"This study will be done in 6 parts. In Part A the dose of intravenous (IV) pembrolizumab (MK-3475) will be escalated to find the maximum tolerated dose (MTD) and recommended Phase 2 dose (RP2D) for participants with a histologically or cytologically confirmed diagnosis of any type of carcinoma or melanoma (MEL). Part B of the study will explore the safety, tolerability, and efficacy of the drug in participants with advanced or metastatic MEL and compare every 2 week to every 3 week dosing. Part C of the study will explore the safety, tolerability, and efficacy of the drug in participants with non-small cell lung carcinoma (NSCLC) that is locally advanced or metastatic. Part D of the study will explore the low and high doses of study drug identified in Parts A and B in participants with advanced or metastatic MEL. Part E (closed with Amendment 7) will explore low, medium, and high doses of study drug in combination with standard chemotherapy in participants with locally advanced or metastatic NSCLC. Part F will explore low and high doses of study drug in treatment-naive and previously-treated participants with NSCLC with programmed cell death 1 ligand (PD-L1) gene expression. In Parts D and F and some of Part B participants will be randomized to one dose level. The primary hypotheses are the following: that pembrolizumab has acceptable safety and tolerability; and that pembrolizumab shows a clinically meaningful response rate (RR) or disease-control rate (DCR) in participants with melanoma (ipilimumab-refractory or not), and a clinically meaningful RR in participants with NSCLC, especially a clinically meaningful RR in those participants with either cancer, whose tumors express PD-L1.\", \"clinical_trial_url\"=>\"https://clinicaltrials.gov/show/NCT01295827\"}]}, \"variant_id\"=>276, \"phenotypes\"=>[]}]}, {\"id\"=>278, \"entrez_name\"=>\"AKT2\", \"entrez_id\"=>208, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>254, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"19\", \"start\"=>40736224, \"stop\"=>40791302, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000392038.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/254/summary/variants/278/summary#variant\", \"evidence_items\"=>[{\"id\"=>706, \"name\"=>\"EID706\", \"description\"=>\"In HER2+ metastatic breast cancer treated with trastuzumab (n = 74 patients), the expression of AKT2 and pAkt-Thr308 and/or pAkt-Ser473 localized in nucleus+cytoplasm was associated with an improved time to progression (TTP) and overall survival (OS) compared to AKT2 negative tumors as measured by immunohistochemistry.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>464, \"name\"=>\"Akt expression and compartmentalization in prediction of clinical outcome in HER2-positive metastatic breast cancer patients treated with trastuzumab.\", \"citation\"=>\"Grell et al., 2012, Int. J. Oncol.\", \"pubmed_id\"=>\"22842582\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22842582\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3583615\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>10}, \"journal\"=>\"Int. J. Oncol.\", \"full_journal_title\"=>\"International journal of oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>278, \"phenotypes\"=>[]}]}, {\"id\"=>282, \"entrez_name\"=>\"SMAD4\", \"entrez_id\"=>4089, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>77, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"18\", \"start\"=>48556583, \"stop\"=>48611409, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000342988.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/77/summary/variants/282/summary#variant\", \"evidence_items\"=>[{\"id\"=>720, \"name\"=>\"EID720\", \"description\"=>\"In this study, SMAD4 underexpression was associated with an epithelial-to-mesenchymal transition and resistance to EGFR-directed treatment with Cetuximab.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>474, \"name\"=>\"Decreased SMAD4 expression is associated with induction of epithelial-to-mesenchymal transition and cetuximab resistance in head and neck squamous cell carcinoma.\", \"citation\"=>\"Cheng et al., 2015, Cancer Biol. Ther.\", \"pubmed_id\"=>\"26046389\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26046389\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4623002\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"Cancer Biol. Ther.\", \"full_journal_title\"=>\"Cancer biology & therapy\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>282, \"phenotypes\"=>[]}]}, {\"id\"=>289, \"entrez_name\"=>\"ERBB3\", \"entrez_id\"=>2065, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>1733, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>27.0, \"coordinates\"=>{\"chromosome\"=>\"12\", \"start\"=>56473645, \"stop\"=>56497289, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000267101.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1733/summary/variants/289/summary#variant\", \"evidence_items\"=>[{\"id\"=>3053, \"name\"=>\"EID3053\", \"description\"=>\"Women with primary breast cancer were randomized (3:1) to 10 to 14 days of preoperative lapatinib or placebo in a multicenter phase II trial. Biopsies pre-/posttreatment were analyzed for Ki67, apoptosis, HER2, EGFR, ER, PgR, pAKT, pERK, and stathmin by IHC. 121 patients (lapatinib, 94; placebo, 27) were randomized; of these, 21% were HER2(+), 78% were HER2(-) nonamplified, 26% were EGFR(+). Paired samples containing tumor were obtained for 98% (118 of 121). Ki67 fell significantly with lapatinib,, but not with placebo (-3%). Whereas Ki67 reduction with lapatinib was greatest in HER2(+) breast cancer, there was a significant Ki67 decrease in HER2(-) breast cancer (-27%) with 14% of HER2(-), demonstrating ≥50% Ki67 reduction with lapatinib. Among HER2(-) tumors, only HER3 mRNA levels were significantly associated with Ki67 response on multivariate analysis. In HER2(-) breast cancer, HER2 and HER3 mRNA levels were highly correlated, with all Ki67 responders having elevated HER3 and HER2 expression.\\nLapatinib has antiproliferative effects in a subgroup of HER2(-) nonamplified tumors characterized by high HER3 expression. The possible role of high HER2:HER3 heterodimers in predicting response to lapatinib merits investigation in HER2(-) tumors.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>45, \"name\"=>\"Lapatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1755, \"name\"=>\"Antiproliferative Effect of Lapatinib in HER2-Positive and HER2-Negative/HER3-High Breast Cancer: Results of the Presurgical Randomized MAPLE Trial (CRUK E/06/039).\", \"citation\"=>\"Leary et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"25398453\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25398453\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>289, \"phenotypes\"=>[]}, {\"id\"=>3050, \"name\"=>\"EID3050\", \"description\"=>\"MEK inhibitors improve survival in cutaneous melanoma patients but show only modest efficacy in metastatic uveal melanoma patients. In this study, we screened for growth factors that elicited resistance in newly characterized metastatic uveal melanoma cell lines to clinical-grade MEK inhibitors, trametinib and selumetinib. We show that neuregulin 1 (NRG1) and hepatocyte growth factor (HGF) provide resistance to MEK inhibition. Mechanistically, trametinib enhances the responsiveness to NRG1 and sustained HGF-mediated activation of AKT. Individually targeting ERBB3 and cMET, the receptors for NRG1 and HGF, respectively, overcome resistance to trametinib\", \"disease\"=>{\"id\"=>188, \"name\"=>\"Uveal Melanoma\", \"display_name\"=>\"Uveal Melanoma\", \"doid\"=>\"6039\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:6039\"}, \"drugs\"=>[{\"id\"=>513, \"name\"=>\"Selumetinib, Trametinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>nil, \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1742, \"name\"=>\"Paracrine Effect of NRG1 and HGF Drives Resistance to MEK Inhibitors in Metastatic Uveal Melanoma.\", \"citation\"=>\"Cheng et al., 2015, Cancer Res.\", \"pubmed_id\"=>\"25952648\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25952648\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4490069\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>289, \"phenotypes\"=>[]}, {\"id\"=>1153, \"name\"=>\"EID1153\", \"description\"=>\"Preclinical study in KRAS mutant lung and colon cancer models. A synthetic lethal screen identified ERBB3 blockade to confer sensitivity to MEK inhibitors. The dual EGFR-ERBB2 (both forming heterodimers with ERBB3) inhibitors afatinib and dacomitinib each showed strong synergy with MEK Inhibition (trametinib or selumetinib) in several KRAS mutant cell lines. MEK Inhibitors alone or the combination with gefitinib (EGFRi) or CP724714 (ERBB2i) showed strong synergy. Results were validated in-vivo. Basal ERBB3 expression was associated with higher synergy scores.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>14, \"name\"=>\"Gefitinib\", \"pubchem_id\"=>nil}, {\"id\"=>122, \"name\"=>\"Selumetinib (AZD6244)\", \"pubchem_id\"=>nil}, {\"id\"=>291, \"name\"=>\"CP724714\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>686, \"name\"=>\"Intrinsic resistance to MEK inhibition in KRAS mutant lung and colon cancer through transcriptional induction of ERBB3.\", \"citation\"=>\"Sun et al., 2014, Cell Rep\", \"pubmed_id\"=>\"24685132\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24685132\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>4, \"day\"=>10}, \"journal\"=>\"Cell Rep\", \"full_journal_title\"=>\"Cell reports\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>289, \"phenotypes\"=>[]}, {\"id\"=>1152, \"name\"=>\"EID1152\", \"description\"=>\"Preclinical study in KRAS mutant lung and colon cancer models. A synthetic lethal screen using a total of 3,530 shRNAs to target 518 human kinases identified ERBB3 knockdown to confer sensitivity of selumetinib-resistant H358 cells (NSCLC cell line) to the MEK inhibitor selumetinib. The dual EGFR-ERBB2 inhibitors afatinib and dacomitinib each showed synergy with MEK Inhibition via selumetinib in 4 KRAS mutant cell lines. Western blot evaluation of 9 NSCLC and 12 CRC cell lines showed ERBB3 expression correlated with the synergy score for combination afatinib and trametinib treatment. I.e. the synergy score was low in cells having low basal levels of ERBB3 protein and high for cells having high basal ERBB3 expression. Using selumetinib in patient-derived CRC xenografts or trametinib in NSCLC xenografts, MEK inhibition by selumetinib or trametinib in combination with afatinib more effectively inhibited increases in tumor volume than either monotherapy.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>122, \"name\"=>\"Selumetinib (AZD6244)\", \"pubchem_id\"=>nil}, {\"id\"=>146, \"name\"=>\"Afatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>686, \"name\"=>\"Intrinsic resistance to MEK inhibition in KRAS mutant lung and colon cancer through transcriptional induction of ERBB3.\", \"citation\"=>\"Sun et al., 2014, Cell Rep\", \"pubmed_id\"=>\"24685132\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24685132\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>4, \"day\"=>10}, \"journal\"=>\"Cell Rep\", \"full_journal_title\"=>\"Cell reports\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>289, \"phenotypes\"=>[]}, {\"id\"=>729, \"name\"=>\"EID729\", \"description\"=>\"High expression of ERBB3 (her3) was associated with shorter overall survival (OS) from Cetuximab treatment in patients with metastatic colorectal cancer. Gene expression measures (qRT-PCR) were available for 103 patients.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>478, \"name\"=>\"Gene expression markers of efficacy and resistance to cetuximab treatment in metastatic colorectal cancer: results from CALGB 80203 (Alliance).\", \"citation\"=>\"Cushman et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"25520391\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25520391\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4772749\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>3, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>289, \"phenotypes\"=>[]}]}, {\"id\"=>290, \"entrez_name\"=>\"NT5E\", \"entrez_id\"=>4907, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>3976, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>86159809, \"stop\"=>86205496, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000257770.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3976/summary/variants/290/summary#variant\", \"evidence_items\"=>[{\"id\"=>730, \"name\"=>\"EID730\", \"description\"=>\"CD73 high expression was identified as a potential marker of improved PFS from cetuximab treatment in patients with metastatic CRC.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>478, \"name\"=>\"Gene expression markers of efficacy and resistance to cetuximab treatment in metastatic colorectal cancer: results from CALGB 80203 (Alliance).\", \"citation\"=>\"Cushman et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"25520391\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25520391\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4772749\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>3, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>290, \"phenotypes\"=>[]}]}, {\"id\"=>291, \"entrez_name\"=>\"EZH2\", \"entrez_id\"=>2146, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>63, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>148504477, \"stop\"=>148581370, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000320356.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/63/summary/variants/291/summary#variant\", \"evidence_items\"=>[{\"id\"=>731, \"name\"=>\"EID731\", \"description\"=>\"High expression of EZH2 was associated with poorer outcome in a meta-analysis encompassing 51 studies and 9444 patients.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>479, \"name\"=>\"Prognostic value of high EZH2 expression in patients with different types of cancer: a systematic review with meta-analysis.\", \"citation\"=>\"Jiang et al., 2016, Oncotarget\", \"pubmed_id\"=>\"26683709\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26683709\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4826228\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>1, \"day\"=>26}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>291, \"phenotypes\"=>[]}]}, {\"id\"=>292, \"entrez_name\"=>\"MYD88\", \"entrez_id\"=>4615, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>3742, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>38179969, \"stop\"=>38184510, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000417037.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3742/summary/variants/292/summary#variant\", \"evidence_items\"=>[{\"id\"=>734, \"name\"=>\"EID734\", \"description\"=>\"In this retrospective analysis of 211 breast cancer tissues, MYD88 overexpression was correlated  with poor survival. In-vitro experiments showed a decrease in sensitivity towards paclitaxel in cell-lines with high MYD88 expression.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>81, \"name\"=>\"Paclitaxel\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>482, \"name\"=>\"Increased expression of MyD88 and association with paclitaxel resistance in breast cancer.\", \"citation\"=>\"Xiang et al., 2016, Tumour Biol.\", \"pubmed_id\"=>\"26596839\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26596839\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>5}, \"journal\"=>\"Tumour Biol.\", \"full_journal_title\"=>\"Tumour biology : the journal of the International Society for Oncodevelopmental Biology and Medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>292, \"phenotypes\"=>[]}]}, {\"id\"=>295, \"entrez_name\"=>\"RET\", \"entrez_id\"=>5979, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>42, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>43572475, \"stop\"=>43625799, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000355710.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/42/summary/variants/295/summary#variant\", \"evidence_items\"=>[{\"id\"=>740, \"name\"=>\"EID740\", \"description\"=>\"Vandetanib was tested in 4 murine models of patient-derived xenografts. In xenografts with high expression of EGFR or RET, vandetanib led to tumor regression, in those without high expression, vandetanib slowed tumor growth.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>117, \"name\"=>\"Vandetanib\", \"pubchem_id\"=>\"\"}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>488, \"name\"=>\"Vandetanib as a potential new treatment for estrogen receptor-negative breast cancers.\", \"citation\"=>\"Hatem et al., 2016, Int. J. Cancer\", \"pubmed_id\"=>\"26686064\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26686064\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>5, \"day\"=>15}, \"journal\"=>\"Int. J. Cancer\", \"full_journal_title\"=>\"International journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>295, \"phenotypes\"=>[]}]}, {\"id\"=>296, \"entrez_name\"=>\"RIT1\", \"entrez_id\"=>6016, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>4875, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>155870065, \"stop\"=>155880706, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000368322.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4875/summary/variants/296/summary#variant\", \"evidence_items\"=>[{\"id\"=>741, \"name\"=>\"EID741\", \"description\"=>\"After preliminary studies in cell lines, GEO datasets and fresh frozen tissues, RIT1 expression was determined in 257 tumors and 31 normal tissues. Elevated expression of RIT1 was correlated with poorer overall survival, pathological type, clinical stage, grade and vascular invasion.\", \"disease\"=>{\"id\"=>108, \"name\"=>\"Endometrial Cancer\", \"display_name\"=>\"Endometrial Cancer\", \"doid\"=>\"1380\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1380\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>489, \"name\"=>\"Elevated expression of RIT1 correlates with poor prognosis in endometrial cancer.\", \"citation\"=>\"Xu et al., 2015, Int J Clin Exp Pathol\", \"pubmed_id\"=>\"26617739\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26617739\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4637554\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"Int J Clin Exp Pathol\", \"full_journal_title\"=>\"International journal of clinical and experimental pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>296, \"phenotypes\"=>[]}]}, {\"id\"=>304, \"entrez_name\"=>\"STK11\", \"entrez_id\"=>6794, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>5534, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"19\", \"start\"=>1205740, \"stop\"=>1228428, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000326873.7\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5534/summary/variants/304/summary#variant\", \"evidence_items\"=>[{\"id\"=>751, \"name\"=>\"EID751\", \"description\"=>\"STK11-deficient cells were sensitive to treatment with MAPK11 inhibitor SB202190.\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[{\"id\"=>133, \"name\"=>\"SB202190\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>497, \"name\"=>\"Loss of STK11 expression is an early event in prostate carcinogenesis and predicts therapeutic response to targeted therapy against MAPK/p38.\", \"citation\"=>\"Grossi et al., 2015, Autophagy\", \"pubmed_id\"=>\"26391455\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26391455\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4824604\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>11, \"day\"=>2}, \"journal\"=>\"Autophagy\", \"full_journal_title\"=>\"Autophagy\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>304, \"phenotypes\"=>[]}]}, {\"id\"=>305, \"entrez_name\"=>\"SYK\", \"entrez_id\"=>6850, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>5572, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>93564069, \"stop\"=>93660831, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000375754.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5572/summary/variants/305/summary#variant\", \"evidence_items\"=>[{\"id\"=>752, \"name\"=>\"EID752\", \"description\"=>\"Overexpression and high p-SYK/SYK ratio were associated with resistance to paclitaxel. Inhibition of SYK with R406 resensitized cell lines to paclitaxel.\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>81, \"name\"=>\"Paclitaxel\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>498, \"name\"=>\"Inhibition of Spleen Tyrosine Kinase Potentiates Paclitaxel-Induced Cytotoxicity in Ovarian Cancer Cells by Stabilizing Microtubules.\", \"citation\"=>\"Yu et al., 2015, Cancer Cell\", \"pubmed_id\"=>\"26096845\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26096845\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>7, \"day\"=>13}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>305, \"phenotypes\"=>[]}]}, {\"id\"=>313, \"entrez_name\"=>\"PTEN\", \"entrez_id\"=>5728, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>41, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>54.5, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>89622870, \"stop\"=>89731687, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000371953.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/41/summary/variants/313/summary#variant\", \"evidence_items\"=>[{\"id\"=>1624, \"name\"=>\"EID1624\", \"description\"=>\"Case report of a 44-year-old patient with recurrent HER2-positive breast cancer. The primary tumor was HR positive; however, the metastatic tumor was HR negative. The patient was resistant to classical chemotherapeutic agents and anti-HER2 treatment. Thus, the combination of everolimus and fulvestrant was administered. The patient experienced stable disease and PFS of 10 months. Genetic testing of pleural effusion after disease progression revealed a PIK3CA H1047R gene mutation but was positive for PTEN expression (IHC).\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}, {\"id\"=>54, \"name\"=>\"Fulvestrant\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>nil, \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1075, \"name\"=>\"Combined treatment with everolimus and fulvestrant reversed anti-HER2 resistance in a patient with refractory advanced breast cancer: a case report.\", \"citation\"=>\"Sun et al., 2016, Onco Targets Ther\", \"pubmed_id\"=>\"27445490\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27445490\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4936809\", \"publication_date\"=>{\"year\"=>2016}, \"journal\"=>\"Onco Targets Ther\", \"full_journal_title\"=>\"OncoTargets and therapy\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>313, \"phenotypes\"=>[]}, {\"id\"=>1129, \"name\"=>\"EID1129\", \"description\"=>\"Expression of EGFRvIII and PTEN was evaluated in 26 patients with glioblastomas treated with erlotinib or gefitinib and no EGFR or Her2/neu kinase domain mutations. 7 patients responded to treatment while 19 progressed rapidly. Coexpression of EGFRvIII and PTEN was associated with clinical response (defined as 25% tumor shrinkage). These results were confirmed using IHC in an independent cohort of 33 patients (8 responders) treated with erlotinib at another institution.\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[{\"id\"=>14, \"name\"=>\"Gefitinib\", \"pubchem_id\"=>nil}, {\"id\"=>15, \"name\"=>\"Erlotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>514, \"name\"=>\"Molecular determinants of the response of glioblastomas to EGFR kinase inhibitors.\", \"citation\"=>\"Mellinghoff et al., 2005, N. Engl. J. Med.\", \"pubmed_id\"=>\"16282176\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16282176\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>11, \"day\"=>10}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>313, \"phenotypes\"=>[]}, {\"id\"=>775, \"name\"=>\"EID775\", \"description\"=>\"In this preclinical study, the EGFR inhibitor dacomitinib had an effect on cell viability, self-renewal and proliferation in EGFR amplified GBM cells in-vitro and in-vivo (with or without the EGFRvIII mutation). It was less effective in one cell-line with PTEN deletion.\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[{\"id\"=>44, \"name\"=>\"Dacomitinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>516, \"name\"=>\"Preclinical Test of Dacomitinib, an Irreversible EGFR Inhibitor, Confirms Its Effectiveness for Glioblastoma.\", \"citation\"=>\"Zahonero et al., 2015, Mol. Cancer Ther.\", \"pubmed_id\"=>\"25939761\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25939761\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>7}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>313, \"phenotypes\"=>[]}, {\"id\"=>774, \"name\"=>\"EID774\", \"description\"=>\"49 patients (26 pretreatment tissues available for analysis) with recurrent glioblastomas who were treated with EGFR TKIs were analyzed in this study. Results were confirmed in an independent cohort of 33 patients. EGFRvIII and PTEN coexpression were associated with responsiveness to EGFR TKI.\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[{\"id\"=>145, \"name\"=>\"EGFR Inhibitor\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>514, \"name\"=>\"Molecular determinants of the response of glioblastomas to EGFR kinase inhibitors.\", \"citation\"=>\"Mellinghoff et al., 2005, N. Engl. J. Med.\", \"pubmed_id\"=>\"16282176\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16282176\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>11, \"day\"=>10}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>313, \"phenotypes\"=>[]}, {\"id\"=>955, \"name\"=>\"EID955\", \"description\"=>\"Trastuzumab was more effective in cell lines (SKBR3 vs. drug-resistant SKBR3/R) and 17 patient samples that exhibit elevated PTEN expression (by IHC).\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>657, \"name\"=>\"PTEN activity could be a predictive marker of trastuzumab efficacy in the treatment of ErbB2-overexpressing breast cancer.\", \"citation\"=>\"Fujita et al., 2006, Br. J. Cancer\", \"pubmed_id\"=>\"16404430\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16404430\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2361109\", \"publication_date\"=>{\"year\"=>2006, \"month\"=>1, \"day\"=>30}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>313, \"phenotypes\"=>[]}]}, {\"id\"=>314, \"entrez_name\"=>\"NRG1\", \"entrez_id\"=>3084, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>2593, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>37.0, \"coordinates\"=>{\"chromosome\"=>\"8\", \"start\"=>31497942, \"stop\"=>32622294, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000523534.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/2593/summary/variants/314/summary#variant\", \"evidence_items\"=>[{\"id\"=>1967, \"name\"=>\"EID1967\", \"description\"=>\"Reactivation of EGFR, HER2 and HER3 occurred within 24 hours of lapatinib treatment in breast cancer cell lines, This was associated with increased expression of NRG1 in cells treated with lapatinib.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>45, \"name\"=>\"Lapatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1382, \"name\"=>\"Combining lapatinib and pertuzumab to overcome lapatinib resistance due to NRG1-mediated signalling in HER2-amplified breast cancer.\", \"citation\"=>\"Leung et al., 2015, Oncotarget\", \"pubmed_id\"=>\"25691057\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25691057\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4467394\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>3, \"day\"=>20}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>314, \"phenotypes\"=>[]}, {\"id\"=>779, \"name\"=>\"EID779\", \"description\"=>\"In-vivo studies of residual NSCLC tumors (xenografts of 5 cell lines) after chemotherapy found increased EGFR-family signaling and NRG1 expression. NRG1 inhibition (YW538.24.71 antibody) led to decreased tumor growth and enhanced magnitude and response to chemotherapy.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>81, \"name\"=>\"Paclitaxel\", \"pubchem_id\"=>nil}, {\"id\"=>85, \"name\"=>\"Gemcitabine\", \"pubchem_id\"=>nil}, {\"id\"=>93, \"name\"=>\"Carboplatin\", \"pubchem_id\"=>nil}, {\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>520, \"name\"=>\"Blocking NRG1 and other ligand-mediated Her4 signaling enhances the magnitude and duration of the chemotherapeutic response of non-small cell lung cancer.\", \"citation\"=>\"Hegde et al., 2013, Sci Transl Med\", \"pubmed_id\"=>\"23390248\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23390248\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>2, \"day\"=>6}, \"journal\"=>\"Sci Transl Med\", \"full_journal_title\"=>\"Science translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>314, \"phenotypes\"=>[]}, {\"id\"=>813, \"name\"=>\"EID813\", \"description\"=>\"In this phase-2 study, 215 patients were randomized to either erlotinib and placebo or erlotinib and HER3 monoclonal antibody patritumab. HRG (NRG1) expression was retrospectively defined as predictive biomarker but prospectively assessed in 102 patients before unblinding. NRG1-high groups demonstrated clinical benefit from additional patritumab with hazard ratios of 0.37 (P = 0.0283) and 0.29 (P = 0.0027) in the high- and low-dose patritumab arms, respectively.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>262, \"name\"=>\"Patritumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>545, \"name\"=>\"Clinical Translation and Validation of a Predictive Biomarker for Patritumab, an Anti-human Epidermal Growth Factor Receptor 3 (HER3) Monoclonal Antibody, in Patients With Advanced Non-small Cell Lung Cancer.\", \"citation\"=>\"Mendell et al., 2015, EBioMedicine\", \"pubmed_id\"=>\"26137564\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26137564\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4484825\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>3}, \"journal\"=>\"EBioMedicine\", \"full_journal_title\"=>\"EBioMedicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>314, \"phenotypes\"=>[]}, {\"id\"=>861, \"name\"=>\"EID861\", \"description\"=>\"This screening study in various tumor xenografts identified NRG1 expression to correlate with sensivity to ERBB3 inhibitory antibody AV-203.\\nERBB3 expression was not predictive of response.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>225, \"name\"=>\"AV-203\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>577, \"name\"=>\"Neuregulin 1 expression is a predictive biomarker for response to AV-203, an ERBB3 inhibitory antibody, in human tumor models.\", \"citation\"=>\"Meetze et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"25542901\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25542901\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>3, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>314, \"phenotypes\"=>[]}, {\"id\"=>778, \"name\"=>\"EID778\", \"description\"=>\"An NRG1-ERBB3 autocrine signal-transducing loop was identified in a subset of ovarian cancers and cell lines. Disruption of this loop led to decreased cell-growth in-vitro and prolonged survival in-vivo. An ERBB3 directed antibody (MM-121) inhibited tumor-growth in-vivo.\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>221, \"name\"=>\"Anti-ERBB3 Agents\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>519, \"name\"=>\"An activated ErbB3/NRG1 autocrine loop supports in vivo proliferation in ovarian cancer cells.\", \"citation\"=>\"Sheng et al., 2010, Cancer Cell\", \"pubmed_id\"=>\"20227043\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20227043\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2897158\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>3, \"day\"=>16}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>314, \"phenotypes\"=>[]}, {\"id\"=>777, \"name\"=>\"EID777\", \"description\"=>\"NRG1 upregulation or ERBB2 amplification were the main causes of resistance to cetuximab in-vivo and in-vitro in this study. Inhibition of ERBB2/ERBB3 signaling restored cetuximab sensitivity.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>518, \"name\"=>\"Activation of ERBB2 signaling causes resistance to the EGFR-directed therapeutic antibody cetuximab.\", \"citation\"=>\"Yonesaka et al., 2011, Sci Transl Med\", \"pubmed_id\"=>\"21900593\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21900593\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3268675\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>9, \"day\"=>7}, \"journal\"=>\"Sci Transl Med\", \"full_journal_title\"=>\"Science translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>314, \"phenotypes\"=>[]}, {\"id\"=>776, \"name\"=>\"EID776\", \"description\"=>\"In non-HER2 amplified cell lines, sensitivity to HER2 kinase inhibitor lapatinib was found mainly in cell-lines with elevated NRG1 expression and HER3 activation. This aberration was mainly identified in head and neck cancer cell lines.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>45, \"name\"=>\"Lapatinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>517, \"name\"=>\"Neuregulin-1-mediated autocrine signaling underlies sensitivity to HER2 kinase inhibitors in a subset of human cancers.\", \"citation\"=>\"Wilson et al., 2011, Cancer Cell\", \"pubmed_id\"=>\"21840482\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21840482\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>8, \"day\"=>16}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>314, \"phenotypes\"=>[]}]}, {\"id\"=>315, \"entrez_name\"=>\"AREG\", \"entrez_id\"=>374, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>389, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>58.0, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>75310851, \"stop\"=>75320726, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000395748.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/389/summary/variants/315/summary#variant\", \"evidence_items\"=>[{\"id\"=>1020, \"name\"=>\"EID1020\", \"description\"=>\"Prospectively planned retrospective biomarker study in archived tumor tissue from 323 patients from the PICCOLO trial (panitumumab plus irinotecan in CRC patients with KRAS wt). Tumors were classified as \\\"high expressor\\\" (either EREG or AREG mRNA level) or \\\"low expressor\\\" (neither EREG nor AREG in top tertile). \\nFor RAS wild-type patients with high ligand expression, median PFS was 8.3 [4.0-11.0] months (irinotecan with panitumumab) vs 4.4 [2.8-6.7] months (irinotecan alone); HR, 0.38 [95% CI, 0.24-0.61]; P < .001. In RAS wild-type patients with low ligand expression, median PFS was 3.2 [2.7-8.1] months (irinotecan with panitumumab) vs 4.0 [2.7-7.5] months (irinotecan); HR, 0.93 [95% CI, 0.64-1.37]; P = .73; interaction test results were significant [P = .01]). Results were less clear for response rate (interaction P = .17) and OS (interaction P = .11).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>28, \"name\"=>\"Panitumumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>707, \"name\"=>\"Combined Epiregulin and Amphiregulin Expression Levels as a Predictive Biomarker for Panitumumab Therapy Benefit or Lack of Benefit in Patients With RAS Wild-Type Advanced Colorectal Cancer.\", \"citation\"=>\"Seligmann et al., 2016, JAMA Oncol\", \"pubmed_id\"=>\"26867820\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26867820\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>2, \"day\"=>11}, \"journal\"=>\"JAMA Oncol\", \"full_journal_title\"=>\"JAMA oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>315, \"phenotypes\"=>[]}, {\"id\"=>846, \"name\"=>\"EID846\", \"description\"=>\"47 HNSCC patients treated with cetuximab/docetaxel were retrospectively analyzed for EGFRvIII, EGFR and AREG expression with IHC. 45% of cases expressed high level of AREG. High AREG expression in tumor cells was associated with reduced overall survival  (hazard ratio: 2.2, P = 0.002) and progression free survival (hazard ratio: 2.2, P = 0.016) compared to patients with low AREG expression score. AREG was an independent prognosticator of OS in multivariate Cox analysis. High EGFRvIII and AREG expression levels identified SCCHN (squamous cell carcinoma, head and neck) patients that were less likely to benefit from combination treatment with cetuximab and docetaxel.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}, {\"id\"=>73, \"name\"=>\"Docetaxel\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>568, \"name\"=>\"Expression of amphiregulin and EGFRvIII affect outcome of patients with squamous cell carcinoma of the head and neck receiving cetuximab-docetaxel treatment.\", \"citation\"=>\"Tinhofer et al., 2011, Clin. Cancer Res.\", \"pubmed_id\"=>\"21653686\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21653686\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>8, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>315, \"phenotypes\"=>[]}, {\"id\"=>780, \"name\"=>\"EID780\", \"description\"=>\"A cell line (DFCI076) with EML1-ALK fusion and resistance mutation L1152R showed no response to ALK inhibition with crizotinib or TAE684. There was co-dependancy on AKT and EGFR-signaling and inhibition of both pathways led to decreased cell-growth. Amphiregulin (AREG) production was identified as a potential mediator of EGFR-activation (no other alterations detectable). The same mechanism could be reproduced in a cell line (H3122) without EML1-ALK mutation and resistance to crizotinib that was acquired in-vitro.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>506, \"name\"=>\"A novel ALK secondary mutation and EGFR signaling cause resistance to ALK kinase inhibitors.\", \"citation\"=>\"Sasaki et al., 2011, Cancer Res.\", \"pubmed_id\"=>\"21791641\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21791641\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3278914\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>9, \"day\"=>15}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>315, \"phenotypes\"=>[]}, {\"id\"=>781, \"name\"=>\"EID781\", \"description\"=>\"Five patients with acquired resistance to crizotinib showed high concentration of amphiregulin in malignant pleural effusions.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>508, \"name\"=>\"Heterogeneity of genetic changes associated with acquired crizotinib resistance in ALK-rearranged lung cancer.\", \"citation\"=>\"Kim et al., 2013, J Thorac Oncol\", \"pubmed_id\"=>\"23344087\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23344087\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>4}, \"journal\"=>\"J Thorac Oncol\", \"full_journal_title\"=>\"Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>315, \"phenotypes\"=>[]}, {\"id\"=>847, \"name\"=>\"EID847\", \"description\"=>\"47 HNSCC patients treated with cetuximab/docetaxel were retrospectively analyzed for EGFRvIII, EGFR and AREG expression with IHC. 45% of cases expressed high level of AREG. High AREG expression in tumor cells was associated with reduced OS (HR: 2.2, P = 0.002) and PFS (HR 2.2, P = 0.019) compared with patients with low expression score. AREG was an independent prognosticator of OS in multivariate Cox analysis.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"rejected\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>568, \"name\"=>\"Expression of amphiregulin and EGFRvIII affect outcome of patients with squamous cell carcinoma of the head and neck receiving cetuximab-docetaxel treatment.\", \"citation\"=>\"Tinhofer et al., 2011, Clin. Cancer Res.\", \"pubmed_id\"=>\"21653686\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21653686\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>8, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>315, \"phenotypes\"=>[]}, {\"id\"=>788, \"name\"=>\"EID788\", \"description\"=>\"226 cetuximab-treated patients with colorectal cancer (CRC) were analyzed for mRNA expression (by qPCR) of EGFR and its ligands (EGF, TGFA, AREG and EREG). High AREG mRNA expression in KRAS wild type tumours was a favorable predictor in a multivariate analysis (median survival 33 vs. 15 months, p=0.0005).  \\nCetuximab-treated patients with AREG-low KRAS wild type CRC had poor survival, similar to KRAS mutated CRC.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>524, \"name\"=>\"Biomarkers of benefit from cetuximab-based therapy in metastatic colorectal cancer: interaction of EGFR ligand expression with RAS/RAF, PIK3CA genotypes.\", \"citation\"=>\"Pentheroudakis et al., 2013, BMC Cancer\", \"pubmed_id\"=>\"23374602\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23374602\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3599697\", \"publication_date\"=>{\"year\"=>2013}, \"journal\"=>\"BMC Cancer\", \"full_journal_title\"=>\"BMC cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>315, \"phenotypes\"=>[]}]}, {\"id\"=>316, \"entrez_name\"=>\"TGFA\", \"entrez_id\"=>7039, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>5742, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>70674412, \"stop\"=>70781147, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000295400.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5742/summary/variants/316/summary#variant\", \"evidence_items\"=>[{\"id\"=>785, \"name\"=>\"EID785\", \"description\"=>\"Primary tumors of 91 patients with head and neck squamous cell carcinoma were analyzed for expression of EGFR and TGF-alpha using IHC. Expression of EGFR or TGF-alpha were associated with reduced disease-free and cause-specific survival.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>521, \"name\"=>\"Levels of TGF-alpha and EGFR protein in head and neck squamous cell carcinoma and patient survival.\", \"citation\"=>\"Rubin Grandis et al., 1998, J. Natl. Cancer Inst.\", \"pubmed_id\"=>\"9625170\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/9625170\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1998, \"month\"=>6, \"day\"=>3}, \"journal\"=>\"J. Natl. Cancer Inst.\", \"full_journal_title\"=>\"Journal of the National Cancer Institute\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>316, \"phenotypes\"=>[]}]}, {\"id\"=>318, \"entrez_name\"=>\"EREG\", \"entrez_id\"=>2069, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>1737, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>35.5, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>75230860, \"stop\"=>75254468, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000244869.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1737/summary/variants/318/summary#variant\", \"evidence_items\"=>[{\"id\"=>1021, \"name\"=>\"EID1021\", \"description\"=>\"Prospectively planned retrospective biomarker study in archived tumor tissue from 323 patients from the PICCOLO trial (panitumumab plus irinotecan in CRC patients with KRAS wt). Tumors were classified as \\\"high expressor\\\" (either EREG or AREG mRNA level) or \\\"low expressor\\\" (neither EREG nor AREG in top tertile). For RAS wild-type patients with high ligand expression, median PFS was 8.3 [4.0-11.0] months (irinotecan with panitumumab) vs 4.4 [2.8-6.7] months (irinotecan alone); HR, 0.38 [95% CI, 0.24-0.61]; P < .001. In RAS wild-type patients with low ligand expression, median PFS was 3.2 [2.7-8.1] months (irinotecan with panitumumab) vs 4.0 [2.7-7.5] months (irinotecan); HR, 0.93 [95% CI, 0.64-1.37]; P = .73; interaction test results were significant [P = .01]). Results were less clear for response rate (interaction P = .17) and OS (interaction P = .11).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>28, \"name\"=>\"Panitumumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>707, \"name\"=>\"Combined Epiregulin and Amphiregulin Expression Levels as a Predictive Biomarker for Panitumumab Therapy Benefit or Lack of Benefit in Patients With RAS Wild-Type Advanced Colorectal Cancer.\", \"citation\"=>\"Seligmann et al., 2016, JAMA Oncol\", \"pubmed_id\"=>\"26867820\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26867820\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>2, \"day\"=>11}, \"journal\"=>\"JAMA Oncol\", \"full_journal_title\"=>\"JAMA oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>318, \"phenotypes\"=>[]}, {\"id\"=>1055, \"name\"=>\"EID1055\", \"description\"=>\"Study reporting an in-vivo two-stage initiation/promotion model, testing the hypothesis that Ereg deficiency would reduce lung tumor promotion in mice. Cancerogenesis with chemical initiators/promoters (3-methylcholanthrene or control, butylated hydroxytoluene) was assessed in mice lacking Ereg (Ereg-/-) and wildtype controls (BALB/ByJ; Ereg+/+) at multiple time points and endpoints (bronchoalveolar lavage analysis, tumor analysis, mRNA expression, ELISA, wound assay). Initially a significantly reduced amount of inflammation (macrophages, PMNs) was observed in the Ereg-/- mice. At 20 wk, tumor multiplicity was also significantly decreased in the Ereg-/- mice.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"E\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>733, \"name\"=>\"Epiregulin is required for lung tumor promotion in a murine two-stage carcinogenesis model.\", \"citation\"=>\"Bauer et al., 2016, Mol. Carcinog.\", \"pubmed_id\"=>\"26894620\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26894620\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>2, \"day\"=>19}, \"journal\"=>\"Mol. Carcinog.\", \"full_journal_title\"=>\"Molecular carcinogenesis\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>318, \"phenotypes\"=>[]}, {\"id\"=>789, \"name\"=>\"EID789\", \"description\"=>\"226 cetuximab-treated patients with CRC were analyzed for mRNA expression of EGFR and its ligands. Favorable predictors in a multivariate analysis were high AREG mRNA in KRAS wild type tumours, high EREG mRNA and low Ephrin A2 receptor mRNA.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>524, \"name\"=>\"Biomarkers of benefit from cetuximab-based therapy in metastatic colorectal cancer: interaction of EGFR ligand expression with RAS/RAF, PIK3CA genotypes.\", \"citation\"=>\"Pentheroudakis et al., 2013, BMC Cancer\", \"pubmed_id\"=>\"23374602\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23374602\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3599697\", \"publication_date\"=>{\"year\"=>2013}, \"journal\"=>\"BMC Cancer\", \"full_journal_title\"=>\"BMC cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>318, \"phenotypes\"=>[]}]}, {\"id\"=>325, \"entrez_name\"=>\"FGFR3\", \"entrez_id\"=>2261, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>23, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>1795039, \"stop\"=>1810599, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000440486.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/23/summary/variants/325/summary#variant\", \"evidence_items\"=>[{\"id\"=>803, \"name\"=>\"EID803\", \"description\"=>\"The effects of FGFR inhibitors (PD173074, TKI-258 and SU5402) in bladder carcinoma cell lines were related to FGFR3 (and/or FGFR1) expression.\", \"disease\"=>{\"id\"=>18, \"name\"=>\"Bladder Carcinoma\", \"display_name\"=>\"Bladder Carcinoma\", \"doid\"=>\"4007\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4007\"}, \"drugs\"=>[{\"id\"=>36, \"name\"=>\"PD173074\", \"pubchem_id\"=>nil}, {\"id\"=>153, \"name\"=>\"SU-5402\", \"pubchem_id\"=>nil}, {\"id\"=>228, \"name\"=>\"Dovitinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>535, \"name\"=>\"Small molecule FGF receptor inhibitors block FGFR-dependent urothelial carcinoma growth in vitro and in vivo.\", \"citation\"=>\"Lamont et al., 2011, Br. J. Cancer\", \"pubmed_id\"=>\"21119661\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21119661\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3039817\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>1, \"day\"=>4}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>325, \"phenotypes\"=>[]}]}, {\"id\"=>326, \"entrez_name\"=>\"CBLC\", \"entrez_id\"=>23624, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>9705, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"19\", \"start\"=>45281126, \"stop\"=>45303891, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000270279.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/9705/summary/variants/326/summary#variant\", \"evidence_items\"=>[{\"id\"=>805, \"name\"=>\"EID805\", \"description\"=>\"The E3 ubiquitin ligase, CBLC, was identified as a candidate biomarker for olaparib sensitivity in a siRNA screen and further validated in-vitro. In other words, expression of CBLC is predicted to confer resistance to olaparib.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>539, \"name\"=>\"Complementary genetic screens identify the E3 ubiquitin ligase CBLC, as a modifier of PARP inhibitor sensitivity.\", \"citation\"=>\"Frankum et al., 2015, Oncotarget\", \"pubmed_id\"=>\"25883215\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25883215\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4484416\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>5, \"day\"=>10}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>326, \"phenotypes\"=>[]}]}, {\"id\"=>328, \"entrez_name\"=>\"DEFA1\", \"entrez_id\"=>1667, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>1428, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"8\", \"start\"=>6835172, \"stop\"=>6837602, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000382692.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1428/summary/variants/328/summary#variant\", \"evidence_items\"=>[{\"id\"=>808, \"name\"=>\"EID808\", \"description\"=>\"Blood mRNA levels of DEFA1, DEFA1B and DEFA3 were associated with response to docetaxel treatment in castration resistant prostate cancer. The discovery cohort consisted of 6 patients, results were validated in 10 patients.\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[{\"id\"=>73, \"name\"=>\"Docetaxel\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>542, \"name\"=>\"Whole blood defensin mRNA expression is a predictive biomarker of docetaxel response in castration-resistant prostate cancer.\", \"citation\"=>\"Kohli et al., 2015, Onco Targets Ther\", \"pubmed_id\"=>\"26261420\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26261420\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4527520\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"Onco Targets Ther\", \"full_journal_title\"=>\"OncoTargets and therapy\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>328, \"phenotypes\"=>[]}]}, {\"id\"=>329, \"entrez_name\"=>\"ERCC1\", \"entrez_id\"=>2067, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>1735, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>30.0, \"coordinates\"=>{\"chromosome\"=>\"19\", \"start\"=>45916692, \"stop\"=>45926824, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000013807.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>\"\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1735/summary/variants/329/summary#variant\", \"evidence_items\"=>[{\"id\"=>810, \"name\"=>\"EID810\", \"description\"=>\"ERCC1 expression as evaluated by IHC in 432 patients was associated with better disease-free survival (HR 0.70, p = 0.028) and cancer specific survival (HR 0.70, p = 0.032) on multivariable analysis.\", \"disease\"=>{\"id\"=>18, \"name\"=>\"Bladder Carcinoma\", \"display_name\"=>\"Bladder Carcinoma\", \"doid\"=>\"4007\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4007\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>543, \"name\"=>\"ERCC1 as a Prognostic and Predictive Biomarker for Urothelial Carcinoma of the Bladder following Radical Cystectomy.\", \"citation\"=>\"Klatte et al., 2015, J. Urol.\", \"pubmed_id\"=>\"26162296\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26162296\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>11}, \"journal\"=>\"J. Urol.\", \"full_journal_title\"=>\"The Journal of urology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>329, \"phenotypes\"=>[]}, {\"id\"=>809, \"name\"=>\"EID809\", \"description\"=>\"Immunhistochemistry of ERCC1 expression in 432 patients revealed no correlation to response to cisplatin based chemotherapy. In-vitro studies also did not show any effects.\", \"disease\"=>{\"id\"=>18, \"name\"=>\"Bladder Carcinoma\", \"display_name\"=>\"Bladder Carcinoma\", \"doid\"=>\"4007\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4007\"}, \"drugs\"=>[{\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>543, \"name\"=>\"ERCC1 as a Prognostic and Predictive Biomarker for Urothelial Carcinoma of the Bladder following Radical Cystectomy.\", \"citation\"=>\"Klatte et al., 2015, J. Urol.\", \"pubmed_id\"=>\"26162296\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26162296\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>11}, \"journal\"=>\"J. Urol.\", \"full_journal_title\"=>\"The Journal of urology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>329, \"phenotypes\"=>[]}]}, {\"id\"=>334, \"entrez_name\"=>\"VEGFA\", \"entrez_id\"=>7422, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>6071, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>43738444, \"stop\"=>43752346, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000372055.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/6071/summary/variants/334/summary#variant\", \"evidence_items\"=>[{\"id\"=>818, \"name\"=>\"EID818\", \"description\"=>\"Decreased peri- and post-therapeutic expression of VEGFA were significantly associated with response to FOLFIRI plus bevacizumab in 57 patients with metastatic CRC.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>33, \"name\"=>\"Bevacizumab\", \"pubchem_id\"=>nil}, {\"id\"=>101, \"name\"=>\"Irinotecan\", \"pubchem_id\"=>nil}, {\"id\"=>215, \"name\"=>\"L-Folinic Acid\", \"pubchem_id\"=>nil}, {\"id\"=>395, \"name\"=>\"5-fluorouracil\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>550, \"name\"=>\"Decreased peritherapeutic VEGF expression could be a predictor of responsiveness to first-line FOLFIRI plus bevacizumab in mCRC patients.\", \"citation\"=>\"Tsai et al., 2015, Int J Clin Exp Pathol\", \"pubmed_id\"=>\"25973082\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25973082\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4396259\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"Int J Clin Exp Pathol\", \"full_journal_title\"=>\"International journal of clinical and experimental pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>334, \"phenotypes\"=>[]}]}, {\"id\"=>335, \"entrez_name\"=>\"DNMT1\", \"entrez_id\"=>1786, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>1510, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>18.0, \"coordinates\"=>{\"chromosome\"=>\"19\", \"start\"=>10244022, \"stop\"=>10305811, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000340748.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1510/summary/variants/335/summary#variant\", \"evidence_items\"=>[{\"id\"=>819, \"name\"=>\"EID819\", \"description\"=>\"DNMT1 expression (mRNA expression from an independent study) correlated with sensitivity to decitabine (ρ, −0.443, Pearson regression analysis) in this preclinical study of 45 solid tumor cell lines. DNMT1 protein expression was also evaluated in two KRAS-mutant ovarian cancer cell lines pre and post-decitabine treatment showing downregulation following treatment.\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>155, \"name\"=>\"Decitabine\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>551, \"name\"=>\"KRAS Genomic Status Predicts the Sensitivity of Ovarian Cancer Cells to Decitabine.\", \"citation\"=>\"Stewart et al., 2015, Cancer Res.\", \"pubmed_id\"=>\"25968887\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25968887\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4506246\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>7, \"day\"=>15}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>335, \"phenotypes\"=>[]}, {\"id\"=>946, \"name\"=>\"EID946\", \"description\"=>\"High DNMT1 expression was found in 105/127 (83%) or patients (by qPCR assay). Low DNMT1 expression was associated with improved histopathological and clinical response and OS in patients with gastric cancer  (P=0.03/P=0.008, P(log-rank)=0.001, respectively). In one cisplatin resistant cell line, DAC treatment and cisplatin had a synergistic effect.\", \"disease\"=>{\"id\"=>147, \"name\"=>\"Gastric Adenocarcinoma\", \"display_name\"=>\"Gastric Adenocarcinoma\", \"doid\"=>\"3717\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3717\"}, \"drugs\"=>[{\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>648, \"name\"=>\"DNA methyltransferase 1 as a predictive biomarker and potential therapeutic target for chemotherapy in gastric cancer.\", \"citation\"=>\"Mutze et al., 2011, Eur. J. Cancer\", \"pubmed_id\"=>\"21458988\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21458988\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>8}, \"journal\"=>\"Eur. J. Cancer\", \"full_journal_title\"=>\"European journal of cancer (Oxford, England : 1990)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>335, \"phenotypes\"=>[]}]}, {\"id\"=>337, \"entrez_name\"=>\"PDCD4\", \"entrez_id\"=>27250, \"name\"=>\"EXPRESSION\", \"description\"=>\"A new study, a retrospective analysis of 77 patients, has implicated PDCD4 expression in the sensitivity to paclitaxel. However, this finding came from a large proteome screen and has not yet been replicated in independent patient sets.\", \"gene_id\"=>10573, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>112631596, \"stop\"=>112659763, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000393104.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/10573/summary/variants/337/summary#variant\", \"evidence_items\"=>[{\"id\"=>821, \"name\"=>\"EID821\", \"description\"=>\"PDCD4 expression was associated with response to paclitaxel in a retrospective analysis of 77 patients after preclinical validation and mechanistic analysis of the biomarker.\", \"disease\"=>{\"id\"=>23, \"name\"=>\"Lung Cancer\", \"display_name\"=>\"Lung Cancer\", \"doid\"=>\"1324\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1324\"}, \"drugs\"=>[{\"id\"=>81, \"name\"=>\"Paclitaxel\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>552, \"name\"=>\"Proteomic Profiling of Paclitaxel Treated Cells Identifies a Novel Mechanism of Drug Resistance Mediated by PDCD4.\", \"citation\"=>\"Xu et al., 2015, J. Proteome Res.\", \"pubmed_id\"=>\"25928036\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25928036\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>6, \"day\"=>5}, \"journal\"=>\"J. Proteome Res.\", \"full_journal_title\"=>\"Journal of proteome research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>337, \"phenotypes\"=>[]}]}, {\"id\"=>339, \"entrez_name\"=>\"TFF3\", \"entrez_id\"=>7033, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>5737, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"21\", \"start\"=>43731777, \"stop\"=>43735761, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000518498.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5737/summary/variants/339/summary#variant\", \"evidence_items\"=>[{\"id\"=>823, \"name\"=>\"EID823\", \"description\"=>\"TFF3 expression was assessed by IHC in 75 patients, 187 patients were identified for the validation cohort. TFF3 expression was associated with response to endocrine therapy (tamoxifen, aminoglutethimide or oophorectomy) and outperformed ER, PR and TFF1 as biomarkers.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>53, \"name\"=>\"Tamoxifen\", \"pubchem_id\"=>nil}, {\"id\"=>235, \"name\"=>\"Aminoglutethimide\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>554, \"name\"=>\"TFF3 is a valuable predictive biomarker of endocrine response in metastatic breast cancer.\", \"citation\"=>\"May et al., 2015, Endocr. Relat. Cancer\", \"pubmed_id\"=>\"25900183\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25900183\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4455223\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>6}, \"journal\"=>\"Endocr. Relat. Cancer\", \"full_journal_title\"=>\"Endocrine-related cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>339, \"phenotypes\"=>[]}]}, {\"id\"=>340, \"entrez_name\"=>\"DKK1\", \"entrez_id\"=>22943, \"name\"=>\"NUCLEAR EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>9171, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>54074056, \"stop\"=>54077417, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000373970.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/9171/summary/variants/340/summary#variant\", \"evidence_items\"=>[{\"id\"=>824, \"name\"=>\"EID824\", \"description\"=>\"Nuclear expression of DICKKOPF-1 was identified in 15% of 699 patients with colorectal cancer and was associated with decreased progression-free survival (PFS) and overall survival (OS) after chemotherapy (FOLFOX, FOLFIRI, or 5-FU) [adjusted HR, 1.65; 95% confidence interval (CI), 1.23-2.21; P = 0.002)].\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>101, \"name\"=>\"Irinotecan\", \"pubchem_id\"=>nil}, {\"id\"=>215, \"name\"=>\"L-Folinic Acid\", \"pubchem_id\"=>nil}, {\"id\"=>237, \"name\"=>\"Oxaliplatin\", \"pubchem_id\"=>nil}, {\"id\"=>395, \"name\"=>\"5-fluorouracil\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>555, \"name\"=>\"Nuclear DICKKOPF-1 as a biomarker of chemoresistance and poor clinical outcome in colorectal cancer.\", \"citation\"=>\"Aguilera et al., 2015, Oncotarget\", \"pubmed_id\"=>\"25788273\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25788273\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4467410\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>3, \"day\"=>20}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>340, \"phenotypes\"=>[]}]}, {\"id\"=>341, \"entrez_name\"=>\"CD44\", \"entrez_id\"=>960, \"name\"=>\"ISOFORM EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>855, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"11\", \"start\"=>35160728, \"stop\"=>35251574, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000428726.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/855/summary/variants/341/summary#variant\", \"evidence_items\"=>[{\"id\"=>825, \"name\"=>\"EID825\", \"description\"=>\"CD44 isoform expression (especially CD44s) was identified to predict response to CD44 antibody RG7356 preclinically. Retrospective evaluation of 13 patients from a phase 1 study, treated with RG7356, identified 3 patients with CD44s isoform. 2/3 patients with CD44s had SD as best response. Overall, 5 patients had an OR.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>240, \"name\"=>\"RG7356\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>556, \"name\"=>\"CD44 Isoform Status Predicts Response to Treatment with Anti-CD44 Antibody in Cancer Patients.\", \"citation\"=>\"Birzele et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"25762343\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25762343\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>6, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>341, \"phenotypes\"=>[]}]}, {\"id\"=>342, \"entrez_name\"=>\"EGF\", \"entrez_id\"=>1950, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>1634, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>110834047, \"stop\"=>110933422, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000265171.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1634/summary/variants/342/summary#variant\", \"evidence_items\"=>[{\"id\"=>826, \"name\"=>\"EID826\", \"description\"=>\"3 tongue cancer cell lines were used for this study. Addition of recombinant human EGF led to increased cetuximab resistance. EGF downregulation suppressed proliferation of cell lines. Results for amphiregulin and epiregulin were inconclusive.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>557, \"name\"=>\"Epidermal growth factor is a potential biomarker for poor cetuximab response in tongue cancer cells.\", \"citation\"=>\"Ansell et al., 2016, J. Oral Pathol. Med.\", \"pubmed_id\"=>\"25677871\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25677871\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>1}, \"journal\"=>\"J. Oral Pathol. Med.\", \"full_journal_title\"=>\"Journal of oral pathology & medicine : official publication of the International Association of Oral Pathologists and the American Academy of Oral Pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>342, \"phenotypes\"=>[]}]}, {\"id\"=>343, \"entrez_name\"=>\"MDM2\", \"entrez_id\"=>4193, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>3465, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"12\", \"start\"=>69201956, \"stop\"=>69239214, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000462284.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3465/summary/variants/343/summary#variant\", \"evidence_items\"=>[{\"id\"=>827, \"name\"=>\"EID827\", \"description\"=>\"72 patients with malignant pleural mesothelioma were analyzed, MDM2 immunoexpression was assessed in 65 patients. MDM2 mRNA and protein expression correlated with poor OS and PFS for patients treated with cisplatin and pemetrexed (standard of care therapy).\", \"disease\"=>{\"id\"=>199, \"name\"=>\"Malignant Pleural Mesothelioma\", \"display_name\"=>\"Malignant Pleural Mesothelioma\", \"doid\"=>\"7474\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:7474\"}, \"drugs\"=>[{\"id\"=>156, \"name\"=>\"Pemetrexed\", \"pubchem_id\"=>nil}, {\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>558, \"name\"=>\"MDM2 is an important prognostic and predictive factor for platin-pemetrexed therapy in malignant pleural mesotheliomas and deregulation of P14/ARF (encoded by CDKN2A) seems to contribute to an MDM2-driven inactivation of P53.\", \"citation\"=>\"Walter et al., 2015, Br. J. Cancer\", \"pubmed_id\"=>\"25668009\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25668009\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4453955\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>3, \"day\"=>3}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>343, \"phenotypes\"=>[]}, {\"id\"=>828, \"name\"=>\"EID828\", \"description\"=>\"72 patients with malignant pleural mesothelioma were analyzed, MDM2 immunoexpression was assessed in 65 patients. MDM2 mRNA and protein expression correlated with poor OS and PFS for patients treated with standard of care therapy (cisplatin and pemetrexed).\", \"disease\"=>{\"id\"=>199, \"name\"=>\"Malignant Pleural Mesothelioma\", \"display_name\"=>\"Malignant Pleural Mesothelioma\", \"doid\"=>\"7474\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:7474\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>558, \"name\"=>\"MDM2 is an important prognostic and predictive factor for platin-pemetrexed therapy in malignant pleural mesotheliomas and deregulation of P14/ARF (encoded by CDKN2A) seems to contribute to an MDM2-driven inactivation of P53.\", \"citation\"=>\"Walter et al., 2015, Br. J. Cancer\", \"pubmed_id\"=>\"25668009\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25668009\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4453955\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>3, \"day\"=>3}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>343, \"phenotypes\"=>[]}]}, {\"id\"=>344, \"entrez_name\"=>\"HMOX1\", \"entrez_id\"=>3162, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>2657, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"22\", \"start\"=>35776828, \"stop\"=>35790207, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000216117.8\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/2657/summary/variants/344/summary#variant\", \"evidence_items\"=>[{\"id\"=>829, \"name\"=>\"EID829\", \"description\"=>\"In 66 patients with advanced renal cell carcinoma, high expression of heme oxygenase-1 (HO-1) was associated with poor overall response rate (2.6% versus 53.6%, P<0.01), clinical benefit rate (47.4% versus 92.9%, P<0.01), shorter progression-free survival (4.4 versus 42 months, P=0.022) and poor overall survival (χ (2)=4.775, P=0.029) in patients receiving sorafenib or sunitinib.\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[{\"id\"=>6, \"name\"=>\"Sorafenib\", \"pubchem_id\"=>nil}, {\"id\"=>157, \"name\"=>\"Sunitinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>559, \"name\"=>\"Heme oxygenase-1 is a predictive biomarker for therapeutic targeting of advanced clear cell renal cell carcinoma treated with sorafenib or sunitinib.\", \"citation\"=>\"Zheng et al., 2015, Onco Targets Ther\", \"pubmed_id\"=>\"26309414\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26309414\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4539079\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"Onco Targets Ther\", \"full_journal_title\"=>\"OncoTargets and therapy\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>344, \"phenotypes\"=>[]}]}, {\"id\"=>345, \"entrez_name\"=>\"SPHK1\", \"entrez_id\"=>8877, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>7041, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"17\", \"start\"=>74380731, \"stop\"=>74383936, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000323374.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/7041/summary/variants/345/summary#variant\", \"evidence_items\"=>[{\"id\"=>830, \"name\"=>\"EID830\", \"description\"=>\"SPHK1 protein expression was assessed in 287 cervical cancer and 5 normal cervical tissue samples using IHC. Univariate analysis for OS showed SPHK1 expression (p = 0.033; Figure 1C), tumor size (p = 0.033), and lymph node metastasis (p = 0.008) as associated with OS. SPHK1 expression did not directly predict OS.\\nHigh SPHK1 expression was also associated with larger tumor size (p < 0.001), deeper invasion depth (p < 0.001), presence of lymph node metastasis (p = 0.029), higher FIGO stage (p = 0.029), presence of lymphovascular invasion (p = 0.045), and higher preoperative squamous cell carcinoma (SCC) antigen level (p = 0.009).\", \"disease\"=>{\"id\"=>159, \"name\"=>\"Cervical Cancer\", \"display_name\"=>\"Cervical Cancer\", \"doid\"=>\"4362\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4362\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>560, \"name\"=>\"Sphingosine kinase 1 is a reliable prognostic factor and a novel therapeutic target for uterine cervical cancer.\", \"citation\"=>\"Kim et al., 2015, Oncotarget\", \"pubmed_id\"=>\"26311741\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26311741\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4694949\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>9, \"day\"=>29}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>345, \"phenotypes\"=>[]}]}, {\"id\"=>346, \"entrez_name\"=>\"CASP8\", \"entrez_id\"=>841, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>761, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>202122759, \"stop\"=>202152434, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000358485.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/761/summary/variants/346/summary#variant\", \"evidence_items\"=>[{\"id\"=>831, \"name\"=>\"EID831\", \"description\"=>\"In-vitro and in-vivo sensitivity to DR5 antibody conatumumab was correlated with expression of caspase-8.\", \"disease\"=>{\"id\"=>135, \"name\"=>\"Bone Ewing's Sarcoma\", \"display_name\"=>\"Bone Ewing's Sarcoma\", \"doid\"=>\"3368\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3368\"}, \"drugs\"=>[{\"id\"=>158, \"name\"=>\"Conatumumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>561, \"name\"=>\"Caspase-8 expression is predictive of tumour response to death receptor 5 agonist antibody in Ewing's sarcoma.\", \"citation\"=>\"Kang et al., 2015, Br. J. Cancer\", \"pubmed_id\"=>\"26291055\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26291055\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4578089\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>9, \"day\"=>15}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>346, \"phenotypes\"=>[]}]}, {\"id\"=>347, \"entrez_name\"=>\"TYMS\", \"entrez_id\"=>7298, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>5971, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>40.0, \"coordinates\"=>{\"chromosome\"=>\"18\", \"start\"=>657604, \"stop\"=>673578, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000323274.10\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5971/summary/variants/347/summary#variant\", \"evidence_items\"=>[{\"id\"=>929, \"name\"=>\"EID929\", \"description\"=>\"In 125 gastric tumors, plasma and tissue mRNA levels of TYMS were lower in the raltitrexed sensitive group compared to the resistant group (p = 0.007 and 0.013, respectively).\", \"disease\"=>{\"id\"=>147, \"name\"=>\"Gastric Adenocarcinoma\", \"display_name\"=>\"Gastric Adenocarcinoma\", \"doid\"=>\"3717\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3717\"}, \"drugs\"=>[{\"id\"=>176, \"name\"=>\"Raltitrexed\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>635, \"name\"=>\"Thymidylate synthase mRNA levels in plasma and tumor as potential predictive biomarkers for raltitrexed sensitivity in gastric cancer.\", \"citation\"=>\"Shen et al., 2012, Int. J. Cancer\", \"pubmed_id\"=>\"22422354\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22422354\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>9, \"day\"=>15}, \"journal\"=>\"Int. J. Cancer\", \"full_journal_title\"=>\"International journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>347, \"phenotypes\"=>[]}, {\"id\"=>922, \"name\"=>\"EID922\", \"description\"=>\"mRNA levels of TYMS, DHFR and GARFT were assessed (rt-PCR) in 50 patient samples of NSCLC. TYMS mRNA levels were lower in responders compared to non-responders to pemetrexed (1.671 ± 0.844 versus 5.978 ± 1.895, p=0.0142).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>156, \"name\"=>\"Pemetrexed\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>628, \"name\"=>\"Association between expression of thymidylate synthase, dihydrofolate reductase, and glycinamide ribonucleotide formyltransferase and efficacy of pemetrexed in advanced non-small cell lung cancer.\", \"citation\"=>\"Shimizu et al., 2012, Anticancer Res.\", \"pubmed_id\"=>\"23060591\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23060591\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>10}, \"journal\"=>\"Anticancer Res.\", \"full_journal_title\"=>\"Anticancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>347, \"phenotypes\"=>[]}, {\"id\"=>832, \"name\"=>\"EID832\", \"description\"=>\"mRNA expression of thymidylate synthase (TYMS) in 62 NSCLC patients correlated with response rate to pemetrexed mono- or combination therapy. Response rate with low TYMS expression was 0.29 compared with 0.03 in patients with overexpression (P = 0.025). Patients with low expression exhibited a benefit in time to progression (average TTP = 56 vs. 23 months, P = 0.001) and in overall survival (average OS = 60 vs. 25 months, P = 0.002).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>156, \"name\"=>\"Pemetrexed\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>562, \"name\"=>\"Thymidylate synthase expression as a predictive biomarker of pemetrexed sensitivity in advanced non-small cell lung cancer.\", \"citation\"=>\"Chamizo et al., 2015, BMC Pulm Med\", \"pubmed_id\"=>\"26502926\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26502926\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4623912\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"BMC Pulm Med\", \"full_journal_title\"=>\"BMC pulmonary medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>347, \"phenotypes\"=>[]}]}, {\"id\"=>348, \"entrez_name\"=>\"PBK\", \"entrez_id\"=>55872, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>13276, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"8\", \"start\"=>27667137, \"stop\"=>27695612, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000301905.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/13276/summary/variants/348/summary#variant\", \"evidence_items\"=>[{\"id\"=>835, \"name\"=>\"EID835\", \"description\"=>\"TOPK (PBK) silencing sensitized lung cancer cells to gefitinib in preclinical xenograft models. Over-expression was also associated with resistance in various lung cancer cell lines.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>14, \"name\"=>\"Gefitinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>563, \"name\"=>\"TOPK promotes lung cancer resistance to EGFR tyrosine kinase inhibitors by phosphorylating and activating c-Jun.\", \"citation\"=>\"Li et al., 2016, Oncotarget\", \"pubmed_id\"=>\"26745678\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26745678\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4872746\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>2, \"day\"=>9}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>348, \"phenotypes\"=>[]}]}, {\"id\"=>349, \"entrez_name\"=>\"SMARCA4\", \"entrez_id\"=>6597, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>78, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>30.0, \"coordinates\"=>{\"chromosome\"=>\"19\", \"start\"=>11071598, \"stop\"=>11172958, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000358026.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/78/summary/variants/349/summary#variant\", \"evidence_items\"=>[{\"id\"=>837, \"name\"=>\"EID837\", \"description\"=>\"SMARCA4 expression was determined with gene expression microarray in 133 patients. Improved five-year disease-specific survival (DSS) was noted only in patients with low SMARCA4 expression when treated with adjuvant cisplatin/vinorelbine (n=36) (HR=0.1, 95% CI: 0.0-0.5, P=0.002 [low]; HR 1.0, 95% CI: 0.5-2.3, P=0.92 [high]) compared to patients in the observation arm (N=30).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>96, \"name\"=>\"Vinorelbine\", \"pubchem_id\"=>nil}, {\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>565, \"name\"=>\"SMARCA4/BRG1 Is a Novel Prognostic Biomarker Predictive of Cisplatin-Based Chemotherapy Outcomes in Resected Non-Small Cell Lung Cancer.\", \"citation\"=>\"Bell et al., 2016, Clin. Cancer Res.\", \"pubmed_id\"=>\"26671993\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26671993\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4867280\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>5, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>349, \"phenotypes\"=>[]}, {\"id\"=>838, \"name\"=>\"EID838\", \"description\"=>\"SMARCA4 expression was determined with gene expression microarray in 133 patients. Reduced overall survival was observed in patients with low SMARCA4 expression compared to intermediate (P<0\\n.001) or high expression (P=0.009). In multivariate analysis, compared to low, high SMARCA4 expression predicted a decrease in risk of death (HR=0.6, 95% CI: 0.4-0.8, P=0.002).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>565, \"name\"=>\"SMARCA4/BRG1 Is a Novel Prognostic Biomarker Predictive of Cisplatin-Based Chemotherapy Outcomes in Resected Non-Small Cell Lung Cancer.\", \"citation\"=>\"Bell et al., 2016, Clin. Cancer Res.\", \"pubmed_id\"=>\"26671993\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26671993\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4867280\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>5, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>349, \"phenotypes\"=>[]}]}, {\"id\"=>354, \"entrez_name\"=>\"EGFR\", \"entrez_id\"=>1956, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>19, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>80.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>55086794, \"stop\"=>55279321, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000275493.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/19/summary/variants/354/summary#variant\", \"evidence_items\"=>[{\"id\"=>4877, \"name\"=>\"EID4877\", \"description\"=>\"Double blind Phase II clinical trial found an objective response rate of 18.4% and 19% for two groups of patients treated with 250mg/d and 500mg/d respectively. The disease control rate was found to be 54.4% and 51.4% for the two groups.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>14, \"name\"=>\"Gefitinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>nil, \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2170, \"name\"=>\"Multi-institutional randomized phase II trial of gefitinib for previously treated patients with advanced non-small-cell lung cancer (The IDEAL 1 Trial) [corrected].\", \"citation\"=>\"Fukuoka et al., 2003, J. Clin. Oncol.\", \"pubmed_id\"=>\"12748244\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/12748244\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2003, \"month\"=>6, \"day\"=>15}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>354, \"phenotypes\"=>[]}, {\"id\"=>4213, \"name\"=>\"EID4213\", \"description\"=>\"A phase 3 study of 889 advanced non-small cell lung cancer patients (with non-progressive disease while on standard chemotherapy treatment), reported that patients harboring EGFR mutation (exon 19 deletion and L858R mutation; n= 493, 46% lung adenocarcinoma, 44% lung squamous cell carcinoma) who were treated with erlotinib had an improved progression free survival (44.6 weeks vs. 13 weeks, HR: 0.10, 95% CI 0.04-0.25, P<0.001) as compared to patients who received placebo.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>15, \"name\"=>\"Erlotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1938, \"name\"=>\"Prospective molecular marker analyses of EGFR and KRAS from a randomized, placebo-controlled study of erlotinib maintenance therapy in advanced non-small-cell lung cancer.\", \"citation\"=>\"Brugger et al., 2011, J. Clin. Oncol.\", \"pubmed_id\"=>\"21969500\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21969500\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>11, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>354, \"phenotypes\"=>[]}, {\"id\"=>1709, \"name\"=>\"EID1709\", \"description\"=>\"57 patients with chemorefractory colorectal cancer expressing EGFR (IHC staining) were treated with single-agent cetuximab in this phase 2 study. Five patients (9%; 95% CI, 3% to 19%) achieved a partial response. Twenty-one additional patients had stable disease or minor responses. The median survival in these previously treated patients with chemotherapy-refractory colorectal cancer was 6.4 months.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1152, \"name\"=>\"Phase II trial of cetuximab in patients with refractory colorectal cancer that expresses the epidermal growth factor receptor.\", \"citation\"=>\"Saltz et al., 2004, J. Clin. Oncol.\", \"pubmed_id\"=>\"14993230\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/14993230\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2004, \"month\"=>4, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>354, \"phenotypes\"=>[]}, {\"id\"=>1708, \"name\"=>\"EID1708\", \"description\"=>\"463 patients with 1% or more EGFR tumor cell membrane staining, with metastatic CRC progressive after standard chemotherapy were randomized to panitumumab plus best supportive care (BSC, n = 231) or BSC alone (n = 232). Panitumumab significantly prolonged PFS (hazard ratio [HR], 0.54; 95% CI, 0.44 to 0.66, [P < .0001]). Objective response rates favored panitumumab over BSC; after a 12-month minimum follow-up, response rates were 10% for panitumumab and 0% for BSC (P < .0001). No difference was observed in OS (HR, 1.00; 95% CI, 0.82 to 1.22), but 76% of BSC patients entered the cross-over study.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>28, \"name\"=>\"Panitumumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1151, \"name\"=>\"Open-label phase III trial of panitumumab plus best supportive care compared with best supportive care alone in patients with chemotherapy-refractory metastatic colorectal cancer.\", \"citation\"=>\"Van Cutsem et al., 2007, J. Clin. Oncol.\", \"pubmed_id\"=>\"17470858\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17470858\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>5, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>354, \"phenotypes\"=>[]}, {\"id\"=>1572, \"name\"=>\"EID1572\", \"description\"=>\"Patients with refractory colorectal cancer expressing immunohistochemically detectable EGFR were randomly assigned to receive treatment with cetuximab (n=287) or receive supportive care alone (n=285). Patients receiving cetuximab had increased overall survival (HR=0.77, 95%CI 0.64-0.92; p=0.005), increased progression-free survival (HR=0.68, 95%CI 0.57-0.80; p<0.001), and improved quality of life at 4 months measured by survey on physical deterioration (-5.9 vs -12.5; p=0.03) and global health status (-3.6 vs. -15.2; p<0.001).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1038, \"name\"=>\"Cetuximab for the treatment of colorectal cancer.\", \"citation\"=>\"Jonker et al., 2007, N. Engl. J. Med.\", \"pubmed_id\"=>\"18003960\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18003960\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>11, \"day\"=>15}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>354, \"phenotypes\"=>[]}, {\"id\"=>1984, \"name\"=>\"EID1984\", \"description\"=>\"18 advanced progressing chordoma patients entered this study (median age: 61 years; disease extent: metastatic 72% and locally advanced 28%). Epidermal growth factor receptor (EGFR) expression and activation were evaluated by immunohistochemistry and/or phospho-arrays, real-time polimerase chain reaction, fluorescence immunostaining. Six (33.3%) patients had PR and 7 (38.9%) SD, as their best Choi responses, corresponding to RECIST SD in all cases. Median PFS by Choi was 6 [interquartile (IQ) range 3-8] months. Median PFS by RECIST was 8 (IQ range 4-12) months, with a 22% CBR. This phase II study showed a modest antitumor activity of lapatinib in chordoma\", \"disease\"=>{\"id\"=>747, \"name\"=>\"Chordoma\", \"display_name\"=>\"Chordoma\", \"doid\"=>\"3302\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3302\"}, \"drugs\"=>[{\"id\"=>45, \"name\"=>\"Lapatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1394, \"name\"=>\"Phase II study on lapatinib in advanced EGFR-positive chordoma.\", \"citation\"=>\"Stacchiotti et al., 2013, Ann. Oncol.\", \"pubmed_id\"=>\"23559153\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23559153\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>7}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>354, \"phenotypes\"=>[]}, {\"id\"=>849, \"name\"=>\"EID849\", \"description\"=>\"47 HNSCC patients treated with cetuximab/docetaxel were retrospectively analyzed for EGFRvIII, EGFR and AREG expression with IHC. 73 % of cases expressed high level of EGFRvIII. EGFR expression had no impact on PFS or OS.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}, {\"id\"=>73, \"name\"=>\"Docetaxel\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>568, \"name\"=>\"Expression of amphiregulin and EGFRvIII affect outcome of patients with squamous cell carcinoma of the head and neck receiving cetuximab-docetaxel treatment.\", \"citation\"=>\"Tinhofer et al., 2011, Clin. Cancer Res.\", \"pubmed_id\"=>\"21653686\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21653686\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>8, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>354, \"phenotypes\"=>[]}]}, {\"id\"=>355, \"entrez_name\"=>\"CDKN1A\", \"entrez_id\"=>1026, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>913, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>36646487, \"stop\"=>36655108, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000405375.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/913/summary/variants/355/summary#variant\", \"evidence_items\"=>[{\"id\"=>852, \"name\"=>\"EID852\", \"description\"=>\"112 patients with rectal cancer who received 5-FU based chemotherapy and surgery were retrospectively analyzed for expression of Ki67, TS, BAX, EpCAM, p53, p21, EGFR, CD44, CD133, CD166, HIF1α and ALDH1. High pretreatment p21 (CDKN1A) expression was significantly associated with non-pCR (p = 0.022) and poor disease free survival (median DFS - low vs high p21: 75.8 vs 58.1 months, p = 0.002). Multivariate analysis was also significant (p = 0.001, HR 6.14; 95% CI 2.03, 18.55).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>395, \"name\"=>\"5-fluorouracil\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>570, \"name\"=>\"P21 and CD166 as predictive markers of poor response and outcome after fluorouracil-based chemoradiotherapy for the patients with rectal cancer.\", \"citation\"=>\"Sim et al., 2014, BMC Cancer\", \"pubmed_id\"=>\"24708484\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24708484\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4101833\", \"publication_date\"=>{\"year\"=>2014}, \"journal\"=>\"BMC Cancer\", \"full_journal_title\"=>\"BMC cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>355, \"phenotypes\"=>[]}]}, {\"id\"=>356, \"entrez_name\"=>\"ALCAM\", \"entrez_id\"=>214, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>259, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>105085753, \"stop\"=>105295744, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000306107.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/259/summary/variants/356/summary#variant\", \"evidence_items\"=>[{\"id\"=>853, \"name\"=>\"EID853\", \"description\"=>\"112 patients with rectal cancer who received 5-FU based chemotherapy and surgery were retrospectively analyzed for expression of Ki67, TS, BAX, EpCAM, p53, p21, EGFR, CD44, CD133, CD166, HIF1α and ALDH1. Pretreatment CD166 (ALCAM) expression level was associated with poor disease free survival (p = 0.003; HR 5.61; 95% CI 1.81, 17.35).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>395, \"name\"=>\"5-fluorouracil\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>570, \"name\"=>\"P21 and CD166 as predictive markers of poor response and outcome after fluorouracil-based chemoradiotherapy for the patients with rectal cancer.\", \"citation\"=>\"Sim et al., 2014, BMC Cancer\", \"pubmed_id\"=>\"24708484\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24708484\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4101833\", \"publication_date\"=>{\"year\"=>2014}, \"journal\"=>\"BMC Cancer\", \"full_journal_title\"=>\"BMC cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>356, \"phenotypes\"=>[]}]}, {\"id\"=>357, \"entrez_name\"=>\"STMN1\", \"entrez_id\"=>3925, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>3255, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>26211931, \"stop\"=>26232957, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000426559.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3255/summary/variants/357/summary#variant\", \"evidence_items\"=>[{\"id\"=>855, \"name\"=>\"EID855\", \"description\"=>\"33 patients were retrospectively analyzed for Stathmin expression by IHC. High protein expression was correlated with poor response to paclitaxel but not other chemotherapy. The authors supported this conclusion with pre-clinical studies that showed that knock-down of stathmin improved sensitivity to paclitaxel in endometrial carcinoma cell lines with both naturally higher and lower sensitivity to paclitaxel.\", \"disease\"=>{\"id\"=>123, \"name\"=>\"Endometrial Carcinoma\", \"display_name\"=>\"Endometrial Carcinoma\", \"doid\"=>\"2871\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2871\"}, \"drugs\"=>[{\"id\"=>81, \"name\"=>\"Paclitaxel\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>572, \"name\"=>\"Stathmin protein level, a potential predictive marker for taxane treatment response in endometrial cancer.\", \"citation\"=>\"Werner et al., 2014, PLoS ONE\", \"pubmed_id\"=>\"24587245\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24587245\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3934991\", \"publication_date\"=>{\"year\"=>2014}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>357, \"phenotypes\"=>[]}]}, {\"id\"=>359, \"entrez_name\"=>\"AR\", \"entrez_id\"=>367, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>67, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"X\", \"start\"=>66764465, \"stop\"=>66950461, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000374690.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/67/summary/variants/359/summary#variant\", \"evidence_items\"=>[{\"id\"=>859, \"name\"=>\"EID859\", \"description\"=>\"In-vitro and xenograft studies of response of breast cancer cell lines to PI3K/mTOR inhibitor NVP-BEZ235 showed that sensitivity correlated with expression of the androgen receptor (AR). Reintroduction of AR expression resensitized nonresponsive AR-negative cells to NVP-BEZ235. A dihydrotestosterone (DHT) and NVP-BEZ235 combination achieved tumor regression and complete response in AR/ER expressing tumors (xenografts).\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>343, \"name\"=>\"BEZ235 (NVP-BEZ235, Dactolisib)\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>575, \"name\"=>\"Activation of AR sensitizes breast carcinomas to NVP-BEZ235's therapeutic effect mediated by PTEN and KLLN upregulation.\", \"citation\"=>\"Wang et al., 2014, Mol. Cancer Ther.\", \"pubmed_id\"=>\"24356815\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24356815\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>2}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>359, \"phenotypes\"=>[]}]}, {\"id\"=>360, \"entrez_name\"=>\"CDKN1B\", \"entrez_id\"=>1027, \"name\"=>\"CYTOPLASMIC MISLOCALIZATION\", \"description\"=>\"\", \"gene_id\"=>914, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"12\", \"start\"=>12870058, \"stop\"=>12875305, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000228872.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/914/summary/variants/360/summary#variant\", \"evidence_items\"=>[{\"id\"=>860, \"name\"=>\"EID860\", \"description\"=>\"Cytoplasmic localization of p27 (CDKN1B) leads to lapatinib resistance in HER2+ breast cancer cell lines. p27 knockout resensitiuzed cells to lapatinib.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>45, \"name\"=>\"Lapatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>576, \"name\"=>\"Mislocalization of p27 to the cytoplasm of breast cancer cells confers resistance to anti-HER2 targeted therapy.\", \"citation\"=>\"Zhao et al., 2014, Oncotarget\", \"pubmed_id\"=>\"25587029\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25587029\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4350358\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>12, \"day\"=>30}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>360, \"phenotypes\"=>[]}]}, {\"id\"=>361, \"entrez_name\"=>\"ERBB3\", \"entrez_id\"=>2065, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>1733, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>6.0, \"coordinates\"=>{\"chromosome\"=>\"12\", \"start\"=>56473645, \"stop\"=>56497289, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000267101.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1733/summary/variants/361/summary#variant\", \"evidence_items\"=>[{\"id\"=>3051, \"name\"=>\"EID3051\", \"description\"=>\"A preclinical study demonstrating that in a subgroup of BRAF/NRAS wildtype/wildtype showing high levels of phospho-ErbB3, knockdown of NRG1 reduced viability and decreased activation of ErbB3, ErbB2 and AKT.\\r\\nIn addition, pertuzumab-mediated inhibition of ErbB2 heterodimerization decreased AKT phosphorylation, cell growth in vitro, and xenograft growth in vivo. Pertuzumab also potentiated the effects of MEK inhibitor on WT/WT melanoma growth in vitro and in vivo.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>247, \"name\"=>\"Pertuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1754, \"name\"=>\"ErbB3-ErbB2 Complexes as a Therapeutic Target in a Subset of Wild-type BRAF/NRAS Cutaneous Melanomas.\", \"citation\"=>\"Capparelli et al., 2015, Cancer Res.\", \"pubmed_id\"=>\"26206558\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26206558\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4558382\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>9, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>361, \"phenotypes\"=>[]}, {\"id\"=>862, \"name\"=>\"EID862\", \"description\"=>\"The authors conducted an in vivo efficacy study of AV-203 (ERBB3 inhibitory antibody) using a panel of 32 xenograft models representing 12 human cancer types. To identify biomarkers that can predict response to AV-203, the relationship between tumor growth inhibition (TGI) by AV-203 and the expression levels of ERBB3 and NRG1 were evaluated in these tumor models. ERBB3 expression was not predictive of response.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>225, \"name\"=>\"AV-203\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>577, \"name\"=>\"Neuregulin 1 expression is a predictive biomarker for response to AV-203, an ERBB3 inhibitory antibody, in human tumor models.\", \"citation\"=>\"Meetze et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"25542901\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25542901\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>3, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>361, \"phenotypes\"=>[]}, {\"id\"=>865, \"name\"=>\"EID865\", \"description\"=>\"In mice xenografts expressing ERBB3 (HER3) the combination of ERBB2 (HER2) and ERBB3 inhibitors pertuzumab and 9F7-F11enhanced tumor inhibition and the median survival time. Sensitivity to single-agent pertuzumab might also be increased through ERBB3 expression.\", \"disease\"=>{\"id\"=>141, \"name\"=>\"Pancreatic Ductal Adenocarcinoma\", \"display_name\"=>\"Pancreatic Ductal Adenocarcinoma\", \"doid\"=>\"3498\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3498\"}, \"drugs\"=>[{\"id\"=>247, \"name\"=>\"Pertuzumab\", \"pubchem_id\"=>nil}, {\"id\"=>248, \"name\"=>\"9F7-F11\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>580, \"name\"=>\"HER3 as biomarker and therapeutic target in pancreatic cancer: new insights in pertuzumab therapy in preclinical models.\", \"citation\"=>\"Thomas et al., 2014, Oncotarget\", \"pubmed_id\"=>\"25216528\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25216528\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4196190\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>8, \"day\"=>30}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>361, \"phenotypes\"=>[]}]}, {\"id\"=>363, \"entrez_name\"=>\"UGT1A\", \"entrez_id\"=>7361, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>6023, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>234526291, \"stop\"=>234681956, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000373450.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/6023/summary/variants/363/summary#variant\", \"evidence_items\"=>[{\"id\"=>866, \"name\"=>\"EID866\", \"description\"=>\"Colorectal cancer cell lines were evaluated for response to the HSP90 inhibitor ganetespib.  Correlating cell line-specific IC50s with the corresponding gene expression patterns revealed a possible relationship to UGT1A expression. High UGT1A expression was correlated with ganetespib resistance. Knockdown of UGT1A sensitized HT29 cells to HSP90 inhibition. HCT116 and SW480 cell lines became resistant through UGT1A overexpression.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>165, \"name\"=>\"Ganetespib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>581, \"name\"=>\"UDP glucuronosyltransferase 1A expression levels determine the response of colorectal cancer cells to the heat shock protein 90 inhibitor ganetespib.\", \"citation\"=>\"Landmann et al., 2014, Cell Death Dis\", \"pubmed_id\"=>\"25210794\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25210794\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4540199\", \"publication_date\"=>{\"year\"=>2014}, \"journal\"=>\"Cell Death Dis\", \"full_journal_title\"=>\"Cell death & disease\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>363, \"phenotypes\"=>[]}]}, {\"id\"=>364, \"entrez_name\"=>\"ERBB4\", \"entrez_id\"=>2066, \"name\"=>\"NUCLEAR TRANSLOCATION\", \"description\"=>\"\", \"gene_id\"=>1734, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>212240446, \"stop\"=>213403565, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000342788.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1734/summary/variants/364/summary#variant\", \"evidence_items\"=>[{\"id\"=>867, \"name\"=>\"EID867\", \"description\"=>\"HER4 nuclear translocation was identified as a mediator of trastuzumab resistance in HER2 positive breast cancer cell lines. Increased nuclear staining was also identified in xenograft and patient samples after trastuzumab treatment.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>582, \"name\"=>\"Nuclear HER4 mediates acquired resistance to trastuzumab and is associated with poor outcome in HER2 positive breast cancer.\", \"citation\"=>\"Mohd Nafi et al., 2014, Oncotarget\", \"pubmed_id\"=>\"25153719\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25153719\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4171603\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>8, \"day\"=>15}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>364, \"phenotypes\"=>[]}]}, {\"id\"=>365, \"entrez_name\"=>\"ERBB4\", \"entrez_id\"=>2066, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>1734, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>212240446, \"stop\"=>213403565, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000342788.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1734/summary/variants/365/summary#variant\", \"evidence_items\"=>[{\"id\"=>868, \"name\"=>\"EID868\", \"description\"=>\"siRNA knockdown of HER4 or inhibition with neratinib enhanced trastuzumab response in breast cancer cell lines.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>582, \"name\"=>\"Nuclear HER4 mediates acquired resistance to trastuzumab and is associated with poor outcome in HER2 positive breast cancer.\", \"citation\"=>\"Mohd Nafi et al., 2014, Oncotarget\", \"pubmed_id\"=>\"25153719\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25153719\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4171603\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>8, \"day\"=>15}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>365, \"phenotypes\"=>[]}, {\"id\"=>1961, \"name\"=>\"EID1961\", \"description\"=>\"Preventing HER4 cleavage by a γ-secretase inhibitor and inhibiting HER4 tyrosine kinase activity by neratinib decreased trastuzumab-induced HER4 nuclear translocation and enhanced trastuzumab response. There was also increased nuclear HER4 staining in the tumours from BT474 xenograft mice and human patients treated with trastuzumab. Furthermore, nuclear HER4 predicted poor clinical response to trastuzumab monotherapy in patients and was shown to be an independent poor prognostic factor in HER2 positive breast cancer.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>nil, \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>582, \"name\"=>\"Nuclear HER4 mediates acquired resistance to trastuzumab and is associated with poor outcome in HER2 positive breast cancer.\", \"citation\"=>\"Mohd Nafi et al., 2014, Oncotarget\", \"pubmed_id\"=>\"25153719\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25153719\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4171603\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>8, \"day\"=>15}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>365, \"phenotypes\"=>[]}, {\"id\"=>869, \"name\"=>\"EID869\", \"description\"=>\"ERBB4 protein expression was identified in breast cancer cell lines with acquired lapatinib resistance. Genetic ablation of ERBB4 (not ERBB1-3) led to apoptosis in trastuzumab- and lapatinib resistant cells, probably through a decrease in the PI3K/Akt pathway.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>45, \"name\"=>\"Lapatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>583, \"name\"=>\"Receptor tyrosine kinase ERBB4 mediates acquired resistance to ERBB2 inhibitors in breast cancer cells.\", \"citation\"=>\"Canfield et al., 2015, Cell Cycle\", \"pubmed_id\"=>\"25590338\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25590338\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4614407\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"Cell Cycle\", \"full_journal_title\"=>\"Cell cycle (Georgetown, Tex.)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>365, \"phenotypes\"=>[]}]}, {\"id\"=>366, \"entrez_name\"=>\"SLFN11\", \"entrez_id\"=>91607, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>16872, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"17\", \"start\"=>33677324, \"stop\"=>33700639, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000394566.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/16872/summary/variants/366/summary#variant\", \"evidence_items\"=>[{\"id\"=>5884, \"name\"=>\"EID5884\", \"description\"=>\"SLFN11 is an ETS transcription factor response gene and a predictive marker for therapeutic response to topoisomerase I inhibitors and temozolomide-PARP inhibitor combinations in ETS-activated cancers.\", \"disease\"=>{\"id\"=>50, \"name\"=>\"Ewing Sarcoma\", \"display_name\"=>\"Ewing Sarcoma\", \"doid\"=>\"3369\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3369\"}, \"drugs\"=>[{\"id\"=>11, \"name\"=>\"Temozolomide\", \"pubchem_id\"=>\"\"}, {\"id\"=>547, \"name\"=>\"Niraparib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2378, \"name\"=>\"SLFN11 Is a Transcriptional Target of EWS-FLI1 and a Determinant of Drug Response in Ewing Sarcoma.\", \"citation\"=>\"Tang et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"25779942\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25779942\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4573368\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>9, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>366, \"phenotypes\"=>[]}, {\"id\"=>5883, \"name\"=>\"EID5883\", \"description\"=>\"SLFN11 is a relevant predictive biomarker of sensitivity to PARP inhibitor monotherapy in SCLC as well as combinatorial therapy with TMZ.\", \"disease\"=>{\"id\"=>174, \"name\"=>\"Lung Small Cell Carcinoma\", \"display_name\"=>\"Lung Small Cell Carcinoma\", \"doid\"=>\"5409\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5409\"}, \"drugs\"=>[{\"id\"=>11, \"name\"=>\"Temozolomide\", \"pubchem_id\"=>\"\"}, {\"id\"=>523, \"name\"=>\"Talazoparib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>nil, \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2377, \"name\"=>\"PARP Inhibitor Activity Correlates with SLFN11 Expression and Demonstrates Synergy with Temozolomide in Small Cell Lung Cancer.\", \"citation\"=>\"Lok et al., 2017, Clin. Cancer Res.\", \"pubmed_id\"=>\"27440269\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27440269\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5241177\", \"publication_date\"=>{\"year\"=>2017, \"month\"=>1, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>366, \"phenotypes\"=>[]}, {\"id\"=>870, \"name\"=>\"EID870\", \"description\"=>\"In this study, SLFN11 expression was associated with sensitivity to SN-38 (an active metabolite of topoisomerase inhibitor irinotecan).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>166, \"name\"=>\"SN-38\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>584, \"name\"=>\"Schlafen-11 sensitizes colorectal carcinoma cells to irinotecan.\", \"citation\"=>\"Tian et al., 2014, Anticancer Drugs\", \"pubmed_id\"=>\"25089570\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25089570\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>11}, \"journal\"=>\"Anticancer Drugs\", \"full_journal_title\"=>\"Anti-cancer drugs\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>366, \"phenotypes\"=>[]}]}, {\"id\"=>368, \"entrez_name\"=>\"BIRC5\", \"entrez_id\"=>332, \"name\"=>\"NUCLEAR EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>355, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"17\", \"start\"=>76210267, \"stop\"=>76221717, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000301633.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/355/summary/variants/368/summary#variant\", \"evidence_items\"=>[{\"id\"=>874, \"name\"=>\"EID874\", \"description\"=>\"Patients with nuclear survivin (BIRC5) expression had significantly better responses to taxane-platinum chemotherapy than those with low tumor N-survivin expression (P < 0.001) in a retrospective analysis of 48 patients with non-small cell lung cancer.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>103, \"name\"=>\"PLATINUM\", \"pubchem_id\"=>nil}, {\"id\"=>167, \"name\"=>\"Taxane\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>588, \"name\"=>\"Nuclear survivin expression: a prognostic factor for the response to taxane-platinum chemotherapy in patients with advanced non-small cell lung cancer.\", \"citation\"=>\"Wu et al., 2014, Med. Oncol.\", \"pubmed_id\"=>\"24961465\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24961465\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>8}, \"journal\"=>\"Med. Oncol.\", \"full_journal_title\"=>\"Medical oncology (Northwood, London, England)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>368, \"phenotypes\"=>[]}]}, {\"id\"=>372, \"entrez_name\"=>\"CDKN1B\", \"entrez_id\"=>1027, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>914, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"12\", \"start\"=>12870058, \"stop\"=>12875305, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000228872.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/914/summary/variants/372/summary#variant\", \"evidence_items\"=>[{\"id\"=>889, \"name\"=>\"EID889\", \"description\"=>\"41 HNSSC were analyzed by IHC for expression of EGFR, p53, CCND1, p16, p21, p27 (CDKN2B), p-AKT, HIF-1a, Caspase 3 and BCL2. p27 was the only protein that served as a predictive biomarker of response after multivariate analysis (p=0.025; n=41 patients).\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}, {\"id\"=>395, \"name\"=>\"5-fluorouracil\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>599, \"name\"=>\"p27 and BCL2 expression predicts response to chemotherapy in head and neck squamous cell carcinomas.\", \"citation\"=>\"Moreno-Galindo et al., 2014, Oral Oncol.\", \"pubmed_id\"=>\"24239278\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24239278\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>2}, \"journal\"=>\"Oral Oncol.\", \"full_journal_title\"=>\"Oral oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>372, \"phenotypes\"=>[]}]}, {\"id\"=>373, \"entrez_name\"=>\"AGR2\", \"entrez_id\"=>10551, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>8407, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>16831435, \"stop\"=>16844704, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000419304.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/8407/summary/variants/373/summary#variant\", \"evidence_items\"=>[{\"id\"=>891, \"name\"=>\"EID891\", \"description\"=>\"mRNA expression of AGR2 in 61 patients with tamoxifen treated postmenopausal breast cancer was determined by qRT-PCR and was inversely associated with treatment response and progression free survival (p= 0.0011, p = 0.0366 respectively). AGR2 protein expression as assessed by IHC was not significantly associated with treatment outcome.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>53, \"name\"=>\"Tamoxifen\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>601, \"name\"=>\"AGR2 predicts tamoxifen resistance in postmenopausal breast cancer patients.\", \"citation\"=>\"Hrstka et al., 2013, Dis. Markers\", \"pubmed_id\"=>\"24167368\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24167368\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3776368\", \"publication_date\"=>{\"year\"=>2013}, \"journal\"=>\"Dis. Markers\", \"full_journal_title\"=>\"Disease markers\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>373, \"phenotypes\"=>[]}]}, {\"id\"=>376, \"entrez_name\"=>\"ZEB1\", \"entrez_id\"=>6935, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>5649, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>31608101, \"stop\"=>31818741, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000361642.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5649/summary/variants/376/summary#variant\", \"evidence_items\"=>[{\"id\"=>900, \"name\"=>\"EID900\", \"description\"=>\"Downregulation of ZEB1 expression in mantle cell lymphoma cell lines and mouse xenografts led to enhanced sensitivity towards treatment with doxorubicin, cytarabin and gemcitabine.\\nthe combination of salinomycin (downregulating ZEB1) and doxorubicin was syngergistic.\", \"disease\"=>{\"id\"=>38, \"name\"=>\"Mantle Cell Lymphoma\", \"display_name\"=>\"Mantle Cell Lymphoma\", \"doid\"=>\"0050746\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050746\"}, \"drugs\"=>[{\"id\"=>68, \"name\"=>\"Doxorubicin\", \"pubchem_id\"=>nil}, {\"id\"=>174, \"name\"=>\"Salinomycin\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>607, \"name\"=>\"The EMT activator ZEB1 promotes tumor growth and determines differential response to chemotherapy in mantle cell lymphoma.\", \"citation\"=>\"Sánchez-Tilló et al., 2014, Cell Death Differ.\", \"pubmed_id\"=>\"24013721\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24013721\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3890947\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>2}, \"journal\"=>\"Cell Death Differ.\", \"full_journal_title\"=>\"Cell death and differentiation\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>376, \"phenotypes\"=>[]}, {\"id\"=>899, \"name\"=>\"EID899\", \"description\"=>\"Downregulation of ZEB1 expression in mantle cell lymphoma cell lines and mouse xenografts led to enhanced sensitivity towards treatment with doxorubicin, cytarabin and gemcitabine.\", \"disease\"=>{\"id\"=>38, \"name\"=>\"Mantle Cell Lymphoma\", \"display_name\"=>\"Mantle Cell Lymphoma\", \"doid\"=>\"0050746\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050746\"}, \"drugs\"=>[{\"id\"=>68, \"name\"=>\"Doxorubicin\", \"pubchem_id\"=>nil}, {\"id\"=>85, \"name\"=>\"Gemcitabine\", \"pubchem_id\"=>nil}, {\"id\"=>173, \"name\"=>\"Cytarabine\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>607, \"name\"=>\"The EMT activator ZEB1 promotes tumor growth and determines differential response to chemotherapy in mantle cell lymphoma.\", \"citation\"=>\"Sánchez-Tilló et al., 2014, Cell Death Differ.\", \"pubmed_id\"=>\"24013721\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24013721\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3890947\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>2}, \"journal\"=>\"Cell Death Differ.\", \"full_journal_title\"=>\"Cell death and differentiation\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>376, \"phenotypes\"=>[]}]}, {\"id\"=>377, \"entrez_name\"=>\"TIMP1\", \"entrez_id\"=>7076, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>5777, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>12.0, \"coordinates\"=>{\"chromosome\"=>\"X\", \"start\"=>47441712, \"stop\"=>47446188, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000218388.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5777/summary/variants/377/summary#variant\", \"evidence_items\"=>[{\"id\"=>927, \"name\"=>\"EID927\", \"description\"=>\"High expression of TIMP-1 was assessed by IHC in 99 primary breast cancers that had not spread beyond the breast after 6 postoperative courses of paclitaxel-based chemotherapy. High TIMP1 levels were associated with worse 5-year disease free survival (71.1%) than low levels (88.5 %; P = 0.020). High TIMP-1 levels were also associated with worse 5-year overall survival (78.9 %) than low levels (96.7 %; P = 0.004).\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>81, \"name\"=>\"Paclitaxel\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>633, \"name\"=>\"High expression of TIMP-1 in human breast cancer tissues is a predictive of resistance to paclitaxel-based chemotherapy.\", \"citation\"=>\"Zhu et al., 2012, Med. Oncol.\", \"pubmed_id\"=>\"22544540\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22544540\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>12}, \"journal\"=>\"Med. Oncol.\", \"full_journal_title\"=>\"Medical oncology (Northwood, London, England)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>377, \"phenotypes\"=>[]}, {\"id\"=>901, \"name\"=>\"EID901\", \"description\"=>\"TIMP1 overexpression was associated with resistance to fulvestrant in breast cancer cell lines.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>54, \"name\"=>\"Fulvestrant\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>608, \"name\"=>\"TIMP1 overexpression mediates resistance of MCF-7 human breast cancer cells to fulvestrant and down-regulates progesterone receptor expression.\", \"citation\"=>\"Bjerre et al., 2013, Tumour Biol.\", \"pubmed_id\"=>\"23881388\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23881388\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>12}, \"journal\"=>\"Tumour Biol.\", \"full_journal_title\"=>\"Tumour biology : the journal of the International Society for Oncodevelopmental Biology and Medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>377, \"phenotypes\"=>[]}]}, {\"id\"=>378, \"entrez_name\"=>\"PTP4A3\", \"entrez_id\"=>11156, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>8908, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"8\", \"start\"=>142402093, \"stop\"=>142441394, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000521578.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/8908/summary/variants/378/summary#variant\", \"evidence_items\"=>[{\"id\"=>902, \"name\"=>\"EID902\", \"description\"=>\"In-vitro studies linked PRL-3 (PP4A3) expression to hyperactivation and dependence on EGFR signaling (through downregulation of PTP1B, an inhibitory phosphatase of EGFR). Retrospective analysis of 68 patients showed improved clinical response to the EGFR inhibitor cetuximab in tumors expressing PRL-3.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>609, \"name\"=>\"Metastasis-associated PRL-3 induces EGFR activation and addiction in cancer cells.\", \"citation\"=>\"Al-Aidaroos et al., 2013, J. Clin. Invest.\", \"pubmed_id\"=>\"23867504\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23867504\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4011027\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>8}, \"journal\"=>\"J. Clin. Invest.\", \"full_journal_title\"=>\"The Journal of clinical investigation\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>378, \"phenotypes\"=>[]}]}, {\"id\"=>380, \"entrez_name\"=>\"IGF1R\", \"entrez_id\"=>3480, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>2899, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"15\", \"start\"=>99192200, \"stop\"=>99507759, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000268035.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/2899/summary/variants/380/summary#variant\", \"evidence_items\"=>[{\"id\"=>904, \"name\"=>\"EID904\", \"description\"=>\"Expression of IGF1R in NSCLC cell lines (n=17) was associated with resistance to EGFR inhibitor gefitinib, however, knock-in or knock-out of IGF1R did not affect gefitinib sensitivity. IGF1R expression was also higher in patients with disease progression after gefitinib therapy in a clinical cohort (n=98).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>14, \"name\"=>\"Gefitinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>611, \"name\"=>\"Insulin-like growth factor-1 receptor (IGF-1R) as a biomarker for resistance to the tyrosine kinase inhibitor gefitinib in non-small cell lung cancer.\", \"citation\"=>\"Peled et al., 2013, Cell Oncol (Dordr)\", \"pubmed_id\"=>\"23619944\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23619944\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4186686\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>7}, \"journal\"=>\"Cell Oncol (Dordr)\", \"full_journal_title\"=>\"Cellular oncology (Dordrecht)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>380, \"phenotypes\"=>[]}]}, {\"id\"=>381, \"entrez_name\"=>\"EPHB4\", \"entrez_id\"=>2050, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>1721, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>100400187, \"stop\"=>100425121, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000358173.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1721/summary/variants/381/summary#variant\", \"evidence_items\"=>[{\"id\"=>905, \"name\"=>\"EID905\", \"description\"=>\"EPHB4 expression, as assessed by quantitative RT-PCR in 13 colorectal cancer patients treated with bevacizumab, was higher in non-responders (p = 0.048). No difference was observed in a control group without bevacizumab treatment. qRT-PCR results were also shown to correlate with protein expression measured by IHC.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>33, \"name\"=>\"Bevacizumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>612, \"name\"=>\"Gene expression profiling identifies EPHB4 as a potential predictive biomarker in colorectal cancer patients treated with bevacizumab.\", \"citation\"=>\"Guijarro-Muñoz et al., 2013, Med. Oncol.\", \"pubmed_id\"=>\"23579861\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23579861\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013}, \"journal\"=>\"Med. Oncol.\", \"full_journal_title\"=>\"Medical oncology (Northwood, London, England)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>381, \"phenotypes\"=>[]}]}, {\"id\"=>382, \"entrez_name\"=>\"TYMS\", \"entrez_id\"=>7298, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>5971, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"18\", \"start\"=>657604, \"stop\"=>673578, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000323274.10\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5971/summary/variants/382/summary#variant\", \"evidence_items\"=>[{\"id\"=>908, \"name\"=>\"EID908\", \"description\"=>\"Case report of a patient with metastatic prostate cancer and underexpression of TYMS who showed a nonspecified response to pemetrexed for at least 4 months.\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[{\"id\"=>156, \"name\"=>\"Pemetrexed\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>615, \"name\"=>\"A practical approach to aid physician interpretation of clinically actionable predictive biomarker results in a multi-platform tumor profiling service.\", \"citation\"=>\"Russell et al., 2014, Front Pharmacol\", \"pubmed_id\"=>\"24782778\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24782778\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3997001\", \"publication_date\"=>{\"year\"=>2014}, \"journal\"=>\"Front Pharmacol\", \"full_journal_title\"=>\"Frontiers in pharmacology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>382, \"phenotypes\"=>[]}, {\"id\"=>907, \"name\"=>\"EID907\", \"description\"=>\"49 tumor specimens from 268 NSCLC patients treated with pemetrexed were evaluated for expression of TYMS and DHFR by IHC and compared to treatment outcomes.\\nPatients with low TS (≤150) expression had a longer median progression-free survival (PFS) than those with high TS (>150) expression (4.8 vs. 3.4 months; p=0.01).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>156, \"name\"=>\"Pemetrexed\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>614, \"name\"=>\"Thymidylate synthase and dihydrofolate reductase expression in non-small cell lung carcinoma: the association with treatment efficacy of pemetrexed.\", \"citation\"=>\"Chen et al., 2011, Lung Cancer\", \"pubmed_id\"=>\"21367480\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21367480\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>10}, \"journal\"=>\"Lung Cancer\", \"full_journal_title\"=>\"Lung cancer (Amsterdam, Netherlands)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>382, \"phenotypes\"=>[]}]}, {\"id\"=>383, \"entrez_name\"=>\"TOP2A\", \"entrez_id\"=>7153, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>5848, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"17\", \"start\"=>38544768, \"stop\"=>38574202, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000423485.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5848/summary/variants/383/summary#variant\", \"evidence_items\"=>[{\"id\"=>909, \"name\"=>\"EID909\", \"description\"=>\"In a retrospective study, TOP2A expression was assessed by IHC for 108 breast cancer patients (from TAX 303 trial). TOP2A expression was associated with a higher probability of response in doxorubicin treated but not in docetaxel treated breast cancer patients.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>68, \"name\"=>\"Doxorubicin\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>616, \"name\"=>\"Topoisomerase-II alpha expression as a predictive marker in a population of advanced breast cancer patients randomly treated either with single-agent doxorubicin or single-agent docetaxel.\", \"citation\"=>\"Durbecq et al., 2004, Mol. Cancer Ther.\", \"pubmed_id\"=>\"15486187\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15486187\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2004, \"month\"=>10}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>383, \"phenotypes\"=>[]}]}, {\"id\"=>384, \"entrez_name\"=>\"TOP1\", \"entrez_id\"=>7150, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>5845, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>25.0, \"coordinates\"=>{\"chromosome\"=>\"20\", \"start\"=>39657458, \"stop\"=>39753127, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000361337.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5845/summary/variants/384/summary#variant\", \"evidence_items\"=>[{\"id\"=>910, \"name\"=>\"EID910\", \"description\"=>\"Shorter progression free survival was observed in patients with low expression (IHC) of topoisomerase 1 in 16 patients with relapsed or persistent ovarian or peritoneal carcinoma after treatment with topotecan, cyclophosphamide and carboplatin followed by an autologous hematopoietic stem cell transplant.\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>93, \"name\"=>\"Carboplatin\", \"pubchem_id\"=>nil}, {\"id\"=>175, \"name\"=>\"Topotecan\", \"pubchem_id\"=>nil}, {\"id\"=>250, \"name\"=>\"Cyclophosphamide\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>617, \"name\"=>\"Phase I trial of autologous hematopoietic SCT with escalating doses of topotecan combined with CY and carboplatin in patients with relapsed or persistent ovarian or primary peritoneal carcinoma.\", \"citation\"=>\"Litzow et al., 2010, Bone Marrow Transplant.\", \"pubmed_id\"=>\"19648970\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19648970\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2836387\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>3}, \"journal\"=>\"Bone Marrow Transplant.\", \"full_journal_title\"=>\"Bone marrow transplantation\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>384, \"phenotypes\"=>[]}, {\"id\"=>912, \"name\"=>\"EID912\", \"description\"=>\"498 patients were retrospectively analyzed for topoisomerase 1 expression (IHC). In the subgroup of patients treated with irinotecan (38% of the 498), those patients expressing Topoisomerase I had a better overall survival according to multivariate analysis (HR = 0.47, 95% CI 0.23-0.94, p = 0.033).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>101, \"name\"=>\"Irinotecan\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>619, \"name\"=>\"Topoisomerase I but not thymidylate synthase is associated with improved outcome in patients with resected colorectal cancer treated with irinotecan containing adjuvant chemotherapy.\", \"citation\"=>\"Kostopoulos et al., 2009, BMC Cancer\", \"pubmed_id\"=>\"19775480\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19775480\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2759966\", \"publication_date\"=>{\"year\"=>2009}, \"journal\"=>\"BMC Cancer\", \"full_journal_title\"=>\"BMC cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>384, \"phenotypes\"=>[]}]}, {\"id\"=>385, \"entrez_name\"=>\"PRKAA2\", \"entrez_id\"=>5563, \"name\"=>\"T172 PHOSPHORYLATION\", \"description\"=>\"\", \"gene_id\"=>4503, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>0.5, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>57159476, \"stop\"=>57159478, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000371244.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4503/summary/variants/385/summary#variant\", \"evidence_items\"=>[{\"id\"=>913, \"name\"=>\"EID913\", \"description\"=>\"Low levels of baseline p-AMPK (PRKAA2, measured at Thr172) were predictive of efficacy of ACLY inhibition by siRNA in cell lines of various cancer histologies.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>255, \"name\"=>\"ACLY SiRNA\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"E\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>620, \"name\"=>\"Inhibition of ATP citrate lyase induces an anticancer effect via reactive oxygen species: AMPK as a predictive biomarker for therapeutic impact.\", \"citation\"=>\"Migita et al., 2013, Am. J. Pathol.\", \"pubmed_id\"=>\"23506848\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23506848\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>5}, \"journal\"=>\"Am. J. Pathol.\", \"full_journal_title\"=>\"The American journal of pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>385, \"phenotypes\"=>[]}]}, {\"id\"=>386, \"entrez_name\"=>\"HSPA5\", \"entrez_id\"=>3309, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>2778, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>127997132, \"stop\"=>128003609, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000324460.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/2778/summary/variants/386/summary#variant\", \"evidence_items\"=>[{\"id\"=>914, \"name\"=>\"EID914\", \"description\"=>\"Expression of GRP78 (HSPA5) in colorectal cancer patients (n=396, by IHC) showed improved benefit from adjuvant chemotherapy in stage III patients (52% vs. 28%; p = 0.026). In vitro, GRP78 inhibition reduced response to 5-FU.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>395, \"name\"=>\"5-fluorouracil\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>621, \"name\"=>\"The unfolded protein response regulator GRP78 is a novel predictive biomarker in colorectal cancer.\", \"citation\"=>\"Thornton et al., 2013, Int. J. Cancer\", \"pubmed_id\"=>\"23456958\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23456958\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>9, \"day\"=>15}, \"journal\"=>\"Int. J. Cancer\", \"full_journal_title\"=>\"International journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>386, \"phenotypes\"=>[]}]}, {\"id\"=>387, \"entrez_name\"=>\"EIF4EBP1\", \"entrez_id\"=>1978, \"name\"=>\"PHOSPHORYLATION\", \"description\"=>\"\", \"gene_id\"=>1655, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>4.0, \"coordinates\"=>{\"chromosome\"=>\"8\", \"start\"=>37888200, \"stop\"=>37914663, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000338825.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1655/summary/variants/387/summary#variant\", \"evidence_items\"=>[{\"id\"=>918, \"name\"=>\"EID918\", \"description\"=>\"8 gastric cancer cell lines were treated with the MTOR inhibitor everolimus. Cell proliferation was effectively inhibited in 3 cell lines by everolimus. Phosphorylated EIF4EBP1 (T37/46, T70) levels were higher in cell lines and xenografts sensitive to everolimus.\", \"disease\"=>{\"id\"=>147, \"name\"=>\"Gastric Adenocarcinoma\", \"display_name\"=>\"Gastric Adenocarcinoma\", \"doid\"=>\"3717\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3717\"}, \"drugs\"=>[{\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>624, \"name\"=>\"Phosphorylation of 4E-BP1 predicts sensitivity to everolimus in gastric cancer cells.\", \"citation\"=>\"Nishi et al., 2013, Cancer Lett.\", \"pubmed_id\"=>\"23340172\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23340172\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>5, \"day\"=>1}, \"journal\"=>\"Cancer Lett.\", \"full_journal_title\"=>\"Cancer letters\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>387, \"phenotypes\"=>[]}, {\"id\"=>932, \"name\"=>\"EID932\", \"description\"=>\"12 CRC cell lines were evaluated. 40% of these were intrinsically resistant to mTOR inhibition with  BEZ235, PP242 or WYE354. mTOR independent phosphorylation of EIF4EBP1 (T37/46) was associated with resistance to mTOR inhibitors in one analyzed cell line (SW620).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>179, \"name\"=>\"PP242\", \"pubchem_id\"=>nil}, {\"id\"=>256, \"name\"=>\"WYE354\", \"pubchem_id\"=>nil}, {\"id\"=>343, \"name\"=>\"BEZ235 (NVP-BEZ235, Dactolisib)\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>638, \"name\"=>\"mTOR-independent 4E-BP1 phosphorylation is associated with cancer resistance to mTOR kinase inhibitors.\", \"citation\"=>\"Zhang et al., 2012, Cell Cycle\", \"pubmed_id\"=>\"22262166\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22262166\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3315097\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>2, \"day\"=>1}, \"journal\"=>\"Cell Cycle\", \"full_journal_title\"=>\"Cell cycle (Georgetown, Tex.)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>387, \"phenotypes\"=>[]}]}, {\"id\"=>389, \"entrez_name\"=>\"TUBB3\", \"entrez_id\"=>10381, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>8262, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>12.0, \"coordinates\"=>{\"chromosome\"=>\"16\", \"start\"=>89989687, \"stop\"=>90002505, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000315491.7\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/8262/summary/variants/389/summary#variant\", \"evidence_items\"=>[{\"id\"=>920, \"name\"=>\"EID920\", \"description\"=>\"High expression of class 3 beta-tubulin led to decreased effect of paclitaxel on cell motility in HeLa, MCF-7 and CHO cell lines.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>81, \"name\"=>\"Paclitaxel\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>626, \"name\"=>\"Class III β-tubulin counteracts the ability of paclitaxel to inhibit cell migration.\", \"citation\"=>\"Ganguly et al., 2011, Oncotarget\", \"pubmed_id\"=>\"21576762\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21576762\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3248193\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>5}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>389, \"phenotypes\"=>[]}, {\"id\"=>921, \"name\"=>\"EID921\", \"description\"=>\"High expression of betaIII-tubulin, as assessed in 85 breast cancers by IHC, was associated with better pathological response.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>167, \"name\"=>\"Taxane\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>627, \"name\"=>\"High expression of class III β-tubulin predicts good response to neoadjuvant taxane and doxorubicin/cyclophosphamide-based chemotherapy in estrogen receptor-negative breast cancer.\", \"citation\"=>\"Wang et al., 2013, Clin. Breast Cancer\", \"pubmed_id\"=>\"23218766\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23218766\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4039021\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>4}, \"journal\"=>\"Clin. Breast Cancer\", \"full_journal_title\"=>\"Clinical breast cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>389, \"phenotypes\"=>[]}]}, {\"id\"=>390, \"entrez_name\"=>\"EGFR\", \"entrez_id\"=>1956, \"name\"=>\"Y1092 PHOSPHORYLATION\", \"description\"=>\"Although referred to as Y1068 in the literature, based on the recognition peptide described, this tyrosine occurs at Y1039 (ENST00000454757.2 v75) or Y1092 (ENST00000275493.2 v75).\", \"gene_id\"=>19, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>55272951, \"stop\"=>55272953, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000275493.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/19/summary/variants/390/summary#variant\", \"evidence_items\"=>[{\"id\"=>923, \"name\"=>\"EID923\", \"description\"=>\"While EGFR mutation is a strong predictive factor for EGFR inhibition, some patients with EGFR wild-type also respond.  The authors investigated whether phosphorylated EGFR could be a potential predictive biomarker in patients with wild-type EGFR. Phosphorylation of Y1068 (assessed by IHC) was associated with improved progression free survival (PFS) in 205 patients with stage IIb and IV NSCLC treated with EGFR tyrosine kinase inhibitors (gefitinib or erlotinib) compared to phospho-Y1068 negative patients (median PFS 7.0 months vs. 1.2 months, P < 0.001). In this group, 92 (44.9%) were EGFR mutant. In a subgroup of patients with wild-type EGFR, phospho-Y1068 expression positive patients had a significantly prolonged PFS (4.2 months vs.1.2 months P < 0.001) compared with those without phospho-Y1068 expression.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>14, \"name\"=>\"Gefitinib\", \"pubchem_id\"=>nil}, {\"id\"=>15, \"name\"=>\"Erlotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>629, \"name\"=>\"Phosphorylated EGFR expression may predict outcome of EGFR-TKIs therapy for the advanced NSCLC patients with wild-type EGFR.\", \"citation\"=>\"Wang et al., 2012, J. Exp. Clin. Cancer Res.\", \"pubmed_id\"=>\"22901364\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22901364\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3548765\", \"publication_date\"=>{\"year\"=>2012}, \"journal\"=>\"J. Exp. Clin. Cancer Res.\", \"full_journal_title\"=>\"Journal of experimental & clinical cancer research : CR\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>390, \"phenotypes\"=>[]}]}, {\"id\"=>391, \"entrez_name\"=>\"IGF1R\", \"entrez_id\"=>3480, \"name\"=>\"NUCLEAR EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>2899, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"15\", \"start\"=>99192200, \"stop\"=>99507759, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000268035.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/2899/summary/variants/391/summary#variant\", \"evidence_items\"=>[{\"id\"=>924, \"name\"=>\"EID924\", \"description\"=>\"IGF-1R nuclear immunostaining (IHC) was observed in 9 out of 16 samples of patients with different sarcomas (osteosarcoma, ewing sarcoma, liposarcoma, synovialsarcoma, desmoplastic tumor, rhabdomyosarcoma). Exclusive nuclear immunostaining was associated with better progression free survival and overall survival after therapy with IGF1R antibodies (R1507, IMC-A12, SCH 717454 and CP-751.871).\", \"disease\"=>{\"id\"=>105, \"name\"=>\"Sarcoma\", \"display_name\"=>\"Sarcoma\", \"doid\"=>\"1115\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1115\"}, \"drugs\"=>[{\"id\"=>253, \"name\"=>\"IGF1R Monoclonal Antibody\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>630, \"name\"=>\"Insulin-like growth factor type 1 receptor (IGF-1R) exclusive nuclear staining: a predictive biomarker for IGF-1R monoclonal antibody (Ab) therapy in sarcomas.\", \"citation\"=>\"Asmane et al., 2012, Eur. J. Cancer\", \"pubmed_id\"=>\"22682017\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22682017\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>11}, \"journal\"=>\"Eur. J. Cancer\", \"full_journal_title\"=>\"European journal of cancer (Oxford, England : 1990)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>391, \"phenotypes\"=>[]}]}, {\"id\"=>392, \"entrez_name\"=>\"CFLAR\", \"entrez_id\"=>8837, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>7009, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>201980827, \"stop\"=>202041410, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000309955.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/7009/summary/variants/392/summary#variant\", \"evidence_items\"=>[{\"id\"=>925, \"name\"=>\"EID925\", \"description\"=>\"Expression of c-FLIP (CFLAR) in 50 prostate specimens (assessed by IHC) was increased in castrate-resistant prostate cancer. In-vitro, silencing of CFLAR led to sensitization of two cell lines to bicalutamide.\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[{\"id\"=>78, \"name\"=>\"Bicalutamide\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>631, \"name\"=>\"Elevation of c-FLIP in castrate-resistant prostate cancer antagonizes therapeutic response to androgen receptor-targeted therapy.\", \"citation\"=>\"McCourt et al., 2012, Clin. Cancer Res.\", \"pubmed_id\"=>\"22623731\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22623731\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3512078\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>7, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>392, \"phenotypes\"=>[]}]}, {\"id\"=>393, \"entrez_name\"=>\"PROM1\", \"entrez_id\"=>8842, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>7014, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>15969857, \"stop\"=>16085324, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000510224.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/7014/summary/variants/393/summary#variant\", \"evidence_items\"=>[{\"id\"=>926, \"name\"=>\"EID926\", \"description\"=>\"CD133 (PROM1) expression in 39 samples of hepatocellular carcinomas was correlated with JNK activity. Both were more highly expressed in patients not responding to sorafenib. CD133+ cells (sorted) were xenografted into mice, producing larger, more JNK inhibitor responsive tumors than CD133- xenografts.\", \"disease\"=>{\"id\"=>196, \"name\"=>\"Hepatocellular Carcinoma\", \"display_name\"=>\"Hepatocellular Carcinoma\", \"doid\"=>\"684\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:684\"}, \"drugs\"=>[{\"id\"=>6, \"name\"=>\"Sorafenib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>632, \"name\"=>\"Activation of JNK and high expression level of CD133 predict a poor response to sorafenib in hepatocellular carcinoma.\", \"citation\"=>\"Hagiwara et al., 2012, Br. J. Cancer\", \"pubmed_id\"=>\"22596232\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22596232\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3388555\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>6, \"day\"=>5}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>393, \"phenotypes\"=>[]}]}, {\"id\"=>394, \"entrez_name\"=>\"FOXP3\", \"entrez_id\"=>50943, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>11587, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"X\", \"start\"=>49106897, \"stop\"=>49121288, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000376207.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/11587/summary/variants/394/summary#variant\", \"evidence_items\"=>[{\"id\"=>928, \"name\"=>\"EID928\", \"description\"=>\"1,097 patients with breast cancer were evaluated for FOXP3 expression by IHC. FOXP3 expression was found in 405 tumor samples and associated with improved OS in patients treated with anthracycline-based adjuvant chemotherapy (6 cycles of 'FEC100' aka epirubicin + cyclophosphamide + 5-fluorouracil) but not in patients treated with sequential anthracycline->taxane (3 cycles of FEC100 followed by 3 cycles of docetaxel). In-vitro experiments confirmed that FOXP3 expression enhance anthracycline efficacy. The in vitro data consisted of testing the sensitivity of various breast cancer cell lines to anthracyclines after treatment with valproic acid (which increases FOXP3 expression). The authors observed that, while valproic acid has no intrinsic cytotoxic effect on tumor cells, it enhanced the cytotoxic effect of epirubicin specifically in cell lines where FOXP3 expression was increased.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>257, \"name\"=>\"Epirubicin\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>634, \"name\"=>\"FOXP3 expression in cancer cells and anthracyclines efficacy in patients with primary breast cancer treated with adjuvant chemotherapy in the phase III UNICANCER-PACS 01 trial.\", \"citation\"=>\"Ladoire et al., 2012, Ann. Oncol.\", \"pubmed_id\"=>\"22431701\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22431701\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>10}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology / ESMO\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>394, \"phenotypes\"=>[]}]}, {\"id\"=>395, \"entrez_name\"=>\"RPS6\", \"entrez_id\"=>6194, \"name\"=>\"PHOSPHORYLATION\", \"description\"=>\"\", \"gene_id\"=>5013, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>19376309, \"stop\"=>19376323, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000380394.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5013/summary/variants/395/summary#variant\", \"evidence_items\"=>[{\"id\"=>930, \"name\"=>\"EID930\", \"description\"=>\"54 patients with advanced gastric cancer were treated with everolimus. Two patient responsed and potential biomarkers were explored. High pre-treatment expression of phosphorylated RPS6 (pS6; at Ser240/4) was associated with higher disease control rate and progression free survival (P=0.043, P=0.001 respectively; N=28).\", \"disease\"=>{\"id\"=>147, \"name\"=>\"Gastric Adenocarcinoma\", \"display_name\"=>\"Gastric Adenocarcinoma\", \"doid\"=>\"3717\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3717\"}, \"drugs\"=>[{\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>636, \"name\"=>\"Phase II study of everolimus with biomarker exploration in patients with advanced gastric cancer refractory to chemotherapy including fluoropyrimidine and platinum.\", \"citation\"=>\"Yoon et al., 2012, Br. J. Cancer\", \"pubmed_id\"=>\"22343617\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22343617\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3304416\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>3, \"day\"=>13}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>395, \"phenotypes\"=>[]}]}, {\"id\"=>396, \"entrez_name\"=>\"JAK1\", \"entrez_id\"=>3716, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>3090, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>4.0, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>65298912, \"stop\"=>65432187, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000342505.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3090/summary/variants/396/summary#variant\", \"evidence_items\"=>[{\"id\"=>931, \"name\"=>\"EID931\", \"description\"=>\"Overexpression of JAK1 in 22 lung cancer cell lines was associated with sensitivity to PKC-inhibitor enzastaurin. Silencing of JAK1 (by small molecule JAK 1/2/3 inhibitor) led to resistance.  Overexpression of JAK1 (lentiviral mediated) led to sensitivity to enzastaurin.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>177, \"name\"=>\"Enzastaurin\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>637, \"name\"=>\"Enzastaurin has anti-tumour effects in lung cancers with overexpressed JAK pathway molecules.\", \"citation\"=>\"Shimokawa et al., 2012, Br. J. Cancer\", \"pubmed_id\"=>\"22333600\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22333600\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3305973\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>2, \"day\"=>28}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>396, \"phenotypes\"=>[]}]}, {\"id\"=>397, \"entrez_name\"=>\"BRCA1\", \"entrez_id\"=>672, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>6, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"17\", \"start\"=>41196312, \"stop\"=>41277387, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000357654.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/6/summary/variants/397/summary#variant\", \"evidence_items\"=>[{\"id\"=>933, \"name\"=>\"EID933\", \"description\"=>\"Expression of BRCA1 in 6 mesothelioma cell lines was associated with sensitivity to vinorelbine in functional studies.\", \"disease\"=>{\"id\"=>112, \"name\"=>\"Malignant Mesothelioma\", \"display_name\"=>\"Malignant Mesothelioma\", \"doid\"=>\"1790\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1790\"}, \"drugs\"=>[{\"id\"=>96, \"name\"=>\"Vinorelbine\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>639, \"name\"=>\"BRCA1 is an essential mediator of vinorelbine-induced apoptosis in mesothelioma.\", \"citation\"=>\"Busacca et al., 2012, J. Pathol.\", \"pubmed_id\"=>\"22190288\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22190288\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>6}, \"journal\"=>\"J. Pathol.\", \"full_journal_title\"=>\"The Journal of pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>397, \"phenotypes\"=>[]}]}, {\"id\"=>398, \"entrez_name\"=>\"DUSP6\", \"entrez_id\"=>1848, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>1567, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"12\", \"start\"=>89741009, \"stop\"=>89747048, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000279488.7\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1567/summary/variants/398/summary#variant\", \"evidence_items\"=>[{\"id\"=>934, \"name\"=>\"EID934\", \"description\"=>\"Expression of DUSP6 was associated with sensitivity to MEK inhibitor GSK1120212 (Sensitivity: 79%; specificity: 82%: P = 0.0027) in 218 solid cancer cell lines.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>19, \"name\"=>\"Trametinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>640, \"name\"=>\"Comprehensive predictive biomarker analysis for MEK inhibitor GSK1120212.\", \"citation\"=>\"Jing et al., 2012, Mol. Cancer Ther.\", \"pubmed_id\"=>\"22169769\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22169769\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>3}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>398, \"phenotypes\"=>[]}]}, {\"id\"=>400, \"entrez_name\"=>\"HSPB1\", \"entrez_id\"=>3315, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>2784, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>75931861, \"stop\"=>75933612, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000248553.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/2784/summary/variants/400/summary#variant\", \"evidence_items\"=>[{\"id\"=>939, \"name\"=>\"EID939\", \"description\"=>\"HSP27 (HSPB1) expression in pancreatic adenocarcinoma cell lines was associated with sensitivity to treatment with gemcitabine.\", \"disease\"=>{\"id\"=>141, \"name\"=>\"Pancreatic Ductal Adenocarcinoma\", \"display_name\"=>\"Pancreatic Ductal Adenocarcinoma\", \"doid\"=>\"3498\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3498\"}, \"drugs\"=>[{\"id\"=>85, \"name\"=>\"Gemcitabine\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>642, \"name\"=>\"Heat shock protein 27 as a prognostic and predictive biomarker in pancreatic ductal adenocarcinoma.\", \"citation\"=>\"Schäfer et al., 2012, J. Cell. Mol. Med.\", \"pubmed_id\"=>\"22004109\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22004109\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3822691\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>8}, \"journal\"=>\"J. Cell. Mol. Med.\", \"full_journal_title\"=>\"Journal of cellular and molecular medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>400, \"phenotypes\"=>[]}]}, {\"id\"=>401, \"entrez_name\"=>\"NQO1\", \"entrez_id\"=>1728, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>1463, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"16\", \"start\"=>69743304, \"stop\"=>69760854, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000320623.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1463/summary/variants/401/summary#variant\", \"evidence_items\"=>[{\"id\"=>940, \"name\"=>\"EID940\", \"description\"=>\"Higher expression of NQO1 was correlated with decreased cytotoxicity of amrubicin (amrubicinol is the active metabolite) in 27 NSCLC and SCLC cell lines.  siRNA knockdown of NQO1 (in two cell lines) increased amrubicinol cytotoxicity.\", \"disease\"=>{\"id\"=>23, \"name\"=>\"Lung Cancer\", \"display_name\"=>\"Lung Cancer\", \"doid\"=>\"1324\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1324\"}, \"drugs\"=>[{\"id\"=>181, \"name\"=>\"Amrubicin\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>643, \"name\"=>\"C609T polymorphism of NAD(P)H quinone oxidoreductase 1 as a predictive biomarker for response to amrubicin.\", \"citation\"=>\"Takakuwa et al., 2011, J Thorac Oncol\", \"pubmed_id\"=>\"21964527\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21964527\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>11}, \"journal\"=>\"J Thorac Oncol\", \"full_journal_title\"=>\"Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>401, \"phenotypes\"=>[]}]}, {\"id\"=>403, \"entrez_name\"=>\"BRCA1\", \"entrez_id\"=>672, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>6, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"17\", \"start\"=>41196312, \"stop\"=>41277387, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000357654.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/6/summary/variants/403/summary#variant\", \"evidence_items\"=>[{\"id\"=>942, \"name\"=>\"EID942\", \"description\"=>\"Absent/low expression of BRCA1 (IHC) was associated with improved response to chemotherapy in 292 patients with sporadic ovarian cancer compared to patients with high expression (odds ratio 2.47: 95%CI 1.10-5.55, p=0.029).\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>103, \"name\"=>\"PLATINUM\", \"pubchem_id\"=>nil}, {\"id\"=>167, \"name\"=>\"Taxane\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>644, \"name\"=>\"BRCA1 is both a prognostic and predictive biomarker of response to chemotherapy in sporadic epithelial ovarian cancer.\", \"citation\"=>\"Carser et al., 2011, Gynecol. Oncol.\", \"pubmed_id\"=>\"21920589\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21920589\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>12}, \"journal\"=>\"Gynecol. Oncol.\", \"full_journal_title\"=>\"Gynecologic oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>403, \"phenotypes\"=>[]}]}, {\"id\"=>404, \"entrez_name\"=>\"ABCB1\", \"entrez_id\"=>5243, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>4244, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>87133175, \"stop\"=>87342564, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000265724.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4244/summary/variants/404/summary#variant\", \"evidence_items\"=>[{\"id\"=>944, \"name\"=>\"EID944\", \"description\"=>\"mRNA expression of ABCB1 in 17 NSCLC cell lines was associated with decreased response to paclitaxel (correlation of mRNA expression and paclitaxel IC50: r=0.5322, P=0.0279).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>81, \"name\"=>\"Paclitaxel\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>646, \"name\"=>\"ATP-binding cassette B1 gene polymorphisms, mRNA expression and chemosensitivity to paclitaxel in non-small cell lung cancer cells.\", \"citation\"=>\"Meng et al., 2011, Respirology\", \"pubmed_id\"=>\"21883677\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21883677\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>11}, \"journal\"=>\"Respirology\", \"full_journal_title\"=>\"Respirology (Carlton, Vic.)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>404, \"phenotypes\"=>[]}]}, {\"id\"=>405, \"entrez_name\"=>\"CXCR4\", \"entrez_id\"=>7852, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>6329, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>136871919, \"stop\"=>136873813, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000409817.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/6329/summary/variants/405/summary#variant\", \"evidence_items\"=>[{\"id\"=>2912, \"name\"=>\"EID2912\", \"description\"=>\"CXCR4, a stromal-derived-factor-1 specific chemokine receptor, is highly expressed in non-small cell lung cancer (NSCLC) tissues and participates in cancer progression by regulating cell growth, apoptosis or invasion. In this study, we therefore investigated whether CXCR4 plays a role in the cisplatin associated resistance in NSCLC. We detected the expression of CXCR4 in tissue specimens from 64 NSCLC patients by immunohistochemistry. Cisplatin-resistant NSCLC cells A549/DDP and its parental A549 cells were employed in this study. RNA interference was performed to silence the CXCR4. Finally, we found CXCR4 was significantly highly expressed in cisplatin-resistant NSCLC patients and the A549/DDP cell line. CXCR4 inhibition by siRNA reversed chemoresistance and decreased tumor cell proliferation.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1674, \"name\"=>\"CXCR4 promotes cisplatin-resistance of non-small cell lung cancer in a CYP1B1-dependent manner.\", \"citation\"=>\"Xie et al., 2017, Oncol. Rep.\", \"pubmed_id\"=>\"27922681\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27922681\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>2}, \"journal\"=>\"Oncol. Rep.\", \"full_journal_title\"=>\"Oncology reports\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>405, \"phenotypes\"=>[]}, {\"id\"=>949, \"name\"=>\"EID949\", \"description\"=>\"CXCR4 expression as assessed by microarray analysis, was associated with resistance to docetaxel in 11 cancer cell lines (R^2 = 0.23, p = 0.019). Expression levels (by qtPCR) were also higher in resistant tumors among 25 patient samples.\", \"disease\"=>{\"id\"=>147, \"name\"=>\"Gastric Adenocarcinoma\", \"display_name\"=>\"Gastric Adenocarcinoma\", \"doid\"=>\"3717\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3717\"}, \"drugs\"=>[{\"id\"=>73, \"name\"=>\"Docetaxel\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>651, \"name\"=>\"CXCR4, a potential predictive marker for docetaxel sensitivity in gastric cancer.\", \"citation\"=>\"Xie et al., 2010, Anticancer Res.\", \"pubmed_id\"=>\"20651371\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20651371\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>6}, \"journal\"=>\"Anticancer Res.\", \"full_journal_title\"=>\"Anticancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>405, \"phenotypes\"=>[]}]}, {\"id\"=>406, \"entrez_name\"=>\"MAGEH1\", \"entrez_id\"=>28986, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>11261, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"X\", \"start\"=>55478538, \"stop\"=>55479998, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000342972.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/11261/summary/variants/406/summary#variant\", \"evidence_items\"=>[{\"id\"=>950, \"name\"=>\"EID950\", \"description\"=>\"10 biliary tract carcinoma cell lines were sorted based on their gemcitabine sensitivity. MAGEH1 protein expression was negatively correlated with sensitivity.\\nIn a clinical validation cohort, MAGEH protein expression was found in 4/4 \\\"non-effective\\\" cases and 2/5 effective cases based on RECIST criteria after gemcitabine treatment.\", \"disease\"=>{\"id\"=>168, \"name\"=>\"Bile Duct Adenocarcinoma\", \"display_name\"=>\"Bile Duct Adenocarcinoma\", \"doid\"=>\"4896\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4896\"}, \"drugs\"=>[{\"id\"=>85, \"name\"=>\"Gemcitabine\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>652, \"name\"=>\"Establishment of six new human biliary tract carcinoma cell lines and identification of MAGEH1 as a candidate biomarker for predicting the efficacy of gemcitabine treatment.\", \"citation\"=>\"Ojima et al., 2010, Cancer Sci.\", \"pubmed_id\"=>\"20088962\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20088962\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>4}, \"journal\"=>\"Cancer Sci.\", \"full_journal_title\"=>\"Cancer science\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>406, \"phenotypes\"=>[]}]}, {\"id\"=>408, \"entrez_name\"=>\"ABCC10\", \"entrez_id\"=>89845, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>16656, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>43395292, \"stop\"=>43418168, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000372530.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/16656/summary/variants/408/summary#variant\", \"evidence_items\"=>[{\"id\"=>952, \"name\"=>\"EID952\", \"description\"=>\"Increased expression of ABCC10 in 17 NSCLC cell lines was inversely correlated with paclitaxel sensitivity (r = 0.574; P < 0.05).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>81, \"name\"=>\"Paclitaxel\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>654, \"name\"=>\"MRP7/ABCC10 expression is a predictive biomarker for the resistance to paclitaxel in non-small cell lung cancer.\", \"citation\"=>\"Oguri et al., 2008, Mol. Cancer Ther.\", \"pubmed_id\"=>\"18445659\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18445659\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2008, \"month\"=>5}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>408, \"phenotypes\"=>[]}]}, {\"id\"=>410, \"entrez_name\"=>\"CDX2\", \"entrez_id\"=>1045, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>927, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"13\", \"start\"=>28536274, \"stop\"=>28545276, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000381020.7\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/927/summary/variants/410/summary#variant\", \"evidence_items\"=>[{\"id\"=>1130, \"name\"=>\"EID1130\", \"description\"=>\"In patients with stage II colon cancer, positive CDX2 status was associated with longer overall survival.\", \"disease\"=>{\"id\"=>119, \"name\"=>\"Colon Cancer\", \"display_name\"=>\"Colon Cancer\", \"doid\"=>\"219\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:219\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"rejected\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>781, \"name\"=>\"Prognostic Subgroups among Patients with Stage II Colon Cancer.\", \"citation\"=>\"Boland et al., 2016, N. Engl. J. Med.\", \"pubmed_id\"=>\"26789877\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26789877\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4852473\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>1, \"day\"=>21}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>410, \"phenotypes\"=>[]}, {\"id\"=>956, \"name\"=>\"EID956\", \"description\"=>\"Patients with stage II/III colorectal cancer were analyzed for expression of CDX2 (by microarray and IHC). In the discovery (466 patients) and validation sets (314 patients), 5-year DFS was lower among patients with CDX2 negative tumors (discovery set:hazard ratio for disease recurrence, 3.44; 95% confidence interval [CI], 1.60 to 7.38; P=0.002; validation set: hazard ratio, 2.42; 95% CI, 1.36 to 4.29; P=0.003).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>658, \"name\"=>\"CDX2 as a Prognostic Biomarker in Stage II and Stage III Colon Cancer.\", \"citation\"=>\"Dalerba et al., 2016, N. Engl. J. Med.\", \"pubmed_id\"=>\"26789870\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26789870\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4784450\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>1, \"day\"=>21}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>410, \"phenotypes\"=>[]}]}, {\"id\"=>412, \"entrez_name\"=>\"POU5F1\", \"entrez_id\"=>5460, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>4419, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>31132120, \"stop\"=>31138470, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000259915.8\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4419/summary/variants/412/summary#variant\", \"evidence_items\"=>[{\"id\"=>958, \"name\"=>\"EID958\", \"description\"=>\"OCT4B1 (POU5F1 splice variant) expression reduced sensitivity to oxaliplatin (via P-gp and ABCG2 differential expression) in vitro.\", \"disease\"=>{\"id\"=>119, \"name\"=>\"Colon Cancer\", \"display_name\"=>\"Colon Cancer\", \"doid\"=>\"219\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:219\"}, \"drugs\"=>[{\"id\"=>237, \"name\"=>\"Oxaliplatin\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>660, \"name\"=>\"OCT4B1 promotes cell growth, migration and invasion suppressing sensitivity to οxaliplatin in colon cancer.\", \"citation\"=>\"Wen et al., 2015, Oncol. Rep.\", \"pubmed_id\"=>\"26398480\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26398480\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>12}, \"journal\"=>\"Oncol. Rep.\", \"full_journal_title\"=>\"Oncology reports\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>412, \"phenotypes\"=>[]}]}, {\"id\"=>416, \"entrez_name\"=>\"ERBB2\", \"entrez_id\"=>2064, \"name\"=>\"SERUM LEVELS\", \"description\"=>\"\", \"gene_id\"=>20, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>35.0, \"coordinates\"=>{\"chromosome\"=>\"17\", \"start\"=>37856333, \"stop\"=>37884915, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000269571.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/20/summary/variants/416/summary#variant\", \"evidence_items\"=>[{\"id\"=>972, \"name\"=>\"EID972\", \"description\"=>\"Serum and tissue levels of HER2 (ERBB2) were analyzed in 1,902 breast cancer patients treated with lapatinib or control.\\nSerum HER2 (sHER2) levels measured by ELISA were higher in patients with HER2 amplification (median sHER2 levels were 25.1 ng/mL in patients with HER2-positive tumors). Serum HER2 levels were Independent predictors of lapatinib progression free survival benefit (HR per 10-ng/mL increase in serum HER2: lapatinib-containing therapies, 1.009 v nonlapatinib-containing therapies, 1.044; P interaction < .001). Tissue HER2 levels were also predictive, but within this subgroup, serum HER2 still independently predicted lapatinib PFS benefit.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>45, \"name\"=>\"Lapatinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>672, \"name\"=>\"Serum Human Epidermal Growth Factor 2 Extracellular Domain as a Predictive Biomarker for Lapatinib Treatment Efficacy in Patients With Advanced Breast Cancer.\", \"citation\"=>\"Lee et al., 2016, J. Clin. Oncol.\", \"pubmed_id\"=>\"26811533\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26811533\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>3, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>416, \"phenotypes\"=>[]}, {\"id\"=>973, \"name\"=>\"EID973\", \"description\"=>\"Serum and tissue levels of HER2 (ERBB2) were analyzed in 1,902 breast cancer patients treated with lapatinib or control. In the control group (n = 936), higher serum HER2 (sHER2) was associated with worse prognosis in multivariable analyses (progression free survival hazard ratio per 10 ng/mL: PFS, 1.024; P < .001; and OS, 1.018; P < .001).\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>672, \"name\"=>\"Serum Human Epidermal Growth Factor 2 Extracellular Domain as a Predictive Biomarker for Lapatinib Treatment Efficacy in Patients With Advanced Breast Cancer.\", \"citation\"=>\"Lee et al., 2016, J. Clin. Oncol.\", \"pubmed_id\"=>\"26811533\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26811533\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>3, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>416, \"phenotypes\"=>[]}]}, {\"id\"=>417, \"entrez_name\"=>\"THBS2\", \"entrez_id\"=>7058, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>5761, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>169615875, \"stop\"=>169654139, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000366787.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5761/summary/variants/417/summary#variant\", \"evidence_items\"=>[{\"id\"=>974, \"name\"=>\"EID974\", \"description\"=>\"172 patients treated with neoadjuvant concurrent chemoradiotherapy (CCRT) were retrospectively analyzed for expression of thrombospondin 2 (THBS2) using IHC (discovery set with 46 rectal cancer patients, GSE35452). Low expression was associated with advanced pretreatment tumor, nodal status, post-treatment tumor, vascular invasion and inferior regression grade. In multivariate analysis, low expression served as negative prognostic factor for disease-free survival (Hazard ratio=3.057, P=0.002).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>673, \"name\"=>\"Low thrombospondin 2 expression is predictive of low tumor regression after neoadjuvant chemoradiotherapy in rectal cancer.\", \"citation\"=>\"Lin et al., 2015, Am J Transl Res\", \"pubmed_id\"=>\"26807188\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26807188\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4697720\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"Am J Transl Res\", \"full_journal_title\"=>\"American journal of translational research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>417, \"phenotypes\"=>[]}]}, {\"id\"=>422, \"entrez_name\"=>\"STAG2\", \"entrez_id\"=>10735, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>8553, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>14.0, \"coordinates\"=>{\"chromosome\"=>\"X\", \"start\"=>123095568, \"stop\"=>123236506, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000371157.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/8553/summary/variants/422/summary#variant\", \"evidence_items\"=>[{\"id\"=>1659, \"name\"=>\"EID1659\", \"description\"=>\"A375 melanoma cells containing an inducible shRNA for STAG2 were grown as xenograft tumors in nude mice. The mice in which STAG2 was silenced had tumors that were more sensitive to vemurafenib-induced tumor shrinkage than the tumors of the control mice (P < .01).\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>4, \"name\"=>\"Vemurafenib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1123, \"name\"=>\"Loss of cohesin complex components STAG2 or STAG3 confers resistance to BRAF inhibition in melanoma.\", \"citation\"=>\"Shen et al., 2016, Nat. Med.\", \"pubmed_id\"=>\"27500726\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27500726\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5014622\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>9}, \"journal\"=>\"Nat. Med.\", \"full_journal_title\"=>\"Nature medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>422, \"phenotypes\"=>[]}, {\"id\"=>984, \"name\"=>\"EID984\", \"description\"=>\"Tumor samples from 125 patients with bladder cancer were assessed by IHC for STAG2 expression. STAG2-negativity was associated with lower histological grade (p = 0.009). Complete loss of STAG2 expression was associated with a lower risk of recurrence (P = 0.023) and diminished risk of death (P = 0.034) in log-rank analysis and was a beneficial factor in uni- and multivariate analysis.\", \"disease\"=>{\"id\"=>18, \"name\"=>\"Bladder Carcinoma\", \"display_name\"=>\"Bladder Carcinoma\", \"doid\"=>\"4007\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4007\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>681, \"name\"=>\"Complete loss of STAG2 expression is an indicator of good prognosis in patients with bladder cancer.\", \"citation\"=>\"Qiao et al., 2016, Tumour Biol.\", \"pubmed_id\"=>\"26838030\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26838030\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>2, \"day\"=>2}, \"journal\"=>\"Tumour Biol.\", \"full_journal_title\"=>\"Tumour biology : the journal of the International Society for Oncodevelopmental Biology and Medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>422, \"phenotypes\"=>[]}]}, {\"id\"=>423, \"entrez_name\"=>\"HLA-DRA\", \"entrez_id\"=>3122, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>2622, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>32407619, \"stop\"=>32412823, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000395388.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/2622/summary/variants/423/summary#variant\", \"evidence_items\"=>[{\"id\"=>985, \"name\"=>\"EID985\", \"description\"=>\"Melanoma patients treated with Anti-PD1 or anti-PD-L1(nivolumab, pembrolizumab, MPDL3280A) therapy were analyzed for expression of HLA-DRA (the primary antigen-presenting molecule of the MHC-II pathway). In a discovery cohort, 30 patients underwent IHC for HLA-DRA. 11/14 (79%) patients with positive HLA-DR staining, had complete (n=3) or partial (n=8) response. 6/16 (38%) HLA-DR non-expressing melanomas responded to therapy (overall response rate 79 versus 38%, Fisher’s exact test P=0.033). This finding was confirmed in a second independent data set of 23 melanoma patients with 6/8 (75%) of HLA-DR(+) tumours in comparison to 4/15 (27%) HLA-DR(−) responding (Fisher’s exact test P=0.025). Preclinical models confirmed the result.\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>137, \"name\"=>\"Nivolumab\", \"pubchem_id\"=>nil}, {\"id\"=>138, \"name\"=>\"Pembrolizumab\", \"pubchem_id\"=>nil}, {\"id\"=>261, \"name\"=>\"Atezolizumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>682, \"name\"=>\"Melanoma-specific MHC-II expression represents a tumour-autonomous phenotype and predicts response to anti-PD-1/PD-L1 therapy.\", \"citation\"=>\"Johnson et al., 2016, Nat Commun\", \"pubmed_id\"=>\"26822383\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26822383\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4740184\", \"publication_date\"=>{\"year\"=>2016}, \"journal\"=>\"Nat Commun\", \"full_journal_title\"=>\"Nature communications\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>423, \"phenotypes\"=>[]}]}, {\"id\"=>429, \"entrez_name\"=>\"KIT\", \"entrez_id\"=>3815, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>29, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>55524085, \"stop\"=>55606881, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000288135.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/29/summary/variants/429/summary#variant\", \"evidence_items\"=>[{\"id\"=>1025, \"name\"=>\"EID1025\", \"description\"=>\"Case Report of a 41-year old patient with high-grade stromal endometrial sarcoma and a large abdominal and retroperitoneal mass progressing after chemotherapy. IHC for c-Kit was positive and treatment with imatinib led to an objective Response. Surgery was performed and the patient was free of disease two years later.\", \"disease\"=>{\"id\"=>108, \"name\"=>\"Endometrial Cancer\", \"display_name\"=>\"Endometrial Cancer\", \"doid\"=>\"1380\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1380\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>711, \"name\"=>\"A case of c-kit positive high-grade stromal endometrial sarcoma responding to Imatinib Mesylate.\", \"citation\"=>\"Salvatierra et al., 2006, Gynecol. Oncol.\", \"pubmed_id\"=>\"16487996\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16487996\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2006, \"month\"=>6}, \"journal\"=>\"Gynecol. Oncol.\", \"full_journal_title\"=>\"Gynecologic oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>429, \"phenotypes\"=>[]}, {\"id\"=>1024, \"name\"=>\"EID1024\", \"description\"=>\"Case Report of a 42-year old patient with low-grade endometrial stromal sarcoma. IHC was focally positive for CD117. Imatinib mesylate led to complete metabolic response (PET-CT).\", \"disease\"=>{\"id\"=>108, \"name\"=>\"Endometrial Cancer\", \"display_name\"=>\"Endometrial Cancer\", \"doid\"=>\"1380\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1380\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>710, \"name\"=>\"Detection of complete response to imatinib mesylate (Glivec/Gleevec) with 18F-FDG PET/CT for low-grade endometrial stromal sarcoma.\", \"citation\"=>\"Kalender et al., 2009, Cancer Chemother. Pharmacol.\", \"pubmed_id\"=>\"18607592\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18607592\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>2}, \"journal\"=>\"Cancer Chemother. Pharmacol.\", \"full_journal_title\"=>\"Cancer chemotherapy and pharmacology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>429, \"phenotypes\"=>[]}]}, {\"id\"=>434, \"entrez_name\"=>\"HIF1A\", \"entrez_id\"=>3091, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>2596, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>25.0, \"coordinates\"=>{\"chromosome\"=>\"14\", \"start\"=>62162258, \"stop\"=>62214976, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000394997.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/2596/summary/variants/434/summary#variant\", \"evidence_items\"=>[{\"id\"=>1034, \"name\"=>\"EID1034\", \"description\"=>\"Expression of HIF1A was analyzed by IHC in 49 soft-tissue sarcomas (29 malignant fibrous histiocytomas, 12 synovial sarcomas, 4 leiomyosarcomas and 4 malignant peripheral nerve sheath tumors). High (18.4%) or moderate (35.4%) expression of HIF-1alpha was significatnly associated with shorter overall survival rate in comparison with those with a weak or negative expression in univariate analysis (P = 0.029; log-rank test) and multivariate analysis (P = 0.018).\", \"disease\"=>{\"id\"=>105, \"name\"=>\"Sarcoma\", \"display_name\"=>\"Sarcoma\", \"doid\"=>\"1115\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1115\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>718, \"name\"=>\"Expression of hypoxia-inducible factor (HIF)-1alpha as a biomarker of outcome in soft-tissue sarcomas.\", \"citation\"=>\"Shintani et al., 2006, Virchows Arch.\", \"pubmed_id\"=>\"17103226\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17103226\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2006, \"month\"=>12}, \"journal\"=>\"Virchows Arch.\", \"full_journal_title\"=>\"Virchows Archiv : an international journal of pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>434, \"phenotypes\"=>[]}, {\"id\"=>1032, \"name\"=>\"EID1032\", \"description\"=>\"HIF1A overexpression was assessed by IHC in 731 colorectal cancers from two prospective cohort studies. 142 (19%) tumors showed HIF1A overexpression. Overexpression of HIF1A was associated with higher colorectal cancer-specific mortality in Kaplan-Meier analysis (log-rank test, P < 0.0001), univariate Cox regression (hazard ratio = 1.84; 95% confidence interval, 1.37 to 2.47; P < 0.0001) and multivariate analysis (adjusted hazard ratio = 1.72; 95% confidence interval, 1.26 to 2.36; P = 0.0007, adjusted for clinical and tumoral features: microsatellite instability, TP53 (p53), PTGS2 (cyclooxygenase-2), CpG island methylator phenotype, and KRAS, BRAF, PIK3CA, and LINE-1 methylation).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>717, \"name\"=>\"HIF1A overexpression is associated with poor prognosis in a cohort of 731 colorectal cancers.\", \"citation\"=>\"Baba et al., 2010, Am. J. Pathol.\", \"pubmed_id\"=>\"20363910\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20363910\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2861094\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>5}, \"journal\"=>\"Am. J. Pathol.\", \"full_journal_title\"=>\"The American journal of pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>434, \"phenotypes\"=>[]}]}, {\"id\"=>435, \"entrez_name\"=>\"EPAS1\", \"entrez_id\"=>2034, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>1705, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>30.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>46524541, \"stop\"=>46613836, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000263734.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1705/summary/variants/435/summary#variant\", \"evidence_items\"=>[{\"id\"=>1673, \"name\"=>\"EID1673\", \"description\"=>\"HIF-2A (EPAS1) staining was conducted on 66 tumors.  Of the 66 patients, 25 were classified as responders and 41 were classified as non-responders.  There was no significant difference between responders and non-responders with regards to overall response rate to pazopanib (44% vs 33%; P=0.36).\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[{\"id\"=>56, \"name\"=>\"Pazopanib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"N/A\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1132, \"name\"=>\"The role of aberrant VHL/HIF pathway elements in predicting clinical outcome to pazopanib therapy in patients with metastatic clear-cell renal cell carcinoma.\", \"citation\"=>\"Choueiri et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"23881929\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23881929\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4522695\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>9, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>435, \"phenotypes\"=>[]}, {\"id\"=>1033, \"name\"=>\"EID1033\", \"description\"=>\"Expression of EPAS1 (HIF-2alpha) was assessed in 731 colorectal cancers from two prospective cohort studies. EPAS1 overexpression was found in 322 (46%) cancers by immunohistochemistry and not significantly associated with patient survival.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>717, \"name\"=>\"HIF1A overexpression is associated with poor prognosis in a cohort of 731 colorectal cancers.\", \"citation\"=>\"Baba et al., 2010, Am. J. Pathol.\", \"pubmed_id\"=>\"20363910\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20363910\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2861094\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>5}, \"journal\"=>\"Am. J. Pathol.\", \"full_journal_title\"=>\"The American journal of pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>435, \"phenotypes\"=>[]}]}, {\"id\"=>438, \"entrez_name\"=>\"PSMB8\", \"entrez_id\"=>5696, \"name\"=>\"NUCLEAR EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>4616, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>32808494, \"stop\"=>32812480, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000374881.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4616/summary/variants/438/summary#variant\", \"evidence_items\"=>[{\"id\"=>1051, \"name\"=>\"EID1051\", \"description\"=>\"Microarray expression data from 48 patients with gastric adenocarcinoma were analyzed for genes differentially expressed with regard to overall survival. Expression of 294 and 116 genes was enriched in good and poor prognosis subtypes, respectively. Of these genes, upregulation of proteasome subunit beta type 8 PSMB8 and PDZ binding kinase PBK was confirmed by real-time reverse transcription-PCR Analysis. IHC in 385 gastric cancer patients showed that increased nuclear expression of PSMB8 and PBK was correlated with depth of invasion, lymph node metastasis, and lower survival rates (survival durations of the (-), (+), and (++) groups, classified according to nuclear PSMB8 expression, were 77.2, 70.6, and 62.2 months, respectively; P = 0.004). Cytoplasmic staining of PSMB8 and PBK were not statistically significant. In multivariate Analysis, nuclear expression of PSMB8 and PBK were not significant predictors of overall survival (P = 0.085, P = 0.083, respectively).\", \"disease\"=>{\"id\"=>147, \"name\"=>\"Gastric Adenocarcinoma\", \"display_name\"=>\"Gastric Adenocarcinoma\", \"doid\"=>\"3717\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3717\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>731, \"name\"=>\"PSMB8 and PBK as potential gastric cancer subtype-specific biomarkers associated with prognosis.\", \"citation\"=>\"Kwon et al., 2016, Oncotarget\", \"pubmed_id\"=>\"26894977\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26894977\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>19}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>438, \"phenotypes\"=>[]}]}, {\"id\"=>439, \"entrez_name\"=>\"PBK\", \"entrez_id\"=>55872, \"name\"=>\"NUCLEAR EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>13276, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"8\", \"start\"=>27667137, \"stop\"=>27695612, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000301905.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/13276/summary/variants/439/summary#variant\", \"evidence_items\"=>[{\"id\"=>1052, \"name\"=>\"EID1052\", \"description\"=>\"Microarray expression data from 48 patients with gastric adenocarcinoma were analyzed for genes differentially expressed with regard to overall survival. Upregulation of proteasome subunit beta type 8 (PSMB8) and PDZ binding kinase (PBK) was confirmed by real-time reverse transcription-PCR Analysis. IHC in 385 gastric cancer patients showed that increased nuclear expression of PSMB8 and PBK was correlated with depth of invasion, lymph node metastasis, and lower survival rates (survival duration, months according to nuclear PBK Expression: -, 72.3 ± 1.2; +, 66.2 ± 3.4; ++, 53.1 ± 9.2; P = 0.002). Cytoplasmic staining of PSMB8 and PBK were not statistically significant. In multivariate analysis, nuclear expression of PSMB8 and PBK were not significant predictors of overall survival (P = 0.085, P = 0.083, respectively).\", \"disease\"=>{\"id\"=>147, \"name\"=>\"Gastric Adenocarcinoma\", \"display_name\"=>\"Gastric Adenocarcinoma\", \"doid\"=>\"3717\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3717\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>731, \"name\"=>\"PSMB8 and PBK as potential gastric cancer subtype-specific biomarkers associated with prognosis.\", \"citation\"=>\"Kwon et al., 2016, Oncotarget\", \"pubmed_id\"=>\"26894977\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26894977\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>19}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>439, \"phenotypes\"=>[]}]}, {\"id\"=>440, \"entrez_name\"=>\"PTTG1\", \"entrez_id\"=>9232, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>7297, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"5\", \"start\"=>159848829, \"stop\"=>159855748, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000352433.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/7297/summary/variants/440/summary#variant\", \"evidence_items\"=>[{\"id\"=>1053, \"name\"=>\"EID1053\", \"description\"=>\"144 cases (discovery set: N=62 discovery; independent validation set: N = 82) of meningiomas were analyzed for transcriptional features defining aggressive clinical course (recurrent, malignantly progressing, or WHO grade III). Meningiomas were categorized into non-recurrent (NR), recurrent (R), and tumors undergoing malignant progression (M). Unsupervised transcriptomic analysis in 62 meningiomas revealed a large set of differentially expressed genes in aggressive subgroups (R+M tumors and WHO grade III, n=332; p<0.01, FC>1.25). Multivariate analysis identified PTTG1 and LEPR mRNA expression independent of gender, WHO grade and extent of resection. The patient group with the longest progression free survival was characterized by co-occurrence of low PTTG1 and high LEPR mRNA expression. IHC in 36 tumors validated these results at the protein level.\", \"disease\"=>{\"id\"=>144, \"name\"=>\"Meningioma\", \"display_name\"=>\"Meningioma\", \"doid\"=>\"3565\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3565\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>732, \"name\"=>\"Transcriptomic analysis of aggressive meningiomas identifies PTTG1 and LEPR as prognostic biomarkers independent of WHO grade.\", \"citation\"=>\"Schmidt et al., 2016, Oncotarget\", \"pubmed_id\"=>\"26894859\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26894859\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4924735\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>3, \"day\"=>22}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>440, \"phenotypes\"=>[]}]}, {\"id\"=>441, \"entrez_name\"=>\"LEPR\", \"entrez_id\"=>3953, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>3274, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>65886335, \"stop\"=>66107242, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000349533.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3274/summary/variants/441/summary#variant\", \"evidence_items\"=>[{\"id\"=>1054, \"name\"=>\"EID1054\", \"description\"=>\"144 cases (discovery set: N=62 discovery; Independent validation set: N = 82) of meningiomas were analyzed for transcriptional features defining aggressive clinical course (recurrent, malignantly progressing or WHO grade III). Meningiomas were categorized into non-recurrent (NR), recurrent (R), and tumors undergoing malignant progression (M). Unsupervised transcriptomic analysis in 62 meningiomas revealed a large set of differentially expressed genes in aggressive subgroups (R+M tumors and WHO grade III, n=332; p<0.01, FC>1.25). Multivariate analysis identified PTTG1 and LEPR mRNA expression independent of gender, WHO grade and extent of resection. The patient group with the longest PFS was characterized by co-occurrence of low PTTG1 and high LEPR mRNA expression. IHC in 36 tumors validated these results on protein level.\", \"disease\"=>{\"id\"=>144, \"name\"=>\"Meningioma\", \"display_name\"=>\"Meningioma\", \"doid\"=>\"3565\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3565\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>732, \"name\"=>\"Transcriptomic analysis of aggressive meningiomas identifies PTTG1 and LEPR as prognostic biomarkers independent of WHO grade.\", \"citation\"=>\"Schmidt et al., 2016, Oncotarget\", \"pubmed_id\"=>\"26894859\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26894859\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4924735\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>3, \"day\"=>22}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>441, \"phenotypes\"=>[]}]}, {\"id\"=>444, \"entrez_name\"=>\"NEDD9\", \"entrez_id\"=>4739, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>3852, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>11185051, \"stop\"=>11382581, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000504387.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3852/summary/variants/444/summary#variant\", \"evidence_items\"=>[{\"id\"=>1059, \"name\"=>\"EID1059\", \"description\"=>\"IHC was performed in 123 hepatocellular carcinoma tissues and their adjacent normal liver tissues for protein expression of HEF1 (NEDD9). Overexpression of HEF1 protein was observed in HCC tissue when compared with their adjacent non-malignant liver tissue. High expression of HEF1 correlated with higher advanced tumor, node, metastasis (TNM) stage and vascular invasion (P<0.05). In univariate and multivariate analysis, the expression of HEF1 was identified as an independent prognostic factor (Spearman's rank correlation, Kaplan-Meier plots and Cox regression model). In subgroup analysis, high expression of HEF1 correlated with a poorer prognosis in advanced (TNM III+IV) stages (P<0.05).\", \"disease\"=>{\"id\"=>196, \"name\"=>\"Hepatocellular Carcinoma\", \"display_name\"=>\"Hepatocellular Carcinoma\", \"doid\"=>\"684\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:684\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>734, \"name\"=>\"High expression of HEF1 predicts a poorer prognosis of hepatocellular carcinoma: A retrospective study.\", \"citation\"=>\"Li et al., 2016, Mol Clin Oncol\", \"pubmed_id\"=>\"26893853\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26893853\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4733949\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>2}, \"journal\"=>\"Mol Clin Oncol\", \"full_journal_title\"=>\"Molecular and clinical oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>444, \"phenotypes\"=>[]}]}, {\"id\"=>446, \"entrez_name\"=>\"KRT18\", \"entrez_id\"=>3875, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>3215, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"12\", \"start\"=>53342655, \"stop\"=>53346684, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000388837.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3215/summary/variants/446/summary#variant\", \"evidence_items\"=>[{\"id\"=>1061, \"name\"=>\"EID1061\", \"description\"=>\"Comparison of nuclear matrix protein (NMP) patterns of paclitaxel-resistant PCa cells (DU145-TxR), with that of parental cell (DU145) using two-dimensional electrophoresis, identified cytokeratin 18 (CK18) as downregulated in DU145-TxR cells (validated at mRNA, NMP and total cellular protein levels). An in-vivo assay of xenograft transplantation showed higher tumorigenicity of DU145-TxR cells. Immunohistochemistry showed that CK18 expression in PCa tissues was statistically significant inversely correlated with tumor grade.\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>736, \"name\"=>\"Downregulation of cytokeratin 18 is associated with paclitaxel‑resistance and tumor aggressiveness in prostate cancer.\", \"citation\"=>\"Yin et al., 2016, Int. J. Oncol.\", \"pubmed_id\"=>\"26892177\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26892177\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>4}, \"journal\"=>\"Int. J. Oncol.\", \"full_journal_title\"=>\"International journal of oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>446, \"phenotypes\"=>[]}]}, {\"id\"=>447, \"entrez_name\"=>\"FGF13\", \"entrez_id\"=>2258, \"name\"=>\"CYTOPLASMIC EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>1883, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"X\", \"start\"=>137715011, \"stop\"=>138286269, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000370603.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1883/summary/variants/447/summary#variant\", \"evidence_items\"=>[{\"id\"=>1062, \"name\"=>\"EID1062\", \"description\"=>\"243 primary PCa, 37 metastatic PCa, and \\n20  castration-resistant  PCa  (CRPC) specimens underwent IHC and microarray analysis. Immunohistochemistry (IHC) showed that FGF13 was detectable in the majority of human PCa samples, and FGF13 IHC scores were higher in high-grade prostatic intraepithelial neoplasia, in primary PCa, and in metastatic PCa than in benign prostatic tissue. There was a significant association between high cytoplasmic FGF13 staining and a risk of biochemical recurrence (BCR) after radical prostatectomy. Multivariate regression analysis indicated that high cytoplasmic FGF13 staining was associated with BCR. Results were recapitulated in cell lines and public databases.\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>737, \"name\"=>\"Increased expression of fibroblast growth factor 13 in prostate cancer is associated with shortened time to biochemical recurrence after radical prostatectomy.\", \"citation\"=>\"Yu et al., 2016, Int. J. Cancer\", \"pubmed_id\"=>\"26891277\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26891277\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Int. J. Cancer\", \"full_journal_title\"=>\"International journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>447, \"phenotypes\"=>[]}]}, {\"id\"=>472, \"entrez_name\"=>\"ALDH1A2\", \"entrez_id\"=>8854, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>7024, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>30.0, \"coordinates\"=>{\"chromosome\"=>\"15\", \"start\"=>58245622, \"stop\"=>58358616, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000249750.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/7024/summary/variants/472/summary#variant\", \"evidence_items\"=>[{\"id\"=>1114, \"name\"=>\"EID1114\", \"description\"=>\"IHC for expression of ALDH1A2 and CRABP2 was performed in 89 patients with squamous cell carcinoma of the head and neck. High expression of both, ALDH1A2 and CRABP2 (in comparison to ALDH1A2 low or ALDH1A2 high and CRABP2 low) was significantly associated with OS and PFS in multivariate Cox analysis (OS, HR: 0.303, 95%CI 0.14–0.63, p-value 0.002 PFS, HR: 0.395, 95%CI 0.20–0.77, p-value 0.007).\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>769, \"name\"=>\"Impaired aldehyde dehydrogenase 1 subfamily member 2A-dependent retinoic acid signaling is related with a mesenchymal-like phenotype and an unfavorable prognosis of head and neck squamous cell carcinoma.\", \"citation\"=>\"Seidensaal et al., 2015, Mol. Cancer\", \"pubmed_id\"=>\"26634247\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26634247\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4669670\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"Mol. Cancer\", \"full_journal_title\"=>\"Molecular cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>472, \"phenotypes\"=>[]}, {\"id\"=>1116, \"name\"=>\"EID1116\", \"description\"=>\"ALDH1A2 protein levels were determined (IHC) in 115 patients with oropharyngeal squamous cell carcinoma with known HPV status (n = 89 non–HPV-driven; n = 26 HPV-driven). Low to moderate ALDH1A2 protein levels (combined as ALDH1A2low) were identified in keratinocytes of healthy mucosa as well as 73 (63.5%) primary tumors, and high ALDH1A2 protein levels in 42 (36.5%) primary tumors. Significantly higher ALDH1A2 protein levels were identified in HPV-driven than non-HPV-driven tumors. Moreover, PFS and OS strongly correlated with the ALDH1A2 protein expression level (OS, P < 0.001; PFS, P = 0.001; log-rank/Mantel-Cox test. Previous analyses in the same study identify hypermethylation of ALDH1A2 as the underlying mechanism of differential protein expression.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>770, \"name\"=>\"HPV-related methylation signature predicts survival in oropharyngeal squamous cell carcinomas.\", \"citation\"=>\"Kostareli et al., 2013, J. Clin. Invest.\", \"pubmed_id\"=>\"23635773\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23635773\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3668826\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>6}, \"journal\"=>\"J. Clin. Invest.\", \"full_journal_title\"=>\"The Journal of clinical investigation\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>472, \"phenotypes\"=>[]}]}, {\"id\"=>473, \"entrez_name\"=>\"ALDH1A2\", \"entrez_id\"=>8854, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>7024, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"15\", \"start\"=>58245622, \"stop\"=>58358616, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000249750.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/7024/summary/variants/473/summary#variant\", \"evidence_items\"=>[{\"id\"=>1115, \"name\"=>\"EID1115\", \"description\"=>\"FaDu (a high ALDH1A2 expression cell line) and Cal27 (a low expression cell line) were treated with all-trans retinoic acid (ATRA), Adapalene or Fenretinide. This led to concentration dependent decrease in the survival fraction in both cell lines with higher efficacy in Cal27 (ED50 ≈ 1 μM) as compared to FaDu cells (ED ≈ 44 μM). Treatment with 3 μM ATRA sensitized Cal27 cells to irradiation at a dose of 2 Gy, but had no or only a minor effect on ALDH1A2-postitive cells.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>265, \"name\"=>\"All-trans Retinoic Acid\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>769, \"name\"=>\"Impaired aldehyde dehydrogenase 1 subfamily member 2A-dependent retinoic acid signaling is related with a mesenchymal-like phenotype and an unfavorable prognosis of head and neck squamous cell carcinoma.\", \"citation\"=>\"Seidensaal et al., 2015, Mol. Cancer\", \"pubmed_id\"=>\"26634247\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26634247\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4669670\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"Mol. Cancer\", \"full_journal_title\"=>\"Molecular cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>473, \"phenotypes\"=>[]}]}, {\"id\"=>474, \"entrez_name\"=>\"KIAA1524\", \"entrez_id\"=>57650, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>13948, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>108268716, \"stop\"=>108308299, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000295746.8\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/13948/summary/variants/474/summary#variant\", \"evidence_items\"=>[{\"id\"=>1117, \"name\"=>\"EID1117\", \"description\"=>\"Expression of the protein CIP2A was investigated in 72 cases of pancreatic ductal adenocarcinoma and 27 cases of adjacent tissues. Immunohistochemistry revealed a significant difference in expression between tumor and adjacent normal samples. Furthermore, a statistically significant decrease in overall survival was discovered between positive and negative CIP2A expression.\", \"disease\"=>{\"id\"=>141, \"name\"=>\"Pancreatic Ductal Adenocarcinoma\", \"display_name\"=>\"Pancreatic Ductal Adenocarcinoma\", \"doid\"=>\"3498\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3498\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>774, \"name\"=>\"CIP2A down regulation enhances the sensitivity of pancreatic cancer cells to gemcitabine.\", \"citation\"=>\"Xu et al., 2016, Oncotarget\", \"pubmed_id\"=>\"26894380\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26894380\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4924755\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>3, \"day\"=>22}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>474, \"phenotypes\"=>[]}]}, {\"id\"=>475, \"entrez_name\"=>\"KIAA1524\", \"entrez_id\"=>57650, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>13948, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>108268716, \"stop\"=>108308299, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000295746.8\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/13948/summary/variants/475/summary#variant\", \"evidence_items\"=>[{\"id\"=>1118, \"name\"=>\"EID1118\", \"description\"=>\"Researchers studied the effect of CIP2A expression in relation to  gemcitabine sensitivity via SW1990 cell lines. SW1990 cells with CIP2A expression knocked-down displayed lower IC50 (p=.024), indicating a sensitivity to gemcitabine.\", \"disease\"=>{\"id\"=>141, \"name\"=>\"Pancreatic Ductal Adenocarcinoma\", \"display_name\"=>\"Pancreatic Ductal Adenocarcinoma\", \"doid\"=>\"3498\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3498\"}, \"drugs\"=>[{\"id\"=>85, \"name\"=>\"Gemcitabine\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>774, \"name\"=>\"CIP2A down regulation enhances the sensitivity of pancreatic cancer cells to gemcitabine.\", \"citation\"=>\"Xu et al., 2016, Oncotarget\", \"pubmed_id\"=>\"26894380\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26894380\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4924755\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>3, \"day\"=>22}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>475, \"phenotypes\"=>[]}]}, {\"id\"=>476, \"entrez_name\"=>\"MIR218-1\", \"entrez_id\"=>407000, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>24938, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>20529898, \"stop\"=>20530007, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000384999.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/24938/summary/variants/476/summary#variant\", \"evidence_items\"=>[{\"id\"=>1119, \"name\"=>\"EID1119\", \"description\"=>\"Researchers examined the prognostic role of miR-218 expression in CRC tumor samples when receiving 5-FU based treatment. CRC samples from patients who received first-line 5-FU-based adjuvant chemotherapy were stratified into two groups (complete/partial response and stable/progressive disease) based on RECIST criteria. miR-218 expression levels were found to be significantly higher in the complete/partial response group.  Further Kaplan-Meier survival analysis revealed that in groups with higher miR-218 expression, overall survival was significantly greater.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>775, \"name\"=>\"MicroRNA-218 is a prognostic indicator in colorectal cancer and enhances 5-fluorouracil-induced apoptosis by targeting BIRC5.\", \"citation\"=>\"Li et al., 2015, Carcinogenesis\", \"pubmed_id\"=>\"26442524\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26442524\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>12}, \"journal\"=>\"Carcinogenesis\", \"full_journal_title\"=>\"Carcinogenesis\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>476, \"phenotypes\"=>[]}]}, {\"id\"=>481, \"entrez_name\"=>\"DDX43\", \"entrez_id\"=>55510, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>13005, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>4.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>74104471, \"stop\"=>74127292, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000370336.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/13005/summary/variants/481/summary#variant\", \"evidence_items\"=>[{\"id\"=>1135, \"name\"=>\"EID1135\", \"description\"=>\"MEK inhibitor-resistant (selumetinib, PD0325901, GSK1129212) uveal melanoma cell lines were created from two GNAQ Q209L mutant cell lines. Overexpression of DDX43 mediated RAS-induced selumetinib resistance as demonstrated by depletion of DDX43 decreasing RAS proteins and ectopic expression of DDX43 induced RAS protein levels. These results were confirmed using biopsies from liver metastases in 14 patients with uveal melanoma harboring GNAQ or GNA11 mutations who were treated on a phase II study of selumetinib. 6 patients with partial response or stable disease for ≥16 weeks, were defined as “responders” (R), 8 patients who had disease progression before 16 weeks were defined as “nonresponders” (NR). DDX43 Expression (qPCR) was higher in nonresponders (P = 0.045). Liver metastasis biopsy samples (3 R and 3 NR) were immunoblotted for DDX43, pERK and RAS proteins with consistent results.\", \"disease\"=>{\"id\"=>188, \"name\"=>\"Uveal Melanoma\", \"display_name\"=>\"Uveal Melanoma\", \"doid\"=>\"6039\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:6039\"}, \"drugs\"=>[{\"id\"=>19, \"name\"=>\"Trametinib\", \"pubchem_id\"=>nil}, {\"id\"=>29, \"name\"=>\"PD0325901\", \"pubchem_id\"=>nil}, {\"id\"=>122, \"name\"=>\"Selumetinib (AZD6244)\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>786, \"name\"=>\"Overexpression of DDX43 mediates MEK inhibitor resistance through RAS Upregulation in uveal melanoma cells.\", \"citation\"=>\"Ambrosini et al., 2014, Mol. Cancer Ther.\", \"pubmed_id\"=>\"24899684\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24899684\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>8}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>481, \"phenotypes\"=>[]}]}, {\"id\"=>483, \"entrez_name\"=>\"TBK1\", \"entrez_id\"=>29110, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>11322, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"12\", \"start\"=>64845660, \"stop\"=>64895888, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000331710.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/11322/summary/variants/483/summary#variant\", \"evidence_items\"=>[{\"id\"=>1138, \"name\"=>\"EID1138\", \"description\"=>\"Preclinical study in melanoma cell lines. TBK1 depletion (siRNA) inhibited migration and invasion, whereas its constitutive overexpression led to an increase in invasion. In MEK-resistant NRAS mutant melanoma cell lines, TBK1 depletion or inhibition (AZ909) in combination with MEK inhibitors (selumetinib) promoted apoptosis in 3D culture.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>122, \"name\"=>\"Selumetinib (AZD6244)\", \"pubchem_id\"=>nil}, {\"id\"=>283, \"name\"=>\"AZ909\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>788, \"name\"=>\"Targeting TBK1 inhibits migration and resistance to MEK inhibitors in mutant NRAS melanoma.\", \"citation\"=>\"Vu et al., 2014, Mol. Cancer Res.\", \"pubmed_id\"=>\"24962318\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24962318\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4482471\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>10}, \"journal\"=>\"Mol. Cancer Res.\", \"full_journal_title\"=>\"Molecular cancer research : MCR\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>483, \"phenotypes\"=>[]}]}, {\"id\"=>486, \"entrez_name\"=>\"HAVCR2\", \"entrez_id\"=>84868, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>16394, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.5, \"coordinates\"=>{\"chromosome\"=>\"5\", \"start\"=>156512843, \"stop\"=>156536725, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000307851.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/16394/summary/variants/486/summary#variant\", \"evidence_items\"=>[{\"id\"=>1151, \"name\"=>\"EID1151\", \"description\"=>\"In a preclinical study in an immunocompetent mouse model of lung adenocarcinoma, upregulation of alternative immune checkpoints, notably T-cell immunoglobulin mucin-3 (TIM-3, aka HAVCR2) was observed after progression under PD-1 antibody. A survival advantage was observed with the addition of a TIM-3 blocking antibody.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>289, \"name\"=>\"PD-1 Inhibitor\", \"pubchem_id\"=>nil}, {\"id\"=>290, \"name\"=>\"TIM-3 Inhibitor\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>794, \"name\"=>\"Adaptive resistance to therapeutic PD-1 blockade is associated with upregulation of alternative immune checkpoints.\", \"citation\"=>\"Koyama et al., 2016, Nat Commun\", \"pubmed_id\"=>\"26883990\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26883990\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4757784\", \"publication_date\"=>{\"year\"=>2016}, \"journal\"=>\"Nat Commun\", \"full_journal_title\"=>\"Nature communications\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>486, \"phenotypes\"=>[]}, {\"id\"=>1150, \"name\"=>\"EID1150\", \"description\"=>\"In a preclinical study in an immunocompetent mouse model of lung adenocarcinoma, upregulation of alternative immune checkpoints, notably T-cell immunoglobulin mucin-3 (TIM-3, aka HAVCR2) was observed after progression under PD-1 antibody. A survival advantage was observed with the addition of a TIM-3 blocking antibody. Two patients who developed adaptive resistance to anti-PD-1 treatment also showed TIM-3 upregulation in blocking antibody-bound T cells at treatment failure.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>289, \"name\"=>\"PD-1 Inhibitor\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>794, \"name\"=>\"Adaptive resistance to therapeutic PD-1 blockade is associated with upregulation of alternative immune checkpoints.\", \"citation\"=>\"Koyama et al., 2016, Nat Commun\", \"pubmed_id\"=>\"26883990\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26883990\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4757784\", \"publication_date\"=>{\"year\"=>2016}, \"journal\"=>\"Nat Commun\", \"full_journal_title\"=>\"Nature communications\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>486, \"phenotypes\"=>[]}]}, {\"id\"=>487, \"entrez_name\"=>\"MMP2\", \"entrez_id\"=>4313, \"name\"=>\"SERUM LEVELS\", \"description\"=>\"\", \"gene_id\"=>3549, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>25.0, \"coordinates\"=>{\"chromosome\"=>\"16\", \"start\"=>55512883, \"stop\"=>55540603, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000219070.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3549/summary/variants/487/summary#variant\", \"evidence_items\"=>[{\"id\"=>1156, \"name\"=>\"EID1156\", \"description\"=>\"MMP2 serum levels were assessed (ELISA) at baseline (b) and before surgery in 45 available samples from patients with inflammatory breast cancer treated with bevacizumab-based neoadjuvant therapy. High bMMP2 was correlated with better disease-free survival (DFS, p=0.001) and overall survival (OS, p=0.032). Correlation to DFS was also found after multivariate analysis (p=0.003, Hazard Ratio [HR]: 0.115). bMMP2 was associated with relapse (p=0.002) and death (p=0.049). During treatment, significant increase in MMP2 was observed in 100% of patients.\", \"disease\"=>{\"id\"=>1550, \"name\"=>\"Inflammatory Breast Carcinoma\", \"display_name\"=>\"Inflammatory Breast Carcinoma\", \"doid\"=>\"6263\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:6263\"}, \"drugs\"=>[{\"id\"=>33, \"name\"=>\"Bevacizumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>796, \"name\"=>\"MMP2 and MMP9 serum levels are associated with favorable outcome in patients with inflammatory breast cancer treated with bevacizumab-based neoadjuvant chemotherapy in the BEVERLY-2 study.\", \"citation\"=>\"Tabouret et al., 2016, Oncotarget\", \"pubmed_id\"=>\"26921265\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26921265\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4951307\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>5}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>487, \"phenotypes\"=>[]}, {\"id\"=>1158, \"name\"=>\"EID1158\", \"description\"=>\"Plasma levels of 11 potential biomarkers were tested at baseline and 2 weeks after bevacizumab initiation in a prospective cohort of 26 patients with recurrent high-grade gliomas. Results were validated in a separate retrospective cohort (n = 50) and also tested in patients treated with cytotoxic agents without bevacizumab (n = 34). MMP2 level was correlated with PFS (hazard ratio, 3.92; 95% confidence interval [CI]:1.46-10.52; P = .007) and OS (hazard ratio, 4.62; 95% CI: 1.58-13.53; P = .005) in the discovery and validation cohorts but not in patients treated without bevacizumab.\", \"disease\"=>{\"id\"=>98, \"name\"=>\"Brain Glioma\", \"display_name\"=>\"Brain Glioma\", \"doid\"=>\"0060108\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060108\"}, \"drugs\"=>[{\"id\"=>33, \"name\"=>\"Bevacizumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>797, \"name\"=>\"Association of matrix metalloproteinase 2 plasma level with response and survival in patients treated with bevacizumab for recurrent high-grade glioma.\", \"citation\"=>\"Tabouret et al., 2014, Neuro-oncology\", \"pubmed_id\"=>\"24327581\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24327581\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3922517\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>3}, \"journal\"=>\"Neuro-oncology\", \"full_journal_title\"=>\"Neuro-oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>487, \"phenotypes\"=>[]}]}, {\"id\"=>488, \"entrez_name\"=>\"MMP9\", \"entrez_id\"=>4318, \"name\"=>\"SERUM LEVELS\", \"description\"=>\"\", \"gene_id\"=>3553, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"20\", \"start\"=>44637547, \"stop\"=>44645200, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000372330.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3553/summary/variants/488/summary#variant\", \"evidence_items\"=>[{\"id\"=>1157, \"name\"=>\"EID1157\", \"description\"=>\"MMP9 serum levels were assessed (ELISA) at baseline (b) and before surgery in 45 available samples from patients with inflammatory breast cancer treated with bevacizumab-based neoadjuvant therapy. Low bMMP9 was correlated with better OS (p=0.022) and non-significantly associated with longer DFS (p=0.071). Correlation to DFS was also found after multivariate analysis (p=0.041, HR: 3.511). bMMP9 was associated with death (p=0.035). During treatment, a significant decrease in MMP9 levels (p<0.001) was observed in 87% of patients.\", \"disease\"=>{\"id\"=>1550, \"name\"=>\"Inflammatory Breast Carcinoma\", \"display_name\"=>\"Inflammatory Breast Carcinoma\", \"doid\"=>\"6263\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:6263\"}, \"drugs\"=>[{\"id\"=>33, \"name\"=>\"Bevacizumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>796, \"name\"=>\"MMP2 and MMP9 serum levels are associated with favorable outcome in patients with inflammatory breast cancer treated with bevacizumab-based neoadjuvant chemotherapy in the BEVERLY-2 study.\", \"citation\"=>\"Tabouret et al., 2016, Oncotarget\", \"pubmed_id\"=>\"26921265\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26921265\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4951307\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>5}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>488, \"phenotypes\"=>[]}]}, {\"id\"=>490, \"entrez_name\"=>\"HSPH1\", \"entrez_id\"=>10808, \"name\"=>\"NUCLEAR EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>8614, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>11.0, \"coordinates\"=>{\"chromosome\"=>\"13\", \"start\"=>31710765, \"stop\"=>31736064, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000320027.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/8614/summary/variants/490/summary#variant\", \"evidence_items\"=>[{\"id\"=>1162, \"name\"=>\"EID1162\", \"description\"=>\"HSP110 (HSPH1) expression was immunohistochemically assessed using a tissue microarray in 210 gastric cancer patients. The overall survival rate was significantly lower in patients with high nuclear expression (N=121, 2+ moderate/3+ strong staining) than patients with low nuclear staining (N=89, 0 no/1+ weak staining)(P = 0.0169). Multivariate regression analysis identified expression of nuclear HSP110 as an independent prognostic factor for gastric cancer outcome (P = 0.0068). This was not found for cytoplasmic expression.\", \"disease\"=>{\"id\"=>147, \"name\"=>\"Gastric Adenocarcinoma\", \"display_name\"=>\"Gastric Adenocarcinoma\", \"doid\"=>\"3717\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3717\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>801, \"name\"=>\"Nuclear heat shock protein 110 expression is associated with poor prognosis and chemotherapy resistance in gastric cancer.\", \"citation\"=>\"Kimura et al., 2016, Oncotarget\", \"pubmed_id\"=>\"26943774\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26943774\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4951298\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>5}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>490, \"phenotypes\"=>[]}, {\"id\"=>1163, \"name\"=>\"EID1163\", \"description\"=>\"49 patients gastric cancer patients treated with adjuvant chemotherapy were immunohistochemically assessed for nuclear HSPH1 (HSP110) expression. Overall survival rate in the high expression group (2+ moderate/3+ strong staining) was significantly lower compared with the low expression group (0 no/1+ weak staining) (P = 0.0364). In gastric cancer cell lines (MKN7, MKN45), siRNA mediated HSP110 knockdown led to a higher sensitivity to 5-FU and cisplatin compared with the parent and control cells (p < 0.05).\", \"disease\"=>{\"id\"=>147, \"name\"=>\"Gastric Adenocarcinoma\", \"display_name\"=>\"Gastric Adenocarcinoma\", \"doid\"=>\"3717\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3717\"}, \"drugs\"=>[{\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}, {\"id\"=>395, \"name\"=>\"5-fluorouracil\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>801, \"name\"=>\"Nuclear heat shock protein 110 expression is associated with poor prognosis and chemotherapy resistance in gastric cancer.\", \"citation\"=>\"Kimura et al., 2016, Oncotarget\", \"pubmed_id\"=>\"26943774\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26943774\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4951298\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>5}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>490, \"phenotypes\"=>[]}]}, {\"id\"=>493, \"entrez_name\"=>\"ROBO4\", \"entrez_id\"=>54538, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>12367, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"11\", \"start\"=>124753587, \"stop\"=>124768177, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000306534.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/12367/summary/variants/493/summary#variant\", \"evidence_items\"=>[{\"id\"=>1166, \"name\"=>\"EID1166\", \"description\"=>\"Biomarkers for Response to antiangiogenic treatment were screened for in hepatocellular cancer (n = 11), renal cell cancer (n = 7) and non-small cell lung cancer (n = 2). Increased expression of tumor endothelial markers: Robo4 and ECSCR, nonsignificantly correlated with prolonged antiangiogenic time to progression (ECSCR high = 275 days vs. ECSCR low = 92.5 days; p = 0.07 and for Robo4 high = 387 days vs. Robo4 low = 90.0 days; p = 0.08).\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>295, \"name\"=>\"Antiangiogenic Therapy\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>803, \"name\"=>\"Biomarkers of evasive resistance predict disease progression in cancer patients treated with antiangiogenic therapies.\", \"citation\"=>\"Pircher et al., 2016, Oncotarget\", \"pubmed_id\"=>\"26956051\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26956051\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>12}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>493, \"phenotypes\"=>[]}]}, {\"id\"=>494, \"entrez_name\"=>\"PIM1\", \"entrez_id\"=>5292, \"name\"=>\"NUCLEAR EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>4286, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>37137979, \"stop\"=>37143202, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000373509.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4286/summary/variants/494/summary#variant\", \"evidence_items\"=>[{\"id\"=>1168, \"name\"=>\"EID1168\", \"description\"=>\"The expression of PIM1, p-STAT3, and c-MYC was tested (IHC) in 194 NSCLC archival tumor samples. High nuclear expression of PIM1 was detected in 43.3% of cases and was significantly correlated with advanced clinical stage (p<0.001) and poor pathologic differentiation (P = 0.004) and was an independent unfavorable prognostic factors for overall and disease-free survival (OS, P=0.009; DFS, P=0.034).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>805, \"name\"=>\"Association of Nuclear PIM1 Expression with Lymph Node Metastasis and Poor Prognosis in Patients with Lung Adenocarcinoma and Squamous Cell Carcinoma.\", \"citation\"=>\"Jiang et al., 2016, J Cancer\", \"pubmed_id\"=>\"26918046\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26918046\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4747887\", \"publication_date\"=>{\"year\"=>2016}, \"journal\"=>\"J Cancer\", \"full_journal_title\"=>\"Journal of Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>494, \"phenotypes\"=>[]}]}, {\"id\"=>498, \"entrez_name\"=>\"MIR218-1\", \"entrez_id\"=>407000, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>24938, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>20529898, \"stop\"=>20530007, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000384999.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/24938/summary/variants/498/summary#variant\", \"evidence_items\"=>[{\"id\"=>1184, \"name\"=>\"EID1184\", \"description\"=>\"The study found mir-218 was significantly under-expressed when compared to matched adjacent normal tissue in 107 samples via qPCR. Further, low expression of mir-218 was revealed to be associated with lower overall survival (p < .001), and multivariate analysis revealed that low expression of mir-218 as an independent predictor of poor prognosis (p < .007).\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>817, \"name\"=>\"Reduced miRNA-218 expression in pancreatic cancer patients as a predictor of poor prognosis.\", \"citation\"=>\"Li et al., 2015, Genet. Mol. Res.\", \"pubmed_id\"=>\"26662432\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26662432\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"Genet. Mol. Res.\", \"full_journal_title\"=>\"Genetics and molecular research : GMR\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>498, \"phenotypes\"=>[]}]}, {\"id\"=>525, \"entrez_name\"=>\"B4GALT1\", \"entrez_id\"=>2683, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>2239, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>33110635, \"stop\"=>33167354, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000379731.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/2239/summary/variants/525/summary#variant\", \"evidence_items\"=>[{\"id\"=>1278, \"name\"=>\"EID1278\", \"description\"=>\"438 patients with non-metastatic clear-cell renal carcinoma were were retrospectively analysed for B4GALT1 expression via IHC. High B4GALT1 expression was associated with poor overall survival (P<0.001 and P<0.001 in training and validation cohort) and identified as an independent adverse prognostic factor for survival (P=0.007 and P=0.002).\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>883, \"name\"=>\"Increased B4GALT1 expression associates with adverse outcome in patients with non-metastatic clear cell renal cell carcinoma.\", \"citation\"=>\"Xie et al., 2016, Oncotarget\", \"pubmed_id\"=>\"27092876\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27092876\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>15}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>525, \"phenotypes\"=>[]}]}, {\"id\"=>529, \"entrez_name\"=>\"SIRT1\", \"entrez_id\"=>23411, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>9538, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>16.0, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>69644427, \"stop\"=>69678147, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000212015.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/9538/summary/variants/529/summary#variant\", \"evidence_items\"=>[{\"id\"=>1294, \"name\"=>\"EID1294\", \"description\"=>\"In-vitro study with MiaPaCa-2 and PANC-1 pancreatic cancer cell lines. Inhibition of Sirt1 by increasing concentrations of nicotinamide led to a decrease in viability. Gefitinib treatment with concentrations of 50 μM showed similar effects as observed for the application of 25 mM nicotinamide. Combinatory treatment with gefitinib and nicotinamide showed a synergistic effect on cell viability. In cell lines overexpressing Sirt1, higher concentrations of nicotinamide were required to abrogate increased cell viability mediated by Sirt1 overexpression compared to controls.\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>324, \"name\"=>\"Nicotinamide\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>889, \"name\"=>\"High SIRT1 expression is a negative prognosticator in pancreatic ductal adenocarcinoma.\", \"citation\"=>\"Stenzinger et al., 2013, BMC Cancer\", \"pubmed_id\"=>\"24088390\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24088390\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3850795\", \"publication_date\"=>{\"year\"=>2013}, \"journal\"=>\"BMC Cancer\", \"full_journal_title\"=>\"BMC cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>529, \"phenotypes\"=>[]}, {\"id\"=>1293, \"name\"=>\"EID1293\", \"description\"=>\"Nuclear Sirt1 expression was identified by IHC in 36 (27.9%) of 129 pancreatic ductal adenocarcinoma cases. SIRT1 expression was associated with poor survival in univariate (p = 0.002) and multivariate (HR 1.65, p = 0.045) analyses. Overexpression of Sirt1 in pancreas cancer cell lines MiaPaCa-2 and PANC-1 led to increased cell viability.\", \"disease\"=>{\"id\"=>145, \"name\"=>\"Pancreatic Ductal Carcinoma\", \"display_name\"=>\"Pancreatic Ductal Carcinoma\", \"doid\"=>\"3587\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3587\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>889, \"name\"=>\"High SIRT1 expression is a negative prognosticator in pancreatic ductal adenocarcinoma.\", \"citation\"=>\"Stenzinger et al., 2013, BMC Cancer\", \"pubmed_id\"=>\"24088390\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24088390\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3850795\", \"publication_date\"=>{\"year\"=>2013}, \"journal\"=>\"BMC Cancer\", \"full_journal_title\"=>\"BMC cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>529, \"phenotypes\"=>[]}]}, {\"id\"=>533, \"entrez_name\"=>\"SMAD4\", \"entrez_id\"=>4089, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>77, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"18\", \"start\"=>48556583, \"stop\"=>48611409, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000342988.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/77/summary/variants/533/summary#variant\", \"evidence_items\"=>[{\"id\"=>1306, \"name\"=>\"EID1306\", \"description\"=>\"In 249 patients that have undergone a pancreaticoduodenectomy (Whipple) for the resection of pancreatic ductal adenocarcinoma, expression of SMAD4 is associated with favorable prognosis. SMAD4 was assessed by IHC in 56 of 249 (22%).  Relative hazard of mortality for cancers lacking SMAD4 after adjusting for other prognostic factors was 1.36 (95% confidence interval, 1.01-1.83; P = 0.04).\", \"disease\"=>{\"id\"=>141, \"name\"=>\"Pancreatic Ductal Adenocarcinoma\", \"display_name\"=>\"Pancreatic Ductal Adenocarcinoma\", \"doid\"=>\"3498\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3498\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>899, \"name\"=>\"The SMAD4 protein and prognosis of pancreatic ductal adenocarcinoma.\", \"citation\"=>\"Tascilar et al., 2001, Clin. Cancer Res.\", \"pubmed_id\"=>\"11751510\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/11751510\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2001, \"month\"=>12}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>533, \"phenotypes\"=>[]}]}, {\"id\"=>537, \"entrez_name\"=>\"RB1\", \"entrez_id\"=>5925, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>4795, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"13\", \"start\"=>48877911, \"stop\"=>49056122, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000267163.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4795/summary/variants/537/summary#variant\", \"evidence_items\"=>[{\"id\"=>1313, \"name\"=>\"EID1313\", \"description\"=>\"Positive RB1 expression (2+ by IHC) was associated with adverse outcomes in a cohort of 73 cases with stage I and II NSCLC (P<0.001) in this retrospective study. This finding was independent from other genomic and clinical factors explored by the authors.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>904, \"name\"=>\"Altered p16(INK4) and RB1 Expressions Are Associated with Poor Prognosis in Patients with Nonsmall Cell Lung Cancer.\", \"citation\"=>\"Zhao et al., 2012, J Oncol\", \"pubmed_id\"=>\"22619677\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22619677\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3350860\", \"publication_date\"=>{\"year\"=>2012}, \"journal\"=>\"J Oncol\", \"full_journal_title\"=>\"Journal of oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>537, \"phenotypes\"=>[]}]}, {\"id\"=>539, \"entrez_name\"=>\"PTPN11\", \"entrez_id\"=>5781, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>4685, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"12\", \"start\"=>112856718, \"stop\"=>112947717, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000351677.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4685/summary/variants/539/summary#variant\", \"evidence_items\"=>[{\"id\"=>1316, \"name\"=>\"EID1316\", \"description\"=>\"Study explored the expression of the PTPN11 protein (SHP2) and found that expression was increased in a cohort of 112 laryngeal cancer samples. Expression was also associated with poor survival (p=.02) and in vitro models showed SHP2 expression promoted tumor growth.\", \"disease\"=>{\"id\"=>623, \"name\"=>\"Laryngeal Carcinoma\", \"display_name\"=>\"Laryngeal Carcinoma\", \"doid\"=>\"2600\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2600\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>906, \"name\"=>\"SHP2 promotes laryngeal cancer growth through the Ras/Raf/Mek/Erk pathway and serves as a prognostic indicator for laryngeal cancer.\", \"citation\"=>\"Gu et al., 2014, Int. J. Oncol.\", \"pubmed_id\"=>\"24297342\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24297342\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>2}, \"journal\"=>\"Int. J. Oncol.\", \"full_journal_title\"=>\"International journal of oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>539, \"phenotypes\"=>[]}]}, {\"id\"=>546, \"entrez_name\"=>\"ACTA1\", \"entrez_id\"=>58, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>128, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>229566992, \"stop\"=>229569841, \"reference_bases\"=>nil, \"variant_bases\"=>\"\", \"representative_transcript\"=>\"ENST00000366684.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>\"\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/128/summary/variants/546/summary#variant\", \"evidence_items\"=>[{\"id\"=>1161, \"name\"=>\"EID1161\", \"description\"=>\"ActA level was measured in serum and plasma samples of 64/87 lung adenocarcinoma patients and 46/66 age- and gender-matched controls (ELISA). ActA Levels were higher in patients than in controls and were significantly correlated with disease stage and metastasis. Patients were stratified as having high or low ActA levels using the median expression level. LADC patients with high serum or plasma ActA levels had significantly shorter OS than those with low circulating ActA concentrations (p < 0.0001 in both) and were identified as an independent prognostic factor (p=0.004 and p=0.002, respectively).\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>800, \"name\"=>\"High circulating activin A level is associated with tumor progression and predicts poor prognosis in lung adenocarcinoma.\", \"citation\"=>\"Hoda et al., 2016, Oncotarget\", \"pubmed_id\"=>\"26950277\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26950277\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4924649\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>3, \"day\"=>22}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>546, \"phenotypes\"=>[]}]}, {\"id\"=>547, \"entrez_name\"=>\"RB1\", \"entrez_id\"=>5925, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>4795, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"13\", \"start\"=>48877911, \"stop\"=>49056122, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000267163.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4795/summary/variants/547/summary#variant\", \"evidence_items\"=>[{\"id\"=>1324, \"name\"=>\"EID1324\", \"description\"=>\"In Capan-1 cell lines engineered to reduce pRB expression (by transfection with p16), resensitization to chemotherapy-induced apoptosis was observed. The effect was asserted to be specific to pRB depletion because, in two separate cell lines, also transfected with p16, that retained high pRB expression, chemotherapy resistance was still observed.\", \"disease\"=>{\"id\"=>158, \"name\"=>\"Pancreatic Adenocarcinoma\", \"display_name\"=>\"Pancreatic Adenocarcinoma\", \"doid\"=>\"4074\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4074\"}, \"drugs\"=>[{\"id\"=>68, \"name\"=>\"Doxorubicin\", \"pubchem_id\"=>nil}, {\"id\"=>85, \"name\"=>\"Gemcitabine\", \"pubchem_id\"=>nil}, {\"id\"=>325, \"name\"=>\"Mitomycin C\", \"pubchem_id\"=>nil}, {\"id\"=>395, \"name\"=>\"5-fluorouracil\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>913, \"name\"=>\"Overexpression of pRB in human pancreatic carcinoma cells: function in chemotherapy-induced apoptosis.\", \"citation\"=>\"Plath et al., 2002, J. Natl. Cancer Inst.\", \"pubmed_id\"=>\"11792751\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/11792751\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2002, \"month\"=>1, \"day\"=>16}, \"journal\"=>\"J. Natl. Cancer Inst.\", \"full_journal_title\"=>\"Journal of the National Cancer Institute\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>547, \"phenotypes\"=>[]}]}, {\"id\"=>578, \"entrez_name\"=>\"BIRC5\", \"entrez_id\"=>332, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>355, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"17\", \"start\"=>76210267, \"stop\"=>76221717, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000301633.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/355/summary/variants/578/summary#variant\", \"evidence_items\"=>[{\"id\"=>1449, \"name\"=>\"EID1449\", \"description\"=>\"13 HER2 breast cancer patients were treated with trastuzumab and then trastuzumab + docetaxel prior to surgery. Pretreatment biopsies and post-treatment surgical samples were obtained, and microarray performed. 5 of 13 patients responded to treatment, and pretreatment biopsies from non-responders showed significantly higher survivin (BIRC5) mRNA (p=0.026) levels than responders. In HER2 positive, trastuzumab-sensitive BT474 cells, overexpression of survivin blunted sensitivity to trastuzumab. In HER2 positive trastuzumab-resistant breast cancer cell lines, knockdown of survivin with RNAi or with YM155 induced growth inhibition and apoptosis.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>977, \"name\"=>\"Trastuzumab-resistant cells rely on a HER2-PI3K-FoxO-survivin axis and are sensitive to PI3K inhibitors.\", \"citation\"=>\"Chakrabarty et al., 2013, Cancer Res.\", \"pubmed_id\"=>\"23204226\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23204226\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3563941\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>2, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>578, \"phenotypes\"=>[]}]}, {\"id\"=>597, \"entrez_name\"=>\"RET\", \"entrez_id\"=>5979, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>42, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>7.5, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>43572475, \"stop\"=>43625799, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000355710.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/42/summary/variants/597/summary#variant\", \"evidence_items\"=>[{\"id\"=>2992, \"name\"=>\"EID2992\", \"description\"=>\"The activity of Vandetanib was tested in vivo in three PDX models of TNBC and one model of HER2+ BC with different expression levels of RET and EGFR. Vandetanib induced tumor regression in PDX models with high expression of RET or EGFR. The effect was associated with inhibition of RET/EGFR phosphorylation and MAP kinase pathway and increased necrosis. In a PDX model with no expression of RET nor EGFR, Vandetanib slowed tumor growth without inducing tumor regression. In addition, treatment by Vandetanib decreased expression of murine Vegf receptors and the endothelial marker Cd31 in the four PDX models tested, suggesting inhibition of tumor vascularization.\", \"disease\"=>{\"id\"=>365, \"name\"=>\"Estrogen-receptor Negative Breast Cancer\", \"display_name\"=>\"Estrogen-receptor Negative Breast Cancer\", \"doid\"=>\"0060076\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060076\"}, \"drugs\"=>[{\"id\"=>388, \"name\"=>\"Vandetanib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>488, \"name\"=>\"Vandetanib as a potential new treatment for estrogen receptor-negative breast cancers.\", \"citation\"=>\"Hatem et al., 2016, Int. J. Cancer\", \"pubmed_id\"=>\"26686064\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26686064\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>5, \"day\"=>15}, \"journal\"=>\"Int. J. Cancer\", \"full_journal_title\"=>\"International journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>597, \"phenotypes\"=>[]}, {\"id\"=>1518, \"name\"=>\"EID1518\", \"description\"=>\"A 78-year-old man presented with oral papillary adenocarcinoma and lung metastases that were refractory to standard treatment.  Tumor sequencing showed increased MAPK activity due to RET over-expression and PTEN deletion, therefore sunitinib (RET inhibitor) was initiated.  After 28 days post-administration, the lung metastases decreased in size by 22% (28mm to 21mm) and no new metastatic lesions had appeared.  After 4 months, the patient grew resistant to treatment.\", \"disease\"=>{\"id\"=>706, \"name\"=>\"Papillary Adenocarcinoma\", \"display_name\"=>\"Papillary Adenocarcinoma\", \"doid\"=>\"3112\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3112\"}, \"drugs\"=>[{\"id\"=>157, \"name\"=>\"Sunitinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1011, \"name\"=>\"Evolution of an adenocarcinoma in response to selection by targeted kinase inhibitors.\", \"citation\"=>\"Jones et al., 2010, Genome Biol.\", \"pubmed_id\"=>\"20696054\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20696054\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2945784\", \"publication_date\"=>{\"year\"=>2010}, \"journal\"=>\"Genome Biol.\", \"full_journal_title\"=>\"Genome biology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>597, \"phenotypes\"=>[]}]}, {\"id\"=>602, \"entrez_name\"=>\"CDK6\", \"entrez_id\"=>1021, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>12, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>22.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>92234235, \"stop\"=>92463231, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000265734.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/12/summary/variants/602/summary#variant\", \"evidence_items\"=>[{\"id\"=>1526, \"name\"=>\"EID1526\", \"description\"=>\"In two cohorts (n=45 and n=46) of ER+ breast cancer patients who had received fulvestrant, CDK6 overexpression correlated with shorter median time to progression in both cohorts (2.5 vs. 8.2 months and 3.4 vs. 8.9 months for CDK6 overexpression vs. normal expression). CDK6 expression did not correlate with other lines of treatment (N=68, tamoxifen or endocrine therapy). Four fulvestrant-resistant cell lines were derived from estrogen responsive and fulvestrant sensitive parental breast cancer cell line MCF-7/SO.5. Gene expression profiling showed upregulation of CDK6 in the fulvestrant resistant derivatives. Knockdown of CDK6 in these derivative cells resulted in fulvestrant sensitivity.\", \"disease\"=>{\"id\"=>28, \"name\"=>\"Estrogen-receptor Positive Breast Cancer\", \"display_name\"=>\"Estrogen-receptor Positive Breast Cancer\", \"doid\"=>\"0060075\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060075\"}, \"drugs\"=>[{\"id\"=>54, \"name\"=>\"Fulvestrant\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1018, \"name\"=>\"High CDK6 protects cells from fulvestrant-mediated apoptosis and is a predictor of resistance to fulvestrant in estrogen receptor-positive metastatic breast cancer.\", \"citation\"=>\"Alves et al., 2016, Clin. Cancer Res.\", \"pubmed_id\"=>\"27252418\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27252418\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>6, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>602, \"phenotypes\"=>[]}, {\"id\"=>1527, \"name\"=>\"EID1527\", \"description\"=>\"Four fulvestrant (estrogen receptor degrader) resistant cell lines were derived from parent fulvestrant sensitive ER+ MCF-7/SO.5 breast cancer cell line. Derivative cell lines showed some sensitivity to CDK4/6 inhibitor palbociclib, and increased growth inhibition when treated with fulvestrant and palbociclib together.\", \"disease\"=>{\"id\"=>28, \"name\"=>\"Estrogen-receptor Positive Breast Cancer\", \"display_name\"=>\"Estrogen-receptor Positive Breast Cancer\", \"doid\"=>\"0060075\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060075\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}, {\"id\"=>54, \"name\"=>\"Fulvestrant\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1018, \"name\"=>\"High CDK6 protects cells from fulvestrant-mediated apoptosis and is a predictor of resistance to fulvestrant in estrogen receptor-positive metastatic breast cancer.\", \"citation\"=>\"Alves et al., 2016, Clin. Cancer Res.\", \"pubmed_id\"=>\"27252418\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27252418\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>6, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>602, \"phenotypes\"=>[]}]}, {\"id\"=>603, \"entrez_name\"=>\"FLT3\", \"entrez_id\"=>2322, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>24, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>7.5, \"coordinates\"=>{\"chromosome\"=>\"13\", \"start\"=>28577411, \"stop\"=>28674729, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000241453.7\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/24/summary/variants/603/summary#variant\", \"evidence_items\"=>[{\"id\"=>1528, \"name\"=>\"EID1528\", \"description\"=>\"A 25-year-old Caucasian male presented with a second relapse of B-ALL. RNA sequencing revealed FLT3 overexpression (also confirmed by IHC). The patient was treated with 22 days of sunitinib, permitting the patient to undergo a bone marrow transplant. A bone marrow biopsy showed complete clearance by variant allele frequency and the patient remains in remission 4-years post matched-unrelated donor allograft.\", \"disease\"=>{\"id\"=>2135, \"name\"=>\"Adult B-Lymphoblastic Leukemia\", \"display_name\"=>\"Adult B-Lymphoblastic Leukemia\", \"doid\"=>\"0060592\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060592\"}, \"drugs\"=>[{\"id\"=>157, \"name\"=>\"Sunitinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1019, \"name\"=>\"Comprehensive genomic analysis reveals FLT3 activation and a therapeutic strategy for a patient with relapsed adult B-lymphoblastic leukemia.\", \"citation\"=>\"Griffith et al., 2016, Exp. Hematol.\", \"pubmed_id\"=>\"27181063\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27181063\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4914477\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>7}, \"journal\"=>\"Exp. Hematol.\", \"full_journal_title\"=>\"Experimental hematology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>603, \"phenotypes\"=>[]}]}, {\"id\"=>607, \"entrez_name\"=>\"ESR1\", \"entrez_id\"=>2099, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>21, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>152128686, \"stop\"=>152424406, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000206249.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/21/summary/variants/607/summary#variant\", \"evidence_items\"=>[{\"id\"=>1541, \"name\"=>\"EID1541\", \"description\"=>\"A phase 2 clinical trial evaluated 165 patients with oestrogen receptor positive and HER2-negative breast cancer such that 84 patients received combination therapy of palbociclib plus letrozole and 81 patients received letrozole alone. The combination therapy arm had higher median duration of progression-free survival when compared to the letrozole only group (20.2 months vs. 10.2 months; p<0.0004). A subcategory of patients taking combination therapy, who lacked mutations in p16 or CCND1 (n=34), had an even higher median free survival of 26.1 months.  Noticeable side effects in the combination therapy group included neutropenia (54%) leucopenia (19%), fatigue (4%) back pain (2%), diarrhea (2%) and pulmonary embolism (4%).\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}, {\"id\"=>375, \"name\"=>\"Letrozole\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1024, \"name\"=>\"The cyclin-dependent kinase 4/6 inhibitor palbociclib in combination with letrozole versus letrozole alone as first-line treatment of oestrogen receptor-positive, HER2-negative, advanced breast cancer (PALOMA-1/TRIO-18): a randomised phase 2 study.\", \"citation\"=>\"Finn et al., 2015, Lancet Oncol.\", \"pubmed_id\"=>\"25524798\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25524798\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>1}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>607, \"phenotypes\"=>[]}]}, {\"id\"=>619, \"entrez_name\"=>\"PTGS2\", \"entrez_id\"=>5743, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>4659, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>186640923, \"stop\"=>186649559, \"reference_bases\"=>nil, \"variant_bases\"=>\"\", \"representative_transcript\"=>\"ENST00000367468.5\", \"chromosome2\"=>\"\", \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>\"\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4659/summary/variants/619/summary#variant\", \"evidence_items\"=>[{\"id\"=>1573, \"name\"=>\"EID1573\", \"description\"=>\"Mail-in surveys conducted every two years since 1980 evaluated 82,911 woman and 47,363 men for regular aspirin use.  In this cohort, aspirin use was associated with a lower relative risk of colorectal cancer (RR=0.73; 95% CI 0.69-0.87; p<0.001).  Immunohistochemical analysis was employed on a subset of 636 cases to evaluate COX2 overexpression.  Of these tumors, 67% had moderate or strong COX2 expression and 33% had weak or absent COX-2 expression.  In tumors with COX-2 overexpression, regular aspirin use conferred significant risk reduction of colorectal cancer (RR=0.64; 95%CI 0.52-0.78) however, in tumors with reduced COX2 expression, aspirin use had no influence on colorectal cancer incidence (RR=0.96, 95%CI 0.73-1.26).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>379, \"name\"=>\"Aspirin\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1039, \"name\"=>\"Aspirin and the risk of colorectal cancer in relation to the expression of COX-2.\", \"citation\"=>\"Chan et al., 2007, N. Engl. J. Med.\", \"pubmed_id\"=>\"17522398\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17522398\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>5, \"day\"=>24}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>619, \"phenotypes\"=>[]}]}, {\"id\"=>621, \"entrez_name\"=>\"MET\", \"entrez_id\"=>4233, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>52, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>64.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>116312459, \"stop\"=>116436396, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000318493.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/52/summary/variants/621/summary#variant\", \"evidence_items\"=>[{\"id\"=>3038, \"name\"=>\"EID3038\", \"description\"=>\"In this study the authors screened for growth factors that elicited resistance in newly characterized metastatic uveal melanoma cell lines to clinical-grade MEK inhibitors, trametinib and selumetinib, showing that neuregulin 1 (NRG1) and hepatocyte growth factor (HGF) provide resistance to MEK inhibition. Trametinib enhances the responsiveness to NRG1 and sustained HGF-mediated activation of AKT. Targeting ERBB3 and cMET could overcome resistance to trametinib.\", \"disease\"=>{\"id\"=>188, \"name\"=>\"Uveal Melanoma\", \"display_name\"=>\"Uveal Melanoma\", \"doid\"=>\"6039\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:6039\"}, \"drugs\"=>[{\"id\"=>19, \"name\"=>\"Trametinib\", \"pubchem_id\"=>nil}, {\"id\"=>122, \"name\"=>\"Selumetinib (AZD6244)\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1742, \"name\"=>\"Paracrine Effect of NRG1 and HGF Drives Resistance to MEK Inhibitors in Metastatic Uveal Melanoma.\", \"citation\"=>\"Cheng et al., 2015, Cancer Res.\", \"pubmed_id\"=>\"25952648\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25952648\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4490069\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>621, \"phenotypes\"=>[]}, {\"id\"=>5835, \"name\"=>\"EID5835\", \"description\"=>\"In this randomised, double-blind, placebo-controlled phase 3 trial, 609 patients with MET+ (expression of at least 1+ intensity on 25% or more tumor cells) gastric cancer or gastroesophageal adenocarcinoma were randomized to chemotherapy (epirubicin, cisplatin, capecitabine) with placebo or MET inhibitor rilotumumab. Study treatment was stopped early because of a higher number of deaths in the rilotumumab arm. Median overall survival was 8·8 months (95% CI 7·7-10·2) in the rilotumumab group compared with 10·7 months (9·6-12·4) in the placebo group (stratified hazard ratio 1·34, 95% CI 1·10-1·63; p=0·003).\", \"disease\"=>{\"id\"=>1322, \"name\"=>\"Stomach Carcinoma\", \"display_name\"=>\"Stomach Carcinoma\", \"doid\"=>\"5517\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5517\"}, \"drugs\"=>[{\"id\"=>546, \"name\"=>\"Rilotumumab\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2361, \"name\"=>\"Rilotumumab plus epirubicin, cisplatin, and capecitabine as first-line therapy in advanced MET-positive gastric or gastro-oesophageal junction cancer (RILOMET-1): a randomised, double-blind, placebo-controlled, phase 3 trial.\", \"citation\"=>\"Catenacci et al., 2017, Lancet Oncol.\", \"pubmed_id\"=>\"28958504\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28958504\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>9, \"day\"=>25}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>621, \"phenotypes\"=>[]}, {\"id\"=>3037, \"name\"=>\"EID3037\", \"description\"=>\"MEK inhibitors improve survival in cutaneous melanoma patients but show only modest efficacy in metastatic uveal melanoma patients. In this study, we screened for growth factors that elicited resistance in newly characterized metastatic uveal melanoma cell lines to clinical-grade MEK inhibitors, trametinib and selumetinib. We show that neuregulin 1 (NRG1) and hepatocyte growth factor (HGF) provide resistance to MEK inhibition. Individually targeting ERBB3 and cMET, the receptors for NRG1 and HGF, respectively, overcome resistance to trametinib provided by these growth factors and by conditioned medium from fibroblasts that produce NRG1 and HGF. Together, these results provide preclinical evidence for the use of MEK inhibitors in combination with clinical-grade anti-ERBB3 or anti-cMET monoclonal antibodies in metastatic uveal melanoma.\", \"disease\"=>{\"id\"=>188, \"name\"=>\"Uveal Melanoma\", \"display_name\"=>\"Uveal Melanoma\", \"doid\"=>\"6039\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:6039\"}, \"drugs\"=>[{\"id\"=>512, \"name\"=>\"MET Inhibition, ErbB3 Inhibition\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1742, \"name\"=>\"Paracrine Effect of NRG1 and HGF Drives Resistance to MEK Inhibitors in Metastatic Uveal Melanoma.\", \"citation\"=>\"Cheng et al., 2015, Cancer Res.\", \"pubmed_id\"=>\"25952648\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25952648\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4490069\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>621, \"phenotypes\"=>[]}, {\"id\"=>1584, \"name\"=>\"EID1584\", \"description\"=>\"FFPE tumours from patients enrolled in the BRIM2 (n = 59) and BRIM3 (n = 150) trials of vemurafenib in advanced BRAF mutant (V600E/K) melanoma were analyzed for MET expression (IHC). Pretreatment MET expression was frequent at the ≥1 + cutoff (BRIM3, 31%; BRIM2, 49%), but relatively infrequent at the ≥2 + cutoff (BRIM3, 9%; BRIM2, 19%). Retrospective subset analyses showed that, irrespective of the cutoff used or the treatment arm, MET expression did not show prognostic significance, in terms of objective response rate, progression-free survival, or overall survival.\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>4, \"name\"=>\"Vemurafenib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1047, \"name\"=>\"Impact of MET expression on outcome in BRAF(V600E/K) advanced melanoma.\", \"citation\"=>\"Jubb et al., 2013, Histopathology\", \"pubmed_id\"=>\"23802768\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23802768\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>9}, \"journal\"=>\"Histopathology\", \"full_journal_title\"=>\"Histopathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>621, \"phenotypes\"=>[]}, {\"id\"=>1581, \"name\"=>\"EID1581\", \"description\"=>\"Whole exome sequencing, transcriptome and methylome alterations in 48 melanoma samples with acquired resistance to BRAFi +/- MEKi therapy compared to patient-matched baseline melanoma tissues were analyzed. 39% of resistant melanomas were not accounted for by any validated mutational mechanism. c-MET was overexpressed in 21 of 48 (44%) resistant samples. Methylome analysis revealed three CpG clusters (C1–3) with differential methylation to be negatively correlated with differential c-MET mRNA expression. In nearly all (90%) pairwise comparisons of tumors and cell lines, differential c-MET mRNA expression could be accounted for by at least one differential CpG cluster methylation (Δβ ≥ 10% and FDR adjusted p ≤ 0.05). In contrast, only 48% displayed a concordant differential expression pattern of at least one c-MET transcription factor. In an independent study, c-MET was overexpressed in 8 of 35 (23%) MAPKi-resistant tumors from 7 of 26 (27%) patients. In two human BRAF V600E mutant cell lines, BRAFi treatment led to progressive methylation-expression changes akin to observations across MAPKi-sensitive versus resistant tumors.\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>4, \"name\"=>\"Vemurafenib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1044, \"name\"=>\"Non-genomic and Immune Evolution of Melanoma Acquiring MAPKi Resistance.\", \"citation\"=>\"Hugo et al., 2015, Cell\", \"pubmed_id\"=>\"26359985\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26359985\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4821508\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>9, \"day\"=>10}, \"journal\"=>\"Cell\", \"full_journal_title\"=>\"Cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>621, \"phenotypes\"=>[]}, {\"id\"=>1583, \"name\"=>\"EID1583\", \"description\"=>\"c-MET and ligand HGF were expressed by 6 uveal melanoma cell lines in-vitro. GNA-mutant cell lines (n=3) showed stronger expression of ligand and receptor as well as more receptor phosphorylation. SiRNA mediated MET-knockdown reduced cell migration but not cell growth. Crizotinib treatment led to a decrease in cell growth and migration. At 25 nmol/L of crizotinib, only the migration of G-protein mutant cells lines and not G-protein wild type cell lines was decreased. Crizotinib did not lead to a reduction in tumor volume of two cell lines in vivo but reduced the formation of macrometastases (luciferase activity, p=0.03).\", \"disease\"=>{\"id\"=>188, \"name\"=>\"Uveal Melanoma\", \"display_name\"=>\"Uveal Melanoma\", \"doid\"=>\"6039\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:6039\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1046, \"name\"=>\"Crizotinib, a c-Met inhibitor, prevents metastasis in a metastatic uveal melanoma model.\", \"citation\"=>\"Surriga et al., 2013, Mol. Cancer Ther.\", \"pubmed_id\"=>\"24140933\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24140933\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>12}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>621, \"phenotypes\"=>[]}, {\"id\"=>1582, \"name\"=>\"EID1582\", \"description\"=>\"Preclinical study in uveal melanoma cell lines. c-MET was identified as a transcriptional target of miR-144. miR-144 was downregulated in 4 uveal melanoma cell lines and 5 human uveal melanoma tumor tissues compared to normal controls. siRNA mediated c-MET downregulation reduced profileration and invasion of uveal melanoma cell lines. Overexpression of c-Met promoted cell proliferation and invasion in uveal melanoma.\", \"disease\"=>{\"id\"=>188, \"name\"=>\"Uveal Melanoma\", \"display_name\"=>\"Uveal Melanoma\", \"doid\"=>\"6039\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:6039\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"E\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"rejected\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1045, \"name\"=>\"MiR-144 Inhibits Uveal Melanoma Cell Proliferation and Invasion by Regulating c-Met Expression.\", \"citation\"=>\"Sun et al., 2015, PLoS ONE\", \"pubmed_id\"=>\"25961751\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25961751\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4427317\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>621, \"phenotypes\"=>[]}]}, {\"id\"=>624, \"entrez_name\"=>\"MKI67\", \"entrez_id\"=>4288, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>3529, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>35.0, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>129894923, \"stop\"=>129924649, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000368654.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3529/summary/variants/624/summary#variant\", \"evidence_items\"=>[{\"id\"=>1594, \"name\"=>\"EID1594\", \"description\"=>\"293 cases of clinically localized prostate cancer were analyzed on needle-biopsy FFPE tissue microarray, using the MIB-1 antibody against the Ki67 marker of proliferation. IHC staining was used to generate a Ki67 score, and cases were divided into 2 groups with Ki67 score greater or less than 10%. Primary endpoint was death due to prostate cancer. A hazard ratio for high Ki67 values of 3.42 was seen in univariate analysis, and  Ki67 score also added significant predictive value in multivariate analysis, with a hazard ratio of 2.78.\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1056, \"name\"=>\"Prognostic value of Ki-67 for prostate cancer death in a conservatively managed cohort.\", \"citation\"=>\"Fisher et al., 2013, Br. J. Cancer\", \"pubmed_id\"=>\"23329234\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23329234\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3566811\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>2, \"day\"=>5}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>624, \"phenotypes\"=>[]}, {\"id\"=>1593, \"name\"=>\"EID1593\", \"description\"=>\"A quantitative meta-analysis was performed to assess optimal IHC-calculated cutoff values for the prognostic use of proliferative biomarker Ki67 in breast cancer. The studies included ER positive and negative cases, and the MIB-1 antibody was used in the large majority of studies. 25 studies that employed multivariate analysis were used, and a hazard ratio of 2.05 for overall survival was obtained when Ki67 cutoff value was set at greater or equal to 25%.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1055, \"name\"=>\"Prognostic value of different cut-off levels of Ki-67 in breast cancer: a systematic review and meta-analysis of 64,196 patients.\", \"citation\"=>\"Petrelli et al., 2015, Breast Cancer Res. Treat.\", \"pubmed_id\"=>\"26341751\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26341751\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>10}, \"journal\"=>\"Breast Cancer Res. Treat.\", \"full_journal_title\"=>\"Breast cancer research and treatment\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>624, \"phenotypes\"=>[]}]}, {\"id\"=>625, \"entrez_name\"=>\"RAD23B\", \"entrez_id\"=>5887, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>4760, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>110045560, \"stop\"=>110094475, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000358015.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4760/summary/variants/625/summary#variant\", \"evidence_items\"=>[{\"id\"=>1597, \"name\"=>\"EID1597\", \"description\"=>\"Western blot analyses were performed for HR23b (RAD23B) expression sarcoma cell lines after treatment with vorinostat, belinostat, mocetinostat and entinostat. All HDACi were able to regulate proliferation and apoptosis in vitro. Sensitivity to vorinostat correlated significantly with HR23b protein expression. IHC in 523 patient samples revealed high HR23b expression in 12.5% of sarcomas (among them malignant peripheral nerve sheath tumours, pleomorphic liposarcomas, leiomyosarcomas, dedifferentiated liposarcomas, synovial sarcomas and angiosarcomas) and 23.2% of GIST.\", \"disease\"=>{\"id\"=>105, \"name\"=>\"Sarcoma\", \"display_name\"=>\"Sarcoma\", \"doid\"=>\"1115\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1115\"}, \"drugs\"=>[{\"id\"=>314, \"name\"=>\"Vorinostat (SAHA)\", \"pubchem_id\"=>\"\"}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1059, \"name\"=>\"HR23b expression is a potential predictive biomarker for HDAC inhibitor treatment in mesenchymal tumours and is associated with response to vorinostat.\", \"citation\"=>\"Angelika Ihle et al., 2016, J Pathol Clin Res\", \"pubmed_id\"=>\"27499916\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27499916\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4907056\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>4}, \"journal\"=>\"J Pathol Clin Res\", \"full_journal_title\"=>\"The journal of pathology. Clinical research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>625, \"phenotypes\"=>[]}]}, {\"id\"=>632, \"entrez_name\"=>\"RB1\", \"entrez_id\"=>5925, \"name\"=>\"PHOSPHORYLATION\", \"description\"=>\"\", \"gene_id\"=>4795, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"13\", \"start\"=>48877911, \"stop\"=>49056122, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000267163.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4795/summary/variants/632/summary#variant\", \"evidence_items\"=>[{\"id\"=>1609, \"name\"=>\"EID1609\", \"description\"=>\"The p110alpha-specific PI3K inhibitor BYL719 was used on a panel of five PI3K inhibitor sensitive and five resistant breast cancer cell lines. Sensitive cells (IC50<400nM) showed reduced phospho-Rb (p-Rb) western blot levels in response to BYL719 treatment, while resistant cell lines (IC50>800nM) showed significantly less reduction in p-Rb after BYL719. Paired biopsies before and after treatment were taken from a set of eight patients classified as either responder or non-responder to BYL719. Responder samples showed significantly reduced p-Rb staining after treatment in comparison to non-responder biopsies. In biopsies taken from a group of three responders who progressed, p-Rb levels were increased after progression.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>387, \"name\"=>\"Alpelisib (NVP-BYL719)\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>178, \"name\"=>\"CDK 4/6 inhibitors sensitize PIK3CA mutant breast cancer to PI3K inhibitors.\", \"citation\"=>\"Vora et al., 2014, Cancer Cell\", \"pubmed_id\"=>\"25002028\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25002028\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4155598\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>7, \"day\"=>14}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>632, \"phenotypes\"=>[]}]}, {\"id\"=>639, \"entrez_name\"=>\"JUN\", \"entrez_id\"=>3725, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>3094, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>59246465, \"stop\"=>59249785, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000371222.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3094/summary/variants/639/summary#variant\", \"evidence_items\"=>[{\"id\"=>1633, \"name\"=>\"EID1633\", \"description\"=>\"A 67-year-old female presents with recurrent metastatic colonic adenocarcinoma.  Immunohistochemical and genetic workup showed overexpression of c-JUN and FOS, which presumably signals through the RAAS pathway. The patient was treated with irbesartan and after 5 weeks of therapy, the CEA value decreased from 18 to 3.1 and there was radiological resolution of the disease.  Her 10 month CEA was 1.4.\", \"disease\"=>{\"id\"=>57, \"name\"=>\"Colorectal Adenocarcinoma\", \"display_name\"=>\"Colorectal Adenocarcinoma\", \"doid\"=>\"0050861\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050861\"}, \"drugs\"=>[{\"id\"=>390, \"name\"=>\"Irbesartan\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1081, \"name\"=>\"Response to angiotensin blockade with irbesartan in a patient with metastatic colorectal cancer.\", \"citation\"=>\"Jones et al., 2016, Ann. Oncol.\", \"pubmed_id\"=>\"27022066\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27022066\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4843189\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>5}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>639, \"phenotypes\"=>[]}]}, {\"id\"=>640, \"entrez_name\"=>\"FOS\", \"entrez_id\"=>2353, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>1955, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"14\", \"start\"=>75745477, \"stop\"=>75748933, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000303562.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1955/summary/variants/640/summary#variant\", \"evidence_items\"=>[{\"id\"=>1634, \"name\"=>\"EID1634\", \"description\"=>\"A 67-year-old female presents with recurrent metastatic colonic adenocarcinoma.  Immunohistochemical and genetic workup showed overexpression of c-JUN and FOS, which presumably signals through the RAAS pathway. The patient was treated with Irbesartan and after 5 weeks of therapy, the CEA value decreased from 18 to 3.1 and there was radiological resolution of the disease.  Her 10 month CEA was 1.4.\", \"disease\"=>{\"id\"=>120, \"name\"=>\"Colon Adenocarcinoma\", \"display_name\"=>\"Colon Adenocarcinoma\", \"doid\"=>\"234\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:234\"}, \"drugs\"=>[{\"id\"=>390, \"name\"=>\"Irbesartan\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1081, \"name\"=>\"Response to angiotensin blockade with irbesartan in a patient with metastatic colorectal cancer.\", \"citation\"=>\"Jones et al., 2016, Ann. Oncol.\", \"pubmed_id\"=>\"27022066\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27022066\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4843189\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>5}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>640, \"phenotypes\"=>[]}]}, {\"id\"=>648, \"entrez_name\"=>\"ATRX\", \"entrez_id\"=>546, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>525, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>19.0, \"coordinates\"=>{\"chromosome\"=>\"X\", \"start\"=>76760356, \"stop\"=>77041702, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000373344.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/525/summary/variants/648/summary#variant\", \"evidence_items\"=>[{\"id\"=>5832, \"name\"=>\"EID5832\", \"description\"=>\"Loss of the chromatin-remodeling protein ATRX associates with ALT in cancers. Inhibition of the protein kinase ATR, a critical regulator of recombination recruited by RPA, disrupts ALT and triggers chromosome fragmentation and apoptosis in ALT cells. The cell death induced by ATR inhibitors is highly selective for cancer cells that rely on ALT.\", \"disease\"=>{\"id\"=>134, \"name\"=>\"Osteosarcoma\", \"display_name\"=>\"Osteosarcoma\", \"doid\"=>\"3347\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3347\"}, \"drugs\"=>[{\"id\"=>544, \"name\"=>\"VE-821\", \"pubchem_id\"=>nil}, {\"id\"=>545, \"name\"=>\"AZ20\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2358, \"name\"=>\"Alternative lengthening of telomeres renders cancer cells hypersensitive to ATR inhibitors.\", \"citation\"=>\"Flynn et al., 2015, Science\", \"pubmed_id\"=>\"25593184\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25593184\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4358324\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>1, \"day\"=>16}, \"journal\"=>\"Science\", \"full_journal_title\"=>\"Science (New York, N.Y.)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>648, \"phenotypes\"=>[]}, {\"id\"=>1648, \"name\"=>\"EID1648\", \"description\"=>\"In a glioblastoma mouse model induced by NRAS and p53 knockdown, ATRX loss was associated with a decreased median survival (69 days vs. 84 days; P = .0032). Also, the tumors grew to a larger size at earlier time points. ATRX loss was also associated with greater microsatellite instability.\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1094, \"name\"=>\"ATRX loss promotes tumor growth and impairs nonhomologous end joining DNA repair in glioma.\", \"citation\"=>\"Koschmann et al., 2016, Sci Transl Med\", \"pubmed_id\"=>\"26936505\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26936505\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>3, \"day\"=>2}, \"journal\"=>\"Sci Transl Med\", \"full_journal_title\"=>\"Science translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>648, \"phenotypes\"=>[]}, {\"id\"=>1647, \"name\"=>\"EID1647\", \"description\"=>\"A retrospective tumor sample study found that the loss of ATRX expression (less than 10% of nuclei) in IDH-mutant astrocytomas treated with temozolomide or a combination of procarbazine, lomustine and vincristine was associated with a greater time to treatment failure (55.6 months vs. 31.8 months; P = .0168) and a greater progression-free survival (37.1 months vs. 18.1 months; P = .038).\", \"disease\"=>{\"id\"=>35, \"name\"=>\"Astrocytoma\", \"display_name\"=>\"Astrocytoma\", \"doid\"=>\"3069\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3069\"}, \"drugs\"=>[{\"id\"=>11, \"name\"=>\"Temozolomide\", \"pubchem_id\"=>\"\"}, {\"id\"=>396, \"name\"=>\"Procarbazine\", \"pubchem_id\"=>nil}, {\"id\"=>397, \"name\"=>\"Lomustine\", \"pubchem_id\"=>nil}, {\"id\"=>398, \"name\"=>\"Vincristine\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>nil, \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1093, \"name\"=>\"ATRX loss refines the classification of anaplastic gliomas and identifies a subgroup of IDH mutant astrocytic tumors with better prognosis.\", \"citation\"=>\"Wiestler et al., 2013, Acta Neuropathol.\", \"pubmed_id\"=>\"23904111\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23904111\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>9}, \"journal\"=>\"Acta Neuropathol.\", \"full_journal_title\"=>\"Acta neuropathologica\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>648, \"phenotypes\"=>[]}]}, {\"id\"=>651, \"entrez_name\"=>\"AXL\", \"entrez_id\"=>558, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>533, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"19\", \"start\"=>41725108, \"stop\"=>41767671, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000301178.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/533/summary/variants/651/summary#variant\", \"evidence_items\"=>[{\"id\"=>1655, \"name\"=>\"EID1655\", \"description\"=>\"Immunohistochemical analysis of tumors from 116 esophageal squamous cell carcinoma found that expression of AXL was associated with increased progression of the tumor (P < 0.0001), increased risk of death (HR = 2.09 ; P = 0.028), and increased risk of distant metastasis (OR = 3.96 ; P = 0.029).\", \"disease\"=>{\"id\"=>149, \"name\"=>\"Esophagus Squamous Cell Carcinoma\", \"display_name\"=>\"Esophagus Squamous Cell Carcinoma\", \"doid\"=>\"3748\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3748\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1119, \"name\"=>\"The AXL receptor tyrosine kinase is associated with adverse prognosis and distant metastasis in esophageal squamous cell carcinoma.\", \"citation\"=>\"Hsieh et al., 2016, Oncotarget\", \"pubmed_id\"=>\"27172793\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27172793\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5095051\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>6, \"day\"=>14}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>651, \"phenotypes\"=>[]}]}, {\"id\"=>652, \"entrez_name\"=>\"GAS6\", \"entrez_id\"=>2621, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>2186, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>23.0, \"coordinates\"=>{\"chromosome\"=>\"13\", \"start\"=>114523524, \"stop\"=>114567046, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000357389.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/2186/summary/variants/652/summary#variant\", \"evidence_items\"=>[{\"id\"=>1656, \"name\"=>\"EID1656\", \"description\"=>\"Prostate Cancer (PCa) cell lines were either cultured alone, co-cultured with osteoblasts expressing GAS6, or cultured in the presence of media with GAS6 to examine their response to therapy with docetaxel. Docetaxel efficiently induced apoptosis in the cells cultured without any GAS6, but it was not able to do so in the cells that were in media with GAS6 present or that were co-cultured with osteoblasts expressing GAS6. PCa cells in the presence of GAS6 were found to be in G1 phase for a greater duration which may help explain the increased resistance to docetaxel.\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[{\"id\"=>73, \"name\"=>\"Docetaxel\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1120, \"name\"=>\"Growth Arrest-Specific 6 (GAS6) Promotes Prostate Cancer Survival by G1 Arrest/S Phase Delay and Inhibition of Apoptosis During Chemotherapy in Bone Marrow.\", \"citation\"=>\"Lee et al., 2016, J. Cell. Biochem.\", \"pubmed_id\"=>\"27153245\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27153245\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5223280\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>12}, \"journal\"=>\"J. Cell. Biochem.\", \"full_journal_title\"=>\"Journal of cellular biochemistry\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>652, \"phenotypes\"=>[]}, {\"id\"=>1657, \"name\"=>\"EID1657\", \"description\"=>\"128 patients with oral squamous cell carcinoma (OSCC) and 145 normal control subjects were evaluated for the concentration of GAS6 in their plasma serum. OSCC patients were found to be associated with a significantly higher level of GAS6 in serum ( P = .05 ). Furthermore, within the OSCC patients, increased GAS6 serum concentration was associated with higher stage tumors ( P < .01) , poorly differentiated tumors ( P < .01 ), and increased lymph node metastasis ( OR = 2.79 ; CI = 1.72-4.48 ).\", \"disease\"=>{\"id\"=>58, \"name\"=>\"Oral Squamous Cell Carcinoma\", \"display_name\"=>\"Oral Squamous Cell Carcinoma\", \"doid\"=>\"0050866\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050866\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1121, \"name\"=>\"Elevated Serum Gas6 Is a Novel Prognostic Biomarker in Patients with Oral Squamous Cell Carcinoma.\", \"citation\"=>\"Jiang et al., 2015, PLoS ONE\", \"pubmed_id\"=>\"26207647\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26207647\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4514879\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>652, \"phenotypes\"=>[]}]}, {\"id\"=>653, \"entrez_name\"=>\"ETV4\", \"entrez_id\"=>2118, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>1767, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"17\", \"start\"=>41605212, \"stop\"=>41623800, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000319349.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1767/summary/variants/653/summary#variant\", \"evidence_items\"=>[{\"id\"=>5121, \"name\"=>\"EID5121\", \"description\"=>\"In vitro studies in KRAS G12C lung cancer cells (CALU1) showed that overexpression of ETV4 modulated resistance to trametinib.\", \"disease\"=>{\"id\"=>23, \"name\"=>\"Lung Cancer\", \"display_name\"=>\"Lung Cancer\", \"doid\"=>\"1324\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1324\"}, \"drugs\"=>[{\"id\"=>19, \"name\"=>\"Trametinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2207, \"name\"=>\"ATXN1L, CIC, and ETS Transcription Factors Modulate Sensitivity to MAPK Pathway Inhibition.\", \"citation\"=>\"Wang et al., 2017, Cell Rep\", \"pubmed_id\"=>\"28178529\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28178529\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5313047\", \"publication_date\"=>{\"year\"=>2017, \"month\"=>2, \"day\"=>7}, \"journal\"=>\"Cell Rep\", \"full_journal_title\"=>\"Cell reports\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>653, \"phenotypes\"=>[]}, {\"id\"=>5119, \"name\"=>\"EID5119\", \"description\"=>\"In vitro studies in KRAS G12V pancreatic cancer cells (PATU8902) showed that overexpression of ETV4 reduced sensitivity to trametinib.\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>19, \"name\"=>\"Trametinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2207, \"name\"=>\"ATXN1L, CIC, and ETS Transcription Factors Modulate Sensitivity to MAPK Pathway Inhibition.\", \"citation\"=>\"Wang et al., 2017, Cell Rep\", \"pubmed_id\"=>\"28178529\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28178529\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5313047\", \"publication_date\"=>{\"year\"=>2017, \"month\"=>2, \"day\"=>7}, \"journal\"=>\"Cell Rep\", \"full_journal_title\"=>\"Cell reports\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>653, \"phenotypes\"=>[]}, {\"id\"=>1658, \"name\"=>\"EID1658\", \"description\"=>\"Immunohistochemical methods were used to evaluate the expression levels of ETV4 in prostate cancer (PCa) tumor samples. Increased expression of ETV4 was associated with greater Gleason score (P = 0.045) , pathological tumor stage (P = 0.041),  and an unfavorable prognosis (P = 0.040).\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1122, \"name\"=>\"Overexpression of ETV4 is associated with poor prognosis in prostate cancer: involvement of uPA/uPAR and MMPs.\", \"citation\"=>\"Qi et al., 2015, Tumour Biol.\", \"pubmed_id\"=>\"25544710\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25544710\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>5}, \"journal\"=>\"Tumour Biol.\", \"full_journal_title\"=>\"Tumour biology : the journal of the International Society for Oncodevelopmental Biology and Medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>653, \"phenotypes\"=>[]}]}, {\"id\"=>654, \"entrez_name\"=>\"STAG3\", \"entrez_id\"=>10734, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>8552, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>4.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>99775186, \"stop\"=>99812003, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000426455.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/8552/summary/variants/654/summary#variant\", \"evidence_items\"=>[{\"id\"=>1660, \"name\"=>\"EID1660\", \"description\"=>\"A375 melanoma cells containing an inducible shRNA for STAG3 were grown as xenograft tumors in nude mice. The mice in which STAG3 was silenced had tumors that were more sensitive to vemurafenib-induced tumor shrinkage than the tumors of the control mice (P < .01).\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>4, \"name\"=>\"Vemurafenib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1123, \"name\"=>\"Loss of cohesin complex components STAG2 or STAG3 confers resistance to BRAF inhibition in melanoma.\", \"citation\"=>\"Shen et al., 2016, Nat. Med.\", \"pubmed_id\"=>\"27500726\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27500726\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5014622\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>9}, \"journal\"=>\"Nat. Med.\", \"full_journal_title\"=>\"Nature medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>654, \"phenotypes\"=>[]}]}, {\"id\"=>660, \"entrez_name\"=>\"HIF1A\", \"entrez_id\"=>3091, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>2596, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"14\", \"start\"=>62164340, \"stop\"=>62214489, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000539097.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/2596/summary/variants/660/summary#variant\", \"evidence_items\"=>[{\"id\"=>1674, \"name\"=>\"EID1674\", \"description\"=>\"Immunohistochemical staining was performed on 65 patients for HIF1A such that 23 were considered responders and 42 were considered non-responders.  There was no difference between the two groups with regards to overall response rate to pazopanib.  There was no difference between the two groups with regards to progression free survival.\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[{\"id\"=>56, \"name\"=>\"Pazopanib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"N/A\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1132, \"name\"=>\"The role of aberrant VHL/HIF pathway elements in predicting clinical outcome to pazopanib therapy in patients with metastatic clear-cell renal cell carcinoma.\", \"citation\"=>\"Choueiri et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"23881929\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23881929\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4522695\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>9, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>660, \"phenotypes\"=>[]}]}, {\"id\"=>662, \"entrez_name\"=>\"MLH1\", \"entrez_id\"=>4292, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>3532, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>37034823, \"stop\"=>37092409, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000231790.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3532/summary/variants/662/summary#variant\", \"evidence_items\"=>[{\"id\"=>1680, \"name\"=>\"EID1680\", \"description\"=>\"46 patients with poorly cohesive, signet-cell ring, gastric carcinoma were evaluated for genetic mutations and MLH1 expression (immunohistochemistry). Overall survival rate of MLH1 negative cases were significantly better than MLH1 positive cases (p=0.020).\", \"disease\"=>{\"id\"=>1322, \"name\"=>\"Stomach Carcinoma\", \"display_name\"=>\"Stomach Carcinoma\", \"doid\"=>\"5517\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5517\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1136, \"name\"=>\"Comprehensive mutation profiling of mucinous gastric carcinoma.\", \"citation\"=>\"Rokutan et al., 2016, J. Pathol.\", \"pubmed_id\"=>\"27313181\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27313181\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>10}, \"journal\"=>\"J. Pathol.\", \"full_journal_title\"=>\"The Journal of pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>662, \"phenotypes\"=>[]}]}, {\"id\"=>670, \"entrez_name\"=>\"MYCL\", \"entrez_id\"=>4610, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>3738, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>1.0, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>40361098, \"stop\"=>40367685, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000397332.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3738/summary/variants/670/summary#variant\", \"evidence_items\"=>[{\"id\"=>1706, \"name\"=>\"EID1706\", \"description\"=>\"Four myeloma cell lines were treated with two different BET inhibitors (I-BET151 or JQ1). Proliferation of three cell lines (KMS11, RPMI8226 and MM1S) but not U266 (only partial inhibition) was strongly inhibited by the BET inhibitors. Microarray analysis showed a downregulation of MYC-dependent genes after treatment. qRT-PCR showed that MYCL1 but not c-MYC and MYCN was expressed in U266 cells and I-BET151 induced downregulation of MYCL1. In contrast, c-MYC but not MYCL was expressed in the other cell lines and I-BET151 induced downregulation of c-MYC.\", \"disease\"=>{\"id\"=>41, \"name\"=>\"Multiple Myeloma\", \"display_name\"=>\"Multiple Myeloma\", \"doid\"=>\"9538\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9538\"}, \"drugs\"=>[{\"id\"=>306, \"name\"=>\"JQ1\", \"pubchem_id\"=>nil}, {\"id\"=>402, \"name\"=>\"I-BET151\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1149, \"name\"=>\"Antimyeloma activity of bromodomain inhibitors on the human myeloma cell line U266 by downregulation of MYCL.\", \"citation\"=>\"Suzuki et al., 2016, Anticancer Drugs\", \"pubmed_id\"=>\"27276402\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27276402\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4969056\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>9}, \"journal\"=>\"Anticancer Drugs\", \"full_journal_title\"=>\"Anti-cancer drugs\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>670, \"phenotypes\"=>[]}]}, {\"id\"=>671, \"entrez_name\"=>\"HGF\", \"entrez_id\"=>3082, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>2591, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>81328322, \"stop\"=>81399514, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000222390.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/2591/summary/variants/671/summary#variant\", \"evidence_items\"=>[{\"id\"=>1707, \"name\"=>\"EID1707\", \"description\"=>\"Three metastatic glioblastoma cell lines were compared with their parental cell lines with regard to differential gene expression. HGF signalling was among the top pathways identified. Exogenous HFG led to an increase in MET activity (except in one cell line with high endogenous HGF levels) which was abrogated by MET inhibitor SGX523. Three HGF autocrine cell lines were grown in mice and showed significantly increased sensitivity to SGX523 than three non-HGF autocrine cell lines (P<0.05).\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[{\"id\"=>403, \"name\"=>\"SGX523\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1150, \"name\"=>\"Hepatocyte growth factor (HGF) autocrine activation predicts sensitivity to MET inhibition in glioblastoma.\", \"citation\"=>\"Xie et al., 2012, Proc. Natl. Acad. Sci. U.S.A.\", \"pubmed_id\"=>\"22203985\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22203985\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3258605\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>1, \"day\"=>10}, \"journal\"=>\"Proc. Natl. Acad. Sci. U.S.A.\", \"full_journal_title\"=>\"Proceedings of the National Academy of Sciences of the United States of America\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>671, \"phenotypes\"=>[]}]}, {\"id\"=>673, \"entrez_name\"=>\"FGF2\", \"entrez_id\"=>2247, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>1872, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>4.0, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>123747863, \"stop\"=>123819391, \"reference_bases\"=>nil, \"variant_bases\"=>\"\", \"representative_transcript\"=>\"ENST00000264498.3\", \"chromosome2\"=>\"\", \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>\"\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1872/summary/variants/673/summary#variant\", \"evidence_items\"=>[{\"id\"=>1711, \"name\"=>\"EID1711\", \"description\"=>\"AML MOLM14 cells were cocultured with FLT3 inhibitor AC220 (10 nM) and selected proteins expressed in the bone marrow microenvironment. FL and FGF2 were the only proteins that increased viability >2 standard deviations. FGFR inhibitor PD173074 (and FGFR1 siRNA) was able to attenuate the protective effect of FGF2.  Removing FL or FGF2 from ligand-dependent resistant cultures transiently restored sensitivity to AC220, but accelerated acquisition of secondary resistance mutations. 10 FLT3-ITD AML patients treated with AC220 developed increased FGF2 expression (IHC) in marrow stromal cells, which peaked prior to overt clinical relapse and detection of resistance mutations\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[{\"id\"=>266, \"name\"=>\"Quizartinib (AC220)\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1154, \"name\"=>\"FGF2 from Marrow Microenvironment Promotes Resistance to FLT3 Inhibitors in Acute Myeloid Leukemia.\", \"citation\"=>\"Traer et al., 2016, Cancer Res.\", \"pubmed_id\"=>\"27671675\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27671675\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>9, \"day\"=>26}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>673, \"phenotypes\"=>[]}]}, {\"id\"=>683, \"entrez_name\"=>\"PAX8\", \"entrez_id\"=>7849, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>75, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>40.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>113973574, \"stop\"=>114036527, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000429538.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/75/summary/variants/683/summary#variant\", \"evidence_items\"=>[{\"id\"=>1716, \"name\"=>\"EID1716\", \"description\"=>\"The authors explored the viability of PAX8 expression as a biomarker for differentiating mullerian from non-mullerian tumors. 74 tumors (37 Mullerian/37 non-Mullerian) were examined for PAX8 expression via IHC. Of these 29/37 Mullerian and 1/37 non-Mullerian tumors exhibited PAX8 expression. The sensitivity and specificity of using this as a biomarker to differentiate between the two tumor types was 78.4% and 97.3% respectively.\", \"disease\"=>{\"id\"=>454, \"name\"=>\"Female Reproductive Organ Cancer\", \"display_name\"=>\"Female Reproductive Organ Cancer\", \"doid\"=>\"120\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:120\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1159, \"name\"=>\"Diagnostic utility of PAX8 in differentiation of mullerian from non-mullerian tumors.\", \"citation\"=>\"Heidarpour et al., 2014, Adv Biomed Res\", \"pubmed_id\"=>\"24800185\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24800185\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4007335\", \"publication_date\"=>{\"year\"=>2014}, \"journal\"=>\"Adv Biomed Res\", \"full_journal_title\"=>\"Advanced biomedical research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>683, \"phenotypes\"=>[]}]}, {\"id\"=>693, \"entrez_name\"=>\"SGK1\", \"entrez_id\"=>6446, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>5225, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>24.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>134490384, \"stop\"=>134639196, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000367858.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5225/summary/variants/693/summary#variant\", \"evidence_items\"=>[{\"id\"=>1731, \"name\"=>\"EID1731\", \"description\"=>\"The alpelisib (PI3Kalpha inhibitor) resistant breast cancer cell line HCC1954 was treated with alpelsisb alone or in combination with the SGK1 kinase inhibitor SGK1-Inh. Addition of SGK1 reduced viability over alpelisib alone, and phospho-S6K as well as phospho-4EBP1 levels decreased with combination treatment. HCC1954 xenografts showed increased tumor volume with treatment of alpelisib or SGK1-Inh alone, but only combination treatment resulted in lack of volume increase and decreased phospho-S6K levels in tumors by IHC.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>339, \"name\"=>\"BYL719 (Alpelisib)\", \"pubchem_id\"=>\"\"}, {\"id\"=>408, \"name\"=>\"SGK1-Inh\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1174, \"name\"=>\"PDK1-SGK1 Signaling Sustains AKT-Independent mTORC1 Activation and Confers Resistance to PI3Kα Inhibition.\", \"citation\"=>\"Castel et al., 2016, Cancer Cell\", \"pubmed_id\"=>\"27451907\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27451907\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4982440\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>8, \"day\"=>8}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>693, \"phenotypes\"=>[]}, {\"id\"=>1729, \"name\"=>\"EID1729\", \"description\"=>\"SGK1 mRNA expression was elevated in five breast cancer cell lines which were resistant to PI3Kalpha inhibitor alpelisib in comparison to five sensitive cell lines. Western blots showed clear elevation of protein levels in 3/5 resistant cell lines. Ectopic SGK1 expression in sensitive MDA-MB-361 cells resulted in alpelisib resistance and preservation of phopho-S6K and phospho-4EBP with alpelisib treatment. Knockdown of SGK1 in resistant HCC1954 cells increased alpelisib sensitivity and reduced p-S6K and p-4EBP levels with alpelisib treatment. Promoter methylation at SGK1 was increased in sensitive cells compared to alpelisib resistant cell lines.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>387, \"name\"=>\"Alpelisib (NVP-BYL719)\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1174, \"name\"=>\"PDK1-SGK1 Signaling Sustains AKT-Independent mTORC1 Activation and Confers Resistance to PI3Kα Inhibition.\", \"citation\"=>\"Castel et al., 2016, Cancer Cell\", \"pubmed_id\"=>\"27451907\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27451907\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4982440\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>8, \"day\"=>8}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>693, \"phenotypes\"=>[]}, {\"id\"=>1730, \"name\"=>\"EID1730\", \"description\"=>\"A group of 18 breast cancer patients were treated with PI3Kalpha inhibitor alpelisib plus an aromatase inhbitor (NCT01870505). 3 out of 3 patients expressing high tumor SGK1 mRNA had disease progression as best response with median progression free survival (PFS) of 56 days, while 15 patients classified with low to medium SGK1 mRNA had median PFS 111 days with 4/15 having disease progression as best response.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>387, \"name\"=>\"Alpelisib (NVP-BYL719)\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1174, \"name\"=>\"PDK1-SGK1 Signaling Sustains AKT-Independent mTORC1 Activation and Confers Resistance to PI3Kα Inhibition.\", \"citation\"=>\"Castel et al., 2016, Cancer Cell\", \"pubmed_id\"=>\"27451907\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27451907\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4982440\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>8, \"day\"=>8}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>693, \"phenotypes\"=>[]}]}, {\"id\"=>695, \"entrez_name\"=>\"SMARCB1\", \"entrez_id\"=>6598, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>5356, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"22\", \"start\"=>24129150, \"stop\"=>24176703, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000344921.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5356/summary/variants/695/summary#variant\", \"evidence_items\"=>[{\"id\"=>1739, \"name\"=>\"EID1739\", \"description\"=>\"Treatment with tazemetostat , a small-molecule inhibitor of EZH2, results in concentration-dependent cell growth inhibition and cell death specifically in SS18-SSX-positive, SMARCB1-deficient synovial sarcomas.\", \"disease\"=>{\"id\"=>1310, \"name\"=>\"Synovial Sarcoma\", \"display_name\"=>\"Synovial Sarcoma\", \"doid\"=>\"5485\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5485\"}, \"drugs\"=>[{\"id\"=>411, \"name\"=>\"EPZ-6438\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1185, \"name\"=>\"Preclinical Evidence of Anti-Tumor Activity Induced by EZH2 Inhibition in Human Models of Synovial Sarcoma.\", \"citation\"=>\"Kawano et al., 2016, PLoS ONE\", \"pubmed_id\"=>\"27391784\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27391784\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4938529\", \"publication_date\"=>{\"year\"=>2016}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>695, \"phenotypes\"=>[]}]}, {\"id\"=>700, \"entrez_name\"=>\"KIF23\", \"entrez_id\"=>9493, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>7502, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"15\", \"start\"=>69706688, \"stop\"=>69740756, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000260363.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/7502/summary/variants/700/summary#variant\", \"evidence_items\"=>[{\"id\"=>1745, \"name\"=>\"EID1745\", \"description\"=>\"KIF23 variant 1 was found to be expressed in the cDNA of tumor tissues from a large cohort of HCC patients of chinese descent via PCR. This expression was absent in the normal tissues of these patients for this isoform. Using IHC and custom generated antibody, the overall survival with respect to KIF23 V1 expression was evaluated using the kaplan-meier method with a log rank test. It was found that positive KIF23 V1 staining was correlated with longer overall survival over a period of 5 years (p=.0052). This pattern was seen to a lesser degree over a period of 3 years (p=.1604). KIF23 V1 expression was independent of gender, age, tumor size, or TNM stage (multivariate cox regression).\", \"disease\"=>{\"id\"=>196, \"name\"=>\"Hepatocellular Carcinoma\", \"display_name\"=>\"Hepatocellular Carcinoma\", \"doid\"=>\"684\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:684\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1203, \"name\"=>\"Evaluation of KIF23 variant 1 expression and relevance as a novel prognostic factor in patients with hepatocellular carcinoma.\", \"citation\"=>\"Sun et al., 2015, BMC Cancer\", \"pubmed_id\"=>\"26674738\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26674738\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4682286\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>12, \"day\"=>16}, \"journal\"=>\"BMC Cancer\", \"full_journal_title\"=>\"BMC cancer\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>700, \"phenotypes\"=>[]}]}, {\"id\"=>784, \"entrez_name\"=>\"CDKN2B\", \"entrez_id\"=>1030, \"name\"=>\"METHYLATION\", \"description\"=>\"\", \"gene_id\"=>916, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>22009121, \"stop\"=>22009172, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000276925.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/916/summary/variants/784/summary#variant\", \"evidence_items\"=>[{\"id\"=>1848, \"name\"=>\"EID1848\", \"description\"=>\"This study analyses 45 new acute promyelocytic leukemia patients (22 male and 23 female). CDKN2B was found to be a negative prognostic biomarker for disease free survival (DFS). The 5-year DFS was 9.6% in methylated and 62.5% in unmethylated samples. While methylation also associated with young age, a multivariate analysis found CDKN2B methylation to be the only independent marker with significance (P=0.008) for DFS.\", \"disease\"=>{\"id\"=>379, \"name\"=>\"Acute Promyelocytic Leukemia\", \"display_name\"=>\"Acute Promyelocytic Leukemia\", \"doid\"=>\"0060318\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060318\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1271, \"name\"=>\"CDKN2B methylation is an independent poor prognostic factor in newly diagnosed acute promyelocytic leukemia.\", \"citation\"=>\"Chim et al., 2006, Leukemia\", \"pubmed_id\"=>\"16307009\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16307009\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2006, \"month\"=>1}, \"journal\"=>\"Leukemia\", \"full_journal_title\"=>\"Leukemia\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>784, \"phenotypes\"=>[]}]}, {\"id\"=>795, \"entrez_name\"=>\"RRM1\", \"entrez_id\"=>6240, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>5051, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"11\", \"start\"=>4116039, \"stop\"=>4160106, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000300738.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5051/summary/variants/795/summary#variant\", \"evidence_items\"=>[{\"id\"=>1859, \"name\"=>\"EID1859\", \"description\"=>\"The mRNA expression level of RRM1 was analyzed by RT-PCR in 3 human pancreatic cancer cell lines cultured in incrementally increased concentrations of gemcitabine. Expression levels of RRM1 progressively increased during the development of gemcitabine resistance.\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>85, \"name\"=>\"Gemcitabine\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1279, \"name\"=>\"Gemcitabine chemoresistance and molecular markers associated with gemcitabine transport and metabolism in human pancreatic cancer cells.\", \"citation\"=>\"Nakano et al., 2007, Br. J. Cancer\", \"pubmed_id\"=>\"17224927\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17224927\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2360025\", \"publication_date\"=>{\"year\"=>2007, \"month\"=>2, \"day\"=>12}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>795, \"phenotypes\"=>[]}]}, {\"id\"=>796, \"entrez_name\"=>\"RRM2\", \"entrez_id\"=>6241, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>5052, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>10262455, \"stop\"=>10271545, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000360566.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5052/summary/variants/796/summary#variant\", \"evidence_items\"=>[{\"id\"=>1860, \"name\"=>\"EID1860\", \"description\"=>\"The mRNA expression level of RRM2 was analyzed by RT-PCR in 3 human pancreatic cancer cell lines  cultured in incrementally increased concentrations of gemcitabine. Expression levels of RRM2 progressively increased during the development of gemcitabine resistance.\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>85, \"name\"=>\"Gemcitabine\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1279, \"name\"=>\"Gemcitabine chemoresistance and molecular markers associated with gemcitabine transport and metabolism in human pancreatic cancer cells.\", \"citation\"=>\"Nakano et al., 2007, Br. J. Cancer\", \"pubmed_id\"=>\"17224927\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17224927\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2360025\", \"publication_date\"=>{\"year\"=>2007, \"month\"=>2, \"day\"=>12}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>796, \"phenotypes\"=>[]}]}, {\"id\"=>860, \"entrez_name\"=>\"NTRK1\", \"entrez_id\"=>4914, \"name\"=>\"EXPRESSION\", \"description\"=>\"\", \"gene_id\"=>3983, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>156830686, \"stop\"=>156851434, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000524377.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3983/summary/variants/860/summary#variant\", \"evidence_items\"=>[{\"id\"=>1972, \"name\"=>\"EID1972\", \"description\"=>\"A high level of expression of the TRK proto-oncogene in a neuroblastoma is strongly predictive of a favorable outcome. A tumor with a functional nerve growth factor receptor may be dependent on the neurotrophin nerve growth factor for survival and may regress in its absence.\", \"disease\"=>{\"id\"=>13, \"name\"=>\"Neuroblastoma\", \"display_name\"=>\"Neuroblastoma\", \"doid\"=>\"769\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:769\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1387, \"name\"=>\"Association between high levels of expression of the TRK gene and favorable outcome in human neuroblastoma.\", \"citation\"=>\"Nakagawara et al., 1993, N. Engl. J. Med.\", \"pubmed_id\"=>\"8441429\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/8441429\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1993, \"month\"=>3, \"day\"=>25}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>860, \"phenotypes\"=>[]}]}, {\"id\"=>1255, \"entrez_name\"=>\"MGMT\", \"entrez_id\"=>4255, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>34, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>65.0, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>131265448, \"stop\"=>131566271, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000306010.7\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/34/summary/variants/1255/summary#variant\", \"evidence_items\"=>[{\"id\"=>2899, \"name\"=>\"EID2899\", \"description\"=>\"In this study, patients with low MGMT expression, compared with patients with high MGMT expression, had a significantly higher response rate (55% v 7%, respectively; P = .004) and improved PFS (median, 5.5 v 1.9 months, respectively; P = .009) and OS (median, 16 v 5 months, respectively; P = .003). Among patients with MGMT data available for analysis, radiotherapy was started and completed in 91% of patients with low MGMT expression. In contrast, among patients with high MGMT expression, only 64% started radiotherapy, and 43% completed it. These results suggest that neoadjuvant chemotherapy may be feasible for inoperable glioblastoma patients expressing low levels of MGMT without compromising subsequent treatment with radiotherapy, whereas patients with high MGMT expression are less likely to respond and more likely to experience progression before or during radiotherapy.\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[{\"id\"=>11, \"name\"=>\"Temozolomide\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1662, \"name\"=>\"Correlation between O6-methylguanine-DNA methyltransferase and survival in inoperable newly diagnosed glioblastoma patients treated with neoadjuvant temozolomide.\", \"citation\"=>\"Chinot et al., 2007, J. Clin. Oncol.\", \"pubmed_id\"=>\"17442989\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17442989\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>4, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1255, \"phenotypes\"=>[]}, {\"id\"=>2903, \"name\"=>\"EID2903\", \"description\"=>\"In 41 glioblastomas treated with temozolomide (TMZ) therapy (MGMT-immunopositive: n = 22, negative: n = 19), both progression free survival and overall survival were significantly better in MGMT-immunonegative glioblastomas. (PFS: P = 0.0015, OS: P = 0.0384).\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[{\"id\"=>11, \"name\"=>\"Temozolomide\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1666, \"name\"=>\"O6-methylguanine DNA methyltransferase expression in tumor cells predicts outcome of radiotherapy plus concomitant and adjuvant temozolomide therapy in patients with primary glioblastoma.\", \"citation\"=>\"Watanabe et al., 2011, Brain Tumor Pathol\", \"pubmed_id\"=>\"21331613\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21331613\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>4}, \"journal\"=>\"Brain Tumor Pathol\", \"full_journal_title\"=>\"Brain tumor pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1255, \"phenotypes\"=>[]}, {\"id\"=>2901, \"name\"=>\"EID2901\", \"description\"=>\"This study showed that low MGMT mRNA expression was strongly predictive for prolonged time to progression, treatment response, and length of survival in univariate and multivariate models (p<0.0001). The degree of MGMT mRNA expression was highly correlated with the MGMT promoter methylation status (p<0.0001); however, discordant findings were seen in 12 glioblastoma patients. Patients with methylated tumors with high MGMT mRNA expression (N = 6) did significantly worse than those with low transcriptional activity (p<0.01). Conversely, unmethylated tumors with low MGMT mRNA expression (N = 6) did better than their counterparts.\", \"disease\"=>{\"id\"=>695, \"name\"=>\"Malignant Glioma\", \"display_name\"=>\"Malignant Glioma\", \"doid\"=>\"3070\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3070\"}, \"drugs\"=>[{\"id\"=>11, \"name\"=>\"Temozolomide\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1664, \"name\"=>\"O-methylguanine-DNA methyltransferase (MGMT) mRNA expression predicts outcome in malignant glioma independent of MGMT promoter methylation.\", \"citation\"=>\"Kreth et al., 2011, PLoS ONE\", \"pubmed_id\"=>\"21365007\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21365007\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3041820\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>2, \"day\"=>18}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1255, \"phenotypes\"=>[]}, {\"id\"=>2902, \"name\"=>\"EID2902\", \"description\"=>\"In this study, loss of chromosome 1p and low MGMT protein expression were associated with objective response (P < .003 and P < .04, respectively). However, only 54% of the study group had evaluation of 1p/19q status, and MGMT protein expression was studied in only 33%. Therefore, the association between 1p deletion and low MGMT protein expression was statistically significant when evaluated further and verified in the extended group of 31 patients with low-grade oligodendrogliomas.\", \"disease\"=>{\"id\"=>715, \"name\"=>\"Oligodendroglioma\", \"display_name\"=>\"Oligodendroglioma\", \"doid\"=>\"3181\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3181\"}, \"drugs\"=>[{\"id\"=>11, \"name\"=>\"Temozolomide\", \"pubchem_id\"=>\"\"}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1665, \"name\"=>\"Progressive low-grade oligodendrogliomas: response to temozolomide and correlation between genetic profile and O6-methylguanine DNA methyltransferase protein expression.\", \"citation\"=>\"Levin et al., 2006, Cancer\", \"pubmed_id\"=>\"16541434\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16541434\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2006, \"month\"=>4, \"day\"=>15}, \"journal\"=>\"Cancer\", \"full_journal_title\"=>\"Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1255, \"phenotypes\"=>[]}, {\"id\"=>2904, \"name\"=>\"EID2904\", \"description\"=>\"Among those patients who received temozolomide-based therapy in this study, the median progression-free survival for patients whose tumors showed intact MGMT expression was 9.3 months compared with 19.2 months for patients with MGMT-deficient tumors (P = 0.11). The median overall survival for patients whose tumors showed intact MGMT expression was 19.1 months; the median overall survival for patients with MGMT deficient tumors was not reached.\", \"disease\"=>{\"id\"=>541, \"name\"=>\"Neuroendocrine Tumor\", \"display_name\"=>\"Neuroendocrine Tumor\", \"doid\"=>\"169\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:169\"}, \"drugs\"=>[{\"id\"=>11, \"name\"=>\"Temozolomide\", \"pubchem_id\"=>\"\"}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1667, \"name\"=>\"O6-methylguanine DNA methyltransferase deficiency and response to temozolomide-based therapy in patients with neuroendocrine tumors.\", \"citation\"=>\"Kulke et al., 2009, Clin. Cancer Res.\", \"pubmed_id\"=>\"19118063\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19118063\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3100724\", \"publication_date\"=>{\"year\"=>2009, \"month\"=>1, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1255, \"phenotypes\"=>[]}]}, {\"id\"=>1257, \"entrez_name\"=>\"RRM1\", \"entrez_id\"=>6240, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>5051, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>85.0, \"coordinates\"=>{\"chromosome\"=>\"11\", \"start\"=>4116039, \"stop\"=>4160106, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000300738.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5051/summary/variants/1257/summary#variant\", \"evidence_items\"=>[{\"id\"=>5506, \"name\"=>\"EID5506\", \"description\"=>\"There were 68 patients enrolled in this study, 28 of which were treated with gemcitabine.  Of those 28, 14 patients had a low RRM1 expression and 14 had a high RRM1 expression. Only the patients with low RRM1 expression received a significant benefit from gemcitabine (p=0.0010) whereas the benefit for those with a high RRM1 expression was not significantly better (p=0.3309).  The study also tested the role of ERCC1 expression and its correlation to RRM1 expression. It was found that patients with a combined high RRM1 and ERCC1 expression had good prognosis with the highest survival rate (P=0.0127).\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>85, \"name\"=>\"Gemcitabine\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2253, \"name\"=>\"Significance of RRM1 and ERCC1 expression in resectable pancreatic adenocarcinoma.\", \"citation\"=>\"Akita et al., 2009, Oncogene\", \"pubmed_id\"=>\"19543324\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19543324\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3913371\", \"publication_date\"=>{\"year\"=>2009, \"month\"=>8, \"day\"=>13}, \"journal\"=>\"Oncogene\", \"full_journal_title\"=>\"Oncogene\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1257, \"phenotypes\"=>[]}, {\"id\"=>2905, \"name\"=>\"EID2905\", \"description\"=>\"In the personalized therapy group of this study, patients received chemotherapy based on RRM-1 IHC expression levels, whereas the standard therapy group received non-customized chemotherapy. There were statistically significant improvements between the personalized therapy group versus the standard therapy group in disease control rate (82.9% vs 55.3%, P=0.004), and progression-free survival (median: 5.5 months vs 3.0 months, P=0.005). Besides, the overall survival had a tendency to become more prolonged (median: 16.0 months vs 12.4 months, P=0.286).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>85, \"name\"=>\"Gemcitabine\", \"pubchem_id\"=>nil}, {\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1668, \"name\"=>\"The tailored chemotherapy based on RRM1 immunohistochemical expression in patients with advanced non-small cell lung cancer.\", \"citation\"=>\"Nie et al., 2013, Cancer Biomark\", \"pubmed_id\"=>\"24595080\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24595080\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>1, \"day\"=>1}, \"journal\"=>\"Cancer Biomark\", \"full_journal_title\"=>\"Cancer biomarkers : section A of Disease markers\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1257, \"phenotypes\"=>[]}, {\"id\"=>5508, \"name\"=>\"EID5508\", \"description\"=>\"In this study, human pancreatic carcinoma cell line MiaPaCa2, which is a gemcitabine-sensitive cell line, was used for the establishment of chemoresistant cells MiaPaCa2-RG. Analysis for identifying genetic alterations relating to gemcitabine resistance revealed that RRM1 was the most upregulated gene with a 4.5 fold expression. RRM1-speciﬁc RNAi transfection significantly reduced the gemcitabine chemoresistance of MiaPaCa2-RG to the same level as that of gemcitabine sensitive MiaPaCa2. Moreover, gemcitabine response of MiaPaCa2 also became more sensitive.  Next, patients with pancreatic cancer were recruited and 18 tissue samples were taken. The patients with a low RRM1 expression had a significantly better outcome after gemcitabine treatment than those patients with high RRM1 expression (P=0.016).\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>85, \"name\"=>\"Gemcitabine\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2257, \"name\"=>\"Involvement of ribonucleotide reductase M1 subunit overexpression in gemcitabine resistance of human pancreatic cancer.\", \"citation\"=>\"Nakahira et al., 2007, Int. J. Cancer\", \"pubmed_id\"=>\"17131328\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17131328\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>3, \"day\"=>15}, \"journal\"=>\"Int. J. Cancer\", \"full_journal_title\"=>\"International journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1257, \"phenotypes\"=>[]}, {\"id\"=>5530, \"name\"=>\"EID5530\", \"description\"=>\"In this study, 229 tumors were analyzed for RRM1 expression. This resulted in 146 that were negative for RRM1 expression and 83 that were positive. These tumors were studied for a relationship between RRM1 expression and chemotherapy regimens, gemcitabine, docetaxel and carboplatin, and vinorelbine and carboplatin. In the patients receiving gemcitabine treatment, the progression-free survival of RRM1-negative patients was significantly higher than that of RRM1-positive patients (8.8 months vs. 7.6 months, P = 0.01). The patients that were positive for RRM1 expression, had a better response to docetaxel or vinorelbine treatment than gemcitabine (P=0.047 and P=0.047). However, there were no differences of response to docetaxel and vinorelbine in regards to RRM1 expression.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>73, \"name\"=>\"Docetaxel\", \"pubchem_id\"=>nil}, {\"id\"=>85, \"name\"=>\"Gemcitabine\", \"pubchem_id\"=>nil}, {\"id\"=>96, \"name\"=>\"Vinorelbine\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2263, \"name\"=>\"Response to first-line chemotherapy in patients with non-small cell lung cancer according to RRM1 expression.\", \"citation\"=>\"Dong et al., 2014, PLoS ONE\", \"pubmed_id\"=>\"24647522\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24647522\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3960222\", \"publication_date\"=>{\"year\"=>2014}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1257, \"phenotypes\"=>[]}, {\"id\"=>5599, \"name\"=>\"EID5599\", \"description\"=>\"The study began with a total of 443 patients that were randomly assigned to either regimen A (Paclitaxel and cisplatin with gemcitabine) or regimen B ( cisplatin and vinorelbine). Of those patients, 261 had tissue available for RRM1 evaluation.  In regimen B, survival increased significantly in patients with low RRM1 expression as compared to those with a high expression (P=0.046). However, in regimen A, the increase in survival for patients with low RRM1 expression as opposed to those with a high RRM1 expression was not as significant (P=0.366).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>81, \"name\"=>\"Paclitaxel\", \"pubchem_id\"=>nil}, {\"id\"=>85, \"name\"=>\"Gemcitabine\", \"pubchem_id\"=>nil}, {\"id\"=>96, \"name\"=>\"Vinorelbine\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Unknown\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2287, \"name\"=>\"Predictive impact of RRM1 protein expression on vinorelbine efficacy in NSCLC patients randomly assigned in a chemotherapy phase III trial.\", \"citation\"=>\"Vilmar et al., 2013, Ann. Oncol.\", \"pubmed_id\"=>\"23038758\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23038758\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>2}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1257, \"phenotypes\"=>[]}]}, {\"id\"=>1299, \"entrez_name\"=>\"MERTK\", \"entrez_id\"=>10461, \"name\"=>\"OVEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>8331, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>112656056, \"stop\"=>112787138, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000295408.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/8331/summary/variants/1299/summary#variant\", \"evidence_items\"=>[{\"id\"=>2991, \"name\"=>\"EID2991\", \"description\"=>\"Highest MERTK expression was observed in metastatic melanomas, followed by primary melanomas, and lowest in benign nevi. Over half of melanoma cell lines overexpressed MERTK compared with normal human melanocytes; however, overexpression did not correlate with mutations in BRAF or RAS. MERTK inhibition via shRNA reduced MERTK-mediated downstream signaling, reduced colony formation by up to 59%, and diminished tumor volume by 60% in a human melanoma murine xenograft model. Treatment of melanoma cells with UNC1062, a novel MERTK-selective small-molecule tyrosine kinase inhibitor, reduced activation of MERTK-mediated downstream signaling, induced apoptosis in culture, reduced colony formation in soft agar, and inhibited invasion of melanoma cells.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>497, \"name\"=>\"UNC1062\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1717, \"name\"=>\"MERTK receptor tyrosine kinase is a therapeutic target in melanoma.\", \"citation\"=>\"Schlegel et al., 2013, J. Clin. Invest.\", \"pubmed_id\"=>\"23585477\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23585477\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3639697\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>5}, \"journal\"=>\"J. Clin. Invest.\", \"full_journal_title\"=>\"The Journal of clinical investigation\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1299, \"phenotypes\"=>[]}]}, {\"id\"=>2108, \"entrez_name\"=>\"ARID1A\", \"entrez_id\"=>8289, \"name\"=>\"UNDEREXPRESSION\", \"description\"=>\"\", \"gene_id\"=>6559, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>183, \"name\"=>\"N/A\", \"display_name\"=>\"N/A\", \"so_id\"=>\"N/A\", \"description\"=>\"No suitable Sequence Ontology term exists.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/N/A\"}], \"civic_actionability_score\"=>30.0, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>27022524, \"stop\"=>27108595, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000324856.7\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/6559/summary/variants/2108/summary#variant\", \"evidence_items\"=>[{\"id\"=>5675, \"name\"=>\"EID5675\", \"description\"=>\"In this study, real-time quantitative RT-PCR assays were used in 66 pairs of cancerous and matched adjacent normal tissues. In 43 of those pairs, the expression level of ARID1A was significantly lower in the cancerous tissue than in the normal tissue (P=0.0029). This was then again tested using Western Blotting. Consistent with the real-time PCR results, there was a lower expression of ARID1A in 13 of the tumor tissue samples compared to the normal tissue samples (P=0.0015). In an immunohistochemical analysis of 224 tissue blocks, 115 of those had a decreased ARID1A expression. The data conveyed that lower ARID1A expression was correlated with depth of tumor infiltration (P=0.001). Lastly, cell proliferation in regards to ARID1A expression was tested in cell lines. The cells with silenced expression of ARID1A had a much greater growth than those with greater expression of the protein.\", \"disease\"=>{\"id\"=>21, \"name\"=>\"Stomach Cancer\", \"display_name\"=>\"Stomach Cancer\", \"doid\"=>\"10534\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10534\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2305, \"name\"=>\"Decreased expression of the ARID1A gene is associated with poor prognosis in primary gastric cancer.\", \"citation\"=>\"Wang et al., 2012, PLoS ONE\", \"pubmed_id\"=>\"22808142\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22808142\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3396657\", \"publication_date\"=>{\"year\"=>2012}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2108, \"phenotypes\"=>[]}, {\"id\"=>5787, \"name\"=>\"EID5787\", \"description\"=>\"In this study, tissue sampling began with 77 ovarian cancer patients. Following, 60 of those samples, the patients withe clear cell ovarian cancer, were analyzed for ARID1A expression using immunohistochemistry.  Of those 60, 9 patients had a loss of ARID1A expression. Also, 17 high-grade serous adenocarcinomas were also analyzed and none of them showed a loss of ARID1A expression. The loss was more frequent in clear cell ovarian carcinoma than in the high-grade serous adenocarcinoma. Results showed that a low ARID1A expression was significantly related to advanced FIGO stages and high CAI25 levels (P=0.02). Those patients with a low expression experienced shorter progression free survival than those with no loss of expression (P<0.01). In a multivariate analysis it was found that a loss of ARID1A expression was significant for shorter disease free survival (P=0.03) and shorter overall survival for patients whose ovarian clear cell carcinoma was treated with platinum-based chemotherapy (P=0.015). Lastly, tumors with low expression were significantly more likely to be resistant to primary adjuvant chemotherapy than tumors with a higher expression (P=0.04).\", \"disease\"=>{\"id\"=>88, \"name\"=>\"Ovarian Clear Cell Carcinoma\", \"display_name\"=>\"Ovarian Clear Cell Carcinoma\", \"doid\"=>\"0050934\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050934\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>nil, \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2336, \"name\"=>\"Loss of ARID1A expression is related to shorter progression-free survival and chemoresistance in ovarian clear cell carcinoma.\", \"citation\"=>\"Katagiri et al., 2012, Mod. Pathol.\", \"pubmed_id\"=>\"22101352\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22101352\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>2}, \"journal\"=>\"Mod. Pathol.\", \"full_journal_title\"=>\"Modern pathology : an official journal of the United States and Canadian Academy of Pathology, Inc\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2108, \"phenotypes\"=>[]}, {\"id\"=>5676, \"name\"=>\"EID5676\", \"description\"=>\"This study included 290 patients with Clear Cell Renal Cell Carcinoma whose tumor tissues were sampled and then analyzed. These samples were then divided according to ARID1A expression after immunohistochemical staining. Of the 290 patients, 213 had a low ARID1A expression and 77 patients had a high expression. There was a significant correlation between low ARID1A expression and large tumors as well as shorter survival. After a multivariate analysis using Cox proportional hazards, it was found that ARID1A expression levels are an independent predictor of progression-free survival in patients with CCRCC, although no significant correlation was found between the expression levels and cancer-specific survival.\", \"disease\"=>{\"id\"=>48, \"name\"=>\"Renal Clear Cell Carcinoma\", \"display_name\"=>\"Renal Clear Cell Carcinoma\", \"doid\"=>\"4467\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4467\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Unknown\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2306, \"name\"=>\"Decreased ARID1A expression correlates with poor prognosis of clear cell renal cell carcinoma.\", \"citation\"=>\"Park et al., 2015, Hum. Pathol.\", \"pubmed_id\"=>\"25628030\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25628030\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>3}, \"journal\"=>\"Hum. Pathol.\", \"full_journal_title\"=>\"Human pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2108, \"phenotypes\"=>[]}]}, {\"id\"=>1, \"entrez_name\"=>\"ABL1\", \"entrez_id\"=>25, \"name\"=>\"BCR-ABL\", \"description\"=>\"The BCR-ABL fusion protein, commonly referred to as the Philadelphia chromosome, is one of the most studied fusion genes in cancer. It has widely been considered the initiating event in chronic myelogenous leukemia (CML). However, despite its ability to initiate disease in mice, its status as an initiating mutation is in dispute. In what is commonly used as the poster-child for targeted therapeutics, the development and use of imatinib in the clinic has led to profound improvements in the prognosis of the disease. However, imatinib resistance is still seen in patients with mutations in the ABL kinase domain of the fusion, most notably the T315I variant. In patients resistant to imatinib, either acquired or otherwise, second generation BCR-ABL TKI's (dasatinib and nilotinib) have seen some success in delivering a tumor response. Third generation ABL1 inhibitor ponatinib is the only FDA approved drug with activity against T315I . However due to risk of life-threatening blood clots and severe narrowing of blood vessels ponatinib is ONLY approved for T315I-positive CML or T315I-positive Ph+ ALL or in cases of CML, Ph+ ALL with resistance or intolerance to other approved ABL1 inhibitors.\", \"gene_id\"=>4, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>238.0, \"coordinates\"=>{\"chromosome\"=>\"22\", \"start\"=>23522397, \"stop\"=>23632600, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000305877.8\", \"chromosome2\"=>\"9\", \"start2\"=>133729451, \"stop2\"=>133763063, \"representative_transcript2\"=>\"ENST00000318560.5\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4/summary/variants/1/summary#variant\", \"evidence_items\"=>[{\"id\"=>344, \"name\"=>\"EID344\", \"description\"=>\"BCR-ABL inhibitors such as imatinib have lead to significantly improved prognosis, response rate, overall survival, and patient outcome in CML patients compared to previous therapeutic regimens.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>156, \"name\"=>\"BCR-ABL tyrosine kinase inhibitors in the treatment of Philadelphia chromosome positive chronic myeloid leukemia: a review.\", \"citation\"=>\"An et al., 2010, Leuk. Res.\", \"pubmed_id\"=>\"20537386\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20537386\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>10}, \"journal\"=>\"Leuk. Res.\", \"full_journal_title\"=>\"Leukemia research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1, \"phenotypes\"=>[]}, {\"id\"=>259, \"name\"=>\"EID259\", \"description\"=>\"Treatment of Philadelphia chromosome positive leukemias with imatinib results in high rates of complete remission in patients with CML.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>164, \"name\"=>\"Novel targeted therapies for Bcr-Abl positive acute leukemias: beyond STI571.\", \"citation\"=>\"Nimmanapalli et al., 2002, Oncogene\", \"pubmed_id\"=>\"12476305\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/12476305\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2002, \"month\"=>12, \"day\"=>9}, \"journal\"=>\"Oncogene\", \"full_journal_title\"=>\"Oncogene\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1, \"phenotypes\"=>[]}, {\"id\"=>260, \"name\"=>\"EID260\", \"description\"=>\"The clinical use of imatinib in patients with BCR-ABL fusion has resulted in drastic prognostic improvements in patients with CML.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>156, \"name\"=>\"BCR-ABL tyrosine kinase inhibitors in the treatment of Philadelphia chromosome positive chronic myeloid leukemia: a review.\", \"citation\"=>\"An et al., 2010, Leuk. Res.\", \"pubmed_id\"=>\"20537386\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20537386\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>10}, \"journal\"=>\"Leuk. Res.\", \"full_journal_title\"=>\"Leukemia research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1, \"phenotypes\"=>[]}, {\"id\"=>213, \"name\"=>\"EID213\", \"description\"=>\"Data from deep sequencing of a Ph-negative clone of a Ph-positive CML patient has found a driving mutation in DNMT3A that preceeded the BCR-ABL fusion, and may imply the possibility that BCR-ABL is not universally the initiating event in CML.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>151, \"name\"=>\"Molecular-defined clonal evolution in patients with chronic myeloid leukemia independent of the BCR-ABL status.\", \"citation\"=>\"Schmidt et al., 2014, Leukemia\", \"pubmed_id\"=>\"25212276\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25212276\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>12}, \"journal\"=>\"Leukemia\", \"full_journal_title\"=>\"Leukemia\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1, \"phenotypes\"=>[]}, {\"id\"=>232, \"name\"=>\"EID232\", \"description\"=>\"BCR-ABL fusions have been found to contribute to imatinib resistance in AML cell lines.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>164, \"name\"=>\"Novel targeted therapies for Bcr-Abl positive acute leukemias: beyond STI571.\", \"citation\"=>\"Nimmanapalli et al., 2002, Oncogene\", \"pubmed_id\"=>\"12476305\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/12476305\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2002, \"month\"=>12, \"day\"=>9}, \"journal\"=>\"Oncogene\", \"full_journal_title\"=>\"Oncogene\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1, \"phenotypes\"=>[]}, {\"id\"=>220, \"name\"=>\"EID220\", \"description\"=>\"The presence of BCR-ABL fusion is considered the characterizing feature of chronic myeloid leukemia, and has been widely thought of as the initiating event in the disease.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>156, \"name\"=>\"BCR-ABL tyrosine kinase inhibitors in the treatment of Philadelphia chromosome positive chronic myeloid leukemia: a review.\", \"citation\"=>\"An et al., 2010, Leuk. Res.\", \"pubmed_id\"=>\"20537386\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20537386\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>10}, \"journal\"=>\"Leuk. Res.\", \"full_journal_title\"=>\"Leukemia research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1, \"phenotypes\"=>[]}, {\"id\"=>261, \"name\"=>\"EID261\", \"description\"=>\"The use of nilotinib and dasatinib, second-generation BCR-ABL targeted therapies, has been effective in patients resistant to imatinib, leading to their FDA-approval.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>13, \"name\"=>\"Nilotinib\", \"pubchem_id\"=>nil}, {\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>156, \"name\"=>\"BCR-ABL tyrosine kinase inhibitors in the treatment of Philadelphia chromosome positive chronic myeloid leukemia: a review.\", \"citation\"=>\"An et al., 2010, Leuk. Res.\", \"pubmed_id\"=>\"20537386\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20537386\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>10}, \"journal\"=>\"Leuk. Res.\", \"full_journal_title\"=>\"Leukemia research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1, \"phenotypes\"=>[]}]}, {\"id\"=>2, \"entrez_name\"=>\"ABL1\", \"entrez_id\"=>25, \"name\"=>\"BCR-ABL T334I\", \"description\"=>\"While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation T315I (aka T334I) has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and ponatinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\", \"gene_id\"=>4, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>52.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>133748283, \"stop\"=>133748283, \"reference_bases\"=>\"C\", \"variant_bases\"=>\"T\", \"representative_transcript\"=>\"ENST00000372348.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4/summary/variants/2/summary#variant\", \"evidence_items\"=>[{\"id\"=>2667, \"name\"=>\"EID2667\", \"description\"=>\"A panel of 18 mutations representing 85% of known mutations seen in imatinib CML patents were used to evaluate the effectiveness of drug binding. All mutations were compared with non-mutated BCR-ABL protein and were grouped according to their anti-proliferative activity. Mutant cell lines that exhibited a greater than 4-fold increase in IC50 value were considered resistant to drug, 2-4 fold increase in IC50 were moderately resistant and less than 2-fold change in IC50 was sensitive to drug treatment. The T315I mutation was resistant to imatinib.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1529, \"name\"=>\"Activity of bosutinib, dasatinib, and nilotinib against 18 imatinib-resistant BCR/ABL mutants.\", \"citation\"=>\"Redaelli et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19075254\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19075254\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4373, \"name\"=>\"EID4373\", \"description\"=>\"In a retrospective study of 84 imatinib-resistant or -intolerant Philadelphia chromosome BCR-ABL1 fusion positive (Ph+) chronic myeloid leukemia (in accelerated-phase or with blast crisis) or Ph+ acute lymphoblastic leukemia patients, patients with the T315I mutation in the ABL1 portion of the BCR-ABL1 gene fusion (n=3) did not achieve a response to dasatinib monotherapy. In a retrospective study of 84 imatinib-resistant or -intolerant Philadelphia chromosome BCR-ABL1 fusion positive (Ph+) chronic myeloid leukemia (in accelerated-phase or with blast crisis) or Ph+ acute lymphoblastic leukemia patients, patients with the T315I mutation in the ABL1 portion of the BCR-ABL1 gene fusion (n=3) did not achieve a response to dasatinib monotherapy.\", \"disease\"=>{\"id\"=>43, \"name\"=>\"Acute Lymphocytic Leukemia\", \"display_name\"=>\"Acute Lymphocytic Leukemia\", \"doid\"=>\"9952\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9952\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2096, \"name\"=>\"Dasatinib in imatinib-resistant Philadelphia chromosome-positive leukemias.\", \"citation\"=>\"Talpaz et al., 2006, N. Engl. J. Med.\", \"pubmed_id\"=>\"16775234\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16775234\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2006, \"month\"=>6, \"day\"=>15}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4379, \"name\"=>\"EID4379\", \"description\"=>\"In an in vitro study, the recombinant ABL1 T315I mutation kinase was associated with reduced sensitivity to dasatinib treatment (IC50: 1648nM vs. 0.3nM), as compared to wild-type ABL1 kinase. Sensitivity was assessed by analyzing kinase inhibition.\", \"disease\"=>{\"id\"=>43, \"name\"=>\"Acute Lymphocytic Leukemia\", \"display_name\"=>\"Acute Lymphocytic Leukemia\", \"doid\"=>\"9952\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9952\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1994, \"name\"=>\"Global target profile of the kinase inhibitor bosutinib in primary chronic myeloid leukemia cells.\", \"citation\"=>\"Remsing Rix et al., 2009, Leukemia\", \"pubmed_id\"=>\"19039322\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19039322\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>3}, \"journal\"=>\"Leukemia\", \"full_journal_title\"=>\"Leukemia\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4378, \"name\"=>\"EID4378\", \"description\"=>\"In an in vitro study, a Ba/F3 cell line stably expressing the T315I mutation in the ABL1 portion of the BCR-ABL1 fusion protein was found to have reduced sensitivity to dasatinib treatment, comparable to Ba/F3 cells expressing BCR-ABL1 fusion protein. In an in vivo experiment, xenografts tumors expressing the T315I mutation in the ABL1 portion of the BCR-ABL1 fusion protein also failed to respond to treatment with dasatinib. Resistance was determined by assessing cell proliferation and tumor growth.\", \"disease\"=>{\"id\"=>43, \"name\"=>\"Acute Lymphocytic Leukemia\", \"display_name\"=>\"Acute Lymphocytic Leukemia\", \"doid\"=>\"9952\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9952\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2001, \"name\"=>\"Overriding imatinib resistance with a novel ABL kinase inhibitor.\", \"citation\"=>\"Shah et al., 2004, Science\", \"pubmed_id\"=>\"15256671\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15256671\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2004, \"month\"=>7, \"day\"=>16}, \"journal\"=>\"Science\", \"full_journal_title\"=>\"Science (New York, N.Y.)\", \"status\"=>\"submitted\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4377, \"name\"=>\"EID4377\", \"description\"=>\"In an in vitro study, a Ba/F3  cell line stably expressing the T315I mutation in the ABL1 portion of the BCR-ABL1 fusion protein was associated with reduced sensitivity to dasatinib treatment (IC50: 137.3nM vs. IC50: 1.83nM) as compared to Ba/F3 cells expressing BCR-ABL1 fusion protein. Resistance was determined by assessing cell proliferation.\", \"disease\"=>{\"id\"=>43, \"name\"=>\"Acute Lymphocytic Leukemia\", \"display_name\"=>\"Acute Lymphocytic Leukemia\", \"doid\"=>\"9952\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9952\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1529, \"name\"=>\"Activity of bosutinib, dasatinib, and nilotinib against 18 imatinib-resistant BCR/ABL mutants.\", \"citation\"=>\"Redaelli et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19075254\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19075254\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4376, \"name\"=>\"EID4376\", \"description\"=>\"In a retrospective study of 107 imatinib-resistant or -intolerant  Philadelphia chromosome BCR-ABL1 fusion positive accelerated phase chronic myeloid leukemia patients, patients with the T315I mutation in the ABL1 portion of the BCR-ABL gene fusion (n=5) did not achieve major hematologic or major cytogenetic responses to dasatinib monotherapy.\", \"disease\"=>{\"id\"=>43, \"name\"=>\"Acute Lymphocytic Leukemia\", \"display_name\"=>\"Acute Lymphocytic Leukemia\", \"doid\"=>\"9952\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9952\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1998, \"name\"=>\"Dasatinib induces significant hematologic and cytogenetic responses in patients with imatinib-resistant or -intolerant chronic myeloid leukemia in accelerated phase.\", \"citation\"=>\"Guilhot et al., 2007, Blood\", \"pubmed_id\"=>\"17264298\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17264298\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>5, \"day\"=>15}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4375, \"name\"=>\"EID4375\", \"description\"=>\"In a retrospective study of 174 imatinib-resistant or -intolerant Philadelphia chromosome BCR-ABL1 fusion positive accelerated phase chronic myeloid leukemia patients, patients with the T315I mutation in the ABL1 portion of the BCR-ABL1 gene fusion (n=9) did not achieve major hematologic or major cytogenetic responses to dasatinib monotherapy .\", \"disease\"=>{\"id\"=>43, \"name\"=>\"Acute Lymphocytic Leukemia\", \"display_name\"=>\"Acute Lymphocytic Leukemia\", \"doid\"=>\"9952\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9952\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1999, \"name\"=>\"Dasatinib in the treatment of chronic myeloid leukemia in accelerated phase after imatinib failure: the START a trial.\", \"citation\"=>\"Apperley et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19487385\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19487385\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4979080\", \"publication_date\"=>{\"year\"=>2009, \"month\"=>7, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4374, \"name\"=>\"EID4374\", \"description\"=>\"In a retrospective study of 45 imatinib-resistant or -intolerant Philadelphia chromosome BCR-ABL fusion positive (Ph+) chronic myeloid leukemia or Ph+ acute lymphoblastic leukemia (ALL) patients, patients harboring T315I mutation in the ABL1 portion of the BCR-ABL gene fusion (n=9) were associated with primary (n=4) or acquired resistance (n=5) to dasatinib monotherapy of 21 dasatinib resistant or non-responsive patients.. In a retrospective study of 45 imatinib-resistant or -intolerant Philadelphia chromosome BCR-ABL fusion positive (Ph+) chronic myeloid leukemia or Ph+ acute lymphoblastic leukemia (ALL) patients, patients harboring T315I mutation in the ABL1 portion of the BCR-ABL gene fusion (n=9) were associated with primary (n=4) or acquired resistance (n=5) to dasatinib monotherapy of 21 dasatinib resistant or non-responsive patients..\", \"disease\"=>{\"id\"=>43, \"name\"=>\"Acute Lymphocytic Leukemia\", \"display_name\"=>\"Acute Lymphocytic Leukemia\", \"doid\"=>\"9952\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9952\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2095, \"name\"=>\"Resistance to dasatinib in Philadelphia-positive leukemia patients and the presence or the selection of mutations at residues 315 and 317 in the BCR-ABL kinase domain.\", \"citation\"=>\"Soverini et al., 2007, Haematologica\", \"pubmed_id\"=>\"17339191\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17339191\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>3}, \"journal\"=>\"Haematologica\", \"full_journal_title\"=>\"Haematologica\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4372, \"name\"=>\"EID4372\", \"description\"=>\"In an in vitro study, a KBM-5 cell line expressing T315I mutation in the ABL1 portion of the BCR-ABL fusion protein (overexpression) demonstrated sensitivity to axitinib treatment, comparable to KBM-5 cells expressing BCR-ABL1 fusion protein (overexpression). Sensitivity was determined by assessing cell metabolic activity, ABL, CRKL, ERK and Gab2 phosphorylation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>519, \"name\"=>\"Axitinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2092, \"name\"=>\"Axitinib and sorafenib are potent in tyrosine kinase inhibitor resistant chronic myeloid leukemia cells.\", \"citation\"=>\"Halbach et al., 2016, Cell Commun. Signal\", \"pubmed_id\"=>\"26912052\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26912052\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4765141\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>2, \"day\"=>24}, \"journal\"=>\"Cell Commun. Signal\", \"full_journal_title\"=>\"Cell communication and signaling : CCS\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4370, \"name\"=>\"EID4370\", \"description\"=>\"In an in vitro study, primary tumor cells from 2 chronic myeloid leukemia patients (endogenous) and pro-B Ba/F3 cells (overexpression) harboring T315I mutation in the ABL1 portion of the BCR-ABL1 fusion protein demonstrated sensitivity to axitinib treatment. Sensitivity was determined by assessing cell proliferation, BCR-ABL1 auto-phosphorylation or phosphorylation of downstream BCR-ABL targets.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>519, \"name\"=>\"Axitinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2094, \"name\"=>\"Anti-leukemic activity of axitinib against cells harboring the BCR-ABL T315I point mutation.\", \"citation\"=>\"Okabe et al., 2015, J Hematol Oncol\", \"pubmed_id\"=>\"26239229\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26239229\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4523922\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>8, \"day\"=>4}, \"journal\"=>\"J Hematol Oncol\", \"full_journal_title\"=>\"Journal of hematology & oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4369, \"name\"=>\"EID4369\", \"description\"=>\"In a chronic myeloid leukemia (blast crisis) patient harboring T315I mutation in the ABL1 portion of the BCR-ABL1 fusion protein was associated with responsiveness to axitinib monotherapy. The patient had been treated with standard chemotherapy, imatinib and dasatinib with disease progression; subsequently, the patient responded to axitinib treatment as assessed by rapid clearance of BCR-ABL1 (T315I) positive transcripts/cells. In an in vitro study, a pro-B Ba/F3 cell line expressing T315I mutation in the ABL1 portion of the BCR-ABL fusion protein (overexpression) demonstrated increased sensitivity to axitinib treatment (IC50: 98nM vs. 823nM), compared to Ba/F3 expressing the BCR-ABL1 fusion protein (overexpression). Sensitivity was determined by assessing and cell proliferation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>519, \"name\"=>\"Axitinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2002, \"name\"=>\"Axitinib effectively inhibits BCR-ABL1(T315I) with a distinct binding conformation.\", \"citation\"=>\"Pemovska et al., 2015, Nature\", \"pubmed_id\"=>\"25686603\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25686603\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>3, \"day\"=>5}, \"journal\"=>\"Nature\", \"full_journal_title\"=>\"Nature\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4368, \"name\"=>\"EID4368\", \"description\"=>\"In a study of 51 imatinib-resistant chronic myeloid leukemia patients, patients with the T315I mutation in the ABL1 portion of the BCR-ABL1 gene fusion (n=13) were associated with acquired resistant to dasatinib monotherapy.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2093, \"name\"=>\"In chronic myeloid leukemia patients on second-line tyrosine kinase inhibitor therapy, deep sequencing of BCR-ABL1 at the time of warning may allow sensitive detection of emerging drug-resistant mutants.\", \"citation\"=>\"Soverini et al., 2016, BMC Cancer\", \"pubmed_id\"=>\"27485109\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27485109\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4970247\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>8, \"day\"=>2}, \"journal\"=>\"BMC Cancer\", \"full_journal_title\"=>\"BMC cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4367, \"name\"=>\"EID4367\", \"description\"=>\"In an in vitro study, a KBM-5 cell line expressing T315I mutation in the ABL1 portion of the BCR-ABL fusion protein (overexpression) demonstrated resistance to dasatinib treatment, compared to KBM-5 cells expressing BCR-ABL1 fusion protein (overexpression). Resistance was determined by assessing cell metabolic activity, ABL, CRKL, ERK and Gab2 phosphorylation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2092, \"name\"=>\"Axitinib and sorafenib are potent in tyrosine kinase inhibitor resistant chronic myeloid leukemia cells.\", \"citation\"=>\"Halbach et al., 2016, Cell Commun. Signal\", \"pubmed_id\"=>\"26912052\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26912052\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4765141\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>2, \"day\"=>24}, \"journal\"=>\"Cell Commun. Signal\", \"full_journal_title\"=>\"Cell communication and signaling : CCS\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4366, \"name\"=>\"EID4366\", \"description\"=>\"In a retrospective study (of phase 3 clinical trial NCT00481247) of 363 Philadelphia chromosome-positive (Ph+) chronic myeloid leukemia patients in chronic phase, patients with the T315I mutation in the ABL1 portion of the BCR-ABL1 gene fusion treated with dasatinib monotherapy (3/169) were associated with accelerated phase/blast phase and disease progression.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2091, \"name\"=>\"BCR-ABL1 mutation development during first-line treatment with dasatinib or imatinib for chronic myeloid leukemia in chronic phase.\", \"citation\"=>\"Hughes et al., 2015, Leukemia\", \"pubmed_id\"=>\"26118315\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26118315\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4559757\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>9}, \"journal\"=>\"Leukemia\", \"full_journal_title\"=>\"Leukemia\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4365, \"name\"=>\"EID4365\", \"description\"=>\"In a retrospective study of 1084 imatinib-resistant or -intolerant, chronic myeloid leukemia (chronic-phase) patients, 6 patients with the T315I mutation (n=21) in the ABL1 portion of the BCR-ABL1 achieved a complete cytogenetic response to dasatinib monotherapy, no patients achieved a complete hematologic or major cytogenetic response.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1532, \"name\"=>\"Dasatinib treatment of chronic-phase chronic myeloid leukemia: analysis of responses according to preexisting BCR-ABL mutations.\", \"citation\"=>\"Müller et al., 2009, Blood\", \"pubmed_id\"=>\"19779040\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19779040\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4916940\", \"publication_date\"=>{\"year\"=>2009, \"month\"=>12, \"day\"=>3}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"submitted\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4364, \"name\"=>\"EID4364\", \"description\"=>\"In a retrospective study of 288 imatinib-resistant or -intolerant Philadelphia chromosome BCR-ABL1 fusion positive chronic phase chronic myeloid leukemia patients, 1 of 2 patients with the T315I mutation in the ABL1 portion of the BCR-ABL gene fusion achieved a complete hematologic response; however 0 of 3 patients achieved a major cytogenetic responses to bosutinib monotherapy.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>520, \"name\"=>\"Bosutinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1996, \"name\"=>\"Safety and efficacy of bosutinib (SKI-606) in chronic phase Philadelphia chromosome-positive chronic myeloid leukemia patients with resistance or intolerance to imatinib.\", \"citation\"=>\"Cortes et al., 2011, Blood\", \"pubmed_id\"=>\"21865346\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21865346\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4916618\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>10, \"day\"=>27}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4363, \"name\"=>\"EID4363\", \"description\"=>\"In an in vitro study, the recombinant ABL1 T315I mutation kinase was associated with sensitivity to bosutinib (IC50: 26nM), comparable to wild-type ABL1 (IC50: 0.5nM). Sensitivity was assessed by analyzing kinase inhibition.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>520, \"name\"=>\"Bosutinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1994, \"name\"=>\"Global target profile of the kinase inhibitor bosutinib in primary chronic myeloid leukemia cells.\", \"citation\"=>\"Remsing Rix et al., 2009, Leukemia\", \"pubmed_id\"=>\"19039322\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19039322\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>3}, \"journal\"=>\"Leukemia\", \"full_journal_title\"=>\"Leukemia\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4362, \"name\"=>\"EID4362\", \"description\"=>\"In a retrospective study of 118 imatinib-resistant or -intolerant, Philadelphia chromosome BCR-ABL1 fusion positive, chronic myeloid leukemia (chronic phase), 2 of 7 patients with the T315I mutation in the ABL1 portion of the BCR-ABL gene achieved a complete hematologic response to bosutinib monotherapy; however no patients achieved a major cytogenetic response.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>520, \"name\"=>\"Bosutinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1997, \"name\"=>\"Bosutinib is active in chronic phase chronic myeloid leukemia after imatinib and dasatinib and/or nilotinib therapy failure.\", \"citation\"=>\"Khoury et al., 2012, Blood\", \"pubmed_id\"=>\"22371878\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22371878\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4916559\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>4, \"day\"=>12}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4361, \"name\"=>\"EID4361\", \"description\"=>\"In an in vitro study, a Ba/F3 murine cell line stably expressing the T315I mutation in the ABL1 portion of the BCR-ABL1 fusion protein was associated with reduced sensitivity to bosutinib treatment (IC50: 1890nM vs. IC50: 41.61nM) as compared to Ba/F3 cells expressing BCR-ABL1 fusion protein. Resistance was assessed by determining cell proliferation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>520, \"name\"=>\"Bosutinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1529, \"name\"=>\"Activity of bosutinib, dasatinib, and nilotinib against 18 imatinib-resistant BCR/ABL mutants.\", \"citation\"=>\"Redaelli et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19075254\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19075254\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>4371, \"name\"=>\"EID4371\", \"description\"=>\"In an in vitro study, pro-B Ba/F3 cell line expressing T315I mutation in the ABL1 portion of the BCR-ABL1 fusion protein (overexpression) demonstrated increased sensitivity to axitinib treatment (IC50: 145.7nM vs. 811.2nM), compared to Ba/F3 expressing BCR-ABL fusion protein (overexpression). Sensitivity was determined by assessing cell viability.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>519, \"name\"=>\"Axitinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2090, \"name\"=>\"Extreme mutational selectivity of axitinib limits its potential use as a targeted therapeutic for BCR-ABL1-positive leukemia.\", \"citation\"=>\"Zabriskie et al., 2016, Leukemia\", \"pubmed_id\"=>\"26582647\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26582647\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4873472\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>6}, \"journal\"=>\"Leukemia\", \"full_journal_title\"=>\"Leukemia\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>2668, \"name\"=>\"EID2668\", \"description\"=>\"In a study with 144 chronic myeloid leukemia (CML) patients, mutations in the BCR-ABL1 fusion gene were detected in 27 samples. Resistance to first-line chemotherapy drug imatinib was observed in 24 of the 27 patients with mutations.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1530, \"name\"=>\"Detection of BCR-ABL mutations in patients with CML treated with imatinib is virtually always accompanied by clinical resistance, and mutations in the ATP phosphate-binding loop (P-loop) are associated with a poor prognosis.\", \"citation\"=>\"Branford et al., 2003, Blood\", \"pubmed_id\"=>\"12623848\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/12623848\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2003, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>2669, \"name\"=>\"EID2669\", \"description\"=>\"In two cell-based studies, a panel of either 18 or 16 mutations representing 85% of known mutations seen in imatinib CML patents were used to evaluate the effectiveness of drug binding. All mutations were compared with non-mutated BCR-ABL protein and were grouped according to their anti-proliferative activity. Mutant cell lines that exhibited a greater than 4-fold increase in IC50 value were considered resistant to drug, 2-4 fold increase in IC50 were moderately resistant and less than 2-fold change in IC50 was sensitive to drug treatment. The T315I mutation was  resistant to imatinib .\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1531, \"name\"=>\"In vitro activity of Bcr-Abl inhibitors AMN107 and BMS-354825 against clinically relevant imatinib-resistant Abl kinase domain mutants.\", \"citation\"=>\"O'Hare et al., 2005, Cancer Res.\", \"pubmed_id\"=>\"15930265\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15930265\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>6, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>2673, \"name\"=>\"EID2673\", \"description\"=>\"In an in vitro study, a Ba/F3 cell line expressing a T315I mutation in the ABL1 portion of the BCR-ABL1 fusion protein, demonstrated sensitivity to ponatinib treatment (IC50: 11nM). Sensitivity was determined by assessing cell proliferation.\", \"disease\"=>{\"id\"=>43, \"name\"=>\"Acute Lymphocytic Leukemia\", \"display_name\"=>\"Acute Lymphocytic Leukemia\", \"doid\"=>\"9952\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9952\"}, \"drugs\"=>[{\"id\"=>55, \"name\"=>\"Ponatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1629, \"name\"=>\"AP24534, a pan-BCR-ABL inhibitor for chronic myeloid leukemia, potently inhibits the T315I mutant and overcomes mutation-based resistance.\", \"citation\"=>\"O'Hare et al., 2009, Cancer Cell\", \"pubmed_id\"=>\"19878872\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19878872\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2804470\", \"publication_date\"=>{\"year\"=>2009, \"month\"=>11, \"day\"=>6}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>2672, \"name\"=>\"EID2672\", \"description\"=>\"In a prospective study (NCT01207440) of 449 previously treated chronic myeloid leukemia (CML) and Philadelphia chromosome-positive acute lymphoblastic leukemia patients, chronic-phase CML patients (n=267) harboring a T315I mutation in the ABL1 portion of the  gene fusion (64/267) were associated with an improved response to ponatinib monotherapy; 66% and 56% of patients experienced a complete cytogenetic response and a major molecular response, respectively.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>55, \"name\"=>\"Ponatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1624, \"name\"=>\"A phase 2 trial of ponatinib in Philadelphia chromosome-positive leukemias.\", \"citation\"=>\"Cortes et al., 2013, N. Engl. J. Med.\", \"pubmed_id\"=>\"24180494\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24180494\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3886799\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>11, \"day\"=>7}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>2671, \"name\"=>\"EID2671\", \"description\"=>\"In an in vitro study, a Ba/F3 cell line stably expressing the T315I mutation in the ABL1 portion of the BCR-ABL1 fusion protein was associated with resistance to nilotinib treatment (IC50: 2000.0 nM vs. > 2000.0 nM), comparable to Ba/F3 parental cells. Sensitivity was determined by assessing cell proliferation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>467, \"name\"=>\"Nilotinib\", \"pubchem_id\"=>\"\"}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1621, \"name\"=>\"Comparison of imatinib, dasatinib, nilotinib and INNO-406 in imatinib-resistant cell lines.\", \"citation\"=>\"Deguchi et al., 2008, Leuk. Res.\", \"pubmed_id\"=>\"18191450\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18191450\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2008, \"month\"=>6}, \"journal\"=>\"Leuk. Res.\", \"full_journal_title\"=>\"Leukemia research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>2670, \"name\"=>\"EID2670\", \"description\"=>\"In an in vitro study, a Ba/F3  cell line stably expressing the T315I mutation in the ABL1 portion of the BCR-ABL1 fusion protein was associated with resistance to nilotinib treatment (IC50: 39.41-fold), as compared to Ba/F3 cells expressing BCR-ABL1 fusion protein (reported to be sensitizing to nilotinib). Sensitivity was determined by assessing cell proliferation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>13, \"name\"=>\"Nilotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1529, \"name\"=>\"Activity of bosutinib, dasatinib, and nilotinib against 18 imatinib-resistant BCR/ABL mutants.\", \"citation\"=>\"Redaelli et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19075254\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19075254\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>1390, \"name\"=>\"EID1390\", \"description\"=>\"Study describes a phase 1 clinical trial of ponatinib in treatment of 81 refractory Philadelphia chromosome-positive leukemias. 43 of these patients were considered to have chronic-phase CML. All 12 chronic-phase CML patients with a T315I mutation showed a complete hematologic response. Furthermore, 91% of these patients were estimated to  remain in response at 1 year (95% CI, Kaplan-Meier analysis).\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>55, \"name\"=>\"Ponatinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>942, \"name\"=>\"Ponatinib in refractory Philadelphia chromosome-positive leukemias.\", \"citation\"=>\"Cortes et al., 2012, N. Engl. J. Med.\", \"pubmed_id\"=>\"23190221\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23190221\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3777383\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>11, \"day\"=>29}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>235, \"name\"=>\"EID235\", \"description\"=>\"COS7 cell lines transfected with BCR-ABL constructs harboring T315I mutations were shown to be resistant to imatinib and exhibited increased kinase activity.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>165, \"name\"=>\"The two major imatinib resistance mutations E255K and T315I enhance the activity of BCR/ABL fusion kinase.\", \"citation\"=>\"Yamamoto et al., 2004, Biochem. Biophys. Res. Commun.\", \"pubmed_id\"=>\"15194504\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15194504\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2004, \"month\"=>7, \"day\"=>9}, \"journal\"=>\"Biochem. Biophys. Res. Commun.\", \"full_journal_title\"=>\"Biochemical and biophysical research communications\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}, {\"id\"=>234, \"name\"=>\"EID234\", \"description\"=>\"In chronic myeloid leukemia patients with the ABL T315I mutation, tumors have shown to be resistant to imatinib treatment.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>156, \"name\"=>\"BCR-ABL tyrosine kinase inhibitors in the treatment of Philadelphia chromosome positive chronic myeloid leukemia: a review.\", \"citation\"=>\"An et al., 2010, Leuk. Res.\", \"pubmed_id\"=>\"20537386\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20537386\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>10}, \"journal\"=>\"Leuk. Res.\", \"full_journal_title\"=>\"Leukemia research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2, \"phenotypes\"=>[]}]}, {\"id\"=>3, \"entrez_name\"=>\"ABL1\", \"entrez_id\"=>25, \"name\"=>\"BCR-ABL E255K\", \"description\"=>\"While the efficacy of imatinib has revolutionized chronic myelogenous leukemia (CML) treatment, it is still not a cure-all. Both initial resistance and acquired resistance as a result of selection have been seen in a small subset of CML patients. The ABL kinase domain mutation E255K has been shown to be one such mutation that confers resistance to imatinib. Second generation TKI's (dasatinib and nilotinib) specific to BCR-ABL have shown efficacy in treating resistant cases.\", \"gene_id\"=>4, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>12.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>133738363, \"stop\"=>133738363, \"reference_bases\"=>\"G\", \"variant_bases\"=>\"A\", \"representative_transcript\"=>\"ENST00000318560.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4/summary/variants/3/summary#variant\", \"evidence_items\"=>[{\"id\"=>2662, \"name\"=>\"EID2662\", \"description\"=>\"In an in vitro study, Ba/F3 cell line stably expressing the E255K mutation in the ABL1 portion of the BCR-ABL1 fusion protein was associated with resistance to nilotinib treatment (IC50: 6.69-fold), as compared to Ba/F3 cells expressing BCR-ABL1 fusion protein (reported to be sensitizing to nilotinib). Sensitivity was determined by assessing cell proliferation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>13, \"name\"=>\"Nilotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1529, \"name\"=>\"Activity of bosutinib, dasatinib, and nilotinib against 18 imatinib-resistant BCR/ABL mutants.\", \"citation\"=>\"Redaelli et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19075254\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19075254\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>3, \"phenotypes\"=>[]}, {\"id\"=>4350, \"name\"=>\"EID4350\", \"description\"=>\"In a retrospective study of 1084 imatinib-resistant or -intolerant, chronic myeloid leukemia (chronic-phase) patients, patients with a mutation in the ABL1 portion of the BCR-ABL gene fusion were associated with a favorable complete hematologic response (CHR), major cytogenetic response (MCyR), complete cytogenetic response (CCyR), and major molecular response (MMR) to dasatinib monotherapy (CHR= 92% (362/402) vs. CHR= 90% (593/641), MCyR= 56% (225/402) vs. MCyR= 65% (414/614), CCyR= 44% (175/402) vs. CCyR= 56% (358/641), MMR= 33% (125/383) vs. MMR= 45% (274/609), comparable to patients without mutation in the ABL1 portion. In a retrospective study of 1084 imatinib-resistant or -intolerant, chronic myeloid leukemia (chronic-phase) patients, 6, 9, and 14 patients with the E255K mutation in the ABL1 portion of the BCR-ABL1 gene fusion achieved a complete hematologic response , major cytogenetic response, and complete cytogenetic response  to dasatinib monotherapy, respectively.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1532, \"name\"=>\"Dasatinib treatment of chronic-phase chronic myeloid leukemia: analysis of responses according to preexisting BCR-ABL mutations.\", \"citation\"=>\"Müller et al., 2009, Blood\", \"pubmed_id\"=>\"19779040\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19779040\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4916940\", \"publication_date\"=>{\"year\"=>2009, \"month\"=>12, \"day\"=>3}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"submitted\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>3, \"phenotypes\"=>[]}, {\"id\"=>4353, \"name\"=>\"EID4353\", \"description\"=>\"In a retrospective study of 84 imatinib-resistant or -intolerant Philadelphia chromosome BCR-ABL1 fusion positive (Ph+) chronic myeloid leukemia (in accelerated-phase or with blast crisis) or Ph+ acute lymphoblastic leukemia patients, patients with the E255K mutation in the ABL1 portion of the BCR-ABL1 gene fusion were associated with a  complete cytogenetic response to dasatinib monotherapy.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2096, \"name\"=>\"Dasatinib in imatinib-resistant Philadelphia chromosome-positive leukemias.\", \"citation\"=>\"Talpaz et al., 2006, N. Engl. J. Med.\", \"pubmed_id\"=>\"16775234\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16775234\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2006, \"month\"=>6, \"day\"=>15}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>3, \"phenotypes\"=>[]}, {\"id\"=>4351, \"name\"=>\"EID4351\", \"description\"=>\"In a retrospective study of 174 imatinib-resistant or -intolerant Philadelphia chromosome BCR-ABL1 fusion positive accelerated phase chronic myeloid leukemia patients, patients with mutations in the ABL1 portion of the BCR-ABL1 gene fusion were associated with an improved major hematologic response (MaHR) to dasatinib monotherapy (MaHR=73% (64/80) vs. MaHR=55% (38/68) as compared to patients without mutation in the ABL1 portion (n=45). In a retrospective study of 174 imatinib-resistant or -intolerant Philadelphia chromosome BCR-ABL1 fusion positive accelerated phase chronic myeloid leukemia patients, 5 and 1 patient achieved a major hematologic response or major cytogenetic response to dasatinib monotherapy, respectively, of 8 patients with the E255K mutation in the ABL1 portion of the BCR-ABL1 gene fusion.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1999, \"name\"=>\"Dasatinib in the treatment of chronic myeloid leukemia in accelerated phase after imatinib failure: the START a trial.\", \"citation\"=>\"Apperley et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19487385\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19487385\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4979080\", \"publication_date\"=>{\"year\"=>2009, \"month\"=>7, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>3, \"phenotypes\"=>[]}, {\"id\"=>4352, \"name\"=>\"EID4352\", \"description\"=>\"In a retrospective study of 107 imatinib-resistant or -intolerant, Philadelphia chromosome BCR-ABL1 fusion positive, chronic myeloid leukemia (accelerated phase) patients, patients with mutations in the ABL1 portion of the BCR-ABL gene fusion were associated with an improved major hematologic response (MaHR) to dasatinib monotherapy (MaHR=73% (44/60) vs. MaHR=55% (22/40), as compared to patients without mutation in the ABL1 portion (n=45). In a retrospective study of 107 imatinib-resistant or -intolerant, Philadelphia chromosome BCR-ABL1 fusion positive, accelerated phase chronic myeloid leukemia patients, 4 and 0  patients achieved a major hematologic response or major cytogenetic response to dasatinib monotherapy respectively, of 6 patients with the E255K mutation in the ABL1 portion of the BCR-ABL gene fusion.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1998, \"name\"=>\"Dasatinib induces significant hematologic and cytogenetic responses in patients with imatinib-resistant or -intolerant chronic myeloid leukemia in accelerated phase.\", \"citation\"=>\"Guilhot et al., 2007, Blood\", \"pubmed_id\"=>\"17264298\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17264298\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>5, \"day\"=>15}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>3, \"phenotypes\"=>[]}, {\"id\"=>4348, \"name\"=>\"EID4348\", \"description\"=>\"In a retrospective study of 288 imatinib-resistant or -intolerant, Philadelphia chromosome BCR-ABL1 fusion positive, chronic myeloid leukemia (chronic phase) patients, patients with mutations in the ABL1 portion of the BCR-ABL gene fusion were associated with an complete hematologic response (MaHR) or complete major cytogenic response (MCyR) to bosutinib monotherapy (MaHR=97% (33/34) vs. MaHR=100% (27/27), MCyR=69% (41/59) vs. MCyR=54% (19/35), comparable to patients without mutation in the ABL1 portion. In a retrospective study of 288 imatinib-resistant or -intolerant, Philadelphia chromosome BCR-ABL1 fusion positive, chronic myeloid leukemia (chronic phase) patients, 2/2 and 1/2 of evaluated patients with the E255K mutation in the ABL1 portion of the BCR-ABL gene fusion achieved a complete hematologic response or major cytogenetic response to bosutinib monotherapy respectively.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>520, \"name\"=>\"Bosutinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1996, \"name\"=>\"Safety and efficacy of bosutinib (SKI-606) in chronic phase Philadelphia chromosome-positive chronic myeloid leukemia patients with resistance or intolerance to imatinib.\", \"citation\"=>\"Cortes et al., 2011, Blood\", \"pubmed_id\"=>\"21865346\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21865346\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4916618\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>10, \"day\"=>27}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>3, \"phenotypes\"=>[]}, {\"id\"=>4355, \"name\"=>\"EID4355\", \"description\"=>\"In an in vitro study, a pro-B Ba/F3 cell line expressing E255K mutation in the ABL1 portion of the BCR-ABL1 fusion protein (overexpression) demonstrated resistance to axitinib treatment (IC50: 1282nM), similar to Ba/F3 expressing the BCR-ABL1 fusion protein (overexpression, IC50: 823nM). Sensitivity was determined by assessing cell proliferation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>519, \"name\"=>\"Axitinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2002, \"name\"=>\"Axitinib effectively inhibits BCR-ABL1(T315I) with a distinct binding conformation.\", \"citation\"=>\"Pemovska et al., 2015, Nature\", \"pubmed_id\"=>\"25686603\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25686603\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>3, \"day\"=>5}, \"journal\"=>\"Nature\", \"full_journal_title\"=>\"Nature\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>3, \"phenotypes\"=>[]}, {\"id\"=>4354, \"name\"=>\"EID4354\", \"description\"=>\"In an in vitro study, a Ba/F3 cell line stably expressing the E255K mutation in the ABL1 portion of the BCR-ABL1 fusion protein was found to have increased sensitivity to dasatinib treatment, as compared to Ba/F3 cells expressing BCR-ABL1 fusion protein. Sensitivity was assessed by determining cell proliferation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2001, \"name\"=>\"Overriding imatinib resistance with a novel ABL kinase inhibitor.\", \"citation\"=>\"Shah et al., 2004, Science\", \"pubmed_id\"=>\"15256671\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15256671\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2004, \"month\"=>7, \"day\"=>16}, \"journal\"=>\"Science\", \"full_journal_title\"=>\"Science (New York, N.Y.)\", \"status\"=>\"submitted\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>3, \"phenotypes\"=>[]}, {\"id\"=>4349, \"name\"=>\"EID4349\", \"description\"=>\"In an in vitro study, a Ba/F3 cell line expressing an E255K mutation in the ABL1 portion of the BCR-ABL1 fusion protein was associated with sensitivity to dasatinib treatment (5.6 nmol/L vs. 0.8 nmol/L), comparable to Ba/F3 cells expressing the BCR-ABL1 fusion protein. Sensitivity was determined by assessing cell proliferation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1531, \"name\"=>\"In vitro activity of Bcr-Abl inhibitors AMN107 and BMS-354825 against clinically relevant imatinib-resistant Abl kinase domain mutants.\", \"citation\"=>\"O'Hare et al., 2005, Cancer Res.\", \"pubmed_id\"=>\"15930265\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15930265\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>6, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>3, \"phenotypes\"=>[]}, {\"id\"=>4347, \"name\"=>\"EID4347\", \"description\"=>\"In a retrospective study of 118 imatinib-resistant or -intolerant, Philadelphia chromosome BCR-ABL1 fusion positive, chronic myeloid leukemia (chronic phase) patients, no response to bosutinib monotherapy was detected, in the 1  patient with the E255K mutation in the ABL1 portion of the BCR-ABL gene fusion.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>520, \"name\"=>\"Bosutinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1997, \"name\"=>\"Bosutinib is active in chronic phase chronic myeloid leukemia after imatinib and dasatinib and/or nilotinib therapy failure.\", \"citation\"=>\"Khoury et al., 2012, Blood\", \"pubmed_id\"=>\"22371878\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22371878\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4916559\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>4, \"day\"=>12}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>3, \"phenotypes\"=>[]}, {\"id\"=>4346, \"name\"=>\"EID4346\", \"description\"=>\"In an in vitro study, a Ba/F3 murine cell line stably expressing the E255K mutation in the ABL1 portion of the BCR-ABL1 fusion protein was associated with reduced sensitivity to bosutinib treatment (IC50: 394nM vs. IC50: 41.61nM), as compared to Ba/F3 cells expressing BCR-ABL1 fusion protein. Resistance was assessed by determining cell proliferation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>520, \"name\"=>\"Bosutinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1529, \"name\"=>\"Activity of bosutinib, dasatinib, and nilotinib against 18 imatinib-resistant BCR/ABL mutants.\", \"citation\"=>\"Redaelli et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19075254\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19075254\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>3, \"phenotypes\"=>[]}, {\"id\"=>2659, \"name\"=>\"EID2659\", \"description\"=>\"A panel of 18 mutations representing 85% of known mutations seen in imatinib CML patents were used to evaluate the effectiveness of drug binding. All mutations were compared with non-mutated BCR-ABL protein and were grouped according to their anti-proliferative activity. Mutant cell lines that exhibited a greater than 4-fold increase in IC50 value were considered resistant to drug, 2-4 fold increase in IC50 were moderately resistant and less than 2-fold change in IC50 was sensitive to drug treatment. The E255K mutation was resistant to imatinib.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1529, \"name\"=>\"Activity of bosutinib, dasatinib, and nilotinib against 18 imatinib-resistant BCR/ABL mutants.\", \"citation\"=>\"Redaelli et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19075254\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19075254\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>3, \"phenotypes\"=>[]}, {\"id\"=>2660, \"name\"=>\"EID2660\", \"description\"=>\"In a study with 144 chronic myeloid leukemia (CML) patients, mutations in the BCR-ABL1 fusion gene were detected in 27 samples. Resistance to first-line chemotherapy drug imatinib was observed in 24 of the 27 patients with mutations. There was reduced overall survival in patients with a mutation in the phosphate binding (P-Loop) region (92%, n=13), with a decreased median survival of 4.5 months post detection of mutation compared with non-P loop mutations (4.5mo vs. 10.5mo).\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1530, \"name\"=>\"Detection of BCR-ABL mutations in patients with CML treated with imatinib is virtually always accompanied by clinical resistance, and mutations in the ATP phosphate-binding loop (P-loop) are associated with a poor prognosis.\", \"citation\"=>\"Branford et al., 2003, Blood\", \"pubmed_id\"=>\"12623848\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/12623848\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2003, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>3, \"phenotypes\"=>[]}, {\"id\"=>2661, \"name\"=>\"EID2661\", \"description\"=>\"A panel of 16 mutations seen in imatinib CML patents were used to evaluate the effectiveness of drug binding. All mutations were compared with non-mutated BCR-ABL protein and evaluated for anti-proliferative activity and substrate or auto-phosphorylation assays. The E255K mutation was resistant to imatinib (20-fold IC50 increase for the proliferation assay).\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1531, \"name\"=>\"In vitro activity of Bcr-Abl inhibitors AMN107 and BMS-354825 against clinically relevant imatinib-resistant Abl kinase domain mutants.\", \"citation\"=>\"O'Hare et al., 2005, Cancer Res.\", \"pubmed_id\"=>\"15930265\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15930265\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>6, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>3, \"phenotypes\"=>[]}, {\"id\"=>233, \"name\"=>\"EID233\", \"description\"=>\"COS7 cell lines transfected with BCR-ABL constructs harboring E255K mutations were shown to be resistant to imatinib and exhibited increased kinase activity.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>165, \"name\"=>\"The two major imatinib resistance mutations E255K and T315I enhance the activity of BCR/ABL fusion kinase.\", \"citation\"=>\"Yamamoto et al., 2004, Biochem. Biophys. Res. Commun.\", \"pubmed_id\"=>\"15194504\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15194504\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2004, \"month\"=>7, \"day\"=>9}, \"journal\"=>\"Biochem. Biophys. Res. Commun.\", \"full_journal_title\"=>\"Biochemical and biophysical research communications\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>3, \"phenotypes\"=>[]}]}, {\"id\"=>5, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"EML4-ALK\", \"description\"=>\"The EML4-ALK fusion variant 1 consisting of ALK kinase domain (exons 20-29) fused to EML4 exons 1-13 is the most common EML4-ALK variant, and was discovered in non-small cell lung cancer. Multiple EML4 breakpoint shave been described with differential sensitivity to inhibitors with variant 1 showing greater sensitivity than 3a in cell lines. EML4-ALK is crizotinib sensitive; however, several mutations that confer resistance mutations have been described in case studies. In the only clinical trial for crizotinib that included determination of EML4-ALK variant type in a subset of its participants, a very high response rate was observed, although the numbers were insufficient to validate correlation of variant type to outcome. Preclinical studies with this variant have indicated sensitivity to Hsp90 inhibitors.\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>48.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>42396490, \"stop\"=>42522656, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000318522.5\", \"chromosome2\"=>\"2\", \"start2\"=>29415640, \"stop2\"=>29446394, \"representative_transcript2\"=>\"ENST00000389048.3\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/5/summary#variant\", \"evidence_items\"=>[{\"id\"=>1332, \"name\"=>\"EID1332\", \"description\"=>\"EML4-ALK variant 1 containing H-3122 NSCLC cells were treated with second generation ALK inhibitor PF-06463922. PF-06463922 was 125x more potent at reducing cell viability than crizotinib. Control neuroblastoma cell lines with wild-type ALK did not show inhibition below 3800 nM with either drug. At values of 10 and 100 nM, PF-06463922 showed increased ability to inhibit ALK 1278 tyrosine phosphorylation in H3122 cells.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>241, \"name\"=>\"Lorlatinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>919, \"name\"=>\"The ALK/ROS1 Inhibitor PF-06463922 Overcomes Primary Resistance to Crizotinib in ALK-Driven Neuroblastoma.\", \"citation\"=>\"Infarinato et al., 2016, Cancer Discov\", \"pubmed_id\"=>\"26554404\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26554404\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4707106\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>1}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>5, \"phenotypes\"=>[]}, {\"id\"=>1207, \"name\"=>\"EID1207\", \"description\"=>\"In the PROFILE 1001 study of crizotinib in 82 patients with ALK-rearranged NSCLC, 31 patients with sufficient tumor material were tested using RT-PCR for the presence of certain EML4-ALK variants.  13/31 tested patients had the EML4-ALK variant 1. Of these, 12/13 had a partial response, and 1 had stable disease. This response rate of 92% is in contrast to the 57% response rate in the overall patient population, but the numbers of patients with specific EML4-ALK breakpoints were insufficient to correlate with patient response rate.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>819, \"name\"=>\"Anaplastic lymphoma kinase inhibition in non-small-cell lung cancer.\", \"citation\"=>\"Kwak et al., 2010, N. Engl. J. Med.\", \"pubmed_id\"=>\"20979469\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20979469\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3014291\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>10, \"day\"=>28}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>5, \"phenotypes\"=>[]}, {\"id\"=>1206, \"name\"=>\"EID1206\", \"description\"=>\"Transduction of Ba/F3 cells with EML4-ALK variant 1 causes IL3 independent growth. Incubation of EML4-ALK variant 1 transduced Ba/F3 cells with Hsp90 inhibitor 17-DMAG caused cell death that was reversible with IL3 addition, suggesting that 17-DMAG was effecting the EML4-ALK driver of IL3-independant proliferation, and that EML4-ALK variant 1 is a client protein of Hsp90. The degree of Hsp90 dependence, and targetability of EML4-ALK for Hsp90 inhibition, may vary among ALK fusion variants.  Addition of crizotinib to 17-DMAG caused synergistic death effects, suggesting potential clinical benefit for this inhibitor combination in targeting EML4-ALK variant 1.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}, {\"id\"=>304, \"name\"=>\"17-DMAG\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>826, \"name\"=>\"Differential protein stability and ALK inhibitor sensitivity of EML4-ALK fusion variants.\", \"citation\"=>\"Heuckmann et al., 2012, Clin. Cancer Res.\", \"pubmed_id\"=>\"22912387\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22912387\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>9, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>5, \"phenotypes\"=>[]}, {\"id\"=>1203, \"name\"=>\"EID1203\", \"description\"=>\"In preclinical work using H3122 NSCLC cells which contain the EML4-ALK variant 1, administration of the Hsp90 inhibitor IPI-504 caused marked decrease of EML4-ALK protein levels at concentrations lower than those used to knock down protein levels of drivers such as HER2 or EGFR in other cell lines, making it the most sensitive Hsp90 client protein the authors had encountered.  IPI-504 treatment caused regression of H3122 tumors in nude mice, whereas Hsp90 inhibitors usually only cause growth inhibition in these models.  Combined treatment of these mice with IPI-504 and crizotinib caused an even stronger tumor regression.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}, {\"id\"=>303, \"name\"=>\"IPI-504\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>832, \"name\"=>\"The Hsp90 inhibitor IPI-504 rapidly lowers EML4-ALK levels and induces tumor regression in ALK-driven NSCLC models.\", \"citation\"=>\"Normant et al., 2011, Oncogene\", \"pubmed_id\"=>\"21258415\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21258415\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>6, \"day\"=>2}, \"journal\"=>\"Oncogene\", \"full_journal_title\"=>\"Oncogene\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>5, \"phenotypes\"=>[]}, {\"id\"=>1188, \"name\"=>\"EID1188\", \"description\"=>\"A retroviral cDNA expression library was constructed using a lung adenocarcinoma specimen from a 67 year old NSCLC patient, and transfected into mouse 3T3 fibroblasts which were assessed for transformed foci.  Analysis of clones from one transformed focus yielded a novel fusion. This fusion generated a protein with N-terminal consisting of EML4 exons 1-13 and C-terminal consisting of ALK exons 20-29, which comprise the intracellular kinase domain of ALK. In subsequent experiments using EML4-ALK or altered versions, the transforming potential of EML4-ALK was found to be dependent on a functional ALK kinase activity as well as the EML4 basic domain, which was shown to have an important role in EML4-ALK dimer formation.  BA/F3 cells expressing EML4-ALK were able to grow independently of IL3 addition, but addition of ALK inhibitor WHI-P154 induced cell death and inhibited tyrosine phosphorylation of EML4-ALK. These results indicated that EML4-ALK may be a driver mutation in NSCLC,  and a target for ALK inhibitor treatment.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>301, \"name\"=>\"WHI-P154\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>820, \"name\"=>\"Identification of the transforming EML4-ALK fusion gene in non-small-cell lung cancer.\", \"citation\"=>\"Soda et al., 2007, Nature\", \"pubmed_id\"=>\"17625570\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17625570\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>8, \"day\"=>2}, \"journal\"=>\"Nature\", \"full_journal_title\"=>\"Nature\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>5, \"phenotypes\"=>[]}, {\"id\"=>1121, \"name\"=>\"EID1121\", \"description\"=>\"A 39-year-old man presented was observed to have lung acinar adenocarcinoma. Treatment with chemotherapy produced no remarkable response. A EGFR L858R point mutation was observed; the patient was therefore treated with erlotinib, but the disease progressed after 30 days. A second biopsy identified a EML4-ALK fusion, which may explain the therapeutic resistance.\", \"disease\"=>{\"id\"=>1594, \"name\"=>\"Lung Acinar Adenocarcinoma\", \"display_name\"=>\"Lung Acinar Adenocarcinoma\", \"doid\"=>\"6482\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:6482\"}, \"drugs\"=>[{\"id\"=>15, \"name\"=>\"Erlotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>776, \"name\"=>\"A case of lung adenocarcinoma harboring EGFR mutation and EML4-ALK fusion gene.\", \"citation\"=>\"Tanaka et al., 2012, BMC Cancer\", \"pubmed_id\"=>\"23181703\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23181703\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3515412\", \"publication_date\"=>{\"year\"=>2012}, \"journal\"=>\"BMC Cancer\", \"full_journal_title\"=>\"BMC cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>5, \"phenotypes\"=>[]}, {\"id\"=>262, \"name\"=>\"EID262\", \"description\"=>\"In patients with non-small cell lung cancer harboring the EML4-ALK fusion, treatment with crizotinib has shown to be effective in acheiving stable disease.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>166, \"name\"=>\"EML4-ALK mutations in lung cancer that confer resistance to ALK inhibitors.\", \"citation\"=>\"Choi et al., 2010, N. Engl. J. Med.\", \"pubmed_id\"=>\"20979473\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20979473\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>10, \"day\"=>28}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>5, \"phenotypes\"=>[]}]}, {\"id\"=>6, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"EML4-ALK C1156Y\", \"description\"=>\"In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, C1156Y has been shown to confer resistance to crizotinib. Case reports indicate that secondary mutations can modulate drug sensitivity. EML4-ALK C1156Y/L1196M maintained crizotinib resistance while the lorlatinib resistant combination EML4-ALK C1156Y/L1198F re-sensitized the tumor to crizotinib treatment.\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>19.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>29445258, \"stop\"=>29445258, \"reference_bases\"=>\"C\", \"variant_bases\"=>\"T\", \"representative_transcript\"=>\"ENST00000389048.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/6/summary#variant\", \"evidence_items\"=>[{\"id\"=>4046, \"name\"=>\"EID4046\", \"description\"=>\"In a lung adenocarcinoma cancer patient with an EML4-ALK gene rearrangement (a known sensitizing alteration to crizotinib) and an ALK C1156Y co-mutation, ALK C1156Y was reported to be refractory to crizotinib treatment. The patient was treated with two courses of cisplatin and pemetrexed and then crizotinib, but then experience disease progression.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2038, \"name\"=>\"Genomic Aberrations in Crizotinib Resistant Lung Adenocarcinoma Samples Identified by Transcriptome Sequencing.\", \"citation\"=>\"Saber et al., 2016, PLoS ONE\", \"pubmed_id\"=>\"27045755\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27045755\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4821611\", \"publication_date\"=>{\"year\"=>2016}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"submitted\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>6, \"phenotypes\"=>[]}, {\"id\"=>4043, \"name\"=>\"EID4043\", \"description\"=>\"In an in vitro study, H3122 cells expressing EML4-ALK fusion (a known sensitizing alteration to crizotinib) and an ALK C1156Y co-mutation demonstrated resistance to crizotinib treatment (240 nmol/L vs. 70 nmol/L) compared to H3122 cells expressing EML4-ALK fusion. Resistance was determined by assessing cell viability.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>506, \"name\"=>\"A novel ALK secondary mutation and EGFR signaling cause resistance to ALK kinase inhibitors.\", \"citation\"=>\"Sasaki et al., 2011, Cancer Res.\", \"pubmed_id\"=>\"21791641\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21791641\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3278914\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>9, \"day\"=>15}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>6, \"phenotypes\"=>[]}, {\"id\"=>4042, \"name\"=>\"EID4042\", \"description\"=>\"In a non-small cell lung cancer patient with a liver metastatic tumor harboring an ALK gene rearrangement and an ALK C1156Y co-mutation, ALK C1156Y was reported to be refractory to 7-month crizotinib treatment.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2035, \"name\"=>\"Multiplexed deep sequencing analysis of ALK kinase domain identifies resistance mutations in relapsed patients following crizotinib treatment.\", \"citation\"=>\"Huang et al., 2013, Genomics\", \"pubmed_id\"=>\"23434628\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23434628\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>9}, \"journal\"=>\"Genomics\", \"full_journal_title\"=>\"Genomics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>6, \"phenotypes\"=>[]}, {\"id\"=>4045, \"name\"=>\"EID4045\", \"description\"=>\"In an in vitro study, a Ba/F3 cell line expressing EML4-ALK fusion (a known sensitizing alteration to crizotinib) and an ALK C1156Y co-mutation demonstrated resistance to crizotinib treatment compared to Ba/F3 cells expressing EML4-ALK fusion. Resistance was determined by assessing cell viability and ALK auto-phosphorylation .\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>166, \"name\"=>\"EML4-ALK mutations in lung cancer that confer resistance to ALK inhibitors.\", \"citation\"=>\"Choi et al., 2010, N. Engl. J. Med.\", \"pubmed_id\"=>\"20979473\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20979473\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>10, \"day\"=>28}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>6, \"phenotypes\"=>[]}, {\"id\"=>1346, \"name\"=>\"EID1346\", \"description\"=>\"EML4-ALK variant 3 containing NSCLC cell line H2228 xenografts in SCID beige mice were treated with increasing doses of crizotinib up to 100 mg/kg per day by oral gavage. At approximately 40 days tumors progressed. 80 progressed tumors were tested for ALK mutations, and 6 mutant ALK variants were found. Two of the mutations were at ALK C1156Y. In Ba/F3 cells, EML4-ALK variant 3 with C1156Y was introduced and tested for crizotinib sensitivity. Survival IC50 value of 177 nM was reported, which is 9.3 times the IC50 reported for crizotinib in EML4-ALK wild-type Ba/F3 cells.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>924, \"name\"=>\"The ALK inhibitor ceritinib overcomes crizotinib resistance in non-small cell lung cancer.\", \"citation\"=>\"Friboulet et al., 2014, Cancer Discov\", \"pubmed_id\"=>\"24675041\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24675041\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4068971\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>6, \"phenotypes\"=>[]}, {\"id\"=>4044, \"name\"=>\"EID4044\", \"description\"=>\"In an in vitro study, a Ba/F3 cell line expressing EML4-ALK  fusion (a known sensitizing alteration to crizotinib) and an ALK C1156Y co-mutation demonstrated resistance to crizotinib treatment (180 nmol/L vs. 22 nmol/L) compared to Ba/F3 cells expressing EML4-ALK fusion. Resistance was determined by assessing cell viability and colony formation.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>507, \"name\"=>\"Mechanisms of resistance to crizotinib in patients with ALK gene rearranged non-small cell lung cancer.\", \"citation\"=>\"Doebele et al., 2012, Clin. Cancer Res.\", \"pubmed_id\"=>\"22235099\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22235099\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3311875\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>3, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>6, \"phenotypes\"=>[]}, {\"id\"=>842, \"name\"=>\"EID842\", \"description\"=>\"Case report of a patient with metastatic ALK-rearranged NSCLC after progression under sequential crizotinib, ceritinib, HSP90 inhibitor AUY922 and chemotherapy followed by crizotinib again. Third generation ALK inhibitor lorlatinib led to a partial response that lasted for 8 months. The C1156Y mutation was confirmed in a sample prior to initial crizotinib (<7% of cells), following crizotinib treatment (~50% of cells) and following lorlatinib resistance (100% of cells). However, a second ALK mutation (C1156Y & L1198F) was detected in the lorlatinib resistant tumor.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>241, \"name\"=>\"Lorlatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>567, \"name\"=>\"Resensitization to Crizotinib by the Lorlatinib ALK Resistance Mutation L1198F.\", \"citation\"=>\"Shaw et al., 2016, N. Engl. J. Med.\", \"pubmed_id\"=>\"26698910\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26698910\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4773904\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>1, \"day\"=>7}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>6, \"phenotypes\"=>[]}, {\"id\"=>841, \"name\"=>\"EID841\", \"description\"=>\"In this case report, a patient with metastatic, ALK-rearranged NSCLC with C1156Y mutation had tumor progression under sequential treatment with first crizotinib, then ceritinib and finally AUY922. Subsequent chemotherapy (carboplatin–pemetrexed) was administered and the patient had a response for 6 months before progression. Exome sequencing revealed the C1156Y mutation was present in <7% of the pretreatment tumor and in ~50% of the crizotinib tumor prior to ceritinib therapy.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}, {\"id\"=>159, \"name\"=>\"Ceritinib\", \"pubchem_id\"=>nil}, {\"id\"=>160, \"name\"=>\"AUY922\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Sequential\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>567, \"name\"=>\"Resensitization to Crizotinib by the Lorlatinib ALK Resistance Mutation L1198F.\", \"citation\"=>\"Shaw et al., 2016, N. Engl. J. Med.\", \"pubmed_id\"=>\"26698910\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26698910\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4773904\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>1, \"day\"=>7}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>6, \"phenotypes\"=>[]}]}, {\"id\"=>7, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"EML4-ALK L1196M\", \"description\"=>\"In patients with non-small cell lung cancer exhibiting EML4-ALK fusion, L1196M has been shown to confer resistance to crizotinib. This was also true in a patient with both EML4-ALK C1156Y & L1196M.\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>31.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>29443631, \"stop\"=>29443631, \"reference_bases\"=>\"G\", \"variant_bases\"=>\"T\", \"representative_transcript\"=>\"ENST00000389048.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/7/summary#variant\", \"evidence_items\"=>[{\"id\"=>4040, \"name\"=>\"EID4040\", \"description\"=>\"In a stage IV Hodgkin's lymphoma patient with metastatic lung adenocarcinoma harboring an EML4-ALK gene rearrangement (a known sensitizing alteration to crizotinib) and an ALK L1196M�co-mutation, ALK L1196M mutation�was associated with resistance to crizotinib monotherapy. Prior to identification of the ALK L1196M mutation, the patient was treated with crizotinib and had a near complete resolution of disease. Subsequently, the patient was treated with ceritinib and disease progressed after 11 months.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2039, \"name\"=>\"Expanded Circulating Tumor Cells from a Patient with ALK-Positive Lung Cancer Present with EML4-ALK Rearrangement Along with Resistance Mutation and Enable Drug Sensitivity Testing: A Case Study.\", \"citation\"=>\"Zhang et al., 2017, J Thorac Oncol\", \"pubmed_id\"=>\"27507192\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27507192\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5263096\", \"publication_date\"=>{\"year\"=>2017, \"month\"=>2}, \"journal\"=>\"J Thorac Oncol\", \"full_journal_title\"=>\"Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>7, \"phenotypes\"=>[]}, {\"id\"=>4039, \"name\"=>\"EID4039\", \"description\"=>\"In an in vitro study, a H3122 cell line expressing L1196M mutation in the ALK portion of the EML4-ALK fusion protein was associated with resistance to crizotinib treatment, as compared to H3122 cells expressing EML4-ALK fusion. Resistance was determined by assessing cell viability, ALK, AKT, and ERK phosphorylation.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2037, \"name\"=>\"De novo ALK kinase domain mutations are uncommon in kinase inhibitor-naïve ALK rearranged lung cancers.\", \"citation\"=>\"Lucena-Araujo et al., 2016, Lung Cancer\", \"pubmed_id\"=>\"27565908\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27565908\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5002311\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>9}, \"journal\"=>\"Lung Cancer\", \"full_journal_title\"=>\"Lung cancer (Amsterdam, Netherlands)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>7, \"phenotypes\"=>[]}, {\"id\"=>4038, \"name\"=>\"EID4038\", \"description\"=>\"In an in vitro study, a Ba/F3 cell line expressing EML4-ALK fusion (a known sensitizing alteration to crizotinib) and an ALK L1196M co-mutation, demonstrated resistance to crizotinib treatment compared to Ba/F3 cells expressing EML4-ALK fusion. Resistance was determined by assessing cell viability  .\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>78, \"name\"=>\"ALK mutations conferring differential resistance to structurally diverse ALK inhibitors.\", \"citation\"=>\"Heuckmann et al., 2011, Clin. Cancer Res.\", \"pubmed_id\"=>\"21948233\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21948233\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3382103\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>12, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>7, \"phenotypes\"=>[]}, {\"id\"=>4037, \"name\"=>\"EID4037\", \"description\"=>\"In an in vitro study, a Ba/F3 cell line expressing EML4-ALK fusion (a known sensitizing alteration to crizotinib) and an ALK L1196M co-mutation, demonstrated resistance to crizotinib treatment compared to Ba/F3 cells expressing EML4-ALK fusion. Resistance was determined by assessing cell viability  .\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2036, \"name\"=>\"Therapeutic strategies to overcome crizotinib resistance in non-small cell lung cancers harboring the fusion oncogene EML4-ALK.\", \"citation\"=>\"Katayama et al., 2011, Proc. Natl. Acad. Sci. U.S.A.\", \"pubmed_id\"=>\"21502504\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21502504\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3088626\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>5, \"day\"=>3}, \"journal\"=>\"Proc. Natl. Acad. Sci. U.S.A.\", \"full_journal_title\"=>\"Proceedings of the National Academy of Sciences of the United States of America\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>7, \"phenotypes\"=>[]}, {\"id\"=>4036, \"name\"=>\"EID4036\", \"description\"=>\"In a stage IV non-small cell lung cancer patient with an EML4-ALK gene rearrangement (a known sensitizing alteration to crizotinib) and an ALK L1196M co-mutation, ALK L1196M was reported to be refractory to crizotinib treatment. The patient was treated with standard chemotherapy, erlotinib and then crizotinib; the patient subsequently succumbed to progressive disease. In an in vitro study, a 293T cell line expressing L1196M mutation in the ALK portion of the EML4-ALK fusion protein was associated with resistance to crizotinib treatment, as compared to 293T cells expressing EML4-ALK fusion. Resistance was determined by assessing cell viability and ALK phosphorylation.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>508, \"name\"=>\"Heterogeneity of genetic changes associated with acquired crizotinib resistance in ALK-rearranged lung cancer.\", \"citation\"=>\"Kim et al., 2013, J Thorac Oncol\", \"pubmed_id\"=>\"23344087\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23344087\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>4}, \"journal\"=>\"J Thorac Oncol\", \"full_journal_title\"=>\"Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>7, \"phenotypes\"=>[]}, {\"id\"=>4035, \"name\"=>\"EID4035\", \"description\"=>\"In a non-small cell lung cancer patient with a liver metastatic tumor harboring an ALK gene rearrangement and an ALK L1196M co-mutation, ALK L1196M was reported to be refractory to 7-month crizotinib treatment.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2035, \"name\"=>\"Multiplexed deep sequencing analysis of ALK kinase domain identifies resistance mutations in relapsed patients following crizotinib treatment.\", \"citation\"=>\"Huang et al., 2013, Genomics\", \"pubmed_id\"=>\"23434628\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23434628\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>9}, \"journal\"=>\"Genomics\", \"full_journal_title\"=>\"Genomics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>7, \"phenotypes\"=>[]}, {\"id\"=>4034, \"name\"=>\"EID4034\", \"description\"=>\"In an in vitro study, a Ba/F3 cell line expressing EML4-ALK fusion (a known sensitizing alteration to crizotinib) and an ALK L1196M co-mutation, demonstrated resistance to crizotinib treatment (766 nmol/L vs. 22 nmol/L) compared to Ba/F3 cells expressing EML4-ALK fusion. Resistance was determined by assessing cell viability and colony formation.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>507, \"name\"=>\"Mechanisms of resistance to crizotinib in patients with ALK gene rearranged non-small cell lung cancer.\", \"citation\"=>\"Doebele et al., 2012, Clin. Cancer Res.\", \"pubmed_id\"=>\"22235099\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22235099\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3311875\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>3, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>7, \"phenotypes\"=>[]}, {\"id\"=>2343, \"name\"=>\"EID2343\", \"description\"=>\"In a prospective study (NCT01283516) of 130 cancer patients, non-small cell lung cancer patients (n=2) harboring a L1196M mutation in the ALK portion of the ALK rearrangement were associated with a 50% response to ceritinib monotherapy, respectively.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>159, \"name\"=>\"Ceritinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>856, \"name\"=>\"Ceritinib in ALK-rearranged non-small-cell lung cancer.\", \"citation\"=>\"Shaw et al., 2014, N. Engl. J. Med.\", \"pubmed_id\"=>\"24670165\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24670165\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4079055\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>3, \"day\"=>27}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>7, \"phenotypes\"=>[]}, {\"id\"=>1341, \"name\"=>\"EID1341\", \"description\"=>\"The crizotinib-resistant MGH045 cell line with EML4-ALK L1196M was treated with crizotinib or ceritinib. In nude mice with MGH045 xenografts, tumor response to crizotinib did not differ appreciably from vehicle, while ceritinib controlled tumor growth. Ceritinib inhibited Ba/F3 cells expressing EML4-ALK L1196M slightly stronger than ALK wild-type, but approximately double the potency as crizotinib, indicating ceritinib sensitivity in Ba/F3 EML4-ALK L1196M cells.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>159, \"name\"=>\"Ceritinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>924, \"name\"=>\"The ALK inhibitor ceritinib overcomes crizotinib resistance in non-small cell lung cancer.\", \"citation\"=>\"Friboulet et al., 2014, Cancer Discov\", \"pubmed_id\"=>\"24675041\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24675041\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4068971\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>7, \"phenotypes\"=>[]}, {\"id\"=>442, \"name\"=>\"EID442\", \"description\"=>\"The L1196M mutation in the EML4-ALK fusion was found to confer resistance to crizotinib in Ba/F3 cells.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>262, \"name\"=>\"Mechanisms of acquired crizotinib resistance in ALK-rearranged lung Cancers.\", \"citation\"=>\"Katayama et al., 2012, Sci Transl Med\", \"pubmed_id\"=>\"22277784\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22277784\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3385512\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>2, \"day\"=>8}, \"journal\"=>\"Sci Transl Med\", \"full_journal_title\"=>\"Science translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>7, \"phenotypes\"=>[]}, {\"id\"=>237, \"name\"=>\"EID237\", \"description\"=>\"In patients with non-small cell lung cancer harboring EML4-ALK fusion, the L1196M variant has been shown to confer resistance to crizotinib.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>166, \"name\"=>\"EML4-ALK mutations in lung cancer that confer resistance to ALK inhibitors.\", \"citation\"=>\"Choi et al., 2010, N. Engl. J. Med.\", \"pubmed_id\"=>\"20979473\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20979473\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>10, \"day\"=>28}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>7, \"phenotypes\"=>[]}, {\"id\"=>141, \"name\"=>\"EID141\", \"description\"=>\"CH5424802 treatment resulted in significant tumor regression in xenograft models produced from Ba/F3 cells expressing EML4-ALK or EML4-ALK with the L1196M mutation.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>321, \"name\"=>\"Alectinib (CH5424802)\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>77, \"name\"=>\"CH5424802, a selective ALK inhibitor capable of blocking the resistant gatekeeper mutant.\", \"citation\"=>\"Sakamoto et al., 2011, Cancer Cell\", \"pubmed_id\"=>\"21575866\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21575866\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>5, \"day\"=>17}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>7, \"phenotypes\"=>[]}]}, {\"id\"=>30, \"entrez_name\"=>\"CSF1R\", \"entrez_id\"=>1436, \"name\"=>\"MEF2D-CSF1R\", \"description\"=>\"MEF2D-CSF1R is a fusion that has been observed in acute lymphocytic leukemia. In cell lines harboring this event, treatment with imatinib has shown notable sensitivity.\", \"gene_id\"=>33, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>156446804, \"stop\"=>156470620, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000348159.4\", \"chromosome2\"=>\"5\", \"start2\"=>149433787, \"stop2\"=>149441412, \"representative_transcript2\"=>\"ENST00000286301.3\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/33/summary/variants/30/summary#variant\", \"evidence_items\"=>[{\"id\"=>403, \"name\"=>\"EID403\", \"description\"=>\"Cell lines with MEF2D-CSF1R fusion show sensitivity to the tyrosine kinase inhibitors Imatinib and GW-2580.\", \"disease\"=>{\"id\"=>43, \"name\"=>\"Acute Lymphocytic Leukemia\", \"display_name\"=>\"Acute Lymphocytic Leukemia\", \"doid\"=>\"9952\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9952\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}, {\"id\"=>70, \"name\"=>\"GW-2580\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>237, \"name\"=>\"RNA-seq identifies clinically relevant fusion genes in leukemia including a novel MEF2D/CSF1R fusion responsive to imatinib.\", \"citation\"=>\"Lilljebjörn et al., 2014, Leukemia\", \"pubmed_id\"=>\"24186003\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24186003\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>4}, \"journal\"=>\"Leukemia\", \"full_journal_title\"=>\"Leukemia\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>30, \"phenotypes\"=>[]}]}, {\"id\"=>31, \"entrez_name\"=>\"PRKACA\", \"entrez_id\"=>5566, \"name\"=>\"DNAJB1-PRKACA\", \"description\"=>\"This fusion has been found to be very recurrent in the rare form of adolescent liver cancer, fibrolamellar hepatocellular carcinoma. In one study, this fusion was observed in 15/15 FL-HCC cases examined and functional studies found that the fusion retained kinase activity. The presence of this fusion may be used as a diagnostic marker for this rare tumor type.\", \"gene_id\"=>17, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>75.0, \"coordinates\"=>{\"chromosome\"=>\"19\", \"start\"=>14628951, \"stop\"=>14629232, \"reference_bases\"=>nil, \"variant_bases\"=>\"\", \"representative_transcript\"=>\"ENST00000254322.2\", \"chromosome2\"=>\"19\", \"start2\"=>14202500, \"stop2\"=>14218221, \"representative_transcript2\"=>\"ENST00000308677.4\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/17/summary/variants/31/summary#variant\", \"evidence_items\"=>[{\"id\"=>1643, \"name\"=>\"EID1643\", \"description\"=>\"A DNAJB1:PRKACA fusion transcript was detected at very high expression levels by RNA-Seq in a case of mixed fibrolamellar hepatocellular carcinoma. Subsequent analysis revealed the presence of this fusion in all primary and metastatic samples, including those with mixed or conventional HCC pathology. A second case of mFL-HCC confirmed that the fusion was detectable in conventional components. An expanded screen of 112 additional HCC samples and 44 adjacent non-tumor liver samples was positive for a third case of HCC with both conventional and fibrolamellar features but negative for all conventional HCC and adjacent non-tumor liver samples.\", \"disease\"=>{\"id\"=>2139, \"name\"=>\"Hepatocellular Mixed Fibrolamellar Carcinoma\", \"display_name\"=>\"Hepatocellular Mixed Fibrolamellar Carcinoma\", \"doid\"=>nil, \"url\"=>\"http://www.disease-ontology.org/\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1089, \"name\"=>\"A genomic case study of mixed fibrolamellar hepatocellular carcinoma.\", \"citation\"=>\"Griffith et al., 2016, Ann. Oncol.\", \"pubmed_id\"=>\"27029710\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27029710\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4880064\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>6}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>31, \"phenotypes\"=>[]}, {\"id\"=>532, \"name\"=>\"EID532\", \"description\"=>\"RT-PCR and FISH assays were developed for the detection of rearrangements of the PRKACA locus. A total of 106 primary liver tumors were studied by RT-PCR (26 fibrolamellar carcinomas, 25 conventional hepatocellular carcinomas, 25 cholangiocarcinomas, 25 hepatic adenomas, and 5 hepatoblastomas). FISH was tested in 19 fibrolamellar carcinomas and in 6 scirrhous hepatocellular carcinomas. Evidence of the DNAJB1-PRKACA fusion was found in all fibrolamellar cases but not in other tumor types for both methods indicating that detection of DNAJB1-PRKACA is a very sensitive and specific finding in support of the diagnosis of fibrolamellar carcinoma.\", \"disease\"=>{\"id\"=>44, \"name\"=>\"Hepatocellular Fibrolamellar Carcinoma\", \"display_name\"=>\"Hepatocellular Fibrolamellar Carcinoma\", \"doid\"=>\"5015\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5015\"}, \"drugs\"=>[], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>260, \"name\"=>\"DNAJB1-PRKACA is specific for fibrolamellar carcinoma.\", \"citation\"=>\"Graham et al., 2015, Mod. Pathol.\", \"pubmed_id\"=>\"25698061\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25698061\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>6}, \"journal\"=>\"Mod. Pathol.\", \"full_journal_title\"=>\"Modern pathology : an official journal of the United States and Canadian Academy of Pathology, Inc\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>31, \"phenotypes\"=>[]}, {\"id\"=>405, \"name\"=>\"EID405\", \"description\"=>\"A head-to-tail in-frame fusion between exon1 of DNAJB1 and exon2 of PRKACA resulting from a ~400kb deletion was observed in 100% (15/15) fibrolamellar hepatocellular carcinomas examined. The fusion was confirmed at both RNA and DNA level in all primary and metastatic tumor samples but not in adjacent normal tissue. Immunoprecipitation and Western blot analyses confirmed that the chimeric protein is expressed in tumor tissue, and a cell culture assay indicated that it retains kinase activity.\", \"disease\"=>{\"id\"=>44, \"name\"=>\"Hepatocellular Fibrolamellar Carcinoma\", \"display_name\"=>\"Hepatocellular Fibrolamellar Carcinoma\", \"doid\"=>\"5015\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5015\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>22, \"name\"=>\"Detection of a recurrent DNAJB1-PRKACA chimeric transcript in fibrolamellar hepatocellular carcinoma.\", \"citation\"=>\"Honeyman et al., 2014, Science\", \"pubmed_id\"=>\"24578576\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24578576\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4286414\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>2, \"day\"=>28}, \"journal\"=>\"Science\", \"full_journal_title\"=>\"Science (New York, N.Y.)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>31, \"phenotypes\"=>[]}, {\"id\"=>754, \"name\"=>\"EID754\", \"description\"=>\"The DNAJB1-PRKACA fusion was detected in 79% (58/73) of fibrolamellar hepatocellular carcinomas assayed.\", \"disease\"=>{\"id\"=>44, \"name\"=>\"Hepatocellular Fibrolamellar Carcinoma\", \"display_name\"=>\"Hepatocellular Fibrolamellar Carcinoma\", \"doid\"=>\"5015\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5015\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>499, \"name\"=>\"Unique genomic profile of fibrolamellar hepatocellular carcinoma.\", \"citation\"=>\"Cornella et al., 2015, Gastroenterology\", \"pubmed_id\"=>\"25557953\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25557953\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4521774\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>4}, \"journal\"=>\"Gastroenterology\", \"full_journal_title\"=>\"Gastroenterology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>31, \"phenotypes\"=>[]}]}, {\"id\"=>52, \"entrez_name\"=>\"FGFR2\", \"entrez_id\"=>2263, \"name\"=>\"FGFR2-MGEA5\", \"description\"=>\"In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has idenified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. Cell lines harboring these fusions were shown to respond to pazopanib. Additionally, tumor size reduction was achieved by both ponatinib and pazopanib treatments administered separately in a single patient with intrahepatic cholangiocarcinoma and this fusion. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.\", \"gene_id\"=>22, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>123243212, \"stop\"=>123357917, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000457416.2\", \"chromosome2\"=>\"10\", \"start2\"=>103544209, \"stop2\"=>103552700, \"representative_transcript2\"=>\"ENST00000361464.3\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/22/summary/variants/52/summary#variant\", \"evidence_items\"=>[{\"id\"=>295, \"name\"=>\"EID295\", \"description\"=>\"In a single patient with this fusion in intrahepatic cholangiocarcinoma. Ponatinib treatment resulted in necrosis and shrinkage of the primary and metastatic sites and overall stable disease. However, this response did not meet criteria for RECIST partial response.\", \"disease\"=>{\"id\"=>29, \"name\"=>\"Cholangiocarcinoma\", \"display_name\"=>\"Cholangiocarcinoma\", \"doid\"=>\"4947\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4947\"}, \"drugs\"=>[{\"id\"=>55, \"name\"=>\"Ponatinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>186, \"name\"=>\"Integrated genomic characterization reveals novel, therapeutically relevant drug targets in FGFR and EGFR pathways in sporadic intrahepatic cholangiocarcinoma.\", \"citation\"=>\"Borad et al., 2014, PLoS Genet.\", \"pubmed_id\"=>\"24550739\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24550739\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3923676\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>2}, \"journal\"=>\"PLoS Genet.\", \"full_journal_title\"=>\"PLoS genetics\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>52, \"phenotypes\"=>[]}]}, {\"id\"=>53, \"entrez_name\"=>\"FGFR3\", \"entrez_id\"=>2261, \"name\"=>\"FGFR3-BAIAP2L1\", \"description\"=>\"In a clinical sequencing program for advanced stage cancers, Wu et al (2013, Cancer Discovery) has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.\", \"gene_id\"=>23, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>1795039, \"stop\"=>1810599, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000340107.4\", \"chromosome2\"=>\"7\", \"start2\"=>97991744, \"stop2\"=>97920963, \"representative_transcript2\"=>\"ENST00000005260.8\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/23/summary/variants/53/summary#variant\", \"evidence_items\"=>[{\"id\"=>144, \"name\"=>\"EID144\", \"description\"=>\"Following identification of this fusion in a patient, this fusion was expressed in HEK 293T cells leading to oligomerization in the absence of ligand, induced morphologic changes in the cell and increased cellular proliferation consistent with FGFR activation. Mouse xenografts using the bladder cell line SW780 which harbors an FGFR3-BAIAP2L1 fusion showed reduced tumor burden when treated with the FGFR inhibitor PD173074.\", \"disease\"=>{\"id\"=>18, \"name\"=>\"Bladder Carcinoma\", \"display_name\"=>\"Bladder Carcinoma\", \"doid\"=>\"4007\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4007\"}, \"drugs\"=>[{\"id\"=>36, \"name\"=>\"PD173074\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>125, \"name\"=>\"Identification of targetable FGFR gene fusions in diverse cancers.\", \"citation\"=>\"Wu et al., 2013, Cancer Discov\", \"pubmed_id\"=>\"23558953\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23558953\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3694764\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>53, \"phenotypes\"=>[]}]}, {\"id\"=>54, \"entrez_name\"=>\"FGFR2\", \"entrez_id\"=>2263, \"name\"=>\"FGFR2-TACC3\", \"description\"=>\"In a clinical sequencing program for advanced stage cancers, Wu et al has identified a number of FGFR fusions in patients with many different cancer types. These fusions were also found to retain oligomerization capability, and result in enhanced cell proliferation. These fusions were shown to respond to pazopanib. The authors use these cases to highlight the need for enhanced clinical sequencing efforts.\", \"gene_id\"=>22, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>7.5, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>123243212, \"stop\"=>123357598, \"reference_bases\"=>nil, \"variant_bases\"=>\"\", \"representative_transcript\"=>\"ENST00000358487.5\", \"chromosome2\"=>\"4\", \"start2\"=>1723266, \"stop2\"=>1741505, \"representative_transcript2\"=>\"ENST00000313288.4\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/22/summary/variants/54/summary#variant\", \"evidence_items\"=>[{\"id\"=>296, \"name\"=>\"EID296\", \"description\"=>\"In a 43 year old female patient with this fusion in intrahepatic cholangiocarcinoma. Both pazopanib and ponatinib separately resulted in minor tumor shrinkage. This fusion is also shown to be recurrent in multiple other cancer types.\", \"disease\"=>{\"id\"=>29, \"name\"=>\"Cholangiocarcinoma\", \"display_name\"=>\"Cholangiocarcinoma\", \"doid\"=>\"4947\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4947\"}, \"drugs\"=>[{\"id\"=>55, \"name\"=>\"Ponatinib\", \"pubchem_id\"=>nil}, {\"id\"=>56, \"name\"=>\"Pazopanib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Sequential\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>186, \"name\"=>\"Integrated genomic characterization reveals novel, therapeutically relevant drug targets in FGFR and EGFR pathways in sporadic intrahepatic cholangiocarcinoma.\", \"citation\"=>\"Borad et al., 2014, PLoS Genet.\", \"pubmed_id\"=>\"24550739\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24550739\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3923676\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>2}, \"journal\"=>\"PLoS Genet.\", \"full_journal_title\"=>\"PLoS genetics\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>54, \"phenotypes\"=>[]}]}, {\"id\"=>108, \"entrez_name\"=>\"PML\", \"entrez_id\"=>5371, \"name\"=>\"PML-RARA\", \"description\"=>\"The PML-RARA fusion is seen as a diagnostic event in acute promyelocytic leukemia (APL). Both in vitro and in vivo studies have shown sensitivity to ATRA in APL patients harboring the PML-RARa fusion. Recent interest has been shown in combining ATRA and arsenic trioxide for treating these patients, and early results seem promising.\", \"gene_id\"=>39, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>132.5, \"coordinates\"=>{\"chromosome\"=>\"15\", \"start\"=>74287058, \"stop\"=>74325755, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000268058.3\", \"chromosome2\"=>\"17\", \"start2\"=>38504568, \"stop2\"=>38513048, \"representative_transcript2\"=>\"ENST00000254066.5\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/39/summary/variants/108/summary#variant\", \"evidence_items\"=>[{\"id\"=>1519, \"name\"=>\"EID1519\", \"description\"=>\"A 39-year old woman who presented with acute myeloid leukemia revealed a complicated genetic profile using conventional metaphase cytogenetics and FISH analysis.  Whole genome sequencing elucidated a PML-RARA fusion, which can be successfully treated with ATRA. Uncovering this mutation prevented an unnecessary allogeneic transplant and the patient remained in remission 15 months post-treatment (date of publication).\", \"disease\"=>{\"id\"=>379, \"name\"=>\"Acute Promyelocytic Leukemia\", \"display_name\"=>\"Acute Promyelocytic Leukemia\", \"doid\"=>\"0060318\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060318\"}, \"drugs\"=>[{\"id\"=>265, \"name\"=>\"All-trans Retinoic Acid\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1012, \"name\"=>\"Use of whole-genome sequencing to diagnose a cryptic fusion oncogene.\", \"citation\"=>\"Welch et al., 2011, JAMA\", \"pubmed_id\"=>\"21505136\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21505136\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3156695\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>4, \"day\"=>20}, \"journal\"=>\"JAMA\", \"full_journal_title\"=>\"JAMA\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>108, \"phenotypes\"=>[]}, {\"id\"=>1091, \"name\"=>\"EID1091\", \"description\"=>\"ATRA/Arsenic combined with standard chemotherapy has been shown to induce a complete remission rate of greater than 75% of patients with acute promyelocytic leukemia harboring the PML-RARa fusion.\", \"disease\"=>{\"id\"=>379, \"name\"=>\"Acute Promyelocytic Leukemia\", \"display_name\"=>\"Acute Promyelocytic Leukemia\", \"doid\"=>\"0060318\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060318\"}, \"drugs\"=>[{\"id\"=>47, \"name\"=>\"Arsenic Trioxide\", \"pubchem_id\"=>nil}, {\"id\"=>265, \"name\"=>\"All-trans Retinoic Acid\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>750, \"name\"=>\"All trans retinoic acid in acute promyelocytic leukemia.\", \"citation\"=>\"Degos et al., 2001, Oncogene\", \"pubmed_id\"=>\"11704842\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/11704842\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2001, \"month\"=>10, \"day\"=>29}, \"journal\"=>\"Oncogene\", \"full_journal_title\"=>\"Oncogene\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>108, \"phenotypes\"=>[]}, {\"id\"=>1090, \"name\"=>\"EID1090\", \"description\"=>\"The presence of the PML-RARa fusion protein is a diagnostic marker of Acute Promyelocytic Leukemia, or M3 AML.\", \"disease\"=>{\"id\"=>379, \"name\"=>\"Acute Promyelocytic Leukemia\", \"display_name\"=>\"Acute Promyelocytic Leukemia\", \"doid\"=>\"0060318\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060318\"}, \"drugs\"=>[], \"rating\"=>5, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>749, \"name\"=>\"The PML/RAR alpha fusion gene in the diagnosis and monitoring of acute promyelocytic leukemia.\", \"citation\"=>\"Diverio et al., Haematologica\", \"pubmed_id\"=>\"7628753\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/7628753\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{}, \"journal\"=>\"Haematologica\", \"full_journal_title\"=>\"Haematologica\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>108, \"phenotypes\"=>[]}, {\"id\"=>316, \"name\"=>\"EID316\", \"description\"=>\"Fusion protein degradation is accelerated upon ATRA treatment of acute promyelocytic cells at pharmalogical concentrations.\", \"disease\"=>{\"id\"=>379, \"name\"=>\"Acute Promyelocytic Leukemia\", \"display_name\"=>\"Acute Promyelocytic Leukemia\", \"doid\"=>\"0060318\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060318\"}, \"drugs\"=>[{\"id\"=>265, \"name\"=>\"All-trans Retinoic Acid\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>197, \"name\"=>\"Accelerated degradation of PML-retinoic acid receptor alpha (PML-RARA) oncoprotein by all-trans-retinoic acid in acute promyelocytic leukemia: possible role of the proteasome pathway.\", \"citation\"=>\"Yoshida et al., 1996, Cancer Res.\", \"pubmed_id\"=>\"8674046\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/8674046\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1996, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>108, \"phenotypes\"=>[]}]}, {\"id\"=>164, \"entrez_name\"=>\"FLI1\", \"entrez_id\"=>2313, \"name\"=>\"EWSR1-FLI1 Type 1\", \"description\"=>\"Although multiple fusion transcripts have been described, type 1 consists of EWSR1 exon 7 fused to FLI1 exon 6, the most common arrangement in Ewings Sarcoma.\", \"gene_id\"=>53, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"22\", \"start\"=>29664007, \"stop\"=>29683123, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000397938.2\", \"chromosome2\"=>\"11\", \"start2\"=>128675261, \"stop2\"=>128683162, \"representative_transcript2\"=>\"ENST00000527786.2\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/53/summary/variants/164/summary#variant\", \"evidence_items\"=>[{\"id\"=>431, \"name\"=>\"EID431\", \"description\"=>\"In patients with Ewing's sarcoma, those with type 1 EWS-FLI1 fusions had longer overall survival than those with other types of EWS-FLI1 fusions.\", \"disease\"=>{\"id\"=>50, \"name\"=>\"Ewing Sarcoma\", \"display_name\"=>\"Ewing Sarcoma\", \"doid\"=>\"3369\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3369\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>255, \"name\"=>\"EWS-FLI1 fusion transcript structure is an independent determinant of prognosis in Ewing's sarcoma.\", \"citation\"=>\"de Alava et al., 1998, J. Clin. Oncol.\", \"pubmed_id\"=>\"9552022\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/9552022\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1998, \"month\"=>4}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>164, \"phenotypes\"=>[]}]}, {\"id\"=>170, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"EML4-ALK AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>11.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>42396490, \"stop\"=>42528380, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000318522.5\", \"chromosome2\"=>\"2\", \"start2\"=>29415640, \"stop2\"=>29446394, \"representative_transcript2\"=>\"ENST00000389048.3\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/170/summary#variant\", \"evidence_items\"=>[{\"id\"=>767, \"name\"=>\"EID767\", \"description\"=>\"One patient with ALK-rearranged non-small cell lung cancer and an acquired resistance to crizotinib showed new ALK amplification (4.1 fold increase compared to pre-crizotinib specimen as measured by FISH) and harbored an EGFR L858R mutation.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>508, \"name\"=>\"Heterogeneity of genetic changes associated with acquired crizotinib resistance in ALK-rearranged lung cancer.\", \"citation\"=>\"Kim et al., 2013, J Thorac Oncol\", \"pubmed_id\"=>\"23344087\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23344087\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>4}, \"journal\"=>\"J Thorac Oncol\", \"full_journal_title\"=>\"Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>170, \"phenotypes\"=>[]}, {\"id\"=>440, \"name\"=>\"EID440\", \"description\"=>\"Amplification of wild-type EML4-ALK in the non-small cell lung carcinoma cell line, H3122, caused resistance to crizotinib.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>262, \"name\"=>\"Mechanisms of acquired crizotinib resistance in ALK-rearranged lung Cancers.\", \"citation\"=>\"Katayama et al., 2012, Sci Transl Med\", \"pubmed_id\"=>\"22277784\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22277784\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3385512\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>2, \"day\"=>8}, \"journal\"=>\"Sci Transl Med\", \"full_journal_title\"=>\"Science translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>170, \"phenotypes\"=>[]}, {\"id\"=>766, \"name\"=>\"EID766\", \"description\"=>\"Two out of 11 patients with an acquired crizotinib resistance showed new ALK copy-number gain (defined as more than 2-fold increase in the mean of the rearranged gene per cell in the posttreatment specimen compared with the pretreatment specimen), both with >4 fold increase. One of them also harbored a resistance mutation (G1269A).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>507, \"name\"=>\"Mechanisms of resistance to crizotinib in patients with ALK gene rearranged non-small cell lung cancer.\", \"citation\"=>\"Doebele et al., 2012, Clin. Cancer Res.\", \"pubmed_id\"=>\"22235099\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22235099\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3311875\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>3, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>170, \"phenotypes\"=>[]}]}, {\"id\"=>171, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"ALK FUSION G1202R\", \"description\"=>\"ALK fusion with G1202R mutation in the ALK kinase domain was first identified in a lung adenocarcinoma patient who had progressed after 10 months of responding to crizotinib. Further characterization of this variant verified crizotinib resistance and additionally demonstrated resistance to multiple next generation ALK inhibitors including ceritinib, alectinib and brigatinib. Experiments using the HSP90 inhibitor  17-AAG have indicated potential sensitivity of EML4-ALK G1202R to this drug, suggesting this variant may be an HSP90 client protein.\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>23.5, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>29443613, \"stop\"=>29443613, \"reference_bases\"=>\"C\", \"variant_bases\"=>\"T\", \"representative_transcript\"=>\"ENST00000389048.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/171/summary#variant\", \"evidence_items\"=>[{\"id\"=>4625, \"name\"=>\"EID4625\", \"description\"=>\"In an in vitro study, a Ba/F3 cell line expressing EML4-ALK fusion (a known sensitizing alteration to crizotinib) and an ALK G1202R co-mutation, demonstrated resistance to crizotinib treatment compared to Ba/F3 cells expressing EML4-ALK fusion. Resistance was determined by assessing cell viability.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>924, \"name\"=>\"The ALK inhibitor ceritinib overcomes crizotinib resistance in non-small cell lung cancer.\", \"citation\"=>\"Friboulet et al., 2014, Cancer Discov\", \"pubmed_id\"=>\"24675041\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24675041\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4068971\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>171, \"phenotypes\"=>[]}, {\"id\"=>4624, \"name\"=>\"EID4624\", \"description\"=>\"In a non-small cell lung cancer patient with acquired crizotinib resistance, the tumor had an ALK gene rearrangement and an ALK G1202R co-mutation. The ALK G1202R tumor was reported to be refractory to 10-month crizotinib treatment.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2034, \"name\"=>\"ALK rearrangements are mutually exclusive with mutations in EGFR or KRAS: an analysis of 1,683 patients with non-small cell lung cancer.\", \"citation\"=>\"Gainor et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"23729361\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23729361\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3874127\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>8, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"submitted\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>171, \"phenotypes\"=>[]}, {\"id\"=>1357, \"name\"=>\"EID1357\", \"description\"=>\"A patient with ALK-rearranged lung adenocarcinoma responded to crizotinib treatment and then had relapse. Biopsy was taken within half a month and total DNA was extracted from the sample. Exons 20-28, which correspond to the tyrosine kinase domain of ALK, were sequenced, and ALK G1202A was found. CKIT focal amplification by FISH was also observed. Tumor sample before crizotinib treatment was not available in this case.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>262, \"name\"=>\"Mechanisms of acquired crizotinib resistance in ALK-rearranged lung Cancers.\", \"citation\"=>\"Katayama et al., 2012, Sci Transl Med\", \"pubmed_id\"=>\"22277784\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22277784\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3385512\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>2, \"day\"=>8}, \"journal\"=>\"Sci Transl Med\", \"full_journal_title\"=>\"Science translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>171, \"phenotypes\"=>[]}, {\"id\"=>1350, \"name\"=>\"EID1350\", \"description\"=>\"Ba/F3 cell line expressing EML4-ALK variant 3 with G1202R ALK mutation was assessed for alectinib sensitivity and IC50 for survival of 868 nM was reported, which was 64.4x higher than the IC50 reported for alectinib on Ba/F3 with EML4-ALK wild-type. Similar experiments using NPM-ALK instead of EML4-ALK yielded survival IC50 of 549 nM of NPM-ALK G1202R to alectinib, which was 27.9 times higher than IC50 for NPM-ALK wild-type. Western blots of Ba/F3 with NPM-ALK wild-type showed ablation of pALK levels above 100 nM alectinib, while pALK levels were essentially unchanged up to 1000 nM in NPM-ALK G1202R Ba/F3 cells.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>321, \"name\"=>\"Alectinib (CH5424802)\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>925, \"name\"=>\"Activity of second-generation ALK inhibitors against crizotinib-resistant mutants in an NPM-ALK model compared to EML4-ALK.\", \"citation\"=>\"Fontana et al., 2015, Cancer Med\", \"pubmed_id\"=>\"25727400\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25727400\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4529334\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>7}, \"journal\"=>\"Cancer Med\", \"full_journal_title\"=>\"Cancer medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>171, \"phenotypes\"=>[]}, {\"id\"=>1352, \"name\"=>\"EID1352\", \"description\"=>\"The G1202R ALK mutation was found in a crizotinib-resistant ALK-rearranged NSCLC, and the mutant ALK fusion was further characterized in Ba/F3 cells. EML4-ALK variant 1 with G1202R mutation was expressed in Ba/F3 and subject to 17-AAG. IC50 for cell number in response to 17-AAG was 33.4 nM, in contrast to IC50 for EML4-ALK wild-type, which was 98.9 nM, suggesting sensitization of this variant to HSP90 inhibition.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>10, \"name\"=>\"17-AAG\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>262, \"name\"=>\"Mechanisms of acquired crizotinib resistance in ALK-rearranged lung Cancers.\", \"citation\"=>\"Katayama et al., 2012, Sci Transl Med\", \"pubmed_id\"=>\"22277784\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22277784\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3385512\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>2, \"day\"=>8}, \"journal\"=>\"Sci Transl Med\", \"full_journal_title\"=>\"Science translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>171, \"phenotypes\"=>[]}, {\"id\"=>1351, \"name\"=>\"EID1351\", \"description\"=>\"This study compared drug responses in Ba/F3 cells expressing either EML4-ALK or NPM-ALK fusions, with or without G1202R ALK mutation. Cells expressing the EML4-ALK variant 3 with G1202R ALK mutation were more resistant to brigatinib treatment (IC50 325 nM) than cells expressing wildtype EML4-ALK variant 3 (IC50 6 nM). These same experiments performed with the NPM-ALK fusion resulted in IC50 of 149 nM for G1202R and 11nM for wild type, consistent with resistance. In Western blots, ablation of pALK with brigatinib treatment above 100 nM was seen in Ba/F3 cells expressing NPM-ALK wildtype, but remained apparent at 1000 nM brigatinib in NPM-ALK G1202R cells, despite dose-dependent reduction.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>318, \"name\"=>\"Brigatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>925, \"name\"=>\"Activity of second-generation ALK inhibitors against crizotinib-resistant mutants in an NPM-ALK model compared to EML4-ALK.\", \"citation\"=>\"Fontana et al., 2015, Cancer Med\", \"pubmed_id\"=>\"25727400\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25727400\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4529334\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>7}, \"journal\"=>\"Cancer Med\", \"full_journal_title\"=>\"Cancer medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>171, \"phenotypes\"=>[]}, {\"id\"=>441, \"name\"=>\"EID441\", \"description\"=>\"The G1202R mutation in the EML4-ALK fusion was found to confer resistance to crizotinib in Ba/F3 cells.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>262, \"name\"=>\"Mechanisms of acquired crizotinib resistance in ALK-rearranged lung Cancers.\", \"citation\"=>\"Katayama et al., 2012, Sci Transl Med\", \"pubmed_id\"=>\"22277784\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22277784\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3385512\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>2, \"day\"=>8}, \"journal\"=>\"Sci Transl Med\", \"full_journal_title\"=>\"Science translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>171, \"phenotypes\"=>[]}, {\"id\"=>1345, \"name\"=>\"EID1345\", \"description\"=>\"Ba/F3 cells expressing EML4-ALK variant 1 with ALK G1202R mutation were assessed for ceritinib sensitivity. Crizotinib was shown to confer significantly more potency than ceritinib (43-fold decrease in IC50). EML4-ALK variant 3 NSCLC cell line H2228 was used in SCID mouse xenografts which were exposed to increasing crizotinib up to 100 mg/kg. Within 40 days tumors progressed, and progressed tumors were assayed for ALK mutation. Of 80 tumors, 6 had ALK mutation and 3 of these were G1202R. Treatment of crizotinib progressed tumors with ceritinib resulted in tumor growth which did not differ appreciably from crizotinib or vehicle treatment of these tumors.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>159, \"name\"=>\"Ceritinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>924, \"name\"=>\"The ALK inhibitor ceritinib overcomes crizotinib resistance in non-small cell lung cancer.\", \"citation\"=>\"Friboulet et al., 2014, Cancer Discov\", \"pubmed_id\"=>\"24675041\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24675041\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4068971\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>171, \"phenotypes\"=>[]}]}, {\"id\"=>172, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"EML4-ALK S1206Y\", \"description\"=>\"\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>6.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>29443600, \"stop\"=>29443600, \"reference_bases\"=>\"G\", \"variant_bases\"=>\"T\", \"representative_transcript\"=>\"ENST00000389048.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/172/summary#variant\", \"evidence_items\"=>[{\"id\"=>4622, \"name\"=>\"EID4622\", \"description\"=>\"In an in vitro study, a Ba/F3 cell line expressing EML4-ALK fusion (a known sensitizing alteration to crizotinib) and an ALK S1206Y co-mutation, demonstrated resistance to crizotinib treatment compared to Ba/F3 cells expressing EML4-ALK fusion. Resistance was determined by assessing cell viability.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>924, \"name\"=>\"The ALK inhibitor ceritinib overcomes crizotinib resistance in non-small cell lung cancer.\", \"citation\"=>\"Friboulet et al., 2014, Cancer Discov\", \"pubmed_id\"=>\"24675041\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24675041\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4068971\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>172, \"phenotypes\"=>[]}, {\"id\"=>2865, \"name\"=>\"EID2865\", \"description\"=>\"In a prospective study (NCT01283516) of 130 cancer patients, a non-small cell lung cancer patient harboring a S1206Y mutation in the ALK portion of the ALK rearrangement was associated with a 60% response to ceritinib monotherapy.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>159, \"name\"=>\"Ceritinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>856, \"name\"=>\"Ceritinib in ALK-rearranged non-small-cell lung cancer.\", \"citation\"=>\"Shaw et al., 2014, N. Engl. J. Med.\", \"pubmed_id\"=>\"24670165\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24670165\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4079055\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>3, \"day\"=>27}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>172, \"phenotypes\"=>[]}, {\"id\"=>1343, \"name\"=>\"EID1343\", \"description\"=>\"Ba/F3 cells expressing the EML4-ALK S1206Y variant were treated with ceritinib, and survival GI50 value of 1.5 nM was obtained, which compares favorably with GI50 of 1.7 for the highly ceritinib sensitive EML4-ALK wild-type variant obtained in parallel experiments, as well as the GI50 value of 16 nM obtained for crizotinib treatment of EML4-ALK wild-type.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>159, \"name\"=>\"Ceritinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>924, \"name\"=>\"The ALK inhibitor ceritinib overcomes crizotinib resistance in non-small cell lung cancer.\", \"citation\"=>\"Friboulet et al., 2014, Cancer Discov\", \"pubmed_id\"=>\"24675041\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24675041\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4068971\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>172, \"phenotypes\"=>[]}, {\"id\"=>443, \"name\"=>\"EID443\", \"description\"=>\"The S1206Y mutation in the EML4-ALK fusion was found to confer resistance to crizotinib in Ba/F3 cells.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>262, \"name\"=>\"Mechanisms of acquired crizotinib resistance in ALK-rearranged lung Cancers.\", \"citation\"=>\"Katayama et al., 2012, Sci Transl Med\", \"pubmed_id\"=>\"22277784\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22277784\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3385512\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>2, \"day\"=>8}, \"journal\"=>\"Sci Transl Med\", \"full_journal_title\"=>\"Science translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>172, \"phenotypes\"=>[]}]}, {\"id\"=>173, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"EML4-ALK T1151INST\", \"description\"=>\"The ALK fusion variant with insertion of T at amino acid 1151 was found in an ALK rearranged lung adenocarcinoma which had progressed after responding to crizotinib treatment. The mutation was absent in tumor sample taken before crizotinib treatment. Characterization of the variant using cell lines expressing EML4-ALK with 1151 T insert found the variant to be highly crizotinib resistant, and other work demonstrated resistance to next generation inhibitor alectinib, while resistance to tool compound TAE684 indicates potential resistance to the next generation ALK inhibitor ceritinib which is derived from TAE684. Other experiments suggest theraputic potential for HSP90 inhibitors, as EML4-ALK T1151 Tins sensitivity to 17-AAG has been observed.\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>106, \"name\"=>\"inframe_insertion\", \"display_name\"=>\"Inframe Insertion\", \"so_id\"=>\"SO:0001821\", \"description\"=>\"An inframe non synonymous variant that inserts bases into in the coding sequence.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001821\"}], \"civic_actionability_score\"=>18.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>29445271, \"stop\"=>29445272, \"reference_bases\"=>nil, \"variant_bases\"=>\"CGT\", \"representative_transcript\"=>\"ENST00000389048.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/173/summary#variant\", \"evidence_items\"=>[{\"id\"=>4608, \"name\"=>\"EID4608\", \"description\"=>\"In an in vitro study,  a Ba/F3 cell line expressing EML4-ALK fusion (a known sensitizing alteration to crizotinib) and an ALK T1151_L1152insT co-mutation, demonstrated resistance to crizotinib treatment (431.8 nmol/L vs. 26.5 nmol/L) compared to Ba/F3 cells expressing EML4-ALK fusion. Resistance was determined by assessing cell viability and ALK auto-phosphorylation .\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>924, \"name\"=>\"The ALK inhibitor ceritinib overcomes crizotinib resistance in non-small cell lung cancer.\", \"citation\"=>\"Friboulet et al., 2014, Cancer Discov\", \"pubmed_id\"=>\"24675041\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24675041\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4068971\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>173, \"phenotypes\"=>[]}, {\"id\"=>444, \"name\"=>\"EID444\", \"description\"=>\"A lung adenocarcinoma patient responded to crizotinib treatment for 12 months, after which the cancer relapsed and sequencing of the ALK tyrosine kinase domain revealed a 1151insT mutation. This mutation was not found in the pre-crizotinib biopsy. EML4-ALK T1151insT was expressed In Ba/F3 cells and an IC50 of 431.8 nM was obtained, which was 16 times higher than for EML4-ALK wild-type, indicating strong crizotinib resistance. EML4-ALK containing H3122 NSCLC cells were incubated in increasing levels of crizotinib inducing resistance. The resulting cell line H3122 CR2 had IC50 10x greater to crizotinib than parental H3122, and was found to contain 1151 insT mutation. In western blots, pALK levels in H3122 CR2 cells remained apparent under 1000 nM crizotinib, whereas pALK signal was ablated under these conditions in parental H3122 cells.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>262, \"name\"=>\"Mechanisms of acquired crizotinib resistance in ALK-rearranged lung Cancers.\", \"citation\"=>\"Katayama et al., 2012, Sci Transl Med\", \"pubmed_id\"=>\"22277784\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22277784\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3385512\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>2, \"day\"=>8}, \"journal\"=>\"Sci Transl Med\", \"full_journal_title\"=>\"Science translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>173, \"phenotypes\"=>[]}, {\"id\"=>1347, \"name\"=>\"EID1347\", \"description\"=>\"Ba/F3 cells expressing EML4-ALK with 1151 Tins treated with alectinib had an IC50 of 165.7 nM, 16x higher than those expressing EML4-ALK wildtype variant 1. EML4-ALK variant 1 expressing H3122 NSCLC cell line was cultured in increasing levels of crizotinib to produce crizotinib resistant cell lines CR1, CR2 and CR3. H3122 CR2 was found to contain an 1151 insT ALK mutation and increased EML4-ALK levels. The H3122 CR2 cell line survival IC50 value for alectinib treatment was approximately 20x higher than that for H3122 cells.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>321, \"name\"=>\"Alectinib (CH5424802)\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>262, \"name\"=>\"Mechanisms of acquired crizotinib resistance in ALK-rearranged lung Cancers.\", \"citation\"=>\"Katayama et al., 2012, Sci Transl Med\", \"pubmed_id\"=>\"22277784\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22277784\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3385512\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>2, \"day\"=>8}, \"journal\"=>\"Sci Transl Med\", \"full_journal_title\"=>\"Science translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>173, \"phenotypes\"=>[]}, {\"id\"=>1349, \"name\"=>\"EID1349\", \"description\"=>\"Ba/F3 cells transduced with EML4-ALK 1151 insT were treated with 17-AAG producing an IC50 value of 44.5 nM, which was 0.4 times that reported for EML4-ALK wildtype in Ba/F3, indicating possible increased dependence of EML4-ALK 1151 insT on HSP90. 17-AAG also reduced survival of the parental IL3-dependent Ba/F3 cells (183.7 nM), indicating increased nonspecific effects of 17_AAG treatment. Crizotinib resistant clones were produced from the parental EML4-ALK variant 1 containing cell line H3122 (CR1, CR2, CR3). H3122 CR2 was found to contain the EML4-ALK InsT variant. H3122 CR2 and H3122 cells were similarly sensitized to 17-AAG death with an IC50 value of approximately 5 nM, while control cells without EML4-ALK had IC50 values approximately 10x higher.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>10, \"name\"=>\"17-AAG\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>262, \"name\"=>\"Mechanisms of acquired crizotinib resistance in ALK-rearranged lung Cancers.\", \"citation\"=>\"Katayama et al., 2012, Sci Transl Med\", \"pubmed_id\"=>\"22277784\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22277784\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3385512\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>2, \"day\"=>8}, \"journal\"=>\"Sci Transl Med\", \"full_journal_title\"=>\"Science translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>173, \"phenotypes\"=>[]}, {\"id\"=>1348, \"name\"=>\"EID1348\", \"description\"=>\"Treatment of Ba/F3 cells expressing EML4-ALK 1151 insT with TAE684 produced an IC50 value (28.3 nM) 20x higher cells expressing EML4-ALK wildtype variant 1. The crizotinib resistant clones were produced from parental H3122 cells (CR1, CR2, CR3). H3122 CR2 was found to contain the 1151 Tins variant as well as elevated EML4-ALK expression. H3122 CR2 cells treated with TAE684 produced IC50 values 20-30 times that of the parental H3122 cells. These results suggest possible variant resistance to the TAE684-derived compound ceritinib.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>18, \"name\"=>\"TAE684\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>262, \"name\"=>\"Mechanisms of acquired crizotinib resistance in ALK-rearranged lung Cancers.\", \"citation\"=>\"Katayama et al., 2012, Sci Transl Med\", \"pubmed_id\"=>\"22277784\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22277784\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3385512\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>2, \"day\"=>8}, \"journal\"=>\"Sci Transl Med\", \"full_journal_title\"=>\"Science translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>173, \"phenotypes\"=>[]}]}, {\"id\"=>184, \"entrez_name\"=>\"BRAF\", \"entrez_id\"=>673, \"name\"=>\"AKAP9-BRAF\", \"description\"=>\"\", \"gene_id\"=>5, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>91570181, \"stop\"=>91625114, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000356239.3\", \"chromosome2\"=>\"7\", \"start2\"=>140434279, \"stop2\"=>140487384, \"representative_transcript2\"=>\"ENST00000288602.6\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5/summary/variants/184/summary#variant\", \"evidence_items\"=>[{\"id\"=>462, \"name\"=>\"EID462\", \"description\"=>\"The AKAP9-BRAF fusion gene was found in 3/28 tumor samples of radiation-associated papillary thyroid carcinoma, and no samples of non-radiation associated papillary thyroid carcinoma. This fusion was associated with elevated BRAF kinase activity, similar to the V600E variant.\", \"disease\"=>{\"id\"=>156, \"name\"=>\"Papillary Thyroid Carcinoma\", \"display_name\"=>\"Papillary Thyroid Carcinoma\", \"doid\"=>\"3969\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3969\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>277, \"name\"=>\"Oncogenic AKAP9-BRAF fusion is a novel mechanism of MAPK pathway activation in thyroid cancer.\", \"citation\"=>\"Ciampi et al., 2005, J. Clin. Invest.\", \"pubmed_id\"=>\"15630448\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15630448\", \"open_access\"=>true, \"pmc_id\"=>\"PMC539203\", \"publication_date\"=>{\"year\"=>2005, \"month\"=>1}, \"journal\"=>\"J. Clin. Invest.\", \"full_journal_title\"=>\"The Journal of clinical investigation\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>184, \"phenotypes\"=>[]}]}, {\"id\"=>194, \"entrez_name\"=>\"ERG\", \"entrez_id\"=>2078, \"name\"=>\"EWSR1-ERG\", \"description\"=>\"EWSR1-ERG is found in 5% of patients with Ewing Sarcoma and considered to be diagnostic of this disease, a small tumor more often found in bone, but less frequently in soft tissue. To better identify EWSR1-ERG in patients, conventional cytogenetics methods were analyzed: FISH and RT-PCR. Ultimately concluding that more than one method is needed to accurately diagnose patients due to the strengths and weaknesses of each individual method.\", \"gene_id\"=>71, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>25.0, \"coordinates\"=>{\"chromosome\"=>\"22\", \"start\"=>29664007, \"stop\"=>29683123, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000397938.2\", \"chromosome2\"=>\"21\", \"start2\"=>39751949, \"stop2\"=>39755845, \"representative_transcript2\"=>\"ENST00000398907.1\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/71/summary/variants/194/summary#variant\", \"evidence_items\"=>[{\"id\"=>475, \"name\"=>\"EID475\", \"description\"=>\"t(21;22) rearrangements of EWSR1-ERG are associated with a diagnosis of Ewing sarcoma and are the second most common gene fusion reported in this disease.\", \"disease\"=>{\"id\"=>50, \"name\"=>\"Ewing Sarcoma\", \"display_name\"=>\"Ewing Sarcoma\", \"doid\"=>\"3369\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3369\"}, \"drugs\"=>[], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>289, \"name\"=>\"Integrated multimodal genetic testing of Ewing sarcoma--a single-institution experience.\", \"citation\"=>\"Warren et al., 2013, Hum. Pathol.\", \"pubmed_id\"=>\"23706910\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23706910\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>10}, \"journal\"=>\"Hum. Pathol.\", \"full_journal_title\"=>\"Human pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>194, \"phenotypes\"=>[]}]}, {\"id\"=>195, \"entrez_name\"=>\"ERG\", \"entrez_id\"=>2078, \"name\"=>\"TMPRSS2-ERG\", \"description\"=>\"TMPRSS2 fusions are prevalent in Prostate Cancer. TMPRSS2 fusion with ETS family members such as ERG are among the most common. Fusion of TMPRSS2 with ERG may result in overexpression of ERG driven by the TMPRSS2 androgen-responsive promoter elements. Due to the high prevalence of this fusion, it may have diagnostic utility for prostate cancer, especially ERG expression positive prostate cancer.\", \"gene_id\"=>71, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"21\", \"start\"=>42879877, \"stop\"=>42879992, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000398585.3\", \"chromosome2\"=>\"21\", \"start2\"=>39751949, \"stop2\"=>39817544, \"representative_transcript2\"=>\"ENST00000398910.1\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/71/summary/variants/195/summary#variant\", \"evidence_items\"=>[{\"id\"=>476, \"name\"=>\"EID476\", \"description\"=>\"TMPRSS2/ERG gene fusion was observed in 23 of 29 (79%) of prostate carcinoma samples, and 90% of ERG expression positive prostate carcinoma samples.\", \"disease\"=>{\"id\"=>101, \"name\"=>\"Prostate Carcinoma\", \"display_name\"=>\"Prostate Carcinoma\", \"doid\"=>\"10286\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10286\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>290, \"name\"=>\"Recurrent fusion of TMPRSS2 and ETS transcription factor genes in prostate cancer.\", \"citation\"=>\"Tomlins et al., 2005, Science\", \"pubmed_id\"=>\"16254181\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16254181\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>10, \"day\"=>28}, \"journal\"=>\"Science\", \"full_journal_title\"=>\"Science (New York, N.Y.)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>195, \"phenotypes\"=>[]}]}, {\"id\"=>197, \"entrez_name\"=>\"FLI1\", \"entrez_id\"=>2313, \"name\"=>\"EWSR1-FLI1\", \"description\"=>\"\", \"gene_id\"=>53, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>25.0, \"coordinates\"=>{\"chromosome\"=>\"22\", \"start\"=>29664007, \"stop\"=>29683123, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000397938.2\", \"chromosome2\"=>\"11\", \"start2\"=>128651853, \"stop2\"=>128683162, \"representative_transcript2\"=>\"ENST00000527786.2\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/53/summary/variants/197/summary#variant\", \"evidence_items\"=>[{\"id\"=>478, \"name\"=>\"EID478\", \"description\"=>\"t(11;22) rearrangements of EWSR1-FLI1 are diagnostic of ewing sarcoma, and are observed in 90% of cytogenetic samples.\", \"disease\"=>{\"id\"=>50, \"name\"=>\"Ewing Sarcoma\", \"display_name\"=>\"Ewing Sarcoma\", \"doid\"=>\"3369\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3369\"}, \"drugs\"=>[], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>289, \"name\"=>\"Integrated multimodal genetic testing of Ewing sarcoma--a single-institution experience.\", \"citation\"=>\"Warren et al., 2013, Hum. Pathol.\", \"pubmed_id\"=>\"23706910\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23706910\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>10}, \"journal\"=>\"Hum. Pathol.\", \"full_journal_title\"=>\"Human pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>197, \"phenotypes\"=>[]}]}, {\"id\"=>209, \"entrez_name\"=>\"PAX8\", \"entrez_id\"=>7849, \"name\"=>\"PAX8-PPARG\", \"description\"=>\"PAX8-PPAR gamma is a oncogene caused by a translocation between 2q13 and 3p25 resulting in an in-frame fusion between the two genes. It has been observed in Follicular Thyroid Cancer (FTC) and is one of two known fusion products involving PPAR gamma in FTC indicating this region may be a breakpoint hotspot. Studies have indicated that the fusion product increases cell viability and proliferation and decreases apoptosis.\", \"gene_id\"=>75, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>114036521, \"stop\"=>113992971, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000429538.3\", \"chromosome2\"=>\"3\", \"start2\"=>12329400, \"stop2\"=>12475810, \"representative_transcript2\"=>\"ENST00000397015.2\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/75/summary/variants/209/summary#variant\", \"evidence_items\"=>[{\"id\"=>498, \"name\"=>\"EID498\", \"description\"=>\"PAX8–PPARG gene fusions were found in 12/33 follicular thyroid carcinoma tumor samples, but were rarely found in thyroid Hürthle cell tumors.\", \"disease\"=>{\"id\"=>246, \"name\"=>\"Follicular Thyroid Carcinoma\", \"display_name\"=>\"Follicular Thyroid Carcinoma\", \"doid\"=>\"3962\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3962\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>304, \"name\"=>\"RAS point mutations and PAX8-PPAR gamma rearrangement in thyroid tumors: evidence for distinct molecular pathways in thyroid follicular carcinoma.\", \"citation\"=>\"Nikiforova et al., 2003, J. Clin. Endocrinol. Metab.\", \"pubmed_id\"=>\"12727991\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/12727991\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2003, \"month\"=>5}, \"journal\"=>\"J. Clin. Endocrinol. Metab.\", \"full_journal_title\"=>\"The Journal of clinical endocrinology and metabolism\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>209, \"phenotypes\"=>[]}]}, {\"id\"=>241, \"entrez_name\"=>\"ABL1\", \"entrez_id\"=>25, \"name\"=>\"BCR-ABL F317L\", \"description\"=>\"BCR-ABL F317L, like the similar BCR-ABL T315I mutation, is becoming a common clinical marker for resistance to front-line therapies in CML. It has been shown to confer resistance to dasatinib, but responds well to ponatinib and other second generation inhibitors.\", \"gene_id\"=>4, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>55.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>133748288, \"stop\"=>133748288, \"reference_bases\"=>\"T\", \"variant_bases\"=>\"C\", \"representative_transcript\"=>\"ENST00000318560.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4/summary/variants/241/summary#variant\", \"evidence_items\"=>[{\"id\"=>2674, \"name\"=>\"EID2674\", \"description\"=>\"A panel of 18 mutations representing 85% of known mutations seen in imatinib CML patents were used to evaluate the effectiveness of drug binding. All mutations were compared with non-mutated BCR-ABL protein and were grouped according to their anti-proliferative activity. Mutant cell lines that exhibited a greater than 4-fold increase in IC50 value were considered resistant to drug, 2-4 fold increase in IC50 were moderately resistant and less than 2-fold change in IC50 was sensitive to drug treatment. The F317L mutation was moderately resistant to imatinib.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1529, \"name\"=>\"Activity of bosutinib, dasatinib, and nilotinib against 18 imatinib-resistant BCR/ABL mutants.\", \"citation\"=>\"Redaelli et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19075254\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19075254\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>241, \"phenotypes\"=>[]}, {\"id\"=>4382, \"name\"=>\"EID4382\", \"description\"=>\"In a retrospective study of 118 imatinib-resistant or -intolerant, Philadelphia chromosome BCR-ABL1 fusion positive, chronic myeloid leukemia (chronic phase) patients, patients with mutations in the ABL1 portion of the BCR-ABL gene fusion were associated with an complete hematologic response (MaHR) and complete major cytogenetic response (MCyR) to bosutinib monotherapy (MaHR:67% (26/39) vs. MaHR:77% (34/44), MCyR:31% (11/35) vs. MCyR: 35% (15/43), comparable to patients without mutation in the ABL1 portion. In a retrospective study of 118 imatinib-resistant or -intolerant, Philadelphia chromosome BCR-ABL1 fusion positive, chronic myeloid leukemia (chronic phase) patients, 4 and 1 patients achieved a complete hematologic response or major cytogenetic response to bosutinib monotherapy respectively, of 8 patients with the F317L mutation in the ABL1 portion of the BCR-ABL gene fusion.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>520, \"name\"=>\"Bosutinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1997, \"name\"=>\"Bosutinib is active in chronic phase chronic myeloid leukemia after imatinib and dasatinib and/or nilotinib therapy failure.\", \"citation\"=>\"Khoury et al., 2012, Blood\", \"pubmed_id\"=>\"22371878\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22371878\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4916559\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>4, \"day\"=>12}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>241, \"phenotypes\"=>[]}, {\"id\"=>4381, \"name\"=>\"EID4381\", \"description\"=>\"In a retrospective study of 288 imatinib-resistant or -intolerant, Philadelphia chromosome BCR-ABL1 fusion positive, chronic myeloid leukemia (chronic phase) patients, patients with mutations in the ABL1 portion of the BCR-ABL gene fusion were associated with an complete hematologic response (MaHR) or complete major cytogenic response (MCyR) to bosutinib monotherapy (MaHR=97% (33/34) vs. MaHR=100% (27/27), MCyR=69% (41/59) vs. MCyR=54% (19/35), comparable to patients without mutation in the ABL1 portion. In a retrospective study of 288 imatinib-resistant or -intolerant, Philadelphia chromosome BCR-ABL1 fusion positive, chronic myeloid leukemia (chronic phase) patients, 3/3 of evaluated patients with the F317L mutation in the ABL1 portion of the BCR-ABL gene fusion achieved a major cytogenetic response to bosutinib monotherapy respectively.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>520, \"name\"=>\"Bosutinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1996, \"name\"=>\"Safety and efficacy of bosutinib (SKI-606) in chronic phase Philadelphia chromosome-positive chronic myeloid leukemia patients with resistance or intolerance to imatinib.\", \"citation\"=>\"Cortes et al., 2011, Blood\", \"pubmed_id\"=>\"21865346\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21865346\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4916618\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>10, \"day\"=>27}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>241, \"phenotypes\"=>[]}, {\"id\"=>4387, \"name\"=>\"EID4387\", \"description\"=>\"In a retrospective study of 84 imatinib-resistant or -intolerant Philadelphia chromosome BCR-ABL1 fusion positive (Ph+) chronic myeloid leukemia (in accelerated-phase or with blast crisis) or Ph+ acute lymphoblastic leukemia patients, patients with the F317L mutation in the ABL1 portion of the BCR-ABL1 gene fusion were associated with a  complete cytogenetic response to dasatinib monotherapy.\", \"disease\"=>{\"id\"=>43, \"name\"=>\"Acute Lymphocytic Leukemia\", \"display_name\"=>\"Acute Lymphocytic Leukemia\", \"doid\"=>\"9952\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9952\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2096, \"name\"=>\"Dasatinib in imatinib-resistant Philadelphia chromosome-positive leukemias.\", \"citation\"=>\"Talpaz et al., 2006, N. Engl. J. Med.\", \"pubmed_id\"=>\"16775234\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16775234\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2006, \"month\"=>6, \"day\"=>15}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>241, \"phenotypes\"=>[]}, {\"id\"=>4386, \"name\"=>\"EID4386\", \"description\"=>\"In a study of 51 imatinib-resistant chronic myeloid leukemia patients, patients with the F317L or F317V mutation in the ABL1 portion of the BCR-ABL1 gene fusion (n=10) were associated with acquired resistant to dasatinib monotherapy.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2093, \"name\"=>\"In chronic myeloid leukemia patients on second-line tyrosine kinase inhibitor therapy, deep sequencing of BCR-ABL1 at the time of warning may allow sensitive detection of emerging drug-resistant mutants.\", \"citation\"=>\"Soverini et al., 2016, BMC Cancer\", \"pubmed_id\"=>\"27485109\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27485109\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4970247\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>8, \"day\"=>2}, \"journal\"=>\"BMC Cancer\", \"full_journal_title\"=>\"BMC cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>241, \"phenotypes\"=>[]}, {\"id\"=>4385, \"name\"=>\"EID4385\", \"description\"=>\"In a retrospective study (of phase 3 clinical trial NCT00481247) of 363 Philadelphia chromosome-positive (Ph+) chronic myeloid leukemia patients in chronic phase, a patient with the F317I mutation in the ABL1 portion of the BCR-ABL1 gene fusion, treated with dasatinib monotherapy (1/169) was associated with accelerated phase/blast phase and disease progression.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2091, \"name\"=>\"BCR-ABL1 mutation development during first-line treatment with dasatinib or imatinib for chronic myeloid leukemia in chronic phase.\", \"citation\"=>\"Hughes et al., 2015, Leukemia\", \"pubmed_id\"=>\"26118315\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26118315\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4559757\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>9}, \"journal\"=>\"Leukemia\", \"full_journal_title\"=>\"Leukemia\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>241, \"phenotypes\"=>[]}, {\"id\"=>4384, \"name\"=>\"EID4384\", \"description\"=>\"In a retrospective study of 45 imatinib-resistant or -intolerant Philadelphia chromosome BCR-ABLfusion positive (Ph+) chronic myeloid leukemia or Ph+ acute lymphoblastic leukemia (ALL) patients, patients harboring F317L mutation in the ABL1 portion of the BCR-ABL gene fusion (n=3) were associated with primary (n=1) or acquired resistance (n=2) to dasatinib monotherapy of 21 dasatinib resistant or non-responsive patients.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2095, \"name\"=>\"Resistance to dasatinib in Philadelphia-positive leukemia patients and the presence or the selection of mutations at residues 315 and 317 in the BCR-ABL kinase domain.\", \"citation\"=>\"Soverini et al., 2007, Haematologica\", \"pubmed_id\"=>\"17339191\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17339191\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>3}, \"journal\"=>\"Haematologica\", \"full_journal_title\"=>\"Haematologica\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>241, \"phenotypes\"=>[]}, {\"id\"=>4383, \"name\"=>\"EID4383\", \"description\"=>\"In an in vitro study, a Ba/F3  cell line stably expressing the F317L mutation in the ABL1 portion of the BCR-ABL1 fusion protein was associated with reduced sensitivity to dasatinib treatment (IC50: 8.16nM vs. IC50: 1.83nM) as compared to Ba/F3 cells expressing BCR-ABL1 fusion protein. Resistance was determined by assessing cell proliferation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1529, \"name\"=>\"Activity of bosutinib, dasatinib, and nilotinib against 18 imatinib-resistant BCR/ABL mutants.\", \"citation\"=>\"Redaelli et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19075254\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19075254\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>241, \"phenotypes\"=>[]}, {\"id\"=>4380, \"name\"=>\"EID4380\", \"description\"=>\"In an in vitro study, a Ba/F3 murine cell line stably expressing the F317L mutation in the ABL1 portion of the BCR-ABL1 fusion protein was associated with reduced sensitivity to bosutinib treatment (IC50: 100.7nM vs. IC50: 41.61nM) as compared to Ba/F3 cells expressing BCR-ABL1 fusion protein. Resistance was assessed by determining cell proliferation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>520, \"name\"=>\"Bosutinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1529, \"name\"=>\"Activity of bosutinib, dasatinib, and nilotinib against 18 imatinib-resistant BCR/ABL mutants.\", \"citation\"=>\"Redaelli et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19075254\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19075254\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>241, \"phenotypes\"=>[]}, {\"id\"=>2675, \"name\"=>\"EID2675\", \"description\"=>\"In a study with 144 chronic myeloid leukemia (CML) patients, mutations in the BCR-ABL1 fusion gene were detected in 27 samples. Resistance to first-line chemotherapy drug imatinib was observed in 24 of the 27 patients with mutations.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1530, \"name\"=>\"Detection of BCR-ABL mutations in patients with CML treated with imatinib is virtually always accompanied by clinical resistance, and mutations in the ATP phosphate-binding loop (P-loop) are associated with a poor prognosis.\", \"citation\"=>\"Branford et al., 2003, Blood\", \"pubmed_id\"=>\"12623848\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/12623848\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2003, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>241, \"phenotypes\"=>[]}, {\"id\"=>2677, \"name\"=>\"EID2677\", \"description\"=>\"In an in vitro study, a Ba/F3 cell line stably expressing the F317L mutation in the ABL1 portion of the BCR-ABL1 fusion protein was associated with sensitivity to nilotinib treatment (IC50: 929.8 nM vs. > 2000.0 nM), as compared to Ba/F3 parental cells. Sensitivity was determined by assessing cell proliferation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>467, \"name\"=>\"Nilotinib\", \"pubchem_id\"=>\"\"}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1621, \"name\"=>\"Comparison of imatinib, dasatinib, nilotinib and INNO-406 in imatinib-resistant cell lines.\", \"citation\"=>\"Deguchi et al., 2008, Leuk. Res.\", \"pubmed_id\"=>\"18191450\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18191450\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2008, \"month\"=>6}, \"journal\"=>\"Leuk. Res.\", \"full_journal_title\"=>\"Leukemia research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>241, \"phenotypes\"=>[]}, {\"id\"=>2676, \"name\"=>\"EID2676\", \"description\"=>\"In two cell-based studies, a panel of either 18 or 16 mutations representing 85% of known mutations seen in imatinib CML patents were used to evaluate the effectiveness of drug binding. All mutations were compared with non-mutated BCR-ABL protein and were grouped according to their anti-proliferative activity. Mutant cell lines that exhibited a greater than 4-fold increase in IC50 value were considered resistant to drug, 2-4 fold increase in IC50 were moderately resistant and less than 2-fold change in IC50 was sensitive to drug treatment. The F317L mutation was  moderately resistant to imatinib .\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1531, \"name\"=>\"In vitro activity of Bcr-Abl inhibitors AMN107 and BMS-354825 against clinically relevant imatinib-resistant Abl kinase domain mutants.\", \"citation\"=>\"O'Hare et al., 2005, Cancer Res.\", \"pubmed_id\"=>\"15930265\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15930265\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>6, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>241, \"phenotypes\"=>[]}, {\"id\"=>638, \"name\"=>\"EID638\", \"description\"=>\"Molecular dynamics simulations and calculated solvated interaction energies indicate BCR-ABL F317L mutations are Ponatinib sensitive.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>55, \"name\"=>\"Ponatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>401, \"name\"=>\"Ponatinib is a pan-BCR-ABL kinase inhibitor: MD simulations and SIE study.\", \"citation\"=>\"Tanneeru et al., 2013, PLoS ONE\", \"pubmed_id\"=>\"24236021\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24236021\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3827254\", \"publication_date\"=>{\"year\"=>2013}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>241, \"phenotypes\"=>[]}, {\"id\"=>639, \"name\"=>\"EID639\", \"description\"=>\"Ponatinib has shown to be effective in patients with BCR-ABL kinase domain mutations like F317L.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>55, \"name\"=>\"Ponatinib\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>402, \"name\"=>\"Resistant mutations in CML and Ph(+)ALL - role of ponatinib.\", \"citation\"=>\"Miller et al., 2014, Biologics\", \"pubmed_id\"=>\"25349473\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25349473\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4208348\", \"publication_date\"=>{\"year\"=>2014}, \"journal\"=>\"Biologics\", \"full_journal_title\"=>\"Biologics : targets & therapy\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>241, \"phenotypes\"=>[]}, {\"id\"=>637, \"name\"=>\"EID637\", \"description\"=>\"BCR-ABL F317L has been shown to confer resistance to dasatinib in patients with CML.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>400, \"name\"=>\"Characteristics and outcome of chronic myeloid leukemia patients with F317L BCR-ABL kinase domain mutation after therapy with tyrosine kinase inhibitors.\", \"citation\"=>\"Jabbour et al., 2008, Blood\", \"pubmed_id\"=>\"18818391\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18818391\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4186640\", \"publication_date\"=>{\"year\"=>2008, \"month\"=>12, \"day\"=>15}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>241, \"phenotypes\"=>[]}]}, {\"id\"=>269, \"entrez_name\"=>\"ROS1\", \"entrez_id\"=>6098, \"name\"=>\"REARRANGEMENT\", \"description\"=>\"ROS1 is a receptor tyrosine kinase that frequently forms gene fusions in several cancer types. Approximately 2% of NSCLC patients harbor a ROS1 gene fusion. Several fusion partners have been described with a constant ROS1-breakpoint, maintaining the kinase domain (Solomon B, 2015). Following case-studies and retrospective analyses, an expansion cohort of a phase-1 study showed clinical benefit of crizotinib in ROS1 rearranged NSCLC. The benefit was independent of the type of ROS1-rearrangement (Shaw et. al.,2014).\", \"gene_id\"=>4941, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>68.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>117609463, \"stop\"=>117747018, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000368508.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4941/summary/variants/269/summary#variant\", \"evidence_items\"=>[{\"id\"=>2951, \"name\"=>\"EID2951\", \"description\"=>\"Two phase I studies in patients with advanced or metastatic solid tumors, including patients with active central nervous system (CNS) disease. Here, we summarize the overall safety and report the antitumor activity of entrectinib in a cohort of patients with tumors harboring NTRK1/2/3, ROS1, or ALK gene fusions Responses were observed in non-small cell lung cancer, colorectal cancer, mammary analogue secretory carcinoma, melanoma, and renal cell carcinoma, as early as 4 weeks after starting treatment and lasting as long as >2 years. Notably, a complete CNS response was achieved in a patient with SQSTM1-NTRK1-rearranged lung cancer.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>426, \"name\"=>\"Entrectinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1689, \"name\"=>\"Safety and Antitumor Activity of the Multitargeted Pan-TRK, ROS1, and ALK Inhibitor Entrectinib: Combined Results from Two Phase I Trials (ALKA-372-001 and STARTRK-1).\", \"citation\"=>\"Drilon et al., 2017, Cancer Discov\", \"pubmed_id\"=>\"28183697\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28183697\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5380583\", \"publication_date\"=>{\"year\"=>2017, \"month\"=>4}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>269, \"phenotypes\"=>[]}, {\"id\"=>1443, \"name\"=>\"EID1443\", \"description\"=>\"Study describes a 31-year old male, never-smoker patient with advanced NSCLC with a ROS1 rearrangement. The patient was treated with crizotinib with marked response at 1 week and resolution of the tumor confirmed at 8 and 12 weeks via CT-scans. It was also demonstrated that crizotinib inhibits ROS1 phosphorylation in HEK 293 cells transfected with a CD14-ROS1 fusion gene, and crizotinib inhibited growth in the HCC78 cell line containing a ROS1 rearrangement to a similar degree as ALK-fusion positive cell lines.\", \"disease\"=>{\"id\"=>1163, \"name\"=>\"Bronchiolo-alveolar Adenocarcinoma\", \"display_name\"=>\"Bronchiolo-alveolar Adenocarcinoma\", \"doid\"=>\"4926\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4926\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>975, \"name\"=>\"ROS1 rearrangements define a unique molecular class of lung cancers.\", \"citation\"=>\"Bergethon et al., 2012, J. Clin. Oncol.\", \"pubmed_id\"=>\"22215748\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22215748\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3295572\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>3, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>269, \"phenotypes\"=>[]}, {\"id\"=>1690, \"name\"=>\"EID1690\", \"description\"=>\"In an evaluation of a patient and never smoker with crizotinib-sensitive lung adenocarcinoma, an SDC4-ROS1 rearrangement was identified using comprehensive genomic profiling. The patient attained a decrease in RESIST target lesions by 26.8% (stable disease); however, the patient relapsed 4 months later. This patient also harbored a TP53 mutation, MCL1 amplification and CDKN2A deletion.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1141, \"name\"=>\"Detection of Crizotinib-Sensitive Lung Adenocarcinomas With MET, ALK, and ROS1 Genomic Alterations via Comprehensive Genomic Profiling.\", \"citation\"=>\"Le et al., 2015, Clin Lung Cancer\", \"pubmed_id\"=>\"25922291\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25922291\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4418215\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>9}, \"journal\"=>\"Clin Lung Cancer\", \"full_journal_title\"=>\"Clinical lung cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>269, \"phenotypes\"=>[]}, {\"id\"=>1301, \"name\"=>\"EID1301\", \"description\"=>\"A tissue microarray of 268 patients with histologically confirmed colorectal adenocarcinoma was performed to search for ROS1 gene fusion events to guide potential targeted therapy in future cases. Break apart FISH assay, which does not distinguish ROS1-fusion partner, was used on the tissue microarray to determine ROS1 fusion status. 2 cases of ROS1 fusions were found. 1 case was verified by RT-PCR to be a SLC34A2-ROS1 fusion, but this case was also positive for BRAF V600E.  The second case of ROS1 fusion was of undetermined fusion partner, where known ROS1 fusion parters SLC34A2, CD74 and SDC4 were tested. This case did not have KRAS or BRAF mutations, indicating that ROS1 fusion may act in a small subset of colorectal cancer as a driver targetable by crizotinib or other inhibitors.\", \"disease\"=>{\"id\"=>57, \"name\"=>\"Colorectal Adenocarcinoma\", \"display_name\"=>\"Colorectal Adenocarcinoma\", \"doid\"=>\"0050861\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050861\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"E\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>894, \"name\"=>\"ROS1 and ALK fusions in colorectal cancer, with evidence of intratumoral heterogeneity for molecular drivers.\", \"citation\"=>\"Aisner et al., 2014, Mol. Cancer Res.\", \"pubmed_id\"=>\"24296758\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24296758\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4140177\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>1}, \"journal\"=>\"Mol. Cancer Res.\", \"full_journal_title\"=>\"Molecular cancer research : MCR\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>269, \"phenotypes\"=>[]}, {\"id\"=>1248, \"name\"=>\"EID1248\", \"description\"=>\"Case report of a 77-year old male patient with lung adenocarcinoma (T2aN1M0, stage IIA) without EGFR, KRAS, or BRAF mutations, and no ALK gene rearrangement. CD72-ROS1 was identified in sequencing after progression on chemotherapy. Crizotinib at 250 mg orally twice daily led to resolution of metastatic disease. The patient remained disease-free for 13 months when a CT-scan showed a relapse with two nodules in the right lower lobe. Stereotactic radiation therapy, gamma knife radiosurgery, and ipilimumab did not lead to a relevant response (new pleural nodules, new intracranial metastases). The patient next received ceritinib on a “Signature Trial\\\" for tumors with aberrations in ALK or ROS1 that are treated with oral ceritinib (LDK378) at 750 mg daily (NCT02186821). Restaging scans after two cycles showed a partial response confirmed after four cycles (56% decrease) per RECIST1.1. MRI showed that his brain metastases decreased as well.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>159, \"name\"=>\"Ceritinib\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>865, \"name\"=>\"Clinical activity of ceritinib in ROS1-rearranged non-small cell lung cancer: Bench to bedside report.\", \"citation\"=>\"Subbiah et al., 2016, Proc. Natl. Acad. Sci. U.S.A.\", \"pubmed_id\"=>\"26917690\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26917690\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4801313\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>3, \"day\"=>15}, \"journal\"=>\"Proc. Natl. Acad. Sci. U.S.A.\", \"full_journal_title\"=>\"Proceedings of the National Academy of Sciences of the United States of America\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>269, \"phenotypes\"=>[]}, {\"id\"=>1442, \"name\"=>\"EID1442\", \"description\"=>\"Study describes the expansion of a phase 1 study of crizotinib in a 50 patient cohort with advanced NSCLC and exhibiting ROS1 rearrangements. Among the 50 patients 72% exhibited a response (3 complete, 33 partial) according to RECIST defined criteria. The median duration of response and median progression free survival were 14.5 and 19.2 months respectively (95% CI).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"rejected\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>447, \"name\"=>\"Crizotinib in ROS1-rearranged non-small-cell lung cancer.\", \"citation\"=>\"Shaw et al., 2014, N. Engl. J. Med.\", \"pubmed_id\"=>\"25264305\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25264305\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4264527\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>11, \"day\"=>20}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>269, \"phenotypes\"=>[]}, {\"id\"=>1030, \"name\"=>\"EID1030\", \"description\"=>\"In a retrospective study, 30 patients were evaluated after crizotinib treatment for stage IV NSCLC confirmed for ROS1 rearrangement. On crizotinib, 24 of the patients achieved objective responses including 5 complete responses. Median progression free survival was 9.1 months and all patients had  been pretreated with 1 or more lines of chemotherapy.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>715, \"name\"=>\"Crizotinib therapy for advanced lung adenocarcinoma and a ROS1 rearrangement: results from the EUROS1 cohort.\", \"citation\"=>\"Mazières et al., 2015, J. Clin. Oncol.\", \"pubmed_id\"=>\"25667280\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25667280\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>3, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>269, \"phenotypes\"=>[]}, {\"id\"=>688, \"name\"=>\"EID688\", \"description\"=>\"An expansion cohort of a phase-1 Crizotinib study found a 72% objective response rate among 50 patients with ROS1 rearrangement. The study did not compare to patients without rearrangement that were treated with Crizotinib.  Several different ROS1 fusion partners were identified, but no clinical differences in outcome were associated with the identity of the 5' partner.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>447, \"name\"=>\"Crizotinib in ROS1-rearranged non-small-cell lung cancer.\", \"citation\"=>\"Shaw et al., 2014, N. Engl. J. Med.\", \"pubmed_id\"=>\"25264305\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25264305\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4264527\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>11, \"day\"=>20}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>269, \"phenotypes\"=>[]}]}, {\"id\"=>273, \"entrez_name\"=>\"RET\", \"entrez_id\"=>5979, \"name\"=>\"KIF5B-RET\", \"description\"=>\"In a study by Takeuchi et. al. a fusion screen was performed in 1529 lung cancer samples. 14 adenocarcinomas with KIF5B-RET and CCDC6-RET fusions were identified. (Takeuchi K et. al., 2012)\", \"gene_id\"=>42, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>32306071, \"stop\"=>32345359, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000302418.4\", \"chromosome2\"=>\"10\", \"start2\"=>43609928, \"stop2\"=>43625799, \"representative_transcript2\"=>\"ENST00000355710.3\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/42/summary/variants/273/summary#variant\", \"evidence_items\"=>[{\"id\"=>1622, \"name\"=>\"EID1622\", \"description\"=>\"Case report of a 74-year old woman with EGFR- and ALK-negative lung adenocarcinoma. FISH identified a deletion of 5'RET (FISH, indicating RET-gene rearrangement) and genomic testing confirmed a fusion of KIF5B and RET. The patient was enrolled in a clinical trial with vandetanib and everolimus (NCT01582191) with everolimus used to enhance brain accumulation of vandetanib. After two cycles of therapy a repeat MRI brain showed a decrease in the intracranial disease burden and PET/CT showed systemic response as well.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}, {\"id\"=>388, \"name\"=>\"Vandetanib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1074, \"name\"=>\"Systemic and CNS activity of the RET inhibitor vandetanib combined with the mTOR inhibitor everolimus in KIF5B-RET re-arranged non-small cell lung cancer with brain metastases.\", \"citation\"=>\"Subbiah et al., 2015, Lung Cancer\", \"pubmed_id\"=>\"25982012\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25982012\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4998046\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>7}, \"journal\"=>\"Lung Cancer\", \"full_journal_title\"=>\"Lung cancer (Amsterdam, Netherlands)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>273, \"phenotypes\"=>[]}, {\"id\"=>698, \"name\"=>\"EID698\", \"description\"=>\"A case study of a patient with EGFR, KRAS, BRAF, HER2, ALK, ROS1 and MET negative adenocarcinoma of the lung. FISH analysis revealed a KIF5B-RET fusion. The RET inhibitor Vandetanib led to remission in the patient.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>117, \"name\"=>\"Vandetanib\", \"pubchem_id\"=>\"\"}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>378, \"name\"=>\"A patient with lung adenocarcinoma and RET fusion treated with vandetanib.\", \"citation\"=>\"Gautschi et al., 2013, J Thorac Oncol\", \"pubmed_id\"=>\"23584301\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23584301\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>5}, \"journal\"=>\"J Thorac Oncol\", \"full_journal_title\"=>\"Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>273, \"phenotypes\"=>[]}]}, {\"id\"=>285, \"entrez_name\"=>\"BRAF\", \"entrez_id\"=>673, \"name\"=>\"AGK-BRAF\", \"description\"=>\"\", \"gene_id\"=>5, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>7.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>141250989, \"stop\"=>141255367, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000355413.4\", \"chromosome2\"=>\"7\", \"start2\"=>140434279, \"stop2\"=>140494267, \"representative_transcript2\"=>\"ENST00000288602.6\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5/summary/variants/285/summary#variant\", \"evidence_items\"=>[{\"id\"=>723, \"name\"=>\"EID723\", \"description\"=>\"BRAF fusion AGK-BRAF was associated with decreased sensitivity to vemurafenib and increased sensitivity to sorafenib in-vitro. A single patient with this fusion showed durable response to sorafenib.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>6, \"name\"=>\"Sorafenib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>475, \"name\"=>\"Recurrent BRAF kinase fusions in melanocytic tumors offer an opportunity for targeted therapy.\", \"citation\"=>\"Botton et al., 2013, Pigment Cell Melanoma Res\", \"pubmed_id\"=>\"23890088\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23890088\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3808507\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>11}, \"journal\"=>\"Pigment Cell Melanoma Res\", \"full_journal_title\"=>\"Pigment cell & melanoma research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>285, \"phenotypes\"=>[]}, {\"id\"=>724, \"name\"=>\"EID724\", \"description\"=>\"A melanoma cell line with AGK-BRAF in-frame fusion showed decreased sensitivity towards Vemurafenib in comparison with BRAF mutated (V600E) cell lines.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>4, \"name\"=>\"Vemurafenib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>475, \"name\"=>\"Recurrent BRAF kinase fusions in melanocytic tumors offer an opportunity for targeted therapy.\", \"citation\"=>\"Botton et al., 2013, Pigment Cell Melanoma Res\", \"pubmed_id\"=>\"23890088\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23890088\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3808507\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>11}, \"journal\"=>\"Pigment Cell Melanoma Res\", \"full_journal_title\"=>\"Pigment cell & melanoma research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>285, \"phenotypes\"=>[]}]}, {\"id\"=>286, \"entrez_name\"=>\"BRAF\", \"entrez_id\"=>673, \"name\"=>\"PAPSS1-BRAF\", \"description\"=>\"\", \"gene_id\"=>5, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>31, \"name\"=>\"gene_fusion\", \"display_name\"=>\"Gene Fusion\", \"so_id\"=>\"SO:0001565\", \"description\"=>\"A sequence variant whereby a two genes have become joined.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001565\"}], \"civic_actionability_score\"=>6.0, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>108603171, \"stop\"=>108641608, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000265174.4\", \"chromosome2\"=>\"7\", \"start2\"=>140434279, \"stop2\"=>140487384, \"representative_transcript2\"=>\"ENST00000288602.6\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5/summary/variants/286/summary#variant\", \"evidence_items\"=>[{\"id\"=>726, \"name\"=>\"EID726\", \"description\"=>\"BRAF-fusion in \\\"pan-negative\\\" melanomas were identified in TCGA data. Cell-lines with a PAPSS1-BRAF fusion were resistant to treatment with Vemurafenib but sensitive to treatment with Trametinib. This fusion is believed to activate MAPK pathway signaling.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>19, \"name\"=>\"Trametinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>476, \"name\"=>\"BRAF fusions define a distinct molecular subset of melanomas with potential sensitivity to MEK inhibition.\", \"citation\"=>\"Hutchinson et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"24345920\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24345920\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3880773\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>12, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>286, \"phenotypes\"=>[]}, {\"id\"=>725, \"name\"=>\"EID725\", \"description\"=>\"BRAF-fusion in \\\"pan-negative\\\" melanomas were identified in TCGA data. Cell-lines with PAPSS1-BRAF fusion were resistant to treatment with Vemurafenib.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>4, \"name\"=>\"Vemurafenib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>476, \"name\"=>\"BRAF fusions define a distinct molecular subset of melanomas with potential sensitivity to MEK inhibition.\", \"citation\"=>\"Hutchinson et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"24345920\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24345920\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3880773\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>12, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>286, \"phenotypes\"=>[]}]}, {\"id\"=>287, \"entrez_name\"=>\"BRAF\", \"entrez_id\"=>673, \"name\"=>\"TRIM24-BRAF\", \"description\"=>\"\", \"gene_id\"=>5, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>6.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>138145079, \"stop\"=>138239711, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000343526.4\", \"chromosome2\"=>\"7\", \"start2\"=>140434279, \"stop2\"=>140487384, \"representative_transcript2\"=>\"ENST00000288602.6\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5/summary/variants/287/summary#variant\", \"evidence_items\"=>[{\"id\"=>1416, \"name\"=>\"EID1416\", \"description\"=>\"Foundation One NGS assay and targeted RNAseq identified PAPSS1-BRAF fusion in a melanoma sample. Further BRAF fusions (TRIM24-BRAF) were identified in TCGA and additional samples. Ectopic expression of engineered cDNA in 293H cells showed that trametinib led to reduced ERK1/2 phosphorylation in fusion positive cells whereas vemurafenib was not effective.\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>4, \"name\"=>\"Vemurafenib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>476, \"name\"=>\"BRAF fusions define a distinct molecular subset of melanomas with potential sensitivity to MEK inhibition.\", \"citation\"=>\"Hutchinson et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"24345920\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24345920\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3880773\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>12, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>287, \"phenotypes\"=>[]}, {\"id\"=>727, \"name\"=>\"EID727\", \"description\"=>\"A TRIM24-BRAF fusion was identified in a single patient with metastatic melanoma that was \\\"pan-negative\\\" for driver mutations. A cell-line (293H) ectopically expressing the TRIM24-BRAF fusion was found to be sensitive to the MEK-inhibitor Trametinib.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>19, \"name\"=>\"Trametinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>476, \"name\"=>\"BRAF fusions define a distinct molecular subset of melanomas with potential sensitivity to MEK inhibition.\", \"citation\"=>\"Hutchinson et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"24345920\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24345920\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3880773\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>12, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>287, \"phenotypes\"=>[]}]}, {\"id\"=>307, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"EML4-ALK L1152R\", \"description\"=>\"\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>29445270, \"stop\"=>29445270, \"reference_bases\"=>\"A\", \"variant_bases\"=>\"C\", \"representative_transcript\"=>\"ENST00000389048.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/307/summary#variant\", \"evidence_items\"=>[{\"id\"=>763, \"name\"=>\"EID763\", \"description\"=>\"Case-report of a patient with an acquired ALK L1152R mutation and resistance to crizotinib treatment. A cell line model also showed resistance against ALK inhibition.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>506, \"name\"=>\"A novel ALK secondary mutation and EGFR signaling cause resistance to ALK kinase inhibitors.\", \"citation\"=>\"Sasaki et al., 2011, Cancer Res.\", \"pubmed_id\"=>\"21791641\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21791641\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3278914\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>9, \"day\"=>15}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>307, \"phenotypes\"=>[]}]}, {\"id\"=>308, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"EML4-ALK G1269A\", \"description\"=>\"The EML4-ALK G1269A mutation has been identified in non-small cell lung cancer patients resistant to crizotinib. Cell line data from multiple groups have shown this mutation enhances resistance to crizotinib treatment in vitro.\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>17.5, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>29432682, \"stop\"=>29432682, \"reference_bases\"=>\"C\", \"variant_bases\"=>\"G\", \"representative_transcript\"=>\"ENST00000389048.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/308/summary#variant\", \"evidence_items\"=>[{\"id\"=>4621, \"name\"=>\"EID4621\", \"description\"=>\"In an in vitro study, a FL5.12 cell line expressing NPM1-ALK or EML4-ALK (known sensitizing alterations to crizotinib) and an ALK G1269A co-mutation, demonstrated resistance to crizotinib treatment (IC50: 871.75 nmol/L and 395.2 nmol/L, respectively), as compared to FL5.12 cells expressing NPM1-ALK or EML4-ALK fusion proteins (IC50: 171.85 nmol/L and 70.59 nmol/L, respectively). Resistance was determined by assessing cell viability.\", \"disease\"=>{\"id\"=>164, \"name\"=>\"Lung Large Cell Carcinoma\", \"display_name\"=>\"Lung Large Cell Carcinoma\", \"doid\"=>\"4556\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4556\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2040, \"name\"=>\"TKI sensitivity patterns of novel kinase-domain mutations suggest therapeutic opportunities for patients with resistant ALK+ tumors.\", \"citation\"=>\"Amin et al., 2016, Oncotarget\", \"pubmed_id\"=>\"27009859\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27009859\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5029658\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>26}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>308, \"phenotypes\"=>[]}, {\"id\"=>4620, \"name\"=>\"EID4620\", \"description\"=>\"In a lung adenocarcinoma cancer patient with an EML4-ALK gene rearrangement (a known sensitizing alteration to crizotinib) and an ALK G1269A co-mutation, ALK G1269A was reported to be refractory to crizotinib treatment. The patient was treated with cisplatin and pemetrexed, pemetrexed monotherapy, and then crizotinib with a near complete response, followed by disease progression. Subsequently, the patient was treated with ceritinib and obtained a complete response.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2038, \"name\"=>\"Genomic Aberrations in Crizotinib Resistant Lung Adenocarcinoma Samples Identified by Transcriptome Sequencing.\", \"citation\"=>\"Saber et al., 2016, PLoS ONE\", \"pubmed_id\"=>\"27045755\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27045755\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4821611\", \"publication_date\"=>{\"year\"=>2016}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"submitted\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>308, \"phenotypes\"=>[]}, {\"id\"=>4619, \"name\"=>\"EID4619\", \"description\"=>\"In a non-small cell lung cancer patient with acquired crizotinib resistance, the tumor had an EML4-ALK gene rearrangement (a known sensitizing alteration to crizotinib) and an ALK G1269A co-mutation. The ALK G1269A tumor was reported to be refractory to crizotinib treatment. In an in vitro study, a Ba/F3 cell line expressing EML4-ALK fusion (a known sensitizing alteration to crizotinib) and an ALK G1269A co-mutation, demonstrated resistance to crizotinib treatment compared to Ba/F3 cells expressing EML4-ALK fusion. Resistance was determined by assessing cell viability.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>924, \"name\"=>\"The ALK inhibitor ceritinib overcomes crizotinib resistance in non-small cell lung cancer.\", \"citation\"=>\"Friboulet et al., 2014, Cancer Discov\", \"pubmed_id\"=>\"24675041\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24675041\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4068971\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>308, \"phenotypes\"=>[]}, {\"id\"=>4617, \"name\"=>\"EID4617\", \"description\"=>\"In a non-small cell lung cancer patient with acquired crizotinib resistance, the tumor had an ALK gene rearrangement and an ALK G1269A co-mutation. The ALK G1269A tumor was reported to be refractory to 11-month crizotinib treatment.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2034, \"name\"=>\"ALK rearrangements are mutually exclusive with mutations in EGFR or KRAS: an analysis of 1,683 patients with non-small cell lung cancer.\", \"citation\"=>\"Gainor et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"23729361\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23729361\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3874127\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>8, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"submitted\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>308, \"phenotypes\"=>[]}, {\"id\"=>4618, \"name\"=>\"EID4618\", \"description\"=>\"In a non-small cell lung cancer patient with a bone metastatic tumor harboring an ALK gene rearrangement and an ALK G1269A co-mutation, ALK G1269A was reported to be refractory to 12-month crizotinib treatment.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2035, \"name\"=>\"Multiplexed deep sequencing analysis of ALK kinase domain identifies resistance mutations in relapsed patients following crizotinib treatment.\", \"citation\"=>\"Huang et al., 2013, Genomics\", \"pubmed_id\"=>\"23434628\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23434628\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>9}, \"journal\"=>\"Genomics\", \"full_journal_title\"=>\"Genomics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>308, \"phenotypes\"=>[]}, {\"id\"=>765, \"name\"=>\"EID765\", \"description\"=>\"2 out of 7 patients with an acquired resistance to crizotinib had a G1269A mutation. One of these patients also harbored a L1196M mutation. In vitro follow-up showed reduced crizotinib-induced ALK dephosphorylation (293T) and apoptosis (Ba/F3) when EML4-ALK G1269A overexpression was compared to EML4-ALK overexpression in these cell lines.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>508, \"name\"=>\"Heterogeneity of genetic changes associated with acquired crizotinib resistance in ALK-rearranged lung cancer.\", \"citation\"=>\"Kim et al., 2013, J Thorac Oncol\", \"pubmed_id\"=>\"23344087\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23344087\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>4}, \"journal\"=>\"J Thorac Oncol\", \"full_journal_title\"=>\"Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>308, \"phenotypes\"=>[]}, {\"id\"=>764, \"name\"=>\"EID764\", \"description\"=>\"11 patients with crizotinib-resistant NSCLC were analyzed for this study. Two harbored the G1269A mutation that also led to crizotinib-resistance in-vitro.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>507, \"name\"=>\"Mechanisms of resistance to crizotinib in patients with ALK gene rearranged non-small cell lung cancer.\", \"citation\"=>\"Doebele et al., 2012, Clin. Cancer Res.\", \"pubmed_id\"=>\"22235099\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22235099\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3311875\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>3, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>308, \"phenotypes\"=>[]}]}, {\"id\"=>352, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"EML4-ALK C1156Y–L1198F\", \"description\"=>\"EML4-ALK C1156Y alone is resistant to crizotinib in non-small cell lung cancer (NSCLC) which has been shown to be modulated (both increased sensitivity and resistance) by additional ALK mutations. A case report showed that an NSCLC harboring EML4-ALK C1156Y–L1198F was resistant to lorlatinib but this second mutation re-sensitized this tumor to crizotinib.\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>29443625, \"stop\"=>29445258, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000389048.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/352/summary#variant\", \"evidence_items\"=>[{\"id\"=>843, \"name\"=>\"EID843\", \"description\"=>\"Case report of a patient with metastatic ALK-rearranged NSCLC. After developing resistance to first and second line ALK inhibitors mediated by a C1156Y mutation, the patient had a partial response with the third generation ALK-inhibitor lorlatinib. The patient progressed on lorlatinib after 8 months, at which point whole exome sequencing revealed an EML4-ALK C1156Y–L1198F double mutation. In vitro, the double mutation was resistant to lorlatinib but sensitive to crizotinib. The patient was started on crizotinib again and had a clinical response that lasted for 6 months.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>241, \"name\"=>\"Lorlatinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>567, \"name\"=>\"Resensitization to Crizotinib by the Lorlatinib ALK Resistance Mutation L1198F.\", \"citation\"=>\"Shaw et al., 2016, N. Engl. J. Med.\", \"pubmed_id\"=>\"26698910\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26698910\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4773904\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>1, \"day\"=>7}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>352, \"phenotypes\"=>[]}, {\"id\"=>844, \"name\"=>\"EID844\", \"description\"=>\"Case report of a patient with metastatic ALK-rearranged NSCLC. After developing resistance to first and second line ALK inhibitors mediated by a C1156Y mutation, the patient had a partial response with the third generation ALK-inhibitor lorlatinib. The patient progressed on lorlatinib after 8 months, at which point whole exome sequencing revealed an EML4-ALK C1156Y–L1198F double mutation. In vitro, the double mutation was resistant to lorlatinib but sensitive to crizotinib. The patient was started on crizotinib again and had a clinical response that lasted for 6 months.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>567, \"name\"=>\"Resensitization to Crizotinib by the Lorlatinib ALK Resistance Mutation L1198F.\", \"citation\"=>\"Shaw et al., 2016, N. Engl. J. Med.\", \"pubmed_id\"=>\"26698910\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26698910\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4773904\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>1, \"day\"=>7}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>352, \"phenotypes\"=>[]}]}, {\"id\"=>419, \"entrez_name\"=>\"NTRK1\", \"entrez_id\"=>4914, \"name\"=>\"NTRK1 FUSIONS\", \"description\"=>\"\", \"gene_id\"=>3983, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>156844175, \"stop\"=>156851434, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000524377.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3983/summary/variants/419/summary#variant\", \"evidence_items\"=>[{\"id\"=>2948, \"name\"=>\"EID2948\", \"description\"=>\"in two phase I studies in patients with advanced or metastatic solid tumors, including patients with active central nervous system (CNS) disease. Here, we summarize the overall safety and report the antitumor activity of entrectinib in a cohort of patients with tumors harboring NTRK1/2/3, ROS1, or ALK gene fusions\\nResponses were observed in non-small cell lung cancer, colorectal cancer, mammary analogue secretory carcinoma, melanoma, and renal cell carcinoma, as early as 4 weeks after starting treatment and lasting as long as >2 years. Notably, a complete CNS response was achieved in a patient with SQSTM1-NTRK1-rearranged lung cancer.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>426, \"name\"=>\"Entrectinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1689, \"name\"=>\"Safety and Antitumor Activity of the Multitargeted Pan-TRK, ROS1, and ALK Inhibitor Entrectinib: Combined Results from Two Phase I Trials (ALKA-372-001 and STARTRK-1).\", \"citation\"=>\"Drilon et al., 2017, Cancer Discov\", \"pubmed_id\"=>\"28183697\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28183697\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5380583\", \"publication_date\"=>{\"year\"=>2017, \"month\"=>4}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>419, \"phenotypes\"=>[]}, {\"id\"=>2956, \"name\"=>\"EID2956\", \"description\"=>\"LOXO-101 is an orally administered inhibitor of the TRK kinase and is highly selective only for the TRK family of receptors. The tumor of a 41-year-old woman with soft-tissue sarcoma metastatic to the lung was found to harbor an LMNA-NTRK1 gene fusion encoding a functional LMNA-TRKA fusion oncoprotein.  This patient's tumors underwent rapid and substantial tumor regression, with an accompanying improvement in pulmonary dyspnea, oxygen saturation, and plasma tumor markers.\", \"disease\"=>{\"id\"=>105, \"name\"=>\"Sarcoma\", \"display_name\"=>\"Sarcoma\", \"doid\"=>\"1115\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1115\"}, \"drugs\"=>[{\"id\"=>486, \"name\"=>\"Larotrectinib (LOXO-101)\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>nil, \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1691, \"name\"=>\"An Oncogenic NTRK Fusion in a Patient with Soft-Tissue Sarcoma with Response to the Tropomyosin-Related Kinase Inhibitor LOXO-101.\", \"citation\"=>\"Doebele et al., 2015, Cancer Discov\", \"pubmed_id\"=>\"26216294\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26216294\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4635026\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>10}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>419, \"phenotypes\"=>[]}, {\"id\"=>2953, \"name\"=>\"EID2953\", \"description\"=>\"Results from a phase I study show that the TRK inhibitor LOXO-101 is well tolerated and effective, with patients whose tumors bear NTRK fusions responding well and durably to this targeted therapy.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>486, \"name\"=>\"Larotrectinib (LOXO-101)\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1690, \"name\"=>\"TRK Inhibitor Shows Early Promise.\", \"citation\"=>\"2016, Cancer Discov\", \"pubmed_id\"=>\"26603524\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26603524\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>1}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>419, \"phenotypes\"=>[]}, {\"id\"=>2947, \"name\"=>\"EID2947\", \"description\"=>\"1378 cases of NSCLC were screened. One patient (frequency 0.1%) with NTRK1 gene rearrangement treated with entrectinib in a phase 1 study. The patient had resolution of prior dyspnea and pain within 3 weeks. CT scans showed a partial response and complete resolution of brain metastases. The patient has continued treatment for over 6 months with ongoing PR.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>426, \"name\"=>\"Entrectinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1288, \"name\"=>\"Durable Clinical Response to Entrectinib in NTRK1-Rearranged Non-Small Cell Lung Cancer.\", \"citation\"=>\"Farago et al., 2015, J Thorac Oncol\", \"pubmed_id\"=>\"26565381\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26565381\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4643748\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>12}, \"journal\"=>\"J Thorac Oncol\", \"full_journal_title\"=>\"Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>419, \"phenotypes\"=>[]}, {\"id\"=>1019, \"name\"=>\"EID1019\", \"description\"=>\"Two patients were found to harbor gene fusions maintaining the kinase domain of NTRK1 (3' partner), specifically: MPRIP-NTRK1 and CD74-NTRK1.  Both of these fusions lead to constitutive kinase activity and were found to be oncogenic. Treatment of cells expressing NTRK1 fusions with inhibitors of TRKA kinase activity (ARRY-470, CEP-701 and, to a lesser extent, crizotinib) inhibited autophosphorylation of TRKA and cell growth.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}, {\"id\"=>259, \"name\"=>\"ARRY-470\", \"pubchem_id\"=>nil}, {\"id\"=>260, \"name\"=>\"Lestaurtinib (CEP-701)\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>706, \"name\"=>\"Oncogenic and drug-sensitive NTRK1 rearrangements in lung cancer.\", \"citation\"=>\"Vaishnavi et al., 2013, Nat. Med.\", \"pubmed_id\"=>\"24162815\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24162815\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3823836\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>11}, \"journal\"=>\"Nat. Med.\", \"full_journal_title\"=>\"Nature medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>419, \"phenotypes\"=>[]}, {\"id\"=>978, \"name\"=>\"EID978\", \"description\"=>\"NTRK1 fusions (one LMNA-NTRK1 and two TPM3-NTRK1) were identified in 2% of Korean colorectal cancer patients (147 patients). TrkA (NTRK1) protein expression was associated with presence of NTRK1 gene fusion. These fusions were mutually exclusive with other oncogenic drivers and showed in-vitro sensitivity to TrkA Inhibitor ARRY470 (one cell line with fusion (KM12), one control cell line).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>259, \"name\"=>\"ARRY-470\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>677, \"name\"=>\"NTRK1 fusions for the therapeutic intervention of Korean patients with colon cancer.\", \"citation\"=>\"Park et al., 2016, Oncotarget\", \"pubmed_id\"=>\"26716414\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26716414\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4885001\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>2, \"day\"=>16}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>419, \"phenotypes\"=>[]}]}, {\"id\"=>431, \"entrez_name\"=>\"CBFB\", \"entrez_id\"=>865, \"name\"=>\"CBFB-MYH11\", \"description\"=>\"CBFB-MYH11 is the result of inv(16)(p13q22) or t(16;16)(p13;q22) and is sufficient for the diagnosis of acute myeloid leukemia (in the correct setting) according to WHO criteria, even when blast counts are <20% (Vardiman et al, Blood 2008). This fusion is typically associated with favorable prognosis relative to other AML subtypes (including cytogenetically normal); however, the impact of additional mutations in the context of this fusion is incompletely understood. AMLs with this fusion are often grouped with other rearrangements affecting genes encoding core binding factor subunits, referred to as CBF AMLs.\", \"gene_id\"=>776, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"16\", \"start\"=>67063148, \"stop\"=>67116211, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000412916.2\", \"chromosome2\"=>\"16\", \"start2\"=>15797029, \"stop2\"=>15820911, \"representative_transcript2\"=>\"ENST00000396324.3\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/776/summary/variants/431/summary#variant\", \"evidence_items\"=>[{\"id\"=>1026, \"name\"=>\"EID1026\", \"description\"=>\"The CBFB-MYH11 fusion has been shown to be associated with favorable prognosis in patients with acute myeloid leukemia.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"rejected\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>712, \"name\"=>\"Acute myeloid leukemia: 2012 update on diagnosis, risk stratification, and management.\", \"citation\"=>\"Estey, 2012, Am. J. Hematol.\", \"pubmed_id\"=>\"22180162\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22180162\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>1}, \"journal\"=>\"Am. J. Hematol.\", \"full_journal_title\"=>\"American journal of hematology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>431, \"phenotypes\"=>[]}, {\"id\"=>1027, \"name\"=>\"EID1027\", \"description\"=>\"The CBFB-MYH11 fusion has been shown to be associated with favorable prognosis in acute myeloid leukemia.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>712, \"name\"=>\"Acute myeloid leukemia: 2012 update on diagnosis, risk stratification, and management.\", \"citation\"=>\"Estey, 2012, Am. J. Hematol.\", \"pubmed_id\"=>\"22180162\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22180162\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>1}, \"journal\"=>\"Am. J. Hematol.\", \"full_journal_title\"=>\"American journal of hematology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>431, \"phenotypes\"=>[]}]}, {\"id\"=>450, \"entrez_name\"=>\"SSX4\", \"entrez_id\"=>6759, \"name\"=>\"SS18-SSX4\", \"description\"=>\"\", \"gene_id\"=>5503, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"18\", \"start\"=>23612363, \"stop\"=>23670589, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000415083.2\", \"chromosome2\"=>\"X\", \"start2\"=>48261524, \"stop2\"=>48265521, \"representative_transcript2\"=>\"ENST00000376884.2\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5503/summary/variants/450/summary#variant\", \"evidence_items\"=>[{\"id\"=>1074, \"name\"=>\"EID1074\", \"description\"=>\"Three synovial sarcomas were assayed for SYT-SSX fusion messenger RNA by nested RT-PCR. In a single case an SYT-SSX4 (SS18-SSX4) fusion was detected. The majority of synovial sarcomas are characterized by SS18-SSX1 or SS18-SSX2, but other rare events such as this SS18-SSX4 may explain some or all of the synovial sarcomas that are negative for the most common fusions.\", \"disease\"=>{\"id\"=>1310, \"name\"=>\"Synovial Sarcoma\", \"display_name\"=>\"Synovial Sarcoma\", \"doid\"=>\"5485\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5485\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>743, \"name\"=>\"A novel fusion gene, SYT-SSX4, in synovial sarcoma.\", \"citation\"=>\"Skytting et al., 1999, J. Natl. Cancer Inst.\", \"pubmed_id\"=>\"10359553\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/10359553\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1999, \"month\"=>6, \"day\"=>2}, \"journal\"=>\"J. Natl. Cancer Inst.\", \"full_journal_title\"=>\"Journal of the National Cancer Institute\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>450, \"phenotypes\"=>[]}]}, {\"id\"=>465, \"entrez_name\"=>\"ROS1\", \"entrez_id\"=>6098, \"name\"=>\"CD74-ROS1 G2032R\", \"description\"=>\"The G2032R missense mutation has been observed in patients with crizotinib-resistant non-small cell lung cancer harboring CD74-ROS1 rearrangement. Independent in vitro studies have confirmed that this mutation confers marked resistance to crizotinib cells over CD74-ROS1 alone. Multiple groups have indicated CD74-ROS1 G2032R expressing cell lines are sensitive to both cabozantinib and foretinib individually administered in vitro. One group additionally showed this mutation conferred relative resistance to alectinib, brigatinib, ceritinib, and AZD3463 in vitro.\", \"gene_id\"=>4941, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>31.5, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>117638347, \"stop\"=>117638347, \"reference_bases\"=>\"C\", \"variant_bases\"=>\"T\", \"representative_transcript\"=>\"ENST00000368508.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4941/summary/variants/465/summary#variant\", \"evidence_items\"=>[{\"id\"=>1255, \"name\"=>\"EID1255\", \"description\"=>\"Ba/F3 cells expressing CD74–ROS1 G2032R showed resistance to crizotinib, compared with those expressing wild-type CD74–ROS1 (~128 fold higher IC50); however, showed relative sensitivity to foretinib (only ~6 fold higher IC50 than WT).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>150, \"name\"=>\"Foretinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>868, \"name\"=>\"Molecular Changes Associated with Acquired Resistance to Crizotinib in ROS1-Rearranged Non-Small Cell Lung Cancer.\", \"citation\"=>\"Song et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"25688157\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25688157\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>5, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>465, \"phenotypes\"=>[]}, {\"id\"=>1250, \"name\"=>\"EID1250\", \"description\"=>\"Preclinical study analyzing the efficacy of dual ROS1/ALK and more specific ROS1 inhibitors on cell lines (Ba/F3) expressing CD74-ROS1 with and without the G2032R mutation. Selective ROS1 inhibitors retained activity against the G2032R mutation (cabozatinib, foretinib), whereas dual ALK/ROS inhibitors (crizotinib, brigatinib, ceritinib, AZD3463) or ALK-selective inhibitors (alectinib) were ineffective.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}, {\"id\"=>159, \"name\"=>\"Ceritinib\", \"pubchem_id\"=>nil}, {\"id\"=>318, \"name\"=>\"Brigatinib\", \"pubchem_id\"=>nil}, {\"id\"=>319, \"name\"=>\"AZD3463\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>866, \"name\"=>\"Structural insight into selectivity and resistance profiles of ROS1 tyrosine kinase inhibitors.\", \"citation\"=>\"Davare et al., 2015, Proc. Natl. Acad. Sci. U.S.A.\", \"pubmed_id\"=>\"26372962\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26372962\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4593101\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>9, \"day\"=>29}, \"journal\"=>\"Proc. Natl. Acad. Sci. U.S.A.\", \"full_journal_title\"=>\"Proceedings of the National Academy of Sciences of the United States of America\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>465, \"phenotypes\"=>[]}, {\"id\"=>1249, \"name\"=>\"EID1249\", \"description\"=>\"Preclinical study analyzing the efficacy of dual ROS1/ALK and more specific ROS1 inhibitors on cell lines (Ba/F3) expressing CD74-ROS1 with and without the G2032R mutation. Selective ROS1 inhibitors retained activity against the G2032R mutation (cabozatinib, foretinib), whereas dual ALK/ROS inhibitors (crizotinib, brigatinib, ceritinib, AZD3463) or ALK-selective inhibitors (alectinib) were ineffective.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>144, \"name\"=>\"Cabozantinib\", \"pubchem_id\"=>nil}, {\"id\"=>150, \"name\"=>\"Foretinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>866, \"name\"=>\"Structural insight into selectivity and resistance profiles of ROS1 tyrosine kinase inhibitors.\", \"citation\"=>\"Davare et al., 2015, Proc. Natl. Acad. Sci. U.S.A.\", \"pubmed_id\"=>\"26372962\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26372962\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4593101\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>9, \"day\"=>29}, \"journal\"=>\"Proc. Natl. Acad. Sci. U.S.A.\", \"full_journal_title\"=>\"Proceedings of the National Academy of Sciences of the United States of America\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>465, \"phenotypes\"=>[]}, {\"id\"=>1254, \"name\"=>\"EID1254\", \"description\"=>\"ROS1 kinase domain mutations were assessed in two patients with stage IIIa/IV pulmonary adenocarcinoma and a cell line. The ROS1 G2032R mutation was identified in a crizotinib-resistant tumor from one of the patients harboring the CD74–ROS1 fusion. Expression of CD74–ROS1 G2032R in Ba/F3 cells conferred resistance to crizotinib combared to CD74–ROS1 (IC50=353.62+/-20.15 vs. 2.76+/-0.82).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>868, \"name\"=>\"Molecular Changes Associated with Acquired Resistance to Crizotinib in ROS1-Rearranged Non-Small Cell Lung Cancer.\", \"citation\"=>\"Song et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"25688157\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25688157\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>5, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>465, \"phenotypes\"=>[]}, {\"id\"=>1251, \"name\"=>\"EID1251\", \"description\"=>\"Preclinical study of PF-06463922 in NSCLC in-vitro, in-vivo and in enzyme assays. PF-06463922 showed inhibition of kinase activity, cell viability and tumor growth in wild-type, G2032R and L2026M mutations in ROS1.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>241, \"name\"=>\"Lorlatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>867, \"name\"=>\"PF-06463922 is a potent and selective next-generation ROS1/ALK inhibitor capable of blocking crizotinib-resistant ROS1 mutations.\", \"citation\"=>\"Zou et al., 2015, Proc. Natl. Acad. Sci. U.S.A.\", \"pubmed_id\"=>\"25733882\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25733882\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4371934\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>3, \"day\"=>17}, \"journal\"=>\"Proc. Natl. Acad. Sci. U.S.A.\", \"full_journal_title\"=>\"Proceedings of the National Academy of Sciences of the United States of America\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>465, \"phenotypes\"=>[]}, {\"id\"=>1100, \"name\"=>\"EID1100\", \"description\"=>\"Resistance to crizotinib developed in a patient with metastatic lung adenocarcinoma harboring a CD74-ROS1 rearrangement who had initially shown a dramatic response to treatment. An acquired G2032R mutation in the ROS1 kinase domain was observed in the resistant tumor, and was found to confer resistance to ROS1 kinase inhibition through steric interference with drug binding.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>381, \"name\"=>\"Acquired resistance to crizotinib from a mutation in CD74-ROS1.\", \"citation\"=>\"Awad et al., 2013, N. Engl. J. Med.\", \"pubmed_id\"=>\"23724914\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23724914\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3878821\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>6, \"day\"=>20}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>465, \"phenotypes\"=>[]}, {\"id\"=>1101, \"name\"=>\"EID1101\", \"description\"=>\"Cabozantinib was assessed in the CD74-ROS1-mutant Ba/F3 cells and crizotinib-resistant patient-derived cancer cells (MGH047) harboring G2032R-mutated CD74-ROS1. It effectively inhibited the survival of CD74-ROS1 wild-type (WT) and resistant mutants harboring Ba/F3 and MGH047 cells. Furthermore, cabozantinib could overcome all the resistance by all newly identified secondary mutations.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>144, \"name\"=>\"Cabozantinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>759, \"name\"=>\"Cabozantinib overcomes crizotinib resistance in ROS1 fusion-positive cancer.\", \"citation\"=>\"Katayama et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"25351743\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25351743\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4286456\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>1, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>465, \"phenotypes\"=>[]}]}, {\"id\"=>499, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"ALK FUSIONS\", \"description\"=>\"Following the initial discovery and characterization of NPM-ALK fusion mutation in anaplastic large cell lymphoma (ALCL), EML4-ALK fusions were found to be driver mutations in a subset of lung adenocarcinomas. ALK fusions bring the C-terminal ALK kinase domain together with the N terminal region of the fusion partner, and have since been found in a range of cancers including inflammatory myofibroblastic tumor, renal cell carcinoma, diffuse large B cell lymphoma and colorectal cancer. \\n\\nIn a Phase I trial for crizotinib, a 92% response rate was seen in the subset of patients with a specific EML4-ALK variant, which was higher than the 57% rate in the overall ALK-rearranged population, but the subpopulation numbers were insufficient to correlate variant type with patient response. \\n\\nDue to restricted wild-type ALK expression, and also mild phenotypes seen in ALK knockout mice, ALK fusions are desirable targets.  ALK inhibitor TAE684 was used in initial characterizations of ALK fusion cancer drivers, and crizotinib – a small molecule ATP-competitor RTK inhibitor which is highly selective for ALK and MET - has been approved as first line therapy in ALK-rearranged NSCLC.   \\n\\nALK-rearrangement defines an NSCLC subtype that makes up 3-5% of NSCLC cases, and is characterized by higher proportions of younger patients and never smokers. EML4-ALK variant 1 is the most common ALK fusion in NSCLC and was the first to be discovered and demonstrate cancer driver properties in vitro and in mouse models. A Phase I study of crizotinib in ALK-rearranged NSCLC demonstrated an increase in overall survival when compared to standard chemotherapy. These and other initial results prompted accelerated approval for crizotinib in NSCLC.  Continued clinical work demonstrated benefit with crizotinib over chemotherapy in multi-armed studies, and as first line therapy. Crizotinib was also found to remain beneficial to NSCLC patients who had progressed on crizotinib. Additionally, crizotinib was shown to increase intracranial disease control in brain metastasis occurring from ALK-rearranged NSCLC.\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>440.5, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>29415640, \"stop\"=>29446394, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000389048.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/499/summary#variant\", \"evidence_items\"=>[{\"id\"=>4858, \"name\"=>\"EID4858\", \"description\"=>\"In a randomized, open-label phase 3 trial 303 patients with ALK-positive, previously untreated NSCLC (including asympt. CNS disease) were randomized to reiceive either Alectinib or Crizotinib. During a median follow-up of 17.6 months (crizotinib) and 18.6 months (alectinib), an event of disease progression/death occurred in 41% (62/152  pt) in the alectinib group and 68% (102/151 patients) in the crizotinib group. 12-month event-free survival rate 68.4% with alectinib vs. 48.7% with crizotinib. A total of 18 patients (12%) in the alectinib group had an event of CNS progression, compared to 68 patients (45%) in the crizotinib group. Response rates were 82.9% (Alectinib) and 75.5% (crizotinib).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}, {\"id\"=>526, \"name\"=>\"Alectinib\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2157, \"name\"=>\"Alectinib versus Crizotinib in Untreated ALK-Positive Non-Small-Cell Lung Cancer.\", \"citation\"=>\"Peters et al., 2017, N. Engl. J. Med.\", \"pubmed_id\"=>\"28586279\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28586279\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>6, \"day\"=>6}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>4835, \"name\"=>\"EID4835\", \"description\"=>\"In a phase II randomized trial we evaluated two regimens of Brigatinib in Crizotinib-refractory ALK-pos. NSCLC. They were randomly assigned (1:1) to oral Brigatinib 90 mg once daily (arm A) or 180 mg once daily with a 7-day lead-in at 90 mg. Of 222 patients enrolled (arm A: n = 112, arm B: n = 110), 154 (69%) had baseline brain metastases and 164 of 222 (74%) had received prior chemotherapy. investigator-assessed confirmed ORR was 45 (arm A) and 54% (B). Investigator-assessed median PFS was 9.2 months (A) and 12.9 months (B). Independent review committee-assessed intracranial ORR in patients with measurable brain metastases at baseline was 42% (11/26, A) and 67% (12/18, B). Brigatinib yielded substantial whole-body and intracranial responses as well as robust PFS; 180 mg (with lead-in) showed consistently better efficacy than 90 mg, with acceptable safety.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>318, \"name\"=>\"Brigatinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2141, \"name\"=>\"Brigatinib in Patients With Crizotinib-Refractory Anaplastic Lymphoma Kinase-Positive Non-Small-Cell Lung Cancer: A Randomized, Multicenter Phase II Trial.\", \"citation\"=>\"Kim et al., 2017, J. Clin. Oncol.\", \"pubmed_id\"=>\"28475456\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28475456\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>5, \"day\"=>5}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>2952, \"name\"=>\"EID2952\", \"description\"=>\"Two phase I studies in patients with advanced or metastatic solid tumors, including patients with active central nervous system (CNS) disease. Here, we summarize the overall safety and report the antitumor activity of entrectinib in a cohort of patients with tumors harboring NTRK1/2/3, ROS1, or ALK gene fusions Responses were observed in non-small cell lung cancer, colorectal cancer, mammary analogue secretory carcinoma, melanoma, and renal cell carcinoma, as early as 4 weeks after starting treatment and lasting as long as >2 years. Notably, a complete CNS response was achieved in a patient with SQSTM1-NTRK1-rearranged lung cancer.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>426, \"name\"=>\"Entrectinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1689, \"name\"=>\"Safety and Antitumor Activity of the Multitargeted Pan-TRK, ROS1, and ALK Inhibitor Entrectinib: Combined Results from Two Phase I Trials (ALKA-372-001 and STARTRK-1).\", \"citation\"=>\"Drilon et al., 2017, Cancer Discov\", \"pubmed_id\"=>\"28183697\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28183697\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5380583\", \"publication_date\"=>{\"year\"=>2017, \"month\"=>4}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1264, \"name\"=>\"EID1264\", \"description\"=>\"A 27 year old female patient with advanced-stage IV, CD20 negative diffuse large B cell lymphoma (DLBCL). ALK rearrangement was determined by FISH, but the ALK fusion partner was not determined due to limited tissue. The patient underwent treatment with CHOP chemotherapy regimen and radiation but did not respond. The patient progressed following salvage chemotherapy and autologous stem cell transplant. Crizotinib was then administered at 250 mg twice daily, and LDH levels rapidly dropped along with disappearance of palpable lymphadenopathy, but progression ensued after 21 days and the patient died soon after. The authors attribute the short response to crizotinib to the possibility of this tumor having an ALK fusion variant less responsive to ALK inhibition and to a high Ki-67 index in the patient, and deem that despite the brief response, crizotinib shows promise in ALK rearranged DLBCL.\", \"disease\"=>{\"id\"=>45, \"name\"=>\"Diffuse Large B-cell Lymphoma\", \"display_name\"=>\"Diffuse Large B-cell Lymphoma\", \"doid\"=>\"0050745\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050745\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>874, \"name\"=>\"Crizotinib in refractory ALK-positive diffuse large B-cell lymphoma: a case report with a short-term response.\", \"citation\"=>\"Wass et al., 2014, Eur. J. Haematol.\", \"pubmed_id\"=>\"24330038\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24330038\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>3}, \"journal\"=>\"Eur. J. Haematol.\", \"full_journal_title\"=>\"European journal of haematology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1202, \"name\"=>\"EID1202\", \"description\"=>\"In this study, heavily pretreated advanced NSCLC patients with various molecular tumor makeup were treated with Hsp90 inhibitor IPI-504 to assess overall response rate. The study did not reach the targeted endpoint of 20% ORR. However, of 3/12 patients found to be ALK-rearranged via break apart FISH assay, 2 showed a partial response and the third stable disease, which was durable for 7.2 months. The authors note that the break apart FISH assay used to determine ALK-rearrangement status does not determine the identity of the ALK fusion partner and the promising results shown here may be dependent on the nature of the ALK fusion partner.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>303, \"name\"=>\"IPI-504\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>831, \"name\"=>\"Activity of IPI-504, a novel heat-shock protein 90 inhibitor, in patients with molecularly defined non-small-cell lung cancer.\", \"citation\"=>\"Sequist et al., 2010, J. Clin. Oncol.\", \"pubmed_id\"=>\"20940188\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20940188\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4676802\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>11, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1273, \"name\"=>\"EID1273\", \"description\"=>\"In this case study of a critically ill 29 year old male nonsmoker with advanced metastatic ALK-positive NSCLC, alectinib (CH5424802), was administered as first line treatment. Where most clinical trials only admit patients with Eastern Cooperative Oncology Performance status (PS) of 0 or 1, this patient had PS 4 due to respiratory failure, multiple bone metastases and complications from disseminated intravascular coagulation (DIC). Administration of alectinib at 300 mg twice daily was followed by marked improvement seen in tests associated with DIC, pulmonary metastasis and change in PS from 4 to 0. Further improvements after a couple of months suggested a rapid complete response. Alectinib was well tolerated in this first line treatment of a critically ill ALK-positive NSCLC patient.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>17, \"name\"=>\"CH5424802\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>879, \"name\"=>\"Rapid and dramatic response to alectinib in an anaplastic lymphoma kinase rearranged non-small-cell lung cancer patient who is critically ill.\", \"citation\"=>\"Yoshida et al., 2016, Anticancer Drugs\", \"pubmed_id\"=>\"26938871\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26938871\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>7}, \"journal\"=>\"Anticancer Drugs\", \"full_journal_title\"=>\"Anti-cancer drugs\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1691, \"name\"=>\"EID1691\", \"description\"=>\"In an evaluation of a patient with crizotinib-sensitive lung adenocarcinoma, an EML4-ALK-E13;A20 rearrangement was identified using CGP.  The patient attained a reduction in the target lesions by 14.2%, however, the patient relapsed 17 months later.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1141, \"name\"=>\"Detection of Crizotinib-Sensitive Lung Adenocarcinomas With MET, ALK, and ROS1 Genomic Alterations via Comprehensive Genomic Profiling.\", \"citation\"=>\"Le et al., 2015, Clin Lung Cancer\", \"pubmed_id\"=>\"25922291\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25922291\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4418215\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>9}, \"journal\"=>\"Clin Lung Cancer\", \"full_journal_title\"=>\"Clinical lung cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1577, \"name\"=>\"EID1577\", \"description\"=>\"A phase-3 clinical trial evaluated 347 patients with ALK-positive lung cancer.  The study compared crizotinib treatment (n=173) with chemotherapy treatment of either pemetrexed or docetaxel (n=174).  In the crizotinib arm, the response rate was significantly higher (65% vs 20%; P<0.001), the progression-free survival was increased (7.7 months vs. 3.0 months; HR=0.49, 95%CI:14 months-26 months; p<0.001), however, there was no significant improvement in overall survival (HR=1.02; 95%CI:0.68-1.54; P=0.54).  Of the patients on crizotinib treatment, global quality of life was improved relative to baseline when compared to chemotherapy patients (+5 mean change, -5 mean change; P<0.001).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1041, \"name\"=>\"Crizotinib versus Chemotherapy in Advanced ALK-Positive Lung Cancer.\", \"citation\"=>\"2015, N. Engl. J. Med.\", \"pubmed_id\"=>\"26466010\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26466010\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>10, \"day\"=>15}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1200, \"name\"=>\"EID1200\", \"description\"=>\"This retrospective study of patients from the PROFILE 1001 and 1005 studies of crizotinib in advanced ALK-rearranged NSCLC assessed the potential benefit of crizotinib beyond progressive disease (CBPD), defined as continuation of crizotinib for 3 or more weeks after disease progression on crizotinib. 194 patients who had developed PD by data cutoff were studied, where 120 received crizotinib for a median duration of 19.4 weeks and 74 discontinued crizotinib after PD. No gross imbalances in clinicopathologic features of the two groups were evident.  Significantly longer overall survival was observed in the crizotinib continuing group from time of PD (16.4 vs. 3.9 months; hazard ratio 0.27, 95% CI: 0.17–0.42; P < 0.0001), and from the time of initial crizotinib (29.6 vs. 10.8 months; HR 0.30, 95% CI: 0.19–0.46; P < 0.0001).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>829, \"name\"=>\"Clinical benefit of continuing ALK inhibition with crizotinib beyond initial disease progression in patients with advanced ALK-positive NSCLC.\", \"citation\"=>\"Ou et al., 2014, Ann. Oncol.\", \"pubmed_id\"=>\"24478318\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24478318\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>2}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology / ESMO\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1783, \"name\"=>\"EID1783\", \"description\"=>\"Fluorescence in situ hybridization with a probe flanking the ALK gene at 2p23 and immunohistochemistry revealed ALK expression, suggesting ALK fusions in three cases of IMT.\", \"disease\"=>{\"id\"=>12, \"name\"=>\"Inflammatory Myofibroblastic Tumor\", \"display_name\"=>\"Inflammatory Myofibroblastic Tumor\", \"doid\"=>\"0050905\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050905\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1245, \"name\"=>\"Recurrent involvement of 2p23 in inflammatory myofibroblastic tumors.\", \"citation\"=>\"Griffin et al., 1999, Cancer Res.\", \"pubmed_id\"=>\"10383129\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/10383129\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1999, \"month\"=>6, \"day\"=>15}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1282, \"name\"=>\"EID1282\", \"description\"=>\"In this Phase II trial of 87 patients (NCT01871805), alectinib was assayed for safety and efficacy in ALK positive NSCLC that had progressed on crizotinib. Of 69 patients with measurable disease, 35 achieved objective response, with median duration of response 13.5 months. Of 16 patients with measurable CNS disease, four achieved CNS complete response and 8 showed CNS partial response. median duration of CNS response was 11.1 months. 2 deaths were observed, and 1 was judged related to study treatment. In early 2016, alectinib was approved by the FDA partly based on this study for treatment of crizotinib-progressed ALK positive NSCLC\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>321, \"name\"=>\"Alectinib (CH5424802)\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>886, \"name\"=>\"Alectinib in ALK-positive, crizotinib-resistant, non-small-cell lung cancer: a single-group, multicentre, phase 2 trial.\", \"citation\"=>\"Shaw et al., 2016, Lancet Oncol.\", \"pubmed_id\"=>\"26708155\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26708155\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4752892\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>2}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1279, \"name\"=>\"EID1279\", \"description\"=>\"In this Phase I trial (NCT01588028) in 47 patients, alectinib was assessed for its effectiveness in treating locally advanced or metastatic ALK-positive NSCLC that had progressed or been resistant to crizotinib treatment. ALK rearrangement was assessed via approved break apart FISH kit, and patients with asymptomatic CNS metastases were also included. Patients were divided into different cohorts receiving different doses of alectinib. In the 600 mg cohort (N=13), there were 4 confirmed partial responses, 3 unconfirmed partial responses and 3 instances of stable disease. Also, five of nine patients throughout the cohorts with measurable baseline CNS lesions showed partial response to alectinib. Cerebrospinal fluid levels of alectinib increased linearly with blood alectinib levels, indicating it effectively crossed the blood-brain barrier.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>321, \"name\"=>\"Alectinib (CH5424802)\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>884, \"name\"=>\"Safety and activity of alectinib against systemic disease and brain metastases in patients with crizotinib-resistant ALK-rearranged non-small-cell lung cancer (AF-002JG): results from the dose-finding portion of a phase 1/2 study.\", \"citation\"=>\"Gadgeel et al., 2014, Lancet Oncol.\", \"pubmed_id\"=>\"25153538\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25153538\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>9}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1334, \"name\"=>\"EID1334\", \"description\"=>\"26 colorectal adenocarcinoma samples were run on Affymetrix human Exon 1.0 arrays and a computational method that assessed differential 5' and 3' gene expression for each gene was used to find potential rearrangements.  One sample showed potential rearrangement and RT-PCR demonstrated an EML4-ALK fusion where EML4 exon 21 was fused to ALK exon 13. This EML4-ALK variant is different from those commonly found in NSCLC. Genomic PCR verified the existence of the fusion breakpoint in intronic space, indicating that some colorectal adenocarcinoma may have targetable ALK-fusion driver mutations.\", \"disease\"=>{\"id\"=>57, \"name\"=>\"Colorectal Adenocarcinoma\", \"display_name\"=>\"Colorectal Adenocarcinoma\", \"doid\"=>\"0050861\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050861\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"E\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>920, \"name\"=>\"Exon array profiling detects EML4-ALK fusion in breast, colorectal, and non-small cell lung cancers.\", \"citation\"=>\"Lin et al., 2009, Mol. Cancer Res.\", \"pubmed_id\"=>\"19737969\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19737969\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>9}, \"journal\"=>\"Mol. Cancer Res.\", \"full_journal_title\"=>\"Molecular cancer research : MCR\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1335, \"name\"=>\"EID1335\", \"description\"=>\"84 breast cancer samples acquired from commercial sources were assayed on Affymetrix human exon 1.0 arrays. A computational method was used to find potential gene fusions which assessed differential exon expression between the 3' and 5' ends of each gene. Two potential fusions were found and assessed using RNA ligase mediated rapid amplifiation of cDNA ends and RT-PCR. The fusions were EML4-ALK variants 1 and 2. Genomic PCR identified fusion breakpoints, and these differed from breakpoints for these EML4-ALK variants reported in NSCLC.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"E\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>920, \"name\"=>\"Exon array profiling detects EML4-ALK fusion in breast, colorectal, and non-small cell lung cancers.\", \"citation\"=>\"Lin et al., 2009, Mol. Cancer Res.\", \"pubmed_id\"=>\"19737969\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19737969\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>9}, \"journal\"=>\"Mol. Cancer Res.\", \"full_journal_title\"=>\"Molecular cancer research : MCR\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1419, \"name\"=>\"EID1419\", \"description\"=>\"In a retrospective study comparing survival outcomes in a subgroup of 82 ALK-positive crizotinib-treated patients enrolled in a phase 1 clinical trial (NCT00585195), crizotinib therapy was associated with improved survival. For the crizotinib-treated, ALK-positive patient group, 1 year survival was 74% and 2 year overall survival was 54%, compared with 72% and 36% for the control group (ALK-positive crizotinib naive patients).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>961, \"name\"=>\"Effect of crizotinib on overall survival in patients with advanced non-small-cell lung cancer harbouring ALK gene rearrangement: a retrospective analysis.\", \"citation\"=>\"Shaw et al., 2011, Lancet Oncol.\", \"pubmed_id\"=>\"21933749\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21933749\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3328296\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>10}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1240, \"name\"=>\"EID1240\", \"description\"=>\"Crizotinib at 250mg daily was given to two patients with relapsed ALK-rearranged anaplastic large cell lymphoma (ALCL). Patient one, 26 yrs old, had been treated with the CHOP-15 chemotherapy regimen, then salvage chemotherapy to prepare for autologous bone marrow transplant but ultimately relapsed. After initiation of crizotinib, PET, CT and bone marrow aspirates showed complete remission of previous lesions at 28 days. At time of publication the patient continued to show persistent response at 6 months. A 20 year old patient relapsed after one month of CHOP, and was treated with further chemotherapy and autologous bone marrow transplantation resulting in partial response of 1 month. Crizotinib was initialized, and after 12 days PET scan showed complete regression of all lesions. On day 60 bone marrow aspirates were ALK-negative (pretreatment evaluation showed 8% ALK-positive). The authors conclude these two cases indicate sensitivity to crizotinib of relapsed ALK-rearranged ALCL.\", \"disease\"=>{\"id\"=>348, \"name\"=>\"Anaplastic Large Cell Lymphoma\", \"display_name\"=>\"Anaplastic Large Cell Lymphoma\", \"doid\"=>\"0050744\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050744\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>858, \"name\"=>\"Crizotinib in anaplastic large-cell lymphoma.\", \"citation\"=>\"Gambacorti-Passerini et al., 2011, N. Engl. J. Med.\", \"pubmed_id\"=>\"21345110\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21345110\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>2, \"day\"=>24}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1272, \"name\"=>\"EID1272\", \"description\"=>\"In this initial report of a single-arm Phase I and II study (JapicCTI-101264) of alectinib (CH5424802), 46 patients with advanced or metastatic ALK-rearranged non small cell lung cancer (NSCLC) were assessed for objective response during phase II. ALK positivity was assessed via IHC and verified with FISH or RT-PCR. Patients had received one or more rounds of previous chemotherapy, but no prior targeted therapy towards ALK. From the 46 patients, 41 showed partial response, and 2 showed complete response. CNS metastases in three patients which had not received prior brain irradiation showed response to alectinib. Alectinib was generally well tolerated, and the authors conclude that alectinib compared favorably to crizotinib and could be an effective and safe option in ALK-rearranged NSCLC.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>321, \"name\"=>\"Alectinib (CH5424802)\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>878, \"name\"=>\"CH5424802 (RO5424802) for patients with ALK-rearranged advanced non-small-cell lung cancer (AF-001JP study): a single-arm, open-label, phase 1-2 study.\", \"citation\"=>\"Seto et al., 2013, Lancet Oncol.\", \"pubmed_id\"=>\"23639470\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23639470\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>6}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1323, \"name\"=>\"EID1323\", \"description\"=>\"Development of second generation ALK inhibitors has been underway in order to address resistance to first line treatments such as crizotinib in ALK-rearranged NSCLC.  Brigatinib is a potent dual inhibitor against ALK and EGFR.  In an update from the Phase I/II trial NCT01449461 of brigatinib in ALK+ NSCLC, patients were given 90 mg/day, 180 mg/day, or 90 followed by 180 mg/day. In crizotinib-naive patients, overall response rate (ORR) was 7/7 patients with progression free survival (PFS) of 56 weeks. In patients who failed on crizotinib treatment ORR was 69% (45/65 patients) with PFS of 47 weeks, and activity against CNS metastases was seen with ORR 37% in 38 patients with PFS 97 weeks indicating substantial brigatinib activity in crizotinib-progressed patients, as well as promise in CNS metastases.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>318, \"name\"=>\"Brigatinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"rejected\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>912, \"name\"=>\"Second- and third-generation ALK inhibitors for non-small cell lung cancer.\", \"citation\"=>\"Wu et al., 2016, J Hematol Oncol\", \"pubmed_id\"=>\"26951079\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26951079\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4782349\", \"publication_date\"=>{\"year\"=>2016}, \"journal\"=>\"J Hematol Oncol\", \"full_journal_title\"=>\"Journal of hematology & oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1242, \"name\"=>\"EID1242\", \"description\"=>\"Four adult ALK-rearranged anaplastic large cell lymphoma (ALCL) patients were resistant to chemotherapy after 3 or more lines of treatment.  The patients were administered 250 mg crizotinib twice daily. Normalization of symptoms was observed at a median time of 5 days, with an overall response rate of 100% (3 complete responses and one partial response)\", \"disease\"=>{\"id\"=>348, \"name\"=>\"Anaplastic Large Cell Lymphoma\", \"display_name\"=>\"Anaplastic Large Cell Lymphoma\", \"doid\"=>\"0050744\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050744\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"rejected\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>860, \"name\"=>\"Brentuximab vedotin and crizotinib in anaplastic large-cell lymphoma.\", \"citation\"=>\"Foyil et al., Cancer J\", \"pubmed_id\"=>\"23006951\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23006951\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{}, \"journal\"=>\"Cancer J\", \"full_journal_title\"=>\"Cancer journal (Sudbury, Mass.)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1243, \"name\"=>\"EID1243\", \"description\"=>\"As part of a phase 1 dose-escalation trial (NCT00939770), seven patients between ages 12 months and 22 years with relapsed or refractory inflammatory myofibroblastic tumors with ALK-rearrangement were given crizotinib. In six patients with measurable disease, responses included 3 with partial response and 3 with stable disease. One patient with evaluable disease had stable disease with treatment ongoing at the time of publication. Crizotinib was well tolerated in this group of pediatric patients.\", \"disease\"=>{\"id\"=>12, \"name\"=>\"Inflammatory Myofibroblastic Tumor\", \"display_name\"=>\"Inflammatory Myofibroblastic Tumor\", \"doid\"=>\"0050905\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050905\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>859, \"name\"=>\"Safety and activity of crizotinib for paediatric patients with refractory solid tumours or anaplastic large-cell lymphoma: a Children's Oncology Group phase 1 consortium study.\", \"citation\"=>\"Mossé et al., 2013, Lancet Oncol.\", \"pubmed_id\"=>\"23598171\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23598171\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3730818\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>5}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1201, \"name\"=>\"EID1201\", \"description\"=>\"This prospective Phase III study named Profile 1014 on NSCLC ALK-rearranged patients assessed intracranial disease control rate (IC-DCR) in the subset of patients entering the study with preexisting treated brain metastasis (tBM). IC-DCR was significantly improved in the tBM group given crizotinib (N=39) over chemotherapy (pemetrexed + cisplatin or carboplatin; N=40) at 12 weeks (85% vs. 45%, P<0.001), and at 24 weeks (56% vs. 25%, P=0.006). Progression free survival was also significantly improved with crizotinib over chemotherapy in the tBM subgroup (9.0 vs. 4.0 months; HR 0.40; P<0.001), where in the intent to treat population it was 10.9 vs. 7.0 months (HR 0.45; P<0.001), indicating that the difficult to treat BM population obtained benefit from crizotinib at both the brain and systemic level.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>830, \"name\"=>\"Intracranial Efficacy of Crizotinib Versus Chemotherapy in Patients With Advanced ALK-Positive Non-Small-Cell Lung Cancer: Results From PROFILE 1014.\", \"citation\"=>\"Solomon et al., 2016, J. Clin. Oncol.\", \"pubmed_id\"=>\"27022118\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27022118\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>8, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1199, \"name\"=>\"EID1199\", \"description\"=>\"This phase 3 trial named PROFILE 1014 (NCT01154140) compared crizotinib to established chemotherapy in first-line treatment for this disease. The study consisted of 343 patients with advanced or metastatic NSCLC that tested positive for ALK rearrangement via break-apart FISH assay, and had no prior systemic treatment. Patients were assigned in a 1:1 ratio to crizotinib (N=172) or pemetrexed with cisplatin or carboplatin (N=171). Progression free survival was significantly longer with crizotinib treatment than chemotherapy (10.9 vs. 7.0 months; hazard ratio for progression or death with crizotinib, 0.45; 95% CI, 0.35 to 0.60;P<0.001), and objective response rate was higher with crizotinib treatment than chemotherapy (75% vs. 45%, P<0.001). Improvement in quality of life was also reported.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>828, \"name\"=>\"First-line crizotinib versus chemotherapy in ALK-positive lung cancer.\", \"citation\"=>\"Solomon et al., 2014, N. Engl. J. Med.\", \"pubmed_id\"=>\"25470694\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25470694\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>12, \"day\"=>4}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1175, \"name\"=>\"EID1175\", \"description\"=>\"Updated results from ASCEND-1 trial assessing safety and activity of ceritinib in ALK rearranged locally advanced or metastatic NSCLC. 255 patients were enrolled and received at least one dose of ceritinib 750 mg/day, of whom 246 had ALK-rearranged NSCLC (Median follow-up: 11.1 months). Overall response rate was 72% [95% CI 61-82] of 83 ALK inhibitor-naive patients and 56% [49-64] of 163 ALK inhibitor-pretreated patients. Median duration of response was 17·0 months (95% CI 11.3-non-estimable [NE]) in ALK inhibitor-naive patients and 8.3 months (6.8-9.7) in ALK inhibitor-pretreated patients. Median progression-free survival was 18.4 months (95% CI 11.1-NE) in ALK inhibitor-naive patients and 6.9 months (5.6-8.7) in ALK inhibitor-pretreated patients. Brain metastases control was reported in 79% [95% CI 54-94] of 19 ALK inhibitor-naive patients and in 65% [54-76] of 75 ALK inhibitor-pretreated patients.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>159, \"name\"=>\"Ceritinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>812, \"name\"=>\"Activity and safety of ceritinib in patients with ALK-rearranged non-small-cell lung cancer (ASCEND-1): updated results from the multicentre, open-label, phase 1 trial.\", \"citation\"=>\"Kim et al., 2016, Lancet Oncol.\", \"pubmed_id\"=>\"26973324\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26973324\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>4}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1187, \"name\"=>\"EID1187\", \"description\"=>\"In the Phase I study PROFILE 1001 (NCT00585195), a recommended crizotinib dose of 250 mg twice daily for 28 day cycles was established. Among 1,500 advanced NSCLC patients who were screened for ALK-rearrangement using a break-apart FISH assay, 82 patients were eligible for crizotinib treatment.  Overall response rate was 57%, with 46 partial responses and one complete response.  Since crizotinib inhibits MET, 33 patients with available tissue were screened for MET amplification and were found negative, further indicating that patient response was due to ALK inhibition.  31 patients were tested via RT-PCR for known EML4-ALK fusions, and 13 were shown to have EML4-ALK variant 1, four cases had variant 3, and one instance of variants 2, 3b and 5 were seen.  9 were unidentified, suggesting other fusion partners in these instances.  On the basis of this and a Phase II study, accelerated approval for crizotinib in ALK-positive advanced NSCLC was granted.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>819, \"name\"=>\"Anaplastic lymphoma kinase inhibition in non-small-cell lung cancer.\", \"citation\"=>\"Kwak et al., 2010, N. Engl. J. Med.\", \"pubmed_id\"=>\"20979469\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20979469\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3014291\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>10, \"day\"=>28}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>1198, \"name\"=>\"EID1198\", \"description\"=>\"This two arm Phase III study named PROFILE 1007 (NCT00932893) was performed to assess crizotinib vs. established chemotherapy in non-small cell lung cancer patients positive for ALK rearrangement (assessed by break-apart FISH) with locally advanced or metastatic disease. 347 patients with disease progression after one round of platinum-based chemotherapy were randomized to receive crizotinib (N=173) or chemotherapy (N=174) consisting of pemetrexed in pemetrexed-naive patients and docetaxel for patients previously treated with pemetrexed. The primary endpoint - progression free survival (PFS) - was found to be significantly better for patients treated with crizotinib (median 7.7 months) than with chemotherapy (median 3.0 months) (hazard ratio for disease progression or death with crizotinib, 0.49; 95% CI, 0.37 to 0.64; P<0.001). Response rates were 65% for crizotinib treatment and 20% for chemotherapy (P<0.001). Improvements in quality of life were also reported.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>827, \"name\"=>\"Crizotinib versus chemotherapy in advanced ALK-positive lung cancer.\", \"citation\"=>\"Shaw et al., 2013, N. Engl. J. Med.\", \"pubmed_id\"=>\"23724913\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23724913\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>6, \"day\"=>20}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}, {\"id\"=>945, \"name\"=>\"EID945\", \"description\"=>\"Among 95 patients with NSCLC in this study, 15 had an ALK translocation. ORR of these patients compared to EGFR mutated or wild-type tumors was superior (46.7 versus 4.7 versus 16.2%, p = 0.001). TYMS expression was lower in ALK-positive cells.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>156, \"name\"=>\"Pemetrexed\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>647, \"name\"=>\"Anaplastic lymphoma kinase translocation: a predictive biomarker of pemetrexed in patients with non-small cell lung cancer.\", \"citation\"=>\"Lee et al., 2011, J Thorac Oncol\", \"pubmed_id\"=>\"21642865\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21642865\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>9}, \"journal\"=>\"J Thorac Oncol\", \"full_journal_title\"=>\"Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>499, \"phenotypes\"=>[]}]}, {\"id\"=>500, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"EML4-ALK E20;A20\", \"description\"=>\"EML4-ALK variant 2 consists of a fusion of the ALK kinase domain (exons 20-29) with EML4 exons 1-20. Variant 2 has cytoplasmic distribution and makes up ~10% of EML4-ALK fusions found in NSCLC,. Although the specific variant is not consistently reported, single cases demonstrating response to crizotinib treatment in NSCLC and a differentiated malignant neoplasm of unknown origin have been reported. In vitro, this variant has shown greater sensitivity to ALK inhibition with TAE684 and crizotinib than the EML4-ALK variants 1 (E13;A20), 3a (E6;A20) and 3b (E6ins33;A20). The variant was reported to be less stable in cells than other variants, and may be a client protein of Hsp90. Along these lines the variant has shown preclinical sensitivity to Hsp90 inhibition.\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>16.5, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>42396490, \"stop\"=>42552694, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000318522.5\", \"chromosome2\"=>\"2\", \"start2\"=>29415640, \"stop2\"=>29446394, \"representative_transcript2\"=>\"ENST00000389048.3\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/500/summary#variant\", \"evidence_items\"=>[{\"id\"=>1245, \"name\"=>\"EID1245\", \"description\"=>\"This case study presented a 53-year-old never-smoker with a poorly differentiated malignant neoplasm. Morphology and immunohistochemistry were unable to establish the primary tumor site. Right upper extremity (RUE), scalp, and multiple lung lesions were apparent. A lung lesion sample with sufficient material for molecular profiling was not attainable but sufficient material was present from the RUE for hybrid capture genomic profiling  (FoundationOne kit was used). EML4-ALK variant 2 was found, and the patient was started on crizotinib. Rapid symptom improvement was seen. After 1 month, a CT scan showed significant size reduction in lung tumors, and after 5 months the patient was asymptomatic with an entirely negative examination.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>862, \"name\"=>\"A Poorly Differentiated Malignant Neoplasm Lacking Lung Markers Harbors an EML4-ALK Rearrangement and Responds to Crizotinib.\", \"citation\"=>\"Chung et al., 2014, Case Rep Oncol\", \"pubmed_id\"=>\"25408655\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25408655\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4209277\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>9}, \"journal\"=>\"Case Rep Oncol\", \"full_journal_title\"=>\"Case reports in oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>500, \"phenotypes\"=>[]}, {\"id\"=>1189, \"name\"=>\"EID1189\", \"description\"=>\"In the PROFILE 1001 study, from 31 NSCLC tumors positive for ALK fusions with sufficient material for RT-PCR aided exon breakpoint identification, 1 case of EML4-ALK variant 2 (EML4 exon 20 fused to ALK exon 20) was found. This patient had adenocarcinoma and a 9 pack year smoking history. Crizotinib treatment duration was over 16 weeks. The patient had multiple lines of prior treatment and showed a partial response to first-time crizotinib treatment. Overall, in the 82 ALK positive patients treated in the study, 46 had partial responses (including this patient) and 1 had a complete response for an overall response rate of 57%.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>819, \"name\"=>\"Anaplastic lymphoma kinase inhibition in non-small-cell lung cancer.\", \"citation\"=>\"Kwak et al., 2010, N. Engl. J. Med.\", \"pubmed_id\"=>\"20979469\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20979469\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3014291\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>10, \"day\"=>28}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>500, \"phenotypes\"=>[]}, {\"id\"=>1204, \"name\"=>\"EID1204\", \"description\"=>\"EML4-ALK fusions occur in 2-7% of lung adenocarcinomas. Transduction of Ba/F3 cells with EML4-ALK variant 2 (EML4 exon 20 fused to ALK exon 20) caused the cells to grow independently of IL3 addition.  EML4-ALK may be a client protein of Hsp90. Addition of Hsp90 inhibitor 17-DMAG to the EML4-ALK variant 2 cells induced cell death which was rescued at lower 17-DMAG concentrations by IL3 addition. This result suggests that 17-DMAG specifically  targeted the EML4-ALK driver in these cells. Finally, 17-DMAG and crizotinib showed a synergistic effect in killing EML4-ALK variant 2 cells.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}, {\"id\"=>304, \"name\"=>\"17-DMAG\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>826, \"name\"=>\"Differential protein stability and ALK inhibitor sensitivity of EML4-ALK fusion variants.\", \"citation\"=>\"Heuckmann et al., 2012, Clin. Cancer Res.\", \"pubmed_id\"=>\"22912387\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22912387\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>9, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>500, \"phenotypes\"=>[]}, {\"id\"=>1196, \"name\"=>\"EID1196\", \"description\"=>\"EML4-ALK variants v1 (E13;A20), v2 (E20;A20), v3a (E6;A20) and v3b (E6ins33;A20) were expressed in Ba/F3 cells using retrovirus.  While all constructs rendered Ba/F3 cells IL3-independant, marked differences in sensitivity to ALK inhibitors TAE684 and crizotinib were observed, where v2 was most sensitized to growth inhibition and v3a the least, with v1 and v3b intermediate.  While phospho-ALK western blots indicated that the ALK inhibitors acted similarly on both v2 and v3a EML4-ALK variants, blots of cyclohexamide-treated cells showed that v2 had a shorter half-life than 3a in the transduced cells, potentially explaining increased v2 sensitivity.  Assays with a panel of mutated EML4-ALK variants of varied length suggested that longer EML4-ALK variants are less stable in cells, although this tendency was not observed across other types of ALK fusion.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>826, \"name\"=>\"Differential protein stability and ALK inhibitor sensitivity of EML4-ALK fusion variants.\", \"citation\"=>\"Heuckmann et al., 2012, Clin. Cancer Res.\", \"pubmed_id\"=>\"22912387\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22912387\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>9, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>500, \"phenotypes\"=>[]}]}, {\"id\"=>501, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"EML4-ALK E2;A20\", \"description\"=>\"\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>1.75, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>42396490, \"stop\"=>42472827, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000318522.5\", \"chromosome2\"=>\"2\", \"start2\"=>29415640, \"stop2\"=>29446394, \"representative_transcript2\"=>\"ENST00000389048.3\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/501/summary#variant\", \"evidence_items\"=>[{\"id\"=>1266, \"name\"=>\"EID1266\", \"description\"=>\"Adenocarcinoma cells were detected via urinary cytology in a 53 year old woman. Cancer morphology suggested papillary renal cell carcinoma (RCC), type 2A.  EML4-ALK variant 5 was discovered in tumor tissue using PCR with EML4 and ALK specific primers. The authors conclude that their findings confirm the potential for ALK targeting therapy in select cases of RCC.\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"E\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>876, \"name\"=>\"Identification of anaplastic lymphoma kinase fusions in renal cancer: large-scale immunohistochemical screening by the intercalated antibody-enhanced polymer method.\", \"citation\"=>\"Sugawara et al., 2012, Cancer\", \"pubmed_id\"=>\"22252991\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22252991\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>9, \"day\"=>15}, \"journal\"=>\"Cancer\", \"full_journal_title\"=>\"Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>501, \"phenotypes\"=>[]}, {\"id\"=>1190, \"name\"=>\"EID1190\", \"description\"=>\"Multiplex RT-PCR to detect in-frame fusions of EML4 to ALK was performed on total RNA from 253 lung adenocarcinomas, and 9 samples were found to contain such fusions, one of which was a novel fusion containing EML4 exon 2 and exon 20 of ALK. This was called variant 5. Two versions of this variant were seen: 5a consisted of EML4 exon 2 fused to ALK exon 20 while 5b contained a 117bp intronic ALK region between the two exons. Genomic PCR showed that EML4-ALK variant 5 tumor cells contained a single EML4-ALK fusion, suggesting 5a and 5b arise from alternate splicing.  5a and 5b showed in-vitro kinase activity and transformed 3T3 cells in cell culture. 5a and 5b transformed 3T3 cells formed tumors when injected in 8/8 nude mice, but 3T3 transformed with ALK alone did not.  These results suggest that EML4-ALK variant 5 is a driver mutation in lung adenocarcinoma targetable by ALK inhibitors like crizotinib.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"E\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>821, \"name\"=>\"Multiplex reverse transcription-PCR screening for EML4-ALK fusion transcripts.\", \"citation\"=>\"Takeuchi et al., 2008, Clin. Cancer Res.\", \"pubmed_id\"=>\"18927303\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18927303\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2008, \"month\"=>10, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>501, \"phenotypes\"=>[]}]}, {\"id\"=>503, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"EML4-ALK E6;A20\", \"description\"=>\"The EML4-ALK variant 3a consists of the ALK kinase domain fused to EML4 exons 1-6, while the 3b variant has an additional 33bp of intronic EML4 sequence between EML4 and ALK regions. The two variants were found to originate from a single genomic rearrangement, and 3a/3b is found in ~ 30% of EML4-ALK positive NSCLC. Variant 3a has cytoplasmic and nuclear distribution. In vitro, these variants have shown sensitivity to ALK inhibitors TAE684 and crizotinib but less sensitive than other EML4-ALK variants. The H2228 cell line contains 3a and 3b variants and is described as having cyctostatic response to ALK inhibition as opposed to apoptotic. In a study of 31 NSCLC patients with EML4-ALK rearrangements, 4/5 patients with the 3a variant showed a partial response while 1 showed progressive disease (the only PD observed). In cells the 3a variant was reported to be more stable than other variants, less sensitive to crizotinib and not to respond to Hsp90 inhibition.\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>25.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>42396490, \"stop\"=>42491871, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000318522.5\", \"chromosome2\"=>\"2\", \"start2\"=>29415640, \"stop2\"=>29446394, \"representative_transcript2\"=>\"ENST00000389048.3\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/503/summary#variant\", \"evidence_items\"=>[{\"id\"=>1205, \"name\"=>\"EID1205\", \"description\"=>\"Transduction of Ba/F3 cells with EML4-ALK variant 3a caused the cells to grow independently of IL3 addition.  While incubation of Ba/F3 cells transduced with EML4-ALK variants 1 and 2 with Hsp90 inhibitor 17-DMAG caused cell death that was reversible with IL3 addition, Ba/F3 cells with variant 3a were not sensitized to 17-DMAG induced death, suggesting that 17-DMAG was effecting the EML4-ALK driver of IL3-independant proliferation. This in turn suggests that variant 3a may not rely on Hsp90 for stability in cancer cells to the extent of other variants.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>304, \"name\"=>\"17-DMAG\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>826, \"name\"=>\"Differential protein stability and ALK inhibitor sensitivity of EML4-ALK fusion variants.\", \"citation\"=>\"Heuckmann et al., 2012, Clin. Cancer Res.\", \"pubmed_id\"=>\"22912387\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22912387\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>9, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>503, \"phenotypes\"=>[]}, {\"id\"=>1192, \"name\"=>\"EID1192\", \"description\"=>\"In a cohort of 75 NSCLC patients that had tested positive for 3 EML4-ALK rearrangements, two individuals were identified via RT-PCR with a novel EML4-ALK translocation involving the first 6 exons of EML4 fused to the kinase domain of ALK. Genomic PCR resolved a single breakpoint, but 2 variants of the fusion were observed in cells. Variant 3a consisted of fusion at EML4 exon 6 and ALK exon 20, while variant 3b fused a 33 bp intronic EML4 region between EML4 exon 6 and ALK exon 20.  3a and 3b showed strong in-vitro kinase activity, transformed mouse 3T3 cells, and 3b-transformed 3T3 cells formed tumors in 8/8 nude mice.  The ALK inhibitor 2,4-pyrimidinediamine inhibited growth of the 3a/b containing NCI-H2228 NSCLC cell line.  The results indicate that 3a/b is a targetable driver-mutation as other EML4-ALK variants.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>302, \"name\"=>\"2,4-pyrimidinediamine\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>823, \"name\"=>\"Identification of novel isoforms of the EML4-ALK transforming gene in non-small cell lung cancer.\", \"citation\"=>\"Choi et al., 2008, Cancer Res.\", \"pubmed_id\"=>\"18593892\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18593892\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2008, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>503, \"phenotypes\"=>[]}, {\"id\"=>1340, \"name\"=>\"EID1340\", \"description\"=>\"Ceritinib was 28x more potent (GI50 3.8nM) than crizotinib (GI50 107nM) at inhibiting growth of  H2228, an NSCLC cell line which harbors the EML4-ALK variant 3a/3b. SCID mice xenografted with H2228 were treated for 14 days with crizotinib (100 mg/kg) or ceritinib (25 or 50 mg/kg). All treatments caused tumor regression, but tumors recurred within 11 days after crizotinib treatment withdrawal. 4/8 mice treated with 25 mg/kg ceritinib showed regrowth after 1 month treatment withdrawal. The remaining 4/8 mice treated at 25 mg/kg ceritinib, and all mice treated at 50 mg/kg ceritinib maintained complete remission for four months. While 100 mg/kg crizotinib in mice approximates levels 3-5 times the maximal tolerated dose in humans, the authors state 25 to 50 mg/kg ceritinib in mice is predicted to approximate tolerable doses in humans.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>159, \"name\"=>\"Ceritinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>924, \"name\"=>\"The ALK inhibitor ceritinib overcomes crizotinib resistance in non-small cell lung cancer.\", \"citation\"=>\"Friboulet et al., 2014, Cancer Discov\", \"pubmed_id\"=>\"24675041\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24675041\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4068971\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>503, \"phenotypes\"=>[]}, {\"id\"=>1193, \"name\"=>\"EID1193\", \"description\"=>\"In a set of 31 patients with NSCLC, RT-PCR was performed to test for known EML4-ALK variants.  Five patients tested positive for the 3a/3b EML4-ALK variant.  Of these, 4 had partial response (PR), and one had disease progression (DP), and this was the only DP in the group of 31.  In contrast, of the 13 testing positive for EML4-ALK variant 1, 12 showed PR, and 1 showed stable disease.  These results potentially mirror findings in the H2228 3a/3b containing NSCLC cell line where some literature indicates it does respond as strongly to ALK inhibition as other EML4-ALK NSCLC cell lines.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>819, \"name\"=>\"Anaplastic lymphoma kinase inhibition in non-small-cell lung cancer.\", \"citation\"=>\"Kwak et al., 2010, N. Engl. J. Med.\", \"pubmed_id\"=>\"20979469\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20979469\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3014291\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>10, \"day\"=>28}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>503, \"phenotypes\"=>[]}, {\"id\"=>1197, \"name\"=>\"EID1197\", \"description\"=>\"Ba/F3 cells transduced with EML4-ALK variants v1 (E13;A20), v2 (E20;A20), v3a (E6;A20) and v3b (E6ins33;A20) showed differential sensitivity to both crizotinib and TAE684, with variant 3a (E6;A20) being the least sensitive. The authors suggest that the type/structure of ALK fusion may have some impact on patient outcome in targeted therapy.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>826, \"name\"=>\"Differential protein stability and ALK inhibitor sensitivity of EML4-ALK fusion variants.\", \"citation\"=>\"Heuckmann et al., 2012, Clin. Cancer Res.\", \"pubmed_id\"=>\"22912387\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22912387\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>9, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>503, \"phenotypes\"=>[]}, {\"id\"=>1195, \"name\"=>\"EID1195\", \"description\"=>\"Exon-array and RT-PCR was used to screen a panel of 83 NSCLC cell lines for EML4-ALK variants. The cell lines NCI-H3122 (containing EML4-ALK variant 1) and NCI-H2228 (containing the EML4 exon 6 and ALK exon 20 fusion-variant 3a/3b) were identified. TAE684 (0.1uM) appreciably inhibited growth as well as Akt and Erk1/2 phosphorylation in variant 1 containing H3122 cells. However, incomplete inhibition in 3a/3b containing H2228 cells was observed. H2228 growth inhibition by TAE684 was more similar to NSCLC cell lines harboring KRAS and EGFR mutations.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>18, \"name\"=>\"TAE684\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>825, \"name\"=>\"EML4-ALK fusion gene and efficacy of an ALK kinase inhibitor in lung cancer.\", \"citation\"=>\"Koivunen et al., 2008, Clin. Cancer Res.\", \"pubmed_id\"=>\"18594010\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18594010\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3025451\", \"publication_date\"=>{\"year\"=>2008, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>503, \"phenotypes\"=>[]}, {\"id\"=>1194, \"name\"=>\"EID1194\", \"description\"=>\"In a panel of 602 cancer cell lines, 3/134 NSCLC cell lines showed <0.5% cell viability following treatment with the ALK inhibitor TAE684. Of these, NCI-H2228 harbored EML4-ALK variant 3a/b (EML4 exon 6 and ALK exon 20). TAE684 inhibited Akt and Erk1/2 phosphorylation in H2228 cells and appeared to induce a cytostatic response.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>18, \"name\"=>\"TAE684\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>824, \"name\"=>\"Genomic alterations of anaplastic lymphoma kinase may sensitize tumors to anaplastic lymphoma kinase inhibitors.\", \"citation\"=>\"McDermott et al., 2008, Cancer Res.\", \"pubmed_id\"=>\"18451166\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18451166\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2008, \"month\"=>5, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>503, \"phenotypes\"=>[]}]}, {\"id\"=>513, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"NPM-ALK\", \"description\"=>\"\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>27.0, \"coordinates\"=>{\"chromosome\"=>\"5\", \"start\"=>170814120, \"stop\"=>170818803, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000517671.1\", \"chromosome2\"=>\"2\", \"start2\"=>29415640, \"stop2\"=>29446394, \"representative_transcript2\"=>\"ENST00000389048.3\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/513/summary#variant\", \"evidence_items\"=>[{\"id\"=>1268, \"name\"=>\"EID1268\", \"description\"=>\"Different ALK fusions were compared for crizotinib sensitivity using the Ba/F3 cell model. In Ba/F3 cells, AK fusions capable of acting as cancer drivers will cause IL3 independent growth, and ALK inhibition will reduce growth and cell viability. It was found that lower concentrations of crizotinib in the range of 0.3 micromolar, would inhibit Ba/F3 cells expressing EML4-ALK variant 2 and KIF5b-ALK, while concentrations around 1 micromolar would inhibit EML4-ALK variant 3a, with NPM-ALK cells being significantly inhibited around 0.7 micromolar crizotinib. 24 hour incubations with and without cyclohexamide (which inhibits protein synthesis) were performed in Ba/F3 cells ectopically expressing ALK fusions. The results indicated that NPM-ALK and EML4-ALK variant 3a protein levels were more stable in cells than other variants since there was more residual protein remaining in cells containing these variants after the cyclohexamide incubations, implying lower rate of protein turnover. These results may suggest subtle differences in NPM-ALK theraputic response compared to some other variants, although these fusions tend to arise in different cancer types.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"rejected\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>826, \"name\"=>\"Differential protein stability and ALK inhibitor sensitivity of EML4-ALK fusion variants.\", \"citation\"=>\"Heuckmann et al., 2012, Clin. Cancer Res.\", \"pubmed_id\"=>\"22912387\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22912387\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>9, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>513, \"phenotypes\"=>[]}, {\"id\"=>1241, \"name\"=>\"EID1241\", \"description\"=>\"In the second part of this Phase I study NCT00939770, nine patients between the ages of 12 months and 22 years with anaplastic large cell lymphoma (ALCL), who were refractory to treatment and for whom no further known treatments were available, were treated with crizotinib. Six patients with detectable NPM-ALK transcript levels responded with partial response, 1 with complete response, and 1 with stable disease.  Crizotinib was well tolerated in these children.\", \"disease\"=>{\"id\"=>348, \"name\"=>\"Anaplastic Large Cell Lymphoma\", \"display_name\"=>\"Anaplastic Large Cell Lymphoma\", \"doid\"=>\"0050744\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050744\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>859, \"name\"=>\"Safety and activity of crizotinib for paediatric patients with refractory solid tumours or anaplastic large-cell lymphoma: a Children's Oncology Group phase 1 consortium study.\", \"citation\"=>\"Mossé et al., 2013, Lancet Oncol.\", \"pubmed_id\"=>\"23598171\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23598171\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3730818\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>5}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>513, \"phenotypes\"=>[]}, {\"id\"=>1239, \"name\"=>\"EID1239\", \"description\"=>\"NPM-ALK in anaplastic large cell lymphoma (ALCL) was the first ALK-fusion to be discovered and characterized.  In this study, targeted activity against NPM-ALK by the ALK and MET inhibitor crizotinib was assessed in cell lines and tumor xenograft models. Treatment of NPM-ALK positive ALCL cell lines Karpas299 and SU-DHL-1 potently inhibited growth, and this was seen to occur at concentrations which also inhibited NPM-ALK phosphorylation in these cells. Administration of 100mg/kg/d crizotinib resulted in complete regression of all tumors, indicating that NPM-ALK could be an effective target for crizotinib in ALCL.\", \"disease\"=>{\"id\"=>348, \"name\"=>\"Anaplastic Large Cell Lymphoma\", \"display_name\"=>\"Anaplastic Large Cell Lymphoma\", \"doid\"=>\"0050744\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050744\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>857, \"name\"=>\"Cytoreductive antitumor activity of PF-2341066, a novel inhibitor of anaplastic lymphoma kinase and c-Met, in experimental models of anaplastic large-cell lymphoma.\", \"citation\"=>\"Christensen et al., 2007, Mol. Cancer Ther.\", \"pubmed_id\"=>\"18089725\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18089725\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>12}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>513, \"phenotypes\"=>[]}, {\"id\"=>1356, \"name\"=>\"EID1356\", \"description\"=>\"In Ba/F3 cells, crizotinib sensitivty of the NPM-ALK variant was compared to crizotinib sensitivity of EML4-ALK variant 3. 50% of maximal inhibitory concentration for NPM-ALK was 42 nM, which was approximately 4x greater than IC50 of 11 nM obtained for EML4-ALK. The IC50 for parental Ba/F3 supplemented with IL3 was 2299 nM, resulting in a fold difference of 209.2 for EML4-ALK and 55.1 for NPM-ALK, indicating some reduced sensitivity for NPM-ALK in direct comparison to EML4-ALK, but substantial sensitization to crizotinib for NPM-ALK in Ba/F3 cells.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>925, \"name\"=>\"Activity of second-generation ALK inhibitors against crizotinib-resistant mutants in an NPM-ALK model compared to EML4-ALK.\", \"citation\"=>\"Fontana et al., 2015, Cancer Med\", \"pubmed_id\"=>\"25727400\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25727400\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4529334\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>7}, \"journal\"=>\"Cancer Med\", \"full_journal_title\"=>\"Cancer medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>513, \"phenotypes\"=>[]}]}, {\"id\"=>514, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"RANBP2-ALK\", \"description\"=>\"\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>109335937, \"stop\"=>109375004, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000283195.6\", \"chromosome2\"=>\"2\", \"start2\"=>29415640, \"stop2\"=>29446394, \"representative_transcript2\"=>\"ENST00000389048.3\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/514/summary#variant\", \"evidence_items\"=>[{\"id\"=>1244, \"name\"=>\"EID1244\", \"description\"=>\"ALK-rearrangements are seen in 50% of inflammatory myofibroblastic tumor (IMT) cases. A 44 year old man was diagnosed with IMT.  He was treated with surgery and catheter placement for administration of cisplatin, doxyrubicin and mitomycin C.  Break-apart FISH showed ALK rearrangement, and immunohistochemical staining was characteristic of RANBP2 rearrangement. After further chemotherapy along with maintenance imatinib, crizotinib was started at 200mg twice daily, and a maximal partial response was achieved 5 months later, with regrowth occurring 2 months later. After further surgery, crizotinib was restarted at 250mg twice daily and almost 2 years later the patient was in complete radiographic remission. Another patient with non-ALK rearranged IMT was administered crizotinib in this study but did not respond.\", \"disease\"=>{\"id\"=>12, \"name\"=>\"Inflammatory Myofibroblastic Tumor\", \"display_name\"=>\"Inflammatory Myofibroblastic Tumor\", \"doid\"=>\"0050905\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050905\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>861, \"name\"=>\"Crizotinib in ALK-rearranged inflammatory myofibroblastic tumor.\", \"citation\"=>\"Butrynski et al., 2010, N. Engl. J. Med.\", \"pubmed_id\"=>\"20979472\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20979472\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3014292\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>10, \"day\"=>28}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>514, \"phenotypes\"=>[]}]}, {\"id\"=>516, \"entrez_name\"=>\"ROS1\", \"entrez_id\"=>6098, \"name\"=>\"CD74-ROS1 L2026M\", \"description\"=>\"The L2026M mutation is considered a gatekeeper mutation located in the inhibitor binding pocket of ROS1 and corresponding to L1196M in ALK and T790M in EGFR. This mutation has not been reportedly observed in a patient but inferred as a variant likely to emerge in crizotinib-resistant tumors. In vitro kinase activity, structural predictions and growth inhibition assays indicate the L2026M mutation in the CD74-ROS1 fusion provides crizotinib resistance. Additional preclinical assays have indicated that crizotinib resistant cells harboring this mutation are sensitive to PF-06463922 and foretinib.\", \"gene_id\"=>4941, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>9.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>117638365, \"stop\"=>117638365, \"reference_bases\"=>\"G\", \"variant_bases\"=>\"T\", \"representative_transcript\"=>\"ENST00000368508.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4941/summary/variants/516/summary#variant\", \"evidence_items\"=>[{\"id\"=>1257, \"name\"=>\"EID1257\", \"description\"=>\"Ba/F3 cells expressing CD74–ROS1 L2026M mutation showed resistance to crizotinib (~10 fold higher IC50 value), compared with those expressing wild-type CD74–ROS1; however, showed similar sensitivity to foretinib.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>868, \"name\"=>\"Molecular Changes Associated with Acquired Resistance to Crizotinib in ROS1-Rearranged Non-Small Cell Lung Cancer.\", \"citation\"=>\"Song et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"25688157\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25688157\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>5, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>516, \"phenotypes\"=>[]}, {\"id\"=>1253, \"name\"=>\"EID1253\", \"description\"=>\"Preclinical study of PF-06463922 in NSCLC in-vitro, in-vivo, and in enzyme assays. After establishing that the CD74-ROS1 L2026M mutation confers crizotinib resistance in kinase activity and cell viability assays (in transduced BaF3 cells), response to PF-06463922 was tested.  Wild-type CD74-ROS1, CD74-ROS1 G2032R, and CD74-ROS1 L2026M all showed sensitivity to PF-06463922 in these assays.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>241, \"name\"=>\"Lorlatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>867, \"name\"=>\"PF-06463922 is a potent and selective next-generation ROS1/ALK inhibitor capable of blocking crizotinib-resistant ROS1 mutations.\", \"citation\"=>\"Zou et al., 2015, Proc. Natl. Acad. Sci. U.S.A.\", \"pubmed_id\"=>\"25733882\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25733882\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4371934\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>3, \"day\"=>17}, \"journal\"=>\"Proc. Natl. Acad. Sci. U.S.A.\", \"full_journal_title\"=>\"Proceedings of the National Academy of Sciences of the United States of America\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>516, \"phenotypes\"=>[]}, {\"id\"=>1252, \"name\"=>\"EID1252\", \"description\"=>\"Preclinical study of PF-06463922 in NSCLC in-vitro, in-vivo and in enzyme assays. BaF3 cells expressing CD74-ROS1 L2026M were more resistant to crizotinib treatment than wildtype CD74-ROS1 in both cell viability and ROS1 phosphorylation. Crizotinib was also less effective at inhibiting kinase activity in ROS1 with L2026M mutation than wildtype. In contrast, PF-06463922 was effectively inhibited the fusion harboring this mutation in these assays.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>867, \"name\"=>\"PF-06463922 is a potent and selective next-generation ROS1/ALK inhibitor capable of blocking crizotinib-resistant ROS1 mutations.\", \"citation\"=>\"Zou et al., 2015, Proc. Natl. Acad. Sci. U.S.A.\", \"pubmed_id\"=>\"25733882\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25733882\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4371934\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>3, \"day\"=>17}, \"journal\"=>\"Proc. Natl. Acad. Sci. U.S.A.\", \"full_journal_title\"=>\"Proceedings of the National Academy of Sciences of the United States of America\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>516, \"phenotypes\"=>[]}, {\"id\"=>1258, \"name\"=>\"EID1258\", \"description\"=>\"Ba/F3 cells expressing CD74–ROS1 L2026M mutation showed similar sensitivity to foretinib compared with those expressing wild-type CD74–ROS1. This is in contrast to the increased resistance to crizotinib associated with this mutation in the same assay.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>150, \"name\"=>\"Foretinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>868, \"name\"=>\"Molecular Changes Associated with Acquired Resistance to Crizotinib in ROS1-Rearranged Non-Small Cell Lung Cancer.\", \"citation\"=>\"Song et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"25688157\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25688157\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>5, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>516, \"phenotypes\"=>[]}]}, {\"id\"=>517, \"entrez_name\"=>\"ROS1\", \"entrez_id\"=>6098, \"name\"=>\"CD74-ROS1 L2155S\", \"description\"=>\"\", \"gene_id\"=>4941, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>117630062, \"stop\"=>117630062, \"reference_bases\"=>\"A\", \"variant_bases\"=>\"G\", \"representative_transcript\"=>\"ENST00000368508.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4941/summary/variants/517/summary#variant\", \"evidence_items\"=>[{\"id\"=>1256, \"name\"=>\"EID1256\", \"description\"=>\"The L2155S mutation in ROS1 was identified in the HCC78CR1, -2 and -3 cell lines which harbor the SLC34A2–ROS1 rearrangement and in 2 patients with NSCLC with progressive disease following crizotinib treatment and CD74–ROS1 rearrangment. Crystal structure predicted ROS1 L2155S mutation to be highly destabilizating and cause protein structural modification. HCC78CR1, 2, and 3 cells and Ba/F3 cells transfected with CD74–ROS1 L2155S, showed resistance to crizotinib compared with parental HCC78 and Ba/F3 cells. In addition, these cell lines showed resistance to foretinib, a potent ROS1 inhibitor that is effective against ROS1 G2032R-mutant cells (16), and TAE684, a potent ALK inhibitor\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}, {\"id\"=>18, \"name\"=>\"TAE684\", \"pubchem_id\"=>nil}, {\"id\"=>150, \"name\"=>\"Foretinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>868, \"name\"=>\"Molecular Changes Associated with Acquired Resistance to Crizotinib in ROS1-Rearranged Non-Small Cell Lung Cancer.\", \"citation\"=>\"Song et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"25688157\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25688157\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>5, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>517, \"phenotypes\"=>[]}]}, {\"id\"=>518, \"entrez_name\"=>\"ROS1\", \"entrez_id\"=>6098, \"name\"=>\"CD74-ROS1 G2101A\", \"description\"=>\"\", \"gene_id\"=>4941, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>4.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>117631376, \"stop\"=>117631376, \"reference_bases\"=>\"C\", \"variant_bases\"=>\"G\", \"representative_transcript\"=>\"ENST00000368508.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4941/summary/variants/518/summary#variant\", \"evidence_items\"=>[{\"id\"=>1260, \"name\"=>\"EID1260\", \"description\"=>\"Ba/F3 cells expressing CD74–ROS1 G2101A showed resistance to crizotinib, compared with those expressing wild-type CD74–ROS1, and were sensitive to foretinib.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>150, \"name\"=>\"Foretinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>868, \"name\"=>\"Molecular Changes Associated with Acquired Resistance to Crizotinib in ROS1-Rearranged Non-Small Cell Lung Cancer.\", \"citation\"=>\"Song et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"25688157\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25688157\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>5, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>518, \"phenotypes\"=>[]}, {\"id\"=>1259, \"name\"=>\"EID1259\", \"description\"=>\"Ba/F3 cells expressing CD74–ROS1 G2101A showed resistance to crizotinib, compared with those expressing wild-type CD74–ROS1, and were sensitive to foretinib.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>868, \"name\"=>\"Molecular Changes Associated with Acquired Resistance to Crizotinib in ROS1-Rearranged Non-Small Cell Lung Cancer.\", \"citation\"=>\"Song et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"25688157\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25688157\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>5, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>518, \"phenotypes\"=>[]}]}, {\"id\"=>520, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"CLTC-ALK\", \"description\"=>\"The t(2;17)(p23;q23) translocation results in the CLTC-ALK fusion protein, the most common ALK fusion observed in diffuse large B cell lymphoma (DLBCL). ALK-rearranged DLBCL is less responsive to CHOP chemotherapy. Preclinical work indicates that CLTC-ALK DLBCL is responsive to ALK inhibition in cell lines and mouse models, and two case studies show short response followed by progression when heavily pretreated and advanced ALK-positive DLBCL is treated with crizotinib (one case CLTC-ALK, one case an unknown ALK fusion).\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>19.0, \"coordinates\"=>{\"chromosome\"=>\"17\", \"start\"=>57697219, \"stop\"=>57768072, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000269122.3\", \"chromosome2\"=>\"2\", \"start2\"=>29415640, \"stop2\"=>29446394, \"representative_transcript2\"=>\"ENST00000389048.3\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/520/summary#variant\", \"evidence_items\"=>[{\"id\"=>1265, \"name\"=>\"EID1265\", \"description\"=>\"A 21 year old male presented with lymphadenopathy and a low density mass in pancreas. Histopathological analysis showed diffuse tumor cell infiltrate (CD20/EMA negative) with cytoplasmic ALK. RT-PCR indicated CLTC-ALK fusion. The patient was diagnosed with splenic ALK + diffuse large B cell lymphoma (DLBCL), Stage III, Group A with an IPI of 3. CHOP-21 chemotherapy regimen was administered with improvement in abdominal lymph nodes but worsening in retroperitoneal lymph nodes assessed via standardized uptake values. Subsequent ICE chemotherapy regiment relieved some symptoms prompting the patient to refuse further treatment despite other indications of disease progression. Therapy was reinstated after worsening of disease. GEMOX chemotherapy was administered with dexamethasone and 250mg crizotinib twice daily resulting in symptom relief, decreased lymphadenopathy and LDH decline. The disease then progressed and the patient died 2 months after crizotinib initiation. The authors conclude that partial response to crizotinib was observed and that in future trials ALK rearranged DLBCL patients might see increased benefit with earlier application of targeted therapy during the treatment course.\", \"disease\"=>{\"id\"=>45, \"name\"=>\"Diffuse Large B-cell Lymphoma\", \"display_name\"=>\"Diffuse Large B-cell Lymphoma\", \"doid\"=>\"0050745\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050745\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>875, \"name\"=>\"Promising response of anaplastic lymphoma kinase-positive large B-cell lymphoma to crizotinib salvage treatment: case report and review of literature.\", \"citation\"=>\"Li et al., 2015, Int J Clin Exp Med\", \"pubmed_id\"=>\"26221234\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26221234\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4509179\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"Int J Clin Exp Med\", \"full_journal_title\"=>\"International journal of clinical and experimental medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>520, \"phenotypes\"=>[]}, {\"id\"=>1261, \"name\"=>\"EID1261\", \"description\"=>\"A cell line called LM1 was established from the bone marrow of a 13 year old female patient with relapsed CLTC-ALK positive diffuse large B-cell lymphoma (DLBCL). The patient had been heavily treated with chemotherapy before establishment of the cell line. The cell line maintained the tumor immunophenotype.  SNP analysis of patient bone marrow mononuclear cells and LM1 showed chromosomal changes associated with the cell line but 94.7% of SNPs were identically called.  ALK kinase inhibitor TAE684 induced cell death in LM1 cells at low nanomolar concentrations, while it did not significantly effect the ALK-rearrangement negative DLBCL cell line Karpas422.  LM1 cells formed tumors in 3/10 SCID and 9/10 NOD-SCID mice.  CD30 and CD138 immmunostaining were maintained between original tumor and LM1 xenografts. In mice, LM1 but not Karpas422 xenografts regressed with TAE684 administration, and complete remission for 13 weeks was observed before experiment was terminated.\", \"disease\"=>{\"id\"=>45, \"name\"=>\"Diffuse Large B-cell Lymphoma\", \"display_name\"=>\"Diffuse Large B-cell Lymphoma\", \"doid\"=>\"0050745\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050745\"}, \"drugs\"=>[{\"id\"=>18, \"name\"=>\"TAE684\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>871, \"name\"=>\"Inhibition of anaplastic lymphoma kinase (ALK) activity provides a therapeutic approach for CLTC-ALK-positive human diffuse large B cell lymphomas.\", \"citation\"=>\"Cerchietti et al., 2011, PLoS ONE\", \"pubmed_id\"=>\"21494621\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21494621\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3072987\", \"publication_date\"=>{\"year\"=>2011}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>520, \"phenotypes\"=>[]}, {\"id\"=>1263, \"name\"=>\"EID1263\", \"description\"=>\"CLTC-ALK is one of the most common ALK fusions found in diffuse large B cell lymphoma (DLBCL). In a retrospective study of 38 patients with ALK-rearranged DLBCL, 3 were CD20 positive. Only one showed cytoplasmic, nuclear and nucleolar staining associated with NPM-ALK, while the others showed granular cytoplasmic ALK stain characteristic for CLTC-ALK.  Seven molecularly characterized cases were positive for CLTC-ALK.  6/7 patients were treated with CHOP chemotherapy with survival of 5, 10, 26 and 27 months, and two alive at end of study, and one CD20 positive patient was treated with CHOP chemotherapy supplemented with CD20-targeted rituximab, with survival of 8 months, resulting in median survival of 26 months for this group.  Although these numbers are not sufficient for statistical comparison, they compare unfavorably to 3.1 year overall survival and 45% 5 year survival of CHOP treated DLBCL patients, and even less favorably to R-CHOP standard of care for CD20 positive DLBCL, suggesting poor prognosis for CLTC-ALK DLBCL and need for updated treatment in this cohort.\", \"disease\"=>{\"id\"=>45, \"name\"=>\"Diffuse Large B-cell Lymphoma\", \"display_name\"=>\"Diffuse Large B-cell Lymphoma\", \"doid\"=>\"0050745\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050745\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>873, \"name\"=>\"Anaplastic lymphoma kinase-positive diffuse large B-cell lymphoma: a rare clinicopathologic entity with poor prognosis.\", \"citation\"=>\"Laurent et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19636007\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19636007\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>9, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>520, \"phenotypes\"=>[]}]}, {\"id\"=>527, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"ALK FUSION I1171\", \"description\"=>\"Multiple case reports indicate that the I1171T missense mutation in ALK-rearranged non-small cell lung cancer (NSCLC) confers resistance to crizotinib and alectinib but is sensitive to ceritinib treatment. These results have been validated in vitro. In vitro evidence indicates this mutation interferes with alectinib binding the active site of ALK and may explain the sensitivity to the structurally distinct ALK inhibitor ceritinib.\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>42.5, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>29445213, \"stop\"=>29445213, \"reference_bases\"=>\"A\", \"variant_bases\"=>\"G\", \"representative_transcript\"=>\"ENST00000389048.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/527/summary#variant\", \"evidence_items\"=>[{\"id\"=>4612, \"name\"=>\"EID4612\", \"description\"=>\"In an in vitro study, Karpas299CR cells expressing NPM-ALK fusion (a known sensitizing alteration to crizotinib) and an ALK I1171T co-mutation, demonstrated resistance to crizotinib treatment compared to Karpas299 cells expressing NPM-ALK fusion. Resistance was determined by assessing cell viability.\", \"disease\"=>{\"id\"=>348, \"name\"=>\"Anaplastic Large Cell Lymphoma\", \"display_name\"=>\"Anaplastic Large Cell Lymphoma\", \"doid\"=>\"0050744\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050744\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2112, \"name\"=>\"Activating mutations in ALK kinase domain confer resistance to structurally unrelated ALK inhibitors in NPM-ALK-positive anaplastic large-cell lymphoma.\", \"citation\"=>\"Zdzalik et al., 2014, J. Cancer Res. Clin. Oncol.\", \"pubmed_id\"=>\"24509625\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24509625\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3949014\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>4}, \"journal\"=>\"J. Cancer Res. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of cancer research and clinical oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>527, \"phenotypes\"=>[]}, {\"id\"=>4611, \"name\"=>\"EID4611\", \"description\"=>\"In an in vitro study, a Ba/F3 cell line expressing EML4-ALK fusion (a known sensitizing alteration to crizotinib) and an ALK I1171T co-mutation, demonstrated resistance to crizotinib treatment compared to Ba/F3 cells expressing EML4-ALK fusion. Resistance was determined by assessing cell viability .\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>924, \"name\"=>\"The ALK inhibitor ceritinib overcomes crizotinib resistance in non-small cell lung cancer.\", \"citation\"=>\"Friboulet et al., 2014, Cancer Discov\", \"pubmed_id\"=>\"24675041\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24675041\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4068971\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>527, \"phenotypes\"=>[]}, {\"id\"=>4610, \"name\"=>\"EID4610\", \"description\"=>\"In a non-small cell lung cancer patient with an EML4-ALK gene rearrangement (a known sensitizing alteration to crizotinib) and an ALK I1171T co-mutation, ALK I1171T was reported to be refractory to crizotinib treatment. The patient was treated with standard chemotherapy and then crizotinib; the patient experienced a partial response, but had progressive disease after 8 months of crizotinib treatment.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2111, \"name\"=>\"Secondary mutations at I1171 in the ALK gene confer resistance to both Crizotinib and Alectinib.\", \"citation\"=>\"Toyokawa et al., 2014, J Thorac Oncol\", \"pubmed_id\"=>\"25393798\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25393798\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>12}, \"journal\"=>\"J Thorac Oncol\", \"full_journal_title\"=>\"Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>527, \"phenotypes\"=>[]}, {\"id\"=>1285, \"name\"=>\"EID1285\", \"description\"=>\"The I1171T ALK kinase domain mutaion was found in the MGH056-1 cell line which was derived from an ALK rearranged NSCLC patient who had progressed on crizotinib and then on alectinib given at lower dose (300mg twice daily).  Ectopic expression of EML4-ALK I1171T in Ba/F3 cells made these cells moderately resistant to crizotinib and resistant to alectinib. In contrast, the frequently used tool compound TAE684 was equally effective against EML4-ALK wild-type and I1171T, causing cell death at 1000x lower concentration then in unaltered Ba/F3 cells supplemented with IL3.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>18, \"name\"=>\"TAE684\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>887, \"name\"=>\"Two novel ALK mutations mediate acquired resistance to the next-generation ALK inhibitor alectinib.\", \"citation\"=>\"Katayama et al., 2014, Clin. Cancer Res.\", \"pubmed_id\"=>\"25228534\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25228534\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4233168\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>11, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>527, \"phenotypes\"=>[]}, {\"id\"=>1368, \"name\"=>\"EID1368\", \"description\"=>\"A 51 year old female never smoker was diagnosed with stage IV NSCLC with liver metastases, and had adverse response to crizotinib treatment, after which she responded to alectinib until disease progressed after 5 months with new liver metastases. Liver tumor biopsy was taken and estimated to be 15% tumor. Hybrid capture next generation sequencing was performed and an I1171N mutation was found in rearranged ALK, and estimated to be in 33% of the tumor cells. Ceritinib was given at 750 mg once daily but dose reduction to 600 mg once daily was required due to persistent nausea. CT showed regression of liver lesions and primary lung lesion and ongoing response was seen at 3 months.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>159, \"name\"=>\"Ceritinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>929, \"name\"=>\"I1171 missense mutation (particularly I1171N) is a common resistance mutation in ALK-positive NSCLC patients who have progressive disease while on alectinib and is sensitive to ceritinib.\", \"citation\"=>\"Ou et al., 2015, Lung Cancer\", \"pubmed_id\"=>\"25736571\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25736571\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>5}, \"journal\"=>\"Lung Cancer\", \"full_journal_title\"=>\"Lung cancer (Amsterdam, Netherlands)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>527, \"phenotypes\"=>[]}, {\"id\"=>1287, \"name\"=>\"EID1287\", \"description\"=>\"The MGH056-1 cell line was derived from a patient with ALK rearranged NSCLC who had progressed on crizotinib and then on lower dose alectinib (300 mg twice daily). The I1171T ALK kinase domain mutation was found in this cell line (established after alectinib resistance). MGH056-1 cells were resistant to crizotinib (IC50 236nmol/L; aproximately 50x higher than for ceritinib). About 50 fold higher concentration of crizotinib was required to inhibit Ba/F3 cells expressing EML4 ALK I1171T than was required for EML4 ALK wild-type.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>887, \"name\"=>\"Two novel ALK mutations mediate acquired resistance to the next-generation ALK inhibitor alectinib.\", \"citation\"=>\"Katayama et al., 2014, Clin. Cancer Res.\", \"pubmed_id\"=>\"25228534\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25228534\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4233168\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>11, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>527, \"phenotypes\"=>[]}, {\"id\"=>1367, \"name\"=>\"EID1367\", \"description\"=>\"A 51 year old female never-smoker was diagnosed with stage IV NSCLC with liver metastases. ALK rearrangement was verified via FISH. She was treated with crizotinib at 250 mg twice daily but discontinued due to adverse response. She then started alectinib at 600 mg twice daily with partial response, but new liver metastases then formed at 5 months. Liver tumor biopsy was performed and was estimated to be 15% tumor tissue. Next-generation sequencing discovered an ALK I1171N mutation. The tumor was not sequenced prior to alectinib resistance.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>321, \"name\"=>\"Alectinib (CH5424802)\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>929, \"name\"=>\"I1171 missense mutation (particularly I1171N) is a common resistance mutation in ALK-positive NSCLC patients who have progressive disease while on alectinib and is sensitive to ceritinib.\", \"citation\"=>\"Ou et al., 2015, Lung Cancer\", \"pubmed_id\"=>\"25736571\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25736571\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>5}, \"journal\"=>\"Lung Cancer\", \"full_journal_title\"=>\"Lung cancer (Amsterdam, Netherlands)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>527, \"phenotypes\"=>[]}, {\"id\"=>1344, \"name\"=>\"EID1344\", \"description\"=>\"Ba/F3 cells expressing EML4-ALK variant 3 with ALK I1171T mutation were treated with ceritinib, and survival IC50 of 2.6 nM was observed, which compares favorably with highly ceritinib sensitive EML4-ALK wild-type variant 1 survival IC50 of 1.7 nM reported from parallel experiments. Tumors from mice with EML4-ALK variant 3 H2228 NSCLC cell xenografts which had progressed after approximately 40 days of crizotinib treatment were sequenced for ALK mutations. I1171T was found as one of 6 ALK resistance mutations from 80 sequenced tumors. Treatment of these mouse xenograft models with ceritinib reduced tumor volume to zero, while in contrast tumor size increased with crizotinib treatment.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>159, \"name\"=>\"Ceritinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>924, \"name\"=>\"The ALK inhibitor ceritinib overcomes crizotinib resistance in non-small cell lung cancer.\", \"citation\"=>\"Friboulet et al., 2014, Cancer Discov\", \"pubmed_id\"=>\"24675041\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24675041\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4068971\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>527, \"phenotypes\"=>[]}, {\"id\"=>1284, \"name\"=>\"EID1284\", \"description\"=>\"The MGH056-1 cell line was derived from an NSCLC patient with ALK-rearrangement who had progressed on crizotinib and then on alectinib. the cell line was also resistant to crizotinib and alectinib, and sequencing of ALK from the cell line revealed the I1171T mutation. Further testing of this mutation in BaF3 cells revealed that cells expressing EML4-ALk with I1171T mutation required 10x higher concentration to induce cell death than BaF3 cells with wild-type EML4-ALK. In contrast, ceritinib caused loss of viability in EML4-ALK wild-type and I1171T BaF3 cells at comparable concentrations. Ceritinib was also effective at killing MGH056-1 cells that were resistant to alectinib, and reduced Akt and ERK phosphorylation in these cells. Finally, administration of ceritinib to the crizotinib and alectinib resistant patient let to tumor regression and partial response lasting 7 months.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>159, \"name\"=>\"Ceritinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>887, \"name\"=>\"Two novel ALK mutations mediate acquired resistance to the next-generation ALK inhibitor alectinib.\", \"citation\"=>\"Katayama et al., 2014, Clin. Cancer Res.\", \"pubmed_id\"=>\"25228534\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25228534\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4233168\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>11, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>527, \"phenotypes\"=>[]}, {\"id\"=>1283, \"name\"=>\"EID1283\", \"description\"=>\"The I1171T mutation in ALK fusions was found in a NSCLC patient shown positive for ALK rearrangement via break apart FISH assay, who progressed after 8 months of crizotinib, and then progressed again after 4 months of alectinib. The MGH056-1 cell line was derived from this patient, and this cell line was resistant to alectinib. Sequencing of MGH056-1 cells revealed the I1171T mutation in ALK. In cell culture studies Ba/F3 cells with EML4-ALK I1171T required more that 10x the concentration of alectinib to significantly reduce viability that was required for EML4-ALK wild-type expressing cells. Computer simulation studies indicated that the I1171T mutation interfered with alectinib binding at the ALK active site.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>321, \"name\"=>\"Alectinib (CH5424802)\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>887, \"name\"=>\"Two novel ALK mutations mediate acquired resistance to the next-generation ALK inhibitor alectinib.\", \"citation\"=>\"Katayama et al., 2014, Clin. Cancer Res.\", \"pubmed_id\"=>\"25228534\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25228534\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4233168\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>11, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>527, \"phenotypes\"=>[]}]}, {\"id\"=>528, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"EML4-ALK  V1180L\", \"description\"=>\"The V1180L missense mutation was identified in a mutagenesis screen to find mutations that conferred resistance to alectinib in a non-small cell lung cancer cell line. This mutation shows resistance to crizotinib and alectinib but sensitivity to TAE684, ceritinib and more mild sensitivity to brigatinib. Similar to I1171T, it resides in the ATP pocket of ALK and likely interferes with alectinib and crizotinib binding as a mechanism of resistance.\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>13.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>29443679, \"stop\"=>29443679, \"reference_bases\"=>\"C\", \"variant_bases\"=>\"G\", \"representative_transcript\"=>\"ENST00000389048.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/528/summary#variant\", \"evidence_items\"=>[{\"id\"=>1289, \"name\"=>\"EID1289\", \"description\"=>\"Passaging of the alectinib sensitive EML4-ALK containing H3122 NSCLC cell line with increasing levels of alectinib resulted in an alectinib resistant derivative of this cell line named H3122 CHR-A1. A V1180L missense mutation was found in the kinase domain of ALK in H3122 CHR-A1 cells. Both H3122 and H3122 CHR-A1 cells were highly and equally sensitive to ceritinib induced death. In Ba/F3 cells, ectopic expression of EML4-ALK V1180L rendered cells significantly more sensitive to ceritinib than Ba/F3 containing EML4-ALK I1171T mutant (alectinib and crizotinib resistant) or EML4-ALK wt.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>159, \"name\"=>\"Ceritinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>887, \"name\"=>\"Two novel ALK mutations mediate acquired resistance to the next-generation ALK inhibitor alectinib.\", \"citation\"=>\"Katayama et al., 2014, Clin. Cancer Res.\", \"pubmed_id\"=>\"25228534\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25228534\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4233168\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>11, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>528, \"phenotypes\"=>[]}, {\"id\"=>1290, \"name\"=>\"EID1290\", \"description\"=>\"The V1180L mutation in the kinase domain of rearranged ALK was found in an alectinib resistant cell line named H3122 CHR-A1, which was derived from the alectinib sensitive EML4-ALK variant 1 containing NSCLC cell line H3122 by passaging into higher concentrations of alectinib. This cell line is a model for gain of alectinib resistance, and also shows resistance to crizotinib. On the other hand, the often used tool ALK inhibitor TAE684 was highly effective in killing H3122 CHR-A1, where H3122 cells were only slightly more sensitized to TAE684-induced cell death. Ectopic expression of EML4-ALK V1180L in Ba/F3 cells caused IL3 independent growth as well as marked sensitization to TAE684, to a level greater than that seen in EML4-ALK I1171T (alectinib resistance mutation) or EML4-ALK wt, and over 1000x greater sensitivity than unaltered Ba/F3 + IL3.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>18, \"name\"=>\"TAE684\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>887, \"name\"=>\"Two novel ALK mutations mediate acquired resistance to the next-generation ALK inhibitor alectinib.\", \"citation\"=>\"Katayama et al., 2014, Clin. Cancer Res.\", \"pubmed_id\"=>\"25228534\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25228534\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4233168\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>11, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>528, \"phenotypes\"=>[]}, {\"id\"=>1291, \"name\"=>\"EID1291\", \"description\"=>\"ALK mutation V1180L was found in an H3122 derivative cell line that had become alectinib resistant after being passaged in increasing levels of alectinib. EML4-ALK V1180L and EML4-ALK wild-type were ectopically expressed in Ba/F3 cells, and brigatinib showed strong activity against both variants. The IC50 for brigatinib against EML4-ALK wild-type was 0.95, which is about 3x higher than the IC50 that was observed for alectinib, but about 6x lower than that for crizotinib. In contrast, the IC50 for brigatinib against the alectinib resistance variant EML4-ALK V1180L was 0.49, indicating increased sensitivity to brigatinib over non mutant EML4-ALK.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>318, \"name\"=>\"Brigatinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>887, \"name\"=>\"Two novel ALK mutations mediate acquired resistance to the next-generation ALK inhibitor alectinib.\", \"citation\"=>\"Katayama et al., 2014, Clin. Cancer Res.\", \"pubmed_id\"=>\"25228534\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25228534\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4233168\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>11, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>528, \"phenotypes\"=>[]}, {\"id\"=>1288, \"name\"=>\"EID1288\", \"description\"=>\"The alectinib resistant cell line H3122 CHR-A1 was generated by culturing of the EML4-ALK (variant 1) containing cell line H3122 with increasing levels of alectinib. Molecular characterization of H3122 CHR-A1 revealed a V1180L ALK kinase domain mutation. The H3122 CHR-A1 cell line required approximately 10 fold increase in crizotinib concentration to induce similar levels of cell death to H3122. Ectopic expression of EML4-ALK V1180L rendered Ba/F3 cells moderately less susceptible to crizotinib than ectopic EML4-ALK.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>887, \"name\"=>\"Two novel ALK mutations mediate acquired resistance to the next-generation ALK inhibitor alectinib.\", \"citation\"=>\"Katayama et al., 2014, Clin. Cancer Res.\", \"pubmed_id\"=>\"25228534\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25228534\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4233168\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>11, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>528, \"phenotypes\"=>[]}, {\"id\"=>1286, \"name\"=>\"EID1286\", \"description\"=>\"The EML4 ALK variant 1 containing NSCLC cell line H3122 is highly sensitive to alectinib, and an alectinib resistant derivative cell line H3122 CHR-A1 was generated by incubation of H3122 in increasing levels of alectinib culminating in 1 micro-molar, over a course of 7 months. The ALK kinase domain mutation V1180L was found in these cells, and Ba/F3 cells expressing EML4 ALK  V1180L were highly resistant to alectinib. Simulations indicated that the V1180L mutation interfered with alectinib binding. The V1180L mutation was found to be more resistant to alectinib than the I1171T mutation found in an ALK-positive NSCLC patient who developed alectinib resistance on lower dose alectinib  (300mg twice daily).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>321, \"name\"=>\"Alectinib (CH5424802)\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>887, \"name\"=>\"Two novel ALK mutations mediate acquired resistance to the next-generation ALK inhibitor alectinib.\", \"citation\"=>\"Katayama et al., 2014, Clin. Cancer Res.\", \"pubmed_id\"=>\"25228534\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25228534\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4233168\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>11, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>528, \"phenotypes\"=>[]}]}, {\"id\"=>551, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"ALK FUSION F1245C\", \"description\"=>\"In neuroblastoma, F1245 is one of three hotspot ALK mutations found at a high frequency. In a case study, a pretreated non-small cell lung cancer (NSCLC) patient with EML4-ALK F1245C mutation responded well to ceritinib after slow disease progression during crizotinib treatment. Crizotinib resistance was attributed to the F1245C mutation, although the ALK-rearrangement was not sequenced prior to crizotinib treatment. The patient's slow onset of disease progression during crizotinib treatment mirrors preclinical findings that indicate ALK-fusion with F1245C has moderate resistance to crizotinib compared to the fusion alone.\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>14.5, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>29436859, \"stop\"=>29436859, \"reference_bases\"=>\"A\", \"variant_bases\"=>\"C\", \"representative_transcript\"=>\"ENST00000389048.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/551/summary#variant\", \"evidence_items\"=>[{\"id\"=>1338, \"name\"=>\"EID1338\", \"description\"=>\"A 52 year old male never smoker was diagnosed with stage IV NSCLC adenocarcinoma and was treated with chemotherapy with eventual disease progression. He tested positive for ALK rearrangement via break apart FISH assay, and started crizotinib with near complete response for 27 months. Progression then occurred and biopsy of a pleural nodule was performed with hybrid-capture NGS. EML4-ALK variant 3 with F1245C mutation was found with allele frequency of 2% and tumor purity 13%. Due to slow progression and low tumor burden crizotinib was maintained for additional 8 months, and then he was transitioned onto ceritinib at 750 mg daily. After 3 months ceritinib, complete radiographic response was seen and the patient was without evidence of disease at 6 months. Dose was reduced to 600 mg ceritinib daily due to gastrointestinal side effects.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>159, \"name\"=>\"Ceritinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>923, \"name\"=>\"A novel acquired ALK F1245C mutation confers resistance to crizotinib in ALK-positive NSCLC but is sensitive to ceritinib.\", \"citation\"=>\"Kodityal et al., 2016, Lung Cancer\", \"pubmed_id\"=>\"26775591\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26775591\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>2}, \"journal\"=>\"Lung Cancer\", \"full_journal_title\"=>\"Lung cancer (Amsterdam, Netherlands)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>551, \"phenotypes\"=>[]}, {\"id\"=>1339, \"name\"=>\"EID1339\", \"description\"=>\"A 52 year old with a non-smoking history was diagnosed with stage IV NSCLC-adenocarcinoma and treated with chemotherapy.  Upon disease progression, the initial diagnostic biopsy tested positive for ALK-rearrangement using break apart FISH, but the ALK fusion was not sequenced for mutations. Break-apart signal was positive in 66% of tumor cells. Crizotinib was initiated and a sustained near complete response for 27 months was observed, after which an FDG-avid cervical lymph node appeared along with pleural effusions and pleural nodules. A pleural nodule was biopsied and profiled using hybrid-capture next generation sequencing, and revealed EML4-ALK variant 3 with F1245C mutation at mutant allele frequency of 2% and tumor purity of 13%. Crizotinib was maintained for 8 additional months due to low tumor burden and slow progression.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>923, \"name\"=>\"A novel acquired ALK F1245C mutation confers resistance to crizotinib in ALK-positive NSCLC but is sensitive to ceritinib.\", \"citation\"=>\"Kodityal et al., 2016, Lung Cancer\", \"pubmed_id\"=>\"26775591\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26775591\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>2}, \"journal\"=>\"Lung Cancer\", \"full_journal_title\"=>\"Lung cancer (Amsterdam, Netherlands)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>551, \"phenotypes\"=>[]}, {\"id\"=>1337, \"name\"=>\"EID1337\", \"description\"=>\"A Ba/F3 cell line expressing EML4-ALK variant 1 was created, and then exposed to the DNA modifying agent ENU. Cells were subsequently cultured in 96 well plates at increasing concentrations of crizotinib (250, 500, 720, 1000, 1440 and 2000 nM). Cell growth occurred in all wells at 250 nM, and was absent at 2000 nM. From 500 through 1440 nM there were progressively fewer resistant clones present. 422 mutations representing 16 different sites were identified. EML4-ALK F1245C was found at 500 nM crizotinib, and was among the 10 most frequently observed mutations. Ba/F3 cells with unaltered EML4-ALK had crizotinib IC50 values for viability of 132 +/- 45 nM. EML4-ALK F1245C clones from the selection assay had IC50 viability value of 425 +/- 100 nM, while Ba/F3 cells expressing reintroduced EML4-ALK F1245C had IC50 value of 269 +/- 194 nM.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>922, \"name\"=>\"Crizotinib-resistant mutants of EML4-ALK identified through an accelerated mutagenesis screen.\", \"citation\"=>\"Zhang et al., 2011, Chem Biol Drug Des\", \"pubmed_id\"=>\"22034911\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22034911\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3265718\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>12}, \"journal\"=>\"Chem Biol Drug Des\", \"full_journal_title\"=>\"Chemical biology & drug design\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>551, \"phenotypes\"=>[]}]}, {\"id\"=>552, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"ALK FUSION G1269A\", \"description\"=>\"\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>29432682, \"stop\"=>29432682, \"reference_bases\"=>\"C\", \"variant_bases\"=>\"G\", \"representative_transcript\"=>\"ENST00000389048.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/552/summary#variant\", \"evidence_items\"=>[{\"id\"=>1342, \"name\"=>\"EID1342\", \"description\"=>\"The NSCLC cell line MGH021-4, harboring SQSTM1-ALK G1269A, was 6 times more sensitive to ceritinib (80nM GI50) than crizotinib (500 nM) in proliferation assays and caused notable reductions in phospho-ALK and phospho-AKT (detected by Western blot) at concentrations 10 fold lower than crizotinib. GI50 values were similar in Ba/F3 cells expressing either EML4-ALK wild-type or EML4-ALK G1269A (1.6nM and 2.2nM, respectively). In contrast, crizotinib treatment of Ba/F3 with EML4-ALK G1269A had survival GI50 of 130nM.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>159, \"name\"=>\"Ceritinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>924, \"name\"=>\"The ALK inhibitor ceritinib overcomes crizotinib resistance in non-small cell lung cancer.\", \"citation\"=>\"Friboulet et al., 2014, Cancer Discov\", \"pubmed_id\"=>\"24675041\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24675041\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4068971\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>552, \"phenotypes\"=>[]}, {\"id\"=>1355, \"name\"=>\"EID1355\", \"description\"=>\"EML4-ALK with G1269A mutation was tested for sensitivity to ceritinib via transduction into Ba/F3 cells and generation of survival curves varying ceritinib concentration. IC50 of 33 nM was seen, which was 1.5 times the IC50 for EML4-ALK wild-type, indicating sensitivity of the G1269A mutation to ceritinib in the context of EML4-ALK. The mutation was also tested for ceritinib sensitivity in the context of NPM-ALK, where IC50 values were reported of 19 nM and 20 nM for mutant and wild-type versions, respectively, also indicating sensitivity. In western blots,  Y1604 pALK levels for wild-type NPM-ALK were ablated at 100 nM, while at 100 nM residual pALK levels were seen in G1269A blotting, with ablation of signal at and above 300 nM, which differed somewhat from the equivalent IC50 values reported for mutant and wild-type in the NPM-ALK background, but still agrees with sensitivity to ceritinib for this mutation.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>159, \"name\"=>\"Ceritinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>925, \"name\"=>\"Activity of second-generation ALK inhibitors against crizotinib-resistant mutants in an NPM-ALK model compared to EML4-ALK.\", \"citation\"=>\"Fontana et al., 2015, Cancer Med\", \"pubmed_id\"=>\"25727400\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25727400\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4529334\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>7}, \"journal\"=>\"Cancer Med\", \"full_journal_title\"=>\"Cancer medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>552, \"phenotypes\"=>[]}, {\"id\"=>1354, \"name\"=>\"EID1354\", \"description\"=>\"EML4-ALK with ALK G1269A mutation was tested for sensitivity to alectinib using Ba/F3 cells. In Ba/F3 cells with ectopic EML4-ALK variant 1 G1269A, IC50 of 84 nM was reported with alectinib, which was a 6 fold increase over IC50 for EML4-ALK wild-type. The authors considered a fold difference of 4 to 10 between the alectinib-sensitized wild-type and mutant variant to indicate a resistant mutant variant. In the background of the NPM-ALK fusion, IC50 for G1269A mutation was 104 nM, which was 5.3 fold that reported for NPM-ALK wild-type. Western blots of alectinib treated NPM-ALK G1269A were performed and reduced pALK Y1604 was seen at 300 nM alectinib, and no pALK at 1000 nM. In contrast pALK signal was ablated at and above 100 nM in NPM-ALK wild-type.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>321, \"name\"=>\"Alectinib (CH5424802)\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>925, \"name\"=>\"Activity of second-generation ALK inhibitors against crizotinib-resistant mutants in an NPM-ALK model compared to EML4-ALK.\", \"citation\"=>\"Fontana et al., 2015, Cancer Med\", \"pubmed_id\"=>\"25727400\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25727400\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4529334\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>7}, \"journal\"=>\"Cancer Med\", \"full_journal_title\"=>\"Cancer medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>552, \"phenotypes\"=>[]}, {\"id\"=>1353, \"name\"=>\"EID1353\", \"description\"=>\"EML4-ALK with ALK G1269A has been seen in crizotinib resistant NSCLC, and this mutation was tested for Brigatinib sensitivity via ectopic expression in Ba/F3 cells, where half maximal inhibitory concentration of 9 nM was seen, which was 1.5 times the IC50 of EML4-ALK wild-type to brigatinib. NPM-ALK G1269A was also tested for brigatinib sensitivity, and IC50 values of 11 nM were reported for both NPM-ALK G1269A and NPM-ALK wild-type. In western blotting, brigatinib at and above 100 nM ablated pALK signal (Y 1604), indicating sensitivity of this variant to brigatinib together with the other results.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>318, \"name\"=>\"Brigatinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>925, \"name\"=>\"Activity of second-generation ALK inhibitors against crizotinib-resistant mutants in an NPM-ALK model compared to EML4-ALK.\", \"citation\"=>\"Fontana et al., 2015, Cancer Med\", \"pubmed_id\"=>\"25727400\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25727400\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4529334\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>7}, \"journal\"=>\"Cancer Med\", \"full_journal_title\"=>\"Cancer medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>552, \"phenotypes\"=>[]}]}, {\"id\"=>569, \"entrez_name\"=>\"FGFR1\", \"entrez_id\"=>2260, \"name\"=>\"FGFR1 FUSIONS\", \"description\"=>\"\", \"gene_id\"=>1885, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>100.0, \"coordinates\"=>{\"chromosome\"=>\"8\", \"start\"=>38268656, \"stop\"=>38325363, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000425967.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1885/summary/variants/569/summary#variant\", \"evidence_items\"=>[{\"id\"=>1435, \"name\"=>\"EID1435\", \"description\"=>\"The 2016 World Health Organization guidelines for the classification of myeloid malignancies and acute leukemia describes a new class of disease defined by myeloid or lymphoid neoplasms associated with eosinophilia and rearrangement of PDGFRA, PDGFRB, or FGFR1 or with PCM1-JAK2.\", \"disease\"=>{\"id\"=>217, \"name\"=>\"Hematologic Cancer\", \"display_name\"=>\"Hematologic Cancer\", \"doid\"=>\"2531\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2531\"}, \"drugs\"=>[], \"rating\"=>5, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>966, \"name\"=>\"The 2016 revision to the World Health Organization classification of myeloid neoplasms and acute leukemia.\", \"citation\"=>\"Arber et al., 2016, Blood\", \"pubmed_id\"=>\"27069254\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27069254\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>5, \"day\"=>19}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>569, \"phenotypes\"=>[]}, {\"id\"=>1427, \"name\"=>\"EID1427\", \"description\"=>\"The 2016 World Health Organization guidelines for the classification of myeloid malignancies uses the detection of PCM1-JAK2 or rearrangements involving PDGFRA, PDGFRB, or FGFR1 as recommended exclusion criteria for a diagnosis of chronic myelomonocytic leukemia, particularly in patients with evidence of eosinophilia.\", \"disease\"=>{\"id\"=>2129, \"name\"=>\"Chronic Myelomonocytic Leukemia\", \"display_name\"=>\"Chronic Myelomonocytic Leukemia\", \"doid\"=>nil, \"url\"=>\"http://www.disease-ontology.org/\"}, \"drugs\"=>[], \"rating\"=>5, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Negative\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>966, \"name\"=>\"The 2016 revision to the World Health Organization classification of myeloid neoplasms and acute leukemia.\", \"citation\"=>\"Arber et al., 2016, Blood\", \"pubmed_id\"=>\"27069254\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27069254\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>5, \"day\"=>19}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>569, \"phenotypes\"=>[]}]}, {\"id\"=>574, \"entrez_name\"=>\"PDGFRA\", \"entrez_id\"=>5156, \"name\"=>\"FIP1L1-PDGFRA\", \"description\"=>\"Prospective trials of patients with hypereosinophilic syndrome harboring the FIP1L1-PDGFRA fusion identified marked response to imatinib treatment (Gotlib and Cool, Leukemia 2008). Imatinib is FDA approved for the treatment of &quot;adult patients with hypereosinophilic syndrome (HES) and/or chronic eosinophilic leukemia (CEL) who have the FIP1L1-PDGFRα fusion kinase&quot; (NDA021588).\", \"gene_id\"=>38, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>60.5, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>54243812, \"stop\"=>54294350, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000337488.6\", \"chromosome2\"=>\"4\", \"start2\"=>55141008, \"stop2\"=>55164414, \"representative_transcript2\"=>\"ENST00000257290.5\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/38/summary/variants/574/summary#variant\", \"evidence_items\"=>[{\"id\"=>1774, \"name\"=>\"EID1774\", \"description\"=>\"A patient with monoblastic myeloid sarcoma harbouring a PCR confirmed FIP1L1-PDGFRA rearrangement was treated with imatinib monotherapy. The patient demonstrated a durable response (10 wks) while on therapy which was reversed when the patient discontinued treatment. Imatinib therapy was subsequently restarted, with peripheral blood and BM analyses returning to normal after 4 weeks.\", \"disease\"=>{\"id\"=>2152, \"name\"=>\"Myeloid And Lymphoid Neoplasms With Eosinophilia And Abnormalities Of PDGFRA, PDGFRB, And FGFR1\", \"display_name\"=>\"Myeloid and lymphoid neoplasms with eosinophilia and abnormalities of PDGFRA, PDGFRB, and FGFR1\", \"doid\"=>\"0080164\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0080164\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1240, \"name\"=>\"Complete response of monoblastic myeloid sarcoma with FIP1L1- PDGFRA rearrangement to imatinib monotherapy.\", \"citation\"=>\"Srinivas et al., 2014, Br. J. Haematol.\", \"pubmed_id\"=>\"24456122\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24456122\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>6}, \"journal\"=>\"Br. J. Haematol.\", \"full_journal_title\"=>\"British journal of haematology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>574, \"phenotypes\"=>[]}, {\"id\"=>1441, \"name\"=>\"EID1441\", \"description\"=>\"A cohort of seven male patients with myeloproliferative hypereosinophilic syndrome and elevated serum tryptase levels were enrolled in a study of imatinib mesylate therapy. All seven patients demonstrated improvement in clinical symptoms and resolution of laboratory abnormalities and bone marrow pathology in response to treatment. Complete molecular remission of FIP1L1-PDGFRA was observed in 5 of 6 patients tested post-therapy.\", \"disease\"=>{\"id\"=>2152, \"name\"=>\"Myeloid And Lymphoid Neoplasms With Eosinophilia And Abnormalities Of PDGFRA, PDGFRB, And FGFR1\", \"display_name\"=>\"Myeloid and lymphoid neoplasms with eosinophilia and abnormalities of PDGFRA, PDGFRB, and FGFR1\", \"doid\"=>\"0080164\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0080164\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>974, \"name\"=>\"Molecular remission and reversal of myelofibrosis in response to imatinib mesylate treatment in patients with the myeloproliferative variant of hypereosinophilic syndrome.\", \"citation\"=>\"Klion et al., 2004, Blood\", \"pubmed_id\"=>\"14504092\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/14504092\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2004, \"month\"=>1, \"day\"=>15}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>574, \"phenotypes\"=>[]}, {\"id\"=>1445, \"name\"=>\"EID1445\", \"description\"=>\"Of eleven patients prospectively accrued with hypereosinophilic syndrome, nine had favorable responses to imatinib that lasted >3 months post-treatment. Of these nine patients, five were found to have a FIP1L1-PDGFRA fusion.\", \"disease\"=>{\"id\"=>2152, \"name\"=>\"Myeloid And Lymphoid Neoplasms With Eosinophilia And Abnormalities Of PDGFRA, PDGFRB, And FGFR1\", \"display_name\"=>\"Myeloid and lymphoid neoplasms with eosinophilia and abnormalities of PDGFRA, PDGFRB, and FGFR1\", \"doid\"=>\"0080164\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0080164\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>376, \"name\"=>\"A tyrosine kinase created by fusion of the PDGFRA and FIP1L1 genes as a therapeutic target of imatinib in idiopathic hypereosinophilic syndrome.\", \"citation\"=>\"Cools et al., 2003, N. Engl. J. Med.\", \"pubmed_id\"=>\"12660384\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/12660384\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2003, \"month\"=>3, \"day\"=>27}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>574, \"phenotypes\"=>[]}, {\"id\"=>1777, \"name\"=>\"EID1777\", \"description\"=>\"In a FIP1L1-PDGFRA positive eosinophil cell line growth-inhibitory effects were seen with all PDGFR-blocking agents, (ponatinib, sorafenib, masitinib, nilotinib, dasatinib, sunitinib, midostaurin) with ponatinib found to be the most potent compound.\", \"disease\"=>{\"id\"=>2152, \"name\"=>\"Myeloid And Lymphoid Neoplasms With Eosinophilia And Abnormalities Of PDGFRA, PDGFRB, And FGFR1\", \"display_name\"=>\"Myeloid and lymphoid neoplasms with eosinophilia and abnormalities of PDGFRA, PDGFRB, and FGFR1\", \"doid\"=>\"0080164\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0080164\"}, \"drugs\"=>[{\"id\"=>55, \"name\"=>\"Ponatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>nil, \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1241, \"name\"=>\"Identification of Ponatinib as a potent inhibitor of growth, migration, and activation of neoplastic eosinophils carrying FIP1L1-PDGFRA.\", \"citation\"=>\"Sadovnik et al., 2014, Exp. Hematol.\", \"pubmed_id\"=>\"24407160\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24407160\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4338611\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>4}, \"journal\"=>\"Exp. Hematol.\", \"full_journal_title\"=>\"Experimental hematology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>574, \"phenotypes\"=>[]}]}, {\"id\"=>576, \"entrez_name\"=>\"ROS1\", \"entrez_id\"=>6098, \"name\"=>\"TFG-ROS1\", \"description\"=>\"\", \"gene_id\"=>4941, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>7.5, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>100428269, \"stop\"=>100447702, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000240851.4\", \"chromosome2\"=>\"6\", \"start2\"=>117609463, \"stop2\"=>117642557, \"representative_transcript2\"=>\"ENST00000368508.3\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4941/summary/variants/576/summary#variant\", \"evidence_items\"=>[{\"id\"=>1444, \"name\"=>\"EID1444\", \"description\"=>\"Case report of a pediatric patient with treatment-refactory inflammatory myofibroblastic tumor negative for ALK expression and rearrangement. Next generation sequencing of the patient revealed a TFG-ROS1 fusion. CT scans after 3 cycles of crizotinib treatment showed a reduction of the tumor mass which continued to decrease at the time of publication after 4 months of crizonitib treatment.\", \"disease\"=>{\"id\"=>12, \"name\"=>\"Inflammatory Myofibroblastic Tumor\", \"display_name\"=>\"Inflammatory Myofibroblastic Tumor\", \"doid\"=>\"0050905\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050905\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>976, \"name\"=>\"Inflammatory myofibroblastic tumors harbor multiple potentially actionable kinase fusions.\", \"citation\"=>\"Lovly et al., 2014, Cancer Discov\", \"pubmed_id\"=>\"24875859\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24875859\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4125481\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>8}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>576, \"phenotypes\"=>[]}]}, {\"id\"=>577, \"entrez_name\"=>\"PDGFRA\", \"entrez_id\"=>5156, \"name\"=>\"FIP1L1-PDGFRA T674I\", \"description\"=>\"A case study of relapsed myeloproliferative hypereosinophilic syndrome suggests that a T674I mutation in the PDGFRA split kinase domain of FIP1L1-PDGFRA fusions may confer resistance to imatinib.\", \"gene_id\"=>38, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>8.0, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>55144547, \"stop\"=>55144547, \"reference_bases\"=>\"C\", \"variant_bases\"=>\"T\", \"representative_transcript\"=>\"ENST00000257290.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/38/summary/variants/577/summary#variant\", \"evidence_items\"=>[{\"id\"=>4420, \"name\"=>\"EID4420\", \"description\"=>\"In an in vitro study, a Ba/F3 cell line stably expressing the T674I mutation in the PDGFRA portion of the FIPL1-PDGFRA fusion protein was found to have reduced sensitivity to dasatinib treatment (IC50: approx. 1000 nM vs. <10 nM) compared to cells expressing the FIPL1-PDGFRA fusion protein. Resistance was determined by assessing cell growth inhibition, PDGFRA auto-phosphorylation and ERK 1/2 phosphorylation.\", \"disease\"=>{\"id\"=>2718, \"name\"=>\"Chronic Eosinophilic Leukemia\", \"display_name\"=>\"Chronic Eosinophilic Leukemia\", \"doid\"=>nil, \"url\"=>\"http://www.disease-ontology.org/\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2097, \"name\"=>\"FIP1L1-PDGFRalpha D842V, a novel panresistant mutant, emerging after treatment of FIP1L1-PDGFRalpha T674I eosinophilic leukemia with single agent sorafenib.\", \"citation\"=>\"Lierman et al., 2009, Leukemia\", \"pubmed_id\"=>\"19212337\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19212337\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>5}, \"journal\"=>\"Leukemia\", \"full_journal_title\"=>\"Leukemia\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>577, \"phenotypes\"=>[]}, {\"id\"=>1446, \"name\"=>\"EID1446\", \"description\"=>\"A patient treated with imatinib for a FIP1L1-PDGFRA mutation relapsed during treatment. A T674I mutation in the PDGFRA split kinase domain was hypothesized to confer resistance, due to the strong evidence of this fusion in driving myeloproliferative hypereosinophilic syndrome and the contextual similarity to T315I mutations in BCR-ABL.\", \"disease\"=>{\"id\"=>2152, \"name\"=>\"Myeloid And Lymphoid Neoplasms With Eosinophilia And Abnormalities Of PDGFRA, PDGFRB, And FGFR1\", \"display_name\"=>\"Myeloid and lymphoid neoplasms with eosinophilia and abnormalities of PDGFRA, PDGFRB, and FGFR1\", \"doid\"=>\"0080164\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0080164\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>376, \"name\"=>\"A tyrosine kinase created by fusion of the PDGFRA and FIP1L1 genes as a therapeutic target of imatinib in idiopathic hypereosinophilic syndrome.\", \"citation\"=>\"Cools et al., 2003, N. Engl. J. Med.\", \"pubmed_id\"=>\"12660384\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/12660384\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2003, \"month\"=>3, \"day\"=>27}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>577, \"phenotypes\"=>[]}, {\"id\"=>1779, \"name\"=>\"EID1779\", \"description\"=>\"Ba/F3 cells expressing two clinically relevant, imatinib-resistant, mutant forms of FIP1L1-PDGFRA, namely T674I and D842V were tested for growth inhibition by 8 PDGFR-blocking TKIs. Strong inhibitory effects on both mutants were seen only with ponatinib.\", \"disease\"=>{\"id\"=>2152, \"name\"=>\"Myeloid And Lymphoid Neoplasms With Eosinophilia And Abnormalities Of PDGFRA, PDGFRB, And FGFR1\", \"display_name\"=>\"Myeloid and lymphoid neoplasms with eosinophilia and abnormalities of PDGFRA, PDGFRB, and FGFR1\", \"doid\"=>\"0080164\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0080164\"}, \"drugs\"=>[{\"id\"=>55, \"name\"=>\"Ponatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1241, \"name\"=>\"Identification of Ponatinib as a potent inhibitor of growth, migration, and activation of neoplastic eosinophils carrying FIP1L1-PDGFRA.\", \"citation\"=>\"Sadovnik et al., 2014, Exp. Hematol.\", \"pubmed_id\"=>\"24407160\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24407160\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4338611\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>4}, \"journal\"=>\"Exp. Hematol.\", \"full_journal_title\"=>\"Experimental hematology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>577, \"phenotypes\"=>[]}]}, {\"id\"=>588, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"HIP1-ALK I1171N\", \"description\"=>\"\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>29445213, \"stop\"=>29445213, \"reference_bases\"=>\"A\", \"variant_bases\"=>\"T\", \"representative_transcript\"=>\"ENST00000389048.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/588/summary#variant\", \"evidence_items\"=>[{\"id\"=>4633, \"name\"=>\"EID4633\", \"description\"=>\"In a stage IV lung adenocarcinoma patient, ALK I1171N was associated with  acquired resistance to crizotinib monotherapy. An ALK gene rearrangement had been previously identified and was associated with an 8 month response with crizotinib treatment, followed by disease progression.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2115, \"name\"=>\"Identification of I1171N resistance mutation in ALK-positive non-small-cell lung cancer tumor sample and circulating tumor DNA.\", \"citation\"=>\"Johnson et al., 2016, Lung Cancer\", \"pubmed_id\"=>\"27565911\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27565911\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>9}, \"journal\"=>\"Lung Cancer\", \"full_journal_title\"=>\"Lung cancer (Amsterdam, Netherlands)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>588, \"phenotypes\"=>[]}, {\"id\"=>4632, \"name\"=>\"EID4632\", \"description\"=>\"In an in vitro study, cell lines expressing NPM-ALK I1171N, SUPM2CR03 (endogenous expression) and Ba/F3 (overexpression), demonstrated resistance to crizotinib treatment (IC50: 338 nmol/L and IC50: 215 nmol/L, respectively) compared to SUPM2 parental cells and Ba/F3 cells expressing NPM-ALK fusion (IC50: 23 nmol/L and 35 nmol/L, respectively). Resistance was determined by assessing cell viability and ALK auto-phosphorylation.\", \"disease\"=>{\"id\"=>348, \"name\"=>\"Anaplastic Large Cell Lymphoma\", \"display_name\"=>\"Anaplastic Large Cell Lymphoma\", \"doid\"=>\"0050744\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050744\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2110, \"name\"=>\"Crizotinib-resistant NPM-ALK mutants confer differential sensitivity to unrelated Alk inhibitors.\", \"citation\"=>\"Ceccon et al., 2013, Mol. Cancer Res.\", \"pubmed_id\"=>\"23239810\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23239810\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>2}, \"journal\"=>\"Mol. Cancer Res.\", \"full_journal_title\"=>\"Molecular cancer research : MCR\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>588, \"phenotypes\"=>[]}, {\"id\"=>1483, \"name\"=>\"EID1483\", \"description\"=>\"Case study describes a female (age 58) never-smoker presenting with stage IV NSCLC with an ALK rearrangement (confirmed by fish). Patient was treated with crizotinib (partial response) and alectinib (complete response). Liver metastasis was noted after 12 months with sequencing revealing a HIP1-ALK fusion with a I1171N variant.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}, {\"id\"=>321, \"name\"=>\"Alectinib (CH5424802)\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Sequential\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>995, \"name\"=>\"Identification of a novel HIP1-ALK fusion variant in Non-Small-Cell Lung Cancer (NSCLC) and discovery of ALK I1171 (I1171N/S) mutations in two ALK-rearranged NSCLC patients with resistance to Alectinib.\", \"citation\"=>\"Ou et al., 2014, J Thorac Oncol\", \"pubmed_id\"=>\"25393796\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25393796\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>12}, \"journal\"=>\"J Thorac Oncol\", \"full_journal_title\"=>\"Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>588, \"phenotypes\"=>[]}]}, {\"id\"=>589, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"EML4-ALK I1171S\", \"description\"=>\"\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>29445213, \"stop\"=>29445213, \"reference_bases\"=>\"A\", \"variant_bases\"=>\"C\", \"representative_transcript\"=>\"ENST00000389048.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/589/summary#variant\", \"evidence_items\"=>[{\"id\"=>4794, \"name\"=>\"EID4794\", \"description\"=>\"In an in vitro study, a FL5.12 cell line expressing NPM1-ALK or EML4-ALK (known sensitizing alterations to crizotinib) and an ALK I1171S co-mutation, demonstrated resistance to crizotinib treatment (IC50: 345.03 nmol/L and 260.38 nmol/L, respectively), as compared to FL5.12 cells expressing NPM1-ALK or EML4-ALK fusion proteins (IC50: 171.85 nmol/L and 70.59 nmol/L, respectively). Resistance was determined by assessing cell viability.\", \"disease\"=>{\"id\"=>164, \"name\"=>\"Lung Large Cell Carcinoma\", \"display_name\"=>\"Lung Large Cell Carcinoma\", \"doid\"=>\"4556\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4556\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2040, \"name\"=>\"TKI sensitivity patterns of novel kinase-domain mutations suggest therapeutic opportunities for patients with resistant ALK+ tumors.\", \"citation\"=>\"Amin et al., 2016, Oncotarget\", \"pubmed_id\"=>\"27009859\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27009859\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5029658\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>26}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>589, \"phenotypes\"=>[]}, {\"id\"=>1484, \"name\"=>\"EID1484\", \"description\"=>\"Case study describes a caucasian male (age 38) never smoker presenting with stage IV non-small cell lung cancer with an ALK rearrangement. Patient was given crizotinib and exhibited a partial response after 5 months with brain metastasis occurring at 21 months. Patient then received alectinib and exhibited partial response however liver metastasis presented after 12 months on alectinib. Sequencing of this liver metastasis revealed an EML4-ALK fusion with an I1171S variant at a variant frequency of 36% (tumor purity estimated as 70%).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}, {\"id\"=>321, \"name\"=>\"Alectinib (CH5424802)\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Sequential\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>995, \"name\"=>\"Identification of a novel HIP1-ALK fusion variant in Non-Small-Cell Lung Cancer (NSCLC) and discovery of ALK I1171 (I1171N/S) mutations in two ALK-rearranged NSCLC patients with resistance to Alectinib.\", \"citation\"=>\"Ou et al., 2014, J Thorac Oncol\", \"pubmed_id\"=>\"25393796\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25393796\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>12}, \"journal\"=>\"J Thorac Oncol\", \"full_journal_title\"=>\"Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>589, \"phenotypes\"=>[]}]}, {\"id\"=>599, \"entrez_name\"=>\"CTLA4\", \"entrez_id\"=>1493, \"name\"=>\"CTLA4-CD28\", \"description\"=>\"A survey of 115 T-cell lymphoma samples of diverse subtypes identified this fusion in 26 of 45 cases of angioimmunoblastic T-cell lymphomas (58%), nine of 39 peripheral T-cell lymphomas, not otherwise specified (23%), and nine of 31 extranodal NK/T cell lymphomas (29%).\", \"gene_id\"=>1285, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>204732509, \"stop\"=>204736210, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000302823.3\", \"chromosome2\"=>\"2\", \"start2\"=>204599507, \"stop2\"=>204603635, \"representative_transcript2\"=>\"ENST00000324106.8\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1285/summary/variants/599/summary#variant\", \"evidence_items\"=>[{\"id\"=>1520, \"name\"=>\"EID1520\", \"description\"=>\"A 67-year-old Caucasian female with stage IVA Sezary syndrome presented refractory to traditional therapy and was enrolled in a whole genome study trial. Sequencing revealed a CTLA4-CD28 fusion protein such that the inhibitory CTLA4 receptor activated a stimulatory CD28 intracellular domain that promoted proliferation. The patient was treated with iplimumab, a CTLA4 receptor inhibitor, and after 10 days, there was a reduction in erythema and ulceration by 50%, a reduction in tumor size by 75%, and overall improved quality of life. However, during the sixth week of therapy, the patient rapidly developed skin tumors and died 3 months later.\", \"disease\"=>{\"id\"=>2060, \"name\"=>\"Sezary's Disease\", \"display_name\"=>\"Sezary's Disease\", \"doid\"=>\"8541\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8541\"}, \"drugs\"=>[{\"id\"=>365, \"name\"=>\"Ipilimumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1013, \"name\"=>\"Personalized treatment of Sézary syndrome by targeting a novel CTLA4:CD28 fusion.\", \"citation\"=>\"Sekulic et al., 2015, Mol Genet Genomic Med\", \"pubmed_id\"=>\"25802883\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25802883\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4367085\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>3}, \"journal\"=>\"Mol Genet Genomic Med\", \"full_journal_title\"=>\"Molecular genetics & genomic medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>599, \"phenotypes\"=>[]}]}, {\"id\"=>608, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"EML4-ALK C1156Y–L1196M\", \"description\"=>\"\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>29443631, \"stop\"=>29445258, \"reference_bases\"=>nil, \"variant_bases\"=>\"\", \"representative_transcript\"=>\"ENST00000389048.3\", \"chromosome2\"=>\"\", \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>\"\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/608/summary#variant\", \"evidence_items\"=>[{\"id\"=>236, \"name\"=>\"EID236\", \"description\"=>\"A 28 year old patient with T4N3M1 stage lung adenocarcinoma harboring an EML4-ALK variant 1 fusion was treated with crizotinib after failing conventional therapy. The patient achieved a partial response but progressed after 5 months of treatment. Molecular analysis at this time identified two missense mutations in ALK C1156Y and L1196M. Ba/F3 cells expressing EML4-ALK L1196M or EML4-ALK C1156Y were more resistant to crizontinib treatment than those expressing EML4-ALK wildtype.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>166, \"name\"=>\"EML4-ALK mutations in lung cancer that confer resistance to ALK inhibitors.\", \"citation\"=>\"Choi et al., 2010, N. Engl. J. Med.\", \"pubmed_id\"=>\"20979473\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20979473\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>10, \"day\"=>28}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>608, \"phenotypes\"=>[]}]}, {\"id\"=>617, \"entrez_name\"=>\"BRAF\", \"entrez_id\"=>673, \"name\"=>\"PPFIBP2-BRAF\", \"description\"=>\"\", \"gene_id\"=>5, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"11\", \"start\"=>7535001, \"stop\"=>7586998, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000299492.4\", \"chromosome2\"=>\"7\", \"start2\"=>140434279, \"stop2\"=>140481493, \"representative_transcript2\"=>\"ENST00000288602.6\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5/summary/variants/617/summary#variant\", \"evidence_items\"=>[{\"id\"=>1570, \"name\"=>\"EID1570\", \"description\"=>\"Case report of a 47year old female patient with metastatic melanoma (BRAF, NRAS, KIT negative). A PPFIBP2-BRAF fusion was identified from DNA from a brain metastasis (inton 3 of PPFIBP2 fused to intron 10 of BRAF). Trametinib was introduced and anemia and ECOG status improved. Imaging revealed a 90% decrease in extracranial and 19% decrease in intracranial metastases with no new metastases and no progressing sites at 6 weeks. Trametinib was stopped and pembrolizumab introduced at this time. Progressive disease was noted after 5 cycles of pembrolizumab but re-introduction of trametinib did not show an effect.\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>19, \"name\"=>\"Trametinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1037, \"name\"=>\"Clinical activity of the MEK inhibitor trametinib in metastatic melanoma containing BRAF kinase fusion.\", \"citation\"=>\"Menzies et al., 2015, Pigment Cell Melanoma Res\", \"pubmed_id\"=>\"26072686\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26072686\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4539279\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>9}, \"journal\"=>\"Pigment Cell Melanoma Res\", \"full_journal_title\"=>\"Pigment cell & melanoma research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>617, \"phenotypes\"=>[]}]}, {\"id\"=>618, \"entrez_name\"=>\"BRAF\", \"entrez_id\"=>673, \"name\"=>\"KIAA1549-BRAF\", \"description\"=>\"\", \"gene_id\"=>5, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>7.5, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>138545885, \"stop\"=>138666064, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000440172.1\", \"chromosome2\"=>\"7\", \"start2\"=>140434279, \"stop2\"=>140487384, \"representative_transcript2\"=>\"ENST00000288602.6\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5/summary/variants/618/summary#variant\", \"evidence_items\"=>[{\"id\"=>1571, \"name\"=>\"EID1571\", \"description\"=>\"A 65 yr old male patient with metastatic acral lentiginous melanoma (BRAF, NRAS, KIT negative) was found to harbor a KIAA1549-BRAF (intron 15-intron 8) fusion in a subcutaneous metastasis sample after disease progression. Trametinib was started and fatigue and ECOG status improved but imaging revealed slight disease progression after 2 weeks (15 sites measurable, 9 stable, 6 progressive). No new metastases were identified. The patient was switched to pembrolizumab and major disease progression was noted.\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>19, \"name\"=>\"Trametinib\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1037, \"name\"=>\"Clinical activity of the MEK inhibitor trametinib in metastatic melanoma containing BRAF kinase fusion.\", \"citation\"=>\"Menzies et al., 2015, Pigment Cell Melanoma Res\", \"pubmed_id\"=>\"26072686\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26072686\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4539279\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>9}, \"journal\"=>\"Pigment Cell Melanoma Res\", \"full_journal_title\"=>\"Pigment cell & melanoma research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>618, \"phenotypes\"=>[]}, {\"id\"=>1664, \"name\"=>\"EID1664\", \"description\"=>\"A patient with a malignant spindle cell tumor of the chest wall treated as a soft tissue sarcoma was identified to harbor a KIAA1549-BRAF fusion. This patient responded to treatment with the pan-kinase inhibitor sorafenib in combination with bevacizumab and temsirolimus, achieving stable disease after 2 cycles extending into 11 cycles at which time she expired due to co-morbidities (acute myocardial infarction, hypotension). Of note, sequencing of 236 cancer-related genes identified CDKN2A A68fs*51, SUFU E283fs*3, MAP3K1 N325fs*3 and homozygous deletion of PTEN as well.\", \"disease\"=>{\"id\"=>105, \"name\"=>\"Sarcoma\", \"display_name\"=>\"Sarcoma\", \"doid\"=>\"1115\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1115\"}, \"drugs\"=>[{\"id\"=>6, \"name\"=>\"Sorafenib\", \"pubchem_id\"=>nil}, {\"id\"=>33, \"name\"=>\"Bevacizumab\", \"pubchem_id\"=>nil}, {\"id\"=>83, \"name\"=>\"Temsirolimus\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1198, \"name\"=>\"Targeted therapy by combined inhibition of the RAF and mTOR kinases in malignant spindle cell neoplasm harboring the KIAA1549-BRAF fusion protein.\", \"citation\"=>\"Subbiah et al., 2014, J Hematol Oncol\", \"pubmed_id\"=>\"24422672\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24422672\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3896681\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>1, \"day\"=>14}, \"journal\"=>\"J Hematol Oncol\", \"full_journal_title\"=>\"Journal of hematology & oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>618, \"phenotypes\"=>[]}]}, {\"id\"=>656, \"entrez_name\"=>\"BRAF\", \"entrez_id\"=>673, \"name\"=>\"BRAF-CUL1\", \"description\"=>\"\", \"gene_id\"=>5, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>140434279, \"stop\"=>140487384, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000288602.6\", \"chromosome2\"=>\"7\", \"start2\"=>148395737, \"stop2\"=>148457588, \"representative_transcript2\"=>\"ENST00000602748.1\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5/summary/variants/656/summary#variant\", \"evidence_items\"=>[{\"id\"=>1662, \"name\"=>\"EID1662\", \"description\"=>\"One patient with low-grade serous ovarian cancer had an in-frame fusion between the BRAF kinase domain and CUL1 identified by panel sequencing (MSK-IMPACT), with expression confirmed by whole-transcriptome sequencing. This patient with metastatic disease after treatment with carboplatin and paclitaxel, was enrolled onto a study of paclitaxel in combination with an oral MEK inhibitor and achieved a CR. She continued to receive therapy for 7 months, until discontinuation because of the development of pneumonitis. At publication, sustained CR had lasted > 18 months.\", \"disease\"=>{\"id\"=>87, \"name\"=>\"Ovarian Serous Carcinoma\", \"display_name\"=>\"Ovarian Serous Carcinoma\", \"doid\"=>\"0050933\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050933\"}, \"drugs\"=>[{\"id\"=>110, \"name\"=>\"MEK Inhibitor\", \"pubchem_id\"=>\"\"}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1124, \"name\"=>\"Extreme Outlier Analysis Identifies Occult Mitogen-Activated Protein Kinase Pathway Mutations in Patients With Low-Grade Serous Ovarian Cancer.\", \"citation\"=>\"Grisham et al., 2015, J. Clin. Oncol.\", \"pubmed_id\"=>\"26324360\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26324360\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4669594\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>12, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>656, \"phenotypes\"=>[]}]}, {\"id\"=>657, \"entrez_name\"=>\"BRAF\", \"entrez_id\"=>673, \"name\"=>\"ZKSCAN1-BRAF\", \"description\"=>\"\", \"gene_id\"=>5, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>7.5, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>99613222, \"stop\"=>99627575, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000426572.1\", \"chromosome2\"=>\"7\", \"start2\"=>140434279, \"stop2\"=>140482957, \"representative_transcript2\"=>\"ENST00000288602.6\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5/summary/variants/657/summary#variant\", \"evidence_items\"=>[{\"id\"=>1663, \"name\"=>\"EID1663\", \"description\"=>\"Analysis of BRAF fusions in 20,573 tumors, across 12 distinct tumor types. BRAF fusions were identified in 55 (0.3%) patients and enriched in spitzoid melanoma, pilocytic astrocytomas, pancreatic acinar and papillary thyroid cancers. Clinical data were available for two patients. Among them one 46-year old woman with spitzoid melanoma that  harbored  a\\nZKSCAN1-BRAF fusion responded to treatment with the MEK inhibitor trametinib. Subcutaneous  tumor  nodules  exhibited  clinical responses within 14 days of therapy, and her dominant bulky right lung metastases showed significant response by Day 45. Subsequent robotic-assisted lobectomy was able to remove the previously unresectable tumor with\\nclean surgical margins.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>19, \"name\"=>\"Trametinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1126, \"name\"=>\"The distribution of BRAF gene fusions in solid tumors and response to targeted therapy.\", \"citation\"=>\"Ross et al., 2016, Int. J. Cancer\", \"pubmed_id\"=>\"26314551\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26314551\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5049644\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>2, \"day\"=>15}, \"journal\"=>\"Int. J. Cancer\", \"full_journal_title\"=>\"International journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>657, \"phenotypes\"=>[]}]}, {\"id\"=>706, \"entrez_name\"=>\"EWSR1\", \"entrez_id\"=>2130, \"name\"=>\"EWSR1-FLI1\", \"description\"=>\"In approximately 90% of Ewing sarcoma/primitive neuroectodermal tumor (ES/PNET) a translocation t(11;22)(q24; q12) is identified, fusing the 5’ exons (containing the transactivatin domain, at least exon 1-7) of EWSR1, with the 3’ exons (from exon 9, coding the DNA binding domain) of FLI1.\", \"gene_id\"=>54, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>45.0, \"coordinates\"=>{\"chromosome\"=>\"22\", \"start\"=>29664307, \"stop\"=>29688158, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000414183.2\", \"chromosome2\"=>\"11\", \"start2\"=>128638013, \"stop2\"=>128683162, \"representative_transcript2\"=>\"ENST00000527786.2\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/54/summary/variants/706/summary#variant\", \"evidence_items\"=>[{\"id\"=>1754, \"name\"=>\"EID1754\", \"description\"=>\"An evaluation of alternative strategies for detection of EWS-FLI (FISH break apart, FISH fusion, and RT-PCR).  Of the 22 informative EWS/PNETs, 20 cases (91%) were positive by FISH break apart, FISH fusion, or both.\", \"disease\"=>{\"id\"=>135, \"name\"=>\"Bone Ewing's Sarcoma\", \"display_name\"=>\"Bone Ewing's Sarcoma\", \"doid\"=>\"3368\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3368\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1216, \"name\"=>\"Molecular diagnosis of Ewing sarcoma/primitive neuroectodermal tumor in routinely processed tissue: a comparison of two FISH strategies and RT-PCR in malignant round cell tumors.\", \"citation\"=>\"Bridge et al., 2006, Mod. Pathol.\", \"pubmed_id\"=>\"16258512\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16258512\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2006, \"month\"=>1}, \"journal\"=>\"Mod. Pathol.\", \"full_journal_title\"=>\"Modern pathology : an official journal of the United States and Canadian Academy of Pathology, Inc\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>706, \"phenotypes\"=>[]}, {\"id\"=>1752, \"name\"=>\"EID1752\", \"description\"=>\"A total of 85 unrelated cases of Ewing's sarcoma were examined and 83% we found to harbor the t(11;22)(q24;q12) chromosomal abnormality (EWS/FLI).\", \"disease\"=>{\"id\"=>135, \"name\"=>\"Bone Ewing's Sarcoma\", \"display_name\"=>\"Bone Ewing's Sarcoma\", \"doid\"=>\"3368\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3368\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1212, \"name\"=>\"Chromosomes in Ewing's sarcoma. I. An evaluation of 85 cases of remarkable consistency of t(11;22)(q24;q12).\", \"citation\"=>\"Turc-Carel et al., 1988, Cancer Genet. Cytogenet.\", \"pubmed_id\"=>\"3163261\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/3163261\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1988, \"month\"=>6}, \"journal\"=>\"Cancer Genet. Cytogenet.\", \"full_journal_title\"=>\"Cancer genetics and cytogenetics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>706, \"phenotypes\"=>[]}, {\"id\"=>1753, \"name\"=>\"EID1753\", \"description\"=>\"The authors propose RT-PCR as a diagnostic test for Ewing sarcoma using FFPE materials.  Starting with 40 cases, intact RNA could be isolated from 6, and of these the EWS-FLI event was detected in 2 by nested PCR. Alternative diagnostic assays for this event are also considered.\", \"disease\"=>{\"id\"=>135, \"name\"=>\"Bone Ewing's Sarcoma\", \"display_name\"=>\"Bone Ewing's Sarcoma\", \"doid\"=>\"3368\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3368\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1215, \"name\"=>\"Detection of t(11;22)(q24;q12) translocation of Ewing's sarcoma in paraffin embedded tissue by nested reverse transcription-polymerase chain reaction.\", \"citation\"=>\"Park et al., 1998, J. Korean Med. Sci.\", \"pubmed_id\"=>\"9741544\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/9741544\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3054425\", \"publication_date\"=>{\"year\"=>1998, \"month\"=>8}, \"journal\"=>\"J. Korean Med. Sci.\", \"full_journal_title\"=>\"Journal of Korean medical science\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>706, \"phenotypes\"=>[]}]}, {\"id\"=>717, \"entrez_name\"=>\"DDIT3\", \"entrez_id\"=>1649, \"name\"=>\"EWSR1-DDIT3\", \"description\"=>\"\", \"gene_id\"=>1413, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>7.5, \"coordinates\"=>{\"chromosome\"=>\"22\", \"start\"=>29664307, \"stop\"=>29687588, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000414183.2\", \"chromosome2\"=>\"12\", \"start2\"=>57910376, \"stop2\"=>57911536, \"representative_transcript2\"=>\"ENST00000346473.3\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1413/summary/variants/717/summary#variant\", \"evidence_items\"=>[{\"id\"=>1778, \"name\"=>\"EID1778\", \"description\"=>\"Identified two cases of myxoid liposarcoma with translocations (between 12q13 and 22q12) giving rise to EWSR1-DDIT3 (EWS-CHOP) fusions instead of the more common FUS-DDIT3.\", \"disease\"=>{\"id\"=>1297, \"name\"=>\"Myxoid Liposarcoma\", \"display_name\"=>\"Myxoid Liposarcoma\", \"doid\"=>\"5363\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5363\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1243, \"name\"=>\"Fusion of the EWS and CHOP genes in myxoid liposarcoma.\", \"citation\"=>\"Panagopoulos et al., 1996, Oncogene\", \"pubmed_id\"=>\"8637704\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/8637704\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1996, \"month\"=>2, \"day\"=>1}, \"journal\"=>\"Oncogene\", \"full_journal_title\"=>\"Oncogene\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>717, \"phenotypes\"=>[]}]}, {\"id\"=>739, \"entrez_name\"=>\"FOS\", \"entrez_id\"=>2353, \"name\"=>\"TRUNCATING FUSION\", \"description\"=>\"\", \"gene_id\"=>1955, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>50, \"name\"=>\"stop_gained\", \"display_name\"=>\"Stop Gained\", \"so_id\"=>\"SO:0001587\", \"description\"=>\"A sequence variant whereby at least one base of a codon is changed, resulting in a premature stop codon, leading to a shortened polypeptide.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001587\"}], \"civic_actionability_score\"=>7.5, \"coordinates\"=>{\"chromosome\"=>\"14\", \"start\"=>75745477, \"stop\"=>75748127, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000303562.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1955/summary/variants/739/summary#variant\", \"evidence_items\"=>[{\"id\"=>1802, \"name\"=>\"EID1802\", \"description\"=>\"FOS was found fused  with 3 different genes in 3 cases of bone epithelioid hemangioma. The fusion partners were not recurrent, respectively MBNL1, VIM and a lincRNA.  The truncation of the FOS gene would result in the loss of the transactivation domain (TAD) in all three cases.\", \"disease\"=>{\"id\"=>2161, \"name\"=>\"Bone Epithelioid Hemangioma\", \"display_name\"=>\"bone epithelioid hemangioma\", \"doid\"=>\"6610\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:6610\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1254, \"name\"=>\"Fusion events lead to truncation of FOS in epithelioid hemangioma of bone.\", \"citation\"=>\"van IJzendoorn et al., 2015, Genes Chromosomes Cancer\", \"pubmed_id\"=>\"26173738\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26173738\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>9}, \"journal\"=>\"Genes Chromosomes Cancer\", \"full_journal_title\"=>\"Genes, chromosomes & cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>739, \"phenotypes\"=>[]}]}, {\"id\"=>742, \"entrez_name\"=>\"WWTR1\", \"entrez_id\"=>25937, \"name\"=>\"WWTR1-CAMTA1\", \"description\"=>\"\", \"gene_id\"=>9952, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>149235022, \"stop\"=>149290787, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000465804.1\", \"chromosome2\"=>\"1\", \"start2\"=>7721786, \"stop2\"=>7829766, \"representative_transcript2\"=>\"ENST00000303635.7\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/9952/summary/variants/742/summary#variant\", \"evidence_items\"=>[{\"id\"=>1805, \"name\"=>\"EID1805\", \"description\"=>\"Performed a systematic molecular analysis of 17 epithelioid hemangioendothelioma (EHE) and compared to 13 epithelioid hemangiomas, five epithelioid angiosarcomas, and four epithelioid sarcoma-like EHE. A FISH positional cloning strategy, and RT-PCR analysis identified WWTR1-CAMTA1 in all EHE cases and none of the other benign or malignant epithelioid vascular tumors examined.\", \"disease\"=>{\"id\"=>2163, \"name\"=>\"Epithelioid Hemangioendothelioma\", \"display_name\"=>\"epithelioid hemangioendothelioma\", \"doid\"=>nil, \"url\"=>\"http://www.disease-ontology.org/\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1255, \"name\"=>\"A novel WWTR1-CAMTA1 gene fusion is a consistent abnormality in epithelioid hemangioendothelioma of different anatomic sites.\", \"citation\"=>\"Errani et al., 2011, Genes Chromosomes Cancer\", \"pubmed_id\"=>\"21584898\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21584898\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3264678\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>8}, \"journal\"=>\"Genes Chromosomes Cancer\", \"full_journal_title\"=>\"Genes, chromosomes & cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>742, \"phenotypes\"=>[]}]}, {\"id\"=>774, \"entrez_name\"=>\"PDGFRA\", \"entrez_id\"=>5156, \"name\"=>\"TNKS2-PDGFRA\", \"description\"=>\"\", \"gene_id\"=>38, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>7.5, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>93558069, \"stop\"=>93619405, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000371627.4\", \"chromosome2\"=>\"4\", \"start2\"=>55141008, \"stop2\"=>55164414, \"representative_transcript2\"=>\"ENST00000257290.5\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/38/summary/variants/774/summary#variant\", \"evidence_items\"=>[{\"id\"=>1838, \"name\"=>\"EID1838\", \"description\"=>\"A patient harbouring a novel TNKS2–PDGFRA fusion in a myeloid neoplasm with eosinophilia demonstrated a dramatic and durable response upon treatment with Imatinib mono therapy.\", \"disease\"=>{\"id\"=>2152, \"name\"=>\"Myeloid And Lymphoid Neoplasms With Eosinophilia And Abnormalities Of PDGFRA, PDGFRB, And FGFR1\", \"display_name\"=>\"Myeloid and lymphoid neoplasms with eosinophilia and abnormalities of PDGFRA, PDGFRB, and FGFR1\", \"doid\"=>\"0080164\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0080164\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1266, \"name\"=>\"Comprehensive genomic profiling identifies a novel TNKS2-PDGFRA fusion that defines a myeloid neoplasm with eosinophilia that responded dramatically to imatinib therapy.\", \"citation\"=>\"Chalmers et al., 2015, Blood Cancer J\", \"pubmed_id\"=>\"25658984\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25658984\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4349257\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>2, \"day\"=>6}, \"journal\"=>\"Blood Cancer J\", \"full_journal_title\"=>\"Blood cancer journal\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>774, \"phenotypes\"=>[]}]}, {\"id\"=>782, \"entrez_name\"=>\"FGFR2\", \"entrez_id\"=>2263, \"name\"=>\"FGFR2-BICC1\", \"description\"=>\"\", \"gene_id\"=>22, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>38.0, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>123239371, \"stop\"=>123357917, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000457416.2\", \"chromosome2\"=>\"10\", \"start2\"=>60461834, \"stop2\"=>60591195, \"representative_transcript2\"=>\"ENST00000373886.3\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/22/summary/variants/782/summary#variant\", \"evidence_items\"=>[{\"id\"=>1917, \"name\"=>\"EID1917\", \"description\"=>\"Phase 1 study of pan-FGFR inhibitor JNJ-42756493 in patients with solid tumors. No responses were seen 36 patients without known FGFR pathway aberrations. Among 23 patients with FGFR pathway aberrations, 4 partial responses as well as 16 with stable disease were seen. In patients with FGFR2-BICC1 fusions, 1 patient with urothelial cancer harboring FGFR2-BICC1/FGFR2-CASP7 fusions had a PR.\", \"disease\"=>{\"id\"=>2132, \"name\"=>\"Urothelial Carcinoma\", \"display_name\"=>\"Urothelial Carcinoma\", \"doid\"=>\"2671\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2671\"}, \"drugs\"=>[{\"id\"=>436, \"name\"=>\"JNJ-42756493\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1341, \"name\"=>\"Phase I Dose-Escalation Study of JNJ-42756493, an Oral Pan-Fibroblast Growth Factor Receptor Inhibitor, in Patients With Advanced Solid Tumors.\", \"citation\"=>\"Tabernero et al., 2015, J. Clin. Oncol.\", \"pubmed_id\"=>\"26324363\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26324363\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>10, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>782, \"phenotypes\"=>[]}, {\"id\"=>1851, \"name\"=>\"EID1851\", \"description\"=>\"Expression of the FGFR2-BICC1 fusion kinase in NIH3T3 cells demonstrated transforming ability that was effectively suppressed with the fibroblast growth factor receptor (FGFR) kinase inhibitors BGJ398 and PD173074.\", \"disease\"=>{\"id\"=>29, \"name\"=>\"Cholangiocarcinoma\", \"display_name\"=>\"Cholangiocarcinoma\", \"doid\"=>\"4947\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4947\"}, \"drugs\"=>[{\"id\"=>36, \"name\"=>\"PD173074\", \"pubchem_id\"=>nil}, {\"id\"=>151, \"name\"=>\"BGJ398\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1270, \"name\"=>\"Fibroblast growth factor receptor 2 tyrosine kinase fusions define a unique molecular subtype of cholangiocarcinoma.\", \"citation\"=>\"Arai et al., 2014, Hepatology\", \"pubmed_id\"=>\"24122810\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24122810\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>4}, \"journal\"=>\"Hepatology\", \"full_journal_title\"=>\"Hepatology (Baltimore, Md.)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>782, \"phenotypes\"=>[]}, {\"id\"=>1920, \"name\"=>\"EID1920\", \"description\"=>\"Phase 1 study of pan-FGFR inhibitor JNJ-42756493 in patients with solid tumors. No responses were seen 36 patients without FGFR pathway aberrations. Among 23 patients with FGFR pathway aberrations, 4 partial responses as well as 16 with stable disease were seen. A patient with endometrial cancer with synchronous FGFR3-TACC3 and FGFR2-BICC1/FGFR2-OFD1 fusions had a PR.\", \"disease\"=>{\"id\"=>108, \"name\"=>\"Endometrial Cancer\", \"display_name\"=>\"Endometrial Cancer\", \"doid\"=>\"1380\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1380\"}, \"drugs\"=>[{\"id\"=>436, \"name\"=>\"JNJ-42756493\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1341, \"name\"=>\"Phase I Dose-Escalation Study of JNJ-42756493, an Oral Pan-Fibroblast Growth Factor Receptor Inhibitor, in Patients With Advanced Solid Tumors.\", \"citation\"=>\"Tabernero et al., 2015, J. Clin. Oncol.\", \"pubmed_id\"=>\"26324363\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26324363\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>10, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>782, \"phenotypes\"=>[]}, {\"id\"=>1846, \"name\"=>\"EID1846\", \"description\"=>\"Massively parallel whole transcriptome sequencing in eight specimens from cholangiocarcinoma patients identified two fusion kinase genes, FGFR2-AHCYL1 and FGFR2-BICC1. In RT-PCR screening, the FGFR2 fusion was detected in nine patients with cholangiocarcinoma (9/102), exclusively in the intrahepatic subtype (9/66, 13.6%).\", \"disease\"=>{\"id\"=>29, \"name\"=>\"Cholangiocarcinoma\", \"display_name\"=>\"Cholangiocarcinoma\", \"doid\"=>\"4947\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4947\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1270, \"name\"=>\"Fibroblast growth factor receptor 2 tyrosine kinase fusions define a unique molecular subtype of cholangiocarcinoma.\", \"citation\"=>\"Arai et al., 2014, Hepatology\", \"pubmed_id\"=>\"24122810\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24122810\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>4}, \"journal\"=>\"Hepatology\", \"full_journal_title\"=>\"Hepatology (Baltimore, Md.)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>782, \"phenotypes\"=>[]}]}, {\"id\"=>783, \"entrez_name\"=>\"FGFR2\", \"entrez_id\"=>2263, \"name\"=>\"FGFR2-AHCYL1\", \"description\"=>\"\", \"gene_id\"=>22, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>23.0, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>123239371, \"stop\"=>123357917, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000457416.2\", \"chromosome2\"=>\"1\", \"start2\"=>110554984, \"stop2\"=>110566357, \"representative_transcript2\"=>\"ENST00000369799.5\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/22/summary/variants/783/summary#variant\", \"evidence_items\"=>[{\"id\"=>1853, \"name\"=>\"EID1853\", \"description\"=>\"Expression of the FGFR2-AHCYL1 fusion kinase in NIH3T3 cells demonstrated transforming ability that was effectively suppressed with the fibroblast growth factor receptor (FGFR) kinase inhibitors BGJ398 and PD173074.\", \"disease\"=>{\"id\"=>29, \"name\"=>\"Cholangiocarcinoma\", \"display_name\"=>\"Cholangiocarcinoma\", \"doid\"=>\"4947\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4947\"}, \"drugs\"=>[{\"id\"=>36, \"name\"=>\"PD173074\", \"pubchem_id\"=>nil}, {\"id\"=>151, \"name\"=>\"BGJ398\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1270, \"name\"=>\"Fibroblast growth factor receptor 2 tyrosine kinase fusions define a unique molecular subtype of cholangiocarcinoma.\", \"citation\"=>\"Arai et al., 2014, Hepatology\", \"pubmed_id\"=>\"24122810\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24122810\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>4}, \"journal\"=>\"Hepatology\", \"full_journal_title\"=>\"Hepatology (Baltimore, Md.)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>783, \"phenotypes\"=>[]}, {\"id\"=>1847, \"name\"=>\"EID1847\", \"description\"=>\"Massively parallel whole transcriptome sequencing in eight specimens from cholangiocarcinoma patients identified two fusion kinase genes, FGFR2-AHCYL1 and FGFR2-BICC1. In RT-PCR screening, the FGFR2 fusion was detected in nine patients with cholangiocarcinoma (9/102), exclusively in the intrahepatic subtype (9/66, 13.6%).\", \"disease\"=>{\"id\"=>29, \"name\"=>\"Cholangiocarcinoma\", \"display_name\"=>\"Cholangiocarcinoma\", \"doid\"=>\"4947\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4947\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>nil, \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1270, \"name\"=>\"Fibroblast growth factor receptor 2 tyrosine kinase fusions define a unique molecular subtype of cholangiocarcinoma.\", \"citation\"=>\"Arai et al., 2014, Hepatology\", \"pubmed_id\"=>\"24122810\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24122810\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>4}, \"journal\"=>\"Hepatology\", \"full_journal_title\"=>\"Hepatology (Baltimore, Md.)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>783, \"phenotypes\"=>[]}]}, {\"id\"=>801, \"entrez_name\"=>\"NTRK3\", \"entrez_id\"=>4916, \"name\"=>\"ETV6-NTRK3\", \"description\"=>\"\", \"gene_id\"=>3985, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>12.5, \"coordinates\"=>{\"chromosome\"=>\"12\", \"start\"=>11802788, \"stop\"=>12022903, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000396373.4\", \"chromosome2\"=>\"15\", \"start2\"=>88483984, \"stop2\"=>88418230, \"representative_transcript2\"=>\"ENST00000394480.2\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3985/summary/variants/801/summary#variant\", \"evidence_items\"=>[{\"id\"=>1870, \"name\"=>\"EID1870\", \"description\"=>\"Case report of a patient with mammary analogue secretory carcinoma (MASC) harboring an ETV6-NTRK3 fusion. The patient was treated with crizotinib and had stable disease but disease progression after 18 weeks. The patient was subsequently treated with entrectinib and exhibited a partial response for 7 months followed by disease progression.\", \"disease\"=>{\"id\"=>66, \"name\"=>\"Salivary Gland Carcinoma\", \"display_name\"=>\"Salivary Gland Carcinoma\", \"doid\"=>\"0050904\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050904\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1289, \"name\"=>\"What hides behind the MASC: clinical response and acquired resistance to entrectinib after ETV6-NTRK3 identification in a mammary analogue secretory carcinoma (MASC).\", \"citation\"=>\"Drilon et al., 2016, Ann. Oncol.\", \"pubmed_id\"=>\"26884591\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26884591\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4843186\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>5}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>801, \"phenotypes\"=>[]}, {\"id\"=>1869, \"name\"=>\"EID1869\", \"description\"=>\"Case report of a patient with mammary analogue secretory carcinoma (MASC) harboring an ETV6-NTRK3 fusion. The patient was treated with entrectinib and exhibited a partial response for 7 months followed by disease progression.\", \"disease\"=>{\"id\"=>66, \"name\"=>\"Salivary Gland Carcinoma\", \"display_name\"=>\"Salivary Gland Carcinoma\", \"doid\"=>\"0050904\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050904\"}, \"drugs\"=>[{\"id\"=>426, \"name\"=>\"Entrectinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1289, \"name\"=>\"What hides behind the MASC: clinical response and acquired resistance to entrectinib after ETV6-NTRK3 identification in a mammary analogue secretory carcinoma (MASC).\", \"citation\"=>\"Drilon et al., 2016, Ann. Oncol.\", \"pubmed_id\"=>\"26884591\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26884591\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4843186\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>5}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>801, \"phenotypes\"=>[]}]}, {\"id\"=>805, \"entrez_name\"=>\"NTRK3\", \"entrez_id\"=>4916, \"name\"=>\"ETV6-NTRK3 G623R\", \"description\"=>\"\", \"gene_id\"=>3985, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"15\", \"start\"=>88476265, \"stop\"=>88476265, \"reference_bases\"=>\"C\", \"variant_bases\"=>\"T\", \"representative_transcript\"=>\"ENST00000394480.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3985/summary/variants/805/summary#variant\", \"evidence_items\"=>[{\"id\"=>1874, \"name\"=>\"EID1874\", \"description\"=>\"Case report of a patient with mammary analogue secretory carcinoma (MASC) harboring an etv6-ntrk3 fusion. The patient was treated with entrectinib and exhibited a partial response for 7 months followed by disease progression.  A NTRK3 exon 16 G623R (c.1867G>A) mutation was identified in the progressive tumor. Cell line and structural models revealed that this mutation conferred relative resistance to entrectinib.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>426, \"name\"=>\"Entrectinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1289, \"name\"=>\"What hides behind the MASC: clinical response and acquired resistance to entrectinib after ETV6-NTRK3 identification in a mammary analogue secretory carcinoma (MASC).\", \"citation\"=>\"Drilon et al., 2016, Ann. Oncol.\", \"pubmed_id\"=>\"26884591\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26884591\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4843186\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>5}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>805, \"phenotypes\"=>[]}]}, {\"id\"=>1152, \"entrez_name\"=>\"ABL1\", \"entrez_id\"=>25, \"name\"=>\"BCR-ABL F486S\", \"description\"=>\"\", \"gene_id\"=>4, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>133755488, \"stop\"=>133755488, \"reference_bases\"=>\"T\", \"variant_bases\"=>\"C\", \"representative_transcript\"=>\"ENST00000318560.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4/summary/variants/1152/summary#variant\", \"evidence_items\"=>[{\"id\"=>4440, \"name\"=>\"EID4440\", \"description\"=>\"In a retrospective study of 118 imatinib-resistant or -intolerant, Philadelphia chromosome BCR-ABL1fusion positive, chronic myeloid leukemia (chronic phase) patients, no response to bosutinib monotherapy was reported in the 1 patient with F486S mutation in the ABL1 portion of the BCR-ABL gene fusion.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>520, \"name\"=>\"Bosutinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1997, \"name\"=>\"Bosutinib is active in chronic phase chronic myeloid leukemia after imatinib and dasatinib and/or nilotinib therapy failure.\", \"citation\"=>\"Khoury et al., 2012, Blood\", \"pubmed_id\"=>\"22371878\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22371878\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4916559\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>4, \"day\"=>12}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1152, \"phenotypes\"=>[]}, {\"id\"=>4439, \"name\"=>\"EID4439\", \"description\"=>\"In an in vitro study, a Ba/F3 cell line stably expressing the F486S mutation in the ABL1 portion of the BCR-ABL1 fusion protein was associated with reduced sensitivity to dasatinib treatment (IC50: 5.56 nM vs. IC50: 1.83 nM) as compared to Ba/F3 cells expressing BCR-ABL1 fusion protein. Resistance was determined by assessing cell proliferation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1529, \"name\"=>\"Activity of bosutinib, dasatinib, and nilotinib against 18 imatinib-resistant BCR/ABL mutants.\", \"citation\"=>\"Redaelli et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19075254\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19075254\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1152, \"phenotypes\"=>[]}, {\"id\"=>4438, \"name\"=>\"EID4438\", \"description\"=>\"In an in vitro study, a pro-B Ba/F3 cell line expressing F486S mutation in the ABL1 portion of the BCR-ABL1 fusion protein (overexpression) demonstrated resistance to axitinib treatment (IC50: 897nM), similar to Ba/F3 expressing the BCR-ABL1 fusion protein (overexpression, IC50: 823nM). Sensitivity was determined by assessing cell proliferation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>519, \"name\"=>\"Axitinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2002, \"name\"=>\"Axitinib effectively inhibits BCR-ABL1(T315I) with a distinct binding conformation.\", \"citation\"=>\"Pemovska et al., 2015, Nature\", \"pubmed_id\"=>\"25686603\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25686603\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>3, \"day\"=>5}, \"journal\"=>\"Nature\", \"full_journal_title\"=>\"Nature\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1152, \"phenotypes\"=>[]}, {\"id\"=>2829, \"name\"=>\"EID2829\", \"description\"=>\"In an in vitro study, a Ba/F3 cell line stably expressing the F486S mutation in the ABL1 portion of the BCR-ABL1 fusion protein was associated with sensitivity to nilotinib treatment (IC50: 1.85-fold), comparable to Ba/F3 cells expressing BCR-ABL fusion protein. Sensitivity was determined by assessing cell proliferation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>13, \"name\"=>\"Nilotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1529, \"name\"=>\"Activity of bosutinib, dasatinib, and nilotinib against 18 imatinib-resistant BCR/ABL mutants.\", \"citation\"=>\"Redaelli et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19075254\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19075254\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1152, \"phenotypes\"=>[]}]}, {\"id\"=>1233, \"entrez_name\"=>\"ABL1\", \"entrez_id\"=>25, \"name\"=>\"BCR-ABL G398R\", \"description\"=>\"\", \"gene_id\"=>4, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>133750361, \"stop\"=>133750361, \"reference_bases\"=>\"G\", \"variant_bases\"=>\"C\", \"representative_transcript\"=>\"ENST00000318560.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4/summary/variants/1233/summary#variant\", \"evidence_items\"=>[{\"id\"=>4561, \"name\"=>\"EID4561\", \"description\"=>\"In an in vitro study, a Ba/F3  cell line stably expressing the G398R mutation in the ABL1 portion of the BCR-ABL1 fusion protein was associated with sensitivity to dasatinib treatment (IC50: 1.27nM vs. IC50: 1.83nM), comparable to Ba/F3 cells expressing BCR-ABL1 fusion protein. Sensitivity was determined by assessing cell proliferation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>20, \"name\"=>\"Dasatinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1529, \"name\"=>\"Activity of bosutinib, dasatinib, and nilotinib against 18 imatinib-resistant BCR/ABL mutants.\", \"citation\"=>\"Redaelli et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19075254\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19075254\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1233, \"phenotypes\"=>[]}, {\"id\"=>4560, \"name\"=>\"EID4560\", \"description\"=>\"In an in vitro study, a Ba/F3 murine cell line stably expressing the G398R mutation in the ABL1 portion of the BCR-ABL1 fusion protein was associated with sensitivity to bosutinib treatment (IC50: 48.13 nM vs. IC50: 41.61 nM), comparable to a Ba/F3 cells expressing BCR-ABL1 fusion protein. Sensitivity was assessed by determining cell proliferation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>520, \"name\"=>\"Bosutinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1529, \"name\"=>\"Activity of bosutinib, dasatinib, and nilotinib against 18 imatinib-resistant BCR/ABL mutants.\", \"citation\"=>\"Redaelli et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19075254\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19075254\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1233, \"phenotypes\"=>[]}, {\"id\"=>2864, \"name\"=>\"EID2864\", \"description\"=>\"In an in vitro study, a Ba/F3 cell line stably expressing the G398R mutation in the ABL1 portion of the BCR-ABL1 gene fusion was associated with sensitivity to nilotinib treatment (IC50: 0.49-fold), comparable to Ba/F3 cells expressing BCR-ABL1 fusion protein (reported to be sensitizing to nilotinib). Sensitivity was determined by assessing cell proliferation.\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[{\"id\"=>13, \"name\"=>\"Nilotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1529, \"name\"=>\"Activity of bosutinib, dasatinib, and nilotinib against 18 imatinib-resistant BCR/ABL mutants.\", \"citation\"=>\"Redaelli et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19075254\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19075254\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1233, \"phenotypes\"=>[]}]}, {\"id\"=>1278, \"entrez_name\"=>\"NTRK1\", \"entrez_id\"=>4914, \"name\"=>\"LMNA-NTRK1\", \"description\"=>\"\", \"gene_id\"=>3983, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>120, \"name\"=>\"transcript_fusion\", \"display_name\"=>\"Transcript Fusion\", \"so_id\"=>\"SO:0001886\", \"description\"=>\"A feature fusion where the deletion brings together transcript regions.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001886\"}], \"civic_actionability_score\"=>12.5, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>156084498, \"stop\"=>156108548, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000368300.4\", \"chromosome2\"=>\"1\", \"start2\"=>156844363, \"stop2\"=>156851434, \"representative_transcript2\"=>\"ENST00000524377.1\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3983/summary/variants/1278/summary#variant\", \"evidence_items\"=>[{\"id\"=>2960, \"name\"=>\"EID2960\", \"description\"=>\"A gene fusion between exon 10 of NTRK1 and exon 11 of the LMNA genes was observed in a patient with metastatic colorectal cancer after therapy with first-line FOLFOX, second-line FOLFIRI/cetuximab, and third-line irinotecan. The patient treated in a clinical trial with pan-TRK kinase inhibitor entrectinib. Treatment was well tolerated and led to a partial response (PR) with 30% tumor shrinkage of multiple liver metastases. Clinical response lasted 4 months.\", \"disease\"=>{\"id\"=>57, \"name\"=>\"Colorectal Adenocarcinoma\", \"display_name\"=>\"Colorectal Adenocarcinoma\", \"doid\"=>\"0050861\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050861\"}, \"drugs\"=>[{\"id\"=>426, \"name\"=>\"Entrectinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1693, \"name\"=>\"Acquired Resistance to the TRK Inhibitor Entrectinib in Colorectal Cancer.\", \"citation\"=>\"Russo et al., 2016, Cancer Discov\", \"pubmed_id\"=>\"26546295\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26546295\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>1}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1278, \"phenotypes\"=>[]}, {\"id\"=>2955, \"name\"=>\"EID2955\", \"description\"=>\"LOXO-101 is an orally administered inhibitor of the TRK kinase and is highly selective only for the TRK family of receptors. The tumor of a 41-year-old woman with soft-tissue sarcoma metastatic to the lung was found to harbor an LMNA-NTRK1 gene fusion encoding a functional LMNA-TRKA fusion oncoprotein.  This patient's tumors underwent rapid and substantial tumor regression, with an accompanying improvement in pulmonary dyspnea, oxygen saturation, and plasma tumor markers.\", \"disease\"=>{\"id\"=>105, \"name\"=>\"Sarcoma\", \"display_name\"=>\"Sarcoma\", \"doid\"=>\"1115\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1115\"}, \"drugs\"=>[{\"id\"=>486, \"name\"=>\"Larotrectinib (LOXO-101)\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1691, \"name\"=>\"An Oncogenic NTRK Fusion in a Patient with Soft-Tissue Sarcoma with Response to the Tropomyosin-Related Kinase Inhibitor LOXO-101.\", \"citation\"=>\"Doebele et al., 2015, Cancer Discov\", \"pubmed_id\"=>\"26216294\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26216294\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4635026\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>10}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1278, \"phenotypes\"=>[]}]}, {\"id\"=>151, \"entrez_name\"=>\"BCL2\", \"entrez_id\"=>596, \"name\"=>\"IGH-BCL2\", \"description\"=>\"The typically reciprocal t(14;18)(q32;q21) translocation involves rearrangement between the immunoglobulin heavy chain locus on chromosome 14 and the 3' UTR of BCL2 on chromosome 18, leading to the overexpression of BCL2. This rearrangment is observed in >80% of patients with follicular lymphoma.\", \"gene_id\"=>59, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>124, \"name\"=>\"transcript_regulatory_region_fusion\", \"display_name\"=>\"Transcript Regulatory Region Fusion\", \"so_id\"=>\"SO:0001890\", \"description\"=>\"A feature fusion where the deletion brings together a regulatory region and a transcript region.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001890\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"14\", \"start\"=>106032614, \"stop\"=>107288051, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>nil, \"chromosome2\"=>\"18\", \"start2\"=>60794268, \"stop2\"=>60987019, \"representative_transcript2\"=>\"ENST00000333681.4\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/59/summary/variants/151/summary#variant\", \"evidence_items\"=>[{\"id\"=>407, \"name\"=>\"EID407\", \"description\"=>\"The t(14;18) transloaction of BCL2 is shown to be associated with germinal-center B-cell–like diffuse large-B-cell lymphomas.\", \"disease\"=>{\"id\"=>45, \"name\"=>\"Diffuse Large B-cell Lymphoma\", \"display_name\"=>\"Diffuse Large B-cell Lymphoma\", \"doid\"=>\"0050745\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050745\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>239, \"name\"=>\"The use of molecular profiling to predict survival after chemotherapy for diffuse large-B-cell lymphoma.\", \"citation\"=>\"Rosenwald et al., 2002, N. Engl. J. Med.\", \"pubmed_id\"=>\"12075054\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/12075054\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2002, \"month\"=>6, \"day\"=>20}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>151, \"phenotypes\"=>[]}]}, {\"id\"=>14, \"entrez_name\"=>\"BRAF\", \"entrez_id\"=>673, \"name\"=>\"V600E AMPLIFICATION\", \"description\"=>\"Amplification of BRAF V600E has been shown to confer resistance to MEK inhibitors. For more information on the V600 locus, see the V600E entry.\", \"gene_id\"=>5, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>1.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>140434279, \"stop\"=>140624564, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000288602.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5/summary/variants/14/summary#variant\", \"evidence_items\"=>[{\"id\"=>92, \"name\"=>\"EID92\", \"description\"=>\"COLO201 and COLO206F cells harboring BRAF V600E mutations were cloned to be MEK inhibitor (AZD6244 [selumetinib]) resistant. The mechanism of this resistance was shown to be amplification of the BRAF V600E gene.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>122, \"name\"=>\"Selumetinib (AZD6244)\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"E\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>102, \"name\"=>\"BRAF gene amplification can promote acquired resistance to MEK inhibitors in cancer cells harboring the BRAF V600E mutation.\", \"citation\"=>\"Corcoran et al., 2010, Sci Signal\", \"pubmed_id\"=>\"21098728\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21098728\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3372405\", \"publication_date\"=>{\"year\"=>2010}, \"journal\"=>\"Sci Signal\", \"full_journal_title\"=>\"Science signaling\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>14, \"phenotypes\"=>[]}]}, {\"id\"=>18, \"entrez_name\"=>\"CCND1\", \"entrez_id\"=>595, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"CCND1 amplification has been implicated in poorer prognosis in non-small cell lung cancer.\", \"gene_id\"=>8, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>47.0, \"coordinates\"=>{\"chromosome\"=>\"11\", \"start\"=>69455855, \"stop\"=>69469242, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000227507.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/8/summary/variants/18/summary#variant\", \"evidence_items\"=>[{\"id\"=>1599, \"name\"=>\"EID1599\", \"description\"=>\"A panel of 40 ovarian cancer cell lines with varied mutation profiles was assayed for sensitivity to the cdk4/6 inhibitor palbociclib (PD0332991) via measurement of IC50 values for inhibition of proliferation. Large variation in palbociclib sensitivity was observed and cell lines (N=40) were classified as above (resistant) or below (sensitive) the mean log of all IC50 values. 6/18 (33%) of the resistant lines and in only 1/22 (5%) of the sensitive lines (P = 0.017) had CCND1 copy number gain. Increased CCND1 expression was not correlated with sensitivity to palbociclib (p=0.671).\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>933, \"name\"=>\"Expression of p16 and retinoblastoma determines response to CDK4/6 inhibition in ovarian cancer.\", \"citation\"=>\"Konecny et al., 2011, Clin. Cancer Res.\", \"pubmed_id\"=>\"21278246\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21278246\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4598646\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>3, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>18, \"phenotypes\"=>[]}, {\"id\"=>1495, \"name\"=>\"EID1495\", \"description\"=>\"Retrospective analysis of somatic mutations and copy number changes in 119 patient treated with carboplatin, paclitaxel, ± sorafenib (CPS) in the E2603 trial. CPS therapy was associated with improved progression-free survival (PFS) compared with CP in patients with tumors with CCND1 gene copy gains (HR, 0.45; P = 0.035).\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>6, \"name\"=>\"Sorafenib\", \"pubchem_id\"=>nil}, {\"id\"=>81, \"name\"=>\"Paclitaxel\", \"pubchem_id\"=>nil}, {\"id\"=>93, \"name\"=>\"Carboplatin\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1002, \"name\"=>\"Copy Number Changes Are Associated with Response to Treatment with Carboplatin, Paclitaxel, and Sorafenib in Melanoma.\", \"citation\"=>\"Wilson et al., 2016, Clin. Cancer Res.\", \"pubmed_id\"=>\"26307133\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26307133\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4821426\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>1, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>18, \"phenotypes\"=>[]}, {\"id\"=>1562, \"name\"=>\"EID1562\", \"description\"=>\"A panel of 47 breast cancer cell lines was tested for sensitivty/resistance to CDK4/6 inhibitor Palbociclib (PD0332991),  where IC50 below 150nM was sensitive, and above 1000nM resistant. Cyclin D1 had higher expression levels in sensitive cell lines.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1031, \"name\"=>\"PD 0332991, a selective cyclin D kinase 4/6 inhibitor, preferentially inhibits proliferation of luminal estrogen receptor-positive human breast cancer cell lines in vitro.\", \"citation\"=>\"Finn et al., 2009, Breast Cancer Res.\", \"pubmed_id\"=>\"19874578\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19874578\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2790859\", \"publication_date\"=>{\"year\"=>2009}, \"journal\"=>\"Breast Cancer Res.\", \"full_journal_title\"=>\"Breast cancer research : BCR\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>18, \"phenotypes\"=>[]}, {\"id\"=>1560, \"name\"=>\"EID1560\", \"description\"=>\"28 renal cell carcinoma cell lines were tested for sensitivity to CDK4/6 inhibitor Palbociclib (PD0332991). 19 were reported sensitive, and 9 resistant. 15 of the 28 cell lines had high levels of Cyclin D1 (CCND1) expression which did not correlate significantly with palbociclib sensitivity (p=0.348).\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>212, \"name\"=>\"PD-0332991, a potent and selective inhibitor of cyclin-dependent kinase 4/6, demonstrates inhibition of proliferation in renal cell carcinoma at nanomolar concentrations and molecular markers predict for sensitivity.\", \"citation\"=>\"Logan et al., 2013, Anticancer Res.\", \"pubmed_id\"=>\"23898052\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23898052\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>8}, \"journal\"=>\"Anticancer Res.\", \"full_journal_title\"=>\"Anticancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>18, \"phenotypes\"=>[]}, {\"id\"=>858, \"name\"=>\"EID858\", \"description\"=>\"In this retrospective analysis, CCND1 amplification was analyzed in 442 patients. No interaction with treatment was observed.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>53, \"name\"=>\"Tamoxifen\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>574, \"name\"=>\"RSF1 and not cyclin D1 gene amplification may predict lack of benefit from adjuvant tamoxifen in high-risk pre-menopausal women in the MA.12 randomized clinical trial.\", \"citation\"=>\"Keilty et al., 2013, PLoS ONE\", \"pubmed_id\"=>\"24367492\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24367492\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3868649\", \"publication_date\"=>{\"year\"=>2013}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>18, \"phenotypes\"=>[]}, {\"id\"=>354, \"name\"=>\"EID354\", \"description\"=>\"Inceased copy number of CCND1 is associated with poorer overall survival.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>210, \"name\"=>\"Cyclin D1 in non-small cell lung cancer: a key driver of malignant transformation.\", \"citation\"=>\"Gautschi et al., 2007, Lung Cancer\", \"pubmed_id\"=>\"17070615\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17070615\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>1}, \"journal\"=>\"Lung Cancer\", \"full_journal_title\"=>\"Lung cancer (Amsterdam, Netherlands)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>18, \"phenotypes\"=>[]}]}, {\"id\"=>153, \"entrez_name\"=>\"FOXP1\", \"entrez_id\"=>27086, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"Amplication in FOXP1 has been shown to be associated with the ABC subtype of DLBCL.\", \"gene_id\"=>56, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>71003844, \"stop\"=>71633140, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000318789.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/56/summary/variants/153/summary#variant\", \"evidence_items\"=>[{\"id\"=>408, \"name\"=>\"EID408\", \"description\"=>\"Amplification of FOXP1 is shown to be associated with ABC DLBCL.\", \"disease\"=>{\"id\"=>45, \"name\"=>\"Diffuse Large B-cell Lymphoma\", \"display_name\"=>\"Diffuse Large B-cell Lymphoma\", \"doid\"=>\"0050745\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050745\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>240, \"name\"=>\"Molecular subtypes of diffuse large B-cell lymphoma arise by distinct genetic pathways.\", \"citation\"=>\"Lenz et al., 2008, Proc. Natl. Acad. Sci. U.S.A.\", \"pubmed_id\"=>\"18765795\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18765795\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2533222\", \"publication_date\"=>{\"year\"=>2008, \"month\"=>9, \"day\"=>9}, \"journal\"=>\"Proc. Natl. Acad. Sci. U.S.A.\", \"full_journal_title\"=>\"Proceedings of the National Academy of Sciences of the United States of America\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>153, \"phenotypes\"=>[]}]}, {\"id\"=>154, \"entrez_name\"=>\"REL\", \"entrez_id\"=>5966, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>57, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>61108709, \"stop\"=>61149800, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000295025.8\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/57/summary/variants/154/summary#variant\", \"evidence_items\"=>[{\"id\"=>409, \"name\"=>\"EID409\", \"description\"=>\"Amplification of REL is shown to be associated with germinal-center B-cell–like diffuse large-B-cell lymphomas.\", \"disease\"=>{\"id\"=>45, \"name\"=>\"Diffuse Large B-cell Lymphoma\", \"display_name\"=>\"Diffuse Large B-cell Lymphoma\", \"doid\"=>\"0050745\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050745\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>239, \"name\"=>\"The use of molecular profiling to predict survival after chemotherapy for diffuse large-B-cell lymphoma.\", \"citation\"=>\"Rosenwald et al., 2002, N. Engl. J. Med.\", \"pubmed_id\"=>\"12075054\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/12075054\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2002, \"month\"=>6, \"day\"=>20}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>154, \"phenotypes\"=>[]}]}, {\"id\"=>180, \"entrez_name\"=>\"AURKA\", \"entrez_id\"=>6790, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>61, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>0.75, \"coordinates\"=>{\"chromosome\"=>\"20\", \"start\"=>54944446, \"stop\"=>54967393, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000395913.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/61/summary/variants/180/summary#variant\", \"evidence_items\"=>[{\"id\"=>456, \"name\"=>\"EID456\", \"description\"=>\"Antitumour effect of alisertib treatment in esophageal adenocarcinomas can be intensified in combination with cisplatin.\", \"disease\"=>{\"id\"=>170, \"name\"=>\"Esophagus Adenocarcinoma\", \"display_name\"=>\"Esophagus Adenocarcinoma\", \"doid\"=>\"4914\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4914\"}, \"drugs\"=>[{\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}, {\"id\"=>363, \"name\"=>\"Alisertib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"E\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>272, \"name\"=>\"The aurora kinase A inhibitor MLN8237 enhances cisplatin-induced cell death in esophageal adenocarcinoma cells.\", \"citation\"=>\"Sehdev et al., 2012, Mol. Cancer Ther.\", \"pubmed_id\"=>\"22302096\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22302096\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3297687\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>3}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>180, \"phenotypes\"=>[]}]}, {\"id\"=>187, \"entrez_name\"=>\"CCNE1\", \"entrez_id\"=>898, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>11, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>27.0, \"coordinates\"=>{\"chromosome\"=>\"19\", \"start\"=>30302805, \"stop\"=>30315215, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000262643.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/11/summary/variants/187/summary#variant\", \"evidence_items\"=>[{\"id\"=>1629, \"name\"=>\"EID1629\", \"description\"=>\"Preclinical study in breast cancer cell lines treated with CDK4/6 inhibitor palbociclib. Cell lines with acquired resistance to palbociclib were developed. Two palbociclib-resistant cell lines were derived through chronic exposure to 1 μmol/L palbociclib during 3 to 4 months. Copy number profiling from exome sequencing of MCF-7 and MCF-7pR, confirmed relative amplification of CCNE1. Silencing of CCNE1 or CDK2 alone in MCF-7pR cells had no effect on cell-cycle arrest, but resulted in substantially increased cell-cycle arrest and reduction in cell growth in combination with palbociclib.\", \"disease\"=>{\"id\"=>28, \"name\"=>\"Estrogen-receptor Positive Breast Cancer\", \"display_name\"=>\"Estrogen-receptor Positive Breast Cancer\", \"doid\"=>\"0060075\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060075\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1078, \"name\"=>\"Early Adaptation and Acquired Resistance to CDK4/6 Inhibition in Estrogen Receptor-Positive Breast Cancer.\", \"citation\"=>\"Herrera-Abreu et al., 2016, Cancer Res.\", \"pubmed_id\"=>\"27020857\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27020857\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"day\"=>15}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>187, \"phenotypes\"=>[]}, {\"id\"=>1735, \"name\"=>\"EID1735\", \"description\"=>\"A high throughput compound screen identified synergistic combinations in CCNE1 amplified high grade serous ovarian carcinoma, including dinaciclib and AKT inhibitors\", \"disease\"=>{\"id\"=>184, \"name\"=>\"Ovarian Serous Cystadenocarcinoma\", \"display_name\"=>\"Ovarian Serous Cystadenocarcinoma\", \"doid\"=>\"5746\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5746\"}, \"drugs\"=>[{\"id\"=>43, \"name\"=>\"MK-2206\", \"pubchem_id\"=>nil}, {\"id\"=>410, \"name\"=>\"Dinaciclib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1178, \"name\"=>\"Selective Targeting of Cyclin E1-Amplified High-Grade Serous Ovarian Cancer by Cyclin-Dependent Kinase 2 and AKT Inhibition.\", \"citation\"=>\"Au-Yeung et al., 2016, Clin. Cancer Res.\", \"pubmed_id\"=>\"27663592\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27663592\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>9, \"day\"=>23}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>187, \"phenotypes\"=>[]}, {\"id\"=>466, \"name\"=>\"EID466\", \"description\"=>\"In a retrospective study of 88 ovarian carcinomas, 18 had amplification of CCNE1. Those with amplification had worse overall and disease-free survival.\", \"disease\"=>{\"id\"=>47, \"name\"=>\"Ovarian Carcinoma\", \"display_name\"=>\"Ovarian Carcinoma\", \"doid\"=>\"4001\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4001\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>280, \"name\"=>\"Gene amplification CCNE1 is related to poor survival and potential therapeutic target in ovarian cancer.\", \"citation\"=>\"Nakayama et al., 2010, Cancer\", \"pubmed_id\"=>\"20336784\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20336784\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>6, \"day\"=>1}, \"journal\"=>\"Cancer\", \"full_journal_title\"=>\"Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>187, \"phenotypes\"=>[]}]}, {\"id\"=>190, \"entrez_name\"=>\"EGFR\", \"entrez_id\"=>1956, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>19, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>133.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>55086794, \"stop\"=>55279321, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000275493.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/19/summary/variants/190/summary#variant\", \"evidence_items\"=>[{\"id\"=>3015, \"name\"=>\"EID3015\", \"description\"=>\"Evidence of wild-type allele-mediated resistance, a novel concept of acquired resistance in response to mutation-selective inhibitor therapy in cancer treatment. This study reports that the Src-AKT pathway contributes to acquired resistance to third generation EGFR TKIs. In addition, amplification of EGFR wild-type alleles but not mutant alleles was sufficient to confer acquired resistance.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>91, \"name\"=>\"Rociletinib\", \"pubchem_id\"=>nil}, {\"id\"=>187, \"name\"=>\"Osimertinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1733, \"name\"=>\"Amplification of EGFR Wild-Type Alleles in Non-Small Cell Lung Cancer Cells Confers Acquired Resistance to Mutation-Selective EGFR Tyrosine Kinase Inhibitors.\", \"citation\"=>\"Nukaga et al., 2017, Cancer Res.\", \"pubmed_id\"=>\"28202511\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28202511\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>4, \"day\"=>15}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>190, \"phenotypes\"=>[]}, {\"id\"=>691, \"name\"=>\"EID691\", \"description\"=>\"EGFR amplification was not a predictor of response to cetuximab/5-FU/platinum in patients with recurrent or metastatic HNSSC from the EXTREME study.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}, {\"id\"=>103, \"name\"=>\"PLATINUM\", \"pubchem_id\"=>nil}, {\"id\"=>395, \"name\"=>\"5-fluorouracil\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>450, \"name\"=>\"Evaluation of EGFR gene copy number as a predictive biomarker for the efficacy of cetuximab in combination with chemotherapy in the first-line treatment of recurrent and/or metastatic squamous cell carcinoma of the head and neck: EXTREME study.\", \"citation\"=>\"Licitra et al., 2011, Ann. Oncol.\", \"pubmed_id\"=>\"21048039\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21048039\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3082162\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>5}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology / ESMO\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>190, \"phenotypes\"=>[]}, {\"id\"=>977, \"name\"=>\"EID977\", \"description\"=>\"In 290 patients with advanced NSCLC, 177 of which were treated with EGFR TKI (erlotinib or gefitinib). EGFR amplification (Multiplex ligation-dependent probe amplification; MLPA) and Mutation (PCR) were analzed. EGFR amplification was associated with EGFR mutation but not independently with patient benefit from EGFR TKI (overall and progression-free survival).\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>14, \"name\"=>\"Gefitinib\", \"pubchem_id\"=>nil}, {\"id\"=>15, \"name\"=>\"Erlotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>676, \"name\"=>\"Epidermal Growth Factor Receptor Gene Amplification in Patients with Advanced-stage NSCLC.\", \"citation\"=>\"Fiala et al., 2016, Anticancer Res.\", \"pubmed_id\"=>\"26722081\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26722081\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>1}, \"journal\"=>\"Anticancer Res.\", \"full_journal_title\"=>\"Anticancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>190, \"phenotypes\"=>[]}, {\"id\"=>948, \"name\"=>\"EID948\", \"description\"=>\"A meta-analysis of EGFR gene copy number as a predictive biomarker for patients receiving tyrosine kinase inhibitor treatment was performed. In total, 20 (1689 patients, 594 with increased gene copy number), 10 (822 patients, 290 with increased gene copy number) and 5 studies (294 patients, 129 with increased gene copy number) were analyzed for overall survival (OS), progression-free survival (PFS) and time-to-progression (TTP) meta-analyses respectively. EGFR amplification was associated with increased OS (HR = 0.77; 95% CI 0.66-0.89; P = 0.001), PFS (HR = 0.60; 95% CI 0.46-0.79; P<0.001) and TTP (HR = 0.50; 95% CI 0.28-0.91; P = 0.02). This effect was shown to influence survival in predominantly white patient populations but not patients of Asian descent.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>145, \"name\"=>\"EGFR Inhibitor\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>650, \"name\"=>\"EGFR gene copy number as a predictive biomarker for patients receiving tyrosine kinase inhibitor treatment: a systematic review and meta-analysis in non-small-cell lung cancer.\", \"citation\"=>\"Dahabreh et al., 2011, Ann. Oncol.\", \"pubmed_id\"=>\"20826716\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20826716\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>3}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology / ESMO\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>190, \"phenotypes\"=>[]}, {\"id\"=>911, \"name\"=>\"EID911\", \"description\"=>\"87 patients with metastatic colorectal cancer treated with cetuximab were retrospectively analyzed for EGFR amplifications by FISH. In multivariate analysis, EGFR amplification predicted response and overall survival independent of KRAS status.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>618, \"name\"=>\"Clinical usefulness of EGFR gene copy number as a predictive marker in colorectal cancer patients treated with cetuximab: a fluorescent in situ hybridization study.\", \"citation\"=>\"Personeni et al., 2008, Clin. Cancer Res.\", \"pubmed_id\"=>\"18794099\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18794099\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2008, \"month\"=>9, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>190, \"phenotypes\"=>[]}, {\"id\"=>473, \"name\"=>\"EID473\", \"description\"=>\"The group of 43 out of 75 tumor samples with increased copy number or amplification of EGFR as determined by FISH (FISH-positive) had worse progression-free and overall survival than FISH-negative tumors.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>287, \"name\"=>\"Increased epidermal growth factor receptor gene copy number is associated with poor prognosis in head and neck squamous cell carcinomas.\", \"citation\"=>\"Chung et al., 2006, J. Clin. Oncol.\", \"pubmed_id\"=>\"16943533\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16943533\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2006, \"month\"=>9, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>190, \"phenotypes\"=>[]}, {\"id\"=>854, \"name\"=>\"EID854\", \"description\"=>\"In this meta-analysis (14 studies, 1021 patients), EGFR gene copy number (GCN) was predictive of improved progression free survival (PFS) and overall survival (OS) with cetuximab or panitumumab regardless of KRAS status. For the pooled analysis, the objective response rate (ORR) was 65.2% (167/256) in patients with high EFGR GCN, while in patients with low EFGR GCN, the pooled ORR was 12.2% (44/361). The odds ratio (OR) was 6.905 (95% CI: 4.489-10.620; Z=8.79, P=0.000). In wild type KRAS patients, the pooled OR was 8.133 (95% CI: 4.316-15.326; Z=6.48, P=0.000).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}, {\"id\"=>28, \"name\"=>\"Panitumumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>571, \"name\"=>\"EGFR gene copy number as a predictive biomarker for resistance to anti-EGFR monoclonal antibodies in metastatic colorectal cancer treatment: a meta-analysis.\", \"citation\"=>\"Shen et al., 2014, Chin. J. Cancer Res.\", \"pubmed_id\"=>\"24653627\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24653627\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3937753\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>2}, \"journal\"=>\"Chin. J. Cancer Res.\", \"full_journal_title\"=>\"Chinese journal of cancer research = Chung-kuo yen cheng yen chiu\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>190, \"phenotypes\"=>[]}, {\"id\"=>471, \"name\"=>\"EID471\", \"description\"=>\"In patients with cervical squamous cell carcinoma, those with gene amplification of EGFR had worse overall survival.\", \"disease\"=>{\"id\"=>148, \"name\"=>\"Cervical Squamous Cell Carcinoma\", \"display_name\"=>\"Cervical Squamous Cell Carcinoma\", \"doid\"=>\"3744\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3744\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>285, \"name\"=>\"EGFR gene amplification is related to adverse clinical outcomes in cervical squamous cell carcinoma, making the EGFR pathway a novel therapeutic target.\", \"citation\"=>\"Iida et al., 2011, Br. J. Cancer\", \"pubmed_id\"=>\"21730982\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21730982\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3172895\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>7, \"day\"=>26}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>190, \"phenotypes\"=>[]}, {\"id\"=>470, \"name\"=>\"EID470\", \"description\"=>\"In patients with Barrett's adenocarcinoma, those with amplification of EGFR had shorter disease-specific survial.\", \"disease\"=>{\"id\"=>228, \"name\"=>\"Barrett's Adenocarcinoma\", \"display_name\"=>\"Barrett's Adenocarcinoma\", \"doid\"=>\"7941\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:7941\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>284, \"name\"=>\"Homogeneous EGFR amplification defines a subset of aggressive Barrett's adenocarcinomas with poor prognosis.\", \"citation\"=>\"Marx et al., 2010, Histopathology\", \"pubmed_id\"=>\"20840671\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20840671\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>9}, \"journal\"=>\"Histopathology\", \"full_journal_title\"=>\"Histopathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>190, \"phenotypes\"=>[]}]}, {\"id\"=>204, \"entrez_name\"=>\"NCOA3\", \"entrez_id\"=>8202, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>74, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"20\", \"start\"=>46130646, \"stop\"=>46285621, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000372004.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/74/summary/variants/204/summary#variant\", \"evidence_items\"=>[{\"id\"=>492, \"name\"=>\"EID492\", \"description\"=>\"In patients with breast cancer, those with amplification of NCOA3 had shorter disease-specific survival.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>300, \"name\"=>\"Prognostic relevance of AIB1 (NCoA3) amplification and overexpression in breast cancer.\", \"citation\"=>\"Burandt et al., 2013, Breast Cancer Res. Treat.\", \"pubmed_id\"=>\"23322234\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23322234\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>2}, \"journal\"=>\"Breast Cancer Res. Treat.\", \"full_journal_title\"=>\"Breast cancer research and treatment\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>204, \"phenotypes\"=>[]}]}, {\"id\"=>212, \"entrez_name\"=>\"PIK3CA\", \"entrez_id\"=>5290, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>37, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>24.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>178866311, \"stop\"=>178957881, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000263967.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/37/summary/variants/212/summary#variant\", \"evidence_items\"=>[{\"id\"=>1403, \"name\"=>\"EID1403\", \"description\"=>\"474 cancer cell lines from the cancer cell line encyclopedia were assessed for response to the PI3Kalpha inhibitor BYL719 (Alpelisib). PIK3CA amplification was significanly associated with BYL719 response. A gastric cancer patient derived xenograft (PDX) was transplanted into two athymic mice and one was treated with BYL719. The untreated PDX grew by over 100 percent while the BYL719 tumor was reduced by 86 percent.\", \"disease\"=>{\"id\"=>1322, \"name\"=>\"Stomach Carcinoma\", \"display_name\"=>\"Stomach Carcinoma\", \"doid\"=>\"5517\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5517\"}, \"drugs\"=>[{\"id\"=>339, \"name\"=>\"BYL719 (Alpelisib)\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>950, \"name\"=>\"Characterization of the novel and specific PI3Kα inhibitor NVP-BYL719 and development of the patient stratification strategy for clinical trials.\", \"citation\"=>\"Fritsch et al., 2014, Mol. Cancer Ther.\", \"pubmed_id\"=>\"24608574\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24608574\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>5}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>212, \"phenotypes\"=>[]}, {\"id\"=>1464, \"name\"=>\"EID1464\", \"description\"=>\"The HNSCC cell line LB771 with PIK3CA amplification was shown to have low antiproliferative IC50 for the p110beta-sparing PI3K inhibitor taselisib. Under increasing taselisib concentrations, reduction in phospho-Akt pro-survival signal was apparent, along with increased levels of pro-apoptotic marker cleaved PARP. A general tendency for taselisib sensitivity to PI3K activation (mutation/amplification) was also seen in a panel of 26 HNSCC cell lines\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>351, \"name\"=>\"Taselisib (GDC-0032)\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>984, \"name\"=>\"Taselisib (GDC-0032), a Potent β-Sparing Small Molecule Inhibitor of PI3K, Radiosensitizes Head and Neck Squamous Carcinomas Containing Activating PIK3CA Alterations.\", \"citation\"=>\"Zumsteg et al., 2016, Clin. Cancer Res.\", \"pubmed_id\"=>\"26589432\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26589432\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4870591\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>212, \"phenotypes\"=>[]}, {\"id\"=>756, \"name\"=>\"EID756\", \"description\"=>\"One platinum-refractory epithelial ovarian cancer patient with PTEN loss and PIK3CA amplification had a partial response on treatment with PI3K inhibitor pictilisib (GDC-0941) in this phase 1 study. A total of 60 patients with advanced solid tumors were enrolled in this study.\", \"disease\"=>{\"id\"=>225, \"name\"=>\"Epithelial Ovarian Cancer\", \"display_name\"=>\"Epithelial Ovarian Cancer\", \"doid\"=>\"2152\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2152\"}, \"drugs\"=>[{\"id\"=>141, \"name\"=>\"Pictilisib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>500, \"name\"=>\"First-in-human phase I study of pictilisib (GDC-0941), a potent pan-class I phosphatidylinositol-3-kinase (PI3K) inhibitor, in patients with advanced solid tumors.\", \"citation\"=>\"Sarker et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"25370471\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25370471\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4287394\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>1, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>212, \"phenotypes\"=>[]}, {\"id\"=>504, \"name\"=>\"EID504\", \"description\"=>\"In patients with gastric cancer, those with amplification of PIK3CA had shorter overall survival than those without amplification.\", \"disease\"=>{\"id\"=>147, \"name\"=>\"Gastric Adenocarcinoma\", \"display_name\"=>\"Gastric Adenocarcinoma\", \"doid\"=>\"3717\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3717\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>309, \"name\"=>\"Highly frequent PIK3CA amplification is associated with poor prognosis in gastric cancer.\", \"citation\"=>\"Shi et al., 2012, BMC Cancer\", \"pubmed_id\"=>\"22292935\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22292935\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3299648\", \"publication_date\"=>{\"year\"=>2012}, \"journal\"=>\"BMC Cancer\", \"full_journal_title\"=>\"BMC cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>212, \"phenotypes\"=>[]}]}, {\"id\"=>219, \"entrez_name\"=>\"TERT\", \"entrez_id\"=>7015, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>79, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"5\", \"start\"=>1253282, \"stop\"=>1295162, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000310581.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/79/summary/variants/219/summary#variant\", \"evidence_items\"=>[{\"id\"=>514, \"name\"=>\"EID514\", \"description\"=>\"In patients with acral lentiginous melanoma, those with amplification of TERT had shorter overall survival.\", \"disease\"=>{\"id\"=>192, \"name\"=>\"Acral Lentiginous Melanoma\", \"display_name\"=>\"Acral Lentiginous Melanoma\", \"doid\"=>\"6367\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:6367\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>317, \"name\"=>\"TERT gene amplification is associated with poor outcome in acral lentiginous melanoma.\", \"citation\"=>\"Diaz et al., 2014, J. Am. Acad. Dermatol.\", \"pubmed_id\"=>\"25219713\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25219713\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>10}, \"journal\"=>\"J. Am. Acad. Dermatol.\", \"full_journal_title\"=>\"Journal of the American Academy of Dermatology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>219, \"phenotypes\"=>[]}]}, {\"id\"=>224, \"entrez_name\"=>\"TTF1\", \"entrez_id\"=>7270, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>80, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>135251008, \"stop\"=>135282209, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000334270.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/80/summary/variants/224/summary#variant\", \"evidence_items\"=>[{\"id\"=>527, \"name\"=>\"EID527\", \"description\"=>\"6 of 89 patients with lung adenocarcinoma had amplification of TTF1. There was no statistical difference in overall survival between patients with and without gene amplification.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"N/A\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>327, \"name\"=>\"Clinical significance of TTF-1 protein expression and TTF-1 gene amplification in lung adenocarcinoma.\", \"citation\"=>\"Barletta et al., 2009, J. Cell. Mol. Med.\", \"pubmed_id\"=>\"19040416\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19040416\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2830395\", \"publication_date\"=>{\"year\"=>2009, \"month\"=>8}, \"journal\"=>\"J. Cell. Mol. Med.\", \"full_journal_title\"=>\"Journal of cellular and molecular medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>224, \"phenotypes\"=>[]}]}, {\"id\"=>251, \"entrez_name\"=>\"BIRC7\", \"entrez_id\"=>79444, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>14842, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>4.0, \"coordinates\"=>{\"chromosome\"=>\"20\", \"start\"=>61867235, \"stop\"=>61871859, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000217169.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/14842/summary/variants/251/summary#variant\", \"evidence_items\"=>[{\"id\"=>661, \"name\"=>\"EID661\", \"description\"=>\"Livin (BIRC7) overexpression in two colon cancer cell lines (RKO and SW620) increased resistance of these cells to cisplatin. siRNA knockdown of Livin rendered the cells more sensitive to cisplatin. The study also further showed that cytoplasmic Livin levels increased in four colon cancer cell lines (RKO, SW620, HCT116, and KM12C) after cisplatin treatment in a dose-dependent manner. This study highlighted Livin as a potential player in colon cancer therapy.\", \"disease\"=>{\"id\"=>119, \"name\"=>\"Colon Cancer\", \"display_name\"=>\"Colon Cancer\", \"doid\"=>\"219\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:219\"}, \"drugs\"=>[{\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>418, \"name\"=>\"Upregulation of the antiapoptotic factor Livin contributes to cisplatin resistance in colon cancer cells.\", \"citation\"=>\"Ding et al., 2013, Tumour Biol.\", \"pubmed_id\"=>\"23188704\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23188704\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>4}, \"journal\"=>\"Tumour Biol.\", \"full_journal_title\"=>\"Tumour biology : the journal of the International Society for Oncodevelopmental Biology and Medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>251, \"phenotypes\"=>[]}]}, {\"id\"=>267, \"entrez_name\"=>\"FGFR1\", \"entrez_id\"=>2260, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"FGFR1 amplifications are the most common aberrations of the FGFR family in cancer. One study reports FGFR1 aberrations in 3.5% of 4,853 patients of which most are FGFR1 amplifications (Helsten et. al., 2016). Preclinically, FGFR1 amplifications have also been shown to confer sensitivity to FGFR1 inhibitors (Weiss et. al., 2010). Clinical studies are currently ongoing (e.g. NCT01004224).\", \"gene_id\"=>1885, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>158.0, \"coordinates\"=>{\"chromosome\"=>\"8\", \"start\"=>38268656, \"stop\"=>38325363, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000425967.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1885/summary/variants/267/summary#variant\", \"evidence_items\"=>[{\"id\"=>1908, \"name\"=>\"EID1908\", \"description\"=>\"Phase 1 study of FGFR-inhibitor BGJ398 in advanced cancer with FGFR-aberrations. In the subgroup of FGFR1-amplified NSCLC, 36 patients were treated at doses >100mg and disease control was achieved in 50% of patients (4 PR and 14 SD) with 6 patients receiving treatment after 16 weeks.\", \"disease\"=>{\"id\"=>153, \"name\"=>\"Lung Squamous Cell Carcinoma\", \"display_name\"=>\"Lung Squamous Cell Carcinoma\", \"doid\"=>\"3907\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3907\"}, \"drugs\"=>[{\"id\"=>151, \"name\"=>\"BGJ398\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1337, \"name\"=>\"Evaluation of BGJ398, a Fibroblast Growth Factor Receptor 1-3 Kinase Inhibitor, in Patients With Advanced Solid Tumors Harboring Genetic Alterations in Fibroblast Growth Factor Receptors: Results of a Global Phase I, Dose-Escalation and Dose-Expansion Study.\", \"citation\"=>\"Nogova et al., 2017, J. Clin. Oncol.\", \"pubmed_id\"=>\"27870574\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27870574\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>1, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>267, \"phenotypes\"=>[]}, {\"id\"=>3027, \"name\"=>\"EID3027\", \"description\"=>\"Clinical trial studied the efficacy of Dovitinib on 26 patients with FGFR1 amplifications. Objective response for the cohort was 11.5%, disease control rate was 50%. 11.5% of patients exhibited partial response, 38.5% exhibited stable disease. Median PFS was 2.9 months, median OS was 5 months, this trend was worse with increases in FGFR1 amplification above average.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>228, \"name\"=>\"Dovitinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1342, \"name\"=>\"Efficacy and safety of dovitinib in pretreated patients with advanced squamous non-small cell lung cancer with FGFR1 amplification: A single-arm, phase 2 study.\", \"citation\"=>\"Lim et al., 2016, Cancer\", \"pubmed_id\"=>\"27315356\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27315356\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>10}, \"journal\"=>\"Cancer\", \"full_journal_title\"=>\"Cancer\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>267, \"phenotypes\"=>[]}, {\"id\"=>1909, \"name\"=>\"EID1909\", \"description\"=>\"Phase 1 study of FGFR-inhibitor BGJ398 in advanced cancer with FGFR-aberrations. Among the 26 patients with breast cancer and FGFR1/2 amplification (n=25,  n= 21 with FGFR1 amplification only) or FGFR3 mutation (n=1) four (15.4%) had a reduced tumor burden. In total, 10 of 32 patients with breast cancer treated at doses >100mg had a best response of SD.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>433, \"name\"=>\"BGJ-398\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>nil, \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1337, \"name\"=>\"Evaluation of BGJ398, a Fibroblast Growth Factor Receptor 1-3 Kinase Inhibitor, in Patients With Advanced Solid Tumors Harboring Genetic Alterations in Fibroblast Growth Factor Receptors: Results of a Global Phase I, Dose-Escalation and Dose-Expansion Study.\", \"citation\"=>\"Nogova et al., 2017, J. Clin. Oncol.\", \"pubmed_id\"=>\"27870574\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27870574\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>1, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>267, \"phenotypes\"=>[]}, {\"id\"=>1911, \"name\"=>\"EID1911\", \"description\"=>\"Phase 1 study of FGFR-inhibitor BGJ398 in advanced cancer with FGFR-aberrations. One patient with FGFR1 amplified bladder/urothelial carcinoma was treated with 125 mg and progressed rapidly in cycle 1 after 11 days of treatment.\", \"disease\"=>{\"id\"=>18, \"name\"=>\"Bladder Carcinoma\", \"display_name\"=>\"Bladder Carcinoma\", \"doid\"=>\"4007\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4007\"}, \"drugs\"=>[{\"id\"=>433, \"name\"=>\"BGJ-398\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1337, \"name\"=>\"Evaluation of BGJ398, a Fibroblast Growth Factor Receptor 1-3 Kinase Inhibitor, in Patients With Advanced Solid Tumors Harboring Genetic Alterations in Fibroblast Growth Factor Receptors: Results of a Global Phase I, Dose-Escalation and Dose-Expansion Study.\", \"citation\"=>\"Nogova et al., 2017, J. Clin. Oncol.\", \"pubmed_id\"=>\"27870574\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27870574\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>1, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>267, \"phenotypes\"=>[]}, {\"id\"=>1991, \"name\"=>\"EID1991\", \"description\"=>\"Activity of the FGFR-selective inhibitor AZD4547 in FGFR1-amplified breast cancer and FGFR2-amplified gastroesophageal cancer was analyzed in this study. 341 patients with advanced cancer were screened by FISH, identifying FGFR1 amplification in 18% of advanced HER2-negative breast cancers and FGFR2 amplification in 9% of advanced gastroesophageal cancer. 1 patient had a confirmed response among 8 patients treated with FGFR1-amplified breast cancer. In a preclinical model, FGFR1 amplification in breast cancer cell lines was less pronounced than FGFR2-amplifications in gastroesophageal cancers, which also had a higher response rate in the trial, indicating that high level amplification is associated with response and less common in FGFR1-amplified breast cancer.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>452, \"name\"=>\"AZD4547\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1397, \"name\"=>\"High-Level Clonal FGFR Amplification and Response to FGFR Inhibition in a Translational Clinical Trial.\", \"citation\"=>\"Pearson et al., 2016, Cancer Discov\", \"pubmed_id\"=>\"27179038\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27179038\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5338732\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>8}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>267, \"phenotypes\"=>[]}, {\"id\"=>1604, \"name\"=>\"EID1604\", \"description\"=>\"Dovitinib was evaluated in-vitro, in xenograft and in a phase 2 study.\\nDovitinib inhibited proliferation in one FGFR1- and one FGFR2-amplified breast cancer cell lines at higher IC50 values than in 11 nonamplified cell lines. Dovitinib inhibited tumor growth and caused regression in both a FGFR1- and FGFR2-amplified xenograft model.\\n81 patients were enrolled in the trial and treated with dovitinib. Unconfirmed response or stable disease >6 months was observed in 25% of FGFR1-amplified/HR-positive vs. 3% FGFR1-nonamplified/HR-positive breast cancer patientes. When FGF3 and FGFR2 amplifications were also assessed, a combined FGF-pathway amplified patient group had a higher mean reduction in target lesion size than patients with FGF-non-amplified tumors.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>228, \"name\"=>\"Dovitinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1064, \"name\"=>\"Targeting FGFR with dovitinib (TKI258): preclinical and clinical data in breast cancer.\", \"citation\"=>\"André et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"23658459\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23658459\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>267, \"phenotypes\"=>[]}, {\"id\"=>1767, \"name\"=>\"EID1767\", \"description\"=>\"In MDA-MB-134 cells (ER-positive breast cancer cells with amplified FGFR1) ponatinib potently inhibited cell growth with a GI50 value of 23 nmol/L and inhibited FGFR1 phosphorylation with an IC50 value of 7 nmol/L.\", \"disease\"=>{\"id\"=>28, \"name\"=>\"Estrogen-receptor Positive Breast Cancer\", \"display_name\"=>\"Estrogen-receptor Positive Breast Cancer\", \"doid\"=>\"0060075\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060075\"}, \"drugs\"=>[{\"id\"=>55, \"name\"=>\"Ponatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1233, \"name\"=>\"Ponatinib (AP24534), a multitargeted pan-FGFR inhibitor with activity in multiple FGFR-amplified or mutated cancer models.\", \"citation\"=>\"Gozgit et al., 2012, Mol. Cancer Ther.\", \"pubmed_id\"=>\"22238366\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22238366\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>3}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>267, \"phenotypes\"=>[]}, {\"id\"=>1247, \"name\"=>\"EID1247\", \"description\"=>\"116 Ewing's sarcoma tumor samples were analyzed in this study. 2 patient's samples underwent whole genome sequencing, tumor samples from 51 patients underwent exome sequencing and expression analysis, 63 samples underwent FGFR1 copy number analysis, mRNA analysis and IHC. A FGFR1 N546K mutation was detected in one patient. Gain of the FGFR1 locus (qPCR) was observed in 13 of 41 patients (31.7%) and Ewing sarcoma cell lines (A673, CADO, SK-NMC, SK-ES, and VH-64). FGFR1 was also highly expressed in Ewing sarcoma (mRNA and IHC) and correlated with copy number. Retroviral transduction of FGFR1 wild-type or the FGFR1 N546K mutant into NIH3T3 cells led to enhanced proliferation especially for cells harboring the FGFR1 mutation. ShRNA mediated knockdown of FGFR1 inhibited growth of Ewing sarcoma and other cancer cells (A549 lung carcinoma cell line or leukemia cell lines HL60 and Kasumi) with almost complete growth suppression only in the Ewing sarcoma cell lines and knockdown of FGFR1 abrogated growth of cell lines in xenograft. Ponatinib treatment inhibited growth of Ewing sarcoma cell lines.\", \"disease\"=>{\"id\"=>135, \"name\"=>\"Bone Ewing's Sarcoma\", \"display_name\"=>\"Bone Ewing's Sarcoma\", \"doid\"=>\"3368\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3368\"}, \"drugs\"=>[{\"id\"=>55, \"name\"=>\"Ponatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>446, \"name\"=>\"Deep Sequencing in Conjunction with Expression and Functional Analyses Reveals Activation of FGFR1 in Ewing Sarcoma.\", \"citation\"=>\"Agelopoulos et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"26179511\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26179511\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>11, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>267, \"phenotypes\"=>[]}, {\"id\"=>682, \"name\"=>\"EID682\", \"description\"=>\"In this study in surgically resected squamous cell lung cancer, FGFR1 amplification has been shown as an independent negative prognostic factor in Asian patients. FGFR1 amplification also correlates with cigarette smoking.\", \"disease\"=>{\"id\"=>153, \"name\"=>\"Lung Squamous Cell Carcinoma\", \"display_name\"=>\"Lung Squamous Cell Carcinoma\", \"doid\"=>\"3907\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3907\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>436, \"name\"=>\"Fibroblast growth factor receptor 1 gene amplification is associated with poor survival and cigarette smoking dosage in patients with resected squamous cell lung cancer.\", \"citation\"=>\"Kim et al., 2013, J. Clin. Oncol.\", \"pubmed_id\"=>\"23182986\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23182986\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>2, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>267, \"phenotypes\"=>[]}, {\"id\"=>684, \"name\"=>\"EID684\", \"description\"=>\"This study shows a reduction in the risk of death for patients with FGFR1 copy-number between 4 and 6. No difference in survival is seen between amplified and non-amplified patients in general.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>438, \"name\"=>\"Fibroblast growth factor receptor 1 (FGFR1) copy number is an independent prognostic factor in non-small cell lung cancer.\", \"citation\"=>\"Tran et al., 2013, Lung Cancer\", \"pubmed_id\"=>\"23806793\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23806793\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>9}, \"journal\"=>\"Lung Cancer\", \"full_journal_title\"=>\"Lung cancer (Amsterdam, Netherlands)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>267, \"phenotypes\"=>[]}, {\"id\"=>685, \"name\"=>\"EID685\", \"description\"=>\"Meta-Analysis on prognostic value of FGFR1 amplification in different types of cancer shows significantly worse overall survival for patients with FGFR1 and FGFR2 amplification.\\nHowever, larger studies are warranted for confirmation.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>439, \"name\"=>\"Prognostic value of FGFR gene amplification in patients with different types of cancer: a systematic review and meta-analysis.\", \"citation\"=>\"Chang et al., 2014, PLoS ONE\", \"pubmed_id\"=>\"25171497\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25171497\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4149366\", \"publication_date\"=>{\"year\"=>2014}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>267, \"phenotypes\"=>[]}, {\"id\"=>687, \"name\"=>\"EID687\", \"description\"=>\"Effect of FGFR1 amplification on survival unclear, no significant correlation with PFS and OS seen in this meta-analysis. However, FGFR1 amplification is significantly correlated with lymph node metastasis.\", \"disease\"=>{\"id\"=>153, \"name\"=>\"Lung Squamous Cell Carcinoma\", \"display_name\"=>\"Lung Squamous Cell Carcinoma\", \"doid\"=>\"3907\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3907\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>441, \"name\"=>\"FGFR1 amplification in lung squamous cell carcinoma: a systematic review with meta-analysis.\", \"citation\"=>\"Jiang et al., 2015, Lung Cancer\", \"pubmed_id\"=>\"25433983\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25433983\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>1}, \"journal\"=>\"Lung Cancer\", \"full_journal_title\"=>\"Lung cancer (Amsterdam, Netherlands)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>267, \"phenotypes\"=>[]}, {\"id\"=>683, \"name\"=>\"EID683\", \"description\"=>\"This study shows an association of FGFR1 amplification with shorter overall and disease-free survival. However, no statistically significant association is seen on multivariate analysis.\", \"disease\"=>{\"id\"=>153, \"name\"=>\"Lung Squamous Cell Carcinoma\", \"display_name\"=>\"Lung Squamous Cell Carcinoma\", \"doid\"=>\"3907\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3907\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>437, \"name\"=>\"FGFR1 amplification is associated with poor prognosis and smoking in non-small-cell lung cancer.\", \"citation\"=>\"Seo et al., 2014, Virchows Arch.\", \"pubmed_id\"=>\"25086725\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25086725\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>11}, \"journal\"=>\"Virchows Arch.\", \"full_journal_title\"=>\"Virchows Archiv : an international journal of pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>267, \"phenotypes\"=>[]}, {\"id\"=>680, \"name\"=>\"EID680\", \"description\"=>\"Preclinical (in-vitro, in-vivo) evidence suggests predictive value of FGFR1 amplification for treatment with FGFR1-inhibitors. 4 of 83 cell lines exhibited FGFR1 amplification, and 3 of these were highly sensitive to PD173074. This sensitivity was confirmed in a mouse xenograft. Clinical trials are currently ongoing.\", \"disease\"=>{\"id\"=>153, \"name\"=>\"Lung Squamous Cell Carcinoma\", \"display_name\"=>\"Lung Squamous Cell Carcinoma\", \"doid\"=>\"3907\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3907\"}, \"drugs\"=>[{\"id\"=>36, \"name\"=>\"PD173074\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>434, \"name\"=>\"Frequent and focal FGFR1 amplification associates with therapeutically tractable FGFR1 dependency in squamous cell lung cancer.\", \"citation\"=>\"Weiss et al., 2010, Sci Transl Med\", \"pubmed_id\"=>\"21160078\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21160078\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3990281\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>12, \"day\"=>15}, \"journal\"=>\"Sci Transl Med\", \"full_journal_title\"=>\"Science translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>267, \"phenotypes\"=>[]}, {\"id\"=>1105, \"name\"=>\"EID1105\", \"description\"=>\"FGFR1-amplified breast cancer cell lines show resistance to 4-hydroxytamoxifen, which is reversed by small interfering RNA silencing of FGFR1.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>277, \"name\"=>\"4-hydroxytamoxifen\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>763, \"name\"=>\"FGFR1 amplification drives endocrine therapy resistance and is a therapeutic target in breast cancer.\", \"citation\"=>\"Turner et al., 2010, Cancer Res.\", \"pubmed_id\"=>\"20179196\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20179196\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2832818\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>3, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>267, \"phenotypes\"=>[]}, {\"id\"=>686, \"name\"=>\"EID686\", \"description\"=>\"Preclinical study showing FGFR1 amplification in NSCLC and efficacy of FGFR-1 inhibition in one amplified cell line.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>36, \"name\"=>\"PD173074\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>440, \"name\"=>\"Inhibitor-sensitive FGFR1 amplification in human non-small cell lung cancer.\", \"citation\"=>\"Dutt et al., 2011, PLoS ONE\", \"pubmed_id\"=>\"21666749\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21666749\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3110189\", \"publication_date\"=>{\"year\"=>2011}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>267, \"phenotypes\"=>[]}]}, {\"id\"=>270, \"entrez_name\"=>\"MET\", \"entrez_id\"=>4233, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>52, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>81.5, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>116312459, \"stop\"=>116436396, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000318493.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/52/summary/variants/270/summary#variant\", \"evidence_items\"=>[{\"id\"=>1971, \"name\"=>\"EID1971\", \"description\"=>\"Functional studies show that MET activation confers resistance to anti-EGFR therapy both in vitro and in vivo. Notably, in patient-derived colorectal cancer xenografts, MET amplification correlated with resistance to EGFR blockade, which could be overcome by MET kinase inhibitors.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>252, \"name\"=>\"Anti-EGFR Monoclonal Antibody\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1386, \"name\"=>\"Amplification of the MET receptor drives resistance to anti-EGFR therapies in colorectal cancer.\", \"citation\"=>\"Bardelli et al., 2013, Cancer Discov\", \"pubmed_id\"=>\"23729478\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23729478\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4078408\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>270, \"phenotypes\"=>[]}, {\"id\"=>1714, \"name\"=>\"EID1714\", \"description\"=>\"Case report of a 59-year-old white female patient (60 pack-years) with a CUP-syndrome. Molecular profiling identified both MET amplification (16 copies) and a KRAS G12V mutation. The patient was treated with crizotinib and experienced a complete normalization of tumor metabolic activity for more than 19 months. Other identified aberrations include CCND1 amplification (8 copies), MYC amplification (9 copies) as well as KRAS G12V, TP53 R273L and CARD11 N184S.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1158, \"name\"=>\"Durable Response to Crizotinib in a MET-Amplified, KRAS-Mutated Carcinoma of Unknown Primary.\", \"citation\"=>\"Palma et al., 2014, Case Rep Oncol\", \"pubmed_id\"=>\"25232318\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25232318\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4164090\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>5}, \"journal\"=>\"Case Rep Oncol\", \"full_journal_title\"=>\"Case reports in oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>270, \"phenotypes\"=>[]}, {\"id\"=>1692, \"name\"=>\"EID1692\", \"description\"=>\"In an evaluation of a patient with crizotinib-sensitive lung adenocarcinoma, an MET-Amplification was identified using CGP.  The patient attained a reduction in the target lesions by 38.7%, and the patient maintained response for at least 5 months post-treatment.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1141, \"name\"=>\"Detection of Crizotinib-Sensitive Lung Adenocarcinomas With MET, ALK, and ROS1 Genomic Alterations via Comprehensive Genomic Profiling.\", \"citation\"=>\"Le et al., 2015, Clin Lung Cancer\", \"pubmed_id\"=>\"25922291\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25922291\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4418215\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>9}, \"journal\"=>\"Clin Lung Cancer\", \"full_journal_title\"=>\"Clinical lung cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>270, \"phenotypes\"=>[]}, {\"id\"=>1713, \"name\"=>\"EID1713\", \"description\"=>\"Case report of two patients with MET amplification (MET:CEP7 ratio ≥5) and no concomitant MET exon 14 deletion. A response to crizotinib was observed in both cases suggesting that MET amplification may predict response to this drug even in the absence of MET exon 14 mutations.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1157, \"name\"=>\"Responses to Crizotinib Can Occur in High-Level MET-Amplified Non-Small Cell Lung Cancer Independent of MET Exon 14 Alterations.\", \"citation\"=>\"Caparica et al., 2017, J Thorac Oncol\", \"pubmed_id\"=>\"27664533\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27664533\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>1}, \"journal\"=>\"J Thorac Oncol\", \"full_journal_title\"=>\"Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>270, \"phenotypes\"=>[]}, {\"id\"=>1392, \"name\"=>\"EID1392\", \"description\"=>\"155 patients with lung adenocarcinomas underwent rebiopsy and genotyping (EGFR, AKT1, BRAF, ERBB2, KRAS, MEK1, NRAS, PIK3CA and FISH for MET and HER2) after the development of acquired resistance to EGFR inhibitors (gefitinib or erlotinib). MET amplification was seen in 4 of 75 patients tested (no baseline samples available). One patient had small cell transformation and MET amplification, two patients had EGFR T790M mutation and MET amplification.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>14, \"name\"=>\"Gefitinib\", \"pubchem_id\"=>nil}, {\"id\"=>15, \"name\"=>\"Erlotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>943, \"name\"=>\"Analysis of tumor specimens at the time of acquired resistance to EGFR-TKI therapy in 155 patients with EGFR-mutant lung cancers.\", \"citation\"=>\"Yu et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"23470965\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23470965\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3630270\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>4, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>270, \"phenotypes\"=>[]}, {\"id\"=>1588, \"name\"=>\"EID1588\", \"description\"=>\"Case report of a patient with BRAF V600E mutant metastatic colorectal cancer. Combined EGFR and BRAF inhibition (panitumumab and vemurafenib) showed an initial partial response for 4 months. Re-biopsy at the time of progression showed a MET amplification and overexpression (IHC, ISH) but no secondary mutations in candidate resistance genes (EGFR, KRAS, NRAS, MAP2K1). MET amplification was also present in a subset of tumor cells in the pretreatment sample. MET amplification was also identified in 3/17 (18%) chemonaive BRAF V600E colorectal cancer tumor tissues.\\nOverexpression of MET in a BRAF mutant CRC cell line (WiDr) confered resistance to combined vemurafenib and panitumumab. The addition of crizotinib to this combination restored sensitivity in vitro.\\nThe patient was started on combined vemurafenib and crizotinib in January 2016 and had a metabolic, clinical and tumor marker response that was ongoing at the time of publication (06/16) without significant toxicity.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>4, \"name\"=>\"Vemurafenib\", \"pubchem_id\"=>nil}, {\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1050, \"name\"=>\"MET-Driven Resistance to Dual EGFR and BRAF Blockade May Be Overcome by Switching from EGFR to MET Inhibition in BRAF-Mutated Colorectal Cancer.\", \"citation\"=>\"Pietrantonio et al., 2016, Cancer Discov\", \"pubmed_id\"=>\"27325282\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27325282\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>6, \"day\"=>20}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>270, \"phenotypes\"=>[]}, {\"id\"=>1566, \"name\"=>\"EID1566\", \"description\"=>\"2 MET amplified NSCLC cell lines (EBC-1 and H1993), as well as 2 EGFR mutant and 2 MET and EGFR wild type cell lines were treated with ALK and MET inhibitor crizotinib, and the MET amplified cells had IC50 100 fold lower than the non MET amplified cell lines. Both crizotinib treatment and siRNA against MET increased apoptotic markers in MET amplified cells but not controls, indicating that crizotinib targeting of amplified MET was inducing the apoptotic effects. Tumors caused by MET amplified EBC-1 cells injected into mice were significantly inhibited with crizotinib treatment, while tumors from non-MET amplified cells showed no substantial effect.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1035, \"name\"=>\"MET tyrosine kinase inhibitor crizotinib (PF-02341066) shows differential antitumor effects in non-small cell lung cancer according to MET alterations.\", \"citation\"=>\"Tanizaki et al., 2011, J Thorac Oncol\", \"pubmed_id\"=>\"21716144\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21716144\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>10}, \"journal\"=>\"J Thorac Oncol\", \"full_journal_title\"=>\"Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>270, \"phenotypes\"=>[]}, {\"id\"=>1979, \"name\"=>\"EID1979\", \"description\"=>\"Functional studies show that MET activation confers resistance to anti-EGFR therapy both in vitro and in vivo. Notably, in patient-derived colorectal cancer xenografts, MET amplification correlated with resistance to EGFR blockade, which could be overcome by MET kinase inhibitors.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}, {\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1386, \"name\"=>\"Amplification of the MET receptor drives resistance to anti-EGFR therapies in colorectal cancer.\", \"citation\"=>\"Bardelli et al., 2013, Cancer Discov\", \"pubmed_id\"=>\"23729478\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23729478\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4078408\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>270, \"phenotypes\"=>[]}, {\"id\"=>1565, \"name\"=>\"EID1565\", \"description\"=>\"A 62 year old female was diagnosed with glioblastoma (GBM) WHO grade 4. Treatment included radiation, temozolomide and cediranib. MET amplification was found by FISH, and ALK and MET inhibitor crizotinib treatment was started. A 40% reduction in size of contrast enhancing lesion was seen. After 6 months, progressive disease was seen and crizotinib withdrawn. The authors state this is a first report of response to prospective MET targeted therapy in GBM.\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1034, \"name\"=>\"Rapid radiographic and clinical improvement after treatment of a MET-amplified recurrent glioblastoma with a mesenchymal-epithelial transition inhibitor.\", \"citation\"=>\"Chi et al., 2012, J. Clin. Oncol.\", \"pubmed_id\"=>\"22162573\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22162573\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>270, \"phenotypes\"=>[]}, {\"id\"=>890, \"name\"=>\"EID890\", \"description\"=>\"Case report of a 73 year old patient with advanced squamous cell carcinoma of the lung that tested negative for EGFR, KRAS and PIK3CA mutations and ALK and ROS1 rearrangements. c-MET amplification was detected by FISH. Crizotinib led to a partial response.\", \"disease\"=>{\"id\"=>153, \"name\"=>\"Lung Squamous Cell Carcinoma\", \"display_name\"=>\"Lung Squamous Cell Carcinoma\", \"doid\"=>\"3907\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3907\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>600, \"name\"=>\"Major partial response to crizotinib, a dual MET/ALK inhibitor, in a squamous cell lung (SCC) carcinoma patient with de novo c-MET amplification in the absence of ALK rearrangement.\", \"citation\"=>\"Schwab et al., 2014, Lung Cancer\", \"pubmed_id\"=>\"24192513\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24192513\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>1}, \"journal\"=>\"Lung Cancer\", \"full_journal_title\"=>\"Lung cancer (Amsterdam, Netherlands)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>270, \"phenotypes\"=>[]}, {\"id\"=>836, \"name\"=>\"EID836\", \"description\"=>\"Case report of a patient with MET amplification without high-level MET/CEP7 ratio, MET exon 14 mutation or overexpression who achieved rapid response to crizotinib.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>564, \"name\"=>\"Response to Crizotinib Observed in Lung Adenocarcinoma with MET Copy Number Gain but without a High-Level MET/CEP7 Ratio, MET Overexpression, or Exon 14 Splicing Mutations.\", \"citation\"=>\"Zhang et al., 2016, J Thorac Oncol\", \"pubmed_id\"=>\"26724472\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26724472\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>5}, \"journal\"=>\"J Thorac Oncol\", \"full_journal_title\"=>\"Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>270, \"phenotypes\"=>[]}, {\"id\"=>733, \"name\"=>\"EID733\", \"description\"=>\"In this study a gefitinib-sensitive cell-line (HCC827), acquired gefitinib resistance (HCC827 GR) by a focal amplification of MET.  Resistance was conferred by the MET amplification driving ERBB3 (HER3)-dependent activation of PI3K.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>14, \"name\"=>\"Gefitinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>481, \"name\"=>\"MET amplification leads to gefitinib resistance in lung cancer by activating ERBB3 signaling.\", \"citation\"=>\"Engelman et al., 2007, Science\", \"pubmed_id\"=>\"17463250\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17463250\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>5, \"day\"=>18}, \"journal\"=>\"Science\", \"full_journal_title\"=>\"Science (New York, N.Y.)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>270, \"phenotypes\"=>[]}, {\"id\"=>735, \"name\"=>\"EID735\", \"description\"=>\"A case-study of an ALK-negative patient with MET amplification (MET/CEP7 ratio >5.0) who achieved a durable partial response to MET/ALK-inhibitor crizotinib.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>483, \"name\"=>\"Activity of crizotinib (PF02341066), a dual mesenchymal-epithelial transition (MET) and anaplastic lymphoma kinase (ALK) inhibitor, in a non-small cell lung cancer patient with de novo MET amplification.\", \"citation\"=>\"Ou et al., 2011, J Thorac Oncol\", \"pubmed_id\"=>\"21623265\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21623265\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>5}, \"journal\"=>\"J Thorac Oncol\", \"full_journal_title\"=>\"Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>270, \"phenotypes\"=>[]}, {\"id\"=>689, \"name\"=>\"EID689\", \"description\"=>\"A case-study describing a patient with MET polysomy, MET over-expression and evidence of autocrine HGF-production (the growth factor ligand of MET). The patient initially achieved a complete response on MET-Inhibitor MetMAb (Onartuzumab) for 2 years and mixed response after recurrence with resistance in some of the newly developing metastases.\", \"disease\"=>{\"id\"=>147, \"name\"=>\"Gastric Adenocarcinoma\", \"display_name\"=>\"Gastric Adenocarcinoma\", \"doid\"=>\"3717\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3717\"}, \"drugs\"=>[{\"id\"=>121, \"name\"=>\"Onartuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>448, \"name\"=>\"Durable complete response of metastatic gastric cancer with anti-Met therapy followed by resistance at recurrence.\", \"citation\"=>\"Catenacci et al., 2011, Cancer Discov\", \"pubmed_id\"=>\"22389872\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22389872\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3289149\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>12}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>270, \"phenotypes\"=>[]}]}, {\"id\"=>298, \"entrez_name\"=>\"MYCN\", \"entrez_id\"=>4613, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>3741, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>43.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>16080686, \"stop\"=>16087129, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000281043.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3741/summary/variants/298/summary#variant\", \"evidence_items\"=>[{\"id\"=>5327, \"name\"=>\"EID5327\", \"description\"=>\"In a patient derived xenograft from a sonic hedgehog driven medulloblastoma patient,  MYCN amplification conferred to resistance to SMO inhibition with Sonidegib. This PDX was shown to be responsive to GLI inhibitor Arsenic trioxide.\", \"disease\"=>{\"id\"=>361, \"name\"=>\"Medulloblastoma\", \"display_name\"=>\"Medulloblastoma\", \"doid\"=>\"0050902\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050902\"}, \"drugs\"=>[{\"id\"=>47, \"name\"=>\"Arsenic Trioxide\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2228, \"name\"=>\"Itraconazole and arsenic trioxide inhibit Hedgehog pathway activation and tumor growth associated with acquired resistance to smoothened antagonists.\", \"citation\"=>\"Kim et al., 2013, Cancer Cell\", \"pubmed_id\"=>\"23291299\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23291299\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3548977\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>1, \"day\"=>14}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>298, \"phenotypes\"=>[]}, {\"id\"=>5325, \"name\"=>\"EID5325\", \"description\"=>\"In a sonic hedgehog driven medulloblastoma patient derived xenograft harbouring a MYCN amplification, SMO inhibitor Sonidegib did not have an effect on cell proliferation.\", \"disease\"=>{\"id\"=>361, \"name\"=>\"Medulloblastoma\", \"display_name\"=>\"Medulloblastoma\", \"doid\"=>\"0050902\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050902\"}, \"drugs\"=>[{\"id\"=>533, \"name\"=>\"Sonidegib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>494, \"name\"=>\"Genome sequencing of SHH medulloblastoma predicts genotype-related response to smoothened inhibition.\", \"citation\"=>\"Kool et al., 2014, Cancer Cell\", \"pubmed_id\"=>\"24651015\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24651015\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4493053\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>3, \"day\"=>17}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>298, \"phenotypes\"=>[]}, {\"id\"=>1328, \"name\"=>\"EID1328\", \"description\"=>\"In a cohort of 1596 neuroblastoma patients.   Five year event free survival was 74 months in non-MYCN amplified cases and 39 months in amplified cases. Differences were significant with p < 0.0001. Five year overall survival was 83 months and non-MYCN amplified cases and 46  months in amplified cases, with p < 0.0001, indicating poorer prognosis for MYCN-amplified cases.\", \"disease\"=>{\"id\"=>13, \"name\"=>\"Neuroblastoma\", \"display_name\"=>\"Neuroblastoma\", \"doid\"=>\"769\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:769\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>918, \"name\"=>\"ALK mutations confer differential oncogenic activation and sensitivity to ALK inhibition therapy in neuroblastoma.\", \"citation\"=>\"Bresler et al., 2014, Cancer Cell\", \"pubmed_id\"=>\"25517749\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25517749\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4269829\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>11, \"day\"=>10}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>298, \"phenotypes\"=>[]}, {\"id\"=>744, \"name\"=>\"EID744\", \"description\"=>\"The drug CBL0137 (a drug that interacts with SSRP1 aka FACT) exhibited a synergistic effect with standard chemotherapy (cyclophosphamide, etoposide, cisplatin, vincristine, etc.) by blocking repair of DNA damage caused by genotoxic drugs, thus creating a synthetic lethal environment in MYCN-amplified neuroblastoma cells. High MYCN expression was found to sensitize neuroblastoma cells to CBL0137.\", \"disease\"=>{\"id\"=>13, \"name\"=>\"Neuroblastoma\", \"display_name\"=>\"Neuroblastoma\", \"doid\"=>\"769\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:769\"}, \"drugs\"=>[{\"id\"=>193, \"name\"=>\"CBL0137\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>491, \"name\"=>\"Therapeutic targeting of the MYC signal by inhibition of histone chaperone FACT in neuroblastoma.\", \"citation\"=>\"Carter et al., 2015, Sci Transl Med\", \"pubmed_id\"=>\"26537256\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26537256\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>11, \"day\"=>4}, \"journal\"=>\"Sci Transl Med\", \"full_journal_title\"=>\"Science translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>298, \"phenotypes\"=>[]}]}, {\"id\"=>306, \"entrez_name\"=>\"ERBB2\", \"entrez_id\"=>2064, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"Her2 (ERBB2) amplifications are seen in up to 20% of breast cancers and were associated with aggressive disease and poor prognosis when discovered. Trastuzumab first found considerable success, and was approved for treatment, in HER2 positive metastatic breast cancer (MBC) which had progressed under chemotherapy.  These metastatic cancers nonetheless progressed under trastuzumab, and other forms of therapy were studied for next line treatments, including tyrosine kinase inhibitors.  Lapatinib, a reversible inhibitor of tyrosine kinase activity in ErbB1 and ErbB2, and afatinib, an irreversible inhibitor of all 4 ErbB forms were shown to have activity in trastuzumab-progressed HER2 MBC.  Interestingly trastuzumab itself also turned out to remain effective in treatment of trastuzumab-progressed HER2 MBC.  The LUX-Breast 1 trial compared a TKI-based treatment (afatinib) to a trastuzumab-based treatment of trastuzumab-progressed HER2 MBC, and the LUX-Breast 3 trial addressed TKI application in the context of HER2 MBC with difficult to treat brain metastases.  In neither of these cases was the TKI more effective, making successful future replacement of trastuzumab-based therapy with a TKI in trastuzumab-progressed HER2 MBC seem unlikely.  Instead, a potential role for TKIs in this disease context is revealed in preclinical work suggesting synergism between TKI and trastuzumab action, and clinical trials with lapatinib or afatinib combined with trastuzumab indicate that dual HER2 blockade may be an effective therapy.  Dual antibody HER2 blockade with pertuzumab instead of TKI in HER2 MBC has also recently shown strong results in the CLEOPATRA trial. HER2 overexpression is targeted in neoadjuvant breast cancer treatment, and has also been successfully targeted in gastric cancer.\", \"gene_id\"=>20, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>822.5, \"coordinates\"=>{\"chromosome\"=>\"17\", \"start\"=>37856333, \"stop\"=>37884915, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000269571.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/20/summary/variants/306/summary#variant\", \"evidence_items\"=>[{\"id\"=>1011, \"name\"=>\"EID1011\", \"description\"=>\"This Phase II trial (LUX-Breast 3 - NCT01441596) compared afatinib, afatinib plus vinorelbine, or investigator's choice of treatment.  121 patients with HER2 overexpressing breast cancer with CNS recurrence or regression who had previously progressed on trastuzumab or lapatinib were assigned to 3 arms, with endpoint in the intention-to-treat population being patient benefit at 12 weeks defined as absence of CNS or extra CNS progression, no tumor related worsening of neurological symptoms and no increase in corticosteroid dose. Benefit was seen in 30% of 40 patients assigned to afatinib alone, 34.2% of 38 assigned to afatinib plus vinorelbine, and 41.9% of 43 assigned to investigator's treatment choice.  Differences were not statistically significant, and afatanib patients experienced more adverse effects.  While Afatanib did show response in HER2 breast cancer patients, it was not better than investigator's treatment choice and the investigators state that no further development of afatinib for HER2 breast cancer will be planned.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>146, \"name\"=>\"Afatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>699, \"name\"=>\"Afatinib alone or afatinib plus vinorelbine versus investigator's choice of treatment for HER2-positive breast cancer with progressive brain metastases after trastuzumab, lapatinib, or both (LUX-Breast 3): a randomised, open-label, multicentre, phase 2 trial.\", \"citation\"=>\"Cortés et al., 2015, Lancet Oncol.\", \"pubmed_id\"=>\"26596672\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26596672\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>12}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1013, \"name\"=>\"EID1013\", \"description\"=>\"Trastuzumab resistance is thought to consist of multiple compensatory mechanisms involving ErbB family members such as increased signaling through ErbB family heterodimers.  The irreversible TKI afatinib blocks EGFR, HER2, ErbB3 and ErbB4.  Thus afatinib was studied in this Phase I trial of trastuzumab-progressed HER2 positive metastatic BC (MBC) in combination with continued trastuzumab treatment, where trastuzumab is still the perferred treatment for trastuzumab-progressed MBC.  Objective response and disease control rates were 11% and 39%.  The authors conclude that the clinical activity observed in this trial warrants further work with afatinib and trastuzumab combination therapy.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}, {\"id\"=>146, \"name\"=>\"Afatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>701, \"name\"=>\"Phase I Study to Assess the Combination of Afatinib with Trastuzumab in Patients with Advanced or Metastatic HER2-Positive Breast Cancer.\", \"citation\"=>\"Ring et al., 2015, Clin. Cancer Res.\", \"pubmed_id\"=>\"25370464\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25370464\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>6, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1159, \"name\"=>\"EID1159\", \"description\"=>\"The Phase III trial, TH3RESA (NCT01419197), was performed in patients with advanced HER2 breast cancer that was heavily pretreated.  Patients were required to have prior exposure to trastuzumab and lapatinib as well as a taxane.  This patient population has not seen many studies, responds poorly to late-line treatment with trastuzumab, and in this study was treated with trastuzumab emtansine (T-DM1, n=404) vs. physician's choice (usually trastuzumab plus chemotherapy, n=198).  Results showed progression free survival was 6.2 months in T-DM1 treatment vs. 3.3 months under physician's choice, along with lower incidence of grade 3 or worse events with T-DM1.  The authors conclude that their findings in this heavily pretreated population, along with findings in the EMILIA trial, indicate promise for antibody-drug conjugates.  The NCI-MATCH trial with further study T-DM1 in the context of HER2 amplification.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>293, \"name\"=>\"Trastuzumab Emtansine\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>798, \"name\"=>\"Trastuzumab emtansine versus treatment of physician's choice for pretreated HER2-positive advanced breast cancer (TH3RESA): a randomised, open-label, phase 3 trial.\", \"citation\"=>\"Krop et al., 2014, Lancet Oncol.\", \"pubmed_id\"=>\"24793816\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24793816\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>6}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1133, \"name\"=>\"EID1133\", \"description\"=>\"In this Phase III trial, trastuzumab-based treatment was assessed in women with locally advanced or metastatic HER2 positive breast cancer who had progressed on trastuzumab.  Patients had 12 weeks or greater previous trastuzumab treatment with time since end of last cycle less than 6 weeks.  Primary endpoint was time to progression.  78 patients were assigned to each group, with median time to progression of 5.6 months in capecitabine group and 8.2 months in trastuzumab plus capecitabine group.  The results are in contrast to the principle of change of treatment on disease progression, and authors suggest that chemotherapy-sensitizing mechanisms of trastuzumab remain intact in HER2 breast cancer cells with trastuzumab progression.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>32, \"name\"=>\"Capecitabine\", \"pubchem_id\"=>nil}, {\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>784, \"name\"=>\"Trastuzumab beyond progression in human epidermal growth factor receptor 2-positive advanced breast cancer: a german breast group 26/breast international group 03-05 study.\", \"citation\"=>\"von Minckwitz et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19289619\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19289619\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>4, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1012, \"name\"=>\"EID1012\", \"description\"=>\"LUX-Breast 1 is a phase 3 trial (NCT01125566) which enrolled 508 patients with metastatic HER2 positive BC who had progressed after adjuvant or first-line trastuzumab. It was thought that the wider range of ErbB family blocking and irreversible properties of afatinib could be beneficial in trastuzumab resistant cancer cells, since ErbB family members are thought to play a role in trastuzumab resistance. Primary endpoint was intention-to-treat group PFS. 339 patients were assigned to afatanib + vinorelbine and 169  to trastuzumab + vinorelbine.  Recruitment was stopped prematurely when it was deemed unlikely that the increased PFS would be seen in the afatinib group. Median PFS was 5.5 months in the afatinib group and 5.6 months in the trastuzumab group.  Afatanib plus vinorelbine was less well tolerated than trastuzumab plus vinorelbine, and the authors conclude that trastuzumab remains the best choice for treatment of trastuzumab-progressed HER2 positive metastatic BC.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>146, \"name\"=>\"Afatinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>700, \"name\"=>\"Afatinib plus vinorelbine versus trastuzumab plus vinorelbine in patients with HER2-overexpressing metastatic breast cancer who had progressed on one previous trastuzumab treatment (LUX-Breast 1): an open-label, randomised, phase 3 trial.\", \"citation\"=>\"Harbeck et al., 2016, Lancet Oncol.\", \"pubmed_id\"=>\"26822398\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26822398\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>3}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1113, \"name\"=>\"EID1113\", \"description\"=>\"In this two arm, Phase III study (NCT00878709) of the efficacy of a 12 month course of the TKI neratinib after trastuzumab-based adjuvant therapy of early stage HER2-positive breast cancer was assessed using invasive disease-free survival as primary endpoint.  HER2 positivity was assessed via FISH.  At 2 year follow-up, 70 vs. 109 disease-free survival events had occurred and disease-free survival rates were 93.9% vs. 91.6% (significant) in neratinib vs. placebo groups respectively.  Distant disease-free survival and time to distant recurrence were similar in both groups.  These results are an interesting contrast to results of long-term adjuvant therapy with trastuzumab (HERA) which did not show significant benefit.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>52, \"name\"=>\"Neratinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>768, \"name\"=>\"Neratinib after trastuzumab-based adjuvant therapy in patients with HER2-positive breast cancer (ExteNET): a multicentre, randomised, double-blind, placebo-controlled, phase 3 trial.\", \"citation\"=>\"Chan et al., 2016, Lancet Oncol.\", \"pubmed_id\"=>\"26874901\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26874901\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>3}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1160, \"name\"=>\"EID1160\", \"description\"=>\"The EMILIA study (NCT00829166) was a Phase II trial to assess the efficacy of trastuzumab emtansine (T-DM1, n=495) in comparison to lapatinib plus capecitabine (n=496), in treatment of advanced HER2 positive breast cancer.  Patients were required to have progressed on or after treatments involving trastuzumab and a taxane, and no prior exposure to lapatinib or T-DM1.   Primary endpoints included independently assessed progression free survival (PFS) and safety. PFS was 9.6 months in T-DM1 group vs. 6.4 months in lapatinib plus capecitabine.  Rates of adverse events of grade 3 or higher were greater with lapatinib plus capecitabine.  These results, together with other trials such as TH3RESA indicate promise for T-DM1 therapy in HER2 amplification, and this will be further studied in an arm of the NCI-MATCH trial.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>293, \"name\"=>\"Trastuzumab Emtansine\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>799, \"name\"=>\"Trastuzumab emtansine for HER2-positive advanced breast cancer.\", \"citation\"=>\"Verma et al., 2012, N. Engl. J. Med.\", \"pubmed_id\"=>\"23020162\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23020162\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>11, \"day\"=>8}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1007, \"name\"=>\"EID1007\", \"description\"=>\"In light of results showing increased PFS using mAB plus TKI in heavily pretreated patients, the efficacy of trastuzumab plus lapatinib in a neoadjuvant setting with untreated stage II or IIIA HER2 positive breast cancer was assessed.  In this randomized 3 arm phase II study (CHER-LOB), patients received (A) chemotherapy plus trastuzumab, (B) chemotherapy plus lapatinib, or (C) chemotherapy plus trastuzumab and lapatinib, with pathologic complete response (pCR) as endpoint.  From 121 patients, pCR rates were 25% in arm A, 26.3% in arm B, and 46.7% in arm C.  The authors conclude these results support the superiority of dual-HER2 inhibition in the neoadjuvant context of HER2 positive BC.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>45, \"name\"=>\"Lapatinib\", \"pubchem_id\"=>nil}, {\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>696, \"name\"=>\"Preoperative chemotherapy plus trastuzumab, lapatinib, or both in human epidermal growth factor receptor 2-positive operable breast cancer: results of the randomized phase II CHER-LOB study.\", \"citation\"=>\"Guarneri et al., 2012, J. Clin. Oncol.\", \"pubmed_id\"=>\"22493419\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22493419\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>6, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1008, \"name\"=>\"EID1008\", \"description\"=>\"This Phase III trial compared the activity of lapatinib vs. trastuzumab vs. sequential trastuzumab followed by lapatinib vs. lapatinib plus trastuzumab in an adjuvant treatment setting.  8381 women were randomized to one of four treatment arms. Patients had HER2 positive non-metastatic operable BC. The primary endpoint was disease free survival.  First results report that after median 4.5 year follow up, a non significant 16% improvement in DFS was seen in trastuzumab + lapatinib vs. trastuzumab alone.  The results are interpreted as showing no added benefit for labatinib addition to trastuzumab in the adjuvant treatment context.  This trial is ongoing but not recruiting patients (NCT00490139).\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>45, \"name\"=>\"Lapatinib\", \"pubchem_id\"=>nil}, {\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>697, \"name\"=>\"Optimal adjuvant treatment for patients with HER2-positive breast cancer in 2015.\", \"citation\"=>\"Zardavas et al., 2015, Breast\", \"pubmed_id\"=>\"26255196\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26255196\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>11}, \"journal\"=>\"Breast\", \"full_journal_title\"=>\"Breast (Edinburgh, Scotland)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>529, \"name\"=>\"EID529\", \"description\"=>\"A randomized clinical trial of 186 patients with previously untreated, HER2-positive, metastatic breast cancer demonstrated improved overall survival, response rate, response duration, time to progression, and time to treatment failure for patients who received trastuzumab in addition to chemotherapy (docetaxel) compared to chemotherapy alone.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>329, \"name\"=>\"Randomized phase II trial of the efficacy and safety of trastuzumab combined with docetaxel in patients with human epidermal growth factor receptor 2-positive metastatic breast cancer administered as first-line treatment: the M77001 study group.\", \"citation\"=>\"Marty et al., 2005, J. Clin. Oncol.\", \"pubmed_id\"=>\"15911866\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15911866\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>528, \"name\"=>\"EID528\", \"description\"=>\"A randomized clinical trial of 469 patients with previously untreated, HER2-positive, metastatic breast cancer demonstrated improved time to disease progression, objective response rate, and duration of response for patients who received trastuzumab in addition to chemotherapy compared to chemotherapy alone.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>328, \"name\"=>\"Use of chemotherapy plus a monoclonal antibody against HER2 for metastatic breast cancer that overexpresses HER2.\", \"citation\"=>\"Slamon et al., 2001, N. Engl. J. Med.\", \"pubmed_id\"=>\"11248153\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/11248153\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2001, \"month\"=>3, \"day\"=>15}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>5985, \"name\"=>\"EID5985\", \"description\"=>\"The phase 2a MyPathway study assigned patients with HER2, EGFR, BRAF or SHH alterations to treatment with pertuzumab plus trastuzumab, erlotinib, vemurafenib, or vismodegib, respectively. Two of nine patients (22%) with HER2 amplified/overexpressing pancreatic cancer had a PR with trastuzumab/pertuzumab combination. Another patient had SD > 120days.\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}, {\"id\"=>247, \"name\"=>\"Pertuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2414, \"name\"=>\"Targeted Therapy for Advanced Solid Tumors on the Basis of Molecular Profiles: Results From MyPathway, an Open-Label, Phase IIa Multiple Basket Study.\", \"citation\"=>\"Hainsworth et al., 2018, J. Clin. Oncol.\", \"pubmed_id\"=>\"29320312\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/29320312\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2018, \"month\"=>1, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[{\"nct_id\"=>\"NCT02091141\", \"name\"=>\"My Pathway: A Study Evaluating Herceptin/Perjeta, Tarceva, Zelboraf/Cotellic, Erivedge, Alecensa, and Tecentriq Treatment Targeted Against Certain Molecular Alterations in Participants With Advanced Solid Tumors\", \"description\"=>\"This multicenter, non-randomized, open-label study will evaluate the efficacy and safety of six treatment regimens in participants with advanced solid tumors for whom therapies that will convey clinical benefit are not available and/or are not suitable options per the treating physician's judgment.\", \"clinical_trial_url\"=>\"https://clinicaltrials.gov/show/NCT02091141\"}]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>5986, \"name\"=>\"EID5986\", \"description\"=>\"The phase 2a MyPathway study assigned patients with HER2, EGFR, BRAF or SHH alterations to treatment with pertuzumab plus trastuzumab, erlotinib, vemurafenib, or vismodegib, respectively. None of the 7 patients with HER2 amplified/overexpressing uterus cancer responded to pertuzumab/trastuzumab.\", \"disease\"=>{\"id\"=>825, \"name\"=>\"Uterine Cancer\", \"display_name\"=>\"Uterine Cancer\", \"doid\"=>\"363\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:363\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}, {\"id\"=>247, \"name\"=>\"Pertuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2414, \"name\"=>\"Targeted Therapy for Advanced Solid Tumors on the Basis of Molecular Profiles: Results From MyPathway, an Open-Label, Phase IIa Multiple Basket Study.\", \"citation\"=>\"Hainsworth et al., 2018, J. Clin. Oncol.\", \"pubmed_id\"=>\"29320312\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/29320312\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2018, \"month\"=>1, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[{\"nct_id\"=>\"NCT02091141\", \"name\"=>\"My Pathway: A Study Evaluating Herceptin/Perjeta, Tarceva, Zelboraf/Cotellic, Erivedge, Alecensa, and Tecentriq Treatment Targeted Against Certain Molecular Alterations in Participants With Advanced Solid Tumors\", \"description\"=>\"This multicenter, non-randomized, open-label study will evaluate the efficacy and safety of six treatment regimens in participants with advanced solid tumors for whom therapies that will convey clinical benefit are not available and/or are not suitable options per the treating physician's judgment.\", \"clinical_trial_url\"=>\"https://clinicaltrials.gov/show/NCT02091141\"}]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1993, \"name\"=>\"EID1993\", \"description\"=>\"We identified 142 patients with metastatic colorectal cancer whose tumors harbored both wild-type exons 2, 3, and 4 in KRAS and NRAS, and wild-type exon 15 in BRAF. All patients received cetuximab after oxaliplatin, irinotecan, and fluoropyrimidine failure. HER2 status was determined using immunohistochemistry and silver in situ hybridization and correlated with cetuximab efficacy. Of 142 RAS and BRAF wild-type tumors, we observed 7 cases (4.9%) of HER2 amplification by SISH. After a median follow-up of 13.2 months (range, 1.4-78.1 months), median progression-free survival (PFS) was significantly different according to HER2 status: 3.1 months in patients with HER2 amplification compared with 5.6 months in those with non-amplified HER2 (HR = 2.73; 95% CI = 1.18-6.31; P = 0.019). Overall survival (OS) was not significantly different between groups, although there was a tendency towards shorter OS in patients with HER2-amplified tumors (HR = 0.31; 95% CI = 0.61-2.82; 10.1 vs. 13.5 months; P = 0.488).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1399, \"name\"=>\"HER2 Amplification and Cetuximab Efficacy in Patients With Metastatic Colorectal Cancer Harboring Wild-type RAS and BRAF.\", \"citation\"=>\"Jeong et al., 2017, Clin Colorectal Cancer\", \"pubmed_id\"=>\"28223103\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28223103\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>1, \"day\"=>25}, \"journal\"=>\"Clin Colorectal Cancer\", \"full_journal_title\"=>\"Clinical colorectal cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>5981, \"name\"=>\"EID5981\", \"description\"=>\"The phase 2a MyPathway study assigned patients with HER2, EGFR, BRAF or SHH alterations to treatment with pertuzumab plus trastuzumab, erlotinib, vemurafenib, or vismodegib, respectively. \\nThirty of 114 patients (26%; 95% CI, 19% to 35%) with HER2 amplification/overexpression had objective responses to treatment with trastuzumab plus pertuzumab (two CR, 28 PR). \\nPatients with HER2-amplified/overexpressing metastatic colorectal cancer composed the largest tumor-pathway cohort. In this group of 37 patients with refractory disease (median, four previous lines of therapy), treatment with trastuzumab plus pertuzumab produced PRs in 14 patients (38%; 95% CI, 23% to 55%; Fig 2A). An additional four patients had SD > 120 days. The median DOR was 11 months (range, < 1 to 16+ months; 95% CI, 2.8 months to not estimable).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}, {\"id\"=>247, \"name\"=>\"Pertuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2414, \"name\"=>\"Targeted Therapy for Advanced Solid Tumors on the Basis of Molecular Profiles: Results From MyPathway, an Open-Label, Phase IIa Multiple Basket Study.\", \"citation\"=>\"Hainsworth et al., 2018, J. Clin. Oncol.\", \"pubmed_id\"=>\"29320312\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/29320312\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2018, \"month\"=>1, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[{\"nct_id\"=>\"NCT02091141\", \"name\"=>\"My Pathway: A Study Evaluating Herceptin/Perjeta, Tarceva, Zelboraf/Cotellic, Erivedge, Alecensa, and Tecentriq Treatment Targeted Against Certain Molecular Alterations in Participants With Advanced Solid Tumors\", \"description\"=>\"This multicenter, non-randomized, open-label study will evaluate the efficacy and safety of six treatment regimens in participants with advanced solid tumors for whom therapies that will convey clinical benefit are not available and/or are not suitable options per the treating physician's judgment.\", \"clinical_trial_url\"=>\"https://clinicaltrials.gov/show/NCT02091141\"}]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>5983, \"name\"=>\"EID5983\", \"description\"=>\"The phase 2a MyPathway study assigned patients with HER2, EGFR, BRAF or SHH alterations to treatment with pertuzumab plus trastuzumab, erlotinib, vemurafenib, or vismodegib, respectively. Two of seven patients (29%; 95% CI, 4% to 71%) with biliary cancer and HER2 amplification/overexpression had PR, and three had SD > 120 days\", \"disease\"=>{\"id\"=>1065, \"name\"=>\"Biliary Tract Cancer\", \"display_name\"=>\"Biliary Tract Cancer\", \"doid\"=>\"4607\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4607\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}, {\"id\"=>247, \"name\"=>\"Pertuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2414, \"name\"=>\"Targeted Therapy for Advanced Solid Tumors on the Basis of Molecular Profiles: Results From MyPathway, an Open-Label, Phase IIa Multiple Basket Study.\", \"citation\"=>\"Hainsworth et al., 2018, J. Clin. Oncol.\", \"pubmed_id\"=>\"29320312\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/29320312\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2018, \"month\"=>1, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[{\"nct_id\"=>\"NCT02091141\", \"name\"=>\"My Pathway: A Study Evaluating Herceptin/Perjeta, Tarceva, Zelboraf/Cotellic, Erivedge, Alecensa, and Tecentriq Treatment Targeted Against Certain Molecular Alterations in Participants With Advanced Solid Tumors\", \"description\"=>\"This multicenter, non-randomized, open-label study will evaluate the efficacy and safety of six treatment regimens in participants with advanced solid tumors for whom therapies that will convey clinical benefit are not available and/or are not suitable options per the treating physician's judgment.\", \"clinical_trial_url\"=>\"https://clinicaltrials.gov/show/NCT02091141\"}]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>5982, \"name\"=>\"EID5982\", \"description\"=>\"The phase 2a MyPathway study assigned patients with HER2, EGFR, BRAF or SHH alterations to treatment with pertuzumab plus trastuzumab, erlotinib, vemurafenib, or vismodegib, respectively. Three of nine patients (33%; 95% CI, 8% to 70%) with advanced bladder cancer and HER2 amplification/overexpression had responses (one CR ongoing at 15 months; two PR lasting 1 and 6 months), and two patients had SD > 120 days.\", \"disease\"=>{\"id\"=>18, \"name\"=>\"Bladder Carcinoma\", \"display_name\"=>\"Bladder Carcinoma\", \"doid\"=>\"4007\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4007\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}, {\"id\"=>247, \"name\"=>\"Pertuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2414, \"name\"=>\"Targeted Therapy for Advanced Solid Tumors on the Basis of Molecular Profiles: Results From MyPathway, an Open-Label, Phase IIa Multiple Basket Study.\", \"citation\"=>\"Hainsworth et al., 2018, J. Clin. Oncol.\", \"pubmed_id\"=>\"29320312\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/29320312\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2018, \"month\"=>1, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[{\"nct_id\"=>\"NCT02091141\", \"name\"=>\"My Pathway: A Study Evaluating Herceptin/Perjeta, Tarceva, Zelboraf/Cotellic, Erivedge, Alecensa, and Tecentriq Treatment Targeted Against Certain Molecular Alterations in Participants With Advanced Solid Tumors\", \"description\"=>\"This multicenter, non-randomized, open-label study will evaluate the efficacy and safety of six treatment regimens in participants with advanced solid tumors for whom therapies that will convey clinical benefit are not available and/or are not suitable options per the treating physician's judgment.\", \"clinical_trial_url\"=>\"https://clinicaltrials.gov/show/NCT02091141\"}]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>5984, \"name\"=>\"EID5984\", \"description\"=>\"The phase 2a MyPathway study assigned patients with HER2, EGFR, BRAF or SHH alterations to treatment with pertuzumab plus trastuzumab, erlotinib, vemurafenib, or vismodegib, respectively. Four of five patients with HER2 amplified/overexpressing salivary gland carcinoma (80%; 95% CI, 28% to > 99%) had responses (all PR).\", \"disease\"=>{\"id\"=>66, \"name\"=>\"Salivary Gland Carcinoma\", \"display_name\"=>\"Salivary Gland Carcinoma\", \"doid\"=>\"0050904\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050904\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}, {\"id\"=>247, \"name\"=>\"Pertuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2414, \"name\"=>\"Targeted Therapy for Advanced Solid Tumors on the Basis of Molecular Profiles: Results From MyPathway, an Open-Label, Phase IIa Multiple Basket Study.\", \"citation\"=>\"Hainsworth et al., 2018, J. Clin. Oncol.\", \"pubmed_id\"=>\"29320312\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/29320312\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2018, \"month\"=>1, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[{\"nct_id\"=>\"NCT02091141\", \"name\"=>\"My Pathway: A Study Evaluating Herceptin/Perjeta, Tarceva, Zelboraf/Cotellic, Erivedge, Alecensa, and Tecentriq Treatment Targeted Against Certain Molecular Alterations in Participants With Advanced Solid Tumors\", \"description\"=>\"This multicenter, non-randomized, open-label study will evaluate the efficacy and safety of six treatment regimens in participants with advanced solid tumors for whom therapies that will convey clinical benefit are not available and/or are not suitable options per the treating physician's judgment.\", \"clinical_trial_url\"=>\"https://clinicaltrials.gov/show/NCT02091141\"}]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1437, \"name\"=>\"EID1437\", \"description\"=>\"In a randomized phase 3 trial, HER2 positive metastatic breast cancer patients given first-line treatment of pertuzumab plus trastuzumb plus docetaxel experienced improved progression free survival (18.5 vs 12.4 months,  P<0.001) and increased objective response rate (80.2% vs 69.3%, P=0.001) than the control group given placebo plus trastuzumab plus docetaxel.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>73, \"name\"=>\"Docetaxel\", \"pubchem_id\"=>nil}, {\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}, {\"id\"=>247, \"name\"=>\"Pertuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>971, \"name\"=>\"Pertuzumab plus trastuzumab plus docetaxel for metastatic breast cancer.\", \"citation\"=>\"Baselga et al., 2012, N. Engl. J. Med.\", \"pubmed_id\"=>\"22149875\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22149875\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>1, \"day\"=>12}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1132, \"name\"=>\"EID1132\", \"description\"=>\"This Phase III trial took patients with locally advanced or metastatic HER2 (ERBB2) positive breast cancer that had progressed on prior trastuzumab treatment.  Patients were given capecitabine (n=161) or lapatinib plus capecitabine (n=163). The primary endpoint of the study was time to progression. Median time to progression was 4.4 months in the capecitabine group and 8.4 months in capecitabine plus lapatinib group (hazard ratio was 0.49 ; 95% confidence interval, 0.34 to 0.71; P<0.001).  The results indicated that lapatinib may be useful in addressing the problem of trastuzumab progression in HER2 breast cancer.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>32, \"name\"=>\"Capecitabine\", \"pubchem_id\"=>nil}, {\"id\"=>45, \"name\"=>\"Lapatinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>783, \"name\"=>\"Lapatinib plus capecitabine for HER2-positive advanced breast cancer.\", \"citation\"=>\"Geyer et al., 2006, N. Engl. J. Med.\", \"pubmed_id\"=>\"17192538\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17192538\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2006, \"month\"=>12, \"day\"=>28}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1381, \"name\"=>\"EID1381\", \"description\"=>\"Pharmacologic (A66; p110alpha specific inhibitor) or genetic (cre-mediated knockout) ablation of the p110alpha PI3K isoform reduced tumor growth and tumor formation, respectively, in athymic mice transplanted with cells transduced by MMTV-NeuT (mammary specific overexpression of HER2 that effectively induces tumors). A66 treatment of cells from MMTV-NeuT mouse tumors resulted in decreased lipid kinase activity and phospho Akt levels.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>337, \"name\"=>\"A66\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>937, \"name\"=>\"The p110α and p110β isoforms of PI3K play divergent roles in mammary gland development and tumorigenesis.\", \"citation\"=>\"Utermark et al., 2012, Genes Dev.\", \"pubmed_id\"=>\"22802530\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22802530\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3404385\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>7, \"day\"=>15}, \"journal\"=>\"Genes Dev.\", \"full_journal_title\"=>\"Genes & development\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>961, \"name\"=>\"EID961\", \"description\"=>\"A phase II study involving adult female patients, >=18 years, with stage IIIB or IV HER2-positive (by IHC and/or in situ fluorescence) metastatic breast cancer. All patients were previously treated with trastuzumab and a median of three chemotherapy lines. 41 patients received 50 mg afatinib once-daily until disease progression and the primary endpoint was objective response rate. 4 patients (10% of 41 treated; 11% of evaluable patients) had partial response and 15 patients (37% of 41) had stable disease as best response. Thus a total of 19 (46% of 41) achieved clinical benefit. Median progression-free survival was 15.1 weeks.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>146, \"name\"=>\"Afatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>663, \"name\"=>\"A phase II study of afatinib (BIBW 2992), an irreversible ErbB family blocker, in patients with HER2-positive metastatic breast cancer progressing after trastuzumab.\", \"citation\"=>\"Lin et al., 2012, Breast Cancer Res. Treat.\", \"pubmed_id\"=>\"22418700\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22418700\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3387495\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>6}, \"journal\"=>\"Breast Cancer Res. Treat.\", \"full_journal_title\"=>\"Breast cancer research and treatment\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>887, \"name\"=>\"EID887\", \"description\"=>\"In this phase 2 trial, treatment-naive, ERBB2-positive (by IHC) breast cancer patients with stage IIIA, B, C or inflammatory disease were randomized 1:1:1 to afatinib (n = 10), lapatinib (n = 8), or trastuzumab (n = 11). The primary end point was objective response rate. Objective response was seen in 8 afatinib-, 6 lapatinib-, and 4 trastuzumab-treated patients. Afatinib demonstrated clinical activity that compared favorably to trastuzumab and lapatinib for neoadjuvant treatment of HER2-positive breast cancer.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>45, \"name\"=>\"Lapatinib\", \"pubchem_id\"=>nil}, {\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}, {\"id\"=>146, \"name\"=>\"Afatinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>597, \"name\"=>\"A neoadjuvant, randomized, open-label phase II trial of afatinib versus trastuzumab versus lapatinib in patients with locally advanced HER2-positive breast cancer.\", \"citation\"=>\"Rimawi et al., 2015, Clin. Breast Cancer\", \"pubmed_id\"=>\"25537159\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25537159\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>4}, \"journal\"=>\"Clin. Breast Cancer\", \"full_journal_title\"=>\"Clinical breast cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1499, \"name\"=>\"EID1499\", \"description\"=>\"ToGA, a Phase III study (NCT01041404) addressed use of trastuzumab in HER2 positive (overexpression or amplification) advanced gastric cancer, where chemotherapy using fluoropyrimidine-based and platinum-based combinations had been common practice.  Patients were given a chemotherapy regime containing capecitabine, flourouracil and cisplatin with (n=298) or without (n=296) trastuzumab, and had not been previously treated for metastatic cancer.  Study endpoint was overall survival, which was significantly different in the two populations (13.8 months with chemotherapy + trastuzumab and 11.1 months for chemotherapy alone).  The authors conclude that trastuzumab should be a standard option for HER2 positive advanced gastric cancer.\", \"disease\"=>{\"id\"=>147, \"name\"=>\"Gastric Adenocarcinoma\", \"display_name\"=>\"Gastric Adenocarcinoma\", \"doid\"=>\"3717\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3717\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>708, \"name\"=>\"Trastuzumab in combination with chemotherapy versus chemotherapy alone for treatment of HER2-positive advanced gastric or gastro-oesophageal junction cancer (ToGA): a phase 3, open-label, randomised controlled trial.\", \"citation\"=>\"Bang et al., 2010, Lancet\", \"pubmed_id\"=>\"20728210\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20728210\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>8, \"day\"=>28}, \"journal\"=>\"Lancet\", \"full_journal_title\"=>\"Lancet (London, England)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1050, \"name\"=>\"EID1050\", \"description\"=>\"Efficacy of trastuzumab in HER2 overexpressing NSCLC remains unclear.  In this study using a panel of NSCLC HER2 overexpressing cell lines, antiproliferative effects of trastuzumab emtansine (T-DM1) were observed in a HER2-expression dependant manner, with strongest effects in Calu-3 (high expressing) and no effects in Calu-6 (low expressing).  Trastuzumab did not show notable effects on cell growth in any tested cells.  HER2 positivity in NSCLC is fairly common at 2-20%.  A clinical trial (NCT02289833) is currently underway to asses TDM-1 in HER2 overexpressing lung cancer.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>293, \"name\"=>\"Trastuzumab Emtansine\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>730, \"name\"=>\"Trastuzumab emtansine is active on HER-2 overexpressing NSCLC cell lines and overcomes gefitinib resistance.\", \"citation\"=>\"Cretella et al., 2014, Mol. Cancer\", \"pubmed_id\"=>\"24898067\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24898067\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4058446\", \"publication_date\"=>{\"year\"=>2014}, \"journal\"=>\"Mol. Cancer\", \"full_journal_title\"=>\"Molecular cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1097, \"name\"=>\"EID1097\", \"description\"=>\"HER2 expression in uterine serous papillary carcinoma (USPC) is thought to be associated with poor prognosis and may act as a targetable driver mutation in this cancer.  HER2 overexpression in USPC is more common in African American women.  This report covered 2 African American patients, the first of which presented high-grade endometrial adenocarcinoma with IHC3+ HER2-positivity as well as positivity by FISH.  Prior treatment involved surgery and chemotherapy.  Treatment with pacitaxel/carboplatin with trastuzumab showed clear clinical improvement assessed by CT scan and drop in CA-125 levels from 988 to 7.9 U/mL.  The second patient had USPC, and was treated with surgery and radiation.  Trastuzumab monotheraby was then administered.  After 7 months of treatment, the patient showed no sign of disease progression and CA-125 had dropped from 144 to 41 U/mL.  The investigators conclude there is promise in trastuzumab-based therapies in advanced endometrial cancers, including USPC.\", \"disease\"=>{\"id\"=>185, \"name\"=>\"Uterine Corpus Serous Adenocarcinoma\", \"display_name\"=>\"Uterine Corpus Serous Adenocarcinoma\", \"doid\"=>\"5750\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5750\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>756, \"name\"=>\"Trastuzumab treatment in patients with advanced or recurrent endometrial carcinoma overexpressing HER2/neu.\", \"citation\"=>\"Santin et al., 2008, Int J Gynaecol Obstet\", \"pubmed_id\"=>\"18555254\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18555254\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2008, \"month\"=>8}, \"journal\"=>\"Int J Gynaecol Obstet\", \"full_journal_title\"=>\"International journal of gynaecology and obstetrics: the official organ of the International Federation of Gynaecology and Obstetrics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1044, \"name\"=>\"EID1044\", \"description\"=>\"This report details 2 case studies in relapsed HER2 positive SCLC.  In the first case, a 50 pack-year patient showed intense non-mutant HER2 expression by IHC without DNA amplification, and had undergone multiple prior lines of treatment.  With induction of trastuzumab and irinotecan treatment, tumor regression was observed with progression free survival of 4.5 months.  Case 2 involved a 68 year old never-smoker with relapsed SCLC.  Tumor showed HER2 positive SCLC cells with no amplification or mutation.  Third line trastuzumab plus irinotecan brought about some symptom improvement and PFS of 3.5 months.  While the efficacy of trastuzumab against NSCLC with elevated levels of wild-type HER2 expression remains unclear, the authors conclude that trastuzumab based therapy is a promising strategy for chemo-resistant SCLC.\", \"disease\"=>{\"id\"=>174, \"name\"=>\"Lung Small Cell Carcinoma\", \"display_name\"=>\"Lung Small Cell Carcinoma\", \"doid\"=>\"5409\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5409\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}, {\"id\"=>101, \"name\"=>\"Irinotecan\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>727, \"name\"=>\"Favorable response to trastuzumab plus irinotecan combination therapy in two patients with HER2-positive relapsed small-cell lung cancer.\", \"citation\"=>\"Kinehara et al., 2015, Lung Cancer\", \"pubmed_id\"=>\"25601188\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25601188\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>3}, \"journal\"=>\"Lung Cancer\", \"full_journal_title\"=>\"Lung cancer (Amsterdam, Netherlands)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1686, \"name\"=>\"EID1686\", \"description\"=>\"A cohort of patient-derived xenografts (PDX) from 85 patients with metastatic colorectal cancer was created. PDX were treated with cetuximab and mechanisms of resistance investigated. None of the xenografts harboring KRAS (N=18), NRAS (N=7) or BRAF (N=3) mutations showed a response to cetuximab whereas 1 out of 4 xenografts with a PIK3CA mutation responded to cetuximab.\\nAmong 11 quadruple-negative samples with resistance to cetuximab, HER2 overexpression and amplification (N=3) and amplification (N=1) were identified. In an independent second retrospective patient cohort, 3 out of 17 patients with KRAS wt, cetuximab resitant CRC were found to harbor HER2 overexpression and amplification. In this cohort no HER2 overexpression or amplification was identified in 14 KRAS wt patients with response to cetuximab.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1138, \"name\"=>\"A molecularly annotated platform of patient-derived xenografts (\\\"xenopatients\\\") identifies HER2 as an effective therapeutic target in cetuximab-resistant colorectal cancer.\", \"citation\"=>\"Bertotti et al., 2011, Cancer Discov\", \"pubmed_id\"=>\"22586653\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22586653\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>11}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1687, \"name\"=>\"EID1687\", \"description\"=>\"A cohort of patient-derived xenografts (PDX) from 85 patients with metastatic colorectal cancer was created. PDX were treated with cetuximab and mechanisms of resistance investigated. None of the xenografts harboring KRAS (N=18), NRAS (N=7) or BRAF (N=3) mutations showed resistance whereas 1 out of 4 xenografts with a PIK3CA mutation responded to cetuximab. Among 11 quadruple-negative samples with resistance to cetuximab, HER2 overexpression and amplification (N=3) and amplification (N=1) were identified. In an independent second retrospective cohort, 3 out of 17 patients with KRAS wt, cetuximab resitant CRC were found to harbor HER2 overexpression and amplification. In this cohort no HER2 overexpression or amplification was identified in 14 KRAS wt patients with response to cetuximab.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1138, \"name\"=>\"A molecularly annotated platform of patient-derived xenografts (\\\"xenopatients\\\") identifies HER2 as an effective therapeutic target in cetuximab-resistant colorectal cancer.\", \"citation\"=>\"Bertotti et al., 2011, Cancer Discov\", \"pubmed_id\"=>\"22586653\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22586653\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>11}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1688, \"name\"=>\"EID1688\", \"description\"=>\"HER2 amplification (FISH) was assessed in 170 KRAS wt mCRC patients treated with cetuximab or panitumumab.\\nHER2 copy number status was significantly correlated with respoinse rate, PFS and OS. Patients with amplification in all neoplastic cells (4%) had worse outcome compared to patients without amplification (35%, intermediate outcome) or amplification in minor clones (61%, highest survival probability)\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}, {\"id\"=>28, \"name\"=>\"Panitumumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1139, \"name\"=>\"HER2 gene copy number status may influence clinical efficacy to anti-EGFR monoclonal antibodies in metastatic colorectal cancer patients.\", \"citation\"=>\"Martin et al., 2013, Br. J. Cancer\", \"pubmed_id\"=>\"23348520\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23348520\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3593567\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>2, \"day\"=>19}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1689, \"name\"=>\"EID1689\", \"description\"=>\"Phase 2 study in 27 patients with HER2 positive (IHC, FISH), KRAS wt metastatic colorectal cancer refractory to standard of care received trastuzumab and lapatinib.\\nAfter a median follow-up of 94 weeks, eight (30%, 95% CI 14-50) of 27 patients had achieved an objective response, with one patient (4%, 95% CI -3 to 11) achieving a complete response, and seven (26%, 95% CI 9-43) achieving partial responses; 12 (44%, 95% CI 25-63) patients had stable disease.\\nA total of 914 patients with KRAS wt were screened for this study, 48 patients (5%) were identified to have HER2-positive tumors.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>45, \"name\"=>\"Lapatinib\", \"pubchem_id\"=>nil}, {\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1140, \"name\"=>\"Dual-targeted therapy with trastuzumab and lapatinib in treatment-refractory, KRAS codon 12/13 wild-type, HER2-positive metastatic colorectal cancer (HERACLES): a proof-of-concept, multicentre, open-label, phase 2 trial.\", \"citation\"=>\"Sartore-Bianchi et al., 2016, Lancet Oncol.\", \"pubmed_id\"=>\"27108243\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27108243\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>6}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1764, \"name\"=>\"EID1764\", \"description\"=>\"In the single arm Phase II study (TDM4258g) breast cancer patients who had progressed on previous anti-HER2 therapy were given transtuzumab emtansine (T-DM1), a HER2 monoclonal antibody and antimicrotubule drug conjugate. 95 trial participants with archival primary tumor were reassessed for HER2 status (positivity defined by FISH or IHC 3+), and 74 patients were confirmed for HER2 positivity and 21 were normal. Overall response rate was 33.8% (95% CI, 23.2% to 44.9%) in patients with confirmed HER2-positive tumors and 4.8% (95% CI, 1.0% to 21.8%) in HER2 normal patients. Median PFS was 8.2 months (95% CI, 4.4 months to not estimable) in patients with confirmed HER2-positive tumors and 2.6 months (95% CI, 1.4 to 3.9 months) in HER2 normal patients.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>293, \"name\"=>\"Trastuzumab Emtansine\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1229, \"name\"=>\"Phase II study of the antibody drug conjugate trastuzumab-DM1 for the treatment of human epidermal growth factor receptor 2 (HER2)-positive breast cancer after prior HER2-directed therapy.\", \"citation\"=>\"Burris et al., 2011, J. Clin. Oncol.\", \"pubmed_id\"=>\"21172893\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21172893\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>2, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1448, \"name\"=>\"EID1448\", \"description\"=>\"In 5 trastuzumab resistant HER2 overexpressing breast cancer cell lines, combination of the pan-PI3K inhibitor XL147 and trastuzumab treatment induced cell death to similar or greater levels than XL147 alone. In three trastuzumab resistant cell lines tested, colony formation in matrigel was markedly inhibited by combination treatment. Apoptotic markers were apparent with combination treatment. In athymic mice, trastuzumab resistant cell line HR6 was used in tumor xenografts and tumors were allowed to form. Mice subsequently treated with trastuzumab or XL147 showed continued tumor growth without marked difference from control, while combination treatment essentially halted tumor growth rate.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}, {\"id\"=>349, \"name\"=>\"XL147 (Pilaralisib)\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>977, \"name\"=>\"Trastuzumab-resistant cells rely on a HER2-PI3K-FoxO-survivin axis and are sensitive to PI3K inhibitors.\", \"citation\"=>\"Chakrabarty et al., 2013, Cancer Res.\", \"pubmed_id\"=>\"23204226\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23204226\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3563941\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>2, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1042, \"name\"=>\"EID1042\", \"description\"=>\"In this phase II trial a small group of patients with IHC2+ or IHC3+ HER2 NSCLC, in which primary platinum-based therapy had failed, were given either docetaxel or trastuzumab after which both groups were given docetaxel and trastuzumab together.  there was one partial response in the docetaxel group and none in the trastuzumab group.  The authors report that due to the limited clinical activity of trastuzumab-based therapy in this cohort, as well as similar results in other trastuzumab studies, that targets other than HER2 in NSCLC should be considered for future study.  This 2004 study remains cited in 2016 literature in discussing the unclear efficacy of trastuzumab in wild-type HER2 overexpressing NSCLC, but mutant HER2 in NSCLC has been further studied and a picture of different HER2 mutations being potentially responsive to specific targeted therapies is emerging.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>725, \"name\"=>\"Trastuzumab plus docetaxel in HER2/neu-positive non-small-cell lung cancer: a California Cancer Consortium screening and phase II trial.\", \"citation\"=>\"Lara et al., 2004, Clin Lung Cancer\", \"pubmed_id\"=>\"14967075\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/14967075\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2004, \"month\"=>1}, \"journal\"=>\"Clin Lung Cancer\", \"full_journal_title\"=>\"Clinical lung cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1380, \"name\"=>\"EID1380\", \"description\"=>\"The HER2/Neu mammary tumor mouse model MMTV-NeuT was employed to demonstrate dependence of HER2 tumor formation on PIK3 p110alpha isoform. MMTV-NeuT mice were crossed with floxed p110alpha mice. Diassacociated tumors cells from these mice were treated with Cre adenovirus to delete p110alpha, or GFP adenovirus, and cells were transplanted into athymic mice. GFP treated cells formed new tumors, while p110alpha deletion blocked tumor formation indicating tumor dependence on PI3KA. Next, MMTV-NeuT tumors were dissociated and treated with pan-PI3K inhibitor GDC-0941. Reduced lipid kinase activity and reduced phospho-Akt levels were seen in these cells. Administration of 125 mg/kg GDC-0941 to athymic mice with transplantation of tumor cells caused marked reduction of tumor cell volume in comparison to control treatment.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>141, \"name\"=>\"Pictilisib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>937, \"name\"=>\"The p110α and p110β isoforms of PI3K play divergent roles in mammary gland development and tumorigenesis.\", \"citation\"=>\"Utermark et al., 2012, Genes Dev.\", \"pubmed_id\"=>\"22802530\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22802530\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3404385\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>7, \"day\"=>15}, \"journal\"=>\"Genes Dev.\", \"full_journal_title\"=>\"Genes & development\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1439, \"name\"=>\"EID1439\", \"description\"=>\"In a randomized open label phase 3 study (NCT00553358), women with HER2 positive primary breast cancer treated with lapatinib plus trasuzumab had higher rates of pathological complete responses than seen in control groups receiving either IV trastuzumab or oral lapatinib alone (combination treatment pCR rate of 51% vs 29.5% and 24.7% for trastuzumab and lapatinib alone, respectively).\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>45, \"name\"=>\"Lapatinib\", \"pubchem_id\"=>nil}, {\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>972, \"name\"=>\"Lapatinib with trastuzumab for HER2-positive early breast cancer (NeoALTTO): a randomised, open-label, multicentre, phase 3 trial.\", \"citation\"=>\"Baselga et al., 2012, Lancet\", \"pubmed_id\"=>\"22257673\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22257673\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>2, \"day\"=>18}, \"journal\"=>\"Lancet\", \"full_journal_title\"=>\"Lancet (London, England)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1382, \"name\"=>\"EID1382\", \"description\"=>\"Treatment of cells obtained from MMTV-NeuT mouse mammary tumors with the p110beta selective inhibitor TGX-221 showed significant but minimal reductions in lipid kinase activity and negligible reductions in phospho Akt levels. In simmilar cells subjected to p110alpha isoform knockout, TGX-221 treatment fully blocked Akt activation, indicating that TGX-221 was fully inhibiting the p110beta isoform. Athymic mice recieved MMTV-NeuT tumor cell transplants and were treated with TGX-221, which resulted in significant but minimal reduction in tumor growth compared to controls.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>338, \"name\"=>\"Tgx 221\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>937, \"name\"=>\"The p110α and p110β isoforms of PI3K play divergent roles in mammary gland development and tumorigenesis.\", \"citation\"=>\"Utermark et al., 2012, Genes Dev.\", \"pubmed_id\"=>\"22802530\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22802530\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3404385\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>7, \"day\"=>15}, \"journal\"=>\"Genes Dev.\", \"full_journal_title\"=>\"Genes & development\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1693, \"name\"=>\"EID1693\", \"description\"=>\"A 71-year old male with metastatic extramammary Paget’s disease of the scrotum overexpressing Her2/neu was treated with trastuzumab, achieving a complete response. At 12 months, he maintains stable disease.\", \"disease\"=>{\"id\"=>779, \"name\"=>\"Scrotum Paget's Disease\", \"display_name\"=>\"Scrotum Paget's Disease\", \"doid\"=>\"3444\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3444\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1142, \"name\"=>\"Metastatic Extramammary Paget's Disease of Scrotum Responds Completely to Single Agent Trastuzumab in a Hemodialysis Patient: Case Report, Molecular Profiling and Brief Review of the Literature.\", \"citation\"=>\"Barth et al., 2015, Case Rep Oncol Med\", \"pubmed_id\"=>\"25692060\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25692060\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4322830\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"Case Rep Oncol Med\", \"full_journal_title\"=>\"Case reports in oncological medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1393, \"name\"=>\"EID1393\", \"description\"=>\"155 patients with lung adenocarcinomas underwent rebiopsy and genotyping (EGFR, AKT1, BRAF, ERBB2, KRAS, MEK1, NRAS, PIK3CA and FISH for MET and HER2) after the development of acquired resistance to EGFR inhibitors (gefitinib or erlotinib). 2 of 24 patients tested were found to have HER2 amplification (but no baseline samples available for analysis). One patient of which had a metastasis sample which was reported to have only HER2 amplification whereas another autopsy specimen from the same patient was HER2 negative but EGFR T790M positive.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>14, \"name\"=>\"Gefitinib\", \"pubchem_id\"=>nil}, {\"id\"=>15, \"name\"=>\"Erlotinib\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>943, \"name\"=>\"Analysis of tumor specimens at the time of acquired resistance to EGFR-TKI therapy in 155 patients with EGFR-mutant lung cancers.\", \"citation\"=>\"Yu et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"23470965\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23470965\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3630270\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>4, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1891, \"name\"=>\"EID1891\", \"description\"=>\"Phase 2 trial of ERBB-inhibitor dacomitinib in stage IIIB/IV lung cancers with HER2 mutations or amplification. 4 patients with HER2 amplified NSCLC were enrolled in the trial, no one among them had a response.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>44, \"name\"=>\"Dacomitinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>729, \"name\"=>\"Targeting HER2 aberrations as actionable drivers in lung cancers: phase II trial of the pan-HER tyrosine kinase inhibitor dacomitinib in patients with HER2-mutant or amplified tumors.\", \"citation\"=>\"Kris et al., 2015, Ann. Oncol.\", \"pubmed_id\"=>\"25899785\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25899785\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>7}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology / ESMO\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1432, \"name\"=>\"EID1432\", \"description\"=>\"In a randomized phase II study, first line treatment of HER2-positive metastatic breast cancer with trastuzumab emtansine (T-DM1) resulted in improved progression free survival and fewer serious adverse effects when compared to treatment with combination trastuzumab plus docetaxel (HT) (median follow up 14 months, median PFS with T-DM1 14.2 vs 9.2 with HT).\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>293, \"name\"=>\"Trastuzumab Emtansine\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>970, \"name\"=>\"Phase II randomized study of trastuzumab emtansine versus trastuzumab plus docetaxel in patients with human epidermal growth factor receptor 2-positive metastatic breast cancer.\", \"citation\"=>\"Hurvitz et al., 2013, J. Clin. Oncol.\", \"pubmed_id\"=>\"23382472\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23382472\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>3, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1098, \"name\"=>\"EID1098\", \"description\"=>\"This phase II single arm study assessed trastuzumab monotherapy in HER2 positive endometrial cancers with tumor response as primary endpoint.  Participants had stage III, IV or recurrent HER2 positive endometrial cancer.  During the trial's initial phase, 23 of 132 patients screened met the criteria of IHC2+ or 3+ or FISH positivity for HER2 status.  In the absence of results, the trial entered phase B, where only patients with HER2 amplification by FISH were admitted.  The trial ended early due to poor accrual with 11 of 154 screened meeting the phase B criteria. No major objective tumor responses were observed in the study.  28% of the cancers were of the serous type that had shown trastuzumab response in case studies.  The investigators note that in other HER2 cancers, such as breast and gastrointestinal, trastuzumab in combination with chemotherapy has been found most effective, and could warrant more study.  Currently a clinical trial (NCT01367002) is evaluating carboplatin/paclitaxel with or without trastuzumab in uterine serous cancer.\", \"disease\"=>{\"id\"=>108, \"name\"=>\"Endometrial Cancer\", \"display_name\"=>\"Endometrial Cancer\", \"doid\"=>\"1380\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1380\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>757, \"name\"=>\"Phase II trial of trastuzumab in women with advanced or recurrent, HER2-positive endometrial carcinoma: a Gynecologic Oncology Group study.\", \"citation\"=>\"Fleming et al., 2010, Gynecol. Oncol.\", \"pubmed_id\"=>\"19840887\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19840887\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2804260\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>1}, \"journal\"=>\"Gynecol. Oncol.\", \"full_journal_title\"=>\"Gynecologic oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1077, \"name\"=>\"EID1077\", \"description\"=>\"CLEOPATRA (NCT00567190) was a Phase III, randomized and double-blind, placebo-controlled study of 808 patients with HER2-positive metastatic breast cancer.  Patients were HER2-positive by IHC3+ or FISH, and patients who had received prior hormonal treatment or adjuvant/neo-adjuvant therapy with or without trastuzumab for longer than 12 months before randomization were also eligible.  The two arms of the study were trastuzumab and docetaxel with either placebo or pertuzumab.  Median PFS in the placebo arm was 12.4 months while in the petuzumab arm 18.5 months was achieved.  Median OS in the control arm was 40.8 months and 56.5 months with pertuzumab.  These strong results make a case for dual antibody blockade in first line treatments of HER2-positive metastatic breast cancer.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>73, \"name\"=>\"Docetaxel\", \"pubchem_id\"=>nil}, {\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}, {\"id\"=>247, \"name\"=>\"Pertuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>745, \"name\"=>\"Pertuzumab, trastuzumab, and docetaxel in HER2-positive metastatic breast cancer.\", \"citation\"=>\"Swain et al., 2015, N. Engl. J. Med.\", \"pubmed_id\"=>\"25693012\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25693012\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>2, \"day\"=>19}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1096, \"name\"=>\"EID1096\", \"description\"=>\"In a study of 27 African American and caucasian patients, endometrial adenocarcinoma samples were assessed for HER2 via IHC staining, and association studies were performed using corresponding patient information.  Earlier deaths were seen with heavier HER2 expression.  Multivariate Cox regression indicated that shorter survival was associated with higher HER2 expression, and not with age or ethnicity, and that observed differences in survival rates between ethnicities can be ascribed to higher rates of HER2 overexpression in African American patients.  The authors conclude that high HER2 expression in USPC is associated with poor outcome and explains observed differences in outcome between ethnicities.\", \"disease\"=>{\"id\"=>185, \"name\"=>\"Uterine Corpus Serous Adenocarcinoma\", \"display_name\"=>\"Uterine Corpus Serous Adenocarcinoma\", \"doid\"=>\"5750\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5750\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>755, \"name\"=>\"Racial differences in the overexpression of epidermal growth factor type II receptor (HER2/neu): a major prognostic indicator in uterine serous papillary cancer.\", \"citation\"=>\"Santin et al., 2005, Am. J. Obstet. Gynecol.\", \"pubmed_id\"=>\"15746676\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15746676\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>3}, \"journal\"=>\"Am. J. Obstet. Gynecol.\", \"full_journal_title\"=>\"American journal of obstetrics and gynecology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1041, \"name\"=>\"EID1041\", \"description\"=>\"In this phase II trial with 2 arms, 103 patients with untreated stage IIIB/IV HER2 positive NSCLC were given gemcitabine-cisplatin with or without trastuzumab.  HER2 positivity was broadly defined as IHC2+/3+, positive by FISH, or increased circulating HER2 extracellular domain.  Significant differences between the treatment arms were not observed, but in a small subset of IHC3+/FISH positive patients, 5 of 6 responded to trastuzumab and had increased PFS of 8.5 months in comparison to 6.1 months in the IHC2+ group.  Although from 2004, this study remains cited in 2016 literature when discussing the unclear efficacy of trastuzumab in wild-type HER2 overexpressing NSCLC. The authors conclude that future studies of trastuzumab in HER2 positive NSCLC should focus on high expressing subjects.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>724, \"name\"=>\"Randomized phase II trial of gemcitabine-cisplatin with or without trastuzumab in HER2-positive non-small-cell lung cancer.\", \"citation\"=>\"Gatzemeier et al., 2004, Ann. Oncol.\", \"pubmed_id\"=>\"14679114\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/14679114\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2004, \"month\"=>1}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology / ESMO\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1122, \"name\"=>\"EID1122\", \"description\"=>\"HERA was a Phase III trial assessing application of trastuzumab in an extended adjuvant setting.  Patients were HER2 positive with completely excised invasive BC, node positive or negative, and having undergone prior adjuvant or neo-adjuvant chemotherapy.  In the 3 arm study patients were given trastuzumab courses of 1 year, 2 years, or untreated.  At the first planned interim analysis, trastuzumab treatment for one year was compared with observation alone.  A significant difference in disease-free survival was seen with 220 DFS events out of 1693 in the observation arm versus 127 DFS events out of 1694 in the 1 year trastuzumab arm.  1 year adjuvant trastuzumab is currently the standard of care.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>777, \"name\"=>\"Trastuzumab after adjuvant chemotherapy in HER2-positive breast cancer.\", \"citation\"=>\"Piccart-Gebhart et al., 2005, N. Engl. J. Med.\", \"pubmed_id\"=>\"16236737\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16236737\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>10, \"day\"=>20}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1561, \"name\"=>\"EID1561\", \"description\"=>\"In a panel of 47 breast cancer cell lines, HER2 overexpressing cells were among those more sensitive to CDK4/6 inhibitor Palbociclib (PD0332991). In the trastuzumab sensitive cell line Bt474, palbociclib synergized with trastuzumab in growth inhibition, while in 2 trastuzumab resistant cell lines, addition of trastuzumab to palbociclib treatment resulted in increased growth inhibition over palbociclib alone.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}, {\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1031, \"name\"=>\"PD 0332991, a selective cyclin D kinase 4/6 inhibitor, preferentially inhibits proliferation of luminal estrogen receptor-positive human breast cancer cell lines in vitro.\", \"citation\"=>\"Finn et al., 2009, Breast Cancer Res.\", \"pubmed_id\"=>\"19874578\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19874578\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2790859\", \"publication_date\"=>{\"year\"=>2009}, \"journal\"=>\"Breast Cancer Res.\", \"full_journal_title\"=>\"Breast cancer research : BCR\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1613, \"name\"=>\"EID1613\", \"description\"=>\"In a panel of 31 breast cancer cell lines with varied mutation backgrounds, 10 cell lines harbored HER2 (ERBB2) amplification. Sensitivity to the mTORC1/2 inhibitor PP242 was assessed, and 8/10 HER2 positive cell lines were reported to be sensitive (IC50 < 1 micro molar PP242).  8/21 without HER2 amplification were reported as resistant (IC50 > 1 micro molar PP242). This pattern of HER2 positive cell sensitivity was not reported with the rapamycin analog  mTORC1 inhibitor everolimus.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>179, \"name\"=>\"PP242\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1066, \"name\"=>\"PIK3CA mutation, but not PTEN loss of function, determines the sensitivity of breast cancer cells to mTOR inhibitory drugs.\", \"citation\"=>\"Weigelt et al., 2011, Oncogene\", \"pubmed_id\"=>\"21358673\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21358673\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>7, \"day\"=>21}, \"journal\"=>\"Oncogene\", \"full_journal_title\"=>\"Oncogene\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1022, \"name\"=>\"EID1022\", \"description\"=>\"ToGA, a Phase III study (NCT01041404) addressed use of trastuzumab in HER2 positive advanced gastric cancer, where chemotherapy using fluoropyrimidine-based and platinum-based combinations had been common practice.  Patients were given a chemotherapy regime containing capecitabine, flourouracil and cisplatin with (n=298) or without (n=296) trastuzumab, and had not been previously treated for metastatic cancer.  Study endpoint was overall survival, which was significantly different in the two populations (13.8 months with chemotherapy + trastuzumab and 11.1 months for chemotherapy alone).  The authors conclude that trastuzumab should be a standard option for HER2 positive advanced gastric cancer.\", \"disease\"=>{\"id\"=>147, \"name\"=>\"Gastric Adenocarcinoma\", \"display_name\"=>\"Gastric Adenocarcinoma\", \"doid\"=>\"3717\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3717\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>708, \"name\"=>\"Trastuzumab in combination with chemotherapy versus chemotherapy alone for treatment of HER2-positive advanced gastric or gastro-oesophageal junction cancer (ToGA): a phase 3, open-label, randomised controlled trial.\", \"citation\"=>\"Bang et al., 2010, Lancet\", \"pubmed_id\"=>\"20728210\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20728210\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>8, \"day\"=>28}, \"journal\"=>\"Lancet\", \"full_journal_title\"=>\"Lancet (London, England)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1010, \"name\"=>\"EID1010\", \"description\"=>\"With multiple mechanisms of HER reprogramming thought to be behind trastuzumab resistance, including increased expression of ErbB members, afatinib, which broadly and irreversibly targets ErbB receptors was tested in a single arm phase II trial on advanced stage HER2 positive metastatic breast cancer. From 41 treated patients, 10% had partial response, 37% had stable disease as best response and 46% achieved clinical benefit.  Median progression free survival (PFS) was 15.1 weeks and median overall survival (OS) was 61.0 weeks. The authors conclude that Afatinib monotherapy shows promise in treatment of metastatic HER2 positive breast cancer that has progressed under trastuzumab.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>146, \"name\"=>\"Afatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>663, \"name\"=>\"A phase II study of afatinib (BIBW 2992), an irreversible ErbB family blocker, in patients with HER2-positive metastatic breast cancer progressing after trastuzumab.\", \"citation\"=>\"Lin et al., 2012, Breast Cancer Res. Treat.\", \"pubmed_id\"=>\"22418700\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22418700\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3387495\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>6}, \"journal\"=>\"Breast Cancer Res. Treat.\", \"full_journal_title\"=>\"Breast cancer research and treatment\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1009, \"name\"=>\"EID1009\", \"description\"=>\"This Phase III study (EGF104900) was motivated by preclinical and other data demonstrating synergistic activity between lapatinib and trastuzumab against HER2 positive breast cancer (BC) cells.  In this study of HER2 positive metastatic BC which had progressed on trastuzumab, two arms were compared: lapatinib vs. lapatinib + trastuzumab. Progression free survival (PFS) was the primary endpoint. For 296 patients, 148 in each arm, median PFS of 8.1 weeks for lapatinib and 12.0 for combination therapy was observed (P=0.008).  The authors conclude that in the context of metastatic trastuzumab-progressed HER2 positive BC, trastuzumab with lapatinib TKI addition is a favorable treatment.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>45, \"name\"=>\"Lapatinib\", \"pubchem_id\"=>nil}, {\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>698, \"name\"=>\"Randomized study of Lapatinib alone or in combination with trastuzumab in women with ErbB2-positive, trastuzumab-refractory metastatic breast cancer.\", \"citation\"=>\"Blackwell et al., 2010, J. Clin. Oncol.\", \"pubmed_id\"=>\"20124187\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20124187\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>3, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>962, \"name\"=>\"EID962\", \"description\"=>\"Afatinib was growth-inhibitory for three pancreatic cancer cell lines but cytotoxic only toward BxPC3 (KRAS wt) and Capan-2 (KRAS mut) cells, both of which express high levels of EGFR, ERBB2, and ERBB3 receptors. Afatinib increased the radiosensitivity of BxPC3 and Capan-2 cells. In comparison, Panc-1 cells (KRAS mut) expressing low levels of EGFR family receptors were resistant to afatinib-induced radiosensitization.\", \"disease\"=>{\"id\"=>158, \"name\"=>\"Pancreatic Adenocarcinoma\", \"display_name\"=>\"Pancreatic Adenocarcinoma\", \"doid\"=>\"4074\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4074\"}, \"drugs\"=>[{\"id\"=>146, \"name\"=>\"Afatinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>664, \"name\"=>\"Afatinib, an Irreversible EGFR Family Inhibitor, Shows Activity Toward Pancreatic Cancer Cells, Alone and in Combination with Radiotherapy, Independent of KRAS Status.\", \"citation\"=>\"Huguet et al., 2016, Target Oncol\", \"pubmed_id\"=>\"26668065\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26668065\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>6}, \"journal\"=>\"Target Oncol\", \"full_journal_title\"=>\"Targeted oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>895, \"name\"=>\"EID895\", \"description\"=>\"141 patients were analyzed for ERBB2 (HER2) positivity by FISH and/or IHC. Only 6 (4.3%) were ERBB2 positive. These patients did not show a difference in outcome after capecitabine, oxaliplatin and chemoradiotherapy, with or without cetuximab.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}, {\"id\"=>32, \"name\"=>\"Capecitabine\", \"pubchem_id\"=>nil}, {\"id\"=>237, \"name\"=>\"Oxaliplatin\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>603, \"name\"=>\"HER2 in high-risk rectal cancer patients treated in EXPERT-C, a randomized phase II trial of neoadjuvant capecitabine and oxaliplatin (CAPOX) and chemoradiotherapy (CRT) with or without cetuximab.\", \"citation\"=>\"Sclafani et al., 2013, Ann. Oncol.\", \"pubmed_id\"=>\"24146218\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24146218\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>12}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology / ESMO\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>886, \"name\"=>\"EID886\", \"description\"=>\"Eight uterine cancer cell lines (four with ERBB2 amplification and four without) were treated with scalar concentrations of afatinib. Primary chemotherapy-resistant USC cell lines harbouring ERBB2 gene amplification were sensitive to afatinib exposure and significantly more sensitive than non-amplified cell lines.\", \"disease\"=>{\"id\"=>93, \"name\"=>\"Uterine Corpus Endometrial Carcinoma\", \"display_name\"=>\"Uterine Corpus Endometrial Carcinoma\", \"doid\"=>\"0050939\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050939\"}, \"drugs\"=>[{\"id\"=>146, \"name\"=>\"Afatinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>596, \"name\"=>\"Afatinib demonstrates remarkable activity against HER2-amplified uterine serous endometrial cancer in vitro and in vivo.\", \"citation\"=>\"Schwab et al., 2014, Br. J. Cancer\", \"pubmed_id\"=>\"25268372\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25268372\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4453741\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>10, \"day\"=>28}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1023, \"name\"=>\"EID1023\", \"description\"=>\"TyTAN, a Phase III study in Asian patients, looked at addition of tyrosine kinase inhibitor lapatinib to chemptherapy in advanced HER2 positive gastric cancer.  Primary endpoint was Overall Survival (OS).  Participants had progressed under previous treatment and only a small percentage had seen first line trastuzumab.  Patients were a mix of those having undergone gastrectomy and not.  OS was not significantly  different in the ITT population, but certain subpopulations showed significant differences.  In the subset of patients showing a stronger immunohistochemistry result for HER2 (IHC3+), progression free survival was significantly different at 5.6 months for lapatinib + pacitaxel vs. 4.2 months for pacitaxel alone.  The authors conclude that proof of concept for lapatanib addition to chemotherapy for certain subgroups is provided in this study.\", \"disease\"=>{\"id\"=>147, \"name\"=>\"Gastric Adenocarcinoma\", \"display_name\"=>\"Gastric Adenocarcinoma\", \"doid\"=>\"3717\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3717\"}, \"drugs\"=>[{\"id\"=>45, \"name\"=>\"Lapatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>709, \"name\"=>\"Lapatinib plus paclitaxel versus paclitaxel alone in the second-line treatment of HER2-amplified advanced gastric cancer in Asian populations: TyTAN--a randomized, phase III study.\", \"citation\"=>\"Satoh et al., 2014, J. Clin. Oncol.\", \"pubmed_id\"=>\"24868024\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24868024\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>1006, \"name\"=>\"EID1006\", \"description\"=>\"In-vitro studies using ERBB2 over-expressing breast cancer cell lines BT474 and SKBR3 (with ERBB2 amplification) showed apoptosis under either labatinib or trastuzumab treatment.  A marked synergistic apoptotic effect under lapatinib and trastuzumab co-treatment was observed in BT474 cells, and a milder synergistic effect was observed in SKBR3 cells.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>45, \"name\"=>\"Lapatinib\", \"pubchem_id\"=>nil}, {\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>695, \"name\"=>\"Combining lapatinib (GW572016), a small molecule inhibitor of ErbB1 and ErbB2 tyrosine kinases, with therapeutic anti-ErbB2 antibodies enhances apoptosis of ErbB2-overexpressing breast cancer cells.\", \"citation\"=>\"Xia et al., 2005, Oncogene\", \"pubmed_id\"=>\"16091755\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16091755\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>9, \"day\"=>15}, \"journal\"=>\"Oncogene\", \"full_journal_title\"=>\"Oncogene\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>760, \"name\"=>\"EID760\", \"description\"=>\"Xenografts established from breast cancer cell lines with ERBB2 amplification and PIK3CA mutation (BT474) or PIK3CA mutation alone (MCF7) were effectively inhibited in-vivo with an AKT1/2-inhibitor (AKTi-1/2; naphthyridinone).\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>169, \"name\"=>\"AKTi-1/2\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>503, \"name\"=>\"Breast tumor cells with PI3K mutation or HER2 amplification are selectively addicted to Akt signaling.\", \"citation\"=>\"She et al., 2008, PLoS ONE\", \"pubmed_id\"=>\"18725974\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18725974\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2516933\", \"publication_date\"=>{\"year\"=>2008}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}, {\"id\"=>761, \"name\"=>\"EID761\", \"description\"=>\"Oral neratinib showed objective response rates of 24 and 56% respectively in prior-trastuzumab and trastuzumab-naive cohorts in a Phase II trial with a primary endpoint of 16-week progression free survival.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>52, \"name\"=>\"Neratinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>504, \"name\"=>\"Neratinib, an irreversible ErbB receptor tyrosine kinase inhibitor, in patients with advanced ErbB2-positive breast cancer.\", \"citation\"=>\"Burstein et al., 2010, J. Clin. Oncol.\", \"pubmed_id\"=>\"20142587\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20142587\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>3, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>306, \"phenotypes\"=>[]}]}, {\"id\"=>321, \"entrez_name\"=>\"MAPK1\", \"entrez_id\"=>5594, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>4532, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>4.0, \"coordinates\"=>{\"chromosome\"=>\"22\", \"start\"=>22108789, \"stop\"=>22221919, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000215832.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4532/summary/variants/321/summary#variant\", \"evidence_items\"=>[{\"id\"=>792, \"name\"=>\"EID792\", \"description\"=>\"Resistance to WZ4002 was induced in cell lines. Amplification of MAPK1 or downregulation of ERK-pathway inhibitors led to resistance to EGFR inhibition with WZ4002. Chemical inhibition (MEK inhibition with either CI-1040 or GSK1120212) or shRNA knockdown of MAPK1 resensitized cells to EGFR inhibition with WZ4002 in-vitro. Additionally, MAPK1 amplification was identified in 1/21 erlotinib-resistant NSCLC patients. MAPK1 amplification was not observed in the pretreatment tumor, only the post-treatment resistant tumor which lacked other more common drug-resistance mechanisms EGFR T790M or MET amplification.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>147, \"name\"=>\"WZ4002\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>526, \"name\"=>\"Reactivation of ERK signaling causes resistance to EGFR kinase inhibitors.\", \"citation\"=>\"Ercan et al., 2012, Cancer Discov\", \"pubmed_id\"=>\"22961667\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22961667\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3477553\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>10}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>321, \"phenotypes\"=>[]}]}, {\"id\"=>330, \"entrez_name\"=>\"NOTCH1\", \"entrez_id\"=>4851, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>50, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>12.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>139388896, \"stop\"=>139440314, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000277541.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/50/summary/variants/330/summary#variant\", \"evidence_items\"=>[{\"id\"=>811, \"name\"=>\"EID811\", \"description\"=>\"NOTCH1 copy-number gain in a patient-derived tumor xenograft model of colorectal cancer, was associated with increased NOTCH1 and JAG1 activity and response to a NOTCH1-targeting antibody (PF-06293622).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>230, \"name\"=>\"NOTCH1 Antibody\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>544, \"name\"=>\"A NOTCH1 gene copy number gain is a prognostic indicator of worse survival and a predictive biomarker to a Notch1 targeting antibody in colorectal cancer.\", \"citation\"=>\"Arcaroli et al., 2016, Int. J. Cancer\", \"pubmed_id\"=>\"26152787\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26152787\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4618491\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>1, \"day\"=>1}, \"journal\"=>\"Int. J. Cancer\", \"full_journal_title\"=>\"International journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>330, \"phenotypes\"=>[]}, {\"id\"=>812, \"name\"=>\"EID812\", \"description\"=>\"NOTCH1 amplifications (as determined by FISH) were significantly (log-rank p = 0.025) associated with worse survival in a cohort of 116 patients with colorectal cancer.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>544, \"name\"=>\"A NOTCH1 gene copy number gain is a prognostic indicator of worse survival and a predictive biomarker to a Notch1 targeting antibody in colorectal cancer.\", \"citation\"=>\"Arcaroli et al., 2016, Int. J. Cancer\", \"pubmed_id\"=>\"26152787\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26152787\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4618491\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>1, \"day\"=>1}, \"journal\"=>\"Int. J. Cancer\", \"full_journal_title\"=>\"International journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>330, \"phenotypes\"=>[]}]}, {\"id\"=>358, \"entrez_name\"=>\"RSF1\", \"entrez_id\"=>51773, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>12121, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"11\", \"start\"=>77371041, \"stop\"=>77532063, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000308488.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/12121/summary/variants/358/summary#variant\", \"evidence_items\"=>[{\"id\"=>857, \"name\"=>\"EID857\", \"description\"=>\"In retrospective study, 413 patients were assessed for RSF1 amplification. 28 patients with RSF1 amplification did not show a benefit from adjuvant tamoxifen compared to placebo (12 treated vs 16 placebo). 381 patients without amplification did show a benefit compared to placebo (187 treated vs 194 placebo).\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>53, \"name\"=>\"Tamoxifen\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>574, \"name\"=>\"RSF1 and not cyclin D1 gene amplification may predict lack of benefit from adjuvant tamoxifen in high-risk pre-menopausal women in the MA.12 randomized clinical trial.\", \"citation\"=>\"Keilty et al., 2013, PLoS ONE\", \"pubmed_id\"=>\"24367492\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24367492\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3868649\", \"publication_date\"=>{\"year\"=>2013}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>358, \"phenotypes\"=>[]}]}, {\"id\"=>371, \"entrez_name\"=>\"TOP1\", \"entrez_id\"=>7150, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>5845, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"20\", \"start\"=>39657458, \"stop\"=>39753127, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000361337.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5845/summary/variants/371/summary#variant\", \"evidence_items\"=>[{\"id\"=>888, \"name\"=>\"EID888\", \"description\"=>\"FFPE tumor samples from 78 patients with colorectal cancer, who had received irinotecan monotherapy were analyzed with a FISH dual-probe.\\nTOP1 gain was nonsignificantly associated with objective response to irinotecan Odds ratio (OR): 1.62; p = 0.07.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>101, \"name\"=>\"Irinotecan\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>598, \"name\"=>\"Assessment of the topoisomerase I gene copy number as a predictive biomarker of objective response to irinotecan in metastatic colorectal cancer.\", \"citation\"=>\"Nygård et al., 2014, Scand. J. Gastroenterol.\", \"pubmed_id\"=>\"24256029\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24256029\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>1}, \"journal\"=>\"Scand. J. Gastroenterol.\", \"full_journal_title\"=>\"Scandinavian journal of gastroenterology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>371, \"phenotypes\"=>[]}]}, {\"id\"=>379, \"entrez_name\"=>\"TYMS\", \"entrez_id\"=>7298, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>5971, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"18\", \"start\"=>657604, \"stop\"=>673578, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000323274.10\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5971/summary/variants/379/summary#variant\", \"evidence_items\"=>[{\"id\"=>903, \"name\"=>\"EID903\", \"description\"=>\"TYMS copy number was lower in clinical samples responding to pemetrexed in combination with platinum (n=25). The same was found in 17 cell lines.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>156, \"name\"=>\"Pemetrexed\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>610, \"name\"=>\"Thymidylate synthase gene copy number as a predictive marker for response to pemetrexed treatment of lung adenocarcinoma.\", \"citation\"=>\"Kasai et al., 2013, Anticancer Res.\", \"pubmed_id\"=>\"23645741\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23645741\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>5}, \"journal\"=>\"Anticancer Res.\", \"full_journal_title\"=>\"Anticancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>379, \"phenotypes\"=>[]}]}, {\"id\"=>407, \"entrez_name\"=>\"ABCC3\", \"entrez_id\"=>8714, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>6906, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"17\", \"start\"=>48712218, \"stop\"=>48769613, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000285238.8\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/6906/summary/variants/407/summary#variant\", \"evidence_items\"=>[{\"id\"=>951, \"name\"=>\"EID951\", \"description\"=>\"44 cell lines and 145 patient samples were utilized in various aspects of this study. 31 breast cancer cell lines for in vitro sensitivity to paclitaxel and Monomethyl Auristatin E (MMAE). Genome wide amplification was assessed by Affymetrix SNP arrays and expression was assessed by Affymetrix gene expression arrays. ABCC3 amplification (and expression) in ERBB2 (HER2) amplified breast cancer cell lines was associated with resistance to paclitaxel and  MMAE. Three independently derived EVSA-T cell lines overexpressing ABCC3 were at least 20-fold less sensitive to paclitaxel and MMAE based on EC50 values.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>81, \"name\"=>\"Paclitaxel\", \"pubchem_id\"=>nil}, {\"id\"=>183, \"name\"=>\"Monomethyl Auristatin E\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>653, \"name\"=>\"Functional genomics identifies ABCC3 as a mediator of taxane resistance in HER2-amplified breast cancer.\", \"citation\"=>\"O'Brien et al., 2008, Cancer Res.\", \"pubmed_id\"=>\"18593940\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18593940\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2008, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>407, \"phenotypes\"=>[]}]}, {\"id\"=>409, \"entrez_name\"=>\"ASNS\", \"entrez_id\"=>440, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>446, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>97481430, \"stop\"=>97501854, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000175506.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/446/summary/variants/409/summary#variant\", \"evidence_items\"=>[{\"id\"=>953, \"name\"=>\"EID953\", \"description\"=>\"ASNS amplification and expression was associated with resistance to L-asparaginase in ovarian cancer and leukemia cell lines. ASNS silencing increased sensitivity to l-ASP in 2/3 of ovarian cancer cell lines.\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>258, \"name\"=>\"L-asparaginase\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>655, \"name\"=>\"Asparagine synthetase as a causal, predictive biomarker for L-asparaginase activity in ovarian cancer cells.\", \"citation\"=>\"Lorenzi et al., 2006, Mol. Cancer Ther.\", \"pubmed_id\"=>\"17088436\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17088436\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2006, \"month\"=>11}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>409, \"phenotypes\"=>[]}]}, {\"id\"=>464, \"entrez_name\"=>\"MET\", \"entrez_id\"=>4233, \"name\"=>\"EXON 14 MUTATION + AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>52, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}, {\"id\"=>38, \"name\"=>\"exon_loss_variant\", \"display_name\"=>\"Exon Loss Variant\", \"so_id\"=>\"SO:0001572\", \"description\"=>\"A sequence variant whereby an exon is lost from the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001572\"}], \"civic_actionability_score\"=>7.5, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>116312459, \"stop\"=>116436396, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000318493.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/52/summary/variants/464/summary#variant\", \"evidence_items\"=>[{\"id\"=>1095, \"name\"=>\"EID1095\", \"description\"=>\"64-year old female (non-smoker) was diagnosed with NSCLC. A c.3028 G->A mutation was identified in exon 14 resulting in exon 14 skipping. Further, amplification of the MET gene was observed. The patient was given crizotinib and CT scans showed a positive response after 8 weeks of treatment.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>754, \"name\"=>\"MET Exon 14 Mutations in Non-Small-Cell Lung Cancer Are Associated With Advanced Age and Stage-Dependent MET Genomic Amplification and c-Met Overexpression.\", \"citation\"=>\"Awad et al., 2016, J. Clin. Oncol.\", \"pubmed_id\"=>\"26729443\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26729443\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>3, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>464, \"phenotypes\"=>[]}]}, {\"id\"=>553, \"entrez_name\"=>\"CDK4\", \"entrez_id\"=>1019, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>13, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>30.0, \"coordinates\"=>{\"chromosome\"=>\"12\", \"start\"=>58141510, \"stop\"=>58146304, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000257904.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/13/summary/variants/553/summary#variant\", \"evidence_items\"=>[{\"id\"=>4841, \"name\"=>\"EID4841\", \"description\"=>\"More than 90% of well-differentiated or dedifferentiated liposarcomas (WD/DDLS) have CDK4 amplification. \\nA prior phase 2 study demonstrated that treatment with palbociclib (200 mg/d for 14 days every 21 days) resulted in clinical benefit in WD/DDLS but moderate hematologic toxic effects. In this phase 2, nonrandomized, open-label clinical trial conducted at the Memorial Sloan Kettering Cancer Center, 60 patients 18 years and older with advanced WD/DDLS and measurable disease by RECIST 1.1 were enrolled from December 2011 to January 2014 and followed to March 2015. Patients received oral palbociclib at new dose scheme, 125 mg/d for 21 days in 28-day cycles. Progression-free survival at 12 weeks was 57.2% and overall median PFS was 17.9 weeks. There was 1 complete response. Toxic effects were primarily hematologic and included neutropenia (grade 3, n = 20 [33%]; grade 4, n = 2 [3%]) but no neutropenic fever. \\nCompared to other dose scheme in PMID 23569312 with 200mg/d: 12week PFS 66%, overall median PFS 18 weeks.\", \"disease\"=>{\"id\"=>137, \"name\"=>\"Liposarcoma\", \"display_name\"=>\"Liposarcoma\", \"doid\"=>\"3382\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3382\"}, \"drugs\"=>[{\"id\"=>429, \"name\"=>\"Palbociclib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2145, \"name\"=>\"Progression-Free Survival Among Patients With Well-Differentiated or Dedifferentiated Liposarcoma Treated With CDK4 Inhibitor Palbociclib: A Phase 2 Clinical Trial.\", \"citation\"=>\"Dickson et al., 2016, JAMA Oncol\", \"pubmed_id\"=>\"27124835\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27124835\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4991028\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"JAMA Oncol\", \"full_journal_title\"=>\"JAMA oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>553, \"phenotypes\"=>[]}, {\"id\"=>1366, \"name\"=>\"EID1366\", \"description\"=>\"In a phase 2 clinical trial, NCT01209598, 30 patients with CDK4-amplified and RB-expressing well differentiated and dedifferentiated liposarcomas (WDLS/DDLS) were treated with a selective CDK 4/6 inhibitor, PD0332991. Treatment with PD0332991 was associated with favorable progression free survival rates (66% PFS rate, primary end point being greater than 40%) suggesting that CDK4 amplification is predictive of sensitivity to CDK4 inhibitor therapy.\", \"disease\"=>{\"id\"=>137, \"name\"=>\"Liposarcoma\", \"display_name\"=>\"Liposarcoma\", \"doid\"=>\"3382\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3382\"}, \"drugs\"=>[{\"id\"=>149, \"name\"=>\"PD0332991\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>360, \"name\"=>\"Phase II trial of the CDK4 inhibitor PD0332991 in patients with advanced CDK4-amplified well-differentiated or dedifferentiated liposarcoma.\", \"citation\"=>\"Dickson et al., 2013, J. Clin. Oncol.\", \"pubmed_id\"=>\"23569312\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23569312\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3661937\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>6, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>553, \"phenotypes\"=>[]}]}, {\"id\"=>573, \"entrez_name\"=>\"RICTOR\", \"entrez_id\"=>253260, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>20480, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"5\", \"start\"=>38938021, \"stop\"=>39074510, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000357387.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/20480/summary/variants/573/summary#variant\", \"evidence_items\"=>[{\"id\"=>1916, \"name\"=>\"EID1916\", \"description\"=>\"RICTOR amplification was observed in ca. 14% of 42 SCLC patients with panel sequencing results. 4 cell lines (2 with RICTOR CN gain, 2 without) were tested for the efficacy of mTOR pathway inhibitors. Cells with RICTOR copy number gain showed increased sensitivity to three mTOR inhibitors, AZD8055, AZD2014 and INK128 in cell growth assays. SCLC cells with RICTOR CN gain also migrated more rapidly in chemotaxis and scratch wound assays and patients with RICTOR amplification had a significantly decreased OS (p = 0.021).\", \"disease\"=>{\"id\"=>174, \"name\"=>\"Lung Small Cell Carcinoma\", \"display_name\"=>\"Lung Small Cell Carcinoma\", \"doid\"=>\"5409\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5409\"}, \"drugs\"=>[{\"id\"=>124, \"name\"=>\"AZD8055\", \"pubchem_id\"=>nil}, {\"id\"=>434, \"name\"=>\"AZD2014\", \"pubchem_id\"=>nil}, {\"id\"=>435, \"name\"=>\"INK128\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1340, \"name\"=>\"RICTOR amplification identifies a subgroup in small cell lung cancer and predicts response to drugs targeting mTOR.\", \"citation\"=>\"Sakre et al., 2017, Oncotarget\", \"pubmed_id\"=>\"27863413\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27863413\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>1, \"day\"=>24}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>573, \"phenotypes\"=>[]}, {\"id\"=>1915, \"name\"=>\"EID1915\", \"description\"=>\"Tumor samples from 640 patients with metastatic solid tumors were prospectively analyzed with a next-generation sequencing assay to identify molecular targets. 13 (2%) of 640 patients had a RICTOR-aamplification (6 gastric, 3 NSCLC, 1 SCLC, 1 CRC, 1 sarcoma, 1 MUO). Of these, 7 patients had RICTOR protein overexpression by IHC. The prevalence of RICTOR amplification in gastric cancer was 3.8 % (6/160). FISH testing confirmed amplificationin 5/13 samples, including 4 gastric cancers and 1 lung cancer sample. A RICTOR-amplified patient-derived cell (PDC) line was generated and used to investigate the effectiveness of selective AKT, mTORC1, and mTORC1/2 inhibition. Sensitivity to the mTORC1/2 inhibitor (AZD2014) was seen which could be reversed by the knockdown of RICTOR.\", \"disease\"=>{\"id\"=>147, \"name\"=>\"Gastric Adenocarcinoma\", \"display_name\"=>\"Gastric Adenocarcinoma\", \"doid\"=>\"3717\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3717\"}, \"drugs\"=>[{\"id\"=>434, \"name\"=>\"AZD2014\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1339, \"name\"=>\"Rapamycin-insensitive companion of mTOR (RICTOR) Amplification Defines a Subset of Advanced Gastric Cancer and is Sensitive to AZD2014-mediated mTORC1/2 Inhibition.\", \"citation\"=>\"Kim et al., 2016, Ann. Oncol.\", \"pubmed_id\"=>\"28028034\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28028034\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>12, \"day\"=>27}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>573, \"phenotypes\"=>[]}, {\"id\"=>1440, \"name\"=>\"EID1440\", \"description\"=>\"Study describes an 18 year old never smoker patient presenting with lung adenocarcinoma. A genomic profiling assay (FoundationOne) revealed a 7-copy amplification of RICTOR as the only somatic alteration. RICTOR amplification was confirmed via FISH and IHC. On two separate treatments with dual mTOR1/2 inhibitors patient exhibited stable disease for more than 18 months.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>347, \"name\"=>\"Cc-223\", \"pubchem_id\"=>nil}, {\"id\"=>348, \"name\"=>\"MLN0128\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>973, \"name\"=>\"RICTOR Amplification Defines a Novel Subset of Patients with Lung Cancer Who May Benefit from Treatment with mTORC1/2 Inhibitors.\", \"citation\"=>\"Cheng et al., 2015, Cancer Discov\", \"pubmed_id\"=>\"26370156\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26370156\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4670806\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>12}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>573, \"phenotypes\"=>[]}]}, {\"id\"=>586, \"entrez_name\"=>\"KIT\", \"entrez_id\"=>3815, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>29, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>55524085, \"stop\"=>55606881, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000288135.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/29/summary/variants/586/summary#variant\", \"evidence_items\"=>[{\"id\"=>5907, \"name\"=>\"EID5907\", \"description\"=>\"This phase II trial showed that nilotinib demonstrates response in a subset of melanoma patients (acral and mucosal subtypes) with KIT mutations. Median duration of response was 34 weeks. Of the 42 patients, 7 responded - 6 out of 27 with KIT mutations (exon 11: 5 patients; exon 17: 1 patient), and 1 of 15 patients with a high-level KIT amplification had a shorter response (5 weeks).\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>13, \"name\"=>\"Nilotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2383, \"name\"=>\"Phase II Trial of Nilotinib in Patients With Metastatic Malignant Melanoma Harboring KIT Gene Aberration: A Multicenter Trial of Korean Cancer Study Group (UN10-06).\", \"citation\"=>\"Lee et al., 2015, Oncologist\", \"pubmed_id\"=>\"26424760\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26424760\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4718426\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>11}, \"journal\"=>\"Oncologist\", \"full_journal_title\"=>\"The oncologist\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>586, \"phenotypes\"=>[]}, {\"id\"=>1466, \"name\"=>\"EID1466\", \"description\"=>\"Phase 2 trial in metastatic mucosal, acral or chronically sun-damaged melanoma. Twenty-five patients were enrolled (24 evaluable). Eight patients (33%) had tumors with KIT mutations, 11 (46%) with KIT amplifications, and five (21%) with both. Median follow-up was 10.6 months. Best overall response rate was statistically significantly different by mutation status: 7 of 13 or 54% KIT mutated v 0% KIT amplified only.\", \"disease\"=>{\"id\"=>83, \"name\"=>\"Mucosal Melanoma\", \"display_name\"=>\"Mucosal Melanoma\", \"doid\"=>\"0050929\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050929\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>85, \"name\"=>\"Imatinib for melanomas harboring mutationally activated or amplified KIT arising on mucosal, acral, and chronically sun-damaged skin.\", \"citation\"=>\"Hodi et al., 2013, J. Clin. Oncol.\", \"pubmed_id\"=>\"23775962\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23775962\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4878082\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>9, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>586, \"phenotypes\"=>[]}]}, {\"id\"=>591, \"entrez_name\"=>\"RAF1\", \"entrez_id\"=>5894, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>4767, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>12625100, \"stop\"=>12705725, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000251849.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4767/summary/variants/591/summary#variant\", \"evidence_items\"=>[{\"id\"=>1496, \"name\"=>\"EID1496\", \"description\"=>\"Retrospective analysis of somatic mutations and copy number changes in 119 patients treated with carboplatin, paclitaxel, ± sorafenib (CPS) in the E2603 trial. CPS therapy was associated with improved progression-free survival (PFS) compared with CP in patients with tumors that were positive for RAF1 gene copy gains (HR = 0.372; P = 0.025).\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>6, \"name\"=>\"Sorafenib\", \"pubchem_id\"=>nil}, {\"id\"=>81, \"name\"=>\"Paclitaxel\", \"pubchem_id\"=>nil}, {\"id\"=>93, \"name\"=>\"Carboplatin\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1002, \"name\"=>\"Copy Number Changes Are Associated with Response to Treatment with Carboplatin, Paclitaxel, and Sorafenib in Melanoma.\", \"citation\"=>\"Wilson et al., 2016, Clin. Cancer Res.\", \"pubmed_id\"=>\"26307133\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26307133\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4821426\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>1, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>591, \"phenotypes\"=>[]}]}, {\"id\"=>592, \"entrez_name\"=>\"KRAS\", \"entrez_id\"=>3845, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>30, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"12\", \"start\"=>25362365, \"stop\"=>25403737, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000256078.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/30/summary/variants/592/summary#variant\", \"evidence_items\"=>[{\"id\"=>5836, \"name\"=>\"EID5836\", \"description\"=>\"In this study, KRAS gain (KRAS/CEP12q ratio >1 and <2) or amplification (ratio >2) was detected in 3% (13 of 414) of the primary endometrial carcinomas investigated. Presence of KRAS gain or amplification were highly significantly associated with poor prognosis with a 46% 5-year survival compared with 87% for patients with unamplified status (P<0.001). Amplification of KRAS maintains its independent prognostic impact in Cox multivariate analysis when adjusted for age, histological subtype, grade and FIGO stage (HR=2.6, 95% CI: 1.2–5.8, P=0.02). In contrast, presence of KRAS mutation did not influence prognosis (P=0.67)\", \"disease\"=>{\"id\"=>108, \"name\"=>\"Endometrial Cancer\", \"display_name\"=>\"Endometrial Cancer\", \"doid\"=>\"1380\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1380\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2362, \"name\"=>\"KRAS gene amplification and overexpression but not mutation associates with aggressive and metastatic endometrial cancer.\", \"citation\"=>\"Birkeland et al., 2012, Br. J. Cancer\", \"pubmed_id\"=>\"23099803\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23099803\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3516681\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>12, \"day\"=>4}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>592, \"phenotypes\"=>[]}, {\"id\"=>2928, \"name\"=>\"EID2928\", \"description\"=>\"Paired pre-treatment and post-progression tumor biopsies from BRAF-mutant CRC patients treated with RAF inhibitor combinations were analyzed. Alterations in MAPK pathway genes were found in resistant tumors not present in matched pre-treatment tumors, including KRAS amplification, BRAF amplification, and a MEK1 mutation.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}, {\"id\"=>483, \"name\"=>\"Encorafenib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"submitted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1681, \"name\"=>\"Clinical Acquired Resistance to RAF Inhibitor Combinations in BRAF-Mutant Colorectal Cancer through MAPK Pathway Alterations.\", \"citation\"=>\"Ahronian et al., 2015, Cancer Discov\", \"pubmed_id\"=>\"25673644\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25673644\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4390490\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>4}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>592, \"phenotypes\"=>[]}, {\"id\"=>1497, \"name\"=>\"EID1497\", \"description\"=>\"Retrospective analysis of somatic mutations and copy number changes in 119 patients treated with carboplatin, paclitaxel, ± sorafenib (CPS) in the E2603 trial. CPS therapy was associated with improved overall survival compared with CP in patients with tumors positive for KRAS gene copy gains (HR = 0.25; P = 0.035).\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>6, \"name\"=>\"Sorafenib\", \"pubchem_id\"=>nil}, {\"id\"=>73, \"name\"=>\"Docetaxel\", \"pubchem_id\"=>nil}, {\"id\"=>93, \"name\"=>\"Carboplatin\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1002, \"name\"=>\"Copy Number Changes Are Associated with Response to Treatment with Carboplatin, Paclitaxel, and Sorafenib in Melanoma.\", \"citation\"=>\"Wilson et al., 2016, Clin. Cancer Res.\", \"pubmed_id\"=>\"26307133\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26307133\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4821426\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>1, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>592, \"phenotypes\"=>[]}]}, {\"id\"=>629, \"entrez_name\"=>\"FGFR2\", \"entrez_id\"=>2263, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>22, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>25.0, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>123239371, \"stop\"=>123357917, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000457416.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/22/summary/variants/629/summary#variant\", \"evidence_items\"=>[{\"id\"=>1990, \"name\"=>\"EID1990\", \"description\"=>\"Nine patients with FGFR2-amplified gastroesophageal cancer were treated with AZD4547.  Of these 9 patients, 3 showed confirmed responses to AZD4547. Median progression-free survival was 6.6 months (range=6.2-10.5). Two additional patients had a response in day 15 using FDG-PET but were not considered complete responders by RECIST criteria. High-level FGFR2-amplified (arbitrarily defined as FISH-ratio > 5) cancers had a higher likelihood of response and were associated with a substantially higher expression of both FGFR2 mRNA and FGFR2 protein (IHC). In vitro and in PDX models, FGFR2-amplified cell lines were sensitive to FGFR inhibition. Further preclinical analyses suggested, that FGFR2 amplification mainly signals via the PI3K pathway.\", \"disease\"=>{\"id\"=>1322, \"name\"=>\"Stomach Carcinoma\", \"display_name\"=>\"Stomach Carcinoma\", \"doid\"=>\"5517\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5517\"}, \"drugs\"=>[{\"id\"=>452, \"name\"=>\"AZD4547\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1397, \"name\"=>\"High-Level Clonal FGFR Amplification and Response to FGFR Inhibition in a Translational Clinical Trial.\", \"citation\"=>\"Pearson et al., 2016, Cancer Discov\", \"pubmed_id\"=>\"27179038\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27179038\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5338732\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>8}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>629, \"phenotypes\"=>[]}, {\"id\"=>1605, \"name\"=>\"EID1605\", \"description\"=>\"Dovitinib was evaluated in-vitro, in xenograft and in a phase 2 study. Dovitinib inhibited proliferation in one FGFR1- and one FGFR2-amplified breast cancer cell lines at higher IC50 values than in 11 nonamplified cell lines. Dovitinib inhibited tumor growth and caused regression in both a FGFR1- and FGFR2-amplified xenograft model. 81 patients were enrolled in the trial and treated with dovitinib. 2 patients were reported to harbor FGFR2 amplifications and both had a reduction in tumor size of 18.5 and 28.2%, respectively. When FGFR1, FGF3 and FGFR2 amplifications were combined, a combined FGF-pathway amplified patient group had a higher mean reduction in target lesion size than patients with FGF-non-amplified tumors.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>228, \"name\"=>\"Dovitinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1064, \"name\"=>\"Targeting FGFR with dovitinib (TKI258): preclinical and clinical data in breast cancer.\", \"citation\"=>\"André et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"23658459\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23658459\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>629, \"phenotypes\"=>[]}]}, {\"id\"=>630, \"entrez_name\"=>\"FGF3\", \"entrez_id\"=>2248, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>1873, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"11\", \"start\"=>69624992, \"stop\"=>69633792, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000334134.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1873/summary/variants/630/summary#variant\", \"evidence_items\"=>[{\"id\"=>1966, \"name\"=>\"EID1966\", \"description\"=>\"FGF3/FGF4 amplification was observed in around 2% of HCCs. Although the sample size was relatively small, FGF3/FGF4 amplification, a poorly differentiated histological type, and multiple lung metastases were frequently observed in responders to sorafenib.\", \"disease\"=>{\"id\"=>196, \"name\"=>\"Hepatocellular Carcinoma\", \"display_name\"=>\"Hepatocellular Carcinoma\", \"doid\"=>\"684\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:684\"}, \"drugs\"=>[{\"id\"=>6, \"name\"=>\"Sorafenib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1106, \"name\"=>\"FGF3/FGF4 amplification and multiple lung metastases in responders to sorafenib in hepatocellular carcinoma.\", \"citation\"=>\"Arao et al., 2013, Hepatology\", \"pubmed_id\"=>\"22890726\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22890726\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>4}, \"journal\"=>\"Hepatology\", \"full_journal_title\"=>\"Hepatology (Baltimore, Md.)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>630, \"phenotypes\"=>[]}, {\"id\"=>1606, \"name\"=>\"EID1606\", \"description\"=>\"81 patients were enrolled in a clinical trial with the FGFR inhibitor dovitinib. Four of these patients had FGF3 amplifications (qPCR) and showed tumor reductions of 100%, 30.8%, 23.0%, and 7.5%, respectively. 3 of these 4 patients also had FGFR1 amplifications (≥6 copies by qPCR) and the fourth presented with FGFR1-gene gain (3.4 copies of FGFR1 by qPCR and SISH negative).\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>228, \"name\"=>\"Dovitinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1064, \"name\"=>\"Targeting FGFR with dovitinib (TKI258): preclinical and clinical data in breast cancer.\", \"citation\"=>\"André et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"23658459\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23658459\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>630, \"phenotypes\"=>[]}]}, {\"id\"=>635, \"entrez_name\"=>\"AKT2\", \"entrez_id\"=>208, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>254, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>2.5, \"coordinates\"=>{\"chromosome\"=>\"19\", \"start\"=>40736224, \"stop\"=>40791302, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000392038.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/254/summary/variants/635/summary#variant\", \"evidence_items\"=>[{\"id\"=>1621, \"name\"=>\"EID1621\", \"description\"=>\"Case report of a 74-year old woman with EGFR- and ALK-negative lung adenocarcinoma. FISH identified a deletion of 5'RET (FISH, indicating RET-gene rearrangement) and genomic testing confirmed a fusion of KIF5B and RET, as well as an AKT2 gene amplification. The patient was enrolled in a clinical trial with vandetanib and everolimus (NCT01582191) with everolimus used to enhance brain accumulation of vandetanib. After two cycles of therapy a repeat MRI brain showed a decrease in the intracranial disease burden and PET/CT showed systemic response as well.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}, {\"id\"=>388, \"name\"=>\"Vandetanib\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1074, \"name\"=>\"Systemic and CNS activity of the RET inhibitor vandetanib combined with the mTOR inhibitor everolimus in KIF5B-RET re-arranged non-small cell lung cancer with brain metastases.\", \"citation\"=>\"Subbiah et al., 2015, Lung Cancer\", \"pubmed_id\"=>\"25982012\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25982012\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4998046\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>7}, \"journal\"=>\"Lung Cancer\", \"full_journal_title\"=>\"Lung cancer (Amsterdam, Netherlands)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>635, \"phenotypes\"=>[]}]}, {\"id\"=>716, \"entrez_name\"=>\"PDGFRA\", \"entrez_id\"=>5156, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>38, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>55095264, \"stop\"=>55164414, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000257290.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/38/summary/variants/716/summary#variant\", \"evidence_items\"=>[{\"id\"=>1771, \"name\"=>\"EID1771\", \"description\"=>\"In the sunitinib-sensitive adenosquamous NSCLC cell line, PDGFRA expression was associated with focal PFGRA gene amplification, which was similarly detected in a small fraction of squamous cell NSCLC primary tumor specimens. Moreover, in this NSCLC cell line, focal amplification of the gene encoding the PDGFR ligand PDGFC was also detected.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>157, \"name\"=>\"Sunitinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>2, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1238, \"name\"=>\"Ligand-dependent platelet-derived growth factor receptor (PDGFR)-alpha activation sensitizes rare lung cancer and sarcoma cells to PDGFR kinase inhibitors.\", \"citation\"=>\"McDermott et al., 2009, Cancer Res.\", \"pubmed_id\"=>\"19366796\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19366796\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2676215\", \"publication_date\"=>{\"year\"=>2009, \"month\"=>5, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>716, \"phenotypes\"=>[]}]}, {\"id\"=>1269, \"entrez_name\"=>\"BRAF\", \"entrez_id\"=>673, \"name\"=>\"AMPLIFICATION\", \"description\"=>\"\", \"gene_id\"=>5, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>123, \"name\"=>\"transcript_amplification\", \"display_name\"=>\"Transcript Amplification\", \"so_id\"=>\"SO:0001889\", \"description\"=>\"A feature amplification of a region containing a transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001889\"}], \"civic_actionability_score\"=>7.5, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>140434279, \"stop\"=>140624564, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000288602.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5/summary/variants/1269/summary#variant\", \"evidence_items\"=>[{\"id\"=>2929, \"name\"=>\"EID2929\", \"description\"=>\"Paired pre-treatment and post-progression tumor biopsies from BRAF-mutant CRC patients treated with RAF inhibitor combinations were analyzed. Alterations in MAPK pathway genes were found in resistant tumors not present in matched pre-treatment tumors, including KRAS amplification, BRAF amplification, and a MEK1 mutation.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>22, \"name\"=>\"Dabrafenib\", \"pubchem_id\"=>nil}, {\"id\"=>28, \"name\"=>\"Panitumumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1681, \"name\"=>\"Clinical Acquired Resistance to RAF Inhibitor Combinations in BRAF-Mutant Colorectal Cancer through MAPK Pathway Alterations.\", \"citation\"=>\"Ahronian et al., 2015, Cancer Discov\", \"pubmed_id\"=>\"25673644\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25673644\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4390490\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>4}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1269, \"phenotypes\"=>[]}]}, {\"id\"=>200, \"entrez_name\"=>\"IKZF1\", \"entrez_id\"=>10320, \"name\"=>\"DELETION\", \"description\"=>\"IKZF1 deletions have been reported in a variety of cancer types including pediatric B-cell precursor acute lymphoblastic leukemia (BCP ALL), Chronic Myeloid Leukemia, diffuse large B-cell lymphoma, myelodysplastic syndrome, T-cell ALL as well as solid tumors of the large intestine, lung, and liver. The clinical impact of these deletions is unknown for many cancer types however a poor outcome has been observed in leukemia based disease. This gene has been strongly implicated as a regulator for \\\"normal  hematopoiesis\\\". In regards to BCP ALL, mouse models have shown that mutations in IKZF1 are not sufficient in and of themselves to cause disease.  Haploinsuffeciency in combination with BCR-ABL1 however, will accelerate leukemiagenesis possibly indicating that at least in regards to BCP ALL these are secondary events that may be necessary for the development of leukaemic clones.\", \"gene_id\"=>73, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>127, \"name\"=>\"transcript_ablation\", \"display_name\"=>\"Transcript Ablation\", \"so_id\"=>\"SO:0001893\", \"description\"=>\"A feature ablation whereby the deleted region includes a transcript feature.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001893\"}], \"civic_actionability_score\"=>85.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>50344378, \"stop\"=>50472799, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000331340.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/73/summary/variants/200/summary#variant\", \"evidence_items\"=>[{\"id\"=>484, \"name\"=>\"EID484\", \"description\"=>\"In children with bone marrow relapsed B-cell precursor acute lymphoblastic leukemia, those with deletions of IKZF1 had worse event-free and overall survival than patients without deletions.\", \"disease\"=>{\"id\"=>237, \"name\"=>\"Precursor B Lymphoblastic Lymphoma/leukemia\", \"display_name\"=>\"Precursor B Lymphoblastic Lymphoma/leukemia\", \"doid\"=>\"7061\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:7061\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>294, \"name\"=>\"Prognostic value of genetic alterations in children with first bone marrow relapse of childhood B-cell precursor acute lymphoblastic leukemia.\", \"citation\"=>\"Krentz et al., 2013, Leukemia\", \"pubmed_id\"=>\"22699455\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22699455\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>2}, \"journal\"=>\"Leukemia\", \"full_journal_title\"=>\"Leukemia\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>200, \"phenotypes\"=>[]}, {\"id\"=>629, \"name\"=>\"EID629\", \"description\"=>\"In patients with Philadelphia Chromosome negative B-cell precursor acute lymphoblastic leukemia, focal deletion involving 3-5 exons of IKZF1 was associated with higher cumulative incidence of relapse (CIR) than wildtype IKZF1.\", \"disease\"=>{\"id\"=>237, \"name\"=>\"Precursor B Lymphoblastic Lymphoma/leukemia\", \"display_name\"=>\"Precursor B Lymphoblastic Lymphoma/leukemia\", \"doid\"=>\"7061\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:7061\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>392, \"name\"=>\"Oncogenetics and minimal residual disease are independent outcome predictors in adult patients with acute lymphoblastic leukemia.\", \"citation\"=>\"Beldjord et al., 2014, Blood\", \"pubmed_id\"=>\"24740809\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24740809\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>6, \"day\"=>12}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>200, \"phenotypes\"=>[]}, {\"id\"=>626, \"name\"=>\"EID626\", \"description\"=>\"A targeted copy number screen of 131 precursor B-ALL cases revealed that IKZF1 deletions are significantly associated with poor relapse-free and overall survival rates. Furthermore non-high-risk patients with IKZF1 deletions exhibited a ~12-fold higher relative relapse rate than those without IKZF1 deletions.\", \"disease\"=>{\"id\"=>43, \"name\"=>\"Acute Lymphocytic Leukemia\", \"display_name\"=>\"Acute Lymphocytic Leukemia\", \"doid\"=>\"9952\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9952\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>389, \"name\"=>\"IKZF1 deletions predict relapse in uniformly treated pediatric precursor B-ALL.\", \"citation\"=>\"Kuiper et al., 2010, Leukemia\", \"pubmed_id\"=>\"20445578\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20445578\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>7}, \"journal\"=>\"Leukemia\", \"full_journal_title\"=>\"Leukemia\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>200, \"phenotypes\"=>[]}, {\"id\"=>625, \"name\"=>\"EID625\", \"description\"=>\"IKZF1 deletions in patients with BCR-ABL–positive adult acute lymphoblastic leukemia were associated with reduced disease-free survival compared to wild type patients.\", \"disease\"=>{\"id\"=>43, \"name\"=>\"Acute Lymphocytic Leukemia\", \"display_name\"=>\"Acute Lymphocytic Leukemia\", \"doid\"=>\"9952\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9952\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>388, \"name\"=>\"IKZF1 (Ikaros) deletions in BCR-ABL1-positive acute lymphoblastic leukemia are associated with short disease-free survival and high rate of cumulative incidence of relapse: a GIMEMA AL WP report.\", \"citation\"=>\"Martinelli et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19770381\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19770381\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>11, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>200, \"phenotypes\"=>[]}, {\"id\"=>624, \"name\"=>\"EID624\", \"description\"=>\"IKZF1 deletions are associated with poor outcome in patients with childhood BCR-ABL–positive B-cell precursor acute lymphoblastic leukemia even when treated with imatinib. This phenomenon was observed in patients treated in the pre-TKI era as well as those treated with imatinib when compared to wild-type patients undergoing similar treatment.\", \"disease\"=>{\"id\"=>237, \"name\"=>\"Precursor B Lymphoblastic Lymphoma/leukemia\", \"display_name\"=>\"Precursor B Lymphoblastic Lymphoma/leukemia\", \"doid\"=>\"7061\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:7061\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>387, \"name\"=>\"IKZF1 status as a prognostic feature in BCR-ABL1-positive childhood ALL.\", \"citation\"=>\"van der Veer et al., 2014, Blood\", \"pubmed_id\"=>\"24366361\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24366361\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>3, \"day\"=>13}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>200, \"phenotypes\"=>[]}]}, {\"id\"=>213, \"entrez_name\"=>\"PTEN\", \"entrez_id\"=>5728, \"name\"=>\"DELETION\", \"description\"=>\"\", \"gene_id\"=>41, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>127, \"name\"=>\"transcript_ablation\", \"display_name\"=>\"Transcript Ablation\", \"so_id\"=>\"SO:0001893\", \"description\"=>\"A feature ablation whereby the deleted region includes a transcript feature.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001893\"}], \"civic_actionability_score\"=>55.0, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>89622870, \"stop\"=>89731687, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000371953.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/41/summary/variants/213/summary#variant\", \"evidence_items\"=>[{\"id\"=>1231, \"name\"=>\"EID1231\", \"description\"=>\"In a phase II trial on castration-resistant prostate cancer patients, FISH detection of PTEN deletion was associated with increased progression free survival (p=0.07) and with improved PSA response (p=0.02) in response to everolimus, an mTOR pathway inhibitor.\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[{\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>852, \"name\"=>\"Phase 2 trial of single-agent everolimus in chemotherapy-naive patients with castration-resistant prostate cancer (SAKK 08/08).\", \"citation\"=>\"Templeton et al., 2013, Eur. Urol.\", \"pubmed_id\"=>\"23582881\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23582881\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>7}, \"journal\"=>\"Eur. Urol.\", \"full_journal_title\"=>\"European urology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>213, \"phenotypes\"=>[]}, {\"id\"=>1741, \"name\"=>\"EID1741\", \"description\"=>\"A meta-analysis of 26 published studies including 8097 prostate cancer patients showed that, compared to patients with normal PTEN, patients with PTEN deletion exhibited a greater aggressive Gleason score (OR: 1.284, 95% CI = 1.145–1.439), pathological stage (OR: 1.628, 95% CI = 1.270–2.087), and risk for recurrence of prostate cancer (HR: 1.738, 95% CI = 1.264–2.390).\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1190, \"name\"=>\"The association of Phosphatase and tensin homolog (PTEN) deletion and prostate cancer risk: A meta-analysis.\", \"citation\"=>\"Gao et al., 2016, Biomed. Pharmacother.\", \"pubmed_id\"=>\"27470558\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27470558\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>10}, \"journal\"=>\"Biomed. Pharmacother.\", \"full_journal_title\"=>\"Biomedicine & pharmacotherapy = Biomedecine & pharmacotherapie\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>213, \"phenotypes\"=>[]}, {\"id\"=>505, \"name\"=>\"EID505\", \"description\"=>\"In a study of 107 prostate adenocarcinomas, those with either hemi- or homozygous deletions of PTEN (as determined by interphase FISH) have shorter biochemical recurence-free survival than those without deletions.\", \"disease\"=>{\"id\"=>122, \"name\"=>\"Prostate Adenocarcinoma\", \"display_name\"=>\"Prostate Adenocarcinoma\", \"doid\"=>\"2526\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2526\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>310, \"name\"=>\"FISH analysis of 107 prostate cancers shows that PTEN genomic deletion is associated with poor clinical outcome.\", \"citation\"=>\"Yoshimoto et al., 2007, Br. J. Cancer\", \"pubmed_id\"=>\"17700571\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17700571\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2360375\", \"publication_date\"=>{\"year\"=>2007, \"month\"=>9, \"day\"=>3}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>213, \"phenotypes\"=>[]}]}, {\"id\"=>250, \"entrez_name\"=>\"GSTP1\", \"entrez_id\"=>2950, \"name\"=>\"DELETION\", \"description\"=>\"\", \"gene_id\"=>2473, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>127, \"name\"=>\"transcript_ablation\", \"display_name\"=>\"Transcript Ablation\", \"so_id\"=>\"SO:0001893\", \"description\"=>\"A feature ablation whereby the deleted region includes a transcript feature.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001893\"}], \"civic_actionability_score\"=>4.0, \"coordinates\"=>{\"chromosome\"=>\"11\", \"start\"=>67351066, \"stop\"=>67354131, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000398606.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/2473/summary/variants/250/summary#variant\", \"evidence_items\"=>[{\"id\"=>660, \"name\"=>\"EID660\", \"description\"=>\"Stable GSTP1 knockdown in ovarian carcinoma A2780 cell lines showed increased sensitivity to platinum-based chemotherapy drugs carboplatin, cisplatinum, carboplatin, and paclitaxel. For each drug, 3 biological replicates were performed for this study, and effectiveness was evaluated using an MTT assay. There was not a significant difference in sensitivity to other chemotherapy drugs such as gemcitabine, topotecan, and doxorubicin.\", \"disease\"=>{\"id\"=>47, \"name\"=>\"Ovarian Carcinoma\", \"display_name\"=>\"Ovarian Carcinoma\", \"doid\"=>\"4001\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4001\"}, \"drugs\"=>[{\"id\"=>81, \"name\"=>\"Paclitaxel\", \"pubchem_id\"=>nil}, {\"id\"=>93, \"name\"=>\"Carboplatin\", \"pubchem_id\"=>nil}, {\"id\"=>93, \"name\"=>\"Carboplatin\", \"pubchem_id\"=>nil}, {\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>417, \"name\"=>\"Glutathione S-transferase P1 (GSTP1) directly influences platinum drug chemosensitivity in ovarian tumour cell lines.\", \"citation\"=>\"Sawers et al., 2014, Br. J. Cancer\", \"pubmed_id\"=>\"25010864\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25010864\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4453841\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>9, \"day\"=>9}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>250, \"phenotypes\"=>[]}]}, {\"id\"=>555, \"entrez_name\"=>\"CDKN2B\", \"entrez_id\"=>1030, \"name\"=>\"LOSS\", \"description\"=>\"\", \"gene_id\"=>916, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>127, \"name\"=>\"transcript_ablation\", \"display_name\"=>\"Transcript Ablation\", \"so_id\"=>\"SO:0001893\", \"description\"=>\"A feature ablation whereby the deleted region includes a transcript feature.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001893\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>22002902, \"stop\"=>22009362, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000276925.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/916/summary/variants/555/summary#variant\", \"evidence_items\"=>[{\"id\"=>1880, \"name\"=>\"EID1880\", \"description\"=>\"A patient-derived xenograft (PDX) from a patient with cyclin-dependent kinase inhibitor 2A/B (CDKN2A/B) loss and FUS-ERG fusion positive Ewing's sarcoma was created. The PDX was treated in four cohorts (each N=6) with control, doxorubicin, CDK4/6 inhibitor palbociclib or IGF-1R inhibitor linsitinib. Both linsitinib and palbociclib cohorts were significantly more effective than control and doxorubicin. Doxorubicin was part of the patient's treatment (Neoadjuvant chemotherapy with DOX, vincristine, and cyclophosphamide) and was noted by authors as failing to control tumor growth.\", \"disease\"=>{\"id\"=>50, \"name\"=>\"Ewing Sarcoma\", \"display_name\"=>\"Ewing Sarcoma\", \"doid\"=>\"3369\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3369\"}, \"drugs\"=>[{\"id\"=>429, \"name\"=>\"Palbociclib\", \"pubchem_id\"=>nil}, {\"id\"=>430, \"name\"=>\"Linsitinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1305, \"name\"=>\"Effective molecular targeting of CDK4/6 and IGF-1R in a rare FUS-ERG fusion CDKN2A-deletion doxorubicin-resistant Ewing's sarcoma patient-derived orthotopic xenograft (PDOX) nude-mouse model.\", \"citation\"=>\"Murakami et al., 2016, Oncotarget\", \"pubmed_id\"=>\"27286459\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27286459\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5216960\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>7, \"day\"=>26}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>555, \"phenotypes\"=>[]}, {\"id\"=>1374, \"name\"=>\"EID1374\", \"description\"=>\"Preclinical study of CDK4/6 inhibitor PD-0332991 in 28 renal carcinoma cell lines (25 malignant RCC and three non-malignant, transformed cell lines derived from kidney epithelium). Both p15 and p16 loss (CGH) were tested for association with response to PD-0332991 using Fisher's exact test. Cell lines with p16 and p15 losses were significantly more likely to be classified as responders (defined by pharmacokinetic data-based concentration of <=400nM) to PD-0332991 (p=0.027, relative risk (RR)=2.06 and p=0.047, RR=1.71, respectively). Four cell lines had homozygous deletions of CDKN2A but not CDKN2B, and three of these lines were classified as being sensitive to PD-0332991. There were no cell lines with only CDKN2B deletion.\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>212, \"name\"=>\"PD-0332991, a potent and selective inhibitor of cyclin-dependent kinase 4/6, demonstrates inhibition of proliferation in renal cell carcinoma at nanomolar concentrations and molecular markers predict for sensitivity.\", \"citation\"=>\"Logan et al., 2013, Anticancer Res.\", \"pubmed_id\"=>\"23898052\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23898052\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>8}, \"journal\"=>\"Anticancer Res.\", \"full_journal_title\"=>\"Anticancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>555, \"phenotypes\"=>[]}]}, {\"id\"=>646, \"entrez_name\"=>\"SMAD4\", \"entrez_id\"=>4089, \"name\"=>\"DELETION\", \"description\"=>\"\", \"gene_id\"=>77, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>127, \"name\"=>\"transcript_ablation\", \"display_name\"=>\"Transcript Ablation\", \"so_id\"=>\"SO:0001893\", \"description\"=>\"A feature ablation whereby the deleted region includes a transcript feature.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001893\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"18\", \"start\"=>48556583, \"stop\"=>48611409, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000342988.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/77/summary/variants/646/summary#variant\", \"evidence_items\"=>[{\"id\"=>1644, \"name\"=>\"EID1644\", \"description\"=>\"A retrospective study with 202 colorectal cancer patients found that SMAD4 deletion was associated with worse outcomes during 5-fluorouracil treatment. The hazard ratio for disease-free survival was 2.89 times greater for patients with SMAD4 deletion as compared to those  with normal SMAD4 diploidy (P = .045).  For overall survival, the hazard ratio was 3.23 times greater for SMAD4 deletion (P = .056).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>395, \"name\"=>\"5-fluorouracil\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1090, \"name\"=>\"SMAD4 is a predictive marker for 5-fluorouracil-based chemotherapy in patients with colorectal cancer.\", \"citation\"=>\"Boulay et al., 2002, Br. J. Cancer\", \"pubmed_id\"=>\"12237773\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/12237773\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2364238\", \"publication_date\"=>{\"year\"=>2002, \"month\"=>9, \"day\"=>9}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>646, \"phenotypes\"=>[]}]}, {\"id\"=>647, \"entrez_name\"=>\"LRP1B\", \"entrez_id\"=>53353, \"name\"=>\"DELETION\", \"description\"=>\"\", \"gene_id\"=>12146, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>127, \"name\"=>\"transcript_ablation\", \"display_name\"=>\"Transcript Ablation\", \"so_id\"=>\"SO:0001893\", \"description\"=>\"A feature ablation whereby the deleted region includes a transcript feature.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001893\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>140988992, \"stop\"=>142889270, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000389484.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/12146/summary/variants/647/summary#variant\", \"evidence_items\"=>[{\"id\"=>1645, \"name\"=>\"EID1645\", \"description\"=>\"A retrospective study of 178 glioblastoma patients found that LRP1B deletion (complete transcript, N=4 or focal transcript deletion of >150kb, N=13) was associated with a lower progress free survival (6.4 months vs. 10.1 months; P = 0.004) and overall free survival (13.4 months vs. 17.8 months; P = 0.001). Multivariate analysis adjusting for age and type of surgery also supported LRP1B deletion being associated with a higher hazard ratio.\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1091, \"name\"=>\"LRP1B deletion is associated with poor outcome for glioblastoma patients.\", \"citation\"=>\"Tabouret et al., 2015, J. Neurol. Sci.\", \"pubmed_id\"=>\"26428308\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26428308\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>11, \"day\"=>15}, \"journal\"=>\"J. Neurol. Sci.\", \"full_journal_title\"=>\"Journal of the neurological sciences\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>647, \"phenotypes\"=>[]}]}, {\"id\"=>696, \"entrez_name\"=>\"SMARCB1\", \"entrez_id\"=>6598, \"name\"=>\"DELETION\", \"description\"=>\"\", \"gene_id\"=>5356, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>127, \"name\"=>\"transcript_ablation\", \"display_name\"=>\"Transcript Ablation\", \"so_id\"=>\"SO:0001893\", \"description\"=>\"A feature ablation whereby the deleted region includes a transcript feature.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001893\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"22\", \"start\"=>24129150, \"stop\"=>24176703, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000344921.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5356/summary/variants/696/summary#variant\", \"evidence_items\"=>[{\"id\"=>1964, \"name\"=>\"EID1964\", \"description\"=>\"Sustained low-dose panobinostat (LBH589) treatment of rhabdoid tumor cells led to changes in cellular morphology associated with a marked increase in the induction of differentiation pathways. In mouse xenograft models, sustained low-dose LBH589 treatment caused tumor growth arrest associated with tumor calcification.\", \"disease\"=>{\"id\"=>834, \"name\"=>\"Rhabdoid Cancer\", \"display_name\"=>\"Rhabdoid Cancer\", \"doid\"=>\"3672\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3672\"}, \"drugs\"=>[{\"id\"=>445, \"name\"=>\"Panobinostat\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1380, \"name\"=>\"Low-Dose Histone Deacetylase Inhibitor Treatment Leads to Tumor Growth Arrest and Multi-Lineage Differentiation of Malignant Rhabdoid Tumors.\", \"citation\"=>\"Muscat et al., 2016, Clin. Cancer Res.\", \"pubmed_id\"=>\"26920892\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26920892\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>7, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>696, \"phenotypes\"=>[]}, {\"id\"=>1740, \"name\"=>\"EID1740\", \"description\"=>\"EPZ-6438 (Tazemetostat) induces apoptosis and differentiation specifically in SMARCB1-deleted malignant rhabdoid tumors (MRT) cells. This study compared in vitro EPZ-6438 treatment of four SMARCB1-deleted MRT cell lines to three control (wild-type) cell lines. These data demonstrate the dependency of SMARCB1 mutant MRTs on EZH2 enzymatic activity and suggest treatment of these genetically defined cancers with EZH2 inhibitors.\", \"disease\"=>{\"id\"=>834, \"name\"=>\"Rhabdoid Cancer\", \"display_name\"=>\"Rhabdoid Cancer\", \"doid\"=>\"3672\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3672\"}, \"drugs\"=>[{\"id\"=>411, \"name\"=>\"EPZ-6438\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1186, \"name\"=>\"Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2.\", \"citation\"=>\"Knutson et al., 2013, Proc. Natl. Acad. Sci. U.S.A.\", \"pubmed_id\"=>\"23620515\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23620515\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3651445\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>5, \"day\"=>7}, \"journal\"=>\"Proc. Natl. Acad. Sci. U.S.A.\", \"full_journal_title\"=>\"Proceedings of the National Academy of Sciences of the United States of America\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>696, \"phenotypes\"=>[]}]}, {\"id\"=>191, \"entrez_name\"=>\"EGFR\", \"entrez_id\"=>1956, \"name\"=>\"COPY NUMBER VARIATION\", \"description\"=>\"\", \"gene_id\"=>19, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>29, \"name\"=>\"copy_number_change\", \"display_name\"=>\"Copy Number Change\", \"so_id\"=>\"SO:0001563\", \"description\"=>\"A sequence variant where copies of a feature (CNV) are either increased or decreased.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001563\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>55086794, \"stop\"=>55279321, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000275493.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/19/summary/variants/191/summary#variant\", \"evidence_items\"=>[{\"id\"=>5925, \"name\"=>\"EID5925\", \"description\"=>\"Among TKI-treated patients, increased EGFR gene copy number appears to be associated with improved survival outcomes.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>14, \"name\"=>\"Gefitinib\", \"pubchem_id\"=>nil}, {\"id\"=>15, \"name\"=>\"Erlotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2395, \"name\"=>\"EGFR gene copy number as a predictive/biomarker for patients with non-small-cell lung cancer receiving tyrosine kinase inhibitor treatment: a systematic review and meta-analysis.\", \"citation\"=>\"Zhang et al., 2017, J. Investig. Med.\", \"pubmed_id\"=>\"27664271\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27664271\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>1}, \"journal\"=>\"J. Investig. Med.\", \"full_journal_title\"=>\"Journal of investigative medicine : the official publication of the American Federation for Clinical Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>191, \"phenotypes\"=>[]}, {\"id\"=>5924, \"name\"=>\"EID5924\", \"description\"=>\"The result of the analysis found that EGFR gene copy number was associated with increased OS and PFS, supporting the idea that EGFR gene copy number is a biomarker for response to EGFR-TKI therapy in patients with advanced NSCLC.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>14, \"name\"=>\"Gefitinib\", \"pubchem_id\"=>nil}, {\"id\"=>15, \"name\"=>\"Erlotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>650, \"name\"=>\"EGFR gene copy number as a predictive biomarker for patients receiving tyrosine kinase inhibitor treatment: a systematic review and meta-analysis in non-small-cell lung cancer.\", \"citation\"=>\"Dahabreh et al., 2011, Ann. Oncol.\", \"pubmed_id\"=>\"20826716\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20826716\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>3}, \"journal\"=>\"Ann. Oncol.\", \"full_journal_title\"=>\"Annals of oncology : official journal of the European Society for Medical Oncology / ESMO\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>191, \"phenotypes\"=>[]}, {\"id\"=>472, \"name\"=>\"EID472\", \"description\"=>\"In 134 patients with head and neck squamous cancer 32 had copy number variations. Tumors with variation (either increase or decrease) in copy number of EGFR had worse overall, cancer-specific, and disease-free survival.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>286, \"name\"=>\"Epidermal growth factor receptor copy number alterations correlate with poor clinical outcome in patients with head and neck squamous cancer.\", \"citation\"=>\"Temam et al., 2007, J. Clin. Oncol.\", \"pubmed_id\"=>\"17538160\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17538160\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>6, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>191, \"phenotypes\"=>[]}]}, {\"id\"=>220, \"entrez_name\"=>\"TERT\", \"entrez_id\"=>7015, \"name\"=>\"PROMOTER MUTATION\", \"description\"=>\"TERT promoter mutations are activating and commonly found at two hotspots within the TERT promoter region which occur 124 and 146 base pairs (hg19) upstream of the translation start site. These hotspots, termed C228T and C250T are observed in many cancer types and have a particularly high prevalence in glioblastoma (83%), melanoma (71%), bladder cancer (66%) and hepatocelluar carcinoma (HCC) (47%). The two mutations are usually heterozygous and mutually exclusive and result in an increased similarity to an ETS binding domain and may act as a binding site to an ETS transcription factor. These mutations are thought to be early events in tumorgenesis and aid in cellular immortality. Decreased overall survival in patients with these mutations have been observed in medulloblastoma, thyroid cancer, urogenital cancer, melanoma, and laryngeal tumors. The presence of these mutations has also been shown to be predictive of malignant/premalignant nodules in HCC and meningiomas.\", \"gene_id\"=>79, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>32, \"name\"=>\"regulatory_region_variant\", \"display_name\"=>\"Regulatory Region Variant\", \"so_id\"=>\"SO:0001566\", \"description\"=>\"A sequence variant located within a regulatory region.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001566\"}], \"civic_actionability_score\"=>80.0, \"coordinates\"=>{\"chromosome\"=>\"5\", \"start\"=>1295161, \"stop\"=>1295373, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000310581.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/79/summary/variants/220/summary#variant\", \"evidence_items\"=>[{\"id\"=>976, \"name\"=>\"EID976\", \"description\"=>\"In this meta-analysis of 16 Trials, TERT mutations were more common in glioblastomas (69%), oligodendrogliomas (72%) in comparison to astrocytomas (24%) and oligoastrocytomas (38%).\", \"disease\"=>{\"id\"=>98, \"name\"=>\"Brain Glioma\", \"display_name\"=>\"Brain Glioma\", \"doid\"=>\"0060108\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060108\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>675, \"name\"=>\"TERT mutation in glioma: Frequency, prognosis and risk.\", \"citation\"=>\"Yuan et al., 2016, J Clin Neurosci\", \"pubmed_id\"=>\"26765760\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26765760\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>4}, \"journal\"=>\"J Clin Neurosci\", \"full_journal_title\"=>\"Journal of clinical neuroscience : official journal of the Neurosurgical Society of Australasia\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>220, \"phenotypes\"=>[]}, {\"id\"=>516, \"name\"=>\"EID516\", \"description\"=>\"In 187 patients with glioblastoma, those with TERT promoter mutations had worse cummulative survival than wild-type patients. The most common mutation was C228T (73% of mutants) followed by C250T (27% of mutants). However, most of the Prognostic value of TERT mutations is thought to be derived from their inverse coorelation with IDH1 mutations.\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>319, \"name\"=>\"TERT promoter mutations in primary and secondary glioblastomas.\", \"citation\"=>\"Nonoguchi et al., 2013, Acta Neuropathol.\", \"pubmed_id\"=>\"23955565\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23955565\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>12}, \"journal\"=>\"Acta Neuropathol.\", \"full_journal_title\"=>\"Acta neuropathologica\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>220, \"phenotypes\"=>[]}, {\"id\"=>658, \"name\"=>\"EID658\", \"description\"=>\"TERT promoter mutation C228T is significantly associated with BRAF V600E mutation in papillary thyroid cancer(PTC). Conversely, the BRAF V600E mutation is significantly more common in PTC cases with the TERT C228T mutation than in cases with the wild-type allele of TERT.\", \"disease\"=>{\"id\"=>16, \"name\"=>\"Thyroid Cancer\", \"display_name\"=>\"Thyroid Cancer\", \"doid\"=>\"1781\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1781\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>414, \"name\"=>\"Highly prevalent TERT promoter mutations in aggressive thyroid cancers.\", \"citation\"=>\"Liu et al., 2013, Endocr. Relat. Cancer\", \"pubmed_id\"=>\"23766237\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23766237\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3782569\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>8}, \"journal\"=>\"Endocr. Relat. Cancer\", \"full_journal_title\"=>\"Endocrine-related cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>220, \"phenotypes\"=>[]}, {\"id\"=>515, \"name\"=>\"EID515\", \"description\"=>\"In patients with nonacral cutaneous melanoma, those with mutations in the TERT promoter had shorter overall survival than wild-type patients.\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>318, \"name\"=>\"TERT promoter mutation status as an independent prognostic factor in cutaneous melanoma.\", \"citation\"=>\"Griewank et al., 2014, J. Natl. Cancer Inst.\", \"pubmed_id\"=>\"25217772\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25217772\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4200061\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>9}, \"journal\"=>\"J. Natl. Cancer Inst.\", \"full_journal_title\"=>\"Journal of the National Cancer Institute\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>220, \"phenotypes\"=>[]}, {\"id\"=>657, \"name\"=>\"EID657\", \"description\"=>\"TERT promoter mutations are prevalent in aggressive sub-types of thyroid cancers. This work showed that TERT promoter mutations were significantly more prevalent in the tall cell papillary thyroid carcinoma (PTC) samples than in the conventional PTC and follicular variant PTC samples.\", \"disease\"=>{\"id\"=>16, \"name\"=>\"Thyroid Cancer\", \"display_name\"=>\"Thyroid Cancer\", \"doid\"=>\"1781\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1781\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>414, \"name\"=>\"Highly prevalent TERT promoter mutations in aggressive thyroid cancers.\", \"citation\"=>\"Liu et al., 2013, Endocr. Relat. Cancer\", \"pubmed_id\"=>\"23766237\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23766237\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3782569\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>8}, \"journal\"=>\"Endocr. Relat. Cancer\", \"full_journal_title\"=>\"Endocrine-related cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>220, \"phenotypes\"=>[]}]}, {\"id\"=>248, \"entrez_name\"=>\"TERT\", \"entrez_id\"=>7015, \"name\"=>\"C228T\", \"description\"=>\"\", \"gene_id\"=>79, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>32, \"name\"=>\"regulatory_region_variant\", \"display_name\"=>\"Regulatory Region Variant\", \"so_id\"=>\"SO:0001566\", \"description\"=>\"A sequence variant located within a regulatory region.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001566\"}], \"civic_actionability_score\"=>40.0, \"coordinates\"=>{\"chromosome\"=>\"5\", \"start\"=>1295228, \"stop\"=>1295228, \"reference_bases\"=>\"G\", \"variant_bases\"=>\"A\", \"representative_transcript\"=>\"ENST00000310581.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/79/summary/variants/248/summary#variant\", \"evidence_items\"=>[{\"id\"=>1646, \"name\"=>\"EID1646\", \"description\"=>\"In 308 thyroid fine-needle aspiration biopsies, 0 of the 179 benign tumors displayed either the C228T or the C250T TERT promoter mutation; however, 9 of the 129 differentiated thyroid cancers had either of these mutations. The biomarker displayed 100% diagnostic specificity and 7% sensitivity.\", \"disease\"=>{\"id\"=>16, \"name\"=>\"Thyroid Cancer\", \"display_name\"=>\"Thyroid Cancer\", \"doid\"=>\"1781\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1781\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1092, \"name\"=>\"Diagnostic and prognostic TERT promoter mutations in thyroid fine-needle aspiration biopsy.\", \"citation\"=>\"Liu et al., 2014, Endocr. Relat. Cancer\", \"pubmed_id\"=>\"25121551\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25121551\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4175020\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>10}, \"journal\"=>\"Endocr. Relat. Cancer\", \"full_journal_title\"=>\"Endocrine-related cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>248, \"phenotypes\"=>[]}, {\"id\"=>655, \"name\"=>\"EID655\", \"description\"=>\"In patients with papillary thyroid cancer harboring both BRAF V600E and the TERT promotor mutation C228T, recurrence-free survival is worse than in patients harboring one of these mutations or no mutations in either gene.\", \"disease\"=>{\"id\"=>156, \"name\"=>\"Papillary Thyroid Carcinoma\", \"display_name\"=>\"Papillary Thyroid Carcinoma\", \"doid\"=>\"3969\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3969\"}, \"drugs\"=>[], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>413, \"name\"=>\"BRAF V600E and TERT promoter mutations cooperatively identify the most aggressive papillary thyroid cancer with highest recurrence.\", \"citation\"=>\"Xing et al., 2014, J. Clin. Oncol.\", \"pubmed_id\"=>\"25024077\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25024077\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4145183\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>9, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>248, \"phenotypes\"=>[]}]}, {\"id\"=>332, \"entrez_name\"=>\"FNTB\", \"entrez_id\"=>2342, \"name\"=>\"RS11623866\", \"description\"=>\"\", \"gene_id\"=>1950, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>32, \"name\"=>\"regulatory_region_variant\", \"display_name\"=>\"Regulatory Region Variant\", \"so_id\"=>\"SO:0001566\", \"description\"=>\"A sequence variant located within a regulatory region.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001566\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"14\", \"start\"=>65453063, \"stop\"=>65453063, \"reference_bases\"=>\"G\", \"variant_bases\"=>\"C\", \"representative_transcript\"=>\"ENST00000246166.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1950/summary/variants/332/summary#variant\", \"evidence_items\"=>[{\"id\"=>815, \"name\"=>\"EID815\", \"description\"=>\"57 from 105 patients in the AGo-OVAR-15 trial were retrospectively genotyped for FNTB polymorphisms. Resistance to lonafarnib-containing treatment was limited to patients with a FNTB rs11623866 (c.-609G > C) GG genotype (HRPFS 6.2, 95%CI = 2.01, 19.41, P = 0.002; HROS 9.6, 95%CI = 1.89, 48.54, P = 0.006).\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>270, \"name\"=>\"Lonafarnib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Polymorphism\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>547, \"name\"=>\"The FNTB promoter polymorphism rs11623866 as a potential predictive biomarker for lonafarnib treatment of ovarian cancer patients.\", \"citation\"=>\"Bachmann et al., 2015, Br J Clin Pharmacol\", \"pubmed_id\"=>\"26033044\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26033044\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4631186\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>11}, \"journal\"=>\"Br J Clin Pharmacol\", \"full_journal_title\"=>\"British journal of clinical pharmacology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>332, \"phenotypes\"=>[]}]}, {\"id\"=>161, \"entrez_name\"=>\"PBRM1\", \"entrez_id\"=>55193, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>62, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>15, \"name\"=>\"transcription_variant\", \"display_name\"=>\"Transcription Variant\", \"so_id\"=>\"SO:0001549\", \"description\"=>\"A variant that changes alters the transcription of a transcript with respect to a reference sequence.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001549\"}, {\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>30.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>52581857, \"stop\"=>52719852, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000394830.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/62/summary/variants/161/summary#variant\", \"evidence_items\"=>[{\"id\"=>5996, \"name\"=>\"EID5996\", \"description\"=>\"This study evaluated the relationship between tumor mutations and treatment outcomes in the RECORD-3 clinical trial. With first-line everolimus, PBRM1 and BAP1 mutations were associated with longer (median [95% confidence interval {CI}] 12.8 [8.1, 18.4] vs 5.5 [3.1, 8.4] mo) and shorter (median [95% CI] 4.9 [2.9, 8.1] vs 10.5 [7.3, 12.9] mo) PFS1L, respectively. With first-line sunitinib, KDM5C mutations were associated with longer PFS1L (median [95% CI] of 20.6 [12.4, 27.3] vs 8.3 [7.8, 11.0] mo).\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[{\"id\"=>561, \"name\"=>\"Everolimus (JAN/USAN/INN)\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2234, \"name\"=>\"Genomic Biomarkers of a Randomized Trial Comparing First-line Everolimus and Sunitinib in Patients with Metastatic Renal Cell Carcinoma.\", \"citation\"=>\"Hsieh et al., 2017, Eur. Urol.\", \"pubmed_id\"=>\"27751729\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27751729\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5431298\", \"publication_date\"=>{\"year\"=>2017, \"month\"=>3}, \"journal\"=>\"Eur. Urol.\", \"full_journal_title\"=>\"European urology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>161, \"phenotypes\"=>[]}, {\"id\"=>499, \"name\"=>\"EID499\", \"description\"=>\"In patients with sporatic clear cell renal cell carcinoma, those with mutations in PBRM1 had better overall survival than patients with BAP1 mutations.\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>305, \"name\"=>\"Effects on survival of BAP1 and PBRM1 mutations in sporadic clear-cell renal-cell carcinoma: a retrospective analysis with independent validation.\", \"citation\"=>\"Kapur et al., 2013, Lancet Oncol.\", \"pubmed_id\"=>\"23333114\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23333114\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4674067\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>2}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>161, \"phenotypes\"=>[]}, {\"id\"=>420, \"name\"=>\"EID420\", \"description\"=>\"In renal clear cell carcinoma, patients with PBRM1 mutations did not have a significantly different cancer specific survival.\", \"disease\"=>{\"id\"=>48, \"name\"=>\"Renal Clear Cell Carcinoma\", \"display_name\"=>\"Renal Clear Cell Carcinoma\", \"doid\"=>\"4467\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4467\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"N/A\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>249, \"name\"=>\"Adverse outcomes in clear cell renal cell carcinoma with mutations of 3p21 epigenetic regulators BAP1 and SETD2: a report by MSKCC and the KIRC TCGA research network.\", \"citation\"=>\"Hakimi et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"23620406\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23620406\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3708609\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>6, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>161, \"phenotypes\"=>[]}]}, {\"id\"=>178, \"entrez_name\"=>\"ATM\", \"entrez_id\"=>472, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>69, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>15, \"name\"=>\"transcription_variant\", \"display_name\"=>\"Transcription Variant\", \"so_id\"=>\"SO:0001549\", \"description\"=>\"A variant that changes alters the transcription of a transcript with respect to a reference sequence.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001549\"}, {\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>86.5, \"coordinates\"=>{\"chromosome\"=>\"11\", \"start\"=>108093559, \"stop\"=>108239826, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000278616.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/69/summary/variants/178/summary#variant\", \"evidence_items\"=>[{\"id\"=>1907, \"name\"=>\"EID1907\", \"description\"=>\"Two ATM deficient mantle cell lymphoma (MCL) cell lines were more sensitive to PARP-1 inhibition with olaparib or PJ-34 than 4 other MCL cell lines. shRNA mediated ATM knockdown in a cell line increased sensitivity to olaparib. These results were confirmed in a xenograft model.\", \"disease\"=>{\"id\"=>38, \"name\"=>\"Mantle Cell Lymphoma\", \"display_name\"=>\"Mantle Cell Lymphoma\", \"doid\"=>\"0050746\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050746\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1335, \"name\"=>\"ATM deficiency sensitizes mantle cell lymphoma cells to poly(ADP-ribose) polymerase-1 inhibitors.\", \"citation\"=>\"Williamson et al., 2010, Mol. Cancer Ther.\", \"pubmed_id\"=>\"20124459\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20124459\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3729269\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>2}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>178, \"phenotypes\"=>[]}, {\"id\"=>1960, \"name\"=>\"EID1960\", \"description\"=>\"Response to olaparib was observed in 4 of 5 with prostate cancer patients with ATM aberrations (2 germline).\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>409, \"name\"=>\"DNA-Repair Defects and Olaparib in Metastatic Prostate Cancer.\", \"citation\"=>\"Mateo et al., 2015, N. Engl. J. Med.\", \"pubmed_id\"=>\"26510020\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26510020\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>10, \"day\"=>29}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>178, \"phenotypes\"=>[]}, {\"id\"=>2927, \"name\"=>\"EID2927\", \"description\"=>\"ATM point mutations were found in 8/57 patients with CLL and were associated with a shorter treatment-free interval, suggesting a more aggressive course.\", \"disease\"=>{\"id\"=>52, \"name\"=>\"Chronic Lymphocytic Leukemia\", \"display_name\"=>\"Chronic Lymphocytic Leukemia\", \"doid\"=>\"1040\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1040\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1680, \"name\"=>\"ATM gene alterations in chronic lymphocytic leukemia patients induce a distinct gene expression profile and predict disease progression.\", \"citation\"=>\"Guarini et al., 2012, Haematologica\", \"pubmed_id\"=>\"21993670\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21993670\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3248930\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>1}, \"journal\"=>\"Haematologica\", \"full_journal_title\"=>\"Haematologica\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>178, \"phenotypes\"=>[]}, {\"id\"=>1539, \"name\"=>\"EID1539\", \"description\"=>\"In a study of 406 previously untreated CLL cases, a multivariate analysis indicated that aberrations (N=44; mixed pathogenic germline and somatic) in exons 2-63 of ATM were significantly associated with shorter time to first treatment (HR: 1.36; 95% CI: 1.05-1.76; P = 0.021). These mutations were not significantly associated with overall survival.\", \"disease\"=>{\"id\"=>52, \"name\"=>\"Chronic Lymphocytic Leukemia\", \"display_name\"=>\"Chronic Lymphocytic Leukemia\", \"doid\"=>\"1040\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1040\"}, \"drugs\"=>[], \"rating\"=>5, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1006, \"name\"=>\"Clinical impact of clonal and subclonal TP53, SF3B1, BIRC3, NOTCH1, and ATM mutations in chronic lymphocytic leukemia.\", \"citation\"=>\"Nadeu et al., 2016, Blood\", \"pubmed_id\"=>\"26837699\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26837699\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4912011\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>28}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>178, \"phenotypes\"=>[]}, {\"id\"=>455, \"name\"=>\"EID455\", \"description\"=>\"Olaparib was found to inhibit cancer cell growth in ATM mutant cell lines of several blood cancers including Lymphoblastoid Leukemia, Mantle Cell Lymphoma, Chronic Lymphocytic Leukemia, and in a murine xenograft model of Mantle Cell Lymphoma.\", \"disease\"=>{\"id\"=>217, \"name\"=>\"Hematologic Cancer\", \"display_name\"=>\"Hematologic Cancer\", \"doid\"=>\"2531\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2531\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>271, \"name\"=>\"The PARP inhibitor olaparib induces significant killing of ATM-deficient lymphoid tumor cells in vitro and in vivo.\", \"citation\"=>\"Weston et al., 2010, Blood\", \"pubmed_id\"=>\"20739657\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20739657\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>11, \"day\"=>25}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>178, \"phenotypes\"=>[]}, {\"id\"=>453, \"name\"=>\"EID453\", \"description\"=>\"Melanoma cell lines were shown to have increased sensitivity to Temozolomide when siRNA-induced ATM knockdown was applied.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>11, \"name\"=>\"Temozolomide\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>269, \"name\"=>\"Contribution of ATM and ATR to the resistance of glioblastoma and malignant melanoma cells to the methylating anticancer drug temozolomide.\", \"citation\"=>\"Eich et al., 2013, Mol. Cancer Ther.\", \"pubmed_id\"=>\"23960094\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23960094\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>11}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>178, \"phenotypes\"=>[]}, {\"id\"=>452, \"name\"=>\"EID452\", \"description\"=>\"Glioblastoma cell lines were shown to have increased sensitivity to Temozolomide when siRNA-induced ATM knockdown was applied.\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[{\"id\"=>11, \"name\"=>\"Temozolomide\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>269, \"name\"=>\"Contribution of ATM and ATR to the resistance of glioblastoma and malignant melanoma cells to the methylating anticancer drug temozolomide.\", \"citation\"=>\"Eich et al., 2013, Mol. Cancer Ther.\", \"pubmed_id\"=>\"23960094\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23960094\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>11}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>178, \"phenotypes\"=>[]}]}, {\"id\"=>323, \"entrez_name\"=>\"MET\", \"entrez_id\"=>4233, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>52, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>15, \"name\"=>\"transcription_variant\", \"display_name\"=>\"Transcription Variant\", \"so_id\"=>\"SO:0001549\", \"description\"=>\"A variant that changes alters the transcription of a transcript with respect to a reference sequence.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001549\"}, {\"id\"=>160, \"name\"=>\"gain_of_function_variant\", \"display_name\"=>\"Gain Of Function Variant\", \"so_id\"=>\"SO:0002053\", \"description\"=>\"A sequence variant whereby new or enhanced function is conferred on the gene product.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002053\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>116312459, \"stop\"=>116436396, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000318493.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/52/summary/variants/323/summary#variant\", \"evidence_items\"=>[{\"id\"=>796, \"name\"=>\"EID796\", \"description\"=>\"In a retrospective analysis of 74 patients with papillary renal cell carcinoma, treated with foretinib, MET germline mutation was associated with improved response rate.  A response was observed in 5 of 10 (50%) vs. 5 of 57 (8.8%) patients with and without germline MET mutations, respectively.\", \"disease\"=>{\"id\"=>162, \"name\"=>\"Papillary Renal Cell Carcinoma\", \"display_name\"=>\"Papillary Renal Cell Carcinoma\", \"doid\"=>\"4465\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4465\"}, \"drugs\"=>[{\"id\"=>150, \"name\"=>\"Foretinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>530, \"name\"=>\"Phase II and biomarker study of the dual MET/VEGFR2 inhibitor foretinib in patients with papillary renal cell carcinoma.\", \"citation\"=>\"Choueiri et al., 2013, J. Clin. Oncol.\", \"pubmed_id\"=>\"23213094\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23213094\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3532390\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>1, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>323, \"phenotypes\"=>[]}]}, {\"id\"=>664, \"entrez_name\"=>\"CDH1\", \"entrez_id\"=>999, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>888, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>15, \"name\"=>\"transcription_variant\", \"display_name\"=>\"Transcription Variant\", \"so_id\"=>\"SO:0001549\", \"description\"=>\"A variant that changes alters the transcription of a transcript with respect to a reference sequence.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001549\"}], \"civic_actionability_score\"=>0.5, \"coordinates\"=>{\"chromosome\"=>\"16\", \"start\"=>68771128, \"stop\"=>68869451, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000261769.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/888/summary/variants/664/summary#variant\", \"evidence_items\"=>[{\"id\"=>1682, \"name\"=>\"EID1682\", \"description\"=>\"285 intestinal-type tumors and 183 diffuse-type tumors were evaluated for mutational profiles.  CDH1 mutations are associated with diffuse type mucinous gastric carcinoma, which indicate worse prognosis when compared to intestinal type (p<0.001).\", \"disease\"=>{\"id\"=>1322, \"name\"=>\"Stomach Carcinoma\", \"display_name\"=>\"Stomach Carcinoma\", \"doid\"=>\"5517\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5517\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"E\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1136, \"name\"=>\"Comprehensive mutation profiling of mucinous gastric carcinoma.\", \"citation\"=>\"Rokutan et al., 2016, J. Pathol.\", \"pubmed_id\"=>\"27313181\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27313181\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>10}, \"journal\"=>\"J. Pathol.\", \"full_journal_title\"=>\"The Journal of pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>664, \"phenotypes\"=>[]}]}, {\"id\"=>665, \"entrez_name\"=>\"RHOA\", \"entrez_id\"=>387, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>399, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>15, \"name\"=>\"transcription_variant\", \"display_name\"=>\"Transcription Variant\", \"so_id\"=>\"SO:0001549\", \"description\"=>\"A variant that changes alters the transcription of a transcript with respect to a reference sequence.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001549\"}, {\"id\"=>160, \"name\"=>\"gain_of_function_variant\", \"display_name\"=>\"Gain Of Function Variant\", \"so_id\"=>\"SO:0002053\", \"description\"=>\"A sequence variant whereby new or enhanced function is conferred on the gene product.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002053\"}], \"civic_actionability_score\"=>0.5, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>49412898, \"stop\"=>49405977, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000418115.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/399/summary/variants/665/summary#variant\", \"evidence_items\"=>[{\"id\"=>1683, \"name\"=>\"EID1683\", \"description\"=>\"285 intestinal-type tumors and 183 diffuse-type tumors were evaluated for mutational profiles.  RHOA mutations are associated with diffuse type mucinous gastric carcinoma, which indicate worse prognosis when compared to intestinal type (p<0.001).\", \"disease\"=>{\"id\"=>1322, \"name\"=>\"Stomach Carcinoma\", \"display_name\"=>\"Stomach Carcinoma\", \"doid\"=>\"5517\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5517\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"E\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1136, \"name\"=>\"Comprehensive mutation profiling of mucinous gastric carcinoma.\", \"citation\"=>\"Rokutan et al., 2016, J. Pathol.\", \"pubmed_id\"=>\"27313181\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27313181\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>10}, \"journal\"=>\"J. Pathol.\", \"full_journal_title\"=>\"The Journal of pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>665, \"phenotypes\"=>[]}]}, {\"id\"=>668, \"entrez_name\"=>\"KMT2C\", \"entrez_id\"=>58508, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>14089, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>15, \"name\"=>\"transcription_variant\", \"display_name\"=>\"Transcription Variant\", \"so_id\"=>\"SO:0001549\", \"description\"=>\"A variant that changes alters the transcription of a transcript with respect to a reference sequence.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001549\"}, {\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>151832010, \"stop\"=>152133090, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000262189.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/14089/summary/variants/668/summary#variant\", \"evidence_items\"=>[{\"id\"=>1696, \"name\"=>\"EID1696\", \"description\"=>\"Evaluation of 39 aggressive cutaneous squamous cell carcinomas revealed that tumors with a KMT2C mutation were 5x more likely to invade the bone (53% versus 10%; p=0.008).  Patients with KMT2C mutations also had significantly shorter recurrent free survival (21.6 versus 167.5; p=0.003) and an increased hazards ratio for death (HR=5.16; 95%CI:1.55 to 17.18).  This worse prognosis did not appear to be correlated with bone invasion (p=0.98).\", \"disease\"=>{\"id\"=>130, \"name\"=>\"Skin Squamous Cell Carcinoma\", \"display_name\"=>\"Skin Squamous Cell Carcinoma\", \"doid\"=>\"3151\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3151\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1145, \"name\"=>\"Mutational landscape of aggressive cutaneous squamous cell carcinoma.\", \"citation\"=>\"Pickering et al., 2014, Clin. Cancer Res.\", \"pubmed_id\"=>\"25303977\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25303977\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4367811\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>12, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>668, \"phenotypes\"=>[]}]}, {\"id\"=>64, \"entrez_name\"=>\"JAK2\", \"entrez_id\"=>3717, \"name\"=>\"V617F\", \"description\"=>\"JAK2 V617F is a highly recurrent mutation in myeloproliferative diseases. It is found in around 98% of patients with polycythemia vera, and just over half of the patients with essential thrombocythemia and primary myelofibrosis. While less associated with cancer, when it is seen, it is more likely to be in myeloid leukemias than lymphoid leukemias. The V617F mutation is an activating mutation, resulting in increased kinase activity. The mutation seems to be restricted to hematologic malignancies. Treatment of JAK mutant diseases with ruxolitinib has seen some clinical success.\", \"gene_id\"=>28, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>160, \"name\"=>\"gain_of_function_variant\", \"display_name\"=>\"Gain Of Function Variant\", \"so_id\"=>\"SO:0002053\", \"description\"=>\"A sequence variant whereby new or enhanced function is conferred on the gene product.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002053\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>83.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>5073770, \"stop\"=>5073770, \"reference_bases\"=>\"G\", \"variant_bases\"=>\"T\", \"representative_transcript\"=>\"ENST00000381652.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/28/summary/variants/64/summary#variant\", \"evidence_items\"=>[{\"id\"=>20, \"name\"=>\"EID20\", \"description\"=>\"TG101348 effectively inhibits STAT5 signaling in JAK2 V617F cells.\", \"disease\"=>{\"id\"=>9, \"name\"=>\"Polycythemia Vera\", \"display_name\"=>\"Polycythemia Vera\", \"doid\"=>\"8997\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8997\"}, \"drugs\"=>[{\"id\"=>9, \"name\"=>\"TG101348\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>65, \"name\"=>\"Efficacy of TG101348, a selective JAK2 inhibitor, in treatment of a murine model of JAK2V617F-induced polycythemia vera.\", \"citation\"=>\"Wernig et al., 2008, Cancer Cell\", \"pubmed_id\"=>\"18394554\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18394554\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2008, \"month\"=>4}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>64, \"phenotypes\"=>[]}, {\"id\"=>7, \"name\"=>\"EID7\", \"description\"=>\"The SNP rs10974944 in the Jak2 locus is associated with increased predisposition for JAK2 V617F mutation and its associated cancer.\", \"disease\"=>{\"id\"=>5, \"name\"=>\"Bone Marrow Cancer\", \"display_name\"=>\"Bone Marrow Cancer\", \"doid\"=>\"4960\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4960\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>55, \"name\"=>\"A germline JAK2 SNP is associated with predisposition to the development of JAK2(V617F)-positive myeloproliferative neoplasms.\", \"citation\"=>\"Kilpivaara et al., 2009, Nat. Genet.\", \"pubmed_id\"=>\"19287384\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19287384\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3676425\", \"publication_date\"=>{\"year\"=>2009, \"month\"=>4}, \"journal\"=>\"Nat. Genet.\", \"full_journal_title\"=>\"Nature genetics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>64, \"phenotypes\"=>[]}, {\"id\"=>19, \"name\"=>\"EID19\", \"description\"=>\"In patients with JAK2 V617F, the use of pegylated IFN-alpha-2a leads to reduction in the percentage of cells harboring JAK2 V617F.\", \"disease\"=>{\"id\"=>9, \"name\"=>\"Polycythemia Vera\", \"display_name\"=>\"Polycythemia Vera\", \"doid\"=>\"8997\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8997\"}, \"drugs\"=>[{\"id\"=>8, \"name\"=>\"Pegylated IFN-alpha-2a\", \"pubchem_id\"=>\"\"}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>64, \"name\"=>\"High molecular response rate of polycythemia vera patients treated with pegylated interferon alpha-2a.\", \"citation\"=>\"Kiladjian et al., 2006, Blood\", \"pubmed_id\"=>\"16709929\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16709929\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2006, \"month\"=>9, \"day\"=>15}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>64, \"phenotypes\"=>[]}, {\"id\"=>5, \"name\"=>\"EID5\", \"description\"=>\"JAK2 V617F is associated with myeloid malignanices (AML, MDS, CMML/atypical CML).\", \"disease\"=>{\"id\"=>4, \"name\"=>\"Chronic Myeloid Leukemia\", \"display_name\"=>\"Chronic Myeloid Leukemia\", \"doid\"=>\"8552\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8552\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>51, \"name\"=>\"The JAK2V617F activating mutation occurs in chronic myelomonocytic leukemia and acute myeloid leukemia, but not in acute lymphoblastic leukemia or chronic lymphocytic leukemia.\", \"citation\"=>\"Levine et al., 2005, Blood\", \"pubmed_id\"=>\"16081687\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16081687\", \"open_access\"=>true, \"pmc_id\"=>\"PMC1895066\", \"publication_date\"=>{\"year\"=>2005, \"month\"=>11, \"day\"=>15}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>64, \"phenotypes\"=>[]}, {\"id\"=>1, \"name\"=>\"EID1\", \"description\"=>\"JAK2 V617F is not associated with lymphoid leukemia (B-lineage ALL, T-ALL or CLL).\", \"disease\"=>{\"id\"=>1, \"name\"=>\"Lymphoid Leukemia\", \"display_name\"=>\"Lymphoid Leukemia\", \"doid\"=>\"10747\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10747\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Negative\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>51, \"name\"=>\"The JAK2V617F activating mutation occurs in chronic myelomonocytic leukemia and acute myeloid leukemia, but not in acute lymphoblastic leukemia or chronic lymphocytic leukemia.\", \"citation\"=>\"Levine et al., 2005, Blood\", \"pubmed_id\"=>\"16081687\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16081687\", \"open_access\"=>true, \"pmc_id\"=>\"PMC1895066\", \"publication_date\"=>{\"year\"=>2005, \"month\"=>11, \"day\"=>15}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>64, \"phenotypes\"=>[]}]}, {\"id\"=>208, \"entrez_name\"=>\"NRAS\", \"entrez_id\"=>4893, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>36, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>160, \"name\"=>\"gain_of_function_variant\", \"display_name\"=>\"Gain Of Function Variant\", \"so_id\"=>\"SO:0002053\", \"description\"=>\"A sequence variant whereby new or enhanced function is conferred on the gene product.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002053\"}, {\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>198.0, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>115247090, \"stop\"=>115259515, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000369535.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/36/summary/variants/208/summary#variant\", \"evidence_items\"=>[{\"id\"=>5344, \"name\"=>\"EID5344\", \"description\"=>\"NCCN guidelines indicate colorectal cancer patients with NRAS mutations should not be treated with EGFR inhibitors cetuximab or panitumumab.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}, {\"id\"=>28, \"name\"=>\"Panitumumab\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2236, \"name\"=>nil, \"citation\"=>\"Benson et al., 2017, J Natl Compr Canc Netw\", \"pubmed_id\"=>\"28275037\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28275037\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{}, \"journal\"=>nil, \"full_journal_title\"=>nil, \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>208, \"phenotypes\"=>[]}, {\"id\"=>1429, \"name\"=>\"EID1429\", \"description\"=>\"In six NRAS mutant melanoma cell lines, concomitant use of MEK inhibitor and PI3K/mTOR inhibitor resulted in greater decrease in cell viability than either alone, with the synergistic effect being marked in 4/6 cell lines. MEK inhibitor JTP-74057 and PI3K/mTOR inhibitor GSK2126458 together induced synergistic increase in apoptotic Annexin V+/PI+ cells in these cell lines. Tumor xenografts in nude mice were established with 5 of the 6 cell lines. Effects of single agent treatment with JTP-74057 or GSK2126458 ranged from minor inhibition to tumor stasis, while dual inhibitor treatment showed synergistic effects in each case, with marked tumor reductions in 2/5 cases and the authors report one compete response in a MaMel301 cell line xenograft mouse.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>19, \"name\"=>\"Trametinib\", \"pubchem_id\"=>nil}, {\"id\"=>346, \"name\"=>\"Omipalisib (GSK2126458)\", \"pubchem_id\"=>\"\"}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>968, \"name\"=>\"Combined targeting of MEK and PI3K/mTOR effector pathways is necessary to effectively inhibit NRAS mutant melanoma in vitro and in vivo.\", \"citation\"=>\"Posch et al., 2013, Proc. Natl. Acad. Sci. U.S.A.\", \"pubmed_id\"=>\"23431193\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23431193\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3593920\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>3, \"day\"=>5}, \"journal\"=>\"Proc. Natl. Acad. Sci. U.S.A.\", \"full_journal_title\"=>\"Proceedings of the National Academy of Sciences of the United States of America\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>208, \"phenotypes\"=>[]}, {\"id\"=>2937, \"name\"=>\"EID2937\", \"description\"=>\"NEMO is an ongoing, randomised, open-label phase 3 study of patients with advanced, unresectable, AJCC IIIC/IV NRAS-mutant melanoma. 402 patients that were previously untreated or progressed on or after previous immunotherapy were randomised to receive either binimetinib (269) or dacarbazine (133). The study's primary endpoint was progression free survival. Median follow-up was 1.7 months (IQR 1.4-4.1). Median progression-free survival was 2.8 months (95% CI 2.8-3.6) in the binimetinib group and 1.5 months (1.5-1.7) in the dacarbazine group, demonstrating improved progression-free survival of binimetinib compared with dacarbazine.\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>485, \"name\"=>\"Binimetinib\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1686, \"name\"=>\"Binimetinib versus dacarbazine in patients with advanced NRAS-mutant melanoma (NEMO): a multicentre, open-label, randomised, phase 3 trial.\", \"citation\"=>\"Dummer et al., 2017, Lancet Oncol.\", \"pubmed_id\"=>\"28284557\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28284557\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>4}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>208, \"phenotypes\"=>[]}, {\"id\"=>2931, \"name\"=>\"EID2931\", \"description\"=>\"Inhibition of CDK4 and 6, regulators of the G1/S cell cycle checkpoint, arrests the cell cycle in the G1 phase, suppresses DNA synthesis, and inhibits cancer cell growth. There is an ongoing phase 1/2 trial of binimetinib with the CDK4/6 inhibitor ribociclib (LEE011) in mutant NRAS melanoma patients (NCT01781572). Early results from the 21 evaluable patients were very promising, with 7 patients achieving a partial response (33%) and 11 patients with stable disease (52%)\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>381, \"name\"=>\"Ribociclib\", \"pubchem_id\"=>nil}, {\"id\"=>485, \"name\"=>\"Binimetinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1682, \"name\"=>\"Targeting mutant NRAS signaling pathways in melanoma.\", \"citation\"=>\"Vu et al., 2016, Pharmacol. Res.\", \"pubmed_id\"=>\"26987942\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26987942\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4867277\", \"publication_date\"=>{\"year\"=>2016}, \"journal\"=>\"Pharmacol. Res.\", \"full_journal_title\"=>\"Pharmacological research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>208, \"phenotypes\"=>[]}, {\"id\"=>1703, \"name\"=>\"EID1703\", \"description\"=>\"A cohort of patient-derived xenografts (PDX) from 85 patients with metastatic colorectal cancer was created. PDX were treated with cetuximab and mechanisms of resistance investigated. None of the xenografts harboring KRAS (N=18), NRAS (N=7) or BRAF (N=3) mutations showed a response to cetuximab whereas 1 out of 4 xenografts with a PIK3CA mutation responded to cetuximab.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1138, \"name\"=>\"A molecularly annotated platform of patient-derived xenografts (\\\"xenopatients\\\") identifies HER2 as an effective therapeutic target in cetuximab-resistant colorectal cancer.\", \"citation\"=>\"Bertotti et al., 2011, Cancer Discov\", \"pubmed_id\"=>\"22586653\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22586653\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>11}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>208, \"phenotypes\"=>[]}, {\"id\"=>1509, \"name\"=>\"EID1509\", \"description\"=>\"In a panel of 7 NRAS mutant melanoma cell lines, all 7 were found to be sensitive to the MEK inhibitor MEK162, whereas 3 of 5 melanoma cell lines that were wild type for BRAF and NRAS were not sensitive to MEK162. In further tests using two of the seven NRAS mutant cell lines (YUDOSO Q16K/wt and YUKIM Q16R), colony forming capacity was reduced with increased MEK162 treatment, and MEK162 induced apoptotic markers in these cells.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>355, \"name\"=>\"Binimetinib (MEK162)\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1008, \"name\"=>\"MEK targeting in N-RAS mutated metastatic melanoma.\", \"citation\"=>\"Thumar et al., 2014, Mol. Cancer\", \"pubmed_id\"=>\"24588908\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24588908\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3945937\", \"publication_date\"=>{\"year\"=>2014}, \"journal\"=>\"Mol. Cancer\", \"full_journal_title\"=>\"Molecular cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>208, \"phenotypes\"=>[]}, {\"id\"=>1227, \"name\"=>\"EID1227\", \"description\"=>\"Phase 1 trial in 97 patients with melanoma (81 cutaneous or unknown, 16 uveal). Seven patients were found to harbor an NRAS mutation and two of them experienced stable disease (one 48 weeks on treatment). Patients with concurrent BRAFV600wt/NRASwt (n=20) had a trend of higher response rates (20%) than BRAFV600wt/NRAS-mutant patients (n=11; RR=0%; p=0.27), as well as a trend of higher percentage of patients on study at Week 24 or at 1 year (40% vs. 18% [p=0.26] and 30% vs. 0% [p=0.07], respectively).\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>19, \"name\"=>\"Trametinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>850, \"name\"=>\"Activity of the oral MEK inhibitor trametinib in patients with advanced melanoma: a phase 1 dose-escalation trial.\", \"citation\"=>\"Falchook et al., 2012, Lancet Oncol.\", \"pubmed_id\"=>\"22805292\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22805292\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4109286\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>8}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>208, \"phenotypes\"=>[]}, {\"id\"=>1225, \"name\"=>\"EID1225\", \"description\"=>\"Phase 1 trial of trametinib and docetaxel in patients with advanced melanoma lacking BRAF V600 mutations. 15 patients were enrolled (14 BRAF wt). Six patients had a partial response, including four out of eight patients with NRAS mutations.\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>19, \"name\"=>\"Trametinib\", \"pubchem_id\"=>nil}, {\"id\"=>73, \"name\"=>\"Docetaxel\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>848, \"name\"=>\"PACMEL: a phase 1 dose escalation trial of trametinib (GSK1120212) in combination with paclitaxel.\", \"citation\"=>\"Coupe et al., 2015, Eur. J. Cancer\", \"pubmed_id\"=>\"25542057\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25542057\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>2}, \"journal\"=>\"Eur. J. Cancer\", \"full_journal_title\"=>\"European journal of cancer (Oxford, England : 1990)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>208, \"phenotypes\"=>[]}, {\"id\"=>1476, \"name\"=>\"EID1476\", \"description\"=>\"Preclinical study in melanoma cell lines. Cell lines (2-5/assay) with NRAS mutation were more sensitive to Amuvatinib (inhibitor targets multiple tyrosine kinases) than BRAF mutant cell lines in growth inhibition, tyrosine phosphorylation inhibition, cell cycle and apoptotic (Annexin V) assays.\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>384, \"name\"=>\"Amuvatinib\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>989, \"name\"=>\"Amuvatinib has cytotoxic effects against NRAS-mutant melanoma but not BRAF-mutant melanoma.\", \"citation\"=>\"Fedorenko et al., 2014, Melanoma Res.\", \"pubmed_id\"=>\"24950457\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24950457\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4384823\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>10}, \"journal\"=>\"Melanoma Res.\", \"full_journal_title\"=>\"Melanoma research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>208, \"phenotypes\"=>[]}, {\"id\"=>1472, \"name\"=>\"EID1472\", \"description\"=>\"Phase II study in 71 patients with NRAS or BRAF V600 mutant melanoma treated with MEK162. Six (20%) of 30 patients with NRAS-mutated melanoma had a partial response (three confirmed).\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>355, \"name\"=>\"Binimetinib (MEK162)\", \"pubchem_id\"=>\"\"}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>849, \"name\"=>\"MEK162 for patients with advanced melanoma harbouring NRAS or Val600 BRAF mutations: a non-randomised, open-label phase 2 study.\", \"citation\"=>\"Ascierto et al., 2013, Lancet Oncol.\", \"pubmed_id\"=>\"23414587\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23414587\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>3}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>208, \"phenotypes\"=>[]}, {\"id\"=>1230, \"name\"=>\"EID1230\", \"description\"=>\"Preclinical study in 16 NRAS mutant cell lines (melanoma, lung carcinoma, neuroblastoma with various Q61, G13, G12 mutations). The combination of metformin and trametinib shown synergistic effects in most tested cell lines (no antagonistic effects observed) and a melanoma xenograft model. Metformin-induced PI3K pathway suppression was identified as a potential mechanism.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>19, \"name\"=>\"Trametinib\", \"pubchem_id\"=>nil}, {\"id\"=>308, \"name\"=>\"Metformin\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>851, \"name\"=>\"Metformin and trametinib have synergistic effects on cell viability and tumor growth in NRAS mutant cancer.\", \"citation\"=>\"Vujic et al., 2015, Oncotarget\", \"pubmed_id\"=>\"25504439\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25504439\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4359268\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>208, \"phenotypes\"=>[]}, {\"id\"=>996, \"name\"=>\"EID996\", \"description\"=>\"Phase I expansion and pharmacodynamic study of the oral MEK inhibitor RO4987655 (CH4987655). Among 8 patients with NRAS mutated melanoma, 4 experienced SD or PR (1 Patient received study treatment for 323 days).\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>243, \"name\"=>\"RO4987655\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>688, \"name\"=>\"Phase I expansion and pharmacodynamic study of the oral MEK inhibitor RO4987655 (CH4987655) in selected patients with advanced cancer with RAS-RAF mutations.\", \"citation\"=>\"Zimmer et al., 2014, Clin. Cancer Res.\", \"pubmed_id\"=>\"24947927\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24947927\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>8, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>208, \"phenotypes\"=>[]}, {\"id\"=>496, \"name\"=>\"EID496\", \"description\"=>\"In patients with multiple myeloma, the presence of a mutation in NRAS was not significantly associated with progression-free or overall survival.\", \"disease\"=>{\"id\"=>41, \"name\"=>\"Multiple Myeloma\", \"display_name\"=>\"Multiple Myeloma\", \"doid\"=>\"9538\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9538\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"N/A\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>297, \"name\"=>\"Clinical and biological significance of RAS mutations in multiple myeloma.\", \"citation\"=>\"Chng et al., 2008, Leukemia\", \"pubmed_id\"=>\"18528420\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18528420\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3864109\", \"publication_date\"=>{\"year\"=>2008, \"month\"=>12}, \"journal\"=>\"Leukemia\", \"full_journal_title\"=>\"Leukemia\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>208, \"phenotypes\"=>[]}, {\"id\"=>71, \"name\"=>\"EID71\", \"description\"=>\"In patients with stage IV melanoma, NRAS mutation was associated with reduced median survival compared to patients with wildtype NRAS.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>87, \"name\"=>\"NRAS mutation status is an independent prognostic factor in metastatic melanoma.\", \"citation\"=>\"Jakob et al., 2012, Cancer\", \"pubmed_id\"=>\"22180178\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22180178\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3310961\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>8, \"day\"=>15}, \"journal\"=>\"Cancer\", \"full_journal_title\"=>\"Cancer\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>208, \"phenotypes\"=>[]}, {\"id\"=>56, \"name\"=>\"EID56\", \"description\"=>\"The presence of NRAS mutation in AML patients does not impact diease prognosis (resistant disease, disease-free survival, complete remission rate, relapse rate, induction death).\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"N/A\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>83, \"name\"=>\"RAS mutation in acute myeloid leukemia is associated with distinct cytogenetic subgroups but does not influence outcome in patients younger than 60 years.\", \"citation\"=>\"Bowen et al., 2005, Blood\", \"pubmed_id\"=>\"15951308\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15951308\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>9, \"day\"=>15}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>208, \"phenotypes\"=>[]}, {\"id\"=>27, \"name\"=>\"EID27\", \"description\"=>\"Patients with colorectal cancer harboring NRAS mutation have poorer survival outcome and worse prognosis than patients with wildtype NRAS.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>70, \"name\"=>\"The predictive value of KRAS, NRAS, BRAF, PIK3CA and PTEN for anti-EGFR treatment in metastatic colorectal cancer: A systematic review and meta-analysis.\", \"citation\"=>\"Therkildsen et al., 2014, Acta Oncol\", \"pubmed_id\"=>\"24666267\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24666267\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>7}, \"journal\"=>\"Acta Oncol\", \"full_journal_title\"=>\"Acta oncologica (Stockholm, Sweden)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>208, \"phenotypes\"=>[]}]}, {\"id\"=>283, \"entrez_name\"=>\"MTOR\", \"entrez_id\"=>2475, \"name\"=>\"H1968Y\", \"description\"=>\"\", \"gene_id\"=>2073, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>160, \"name\"=>\"gain_of_function_variant\", \"display_name\"=>\"Gain Of Function Variant\", \"so_id\"=>\"SO:0002053\", \"description\"=>\"A sequence variant whereby new or enhanced function is conferred on the gene product.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002053\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>11188519, \"stop\"=>11188519, \"reference_bases\"=>\"G\", \"variant_bases\"=>\"A\", \"representative_transcript\"=>\"ENST00000361445.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/2073/summary/variants/283/summary#variant\", \"evidence_items\"=>[{\"id\"=>721, \"name\"=>\"EID721\", \"description\"=>\"In a cell line with expression of H1968Y or P2213S mTOR mutations, LY294002 and AZD5363 were more effective than temsirolimus or BYL719 in suppressing cell growth.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>112, \"name\"=>\"AZD5363\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"rejected\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>468, \"name\"=>\"Analysis of mTOR Gene Aberrations in Melanoma Patients and Evaluation of Their Sensitivity to PI3K-AKT-mTOR Pathway Inhibitors.\", \"citation\"=>\"Kong et al., 2016, Clin. Cancer Res.\", \"pubmed_id\"=>\"26490311\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26490311\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>2, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>283, \"phenotypes\"=>[]}, {\"id\"=>713, \"name\"=>\"EID713\", \"description\"=>\"In a cell line (HEK293T) with expression of H1968Y or P2213S mTOR mutations, LY294002 and AZD5363 were more effective than temsirolimus or BYL719 in suppressing cell growth.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>112, \"name\"=>\"AZD5363\", \"pubchem_id\"=>nil}, {\"id\"=>115, \"name\"=>\"LY294002\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>468, \"name\"=>\"Analysis of mTOR Gene Aberrations in Melanoma Patients and Evaluation of Their Sensitivity to PI3K-AKT-mTOR Pathway Inhibitors.\", \"citation\"=>\"Kong et al., 2016, Clin. Cancer Res.\", \"pubmed_id\"=>\"26490311\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26490311\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>2, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>283, \"phenotypes\"=>[]}]}, {\"id\"=>284, \"entrez_name\"=>\"MTOR\", \"entrez_id\"=>2475, \"name\"=>\"P2213S\", \"description\"=>\"\", \"gene_id\"=>2073, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>160, \"name\"=>\"gain_of_function_variant\", \"display_name\"=>\"Gain Of Function Variant\", \"so_id\"=>\"SO:0002053\", \"description\"=>\"A sequence variant whereby new or enhanced function is conferred on the gene product.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002053\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>11184580, \"stop\"=>11184580, \"reference_bases\"=>\"G\", \"variant_bases\"=>\"A\", \"representative_transcript\"=>\"ENST00000361445.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/2073/summary/variants/284/summary#variant\", \"evidence_items\"=>[{\"id\"=>722, \"name\"=>\"EID722\", \"description\"=>\"In a cell line (HEK293T) with expression of H1968Y or P2213S mTOR mutations, LY294002 and AZD5363 were more effective than temsirolimus or BYL719 in suppressing cell growth.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>112, \"name\"=>\"AZD5363\", \"pubchem_id\"=>nil}, {\"id\"=>115, \"name\"=>\"LY294002\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>468, \"name\"=>\"Analysis of mTOR Gene Aberrations in Melanoma Patients and Evaluation of Their Sensitivity to PI3K-AKT-mTOR Pathway Inhibitors.\", \"citation\"=>\"Kong et al., 2016, Clin. Cancer Res.\", \"pubmed_id\"=>\"26490311\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26490311\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>2, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>284, \"phenotypes\"=>[]}]}, {\"id\"=>310, \"entrez_name\"=>\"ERBB4\", \"entrez_id\"=>2066, \"name\"=>\"MUTATION\", \"description\"=>\"ERBB4 mutations have been identified in melanoma, lung adenocarcinoma and medulloblastoma. In-vitro studies suggest that melanoma cell lines harboring an ERBB4 mutation are sensitive to ERBB4 inhibition with lapatinib.\", \"gene_id\"=>1734, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>160, \"name\"=>\"gain_of_function_variant\", \"display_name\"=>\"Gain Of Function Variant\", \"so_id\"=>\"SO:0002053\", \"description\"=>\"A sequence variant whereby new or enhanced function is conferred on the gene product.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002053\"}, {\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>4.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>212295697, \"stop\"=>212578308, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000342788.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1734/summary/variants/310/summary#variant\", \"evidence_items\"=>[{\"id\"=>770, \"name\"=>\"EID770\", \"description\"=>\"ERBB4 was the most commonly mutated protein tyrosine kinase in this study in melanoma (19% in 79 patients). Seven of the identified mutations in ERBB4 (E317K, E452K, E542K, R544W, E563K, E836K and E872K) were transforming and activating in-vitro. ERBB4 inhibition with shRNA or lapatinib led to reduced cell growth.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>45, \"name\"=>\"Lapatinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>509, \"name\"=>\"Analysis of the tyrosine kinome in melanoma reveals recurrent mutations in ERBB4.\", \"citation\"=>\"Prickett et al., 2009, Nat. Genet.\", \"pubmed_id\"=>\"19718025\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19718025\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2897709\", \"publication_date\"=>{\"year\"=>2009, \"month\"=>10}, \"journal\"=>\"Nat. Genet.\", \"full_journal_title\"=>\"Nature genetics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>310, \"phenotypes\"=>[]}]}, {\"id\"=>311, \"entrez_name\"=>\"PIK3CA\", \"entrez_id\"=>5290, \"name\"=>\"MUTATION\", \"description\"=>\"PIK3CA mutations are observed in many cancers among the most prevalent are breast cancer,  colorectal cancer, and head and neck squamous cell carcinoma. These mutations have been observed in up to 40%, 20%, and 21% respectively for each of these cancer types with mutations commonly occurring within the helical exon 9 and kinase domain of exon 20. In breast cancer PIK3CA mutations have been associated with better prognosis, and a greater survival outcome, however the PI3K pathway is also associated with increased drug resistance ( trastuzumab/lapatinib). PIK3CA mutations in both HNSCC and CRC have been associated with resistance to anti-EGFR therapeutic agents and are associated with a poorer prognosis in CRC. Interestingly PIK3CA mutations in HNSCC have been observed to be absent from a number of ethnic groups including greek, german, and vietnamese populations.\", \"gene_id\"=>37, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>160, \"name\"=>\"gain_of_function_variant\", \"display_name\"=>\"Gain Of Function Variant\", \"so_id\"=>\"SO:0002053\", \"description\"=>\"A sequence variant whereby new or enhanced function is conferred on the gene product.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002053\"}, {\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>124.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>178866311, \"stop\"=>178957881, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000263967.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/37/summary/variants/311/summary#variant\", \"evidence_items\"=>[{\"id\"=>3040, \"name\"=>\"EID3040\", \"description\"=>\"In a study of 37 patients with AKT1 (E13K) mutations, 5 patients harbored a concurrent PI3K/mTOR pathway alteration. The PFS of patients with a concurrent PI3K/mTOR pathway alteration was significantly longer than those without when treated with AZD-5363 (median PFS, not reached v 4.3 months, respectively; HR, 0.21; P = 0.045).\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1745, \"name\"=>\"AKT Inhibition in Solid Tumors With AKT1 Mutations.\", \"citation\"=>\"Hyman et al., 2017, J. Clin. Oncol.\", \"pubmed_id\"=>\"28489509\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28489509\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>5, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}, {\"id\"=>1705, \"name\"=>\"EID1705\", \"description\"=>\"A cohort of patient-derived xenografts (PDX) from 85 patients with metastatic colorectal cancer was created. PDX were treated with cetuximab and mechanisms of resistance investigated. None of the xenografts harboring KRAS (N=18), NRAS (N=7) or BRAF (N=3) mutations showed a response to cetuximab and only 1 out of 4 xenografts with a PIK3CA mutation responded to cetuximab.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1138, \"name\"=>\"A molecularly annotated platform of patient-derived xenografts (\\\"xenopatients\\\") identifies HER2 as an effective therapeutic target in cetuximab-resistant colorectal cancer.\", \"citation\"=>\"Bertotti et al., 2011, Cancer Discov\", \"pubmed_id\"=>\"22586653\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22586653\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>11}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}, {\"id\"=>1384, \"name\"=>\"EID1384\", \"description\"=>\"The HER2 overexpressing breast cancer cell line BT-474 transduced with mutant PIK3CA H1047R demonstrated essentially normal proliferation under trastuzumab, while control cells showed negligible proliferation with trastuzumab. Sequencing of 55 HER2 breast cancer samples revealed 25% PIK3CA mutation rate and Kaplan Meier curves showed decreased time to progression for mutant PIK3CA at borderline statistical significance (p=0.052). This was potentially explained by presence of PTEN loss in wild-type PIK3CA sample and combining PTEN loss with PIK3CA mutation resulted in significant differences in time to progression (p=0.007).\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>938, \"name\"=>\"A functional genetic approach identifies the PI3K pathway as a major determinant of trastuzumab resistance in breast cancer.\", \"citation\"=>\"Berns et al., 2007, Cancer Cell\", \"pubmed_id\"=>\"17936563\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17936563\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>10}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}, {\"id\"=>1501, \"name\"=>\"EID1501\", \"description\"=>\"In a panel of 22 gastric cancer cell lines, 4 of 6 cell lines having the highest sensitivity to AZD5363 contained a PIK3CA mutation (E542K, E453K, E545K). A patient derived gastric cancer xenograft mouse model was established with PIK3CA H1047R mutant cells, and treatment with AZD5363 induced significant tumor growth inhibition in comparison to vehicle control.\", \"disease\"=>{\"id\"=>21, \"name\"=>\"Stomach Cancer\", \"display_name\"=>\"Stomach Cancer\", \"doid\"=>\"10534\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10534\"}, \"drugs\"=>[{\"id\"=>112, \"name\"=>\"AZD5363\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1004, \"name\"=>\"The AKT inhibitor AZD5363 is selectively active in PI3KCA mutant gastric cancer, and sensitizes a patient-derived gastric cancer xenograft model with PTEN loss to Taxotere.\", \"citation\"=>\"Li et al., 2013, J Transl Med\", \"pubmed_id\"=>\"24088382\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24088382\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3850695\", \"publication_date\"=>{\"year\"=>2013}, \"journal\"=>\"J Transl Med\", \"full_journal_title\"=>\"Journal of translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}, {\"id\"=>1503, \"name\"=>\"EID1503\", \"description\"=>\"Sensitivity to pan-Akt inhibitor AZD5363 was assessed in a panel of 182 cell lines (160 with PTEN evaluated) derived from solid and hematological tumors. PTEN loss of function due to loss or mutation (N=32) was found to be significantly correlated with AZD5363 sensitivity (p=0.0099), and correlation improved in the subpopulation free of RAS mutation. Mouse xenografts with 786-0 (PTEN-null renal cancer) cells were established. 75 mg/kg twice daily AZD5363 caused tumor growth inhibition while 150 mg/kg twice daily induced moderate tumor regression.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>112, \"name\"=>\"AZD5363\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1005, \"name\"=>\"Preclinical pharmacology of AZD5363, an inhibitor of AKT: pharmacodynamics, antitumor activity, and correlation of monotherapy activity with genetic background.\", \"citation\"=>\"Davies et al., 2012, Mol. Cancer Ther.\", \"pubmed_id\"=>\"22294718\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22294718\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>4}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}, {\"id\"=>1600, \"name\"=>\"EID1600\", \"description\"=>\"Three PI3K mutant breast cancer cell lines were rendered resistant to PI3K inhibitors (BYL719 or GDC-0941), and a panel of 42 inhibitor compounds was tested on the cells for ability to resensitize the cells to PI3K inhibition. The CDK4/6 inhibitor ribociclib (LEE011) acted synergistically with PI3K inhibitor to reduce viability in all three resistant cell lines. In a panel of 12 PIK3CA mutant and 10 wild type cell lines, synergistic effects between P110alpha inhibitor BYL719 and ribociclib were significantly stronger (weighted synergy score calculation, ANOVA, p=0.012) in PIK3CA mutant cells, indicating variant specific sensitization. Mouse xenografts of 4 different PIK3CA mutant breast cancer cells resistant to PI3K inhibitor showed stronger regression under dual PI3K inhibitor and ribociclib treatment than with either treatment alone.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>381, \"name\"=>\"Ribociclib\", \"pubchem_id\"=>nil}, {\"id\"=>382, \"name\"=>\"PI3K Inhibitor\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>178, \"name\"=>\"CDK 4/6 inhibitors sensitize PIK3CA mutant breast cancer to PI3K inhibitors.\", \"citation\"=>\"Vora et al., 2014, Cancer Cell\", \"pubmed_id\"=>\"25002028\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25002028\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4155598\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>7, \"day\"=>14}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}, {\"id\"=>1402, \"name\"=>\"EID1402\", \"description\"=>\"A set of 474 cell lines from the Cancer Cell Line Encyclopedia was tested for responsiveness to the PI3Kalpha specific inhibitor BYL719 (Alpelisib). In the PIK3CA wild type subroup of cell lines, 22% were assessed as responsive, while in the subgroup of cell lines with PIK3CA mutations, 64% were labeled as responders. 7 patient derived xenografts with PIK3CA mutations (H1047R, E545K, M1043I, C420R, Q546K) were transplanted into mice and treated with BYL719. 6/7 mice responded with strong tumor growth inhibition or regression, while one (H1047R) responded with weak growth inhibition.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>339, \"name\"=>\"BYL719 (Alpelisib)\", \"pubchem_id\"=>\"\"}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>950, \"name\"=>\"Characterization of the novel and specific PI3Kα inhibitor NVP-BYL719 and development of the patient stratification strategy for clinical trials.\", \"citation\"=>\"Fritsch et al., 2014, Mol. Cancer Ther.\", \"pubmed_id\"=>\"24608574\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24608574\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>5}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}, {\"id\"=>1360, \"name\"=>\"EID1360\", \"description\"=>\"Preclinical analysis of the predicitive value of PIK3CA mutations for mTOR/PI3K Inhibition with BEZ-235. Two cell lines with PIK3CA Mutation (H1047R) were more sensitive to BEZ-235 than two cell lines with wild-type PIK3CA (in-vitro, results from one cell line also confirmed in-vivo). Two patient-derived xenografts with PIK3CA mutations (E110K, E542K) showed sensitivity to BEZ-235.\", \"disease\"=>{\"id\"=>453, \"name\"=>\"Head And Neck Cancer\", \"display_name\"=>\"Head And Neck Cancer\", \"doid\"=>\"11934\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:11934\"}, \"drugs\"=>[{\"id\"=>343, \"name\"=>\"BEZ235 (NVP-BEZ235, Dactolisib)\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>927, \"name\"=>\"Frequent mutation of the PI3K pathway in head and neck cancer defines predictive biomarkers.\", \"citation\"=>\"Lui et al., 2013, Cancer Discov\", \"pubmed_id\"=>\"23619167\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23619167\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3710532\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>7}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}, {\"id\"=>1358, \"name\"=>\"EID1358\", \"description\"=>\"Analysis of drug response data from the Sanger center cell line screen. Cell lines with either PI3K and/or PTEN mutations did not show a significant difference in response to class I PI3K inhibitor GDC-0941 (p =0.07 for combined PI3K/PTEN aberrations with a trend towards lower IC50 for mutated cell lines).\\nIC50 values with HSP90 inhibitor 17-AAG were significantly lower in PI3K and/or PTEN mutated cell lines in the sanger cell line screening but these results could not be reproduced in the cancer cell line encyclopedia screening.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>10, \"name\"=>\"17-AAG\", \"pubchem_id\"=>nil}, {\"id\"=>141, \"name\"=>\"Pictilisib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>926, \"name\"=>\"Response of head and neck squamous cell carcinoma cells carrying PIK3CA mutations to selected targeted therapies.\", \"citation\"=>\"Wirtz et al., 2015, JAMA Otolaryngol Head Neck Surg\", \"pubmed_id\"=>\"25855885\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25855885\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4474763\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>6}, \"journal\"=>\"JAMA Otolaryngol Head Neck Surg\", \"full_journal_title\"=>\"JAMA otolaryngology-- head & neck surgery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}, {\"id\"=>1296, \"name\"=>\"EID1296\", \"description\"=>\"Archival tumor samples from 549 patients from the BOLERO-1 and -3 trials were analyzed for alterations in the PI3K pathway using next-generation sequencing, immunohistochemistry (IHC) and Sanger sequencing. In the BOLERO trials, patients with HER2-overexpressing advanced breast cancer were treated with trastuzumab, chemotherapy and randomized to receive either everolimus or placebo. In the pooled analysis of both trials, patients with PI3K mutations had longer progression-free survival with everolimus compared to placebo (HR 0.67 95%CI 0.45 to 1.00, P=0.05) whereas patients with PI3K wild type status did not benefit significantly (HR 1.1, 95%CI 0.83 to 1.46, P=0.5). The same was found when PI3K mutations, PTEN loss and AKT E17K mutations were combinedly analyzed as hyperactive PI3K pathway. Patients with PI3K hyperactive pathway had a hazard ratio of 0.67 (95%CI 0.48 to 0.93, P=0.02) compared to HR 1.19 (95%CI 0.87 to 1.62, P=0.28) in patients with PI3K normal pathway. These trends were independently confirmed in both trials but only pooled results showed consistent statistical significance.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>891, \"name\"=>\"Molecular Alterations and Everolimus Efficacy in Human Epidermal Growth Factor Receptor 2-Overexpressing Metastatic Breast Cancers: Combined Exploratory Biomarker Analysis From BOLERO-1 and BOLERO-3.\", \"citation\"=>\"André et al., 2016, J. Clin. Oncol.\", \"pubmed_id\"=>\"27091708\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27091708\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>6, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}, {\"id\"=>1364, \"name\"=>\"EID1364\", \"description\"=>\"6 head and neck cancer cell lines were tested for Response to 17-AAG, GDC-0941. Two of the cell lines did not carry PIK3CA mutations. No clear difference was shown between PIK3CA-mutant and wild-type cell lines and there was a trend towards increased resistance. SCC25 cell line was lentivirally transduced with PIK3CA E545K or H1047R mutation. Engineered cells, compared to the parental cell line, showed decreased sensitivity to 17-AAG, GDC-0941 or trametinib.\", \"disease\"=>{\"id\"=>453, \"name\"=>\"Head And Neck Cancer\", \"display_name\"=>\"Head And Neck Cancer\", \"doid\"=>\"11934\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:11934\"}, \"drugs\"=>[{\"id\"=>10, \"name\"=>\"17-AAG\", \"pubchem_id\"=>nil}, {\"id\"=>19, \"name\"=>\"Trametinib\", \"pubchem_id\"=>nil}, {\"id\"=>141, \"name\"=>\"Pictilisib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>926, \"name\"=>\"Response of head and neck squamous cell carcinoma cells carrying PIK3CA mutations to selected targeted therapies.\", \"citation\"=>\"Wirtz et al., 2015, JAMA Otolaryngol Head Neck Surg\", \"pubmed_id\"=>\"25855885\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25855885\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4474763\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>6}, \"journal\"=>\"JAMA Otolaryngol Head Neck Surg\", \"full_journal_title\"=>\"JAMA otolaryngology-- head & neck surgery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}, {\"id\"=>1607, \"name\"=>\"EID1607\", \"description\"=>\"Three PIK3CA mutant breast cancer cell lines were rendered resistant to PI3K inhibitors (BYL719 or GDC-0941) by chronic exposure. Co-treatment of all three cell lines with PI3K inhibitor and CDK4/6 inhibitor palbociclib resulted in synergistic reductions in cell viability which were not seen with treatment of either compound alone, and resulted in similar suppression to viability as seen in the respective PI3K inhibitor sensitive parental cell lines.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}, {\"id\"=>382, \"name\"=>\"PI3K Inhibitor\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>178, \"name\"=>\"CDK 4/6 inhibitors sensitize PIK3CA mutant breast cancer to PI3K inhibitors.\", \"citation\"=>\"Vora et al., 2014, Cancer Cell\", \"pubmed_id\"=>\"25002028\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25002028\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4155598\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>7, \"day\"=>14}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}, {\"id\"=>771, \"name\"=>\"EID771\", \"description\"=>\"20 patient-derived colorectal cancer xenografts were treated with cabozantinib, a c-MET and VEGFR2 inhibitor. 16 xenografts showed sensitivity (as defined by TGII: tumor growth of treated compared to control ≤ 20%). Among the genes tested for mutations (KRAS, TP53, PIK3CA, MET, VEGFR2, RET and AXL), PIK3CA mutations (4 PDTX) showed a significant enrichment in sensitive xenografts. PIK3CA mutations were observed in 4/5 of the most responsive tumors. Treating xenografts derived from isogenic PIK3CA wildtype or mutant cell lines corroborated this finding with PIK3CA mutant xenografts showing greater sensitivity to cabozantinib.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>144, \"name\"=>\"Cabozantinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>510, \"name\"=>\"Potent antitumor activity of cabozantinib, a c-MET and VEGFR2 inhibitor, in a colorectal cancer patient-derived tumor explant model.\", \"citation\"=>\"Song et al., 2015, Int. J. Cancer\", \"pubmed_id\"=>\"25242168\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25242168\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4323738\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>4, \"day\"=>15}, \"journal\"=>\"Int. J. Cancer\", \"full_journal_title\"=>\"International journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}, {\"id\"=>1616, \"name\"=>\"EID1616\", \"description\"=>\"A panel of 24 endometrioid endometrial cancer (EEC) cell lines was characterized for a set of mutations and sensitivity to pan PI3K class I inhibitor GDC-0941 (pictilisib) was assessed. EEC cell lines with PIK3CA mutations were significantly more sensitive to GDC-0941 (Fisher’s exact test, two-tailed, P=0.038). Some of the PIK3CA mutant cells contained PTEN and KRAS mutations, but in these cells, these mutations did not effect sensitivity.\", \"disease\"=>{\"id\"=>108, \"name\"=>\"Endometrial Cancer\", \"display_name\"=>\"Endometrial Cancer\", \"doid\"=>\"1380\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1380\"}, \"drugs\"=>[{\"id\"=>141, \"name\"=>\"Pictilisib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1067, \"name\"=>\"PI3K pathway dependencies in endometrioid endometrial cancer cell lines.\", \"citation\"=>\"Weigelt et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"23674493\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23674493\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3700760\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}, {\"id\"=>1610, \"name\"=>\"EID1610\", \"description\"=>\"A panel of 31 breast cancer cell lines was tested for sensitivity to mTORC1 inhibitor everolimus and mTORC1/mTORC2 inhibitor PP242. Inhibitor concentrations for 50% survival fraction (SF50) were measured and SF50 less than one micro molar was considered sensitive. PIK3CA mutant cell lines were significantly more sensitive to both everolimus and PP242 than PIK3CA wild type cell lines. Apoptotic indicators and cell cycle analysis indicated that the mTOR inhibitors induced G1 arrest instead of apoptosis in the cell lines.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}, {\"id\"=>179, \"name\"=>\"PP242\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1066, \"name\"=>\"PIK3CA mutation, but not PTEN loss of function, determines the sensitivity of breast cancer cells to mTOR inhibitory drugs.\", \"citation\"=>\"Weigelt et al., 2011, Oncogene\", \"pubmed_id\"=>\"21358673\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21358673\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>7, \"day\"=>21}, \"journal\"=>\"Oncogene\", \"full_journal_title\"=>\"Oncogene\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}, {\"id\"=>915, \"name\"=>\"EID915\", \"description\"=>\"A meta-analysis of 8 metastatic colorectal cancer studies (839 patients) was performed. PIK3CA mutations were significantly associated with shorter PFS and OS following therapy with anti-EGFR monoclonal antibodies (PFS: 8 studies, 839 patients; HR = 1.53; 95 % confidence interval (CI) 1.28-1.84; P < 0.001; OS: 5 studies; 587 patients; HR = 1.28; 95 % CI 1.05-1.56; P = 0.015).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>252, \"name\"=>\"Anti-EGFR Monoclonal Antibody\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>622, \"name\"=>\"PIK3CA mutation is associated with poor survival among patients with metastatic colorectal cancer following anti-EGFR monoclonal antibody therapy: a meta-analysis.\", \"citation\"=>\"Wu et al., 2013, J. Cancer Res. Clin. Oncol.\", \"pubmed_id\"=>\"23435830\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23435830\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>5}, \"journal\"=>\"J. Cancer Res. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of cancer research and clinical oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}, {\"id\"=>892, \"name\"=>\"EID892\", \"description\"=>\"94 endometrial cancer samples of women who had been treated with single agent mTOR inhibitors temsirolimus or ridaforolimus were retrospectively analyzed.\\nNo predictive biomarkers were identified, including PIK3CA mutations (N=21/73), KRAS mutations (10/73) or PTEN loss.\", \"disease\"=>{\"id\"=>108, \"name\"=>\"Endometrial Cancer\", \"display_name\"=>\"Endometrial Cancer\", \"doid\"=>\"1380\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1380\"}, \"drugs\"=>[{\"id\"=>83, \"name\"=>\"Temsirolimus\", \"pubchem_id\"=>nil}, {\"id\"=>170, \"name\"=>\"Ridaforolimus\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>602, \"name\"=>\"Molecular determinants of outcome with mammalian target of rapamycin inhibition in endometrial cancer.\", \"citation\"=>\"Mackay et al., 2014, Cancer\", \"pubmed_id\"=>\"24166148\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24166148\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>2, \"day\"=>15}, \"journal\"=>\"Cancer\", \"full_journal_title\"=>\"Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}, {\"id\"=>1490, \"name\"=>\"EID1490\", \"description\"=>\"A panel of 26 HNSCC cell lines was tested for efficacy of the p110 alpha, delta and gamma isoform inhibitor taselisib (GDC-0032). Cell lines with PIK3CA mutation or amplification were the most sensitive. Four HNSCC cell lines with activating PIK3CA alterations showed synergistic cell death response to the combination of drug with radiation treatment. This synergism was not apparent in 4 tested HNSCC cell lines with wild-type PIK3CA or PTEN mutation. Nude mouse xenografts were established with Cal-33 PIK3CA H1047R cells. Radiation and taselisib treatments alone showed some growth inhibition in comparison to control, but only the combined treatment led durable tumor regressions.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>351, \"name\"=>\"Taselisib (GDC-0032)\", \"pubchem_id\"=>nil}, {\"id\"=>360, \"name\"=>\"Radiation\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>984, \"name\"=>\"Taselisib (GDC-0032), a Potent β-Sparing Small Molecule Inhibitor of PI3K, Radiosensitizes Head and Neck Squamous Carcinomas Containing Activating PIK3CA Alterations.\", \"citation\"=>\"Zumsteg et al., 2016, Clin. Cancer Res.\", \"pubmed_id\"=>\"26589432\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26589432\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4870591\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}, {\"id\"=>1504, \"name\"=>\"EID1504\", \"description\"=>\"HER2 overexpressing breast cancer cell lines with PIK3CA mutation BT474c (K111N) and HCC-1954 (H1047R) were used to establish mouse tumor xenografts. In BT474c xenografts, three dosing schedules of AZD5363 caused significant tumor inhibition (200 mg/kg twice daily > 200 mg/kg once daily > 100 mg/kg once daily). In HCC-1954 xenografts two doses of AZD5363 induced significant tumor inhibition (150 mg/kg twice daily > 75 mg/kg twice daily) while 30 mg/kg twice weekly trastuzumab was inactive.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>112, \"name\"=>\"AZD5363\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1005, \"name\"=>\"Preclinical pharmacology of AZD5363, an inhibitor of AKT: pharmacodynamics, antitumor activity, and correlation of monotherapy activity with genetic background.\", \"citation\"=>\"Davies et al., 2012, Mol. Cancer Ther.\", \"pubmed_id\"=>\"22294718\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22294718\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>4}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>311, \"phenotypes\"=>[]}]}, {\"id\"=>320, \"entrez_name\"=>\"MAPK1\", \"entrez_id\"=>5594, \"name\"=>\"E322K\", \"description\"=>\"\", \"gene_id\"=>4532, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>160, \"name\"=>\"gain_of_function_variant\", \"display_name\"=>\"Gain Of Function Variant\", \"so_id\"=>\"SO:0002053\", \"description\"=>\"A sequence variant whereby new or enhanced function is conferred on the gene product.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002053\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"22\", \"start\"=>22127164, \"stop\"=>22127164, \"reference_bases\"=>\"C\", \"variant_bases\"=>\"T\", \"representative_transcript\"=>\"ENST00000215832.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4532/summary/variants/320/summary#variant\", \"evidence_items\"=>[{\"id\"=>4797, \"name\"=>\"EID4797\", \"description\"=>\"In an in vitro study, a HSC-6 cell line expressing MAPK1 E322K mutation was associated with sensitivity to erlotinib treatment, compared to Cal33 cells expressing wild-type MAPK1. Sensitivity was determined by assessing senescence and EGFR and MAPK phosphorylation.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>15, \"name\"=>\"Erlotinib\", \"pubchem_id\"=>nil}], \"rating\"=>nil, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2132, \"name\"=>\"MAPK1E322K mutation increases head and neck squamous cell carcinoma sensitivity to erlotinib through enhanced secretion of amphiregulin.\", \"citation\"=>\"Wen et al., 2016, Oncotarget\", \"pubmed_id\"=>\"27004400\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27004400\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5029627\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>4, \"day\"=>26}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>320, \"phenotypes\"=>[]}, {\"id\"=>791, \"name\"=>\"EID791\", \"description\"=>\"Case report of a patient with stage IVA HNSCC with near-complete histologic response after 13 days of neoadjuvant erlotinib. Whole-exome sequencing of the pre-treatment tumor revealed a MAPK1 E322K mutation (allelic fraction 0.13). In-vitro studies in two HNSCC cell lines showed enhanced EGFR phosphorylation and erlotinib sensitivity in the mutant line compared with the wild-type line.\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>15, \"name\"=>\"Erlotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>525, \"name\"=>\"Genomic Correlate of Exceptional Erlotinib Response in Head and Neck Squamous Cell Carcinoma.\", \"citation\"=>\"Van Allen et al., 2015, JAMA Oncol\", \"pubmed_id\"=>\"26181029\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26181029\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4557203\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>5}, \"journal\"=>\"JAMA Oncol\", \"full_journal_title\"=>\"JAMA oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>320, \"phenotypes\"=>[]}]}, {\"id\"=>388, \"entrez_name\"=>\"KIT\", \"entrez_id\"=>3815, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>29, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>160, \"name\"=>\"gain_of_function_variant\", \"display_name\"=>\"Gain Of Function Variant\", \"so_id\"=>\"SO:0002053\", \"description\"=>\"A sequence variant whereby new or enhanced function is conferred on the gene product.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002053\"}, {\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>17.0, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>55524085, \"stop\"=>55606881, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000288135.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/29/summary/variants/388/summary#variant\", \"evidence_items\"=>[{\"id\"=>5573, \"name\"=>\"EID5573\", \"description\"=>\"Compared to non-treated control cells, Kasumi-1 cells (KIT-mutated cell lines) that were exposed to nilotinib (30 uM) showed a 60% reduction in colony numbers. Further evidence to support cell death included increased detection of Annexing V-positive apoptotic cells (6.9-fold increase) and increases in caspase-3 and caspase-8.\", \"disease\"=>{\"id\"=>379, \"name\"=>\"Acute Promyelocytic Leukemia\", \"display_name\"=>\"Acute Promyelocytic Leukemia\", \"doid\"=>\"0060318\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060318\"}, \"drugs\"=>[{\"id\"=>13, \"name\"=>\"Nilotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2281, \"name\"=>\"Inactivation of receptor tyrosine kinases reverts aberrant DNA methylation in acute myeloid leukemia.\", \"citation\"=>\"Shen et al., 2017, Clin. Cancer Res.\", \"pubmed_id\"=>\"28720666\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28720666\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>7, \"day\"=>18}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>388, \"phenotypes\"=>[]}, {\"id\"=>1222, \"name\"=>\"EID1222\", \"description\"=>\"In a phase II clinical trial of 43 melanoma patients harboring KIT mutations or amplifications, imatinib treatment exhibited significant anti-tumor activity with a median progression free survival of 3.5 months, tumor regression in 41.9% of patients, and a 1-year overall survival rate of 51%. 9/10 partial responses were in patients with exon 11 or 13 KIT mutations.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>845, \"name\"=>\"Phase II, open-label, single-arm trial of imatinib mesylate in patients with metastatic melanoma harboring c-Kit mutation or amplification.\", \"citation\"=>\"Guo et al., 2011, J. Clin. Oncol.\", \"pubmed_id\"=>\"21690468\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21690468\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>7, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>388, \"phenotypes\"=>[]}, {\"id\"=>919, \"name\"=>\"EID919\", \"description\"=>\"Human GISTs with different KIT mutations were grafted in 136 nude muce. Mice were then treated with control, imatinib, GDC-0941, or GDC-0941 + imatinib. The combination showed superior antitumor efficacy that was sustained after treatment withdrawal.\", \"disease\"=>{\"id\"=>2, \"name\"=>\"Gastrointestinal Stromal Tumor\", \"display_name\"=>\"Gastrointestinal Stromal Tumor\", \"doid\"=>\"9253\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9253\"}, \"drugs\"=>[{\"id\"=>5, \"name\"=>\"Imatinib\", \"pubchem_id\"=>nil}, {\"id\"=>141, \"name\"=>\"Pictilisib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>625, \"name\"=>\"A potent combination of the novel PI3K Inhibitor, GDC-0941, with imatinib in gastrointestinal stromal tumor xenografts: long-lasting responses after treatment withdrawal.\", \"citation\"=>\"Floris et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"23231951\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23231951\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4405777\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>2, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>388, \"phenotypes\"=>[]}]}, {\"id\"=>399, \"entrez_name\"=>\"BRAF\", \"entrez_id\"=>673, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>5, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>30, \"name\"=>\"gene_variant\", \"display_name\"=>\"Gene Variant\", \"so_id\"=>\"SO:0001564\", \"description\"=>\"A sequence variant where the structure of the gene is changed.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001564\"}, {\"id\"=>160, \"name\"=>\"gain_of_function_variant\", \"display_name\"=>\"Gain Of Function Variant\", \"so_id\"=>\"SO:0002053\", \"description\"=>\"A sequence variant whereby new or enhanced function is conferred on the gene product.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002053\"}], \"civic_actionability_score\"=>46.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>140453136, \"stop\"=>140481403, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000288602.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5/summary/variants/399/summary#variant\", \"evidence_items\"=>[{\"id\"=>3046, \"name\"=>\"EID3046\", \"description\"=>\"Next-generation sequencing (NGS) often identifies mutations with unclear clinical or prognostic implications. One such example is BRAF mutations that occur outside of codon 600 (non-V600 BRAF mutations). A multicenter, retrospective cohort study to characterize the clinical, pathologic, and survival implications of non-V600 BRAF mutations in metastatic CRC. A total of 9,643 patients with metastatic CRC underwent NGS testing. We identified 208 patients with non-V600 BRAF mutations, which occurred in 2.2% of all patients tested and accounted for 22% of all BRAF mutations identified. Cancers with non-V600 BRAF mutations, compared with cancers with V600E mutations, were found in patients who were significantly younger (58 v 68 years, respectively), fewer female patients (46% v 65%, respectively), and patients who had fewer high-grade tumors (13% v 64%, respectively) or right-sided primary tumors (36% v 81%, respectively). Median overall survival was significantly longer in patients with non-V600 BRAF-mutant metastatic CRC compared with those with both V600E BRAF-mutant and wild-type BRAF metastatic CRC (60.7 v 11.4 v 43.0 months).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1750, \"name\"=>\"Non-V600 BRAF Mutations Define a Clinically Distinct Molecular Subtype of Metastatic Colorectal Cancer.\", \"citation\"=>\"Jones et al., 2017, J. Clin. Oncol.\", \"pubmed_id\"=>\"28486044\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28486044\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>5, \"day\"=>9}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>399, \"phenotypes\"=>[]}, {\"id\"=>1704, \"name\"=>\"EID1704\", \"description\"=>\"A cohort of patient-derived xenografts (PDX) from 85 patients with metastatic colorectal cancer was created. PDX were treated with cetuximab and mechanisms of resistance investigated. None of the xenografts harboring KRAS (N=18), NRAS (N=7) or BRAF (N=3) mutations showed a response to cetuximab whereas 1 out of 4 xenografts with a PIK3CA mutation responded to cetuximab.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1138, \"name\"=>\"A molecularly annotated platform of patient-derived xenografts (\\\"xenopatients\\\") identifies HER2 as an effective therapeutic target in cetuximab-resistant colorectal cancer.\", \"citation\"=>\"Bertotti et al., 2011, Cancer Discov\", \"pubmed_id\"=>\"22586653\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22586653\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>11}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>399, \"phenotypes\"=>[]}, {\"id\"=>1404, \"name\"=>\"EID1404\", \"description\"=>\"A quantitative synthesis was performed on nine studies comparing treatment of metastatic colorectal cancer with cetuximab or panitumumab and chemotherapy, versus chemotherapy alone, or with other targeted inhibitors. It was found that in the patient subgroup with BRAF mutation (V600E in the majority of cases), there were no benefits to overall survival, progression free survival, or overall response rate with addition of cetuximab or panitumumab to treatment. This conclusion held in the first line treatment as well as general treatment setting.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}, {\"id\"=>28, \"name\"=>\"Panitumumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>951, \"name\"=>\"Predictive role of BRAF mutations in patients with advanced colorectal cancer receiving cetuximab and panitumumab: a meta-analysis.\", \"citation\"=>\"Pietrantonio et al., 2015, Eur. J. Cancer\", \"pubmed_id\"=>\"25673558\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25673558\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>3}, \"journal\"=>\"Eur. J. Cancer\", \"full_journal_title\"=>\"European journal of cancer (Oxford, England : 1990)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>399, \"phenotypes\"=>[]}, {\"id\"=>936, \"name\"=>\"EID936\", \"description\"=>\"In this screen of 218 solid cancer cell lines, BRAF mutations were predictive of response to the MEK inhibitor GSK1120212. 26 of these cell lines had the BRAF V600E mutation, one cell line had a G469A mutation, one had G596R mutation and one had an unspecified mutation. Also of note, in RAF/RAS mutant colon cancer cell lines, co-occurring PIK3CA/PTEN mutations led to a cytostatic response rather than a cytotoxic response.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>19, \"name\"=>\"Trametinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>640, \"name\"=>\"Comprehensive predictive biomarker analysis for MEK inhibitor GSK1120212.\", \"citation\"=>\"Jing et al., 2012, Mol. Cancer Ther.\", \"pubmed_id\"=>\"22169769\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22169769\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>3}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>399, \"phenotypes\"=>[]}]}, {\"id\"=>424, \"entrez_name\"=>\"MYD88\", \"entrez_id\"=>4615, \"name\"=>\"L265P\", \"description\"=>\"\", \"gene_id\"=>3742, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>160, \"name\"=>\"gain_of_function_variant\", \"display_name\"=>\"Gain Of Function Variant\", \"so_id\"=>\"SO:0002053\", \"description\"=>\"A sequence variant whereby new or enhanced function is conferred on the gene product.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002053\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>31.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>38182641, \"stop\"=>38182641, \"reference_bases\"=>\"T\", \"variant_bases\"=>\"C\", \"representative_transcript\"=>\"ENST00000396334.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3742/summary/variants/424/summary#variant\", \"evidence_items\"=>[{\"id\"=>1489, \"name\"=>\"EID1489\", \"description\"=>\"MYD88 mutations were identified in 24 of 1080 screened  CLLs. They were exclusively seen in instances of mutated IGHV genes (18/23, P = 0.0001), a widely recognized prognostic signature for better outcome in CLL.\", \"disease\"=>{\"id\"=>52, \"name\"=>\"Chronic Lymphocytic Leukemia\", \"display_name\"=>\"Chronic Lymphocytic Leukemia\", \"doid\"=>\"1040\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1040\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>997, \"name\"=>\"Recurrent mutations refine prognosis in chronic lymphocytic leukemia.\", \"citation\"=>\"Baliakas et al., 2015, Leukemia\", \"pubmed_id\"=>\"24943832\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24943832\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>2}, \"journal\"=>\"Leukemia\", \"full_journal_title\"=>\"Leukemia\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>424, \"phenotypes\"=>[]}, {\"id\"=>986, \"name\"=>\"EID986\", \"description\"=>\"A prospective study of ibrutinib in 63 symptomatic patients with Waldenström's macroglobulinemia who had received at least one previous treatment. The response rates were highest among patients with MYD88(L265P) and CXCR4(WT). The MYD88 L265P mutation was present in 56 patients (89%).\", \"disease\"=>{\"id\"=>317, \"name\"=>\"Waldenström's Macroglobulinemia\", \"display_name\"=>\"Waldenström's Macroglobulinemia\", \"doid\"=>\"\", \"url\"=>\"http://www.disease-ontology.org/\"}, \"drugs\"=>[{\"id\"=>227, \"name\"=>\"Ibrutinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>683, \"name\"=>\"Ibrutinib in previously treated Waldenström's macroglobulinemia.\", \"citation\"=>\"Treon et al., 2015, N. Engl. J. Med.\", \"pubmed_id\"=>\"25853747\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25853747\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>4, \"day\"=>9}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>424, \"phenotypes\"=>[]}, {\"id\"=>1641, \"name\"=>\"EID1641\", \"description\"=>\"In a study of 30 patients with Waldenström’s macroglobulinemia, 87% had MYD88(L265P) mutations.  Waldenstrom macroglobulinemia cells expressing MYD88(L265P) were treated with an MYD88 homodimerization inhibitor or an IRAK 1/4 kinase inhibitor.  The cells showed a decrease in staining of NF-kappa-B p65 phosphorylation.\", \"disease\"=>{\"id\"=>317, \"name\"=>\"Waldenström's Macroglobulinemia\", \"display_name\"=>\"Waldenström's Macroglobulinemia\", \"doid\"=>\"\", \"url\"=>\"http://www.disease-ontology.org/\"}, \"drugs\"=>[{\"id\"=>393, \"name\"=>\"IRAK-1/4 Inhibitor\", \"pubchem_id\"=>nil}, {\"id\"=>536, \"name\"=>\"IMG-2005-5\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1086, \"name\"=>\"MYD88 L265P somatic mutation in Waldenström's macroglobulinemia.\", \"citation\"=>\"Treon et al., 2012, N. Engl. J. Med.\", \"pubmed_id\"=>\"22931316\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22931316\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>8, \"day\"=>30}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>424, \"phenotypes\"=>[]}]}, {\"id\"=>511, \"entrez_name\"=>\"FGFR2\", \"entrez_id\"=>2263, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>22, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>160, \"name\"=>\"gain_of_function_variant\", \"display_name\"=>\"Gain Of Function Variant\", \"so_id\"=>\"SO:0002053\", \"description\"=>\"A sequence variant whereby new or enhanced function is conferred on the gene product.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002053\"}, {\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>8.0, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>123239371, \"stop\"=>123357917, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000457416.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/22/summary/variants/511/summary#variant\", \"evidence_items\"=>[{\"id\"=>1912, \"name\"=>\"EID1912\", \"description\"=>\"Phase 1 study of FGFR-inhibitor BGJ398 in advanced cancer with FGFR-aberrations. Among three patients with cholangiocarcinoma and FGFR2 aberrations (2 fusions, 1 mutation), all three had stable disease with a reduced tumor burden.\", \"disease\"=>{\"id\"=>29, \"name\"=>\"Cholangiocarcinoma\", \"display_name\"=>\"Cholangiocarcinoma\", \"doid\"=>\"4947\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4947\"}, \"drugs\"=>[{\"id\"=>433, \"name\"=>\"BGJ-398\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1337, \"name\"=>\"Evaluation of BGJ398, a Fibroblast Growth Factor Receptor 1-3 Kinase Inhibitor, in Patients With Advanced Solid Tumors Harboring Genetic Alterations in Fibroblast Growth Factor Receptors: Results of a Global Phase I, Dose-Escalation and Dose-Expansion Study.\", \"citation\"=>\"Nogova et al., 2017, J. Clin. Oncol.\", \"pubmed_id\"=>\"27870574\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27870574\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>1, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>511, \"phenotypes\"=>[]}, {\"id\"=>1236, \"name\"=>\"EID1236\", \"description\"=>\"Preclinical study in endometrial cancer cell lines (MFE-296, AN3CA, MFE-280 and Ishikawa). In cell proliferation assays, the three FGFR2 mutant cell lines were more sensitive to the pan-FGFR inhibitor ponatinib (AP24534) than the FGFR2 wild type cell line (Ishikawa) and human embryonic kidney cell line (HEK293). Ponatinib also inhibited FGFR2 kinase activity and phosphorylation (MFE-296 and MFE-280) as well as cell migration and invasion (MFE-296 and AN3CA) of FGFR2 mutated cancer cell lines.\", \"disease\"=>{\"id\"=>108, \"name\"=>\"Endometrial Cancer\", \"display_name\"=>\"Endometrial Cancer\", \"doid\"=>\"1380\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1380\"}, \"drugs\"=>[{\"id\"=>55, \"name\"=>\"Ponatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>855, \"name\"=>\"Antitumor effects and molecular mechanisms of ponatinib on endometrial cancer cells harboring activating FGFR2 mutations.\", \"citation\"=>\"Kim et al., 2016, Cancer Biol. Ther.\", \"pubmed_id\"=>\"26574622\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26574622\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4847821\", \"publication_date\"=>{\"year\"=>2016}, \"journal\"=>\"Cancer Biol. Ther.\", \"full_journal_title\"=>\"Cancer biology & therapy\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>511, \"phenotypes\"=>[]}]}, {\"id\"=>512, \"entrez_name\"=>\"ALK\", \"entrez_id\"=>238, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>1, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>30, \"name\"=>\"gene_variant\", \"display_name\"=>\"Gene Variant\", \"so_id\"=>\"SO:0001564\", \"description\"=>\"A sequence variant where the structure of the gene is changed.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001564\"}, {\"id\"=>160, \"name\"=>\"gain_of_function_variant\", \"display_name\"=>\"Gain Of Function Variant\", \"so_id\"=>\"SO:0002053\", \"description\"=>\"A sequence variant whereby new or enhanced function is conferred on the gene product.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002053\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>29415640, \"stop\"=>30144432, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000389048.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1/summary/variants/512/summary#variant\", \"evidence_items\"=>[{\"id\"=>1237, \"name\"=>\"EID1237\", \"description\"=>\"In a phase 1 clinical trial (NCT01283516) 130 patients with advanced cancers all harboring ALK alterations were treated with ceritinib.  Of 114 NSCLC patients treated with at least 400 mg/daily ceritinib, 1 had a complete response, 65 had a partial response, 25 had stable disease, 12 had progressive disease, and 11 withdrew from the study early. Ceritinib dosed at 400 mg or higher had similar results in patients who had or had not received prior crizotinib treatment. Overall response rate for patients with prior crizotinib treatment (83 out of 122) was 56% with median progression free survival of 6.9 months. For patients who had no prior crizotinib treatment, median progression free survival was 10.4 months.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>159, \"name\"=>\"Ceritinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>856, \"name\"=>\"Ceritinib in ALK-rearranged non-small-cell lung cancer.\", \"citation\"=>\"Shaw et al., 2014, N. Engl. J. Med.\", \"pubmed_id\"=>\"24670165\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24670165\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4079055\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>3, \"day\"=>27}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>512, \"phenotypes\"=>[]}]}, {\"id\"=>519, \"entrez_name\"=>\"FLT3\", \"entrez_id\"=>2322, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>24, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>160, \"name\"=>\"gain_of_function_variant\", \"display_name\"=>\"Gain Of Function Variant\", \"so_id\"=>\"SO:0002053\", \"description\"=>\"A sequence variant whereby new or enhanced function is conferred on the gene product.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002053\"}, {\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>40.0, \"coordinates\"=>{\"chromosome\"=>\"13\", \"start\"=>28577411, \"stop\"=>28674729, \"reference_bases\"=>nil, \"variant_bases\"=>\"\", \"representative_transcript\"=>\"ENST00000241453.7\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>\"\", \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/24/summary/variants/519/summary#variant\", \"evidence_items\"=>[{\"id\"=>5575, \"name\"=>\"EID5575\", \"description\"=>\"Of the 14 AML patients receiving nilotinib, 80% achieved complete remission or complete remission with incomplete platelet recovery. Genetic testing revealed that 27% of these individuals conferred a FLT3 mutation and 0% had a pathogenic KIT mutation.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[{\"id\"=>13, \"name\"=>\"Nilotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2281, \"name\"=>\"Inactivation of receptor tyrosine kinases reverts aberrant DNA methylation in acute myeloid leukemia.\", \"citation\"=>\"Shen et al., 2017, Clin. Cancer Res.\", \"pubmed_id\"=>\"28720666\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28720666\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>7, \"day\"=>18}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>519, \"phenotypes\"=>[]}, {\"id\"=>5574, \"name\"=>\"EID5574\", \"description\"=>\"Compared to non-treated control cells, MV4-11 cells (FLT3 mutated cell line) that were exposed to nilotinib (30 uM) showed a 58% reduction in colony numbers.Further evidence to support cell death included increased detection of Annexing V-positive apoptotic cells (3.2-fold increase) and increases in caspase-3 and caspase-8.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[{\"id\"=>13, \"name\"=>\"Nilotinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2281, \"name\"=>\"Inactivation of receptor tyrosine kinases reverts aberrant DNA methylation in acute myeloid leukemia.\", \"citation\"=>\"Shen et al., 2017, Clin. Cancer Res.\", \"pubmed_id\"=>\"28720666\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28720666\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>7, \"day\"=>18}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>519, \"phenotypes\"=>[]}, {\"id\"=>5261, \"name\"=>\"EID5261\", \"description\"=>\"Midostaurin is FDA-approved for the treatment of newly-diagnosed acute myeloid leukemia patients harboring FLT3 mutations.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[{\"id\"=>323, \"name\"=>\"Midostaurin\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2224, \"name\"=>\"Midostaurin approved for FLT3-mutated AML.\", \"citation\"=>\"Levis, 2017, Blood\", \"pubmed_id\"=>\"28546144\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28546144\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>6, \"day\"=>29}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>true, \"clinical_trials\"=>[]}, \"variant_id\"=>519, \"phenotypes\"=>[]}, {\"id\"=>156, \"name\"=>\"EID156\", \"description\"=>\"FLT3 mutations did not alter overall survival or remission duration in younger (16-60), cytogenetically normal AML patients with CEBPA mutations. 10/36 patients with CEBPA mutations also had FLT3 mutations (8 FLT3-ITD, 2 FLT3-D835).\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"N/A\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>133, \"name\"=>\"CEBPA mutations in younger adults with acute myeloid leukemia and normal cytogenetics: prognostic relevance and analysis of cooperating mutations.\", \"citation\"=>\"Fröhling et al., 2004, J. Clin. Oncol.\", \"pubmed_id\"=>\"14726504\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/14726504\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2004, \"month\"=>2, \"day\"=>15}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>519, \"phenotypes\"=>[]}]}, {\"id\"=>545, \"entrez_name\"=>\"FGFR2\", \"entrez_id\"=>2263, \"name\"=>\"N550K\", \"description\"=>\"\", \"gene_id\"=>22, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>160, \"name\"=>\"gain_of_function_variant\", \"display_name\"=>\"Gain Of Function Variant\", \"so_id\"=>\"SO:0002053\", \"description\"=>\"A sequence variant whereby new or enhanced function is conferred on the gene product.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002053\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>123258034, \"stop\"=>123258034, \"reference_bases\"=>\"A\", \"variant_bases\"=>\"T\", \"representative_transcript\"=>\"ENST00000457416.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/22/summary/variants/545/summary#variant\", \"evidence_items\"=>[{\"id\"=>1322, \"name\"=>\"EID1322\", \"description\"=>\"The endometrial cancer cell lines, AN3CA and MFE296, which carry the N550K FGFR2 mutation, were found to be sensitive to FGFR inhibition with PD174074 (10 to 40x more sensitive than cell lines with wild type FGFR2.).\", \"disease\"=>{\"id\"=>108, \"name\"=>\"Endometrial Cancer\", \"display_name\"=>\"Endometrial Cancer\", \"doid\"=>\"1380\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1380\"}, \"drugs\"=>[{\"id\"=>36, \"name\"=>\"PD173074\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>952, \"name\"=>\"Inhibition of activated fibroblast growth factor receptor 2 in endometrial cancer cells induces cell death despite PTEN abrogation.\", \"citation\"=>\"Byron et al., 2008, Cancer Res.\", \"pubmed_id\"=>\"18757403\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18757403\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2008, \"month\"=>9, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>545, \"phenotypes\"=>[]}]}, {\"id\"=>633, \"entrez_name\"=>\"PIK3R1\", \"entrez_id\"=>5295, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>4289, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>160, \"name\"=>\"gain_of_function_variant\", \"display_name\"=>\"Gain Of Function Variant\", \"so_id\"=>\"SO:0002053\", \"description\"=>\"A sequence variant whereby new or enhanced function is conferred on the gene product.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002053\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"5\", \"start\"=>67511548, \"stop\"=>67597649, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000521381.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4289/summary/variants/633/summary#variant\", \"evidence_items\"=>[{\"id\"=>1612, \"name\"=>\"EID1612\", \"description\"=>\"Sensitivity to mTORC1/2 inhibitor PP242 was assessed in a panel of breast cancer cell lines with various mutation backgrounds. PTEN and PIK3CA mutations were heavily represented in the panel. However, a PTEN and PIK3CA wild type cell line 'HS578T' was found to harbor a mutation in the PI3K regulatory protein PIK3R1. Like PIK3CA mutant cell lines, the PIK3R1 mutant HS578T line was also determined to be sensitive to PP242. Approximately 70% of PIK3CA wild type lines were not sensitive to PP242. This suggests that mutations in the PI3K holoenzyme may be targetable by mTORC1/2 inhibition.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>179, \"name\"=>\"PP242\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1066, \"name\"=>\"PIK3CA mutation, but not PTEN loss of function, determines the sensitivity of breast cancer cells to mTOR inhibitory drugs.\", \"citation\"=>\"Weigelt et al., 2011, Oncogene\", \"pubmed_id\"=>\"21358673\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21358673\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>7, \"day\"=>21}, \"journal\"=>\"Oncogene\", \"full_journal_title\"=>\"Oncogene\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>633, \"phenotypes\"=>[]}]}, {\"id\"=>1296, \"entrez_name\"=>\"CTNNB1\", \"entrez_id\"=>1499, \"name\"=>\"ACTIVATING MUTATION\", \"description\"=>\"\", \"gene_id\"=>1290, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>160, \"name\"=>\"gain_of_function_variant\", \"display_name\"=>\"Gain Of Function Variant\", \"so_id\"=>\"SO:0002053\", \"description\"=>\"A sequence variant whereby new or enhanced function is conferred on the gene product.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002053\"}, {\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>4.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>41240930, \"stop\"=>41281936, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000349496.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1290/summary/variants/1296/summary#variant\", \"evidence_items\"=>[{\"id\"=>2988, \"name\"=>\"EID2988\", \"description\"=>\"Activation of melanoma-cell-intrinsic WNT/β-catenin signalling pathway contributes to a lack of T-cell infiltration in melanoma. Molecular analysis of human metastatic melanoma samples revealed a correlation between activation of the WNT/β-catenin signalling pathway and absence of a T-cell gene expression signature. In autochthonous mouse melanoma models active β-catenin signalling results in T-cell exclusion and resistance to anti-PD-L1/anti-CTLA-4 monoclonal antibody therapy.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>495, \"name\"=>\"CTLA-4 Antibody\", \"pubchem_id\"=>nil}, {\"id\"=>496, \"name\"=>\"PD-L1 Antibody\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1709, \"name\"=>\"Melanoma-intrinsic β-catenin signalling prevents anti-tumour immunity.\", \"citation\"=>\"Spranger et al., 2015, Nature\", \"pubmed_id\"=>\"25970248\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25970248\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>7, \"day\"=>9}, \"journal\"=>\"Nature\", \"full_journal_title\"=>\"Nature\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1296, \"phenotypes\"=>[]}]}, {\"id\"=>23, \"entrez_name\"=>\"CCND3\", \"entrez_id\"=>896, \"name\"=>\"LOSS\", \"description\"=>\"Cyclin D has been shown in many cancer types to be misregulated. Well established for their oncogenic properties, the cyclins and the cyclin-dependent kinases (CDK's) they activate have been the focus of major research and development efforts over the past decade. The methods by which the cyclins are deregulated are widely variable, and range from genomic amplification to promoter methylation changes. Cyclin D3 loss has been reported in T-cell acute lymphoblastic leukemia (T-ALL), a seemingly unique trend when compared to the amplifcations and overexpressions of the other cyclin D's. Treating cyclin D3 knockout mice with the targeted therapeutic palbociclib significantly increased the median survival of a Notch-driven model of T-ALL.\", \"gene_id\"=>10, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>41902671, \"stop\"=>41909586, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000372991.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/10/summary/variants/23/summary#variant\", \"evidence_items\"=>[{\"id\"=>263, \"name\"=>\"EID263\", \"description\"=>\"Treatment of Notch-driven T-cell leukemia in Ccnd3 knockout mice with Palbociclib (PD-0332991) significantly increased median survival.\", \"disease\"=>{\"id\"=>27, \"name\"=>\"T-cell Leukemia\", \"display_name\"=>\"T-cell Leukemia\", \"doid\"=>\"715\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:715\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>177, \"name\"=>\"Therapeutic targeting of the cyclin D3:CDK4/6 complex in T cell leukemia.\", \"citation\"=>\"Sawai et al., 2012, Cancer Cell\", \"pubmed_id\"=>\"23079656\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23079656\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3493168\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>10, \"day\"=>16}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>23, \"phenotypes\"=>[]}]}, {\"id\"=>29, \"entrez_name\"=>\"CEBPA\", \"entrez_id\"=>1050, \"name\"=>\"MUTATION\", \"description\"=>\"CEBPA mutations are associated with cytogenetically normal acute myeloid leukemia (CN-AML) and a favorable prognosis. However, NPM1 and FLT3 mutations should also be assessed in CN-AML patients as concurrent mutations may have prognostic implications.\", \"gene_id\"=>15, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}, {\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>105.0, \"coordinates\"=>{\"chromosome\"=>\"19\", \"start\"=>33790840, \"stop\"=>33793470, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000498907.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/15/summary/variants/29/summary#variant\", \"evidence_items\"=>[{\"id\"=>167, \"name\"=>\"EID167\", \"description\"=>\"In young patients (age 16-60) with previously untreated, cytogenetically normal acute myeloid leukemia, CEBPA mutation was significantly associated with complete remission in multivariate analysis (odds ratio, 1.33; 95% confidence interval [CI], 1.01 to 1.74). Multivariate analysis was performed on all patients  that were evaluated for a mutation in at least 1 gene (CEBPA, FLT3-ITD or NPM1) totaling 693/872 patients in the study. Only 509 patients in the study were evaluated for CEBPA, 67 of which had a mutation.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>137, \"name\"=>\"Mutations and treatment outcome in cytogenetically normal acute myeloid leukemia.\", \"citation\"=>\"Schlenk et al., 2008, N. Engl. J. Med.\", \"pubmed_id\"=>\"18450602\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18450602\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2008, \"month\"=>5, \"day\"=>1}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>29, \"phenotypes\"=>[]}, {\"id\"=>122, \"name\"=>\"EID122\", \"description\"=>\"In a randomized trial of previously untreated younger (<60) patients with acute myeloid leukemia (excluding acute promyelocytic leukemia) and high-risk myelodysplastic syndrome CEBPA mutation status had no significant impact on patient response to ATRA (N=20) compared to randomized control (N=15).\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[{\"id\"=>265, \"name\"=>\"All-trans Retinoic Acid\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>118, \"name\"=>\"The impact on outcome of the addition of all-trans retinoic acid to intensive chemotherapy in younger patients with nonacute promyelocytic acute myeloid leukemia: overall results and results in genotypic subgroups defined by mutations in NPM1, FLT3, and CEBPA.\", \"citation\"=>\"Burnett et al., 2010, Blood\", \"pubmed_id\"=>\"19965647\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19965647\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>2, \"day\"=>4}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>29, \"phenotypes\"=>[]}, {\"id\"=>467, \"name\"=>\"EID467\", \"description\"=>\"In children with acute myeloid leukemia, those with mutations in CEBPA had better overall and event-free survival and decreased cummulative incidence of relapse.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>281, \"name\"=>\"Prevalence and prognostic implications of CEBPA mutations in pediatric acute myeloid leukemia (AML): a report from the Children's Oncology Group.\", \"citation\"=>\"Ho et al., 2009, Blood\", \"pubmed_id\"=>\"19304957\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19304957\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2943755\", \"publication_date\"=>{\"year\"=>2009, \"month\"=>6, \"day\"=>25}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>29, \"phenotypes\"=>[]}, {\"id\"=>468, \"name\"=>\"EID468\", \"description\"=>\"In a set of 315 pediatric patients with acute myeloid leukemia, 47 had mutations in CEBPA. Those with mutations had better overall survival than wild-type patients.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>282, \"name\"=>\"Prognostic implications of CEBPA mutations in pediatric acute myeloid leukemia: a report from the Japanese Pediatric Leukemia/Lymphoma Study Group.\", \"citation\"=>\"Matsuo et al., 2014, Blood Cancer J\", \"pubmed_id\"=>\"25014773\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25014773\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4219441\", \"publication_date\"=>{\"year\"=>2014}, \"journal\"=>\"Blood Cancer J\", \"full_journal_title\"=>\"Blood cancer journal\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>29, \"phenotypes\"=>[]}, {\"id\"=>168, \"name\"=>\"EID168\", \"description\"=>\"Overall survival and remission duration were significantly longer (p=0.01 and p=0.05, respectively) in younger (16-60), cytogenetically normal acute myeloid leukemia patients with CEBPA mutations (21/236 patients).\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>133, \"name\"=>\"CEBPA mutations in younger adults with acute myeloid leukemia and normal cytogenetics: prognostic relevance and analysis of cooperating mutations.\", \"citation\"=>\"Fröhling et al., 2004, J. Clin. Oncol.\", \"pubmed_id\"=>\"14726504\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/14726504\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2004, \"month\"=>2, \"day\"=>15}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>29, \"phenotypes\"=>[]}, {\"id\"=>166, \"name\"=>\"EID166\", \"description\"=>\"In a group of young (<60 years old) acute myeloid leukemia (exclude acute promyelocytic leukemia) and high-risk MDS patients, those with CEBPA mutations (35/423) showed marginally improved overall survival (36% vs 31%; P = .04) than those with wildtype CEBPA.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>118, \"name\"=>\"The impact on outcome of the addition of all-trans retinoic acid to intensive chemotherapy in younger patients with nonacute promyelocytic acute myeloid leukemia: overall results and results in genotypic subgroups defined by mutations in NPM1, FLT3, and CEBPA.\", \"citation\"=>\"Burnett et al., 2010, Blood\", \"pubmed_id\"=>\"19965647\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19965647\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>2, \"day\"=>4}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>29, \"phenotypes\"=>[]}]}, {\"id\"=>110, \"entrez_name\"=>\"PTEN\", \"entrez_id\"=>5728, \"name\"=>\"R233*\", \"description\"=>\"PTEN R233* has been shown to be a loss of function mutation, and PTEN loss has been the subject of considerable research in breast cancer. PTEN loss may sensitize cells to PI3K-mTOR inhibition. While still being debated, there is data to support that PTEN loss is both associated with poorer prognosis, and no change in prognosis.\", \"gene_id\"=>41, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>50, \"name\"=>\"stop_gained\", \"display_name\"=>\"Stop Gained\", \"so_id\"=>\"SO:0001587\", \"description\"=>\"A sequence variant whereby at least one base of a codon is changed, resulting in a premature stop codon, leading to a shortened polypeptide.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001587\"}, {\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>19.0, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>89717672, \"stop\"=>89717672, \"reference_bases\"=>\"C\", \"variant_bases\"=>\"T\", \"representative_transcript\"=>\"ENST00000371953.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/41/summary/variants/110/summary#variant\", \"evidence_items\"=>[{\"id\"=>343, \"name\"=>\"EID343\", \"description\"=>\"PTEN nonsense mutations, including R233*, have been shown to be inactivating and loss-of-function, but do not have prognostic value in glioblastoma multiforme patients.\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>211, \"name\"=>\"Loss of PTEN is not associated with poor survival in newly diagnosed glioblastoma patients of the temozolomide era.\", \"citation\"=>\"Carico et al., 2012, PLoS ONE\", \"pubmed_id\"=>\"22479427\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22479427\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3315579\", \"publication_date\"=>{\"year\"=>2012}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>110, \"phenotypes\"=>[]}, {\"id\"=>317, \"name\"=>\"EID317\", \"description\"=>\"Cells with PTEN deficiency have been shown to exhibit slowed growth in reponse to PI3K-mTOR inhibitors.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>67, \"name\"=>\"MTOR Inhibitors\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>198, \"name\"=>\"The PI3K pathway as drug target in human cancer.\", \"citation\"=>\"Courtney et al., 2010, J. Clin. Oncol.\", \"pubmed_id\"=>\"20085938\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20085938\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2834432\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>2, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>110, \"phenotypes\"=>[]}]}, {\"id\"=>125, \"entrez_name\"=>\"TSC1\", \"entrez_id\"=>7248, \"name\"=>\"LOSS-OF-FUNCTION\", \"description\"=>\"In a small cohort study of bladder cancer, patients with TSC1 mutations showed better responses to and increased treatment duration tolerance with the mTOR inhibitor everolimus. Additionally, patient-derived bladder cancer cell lines with TSC1 or TSC2 mutations are much more sensitive to everolimus treatment than TSC1/TSC2 wildtype cells.\", \"gene_id\"=>46, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>135766735, \"stop\"=>135820008, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000298552.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/46/summary/variants/125/summary#variant\", \"evidence_items\"=>[{\"id\"=>5837, \"name\"=>\"EID5837\", \"description\"=>\"In a retrospective study of 79 RCC patients treated with diverse rapalogs (mTOR inhibitors) inactivating TSC1 and TSC2-mutations and activating MTOR-mutations were significantly more common in responsers  than in non-responders.\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[{\"id\"=>67, \"name\"=>\"MTOR Inhibitors\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2363, \"name\"=>\"Mutations in TSC1, TSC2, and MTOR Are Associated with Response to Rapalogs in Patients with Metastatic Renal Cell Carcinoma.\", \"citation\"=>\"Kwiatkowski et al., 2016, Clin. Cancer Res.\", \"pubmed_id\"=>\"26831717\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26831717\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4976069\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>5, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>125, \"phenotypes\"=>[]}, {\"id\"=>155, \"name\"=>\"EID155\", \"description\"=>\"Tumors expressing TSC1 or TSC2 mutations show notably more sensitivity to everolimus than their wild-type counterparts.\", \"disease\"=>{\"id\"=>18, \"name\"=>\"Bladder Carcinoma\", \"display_name\"=>\"Bladder Carcinoma\", \"doid\"=>\"4007\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4007\"}, \"drugs\"=>[{\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>132, \"name\"=>\"Genome sequencing identifies a basis for everolimus sensitivity.\", \"citation\"=>\"Iyer et al., 2012, Science\", \"pubmed_id\"=>\"22923433\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22923433\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3633467\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>10, \"day\"=>12}, \"journal\"=>\"Science\", \"full_journal_title\"=>\"Science (New York, N.Y.)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>125, \"phenotypes\"=>[]}]}, {\"id\"=>131, \"entrez_name\"=>\"BRCA1\", \"entrez_id\"=>672, \"name\"=>\"LOSS-OF-FUNCTION\", \"description\"=>\"BRCA1 loss of function mutations have been shown to increase risk of breast and ovarian cancer in those carrying the allele in their germline. Treating BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant response.\", \"gene_id\"=>6, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}, {\"id\"=>97, \"name\"=>\"loss_of_heterozygosity\", \"display_name\"=>\"Loss Of Heterozygosity\", \"so_id\"=>\"SO:0001786\", \"description\"=>\"A functional variant whereby the sequence alteration causes a loss of function of one allele of a gene.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001786\"}], \"civic_actionability_score\"=>104.0, \"coordinates\"=>{\"chromosome\"=>\"17\", \"start\"=>41196312, \"stop\"=>41277387, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000357654.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/6/summary/variants/131/summary#variant\", \"evidence_items\"=>[{\"id\"=>4876, \"name\"=>\"EID4876\", \"description\"=>\"In part 2 of a two-part dose-escalation phase I trial (NCT01286987), 71 patients with Ewings sarcoma, breast, ovarian, small cell lung, or pancreatic cancers received 1.0 mg/day talazoparib treatment. Of patients with BRCA mutation-associated breast and ovarian cancers, confirmed responses were observed in 7 of 14 (50%) and 5 of 12 (42%) patients, respectively.\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>523, \"name\"=>\"Talazoparib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2143, \"name\"=>\"Phase I, Dose-Escalation, Two-Part Trial of the PARP Inhibitor Talazoparib in Patients with Advanced Germline BRCA1/2 Mutations and Selected Sporadic Cancers.\", \"citation\"=>\"de Bono et al., 2017, Cancer Discov\", \"pubmed_id\"=>\"28242752\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28242752\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>131, \"phenotypes\"=>[]}, {\"id\"=>4838, \"name\"=>\"EID4838\", \"description\"=>\"In part 2 of a two-part dose-escalation phase I trial (NCT01286987), 71 patients with Ewings sarcoma, breast, ovarian, small cell lung, or pancreatic cancers received 1.0 mg/day talazoparib treatment. Of patients with BRCA mutation-associated breast and ovarian cancers, confirmed responses were observed in 7 of 14 (50%) and 5 of 12 (42%) patients, respectively.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>523, \"name\"=>\"Talazoparib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2143, \"name\"=>\"Phase I, Dose-Escalation, Two-Part Trial of the PARP Inhibitor Talazoparib in Patients with Advanced Germline BRCA1/2 Mutations and Selected Sporadic Cancers.\", \"citation\"=>\"de Bono et al., 2017, Cancer Discov\", \"pubmed_id\"=>\"28242752\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28242752\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>131, \"phenotypes\"=>[]}, {\"id\"=>5815, \"name\"=>\"EID5815\", \"description\"=>\"302 patients (234 with progression events) with HER2-negative metastatic breast cancer and with a confirmed deleterious or suspected deleterious germline BRCA mutation (164 BRCA1, 129 BRCA2) had significantly improved progression-free survival and better response rate to olaparib (PARP inhibitor) than standard therapy (7.0 mo vs 4.2 mo, hazard ratio 0.58, 95% CI 0.43-0.8) in this randomized phase 3 trial (NCT02000622). For BRCA1 patients specifically, the hazard ratio was 0.54, 95% CI 0.37-0.79).\", \"disease\"=>{\"id\"=>369, \"name\"=>\"Her2-receptor Negative Breast Cancer\", \"display_name\"=>\"Her2-receptor Negative Breast Cancer\", \"doid\"=>\"0060080\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060080\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2329, \"name\"=>\"Olaparib for Metastatic Breast Cancer in Patients with a Germline BRCA Mutation.\", \"citation\"=>\"Robson et al., 2017, N. Engl. J. Med.\", \"pubmed_id\"=>\"28578601\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28578601\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"day\"=>10}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[{\"nct_id\"=>\"NCT02000622\", \"name\"=>\"Assessment of the Efficacy and Safety of Olaparib Monotherapy Versus Physicians Choice Chemotherapy in the Treatment of Metastatic Breast Cancer Patients With Germline BRCA1/2 Mutations.\", \"description\"=>\"This open label, randomised, controlled, multi-centre phase III study will assess the efficacy and safety of single agent olaparib vs standard of care based on physician's choice of capecitabine, vinorelbine or eribulin in metastatic breast cancer patients with gBRCA 1/2 mutations.\", \"clinical_trial_url\"=>\"https://clinicaltrials.gov/show/NCT02000622\"}]}, \"variant_id\"=>131, \"phenotypes\"=>[]}, {\"id\"=>1924, \"name\"=>\"EID1924\", \"description\"=>\"The authors show that in the CX-5461 and CX-3543, G-quadruplex stabilizing agents, demonstrate specific toxicity against BRCA deficient breast cancer derived cells and polyclonal patient-derived xenograft models. Xenografts harbored both a germline BRCA and a somatic BRCA mutation leading to disruption of both. The BRCA and NHEJ pathways are required for the repair of CX-5461 and CX-3543-induced DNA damage and failure to do so leads to lethality.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>439, \"name\"=>\"CX-5461\", \"pubchem_id\"=>nil}, {\"id\"=>440, \"name\"=>\"CX-3543\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Unknown\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1346, \"name\"=>\"CX-5461 is a DNA G-quadruplex stabilizer with selective lethality in BRCA1/2 deficient tumours.\", \"citation\"=>\"Xu et al., 2017, Nat Commun\", \"pubmed_id\"=>\"28211448\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28211448\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>2, \"day\"=>17}, \"journal\"=>\"Nat Commun\", \"full_journal_title\"=>\"Nature communications\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>131, \"phenotypes\"=>[]}, {\"id\"=>1523, \"name\"=>\"EID1523\", \"description\"=>\"In a study of 2,019 patients with prostate cancer, 18 had germline BRCA1 mutations, 61 had germline BRCA2 mutations and 1,940 were non-carriers.  Patients with BRCA1/2 mutations had a 2-fold higher incidence of poorly differentiated cancer (Gleason score >8), a higher incidence of advanced stage (37% vs. 28%; P=0.003), a higher incidence of nodal involvement (15% vs. 5%; P=0.0005), and a higher incidence of metastatic spread (18% vs. 9%; P=0.005).\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1016, \"name\"=>\"Germline BRCA mutations are associated with higher risk of nodal involvement, distant metastasis, and poor survival outcomes in prostate cancer.\", \"citation\"=>\"Castro et al., 2013, J. Clin. Oncol.\", \"pubmed_id\"=>\"23569316\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23569316\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3641696\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>5, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>131, \"phenotypes\"=>[]}, {\"id\"=>211, \"name\"=>\"EID211\", \"description\"=>\"The combination of PARP inhibition (Olaparib) with BRCA mutation results in irreparable and lethal DNA damage. Response rates are higher in BRCA1 mutation positive women with high grade serous or undifferentiated ovarian cancer.\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>150, \"name\"=>\"Advanced ovarian cancer: what should be the standard of care?\", \"citation\"=>\"Goff, 2013, J Gynecol Oncol\", \"pubmed_id\"=>\"23346317\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23346317\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3549512\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>1}, \"journal\"=>\"J Gynecol Oncol\", \"full_journal_title\"=>\"Journal of gynecologic oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>131, \"phenotypes\"=>[]}, {\"id\"=>845, \"name\"=>\"EID845\", \"description\"=>\"1/1 patients with BRCA1 homozygous deletion had a response to olaparib in this phase 2 study.\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>409, \"name\"=>\"DNA-Repair Defects and Olaparib in Metastatic Prostate Cancer.\", \"citation\"=>\"Mateo et al., 2015, N. Engl. J. Med.\", \"pubmed_id\"=>\"26510020\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26510020\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>10, \"day\"=>29}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>131, \"phenotypes\"=>[]}]}, {\"id\"=>132, \"entrez_name\"=>\"BRCA2\", \"entrez_id\"=>675, \"name\"=>\"LOSS-OF-FUNCTION\", \"description\"=>\"BRCA2 loss of function mutations have been shown to increase risk of breast and ovarian cancer in those carrying the allele in their germline. Treating BRCA mutant patients with PARP inhibitors such as Olaparib has led to significant response.\", \"gene_id\"=>7, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>92.0, \"coordinates\"=>{\"chromosome\"=>\"13\", \"start\"=>32889611, \"stop\"=>32973347, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000380152.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/7/summary/variants/132/summary#variant\", \"evidence_items\"=>[{\"id\"=>4875, \"name\"=>\"EID4875\", \"description\"=>\"In part 2 of a two-part dose-escalation phase I trial (NCT01286987), 71 patients with Ewings sarcoma, breast, ovarian, small cell lung, or pancreatic cancers received 1.0 mg/day talazoparib treatment. Of patients with BRCA mutation-associated breast and ovarian cancers, confirmed responses were observed in 7 of 14 (50%) and 5 of 12 (42%) patients, respectively.\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>523, \"name\"=>\"Talazoparib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2143, \"name\"=>\"Phase I, Dose-Escalation, Two-Part Trial of the PARP Inhibitor Talazoparib in Patients with Advanced Germline BRCA1/2 Mutations and Selected Sporadic Cancers.\", \"citation\"=>\"de Bono et al., 2017, Cancer Discov\", \"pubmed_id\"=>\"28242752\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28242752\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>132, \"phenotypes\"=>[]}, {\"id\"=>4839, \"name\"=>\"EID4839\", \"description\"=>\"In part 2 of a two-part dose-escalation phase I trial (NCT01286987), 71 patients with Ewings sarcoma, breast, ovarian, small cell lung, or pancreatic cancers received 1.0 mg/day talazoparib treatment. Of patients with BRCA mutation-associated breast and ovarian cancers, confirmed responses were observed in 7 of 14 (50%) and 5 of 12 (42%) patients, respectively.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>523, \"name\"=>\"Talazoparib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2143, \"name\"=>\"Phase I, Dose-Escalation, Two-Part Trial of the PARP Inhibitor Talazoparib in Patients with Advanced Germline BRCA1/2 Mutations and Selected Sporadic Cancers.\", \"citation\"=>\"de Bono et al., 2017, Cancer Discov\", \"pubmed_id\"=>\"28242752\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28242752\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>6}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>132, \"phenotypes\"=>[]}, {\"id\"=>5816, \"name\"=>\"EID5816\", \"description\"=>\"302 patients (234 with progression events) with HER2-negative metastatic breast cancer and with a confirmed deleterious or suspected deleterious germline BRCA mutation (164 BRCA1, 129 BRCA2) had significantly improved progression-free survival and better response rate to olaparib (PARP inhibitor) than standard therapy (7.0 mo vs 4.2 mo, hazard ratio 0.58, 95% CI 0.43-0.8) in this randomized phase 3 trial (NCT02000622). For BRCA2 patients specifically, the hazard ratio was 0.68, 95% CI 0.45-0.1.07.\", \"disease\"=>{\"id\"=>369, \"name\"=>\"Her2-receptor Negative Breast Cancer\", \"display_name\"=>\"Her2-receptor Negative Breast Cancer\", \"doid\"=>\"0060080\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060080\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2329, \"name\"=>\"Olaparib for Metastatic Breast Cancer in Patients with a Germline BRCA Mutation.\", \"citation\"=>\"Robson et al., 2017, N. Engl. J. Med.\", \"pubmed_id\"=>\"28578601\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28578601\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"day\"=>10}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[{\"nct_id\"=>\"NCT02000622\", \"name\"=>\"Assessment of the Efficacy and Safety of Olaparib Monotherapy Versus Physicians Choice Chemotherapy in the Treatment of Metastatic Breast Cancer Patients With Germline BRCA1/2 Mutations.\", \"description\"=>\"This open label, randomised, controlled, multi-centre phase III study will assess the efficacy and safety of single agent olaparib vs standard of care based on physician's choice of capecitabine, vinorelbine or eribulin in metastatic breast cancer patients with gBRCA 1/2 mutations.\", \"clinical_trial_url\"=>\"https://clinicaltrials.gov/show/NCT02000622\"}]}, \"variant_id\"=>132, \"phenotypes\"=>[]}, {\"id\"=>1925, \"name\"=>\"EID1925\", \"description\"=>\"The authors show that in the CX-5461 and CX-3543, G-quadruplex stabilizing agents, demonstrate specific toxicity against BRCA deficient breast cancer derived cells and polyclonal patient-derived xenograft models. The BRCA and NHE J pathways are required for the repair of CX-5461 and CX-3543-induced DNA damage and failure to do so leads to lethality.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>439, \"name\"=>\"CX-5461\", \"pubchem_id\"=>nil}, {\"id\"=>440, \"name\"=>\"CX-3543\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Unknown\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1346, \"name\"=>\"CX-5461 is a DNA G-quadruplex stabilizer with selective lethality in BRCA1/2 deficient tumours.\", \"citation\"=>\"Xu et al., 2017, Nat Commun\", \"pubmed_id\"=>\"28211448\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28211448\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>2, \"day\"=>17}, \"journal\"=>\"Nat Commun\", \"full_journal_title\"=>\"Nature communications\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>132, \"phenotypes\"=>[]}, {\"id\"=>1524, \"name\"=>\"EID1524\", \"description\"=>\"In a study of 2,019 patients with prostate cancer, 18 had germ-line BRCA1 mutations, 61 had germ-line BRCA2 mutations and 1,940 were non-carriers.  Patients with BRCA1/2 mutations had a 2-fold higher incidence of poorly differentiated cancer (Gleason score >8), a higher incidence of advanced stage (37% vs. 28%; P=0.003), a higher incidence of nodal involvement (15% vs. 5%; P=0.0005), and a higher incidence of metastatic spread (18% vs. 9%; P=0.005).\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1016, \"name\"=>\"Germline BRCA mutations are associated with higher risk of nodal involvement, distant metastasis, and poor survival outcomes in prostate cancer.\", \"citation\"=>\"Castro et al., 2013, J. Clin. Oncol.\", \"pubmed_id\"=>\"23569316\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23569316\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3641696\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>5, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>132, \"phenotypes\"=>[]}, {\"id\"=>1308, \"name\"=>\"EID1308\", \"description\"=>\"In pancreatic cell lines PL11 (FANCC-deficient) and CAPAN1 (BRCA2-mutated), sensitivity to DNA cross-linking agents was observed compared to FANCC/BRCA2 wildtype cell lines (Su86.86, CFPAC, AsPc1, and MiaPaCa2). Mitomycin C specifically led to regression in 8/11 xenografts of the CAPAN1 cell line compared to 2/9 in the CFPAC (BRCA2 wildtype) cell line.\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>325, \"name\"=>\"Mitomycin C\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>900, \"name\"=>\"In vivo therapeutic responses contingent on Fanconi anemia/BRCA2 status of the tumor.\", \"citation\"=>\"van der Heijden et al., 2005, Clin. Cancer Res.\", \"pubmed_id\"=>\"16243825\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16243825\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>10, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>132, \"phenotypes\"=>[]}, {\"id\"=>650, \"name\"=>\"EID650\", \"description\"=>\"In a phase 2 trial of patients with metastatic, castration-resistant prostate cancer, treated with olaparib tablets at a dose of 400 mg twice a day, all 7 patients harbouring BRCA2 loss of function alleles (biallelic somatic loss or germline mutations in combination with somatic copy loss or copy neutral LOH) demonstrated a clinical response.\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>409, \"name\"=>\"DNA-Repair Defects and Olaparib in Metastatic Prostate Cancer.\", \"citation\"=>\"Mateo et al., 2015, N. Engl. J. Med.\", \"pubmed_id\"=>\"26510020\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26510020\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>10, \"day\"=>29}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>132, \"phenotypes\"=>[]}, {\"id\"=>212, \"name\"=>\"EID212\", \"description\"=>\"The combination of PARP inhibition (Olaparib) with BRCA mutation results in irreparable and lethal DNA damage. Response rates are higher in BRCA2 mutation positive women with high grade serous or undifferentiated ovarian cancer.\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>150, \"name\"=>\"Advanced ovarian cancer: what should be the standard of care?\", \"citation\"=>\"Goff, 2013, J Gynecol Oncol\", \"pubmed_id\"=>\"23346317\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23346317\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3549512\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>1}, \"journal\"=>\"J Gynecol Oncol\", \"full_journal_title\"=>\"Journal of gynecologic oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>132, \"phenotypes\"=>[]}]}, {\"id\"=>155, \"entrez_name\"=>\"RUNX1\", \"entrez_id\"=>861, \"name\"=>\"MUTATION\", \"description\"=>\"RUNX1 mutations are associated with poor prognosis in patients with acute myeloid leukemia or adult T-cell acute lymphocytic leukemia. In younger (<60 years old) AML patients, allogeneic stem cell transplant at complete remission led to much better relapse-free survival than repetitive cycles of high-dose cytarabine.\", \"gene_id\"=>43, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}, {\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>110.0, \"coordinates\"=>{\"chromosome\"=>\"21\", \"start\"=>36160098, \"stop\"=>36421641, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000300305.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/43/summary/variants/155/summary#variant\", \"evidence_items\"=>[{\"id\"=>429, \"name\"=>\"EID429\", \"description\"=>\"In 878 non-APL patients with AML, those who had mutations in RUNX1 (N=53) had higher rates of refractory disease than those without (30% versus 19%; P = 0.047).\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>241, \"name\"=>\"RUNX1 mutations in acute myeloid leukemia: results from a comprehensive genetic and clinical analysis from the AML study group.\", \"citation\"=>\"Gaidzik et al., 2011, J. Clin. Oncol.\", \"pubmed_id\"=>\"21343560\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21343560\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>4, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>155, \"phenotypes\"=>[]}, {\"id\"=>509, \"name\"=>\"EID509\", \"description\"=>\"In a study of patients with cytogenically normal acute myeloid leukemia, RUNX1 mutations were more frequent in patients 60 years old or older. Patients with RUNX1 mutations had shorter disease-free (younger, P = 0.058; older, P < 0.001), event-free (both, P < 0.001), and overall survival (younger, P = 0.003; older, P < 0.001) as well as lower complete remission rates (younger, P = 0.005; older, P = 0.006).\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>313, \"name\"=>\"RUNX1 mutations are associated with poor outcome in younger and older patients with cytogenetically normal acute myeloid leukemia and with distinct gene and MicroRNA expression signatures.\", \"citation\"=>\"Mendler et al., 2012, J. Clin. Oncol.\", \"pubmed_id\"=>\"22753902\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22753902\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3732007\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>9, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>155, \"phenotypes\"=>[]}, {\"id\"=>430, \"name\"=>\"EID430\", \"description\"=>\"In 878 non-APL patients with AML (18-60 years), those who had mutations in RUNX1 (N=53) had shorter event-free survival (8% vs 30%, P<0.0001) and relapse-free survival (26% vs 44%, P=0.022) relative to wild-type. EFS was also shorter for the cytogenetically normal (CN) subgroup with RUNX1 mutations (N=34; 10% vs 34%; P<0.0001) compared to wild-type (N=499), but only trending for relapse-free survival (29% vs 43%; P=0.21). For multivariate analysis, RUNX1 was a significant marker of inferior EFS in the entire cohort (P=0.011) and the intermediate-risk (P=0.005) subgroup but not the CN subgroup.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>241, \"name\"=>\"RUNX1 mutations in acute myeloid leukemia: results from a comprehensive genetic and clinical analysis from the AML study group.\", \"citation\"=>\"Gaidzik et al., 2011, J. Clin. Oncol.\", \"pubmed_id\"=>\"21343560\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21343560\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>4, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>155, \"phenotypes\"=>[]}, {\"id\"=>1558, \"name\"=>\"EID1558\", \"description\"=>\"In patients with acute myeloid leukemia, those with RUNX1 mutations (n=245) had significantly worse outcomes when compared to those with RUNX1 wild-type alleles (n=2,194).  The complete remission rate for patients with RUNX1 mutations was significantly lower (48.4% vs. 68.1%; p <0.0001), the refractory disease rate was significantly higher (40.6% 23.4%; p=0.03), the 5-year event free survival rate was significantly lower (9%, 24%; p<0.0001), the 5-year relapse free survival rate was significantly lower (22%, 36%; p=0.0007), and the 5-year overall survival rate was significantly lower (22%, 37%; p=0.0001).\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1029, \"name\"=>\"RUNX1 mutations in acute myeloid leukemia are associated with distinct clinico-pathologic and genetic features.\", \"citation\"=>\"Gaidzik et al., 2016, Leukemia\", \"pubmed_id\"=>\"27137476\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27137476\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>6, \"day\"=>10}, \"journal\"=>\"Leukemia\", \"full_journal_title\"=>\"Leukemia\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>155, \"phenotypes\"=>[]}, {\"id\"=>508, \"name\"=>\"EID508\", \"description\"=>\"In univariate analysis, mutations in RUNX1 were significantly associated with shorter overall survival compared to wildtype RUNX1 in adult (18-87 years old) T-cell acute lymphoblastic leukemia (N=9 mutated, 62 wild type patients; P=0.014) and the early subtype (N=7 mutated, 23 wild type patients; P=0.027). RUNX1 mutations were associated with other poor-risk parameters, early subtype and older age.\", \"disease\"=>{\"id\"=>43, \"name\"=>\"Acute Lymphocytic Leukemia\", \"display_name\"=>\"Acute Lymphocytic Leukemia\", \"doid\"=>\"9952\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9952\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>283, \"name\"=>\"The molecular profile of adult T-cell acute lymphoblastic leukemia: mutations in RUNX1 and DNMT3A are associated with poor prognosis in T-ALL.\", \"citation\"=>\"Grossmann et al., 2013, Genes Chromosomes Cancer\", \"pubmed_id\"=>\"23341344\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23341344\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>4}, \"journal\"=>\"Genes Chromosomes Cancer\", \"full_journal_title\"=>\"Genes, chromosomes & cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>155, \"phenotypes\"=>[]}, {\"id\"=>411, \"name\"=>\"EID411\", \"description\"=>\"In non-APL patients with AML (age 18-60), those who had mutations in RUNX1 had significantly shorter relapse-free survival following conventional post-remission treatment (repetitive cycles of high-dose cytarabine) than those treated with an allogeneic hematopoietic stem-cell transplantation (N=32) at first remission (4-year RFS; 0% vs 52%; P<0.0001).\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[{\"id\"=>173, \"name\"=>\"Cytarabine\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>241, \"name\"=>\"RUNX1 mutations in acute myeloid leukemia: results from a comprehensive genetic and clinical analysis from the AML study group.\", \"citation\"=>\"Gaidzik et al., 2011, J. Clin. Oncol.\", \"pubmed_id\"=>\"21343560\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21343560\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>4, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>155, \"phenotypes\"=>[]}]}, {\"id\"=>157, \"entrez_name\"=>\"TET2\", \"entrez_id\"=>54790, \"name\"=>\"MUTATION\", \"description\"=>\"TET2 mutations are commonly found in acute myeloid leukemia occurring in 9-23% of cases with many occurring in the elderly. Generally these mutations are loss-of-function which have been shown in mouse models to impair hematopoietic stem cell differentiation and cause aberrant hematopoiesis. These mutations have been found to be mutually exclusive with MLL and IDH1/2 mutations and have been associated with shortened event-free-survival and disease-free-survival when co-occurring with NPM1+/FLT3- genotypes. Overall Survival has been found to be shorter when occurring with NPM1+/FLT3+ genotypes. Studies involving \\\"CN AML\\\" and \\\"de novo AML\\\" however have been unable to elucidate a clinical impact.\", \"gene_id\"=>55, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>30, \"name\"=>\"gene_variant\", \"display_name\"=>\"Gene Variant\", \"so_id\"=>\"SO:0001564\", \"description\"=>\"A sequence variant where the structure of the gene is changed.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001564\"}, {\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>120.0, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>106067450, \"stop\"=>106200973, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000513237.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/55/summary/variants/157/summary#variant\", \"evidence_items\"=>[{\"id\"=>427, \"name\"=>\"EID427\", \"description\"=>\"In patients with cytogenetically normal acute myeloid leukemia, those in the ELN favorable-risk group with TET2 mutations had shorter event-free survival, lower rates of complete remission, and shorter disease-free survival.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>250, \"name\"=>\"TET2 mutations improve the new European LeukemiaNet risk classification of acute myeloid leukemia: a Cancer and Leukemia Group B study.\", \"citation\"=>\"Metzeler et al., 2011, J. Clin. Oncol.\", \"pubmed_id\"=>\"21343549\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21343549\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3084003\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>4, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>157, \"phenotypes\"=>[]}, {\"id\"=>426, \"name\"=>\"EID426\", \"description\"=>\"In patients classified as intermediate-risk via cytogenetics, TET2 mutations have been shown to be correlated with poor prognosis.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>253, \"name\"=>\"TET2 mutation is an unfavorable prognostic factor in acute myeloid leukemia patients with intermediate-risk cytogenetics.\", \"citation\"=>\"Chou et al., 2011, Blood\", \"pubmed_id\"=>\"21828143\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21828143\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>10, \"day\"=>6}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>157, \"phenotypes\"=>[]}, {\"id\"=>421, \"name\"=>\"EID421\", \"description\"=>\"In patients with cytogenetically normal acute myeloid leukemia, those in the intermediate-I risk group with TET2 mutations did not have significantly different rates of event-free survival, complete remission, disease-free survival, or overall survival.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"N/A\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>250, \"name\"=>\"TET2 mutations improve the new European LeukemiaNet risk classification of acute myeloid leukemia: a Cancer and Leukemia Group B study.\", \"citation\"=>\"Metzeler et al., 2011, J. Clin. Oncol.\", \"pubmed_id\"=>\"21343549\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21343549\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3084003\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>4, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>157, \"phenotypes\"=>[]}, {\"id\"=>419, \"name\"=>\"EID419\", \"description\"=>\"In patients with cytogenetically normal acute myeloid leukemia, those in the intermediate-I risk group with TET2 mutations had higher rates of complete remission.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>245, \"name\"=>\"TET2 mutations in acute myeloid leukemia (AML): results from a comprehensive genetic and clinical analysis of the AML study group.\", \"citation\"=>\"Gaidzik et al., 2012, J. Clin. Oncol.\", \"pubmed_id\"=>\"22430270\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22430270\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>4, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>157, \"phenotypes\"=>[]}, {\"id\"=>417, \"name\"=>\"EID417\", \"description\"=>\"In patients with myelodysplastic syndromes, those with TET2 mutations have been shown to have better overall survival than patients without TET2 mutations.\", \"disease\"=>{\"id\"=>34, \"name\"=>\"Myelodysplastic Syndrome\", \"display_name\"=>\"Myelodysplastic Syndrome\", \"doid\"=>\"0050908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050908\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>247, \"name\"=>\"TET2 mutation is an independent favorable prognostic factor in myelodysplastic syndromes (MDSs).\", \"citation\"=>\"Kosmider et al., 2009, Blood\", \"pubmed_id\"=>\"19666869\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19666869\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>10, \"day\"=>8}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>157, \"phenotypes\"=>[]}, {\"id\"=>415, \"name\"=>\"EID415\", \"description\"=>\"In a cohort of 783 young adult patients with acute myeloid leukemia, those with TET2 mutations did not have significantly different rates of complete recovery, refractory disease, or hypoplastic death.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"N/A\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>245, \"name\"=>\"TET2 mutations in acute myeloid leukemia (AML): results from a comprehensive genetic and clinical analysis of the AML study group.\", \"citation\"=>\"Gaidzik et al., 2012, J. Clin. Oncol.\", \"pubmed_id\"=>\"22430270\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22430270\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>4, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>157, \"phenotypes\"=>[]}]}, {\"id\"=>159, \"entrez_name\"=>\"VHL\", \"entrez_id\"=>7428, \"name\"=>\"LOSS-OF-FUNCTION\", \"description\"=>\"\", \"gene_id\"=>58, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>10182692, \"stop\"=>10193904, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000256474.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/58/summary/variants/159/summary#variant\", \"evidence_items\"=>[{\"id\"=>4831, \"name\"=>\"EID4831\", \"description\"=>\"A meta-analysis to evaluate the association between the VHL alteration and clinical outcomes in patients with RCC. There were six studies including a total of 633 patients: 244 patients who received anti-VEGF therapy in the predictive value analysis and 419 in the prognostic value analysis. 410 (61.8%) had VHL alteration. The meta-analysis showed no association between the VHL gene alteration and ORR or PFS, as well as no correlation to OS.\\nVEGFR inhibition included: Sunitinib, Sorafenib, Axitinib, Pazopanib, Bevacizumab.\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[{\"id\"=>521, \"name\"=>\"VEGF Inhibition\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2139, \"name\"=>\"Prognostic and predictive value of VHL gene alteration in renal cell carcinoma: a meta-analysis and review.\", \"citation\"=>\"Kim et al., 2017, Oncotarget\", \"pubmed_id\"=>\"28103578\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28103578\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5355155\", \"publication_date\"=>{\"year\"=>2017, \"month\"=>2, \"day\"=>21}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>159, \"phenotypes\"=>[]}, {\"id\"=>4829, \"name\"=>\"EID4829\", \"description\"=>\"VHL mutations were evaluated in 78 tumors samples of patients with metastatic RCC treated with pazopanib as part of a clinical trial. Overall, 87% had one VHL mutation, 12% showed promoter hypermethylation, and 6 patients had both. 8 patients exhibited neither mutation nor methylation of VHL. There was no correlation between VHL status and overall response rate or progression free survival under pazopanib therapy.\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[{\"id\"=>56, \"name\"=>\"Pazopanib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>nil, \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1132, \"name\"=>\"The role of aberrant VHL/HIF pathway elements in predicting clinical outcome to pazopanib therapy in patients with metastatic clear-cell renal cell carcinoma.\", \"citation\"=>\"Choueiri et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"23881929\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23881929\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4522695\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>9, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>159, \"phenotypes\"=>[]}, {\"id\"=>4828, \"name\"=>\"EID4828\", \"description\"=>\"Inhibitors of the kinase mammalian target of rapamycin (mTOR) have shown sporadic activity in cancer trials, leading to confusion about the appropriate clinical setting for their use. Here we show that loss of the Von Hippel-Lindau tumor suppressor gene (VHL) sensitizes kidney cancer cells to the mTOR inhibitor CCI-779 in vitro and in mouse models. Growth arrest caused by CCI-779 correlates with a block in translation of mRNA encoding hypoxia-inducible factor (HIF1A).\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[{\"id\"=>83, \"name\"=>\"Temsirolimus\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>719, \"name\"=>\"Hypoxia-inducible factor determines sensitivity to inhibitors of mTOR in kidney cancer.\", \"citation\"=>\"Thomas et al., 2006, Nat. Med.\", \"pubmed_id\"=>\"16341243\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16341243\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2006, \"month\"=>1}, \"journal\"=>\"Nat. Med.\", \"full_journal_title\"=>\"Nature medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>159, \"phenotypes\"=>[]}, {\"id\"=>428, \"name\"=>\"EID428\", \"description\"=>\"In patients with renal clear cell carcinoma, patients with loss of function VHL mutations had shorter tumor-specific survival.\", \"disease\"=>{\"id\"=>48, \"name\"=>\"Renal Clear Cell Carcinoma\", \"display_name\"=>\"Renal Clear Cell Carcinoma\", \"doid\"=>\"4467\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4467\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>254, \"name\"=>\"VHL mutations and their correlation with tumour cell proliferation, microvessel density, and patient prognosis in clear cell renal cell carcinoma.\", \"citation\"=>\"Schraml et al., 2002, J. Pathol.\", \"pubmed_id\"=>\"11793370\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/11793370\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2002, \"month\"=>2}, \"journal\"=>\"J. Pathol.\", \"full_journal_title\"=>\"The Journal of pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>159, \"phenotypes\"=>[]}]}, {\"id\"=>160, \"entrez_name\"=>\"VHL\", \"entrez_id\"=>7428, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>58, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>30, \"name\"=>\"gene_variant\", \"display_name\"=>\"Gene Variant\", \"so_id\"=>\"SO:0001564\", \"description\"=>\"A sequence variant where the structure of the gene is changed.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001564\"}, {\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>45.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>10182692, \"stop\"=>10193904, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000256474.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/58/summary/variants/160/summary#variant\", \"evidence_items\"=>[{\"id\"=>5323, \"name\"=>\"EID5323\", \"description\"=>\"A randomized, open-label phase 2 study, encompassing 85 patients, to compare the efficacy of dual PIK3/mTOR Inhibitor Apitolisib and mTORC1 inhibitor Everolimus in patients who progressed on or after antiangiogenetic therapy (VEGFR). There was a trend towards association between deleterious VHL alterations and progression-free-survival in the Everolimus arm only.\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[{\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2227, \"name\"=>\"Randomized Open-Label Phase II Trial of Apitolisib (GDC-0980), a Novel Inhibitor of the PI3K/Mammalian Target of Rapamycin Pathway, Versus Everolimus in Patients With Metastatic Renal Cell Carcinoma.\", \"citation\"=>\"Powles et al., 2016, J. Clin. Oncol.\", \"pubmed_id\"=>\"26951309\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26951309\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>5, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>160, \"phenotypes\"=>[]}, {\"id\"=>4832, \"name\"=>\"EID4832\", \"description\"=>\"A meta-analysis to evaluate the association between the VHL alteration and clinical outcomes in patients with RCC. There were six studies including a total of 633 patients: 244 patients who received anti-VEGF therapy in the predictive value analysis and 419 in the prognostic value analysis. 410 (61.8%) had VHL alteration. The meta-analysis showed no association between the VHL gene alteration and ORR or PFS, as well as no correlation to OS. VEGFR inhibition included: Sunitinib, Sorafenib, Axitinib, Pazopanib, Bevacizumab.\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[{\"id\"=>521, \"name\"=>\"VEGF Inhibition\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2139, \"name\"=>\"Prognostic and predictive value of VHL gene alteration in renal cell carcinoma: a meta-analysis and review.\", \"citation\"=>\"Kim et al., 2017, Oncotarget\", \"pubmed_id\"=>\"28103578\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28103578\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5355155\", \"publication_date\"=>{\"year\"=>2017, \"month\"=>2, \"day\"=>21}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>160, \"phenotypes\"=>[]}, {\"id\"=>1672, \"name\"=>\"EID1672\", \"description\"=>\"VHL mutations were evaluated in 78 tumors samples of patients with metastatic RCC treated with pazopanib as part of a clinical trial. Overall, 87% had one VHL mutation, 12% showed promoter hypermethylation, and 6 patients had both. 8 patients exhibited neither mutation nor methylation of VHL. There was no correlation between VHL status and overall response rate or progression free survival under pazopanib therapy.\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[{\"id\"=>56, \"name\"=>\"Pazopanib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1132, \"name\"=>\"The role of aberrant VHL/HIF pathway elements in predicting clinical outcome to pazopanib therapy in patients with metastatic clear-cell renal cell carcinoma.\", \"citation\"=>\"Choueiri et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"23881929\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23881929\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4522695\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>9, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>160, \"phenotypes\"=>[]}, {\"id\"=>422, \"name\"=>\"EID422\", \"description\"=>\"In renal clear cell carcinoma, patients with VHL mutations did not have a significant impact on cancer specific survival.\", \"disease\"=>{\"id\"=>48, \"name\"=>\"Renal Clear Cell Carcinoma\", \"display_name\"=>\"Renal Clear Cell Carcinoma\", \"doid\"=>\"4467\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4467\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"N/A\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>249, \"name\"=>\"Adverse outcomes in clear cell renal cell carcinoma with mutations of 3p21 epigenetic regulators BAP1 and SETD2: a report by MSKCC and the KIRC TCGA research network.\", \"citation\"=>\"Hakimi et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"23620406\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23620406\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3708609\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>6, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>160, \"phenotypes\"=>[]}, {\"id\"=>418, \"name\"=>\"EID418\", \"description\"=>\"In patients with renal clear cell carcinoma, patients with VHL mutations had longer cancer-specific survival.\", \"disease\"=>{\"id\"=>48, \"name\"=>\"Renal Clear Cell Carcinoma\", \"display_name\"=>\"Renal Clear Cell Carcinoma\", \"doid\"=>\"4467\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4467\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>248, \"name\"=>\"Low CAIX expression and absence of VHL gene mutation are associated with tumor aggressiveness and poor survival of clear cell renal cell carcinoma.\", \"citation\"=>\"Patard et al., 2008, Int. J. Cancer\", \"pubmed_id\"=>\"18464292\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18464292\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2721857\", \"publication_date\"=>{\"year\"=>2008, \"month\"=>7, \"day\"=>15}, \"journal\"=>\"Int. J. Cancer\", \"full_journal_title\"=>\"International journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>160, \"phenotypes\"=>[]}]}, {\"id\"=>167, \"entrez_name\"=>\"KMT2D\", \"entrez_id\"=>8085, \"name\"=>\"LOSS-OF-FUNCTION\", \"description\"=>\"\", \"gene_id\"=>64, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"12\", \"start\"=>49412758, \"stop\"=>49449107, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000301067.7\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/64/summary/variants/167/summary#variant\", \"evidence_items\"=>[{\"id\"=>435, \"name\"=>\"EID435\", \"description\"=>\"MLL2 was found to be the most frequently mutated gene in a sample of 31/35 follicular lymphoma patients and was not mutated in normal centroblast samples. Furthermore, MLL2 mutations were as frequently observed as t(14;18) Igh-BCL2 translocations, which are the most common clinically significant genetic event observed in follicular lymphoma. Mutations were predicted to be loss-of-function.\", \"disease\"=>{\"id\"=>51, \"name\"=>\"Follicular Lymphoma\", \"display_name\"=>\"Follicular Lymphoma\", \"doid\"=>\"0050873\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050873\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>258, \"name\"=>\"Frequent mutation of histone-modifying genes in non-Hodgkin lymphoma.\", \"citation\"=>\"Morin et al., 2011, Nature\", \"pubmed_id\"=>\"21796119\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21796119\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3210554\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>8, \"day\"=>18}, \"journal\"=>\"Nature\", \"full_journal_title\"=>\"Nature\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>167, \"phenotypes\"=>[]}]}, {\"id\"=>183, \"entrez_name\"=>\"BAP1\", \"entrez_id\"=>8314, \"name\"=>\"MUTATION\", \"description\"=>\"BAP1 is a tumor suppressor gene, mutations in this gene have been observed in a number of cancers including melanoma, and renal cancers. These mutations generally lead to loss-of-function and mutations in the UHC domain of BAP1 have been shown to lead to structural instability and β-amyloid aggregation in models. Germline mutations are associated with an increased risk of tumorgenesis and can lead to the development of \\\"atypical Spitz tumours (ASTs), cutaneous melanoma (CM), epithelioid malignant mesothelioma (MM) and clear cell RCC\\\".\", \"gene_id\"=>70, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}, {\"id\"=>103, \"name\"=>\"protein_altering_variant\", \"display_name\"=>\"Protein Altering Variant\", \"so_id\"=>\"SO:0001818\", \"description\"=>\"A sequence_variant which is predicted to change the protein encoded in the coding sequence.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001818\"}], \"civic_actionability_score\"=>73.5, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>52435029, \"stop\"=>52444366, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000460680.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/70/summary/variants/183/summary#variant\", \"evidence_items\"=>[{\"id\"=>5339, \"name\"=>\"EID5339\", \"description\"=>\"This study evaluated the effects of everolimus and sunitinib on patients with metastatic renal cell carcinoma. A total of 220 patients with RCC were evaluated for 341 genes at 540x coverage. Of all 220 patients, 19% of had BAP1 mutations. Patients with BAP1 mutations treated with first line everolimus followed by sunitinib (N=87 wildtype, 22 mutant) were associated with shorter progression free survival (median [95% CI] 4.9 [2.9 - 8.1] vs 10.5 [7.3 - 12.9] mo; HR 1.84 [1.1-3.2]; P=0.02). For patients treated with sunitinib followed by everolimus (N=91 wildtype, 20 mutant) were associated with a trend towards shorter progression free survival (median [95% CI] 8.1 [3.1-11.3] vs 11.0 [8.3-13.8] mo; HR 1.69 [0.9-3.2]; P=0.06).\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[{\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}, {\"id\"=>157, \"name\"=>\"Sunitinib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Sequential\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2234, \"name\"=>\"Genomic Biomarkers of a Randomized Trial Comparing First-line Everolimus and Sunitinib in Patients with Metastatic Renal Cell Carcinoma.\", \"citation\"=>\"Hsieh et al., 2017, Eur. Urol.\", \"pubmed_id\"=>\"27751729\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27751729\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5431298\", \"publication_date\"=>{\"year\"=>2017, \"month\"=>3}, \"journal\"=>\"Eur. Urol.\", \"full_journal_title\"=>\"European urology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>183, \"phenotypes\"=>[]}, {\"id\"=>1235, \"name\"=>\"EID1235\", \"description\"=>\"A siRNA screen identified BAP1 associated with HDAC1 and HDAC2 expression. HDAC2 or BAP1 depletion sensitized a mesothelioma cell line (MSTO-211H) to HDAC inhibitors MC1568, vorinostat or mocetinostat. The paper indicates however, that temporary BAP knockdown might not adequately reflect long-term BAP1 mutation.\", \"disease\"=>{\"id\"=>112, \"name\"=>\"Malignant Mesothelioma\", \"display_name\"=>\"Malignant Mesothelioma\", \"doid\"=>\"1790\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1790\"}, \"drugs\"=>[{\"id\"=>314, \"name\"=>\"Vorinostat (SAHA)\", \"pubchem_id\"=>\"\"}, {\"id\"=>315, \"name\"=>\"Mocetinostat\", \"pubchem_id\"=>nil}, {\"id\"=>316, \"name\"=>\"MC1568\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"E\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>854, \"name\"=>\"Loss of the deubiquitylase BAP1 alters class I histone deacetylase expression and sensitivity of mesothelioma cells to HDAC inhibitors.\", \"citation\"=>\"Sacco et al., 2015, Oncotarget\", \"pubmed_id\"=>\"25970771\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25970771\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4537048\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>5, \"day\"=>30}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>183, \"phenotypes\"=>[]}, {\"id\"=>1234, \"name\"=>\"EID1234\", \"description\"=>\"Somatic and germline BAP1 mutations are associated with uveal melanoma. In this preclinical study, gene set enrichment analysis revealed HDAC inhibitors as potential drugs in BAP1 mutant uveal melanomas. HDAC inhibitors VPA, TSA, LBH-589, and SAHA (vorinostat) were tested in-vitro and led to decreased proliferation (all inhibitors) decreased viability and cytotoxicity (TSA, SAHA, LBH-589) in-vitro and in-vivo (VPA). RNAi mediated BAP1 knockdown increased sensitivity to HDAC inhibition (VPA).\", \"disease\"=>{\"id\"=>188, \"name\"=>\"Uveal Melanoma\", \"display_name\"=>\"Uveal Melanoma\", \"doid\"=>\"6039\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:6039\"}, \"drugs\"=>[{\"id\"=>37, \"name\"=>\"Valproic Acid\", \"pubchem_id\"=>nil}, {\"id\"=>311, \"name\"=>\"Trichostatin A (TSA)\", \"pubchem_id\"=>\"\"}, {\"id\"=>312, \"name\"=>\"LBH-589\", \"pubchem_id\"=>nil}, {\"id\"=>314, \"name\"=>\"Vorinostat (SAHA)\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>335, \"name\"=>\"Histone deacetylase inhibitors induce growth arrest and differentiation in uveal melanoma.\", \"citation\"=>\"Landreville et al., 2012, Clin. Cancer Res.\", \"pubmed_id\"=>\"22038994\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22038994\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3261307\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>1, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>183, \"phenotypes\"=>[]}, {\"id\"=>459, \"name\"=>\"EID459\", \"description\"=>\"In patients with clear cell renal cell carcinoma, in a multivariate analysis those with mutations in BAP1 had worse overall survival than wild-type.\", \"disease\"=>{\"id\"=>48, \"name\"=>\"Renal Clear Cell Carcinoma\", \"display_name\"=>\"Renal Clear Cell Carcinoma\", \"doid\"=>\"4467\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4467\"}, \"drugs\"=>[], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>275, \"name\"=>\"Integrated molecular analysis of clear-cell renal cell carcinoma.\", \"citation\"=>\"Sato et al., 2013, Nat. Genet.\", \"pubmed_id\"=>\"23797736\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23797736\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>8}, \"journal\"=>\"Nat. Genet.\", \"full_journal_title\"=>\"Nature genetics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>183, \"phenotypes\"=>[]}, {\"id\"=>461, \"name\"=>\"EID461\", \"description\"=>\"In patients with clear cell renal cell carcinoma, those with nonsilent mutations of BAP1 had worse overall survival than wild-type.\", \"disease\"=>{\"id\"=>48, \"name\"=>\"Renal Clear Cell Carcinoma\", \"display_name\"=>\"Renal Clear Cell Carcinoma\", \"doid\"=>\"4467\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4467\"}, \"drugs\"=>[], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>276, \"name\"=>\"Comprehensive molecular characterization of clear cell renal cell carcinoma.\", \"citation\"=>\"2013, Nature\", \"pubmed_id\"=>\"23792563\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23792563\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3771322\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>7, \"day\"=>4}, \"journal\"=>\"Nature\", \"full_journal_title\"=>\"Nature\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>183, \"phenotypes\"=>[]}]}, {\"id\"=>185, \"entrez_name\"=>\"BRCA1\", \"entrez_id\"=>672, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>6, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>30, \"name\"=>\"gene_variant\", \"display_name\"=>\"Gene Variant\", \"so_id\"=>\"SO:0001564\", \"description\"=>\"A sequence variant where the structure of the gene is changed.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001564\"}, {\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>180.0, \"coordinates\"=>{\"chromosome\"=>\"17\", \"start\"=>41196312, \"stop\"=>41277387, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000357654.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/6/summary/variants/185/summary#variant\", \"evidence_items\"=>[{\"id\"=>5915, \"name\"=>\"EID5915\", \"description\"=>\"Patients with BRCA1/2 associated pancreatic adenocarcinoma were evaluated retrospectively.\\nIn patients with stage 3 and 4, the median overall survival (mOS) was 22 months for the platinum exposed (n=22), compared with 9 months for the non-platinum (n=21) treatment groups (p<0.039)\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>103, \"name\"=>\"PLATINUM\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2389, \"name\"=>\"Overall survival and clinical characteristics of pancreatic cancer in BRCA mutation carriers.\", \"citation\"=>\"Golan et al., 2014, Br. J. Cancer\", \"pubmed_id\"=>\"25072261\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25072261\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4453851\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>9, \"day\"=>9}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>185, \"phenotypes\"=>[]}, {\"id\"=>5914, \"name\"=>\"EID5914\", \"description\"=>\"Twenty-three patients with pancreatic cancer havoring BRCA1/2 germline mutations received olaparib. Eighteen (78%) patients had a BRCA2 mutation. All but one had received gemcitabine, and 65% had received prior platinum. Tumor response rate was 21.7% and stable disease more than 8 weeks was 35%.\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2388, \"name\"=>\"Olaparib monotherapy in patients with advanced cancer and a germline BRCA1/2 mutation.\", \"citation\"=>\"Kaufman et al., 2015, J. Clin. Oncol.\", \"pubmed_id\"=>\"25366685\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25366685\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>185, \"phenotypes\"=>[]}, {\"id\"=>5934, \"name\"=>\"EID5934\", \"description\"=>\"Veliparib 400mg bid was administered to patients with germline BRCA1/2 mutated pancreatic cancer. Sixteen pts was enrolled. All except two pts (12.5%) had previously been treated with platinum therapy. No confirmed PR was observed. One patient who had received prior single-agent gemcitabine only had an unconfirmed PR.\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>556, \"name\"=>\"Veliparib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2406, \"name\"=>\"Phase II trial of veliparib in patients with previously treated BRCA-mutated pancreas ductal adenocarcinoma.\", \"citation\"=>\"Lowery et al., 2018, Eur. J. Cancer\", \"pubmed_id\"=>\"29223478\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/29223478\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2018, \"month\"=>1}, \"journal\"=>\"Eur. J. Cancer\", \"full_journal_title\"=>\"European journal of cancer (Oxford, England : 1990)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[{\"nct_id\"=>\"NCT01585805\", \"name\"=>\"Gemcitabine Hydrochloride and Cisplatin With or Without Veliparib or Veliparib Alone in Treating Patients With Locally Advanced or Metastatic Pancreatic Cancer\", \"description\"=>\"This randomized phase II trial studies how well veliparib together with gemcitabine hydrochloride and cisplatin works compared to gemcitabine hydrochloride and cisplatin alone in treating patients with pancreatic cancer that has spread from where it started to nearby tissue or lymph nodes (locally advanced) or spread from the primary site (place where it started) to other places in the body (metastatic). Veliparib may stop the growth of tumor cells by blocking some of the enzymes needed for cell growth. Drugs used in chemotherapy, such as gemcitabine hydrochloride and cisplatin, work in different ways to stop the growth of tumor cells, either by killing the cells, by stopping them from dividing, or by stopping them from spreading. It is not yet known whether giving veliparib together with gemcitabine hydrochloride and cisplatin is an effective treatment for pancreatic cancer.\", \"clinical_trial_url\"=>\"https://clinicaltrials.gov/show/NCT01585805\"}, {\"nct_id\"=>\"NCT02184195\", \"name\"=>\"Olaparib in gBRCA Mutated Pancreatic Cancer Whose Disease Has Not Progressed on First Line Platinum-Based Chemotherapy\", \"description\"=>\"A Phase III, Randomised, Double Blind, Placebo Controlled, Multicentre Study of Maintenance Olaparib Monotherapy in Patients with gBRCA Mutated Metastatic Pancreatic Cancer whose Disease Has Not Progressed on First Line Platinum Based Chemotherapy\", \"clinical_trial_url\"=>\"https://clinicaltrials.gov/show/NCT02184195\"}]}, \"variant_id\"=>185, \"phenotypes\"=>[]}, {\"id\"=>5932, \"name\"=>\"EID5932\", \"description\"=>\"In Phase I study for pancreatic cancer patients\\nGEM 600 mg/m2 + CDDP 25 mg/m2 + Veliparib 20 - 80 mg twice daily showed response rate of  77.8% (7/9) and mOS of 23.3 months in germline BRCA mutation-positive patients with stage IV disease.\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>85, \"name\"=>\"Gemcitabine\", \"pubchem_id\"=>nil}, {\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}, {\"id\"=>556, \"name\"=>\"Veliparib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2405, \"name\"=>\"Phase 1 trial evaluating cisplatin, gemcitabine, and veliparib in 2 patient cohorts: Germline BRCA mutation carriers and wild-type BRCA pancreatic ductal adenocarcinoma.\", \"citation\"=>\"O'Reilly et al., 2018, Cancer\", \"pubmed_id\"=>\"29338080\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/29338080\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2018, \"month\"=>1, \"day\"=>16}, \"journal\"=>\"Cancer\", \"full_journal_title\"=>\"Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[{\"nct_id\"=>\"NCT01585805\", \"name\"=>\"Gemcitabine Hydrochloride and Cisplatin With or Without Veliparib or Veliparib Alone in Treating Patients With Locally Advanced or Metastatic Pancreatic Cancer\", \"description\"=>\"This randomized phase II trial studies how well veliparib together with gemcitabine hydrochloride and cisplatin works compared to gemcitabine hydrochloride and cisplatin alone in treating patients with pancreatic cancer that has spread from where it started to nearby tissue or lymph nodes (locally advanced) or spread from the primary site (place where it started) to other places in the body (metastatic). Veliparib may stop the growth of tumor cells by blocking some of the enzymes needed for cell growth. Drugs used in chemotherapy, such as gemcitabine hydrochloride and cisplatin, work in different ways to stop the growth of tumor cells, either by killing the cells, by stopping them from dividing, or by stopping them from spreading. It is not yet known whether giving veliparib together with gemcitabine hydrochloride and cisplatin is an effective treatment for pancreatic cancer.\", \"clinical_trial_url\"=>\"https://clinicaltrials.gov/show/NCT01585805\"}]}, \"variant_id\"=>185, \"phenotypes\"=>[]}, {\"id\"=>5830, \"name\"=>\"EID5830\", \"description\"=>\"Randomized, open-label, phase 3 trial in which olaparib monotherapy was compared with standard therapy in patients with a germline BRCA mutation HER2–negative metastatic breast cancer. 205 patients were assigned to receive olaparib and 97 were assigned to receive standard therapy. Median progression-free survival was significantly longer in the olaparib group than in the standard-therapy group (7.0 months vs. 4.2 months; hazard ratio 0.58; 95%CI 0.43 to 0.80; P<0.001). A response to treatment occurred in 100 of the 167 patients who had measurable disease in the olaparib group (59.9%; 95% CI, 52.0 to 67.4) and in 19 of the 66 patients in the standard-therapy group (28.8%; 95% CI, 18.3 to 41.3). A complete response was seen in 9.0% of the patients who had measurable disease in the olaparib group and in 1.5% in the standard-therapy group.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2357, \"name\"=>\"Olaparib for Metastatic Breast Cancer in Patients with a Germline BRCA Mutation.\", \"citation\"=>\"2017, N. Engl. J. Med.\", \"pubmed_id\"=>\"28792849\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28792849\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>8, \"day\"=>9}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>185, \"phenotypes\"=>[]}, {\"id\"=>1897, \"name\"=>\"EID1897\", \"description\"=>\"A phase 2 trial on patients with relapsed, platinum-sensitive high grade ovarian carcinoma (ARIEL2 Part1, NCT01891344) was held to assess PARP inhibitor Rucaparib on 3 genetically defined subsets of patients. Out of 40 BRCA mutant patients, 10 had somatic BRCA1 mutation. Confirmed objective response was seen in 79% of the BRCA1 mutant subgroup, and 74% confirmed objective response was seen in the somatic BRCA mutant subgroup. In a BRCA wild-type subgroup, with low levels of Loss of Heterozygosity, confirmed objective response was observed in 10% of the patients.\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>431, \"name\"=>\"Rucaparib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1328, \"name\"=>\"Rucaparib in relapsed, platinum-sensitive high-grade ovarian carcinoma (ARIEL2 Part 1): an international, multicentre, open-label, phase 2 trial.\", \"citation\"=>\"Swisher et al., 2017, Lancet Oncol.\", \"pubmed_id\"=>\"27908594\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27908594\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>1}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>185, \"phenotypes\"=>[]}, {\"id\"=>1684, \"name\"=>\"EID1684\", \"description\"=>\"In an evaluation of 77 patients, those who had BRCA1/2 germline mutations (n=11) had an increase response rate to cisplatin when compared to those without germline mutations (54.5% versus 19.7%; P=0.022) although progression free survival and overall survival was not significantly different.\", \"disease\"=>{\"id\"=>370, \"name\"=>\"Triple-receptor Negative Breast Cancer\", \"display_name\"=>\"Triple-receptor Negative Breast Cancer\", \"doid\"=>\"0060081\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060081\"}, \"drugs\"=>[{\"id\"=>93, \"name\"=>\"Carboplatin\", \"pubchem_id\"=>nil}, {\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1137, \"name\"=>\"TBCRC009: A Multicenter Phase II Clinical Trial of Platinum Monotherapy With Biomarker Assessment in Metastatic Triple-Negative Breast Cancer.\", \"citation\"=>\"Isakoff et al., 2015, J. Clin. Oncol.\", \"pubmed_id\"=>\"25847936\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25847936\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4451173\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>6, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>185, \"phenotypes\"=>[]}, {\"id\"=>1675, \"name\"=>\"EID1675\", \"description\"=>\"664 patients with non-bulky FIGO Stage II-IV ovarian cancer were evaluated for PFS when treated with Pazopanib versus placebo.  Of the 664 patients, 10% (n=68) were BRCA1+, 4% were BRCA 2+ (n=29) and 85% (n=567) were BCRA1/2 negative.  Longer progression free survival was observed in BRCA1/2 positive carriers versus BRCA1/2 negative carriers in both the Pazopanib arm and the placebo arm, however, due to small sample size, the PFS increase in the Pazopanib arm was not statistically significant.\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1133, \"name\"=>\"BRCA1/2 mutations associated with progression-free survival in ovarian cancer patients in the AGO-OVAR 16 study.\", \"citation\"=>\"Harter et al., 2016, Gynecol. Oncol.\", \"pubmed_id\"=>\"26740259\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26740259\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>3}, \"journal\"=>\"Gynecol. Oncol.\", \"full_journal_title\"=>\"Gynecologic oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>185, \"phenotypes\"=>[]}, {\"id\"=>1677, \"name\"=>\"EID1677\", \"description\"=>\"46 women with platinum-sensitive, high-grade, BRCA1/2+ ovarian cancer received either olaparib monotherapy or olaparib/cediranib combination therapy.  Median PFS was increased in combination therapy (17.7 months versus 9.0 months; HR=0.42; 95%CI 0.23-0.76; p=0.005); overall response rate was increased in the combination therapy (79.6% versus 47.8%; p=0.002); and total deaths were reduced in combination therapy (6 patients versus 10 patients).\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}, {\"id\"=>400, \"name\"=>\"Cediranib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1134, \"name\"=>\"Combination cediranib and olaparib versus olaparib alone for women with recurrent platinum-sensitive ovarian cancer: a randomised phase 2 study.\", \"citation\"=>\"Liu et al., 2014, Lancet Oncol.\", \"pubmed_id\"=>\"25218906\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25218906\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4294183\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>10}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>185, \"phenotypes\"=>[]}, {\"id\"=>1529, \"name\"=>\"EID1529\", \"description\"=>\"In a study of ovarian carcinomas, 81% of tumors with germline BRCA1/2 mutations were platinum sensitive, whereas only 60% of tumors with non-BRCA1/2 mutations were platinum sensitive (N=47; N=158; P=0.01). Sensitivity was defined by maintenance of complete response greater than 6 months post platinum-based therapy.\", \"disease\"=>{\"id\"=>47, \"name\"=>\"Ovarian Carcinoma\", \"display_name\"=>\"Ovarian Carcinoma\", \"doid\"=>\"4001\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4001\"}, \"drugs\"=>[{\"id\"=>103, \"name\"=>\"PLATINUM\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1020, \"name\"=>\"Germline and somatic mutations in homologous recombination genes predict platinum response and survival in ovarian, fallopian tube, and peritoneal carcinomas.\", \"citation\"=>\"Pennington et al., 2014, Clin. Cancer Res.\", \"pubmed_id\"=>\"24240112\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24240112\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3944197\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>2, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>185, \"phenotypes\"=>[]}, {\"id\"=>1531, \"name\"=>\"EID1531\", \"description\"=>\"In a study of ovarian carcinomas, 87% of tumors with somatic BRCA1/2 mutations were platinum sensitive, whereas only 60% of tumors with non-BRCA1/2 mutations were platinum sensitive (N=38; N=158; P=0.002). Sensitivity to platinum was defined by maintenance of complete response greater than 6 months post platinum-based therapy.\", \"disease\"=>{\"id\"=>47, \"name\"=>\"Ovarian Carcinoma\", \"display_name\"=>\"Ovarian Carcinoma\", \"doid\"=>\"4001\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4001\"}, \"drugs\"=>[{\"id\"=>103, \"name\"=>\"PLATINUM\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1020, \"name\"=>\"Germline and somatic mutations in homologous recombination genes predict platinum response and survival in ovarian, fallopian tube, and peritoneal carcinomas.\", \"citation\"=>\"Pennington et al., 2014, Clin. Cancer Res.\", \"pubmed_id\"=>\"24240112\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24240112\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3944197\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>2, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>185, \"phenotypes\"=>[]}, {\"id\"=>1772, \"name\"=>\"EID1772\", \"description\"=>\"In a study of 63 patients with ovarian cancer, 17 patients had BRCA1 or BRCA2 mutations and 46 patients had no BRCA mutations.  Of the BRCA-positive patients, 7/17(41%; 95%CI 22-64) showed objective response to olaparib whereas only 11/46 BRCA-negative patients (24%; 95%CI 14-38) showed objective response to olaparib.\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1239, \"name\"=>\"Olaparib in patients with recurrent high-grade serous or poorly differentiated ovarian carcinoma or triple-negative breast cancer: a phase 2, multicentre, open-label, non-randomised study.\", \"citation\"=>\"Gelmon et al., 2011, Lancet Oncol.\", \"pubmed_id\"=>\"21862407\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21862407\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>9}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>185, \"phenotypes\"=>[]}, {\"id\"=>1775, \"name\"=>\"EID1775\", \"description\"=>\"In a study of 26 patients with breast cancer, 10 patients were BRCA-positive and 16 patients were BRCA-negative.  Regardless of BRCA status, none showed objective response when treated with olaparib.\", \"disease\"=>{\"id\"=>370, \"name\"=>\"Triple-receptor Negative Breast Cancer\", \"display_name\"=>\"Triple-receptor Negative Breast Cancer\", \"doid\"=>\"0060081\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060081\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1239, \"name\"=>\"Olaparib in patients with recurrent high-grade serous or poorly differentiated ovarian carcinoma or triple-negative breast cancer: a phase 2, multicentre, open-label, non-randomised study.\", \"citation\"=>\"Gelmon et al., 2011, Lancet Oncol.\", \"pubmed_id\"=>\"21862407\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21862407\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>9}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>185, \"phenotypes\"=>[]}, {\"id\"=>1370, \"name\"=>\"EID1370\", \"description\"=>\"In this phase 1 trial, 60 patients with advanced solid tumors (ovarian, breast, colorectal, melanoma, sarcoma, prostate) were treated with PARP inhibitor olaparib. Durable antitumor activity was only observed in BRCA1/2 mutation carriers. 19 BRCA carriers had ovarian, breast, or prostate cancers; 12 of these (63%) had a clinical benefit from treatment with olaparib: radiologic or tumor-marker responses or meaningful disease stabilization (stable disease for a period of 4 months or more). Nine BRCA carriers had a response according to RECIST, with the response sustained for more than 76 weeks in one patient. No responses were seen in patients without BRCA mutation.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>359, \"name\"=>\"Inhibition of poly(ADP-ribose) polymerase in tumors from BRCA mutation carriers.\", \"citation\"=>\"Fong et al., 2009, N. Engl. J. Med.\", \"pubmed_id\"=>\"19553641\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19553641\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>7, \"day\"=>9}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>185, \"phenotypes\"=>[]}, {\"id\"=>464, \"name\"=>\"EID464\", \"description\"=>\"In patients with invasive epithelial ovarian cancer, those with mutations in BRCA1 had better five-year overall survival.\", \"disease\"=>{\"id\"=>225, \"name\"=>\"Epithelial Ovarian Cancer\", \"display_name\"=>\"Epithelial Ovarian Cancer\", \"doid\"=>\"2152\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2152\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>279, \"name\"=>\"Association between BRCA1 and BRCA2 mutations and survival in women with invasive epithelial ovarian cancer.\", \"citation\"=>\"Bolton et al., 2012, JAMA\", \"pubmed_id\"=>\"22274685\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22274685\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3727895\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>1, \"day\"=>25}, \"journal\"=>\"JAMA\", \"full_journal_title\"=>\"JAMA\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>185, \"phenotypes\"=>[]}]}, {\"id\"=>186, \"entrez_name\"=>\"BRCA2\", \"entrez_id\"=>675, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>7, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>30, \"name\"=>\"gene_variant\", \"display_name\"=>\"Gene Variant\", \"so_id\"=>\"SO:0001564\", \"description\"=>\"A sequence variant where the structure of the gene is changed.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001564\"}, {\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>160.0, \"coordinates\"=>{\"chromosome\"=>\"13\", \"start\"=>32889611, \"stop\"=>32973347, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000380152.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/7/summary/variants/186/summary#variant\", \"evidence_items\"=>[{\"id\"=>5916, \"name\"=>\"EID5916\", \"description\"=>\"Patients with BRCA1/2 associated pancreatic adenocarcinoma were evaluated retrospectively.\\nIn patients with stage 3 and 4, the median overall survival (mOS) was 22 months for the platinum exposed (n=22), compared with 9 months for the non-platinum (n=21) treatment groups (p<0.039)\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>103, \"name\"=>\"PLATINUM\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2389, \"name\"=>\"Overall survival and clinical characteristics of pancreatic cancer in BRCA mutation carriers.\", \"citation\"=>\"Golan et al., 2014, Br. J. Cancer\", \"pubmed_id\"=>\"25072261\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25072261\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4453851\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>9, \"day\"=>9}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>186, \"phenotypes\"=>[]}, {\"id\"=>5913, \"name\"=>\"EID5913\", \"description\"=>\"Twenty-three patients with pancreatic cancer havoring BRCA1/2 germline mutations received olaparib. Eighteen (78%) patients had a BRCA2 mutation. All but one had received gemcitabine, and 65% had received prior platinum. Tumor response rate was 21.7% and stable disease more than 8 weeks was 35%.\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2388, \"name\"=>\"Olaparib monotherapy in patients with advanced cancer and a germline BRCA1/2 mutation.\", \"citation\"=>\"Kaufman et al., 2015, J. Clin. Oncol.\", \"pubmed_id\"=>\"25366685\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25366685\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>1, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>186, \"phenotypes\"=>[]}, {\"id\"=>5936, \"name\"=>\"EID5936\", \"description\"=>\"Patient with germline BRCA2 mutation underwent resection of pancreatic cancer.\\nAround three years later, tumor recurrence was detected by MRI and PET-CT confirmed it. She was treated with gemcitabine plus iniparib in clinical trial. Excellent response to therapy was observed and surgery was performed but no residual tumor tissue was detected\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>557, \"name\"=>\"Iniparib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2407, \"name\"=>\"Evidence for the efficacy of Iniparib, a PARP-1 inhibitor, in BRCA2-associated pancreatic cancer.\", \"citation\"=>\"Fogelman et al., 2011, Anticancer Res.\", \"pubmed_id\"=>\"21508395\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21508395\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>4}, \"journal\"=>\"Anticancer Res.\", \"full_journal_title\"=>\"Anticancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>186, \"phenotypes\"=>[]}, {\"id\"=>5935, \"name\"=>\"EID5935\", \"description\"=>\"Veliparib 400mg bid was administered to patients with germline BRCA1/2 mutated pancreatic cancer. Sixteen pts was enrolled. All except two pts (12.5%) had previously been treated with platinum therapy. No confirmed PR was observed. One patient who had received prior single-agent gemcitabine only had an unconfirmed PR.\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>556, \"name\"=>\"Veliparib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2406, \"name\"=>\"Phase II trial of veliparib in patients with previously treated BRCA-mutated pancreas ductal adenocarcinoma.\", \"citation\"=>\"Lowery et al., 2018, Eur. J. Cancer\", \"pubmed_id\"=>\"29223478\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/29223478\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2018, \"month\"=>1}, \"journal\"=>\"Eur. J. Cancer\", \"full_journal_title\"=>\"European journal of cancer (Oxford, England : 1990)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[{\"nct_id\"=>\"NCT01585805\", \"name\"=>\"Gemcitabine Hydrochloride and Cisplatin With or Without Veliparib or Veliparib Alone in Treating Patients With Locally Advanced or Metastatic Pancreatic Cancer\", \"description\"=>\"This randomized phase II trial studies how well veliparib together with gemcitabine hydrochloride and cisplatin works compared to gemcitabine hydrochloride and cisplatin alone in treating patients with pancreatic cancer that has spread from where it started to nearby tissue or lymph nodes (locally advanced) or spread from the primary site (place where it started) to other places in the body (metastatic). Veliparib may stop the growth of tumor cells by blocking some of the enzymes needed for cell growth. Drugs used in chemotherapy, such as gemcitabine hydrochloride and cisplatin, work in different ways to stop the growth of tumor cells, either by killing the cells, by stopping them from dividing, or by stopping them from spreading. It is not yet known whether giving veliparib together with gemcitabine hydrochloride and cisplatin is an effective treatment for pancreatic cancer.\", \"clinical_trial_url\"=>\"https://clinicaltrials.gov/show/NCT01585805\"}, {\"nct_id\"=>\"NCT02184195\", \"name\"=>\"Olaparib in gBRCA Mutated Pancreatic Cancer Whose Disease Has Not Progressed on First Line Platinum-Based Chemotherapy\", \"description\"=>\"A Phase III, Randomised, Double Blind, Placebo Controlled, Multicentre Study of Maintenance Olaparib Monotherapy in Patients with gBRCA Mutated Metastatic Pancreatic Cancer whose Disease Has Not Progressed on First Line Platinum Based Chemotherapy\", \"clinical_trial_url\"=>\"https://clinicaltrials.gov/show/NCT02184195\"}]}, \"variant_id\"=>186, \"phenotypes\"=>[]}, {\"id\"=>5933, \"name\"=>\"EID5933\", \"description\"=>\"In Phase I study for pancreatic cancer patients\\nGEM 600 mg/m2 + CDDP 25 mg/m2 + Veliparib 20 - 80 mg twice daily showed response rate of  77.8% (7/9) and mOS of 23.3 months in germline BRCA mutation-positive patients with stage IV disease.\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>85, \"name\"=>\"Gemcitabine\", \"pubchem_id\"=>nil}, {\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}, {\"id\"=>556, \"name\"=>\"Veliparib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2405, \"name\"=>\"Phase 1 trial evaluating cisplatin, gemcitabine, and veliparib in 2 patient cohorts: Germline BRCA mutation carriers and wild-type BRCA pancreatic ductal adenocarcinoma.\", \"citation\"=>\"O'Reilly et al., 2018, Cancer\", \"pubmed_id\"=>\"29338080\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/29338080\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2018, \"month\"=>1, \"day\"=>16}, \"journal\"=>\"Cancer\", \"full_journal_title\"=>\"Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[{\"nct_id\"=>\"NCT01585805\", \"name\"=>\"Gemcitabine Hydrochloride and Cisplatin With or Without Veliparib or Veliparib Alone in Treating Patients With Locally Advanced or Metastatic Pancreatic Cancer\", \"description\"=>\"This randomized phase II trial studies how well veliparib together with gemcitabine hydrochloride and cisplatin works compared to gemcitabine hydrochloride and cisplatin alone in treating patients with pancreatic cancer that has spread from where it started to nearby tissue or lymph nodes (locally advanced) or spread from the primary site (place where it started) to other places in the body (metastatic). Veliparib may stop the growth of tumor cells by blocking some of the enzymes needed for cell growth. Drugs used in chemotherapy, such as gemcitabine hydrochloride and cisplatin, work in different ways to stop the growth of tumor cells, either by killing the cells, by stopping them from dividing, or by stopping them from spreading. It is not yet known whether giving veliparib together with gemcitabine hydrochloride and cisplatin is an effective treatment for pancreatic cancer.\", \"clinical_trial_url\"=>\"https://clinicaltrials.gov/show/NCT01585805\"}]}, \"variant_id\"=>186, \"phenotypes\"=>[]}, {\"id\"=>5831, \"name\"=>\"EID5831\", \"description\"=>\"Randomized, open-label, phase 3 trial in which olaparib monotherapy was compared with standard therapy in patients with a germline BRCA mutation HER2–negative metastatic breast cancer. 205 patients were assigned to receive olaparib and 97 were assigned to receive standard therapy. Median progression-free survival was significantly longer in the olaparib group than in the standard-therapy group (7.0 months vs. 4.2 months; hazard ratio 0.58; 95%CI 0.43 to 0.80; P<0.001). A response to treatment occurred in 100 of the 167 patients who had measurable disease in the olaparib group (59.9%; 95% CI, 52.0 to 67.4) and in 19 of the 66 patients in the standard-therapy group (28.8%; 95% CI, 18.3 to 41.3). A complete response was seen in 9.0% of the patients who had measurable disease in the olaparib group and in 1.5% in the standard-therapy group.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2357, \"name\"=>\"Olaparib for Metastatic Breast Cancer in Patients with a Germline BRCA Mutation.\", \"citation\"=>\"2017, N. Engl. J. Med.\", \"pubmed_id\"=>\"28792849\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28792849\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>8, \"day\"=>9}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>186, \"phenotypes\"=>[]}, {\"id\"=>1685, \"name\"=>\"EID1685\", \"description\"=>\"In an evaluation of 77 patients, those who had BRCA1/2 germline mutations (n=11) had an increase response rate to cisplatin when compared to those without germline mutations (54.5% versus 19.7%; P=0.022) although progression free survival and overall survival was not significantly different.\", \"disease\"=>{\"id\"=>370, \"name\"=>\"Triple-receptor Negative Breast Cancer\", \"display_name\"=>\"Triple-receptor Negative Breast Cancer\", \"doid\"=>\"0060081\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060081\"}, \"drugs\"=>[{\"id\"=>93, \"name\"=>\"Carboplatin\", \"pubchem_id\"=>nil}, {\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1137, \"name\"=>\"TBCRC009: A Multicenter Phase II Clinical Trial of Platinum Monotherapy With Biomarker Assessment in Metastatic Triple-Negative Breast Cancer.\", \"citation\"=>\"Isakoff et al., 2015, J. Clin. Oncol.\", \"pubmed_id\"=>\"25847936\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25847936\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4451173\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>6, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>186, \"phenotypes\"=>[]}, {\"id\"=>1678, \"name\"=>\"EID1678\", \"description\"=>\"46 women with platinum-sensitive, high-grade, BRCA1/2+ ovarian cancer received either olaparib monotherapy or olaparib/cediranib combination therapy.  Median PFS was increased in combination therapy (17.7 months versus 9.0 months; HR=0.42; 95%CI 0.23-0.76; p=0.005); overall response rate was increased in the combination therapy (79.6% versus 47.8%; p=0.002); and total deaths were reduced in combination therapy (6 patients versus 10 patients).\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}, {\"id\"=>400, \"name\"=>\"Cediranib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1134, \"name\"=>\"Combination cediranib and olaparib versus olaparib alone for women with recurrent platinum-sensitive ovarian cancer: a randomised phase 2 study.\", \"citation\"=>\"Liu et al., 2014, Lancet Oncol.\", \"pubmed_id\"=>\"25218906\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25218906\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4294183\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>10}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>186, \"phenotypes\"=>[]}, {\"id\"=>1676, \"name\"=>\"EID1676\", \"description\"=>\"664 patients with non-bulky FIGO Stage II-IV ovarian cancer were evaluated for PFS when treated with Pazopanib versus placebo.  Of the 664 patients, 10% (n=68) were BRCA1+, 4% were BRCA 2+ (n=29) and 85% (n=567) were BCRA1/2 negative.  Longer progression free survival was observed in BRCA1/2 positive carriers versus BRCA1/2 negative carriers in both the Pazopanib arm and the placebo arm, however, due to small sample size, the PFS increase in the Pazopanib arm was not statistically significant.\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1133, \"name\"=>\"BRCA1/2 mutations associated with progression-free survival in ovarian cancer patients in the AGO-OVAR 16 study.\", \"citation\"=>\"Harter et al., 2016, Gynecol. Oncol.\", \"pubmed_id\"=>\"26740259\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26740259\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>3}, \"journal\"=>\"Gynecol. Oncol.\", \"full_journal_title\"=>\"Gynecologic oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>186, \"phenotypes\"=>[]}, {\"id\"=>1532, \"name\"=>\"EID1532\", \"description\"=>\"In a study of ovarian carcinomas, 87% of tumors with somatic BRCA1/2 mutations were platinum sensitive, whereas only 60% of tumors with non-BRCA1/2 mutations were platinum sensitive (N=38; N=158; P=0.002). Sensitivity to platinum was defined by maintenance of complete response greater than 6 months post platinum therapy.\", \"disease\"=>{\"id\"=>47, \"name\"=>\"Ovarian Carcinoma\", \"display_name\"=>\"Ovarian Carcinoma\", \"doid\"=>\"4001\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4001\"}, \"drugs\"=>[{\"id\"=>103, \"name\"=>\"PLATINUM\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1020, \"name\"=>\"Germline and somatic mutations in homologous recombination genes predict platinum response and survival in ovarian, fallopian tube, and peritoneal carcinomas.\", \"citation\"=>\"Pennington et al., 2014, Clin. Cancer Res.\", \"pubmed_id\"=>\"24240112\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24240112\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3944197\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>2, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>186, \"phenotypes\"=>[]}, {\"id\"=>1530, \"name\"=>\"EID1530\", \"description\"=>\"In a study of ovarian carcinomas, 81% of tumors with germline BRCA1/2 mutations were platinum sensitive, whereas only 60% of tumors with non-BRCA1/2 mutations were platinum sensitive (N=47; N=158; P=0.01). Sensitivity was defined by maintenance of complete response greater than 6 months post platinum-based therapy.\", \"disease\"=>{\"id\"=>47, \"name\"=>\"Ovarian Carcinoma\", \"display_name\"=>\"Ovarian Carcinoma\", \"doid\"=>\"4001\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4001\"}, \"drugs\"=>[{\"id\"=>103, \"name\"=>\"PLATINUM\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1020, \"name\"=>\"Germline and somatic mutations in homologous recombination genes predict platinum response and survival in ovarian, fallopian tube, and peritoneal carcinomas.\", \"citation\"=>\"Pennington et al., 2014, Clin. Cancer Res.\", \"pubmed_id\"=>\"24240112\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24240112\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3944197\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>2, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>186, \"phenotypes\"=>[]}, {\"id\"=>1371, \"name\"=>\"EID1371\", \"description\"=>\"In this phase 1 trial 60 patients with advanced solid tumors (ovarian, breast, colorectal, melanoma, sarcoma, prostate) were treated with PARP inhibitor olaparib. Durable antitumor activity was only observed in BRCA1/2 mutation carriers.19 BRCA carriers had ovarian, breast, or prostate cancers; 12 of these (63%) had a clinical benefit from treatment with olaparib: radiologic or tumor-marker responses or meaningful disease stabilization (stable disease for a period of 4 months or more). Nine BRCA carriers had a response according to RECIST, with the response sustained for more than 76 weeks in one patient. No responses were seen in patients without BRCA mutation.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>359, \"name\"=>\"Inhibition of poly(ADP-ribose) polymerase in tumors from BRCA mutation carriers.\", \"citation\"=>\"Fong et al., 2009, N. Engl. J. Med.\", \"pubmed_id\"=>\"19553641\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19553641\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2009, \"month\"=>7, \"day\"=>9}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>186, \"phenotypes\"=>[]}, {\"id\"=>1776, \"name\"=>\"EID1776\", \"description\"=>\"In a study of 26 patients with breast cancer, 10 patients were BRCA-positive and 16 patients were BRCA-negative.  Regardless of BRCA status, none showed objective response when treated with olaparib.\", \"disease\"=>{\"id\"=>370, \"name\"=>\"Triple-receptor Negative Breast Cancer\", \"display_name\"=>\"Triple-receptor Negative Breast Cancer\", \"doid\"=>\"0060081\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060081\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1239, \"name\"=>\"Olaparib in patients with recurrent high-grade serous or poorly differentiated ovarian carcinoma or triple-negative breast cancer: a phase 2, multicentre, open-label, non-randomised study.\", \"citation\"=>\"Gelmon et al., 2011, Lancet Oncol.\", \"pubmed_id\"=>\"21862407\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21862407\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>9}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>186, \"phenotypes\"=>[]}, {\"id\"=>1773, \"name\"=>\"EID1773\", \"description\"=>\"In a study of 63 patients with ovarian cancer, 17 patients had BRCA1 or BRCA2 mutations and 46 patients had no BRCA mutations.  Of the BRCA-positive patients, 7/17(41%; 95%CI 22-64) showed objective response to olaparib whereas only 11/46 BRCA-negative patients (24%; 95%CI 14-38) showed objective response to olaparib.\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1239, \"name\"=>\"Olaparib in patients with recurrent high-grade serous or poorly differentiated ovarian carcinoma or triple-negative breast cancer: a phase 2, multicentre, open-label, non-randomised study.\", \"citation\"=>\"Gelmon et al., 2011, Lancet Oncol.\", \"pubmed_id\"=>\"21862407\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21862407\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>9}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>186, \"phenotypes\"=>[]}, {\"id\"=>465, \"name\"=>\"EID465\", \"description\"=>\"In patients with invasive epithelial ovarian cancer, those with mutations in BRCA2 had better five-year overall survival.\", \"disease\"=>{\"id\"=>225, \"name\"=>\"Epithelial Ovarian Cancer\", \"display_name\"=>\"Epithelial Ovarian Cancer\", \"doid\"=>\"2152\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2152\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>279, \"name\"=>\"Association between BRCA1 and BRCA2 mutations and survival in women with invasive epithelial ovarian cancer.\", \"citation\"=>\"Bolton et al., 2012, JAMA\", \"pubmed_id\"=>\"22274685\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22274685\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3727895\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>1, \"day\"=>25}, \"journal\"=>\"JAMA\", \"full_journal_title\"=>\"JAMA\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>186, \"phenotypes\"=>[]}]}, {\"id\"=>214, \"entrez_name\"=>\"PTEN\", \"entrez_id\"=>5728, \"name\"=>\"LOSS\", \"description\"=>\"\", \"gene_id\"=>41, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>229.0, \"coordinates\"=>{\"chromosome\"=>\"10\", \"start\"=>89622870, \"stop\"=>89731687, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000371953.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/41/summary/variants/214/summary#variant\", \"evidence_items\"=>[{\"id\"=>5840, \"name\"=>\"EID5840\", \"description\"=>\"In a single-arm, open label phase 2 trial in PIK3CA amplified/mutated and/or PTEN loss encompassing 10 patients, no objective responses were observed but a stable disease response in 4 patients, resulting in a median PFS of 1.4 months. Everolimus failed to meet its enpoint.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2364, \"name\"=>\"Prospective phase II trial of everolimus in PIK3CA amplification/mutation and/or PTEN loss patients with advanced solid tumors refractory to standard therapy.\", \"citation\"=>\"Kim et al., 2017, BMC Cancer\", \"pubmed_id\"=>\"28330462\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28330462\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5363054\", \"publication_date\"=>{\"year\"=>2017, \"month\"=>3, \"day\"=>23}, \"journal\"=>\"BMC Cancer\", \"full_journal_title\"=>\"BMC cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>5957, \"name\"=>\"EID5957\", \"description\"=>\"No objective responses were observed in the PTEN deficient cohort. Four out of 6 patients with stable disease (SD) had SD or better for ≥6 cycles, 2 of which had PTEN deficient tumors. Conclusion The addition of buparlisib to high dose carboplatin and paclitaxel was not tolerable. The combination did not reveal significant clinical activity amongst a small and heterogenous group of PTEN deficient tumors.\", \"disease\"=>{\"id\"=>2616, \"name\"=>\"Solid Tumor\", \"display_name\"=>\"Solid Tumor\", \"doid\"=>\"\", \"url\"=>\"http://www.disease-ontology.org/\"}, \"drugs\"=>[{\"id\"=>81, \"name\"=>\"Paclitaxel\", \"pubchem_id\"=>nil}, {\"id\"=>93, \"name\"=>\"Carboplatin\", \"pubchem_id\"=>nil}, {\"id\"=>116, \"name\"=>\"Buparlisib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2413, \"name\"=>\"A phase 1b dose expansion study of the pan-class I PI3K inhibitor buparlisib (BKM120) plus carboplatin and paclitaxel in PTEN deficient tumors and with dose intensified carboplatin and paclitaxel.\", \"citation\"=>\"Smyth et al., 2017, Invest New Drugs\", \"pubmed_id\"=>\"28281183\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28281183\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5591764\", \"publication_date\"=>{\"year\"=>2017, \"month\"=>12}, \"journal\"=>\"Invest New Drugs\", \"full_journal_title\"=>\"Investigational new drugs\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[{\"nct_id\"=>\"NCT01297452\", \"name\"=>\"BKM120 + Carboplatin + Paclitaxel for Patients With Advanced Solid Tumors\", \"description\"=>\"The purpose of this study is to find out the good and bad effects that occur when BKM120 is added to standard chemotherapy with carboplatin and paclitaxel.\", \"clinical_trial_url\"=>\"https://clinicaltrials.gov/show/NCT01297452\"}]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>1502, \"name\"=>\"EID1502\", \"description\"=>\"In a panel of 182 cell lines (162 with PIK3CA mutation status known) from solid and hematological tumors, significant correlation was found between AZD5363 (pan-Akt inhibitor) sensitivity and PIK3CA mutation status (N=33, p=0.0059). In the subpopulation free of RAS mutation (N=19) the correlation was increased.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>112, \"name\"=>\"AZD5363\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1005, \"name\"=>\"Preclinical pharmacology of AZD5363, an inhibitor of AKT: pharmacodynamics, antitumor activity, and correlation of monotherapy activity with genetic background.\", \"citation\"=>\"Davies et al., 2012, Mol. Cancer Ther.\", \"pubmed_id\"=>\"22294718\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22294718\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>4}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>1455, \"name\"=>\"EID1455\", \"description\"=>\"Knockdown of PTEN in trastuzumab sensitive HER2 breast cancer BT474 cells induced trastuzumab resistance. PTEN null cells treated with PI3K inhibitor LY294002 also did not show substantial inhibition, but co-treatment with trastuzumab and LY294002 induced growth inhibition comparable to trastuzumab alone in normal BT474 cells. BT474 mouse xenograft tumors were injected with PTEN antisense (AS) oligonucleotides and this resulted in trastuzumab resitance. Co-treatment of PTEN AS tumor mice with LY294002 and trastuzumab inhibited tumor growth in comparison to PTEN AS tumors treated with trastuzumab or LY294002 alone.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}, {\"id\"=>115, \"name\"=>\"LY294002\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>983, \"name\"=>\"PTEN activation contributes to tumor inhibition by trastuzumab, and loss of PTEN predicts trastuzumab resistance in patients.\", \"citation\"=>\"Nagata et al., 2004, Cancer Cell\", \"pubmed_id\"=>\"15324695\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15324695\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2004, \"month\"=>8}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>1493, \"name\"=>\"EID1493\", \"description\"=>\"Cancer and normal tissue samples were taken from 68 gastric cancer patients undergoing surgery without prior chemotherapy. Tissue samples were divided into high grade and low grade for phospho Akt (p-Akt) expression, and tested for PTEN LOH by sequencing. Patients with PTEN LOH were significantly more likely to have elevated p-Akt (p=0.0081), and high p-Akt samples were significantly less sensitive to chemotherapy agents 5-flourouracil, adriamycin, mitomycin and cis-platinum in MTT assay.\", \"disease\"=>{\"id\"=>1322, \"name\"=>\"Stomach Carcinoma\", \"display_name\"=>\"Stomach Carcinoma\", \"doid\"=>\"5517\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5517\"}, \"drugs\"=>[{\"id\"=>199, \"name\"=>\"Chemotherapy\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1000, \"name\"=>\"Akt phosphorylation associates with LOH of PTEN and leads to chemoresistance for gastric cancer.\", \"citation\"=>\"Oki et al., 2005, Int. J. Cancer\", \"pubmed_id\"=>\"15900596\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15900596\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>11, \"day\"=>10}, \"journal\"=>\"Int. J. Cancer\", \"full_journal_title\"=>\"International journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>1383, \"name\"=>\"EID1383\", \"description\"=>\"A large-scale RNAi genetic sereen in HER2 overexpressing breast cancer cell line BT-474 was performed to discover genes whose loss induced trastuzumab resistance. The most prominently enriched gene in trastuzumab resistant populations was PTEN. Subsequent knockdown of PTEN in BT-474 cells increased cell survival when treated with trastuzumab, and growth curves showed partial restoration of proliferation in PTEN knockdown BT-474 cells treated with trastuzumab.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>938, \"name\"=>\"A functional genetic approach identifies the PI3K pathway as a major determinant of trastuzumab resistance in breast cancer.\", \"citation\"=>\"Berns et al., 2007, Cancer Cell\", \"pubmed_id\"=>\"17936563\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17936563\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>10}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>1492, \"name\"=>\"EID1492\", \"description\"=>\"A cisplatin resistant derivative of ovarian cancer cell line OVCAR-3 was generated by long term passage with increasing cisplatin levels. Western blot showed decreased PTEN level in the resistant cell line, and siRNA against PTEN in parent OVCAR-3 cells induced cisplatin resistance.\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>999, \"name\"=>\"Activation of PI3K/Akt pathway by PTEN reduction and PIK3CA mRNA amplification contributes to cisplatin resistance in an ovarian cancer cell line.\", \"citation\"=>\"Lee et al., 2005, Gynecol. Oncol.\", \"pubmed_id\"=>\"15790433\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/15790433\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>4}, \"journal\"=>\"Gynecol. Oncol.\", \"full_journal_title\"=>\"Gynecologic oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>1385, \"name\"=>\"EID1385\", \"description\"=>\"A subgroup of HER2-positive breast cancer patients that received first-line trastuzumab treatment were assessed for PTEN loss via IHC of FFPE samples. 46 samples showed PTEN loss while 41 did not, and PTEN loss alone was found to be significantly associated with resistance to trastuzumab (p=0.028) and shorter overall survival (p=0.008).\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>939, \"name\"=>\"PTEN, PIK3CA, p-AKT, and p-p70S6K status: association with trastuzumab response and survival in patients with HER2-positive metastatic breast cancer.\", \"citation\"=>\"Esteva et al., 2010, Am. J. Pathol.\", \"pubmed_id\"=>\"20813970\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20813970\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2947262\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>10}, \"journal\"=>\"Am. J. Pathol.\", \"full_journal_title\"=>\"The American journal of pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>1386, \"name\"=>\"EID1386\", \"description\"=>\"While some reports indicate PTEN loss is associated with resistance to trastuzumab, in a subgroup of HER2 breast cancer FFPE samples from patients receiving trastuzumab as later line therapy (second or greater line therapy), with 25 PTEN loss and and 25 normal samples, no association was seen between trastuzumab response and PTEN status (p=1.000) or overall survival and PTEN status (p=0.863).\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>939, \"name\"=>\"PTEN, PIK3CA, p-AKT, and p-p70S6K status: association with trastuzumab response and survival in patients with HER2-positive metastatic breast cancer.\", \"citation\"=>\"Esteva et al., 2010, Am. J. Pathol.\", \"pubmed_id\"=>\"20813970\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20813970\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2947262\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>10}, \"journal\"=>\"Am. J. Pathol.\", \"full_journal_title\"=>\"The American journal of pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>1500, \"name\"=>\"EID1500\", \"description\"=>\"A patient derived mouse xenograft model was established with gastric cancer cells having PTEN loss. The xenograft was resistant to monotherapy with both the pan-Akt inhibitor AZD5363, and also to the commonly used gastric cancer chemotheraputic docetaxel. When used in combination, AZD5363 and docetaxel induced significant tumor growth inhibition in comparison to monotherapy and vehicle control.\", \"disease\"=>{\"id\"=>21, \"name\"=>\"Stomach Cancer\", \"display_name\"=>\"Stomach Cancer\", \"doid\"=>\"10534\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10534\"}, \"drugs\"=>[{\"id\"=>73, \"name\"=>\"Docetaxel\", \"pubchem_id\"=>nil}, {\"id\"=>112, \"name\"=>\"AZD5363\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1004, \"name\"=>\"The AKT inhibitor AZD5363 is selectively active in PI3KCA mutant gastric cancer, and sensitizes a patient-derived gastric cancer xenograft model with PTEN loss to Taxotere.\", \"citation\"=>\"Li et al., 2013, J Transl Med\", \"pubmed_id\"=>\"24088382\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24088382\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3850695\", \"publication_date\"=>{\"year\"=>2013}, \"journal\"=>\"J Transl Med\", \"full_journal_title\"=>\"Journal of translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>1614, \"name\"=>\"EID1614\", \"description\"=>\"A panel of 24 endometroid endometrial cancer (EEC) cell lines was characterized for mutations and 18 cell lines were found to have mutations in PTEN or absence of expression. Cell lines were assessed for sensitivity to mTORC1 allosteric inhibitor temsirolimus, and it was reported that PTEN loss had significant association with temsirolimus sensitivity. .\", \"disease\"=>{\"id\"=>108, \"name\"=>\"Endometrial Cancer\", \"display_name\"=>\"Endometrial Cancer\", \"doid\"=>\"1380\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1380\"}, \"drugs\"=>[{\"id\"=>83, \"name\"=>\"Temsirolimus\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1067, \"name\"=>\"PI3K pathway dependencies in endometrioid endometrial cancer cell lines.\", \"citation\"=>\"Weigelt et al., 2013, Clin. Cancer Res.\", \"pubmed_id\"=>\"23674493\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23674493\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3700760\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>7, \"day\"=>1}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>1297, \"name\"=>\"EID1297\", \"description\"=>\"Archival tumor samples from 549 patients from the BOLERO-1 and -3 trials were analyzed for alterations in the PI3K pathway using next-generation sequencing, immunohistochemistry (IHC) and Sanger sequencing. In the BOLERO trials, patients with HER2-overexpressing advanced breast cancer were treated with trastuzumab, chemotherapy and randomized to receive either everolimus or placebo. In the pooled analysis of both trials, patients with PTEN loss (as defined by IHC staining intensity and proportion of positive tumor cells) had a longer progression-free survival with everolimus compared to placebo (HR 0.54 (95%CI 0.31 to 0.96, P=0.04) whereas patients with PTEN normal status had a hazard ration of 1.00 (95%CI 0.8 to 1.26, P=0.97).\\nThe same was found when PI3K mutations, PTEN loss and AKT E17K mutations were combined and analyzed as hyperactive PI3K pathway. Patients with PI3K hyperactive pathway had a hazard ratio of 0.67 (95%CI 0.48 to 0.93, P=0.02) compared to HR 1.19 (95%CI 0.87 to 1.62, P=0.28) in patients with PI3K normal pathway. These trends were independently confirmed in both trials but only pooled results showed consistent statistical significance.\", \"disease\"=>{\"id\"=>368, \"name\"=>\"Her2-receptor Positive Breast Cancer\", \"display_name\"=>\"Her2-receptor Positive Breast Cancer\", \"doid\"=>\"0060079\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060079\"}, \"drugs\"=>[{\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>891, \"name\"=>\"Molecular Alterations and Everolimus Efficacy in Human Epidermal Growth Factor Receptor 2-Overexpressing Metastatic Breast Cancers: Combined Exploratory Biomarker Analysis From BOLERO-1 and BOLERO-3.\", \"citation\"=>\"André et al., 2016, J. Clin. Oncol.\", \"pubmed_id\"=>\"27091708\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27091708\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>6, \"day\"=>20}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>1611, \"name\"=>\"EID1611\", \"description\"=>\"In a panel of 31 breast cancer cell lines, 11 had loss of PTEN function. Cells were tested for sensitivity to mTORC1 inhibitor everolimus and mTORC1/2 inhibitor PP242, and no significant correlation was found between PTEN status and sensitivity to mTORC inhibition.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}, {\"id\"=>179, \"name\"=>\"PP242\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1066, \"name\"=>\"PIK3CA mutation, but not PTEN loss of function, determines the sensitivity of breast cancer cells to mTOR inhibitory drugs.\", \"citation\"=>\"Weigelt et al., 2011, Oncogene\", \"pubmed_id\"=>\"21358673\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21358673\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>7, \"day\"=>21}, \"journal\"=>\"Oncogene\", \"full_journal_title\"=>\"Oncogene\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>938, \"name\"=>\"EID938\", \"description\"=>\"Among 33 patients treated with pan-AKT inhibitor MK-2206 in this phase 1 study, one patient with pancreatic adenocarcinoma and PTEN loss/KRAS G12D mutation experienced a decrease of approximately 60% in cancer antigen 19-9 levels and 23% shrinkage in tumor measurements.\", \"disease\"=>{\"id\"=>169, \"name\"=>\"Pancreatic Carcinoma\", \"display_name\"=>\"Pancreatic Carcinoma\", \"doid\"=>\"4905\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4905\"}, \"drugs\"=>[{\"id\"=>43, \"name\"=>\"MK-2206\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>641, \"name\"=>\"First-in-man clinical trial of the oral pan-AKT inhibitor MK-2206 in patients with advanced solid tumors.\", \"citation\"=>\"Yap et al., 2011, J. Clin. Oncol.\", \"pubmed_id\"=>\"22025163\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22025163\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>12, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>864, \"name\"=>\"EID864\", \"description\"=>\"In 48 patients with advanced gastric cancer, retrospectively analyzed for PTEN expression using IHC, PTEN loss was associated with a shorter duration of response (median 4.2 vs. 6.1 months, p = 0.04) to HER2-targeted therapy (trastuzumab, N=39; lapatinib, N=9). Response rates did not differ between groups.\", \"disease\"=>{\"id\"=>147, \"name\"=>\"Gastric Adenocarcinoma\", \"display_name\"=>\"Gastric Adenocarcinoma\", \"doid\"=>\"3717\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3717\"}, \"drugs\"=>[{\"id\"=>45, \"name\"=>\"Lapatinib\", \"pubchem_id\"=>nil}, {\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>579, \"name\"=>\"PTEN deficiency as a predictive biomarker of resistance to HER2-targeted therapy in advanced gastric cancer.\", \"citation\"=>\"Zhang et al., 2015, Oncology\", \"pubmed_id\"=>\"25300346\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25300346\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"Oncology\", \"full_journal_title\"=>\"Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>893, \"name\"=>\"EID893\", \"description\"=>\"94 endometrial cancer samples of women who had been treated with single agent mTOR inhibitors temsirolimus or ridaforolimus were retrospectively analyzed.\\nNo predictive biomarkers were identified, including PIK3CA mutations, KRAS mutations or PTEN loss.\", \"disease\"=>{\"id\"=>108, \"name\"=>\"Endometrial Cancer\", \"display_name\"=>\"Endometrial Cancer\", \"doid\"=>\"1380\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1380\"}, \"drugs\"=>[{\"id\"=>83, \"name\"=>\"Temsirolimus\", \"pubchem_id\"=>nil}, {\"id\"=>170, \"name\"=>\"Ridaforolimus\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>602, \"name\"=>\"Molecular determinants of outcome with mammalian target of rapamycin inhibition in endometrial cancer.\", \"citation\"=>\"Mackay et al., 2014, Cancer\", \"pubmed_id\"=>\"24166148\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24166148\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>2, \"day\"=>15}, \"journal\"=>\"Cancer\", \"full_journal_title\"=>\"Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>711, \"name\"=>\"EID711\", \"description\"=>\"Breast and prostate cancer cell lines responsive to AZD8186 were enriched for PTEN deficiency.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>114, \"name\"=>\"AZD8186\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>467, \"name\"=>\"Inhibition of PI3Kβ signaling with AZD8186 inhibits growth of PTEN-deficient breast and prostate tumors alone and in combination with docetaxel.\", \"citation\"=>\"Hancox et al., 2015, Mol. Cancer Ther.\", \"pubmed_id\"=>\"25398829\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25398829\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>1}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>645, \"name\"=>\"EID645\", \"description\"=>\"PTEN loss is associated with loss of sensitivity to the HER2-antibody trastuzumab in breast cancer in smaller trials, a phase III trial (NCCTG N9831) did not show an effect of PTEN on disease-free survival.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>84, \"name\"=>\"Trastuzumab\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>406, \"name\"=>\"Therapeutic targeting of cancers with loss of PTEN function.\", \"citation\"=>\"Dillon et al., 2014, Curr Drug Targets\", \"pubmed_id\"=>\"24387334\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24387334\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4310752\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>1}, \"journal\"=>\"Curr Drug Targets\", \"full_journal_title\"=>\"Current drug targets\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>714, \"name\"=>\"EID714\", \"description\"=>\"All three patients with PTEN loss had benefit from adding Buparlisib to carboplatin+paclitaxel therapy in a phase I study in advanced solid cancers.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>81, \"name\"=>\"Paclitaxel\", \"pubchem_id\"=>nil}, {\"id\"=>93, \"name\"=>\"Carboplatin\", \"pubchem_id\"=>nil}, {\"id\"=>116, \"name\"=>\"Buparlisib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>469, \"name\"=>\"Parallel phase Ib studies of two schedules of buparlisib (BKM120) plus carboplatin and paclitaxel (q21 days or q28 days) for patients with advanced solid tumors.\", \"citation\"=>\"Hyman et al., 2015, Cancer Chemother. Pharmacol.\", \"pubmed_id\"=>\"25672916\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25672916\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>4}, \"journal\"=>\"Cancer Chemother. Pharmacol.\", \"full_journal_title\"=>\"Cancer chemotherapy and pharmacology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>716, \"name\"=>\"EID716\", \"description\"=>\"A case report described a responder to PI3K inhibitor BYL719 for a tumor with PIK3CA E542K mutation. After partial response for 9.5 months, resistance developed. Sequencing in this and other patients as well as preclinical models showed loss of PTEN as the probable mechanism of resistance.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>339, \"name\"=>\"BYL719 (Alpelisib)\", \"pubchem_id\"=>\"\"}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>471, \"name\"=>\"Convergent loss of PTEN leads to clinical resistance to a PI(3)Kα inhibitor.\", \"citation\"=>\"Juric et al., 2015, Nature\", \"pubmed_id\"=>\"25409150\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25409150\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4326538\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>2, \"day\"=>12}, \"journal\"=>\"Nature\", \"full_journal_title\"=>\"Nature\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>708, \"name\"=>\"EID708\", \"description\"=>\"Most BRAF mutant cell lines (mostly V600E) with PTEN mutation or loss of expression were among those sensitive to inhibition with AKT inhibitor\\nGSK2141795B.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>111, \"name\"=>\"GSK2141795\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>465, \"name\"=>\"Effects of AKT inhibitor therapy in response and resistance to BRAF inhibition in melanoma.\", \"citation\"=>\"Lassen et al., 2014, Mol. Cancer\", \"pubmed_id\"=>\"24735930\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24735930\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4021505\", \"publication_date\"=>{\"year\"=>2014}, \"journal\"=>\"Mol. Cancer\", \"full_journal_title\"=>\"Molecular cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>644, \"name\"=>\"EID644\", \"description\"=>\"PTEN loss is associated with reduced response to mTOR inhibitors in patients with bladder cancer.\", \"disease\"=>{\"id\"=>18, \"name\"=>\"Bladder Carcinoma\", \"display_name\"=>\"Bladder Carcinoma\", \"doid\"=>\"4007\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4007\"}, \"drugs\"=>[{\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>407, \"name\"=>\"PTEN deficiency is associated with reduced sensitivity to mTOR inhibitor in human bladder cancer through the unhampered feedback loop driving PI3K/Akt activation.\", \"citation\"=>\"Seront et al., 2013, Br. J. Cancer\", \"pubmed_id\"=>\"23989949\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23989949\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3777009\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>9, \"day\"=>17}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>643, \"name\"=>\"EID643\", \"description\"=>\"PTEN Loss (by FISH) is associated with poorer prognosis in patients with prostate cancer.\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>310, \"name\"=>\"FISH analysis of 107 prostate cancers shows that PTEN genomic deletion is associated with poor clinical outcome.\", \"citation\"=>\"Yoshimoto et al., 2007, Br. J. Cancer\", \"pubmed_id\"=>\"17700571\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17700571\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2360375\", \"publication_date\"=>{\"year\"=>2007, \"month\"=>9, \"day\"=>3}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>507, \"name\"=>\"EID507\", \"description\"=>\"In prostate cancer xenographs, PTEN null tumors had a significant decrease in cell size and proliferation when treated with Temsirolimus (CCI-779).\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[{\"id\"=>83, \"name\"=>\"Temsirolimus\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>312, \"name\"=>\"Enhanced sensitivity of PTEN-deficient tumors to inhibition of FRAP/mTOR.\", \"citation\"=>\"Neshat et al., 2001, Proc. Natl. Acad. Sci. U.S.A.\", \"pubmed_id\"=>\"11504908\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/11504908\", \"open_access\"=>true, \"pmc_id\"=>\"PMC56958\", \"publication_date\"=>{\"year\"=>2001, \"month\"=>8, \"day\"=>28}, \"journal\"=>\"Proc. Natl. Acad. Sci. U.S.A.\", \"full_journal_title\"=>\"Proceedings of the National Academy of Sciences of the United States of America\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}, {\"id\"=>506, \"name\"=>\"EID506\", \"description\"=>\"In 27 patients with chemotherapy-naive or chemotherapy-refractory metastatic colorectal cancer who were treated with cetuximab and chemotherapy, PTEN loss was associated with non-response.\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>311, \"name\"=>\"KRAS, BRAF, PIK3CA, and PTEN mutations: implications for targeted therapies in metastatic colorectal cancer.\", \"citation\"=>\"De Roock et al., 2011, Lancet Oncol.\", \"pubmed_id\"=>\"21163703\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21163703\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>6}, \"journal\"=>\"Lancet Oncol.\", \"full_journal_title\"=>\"The Lancet. Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>214, \"phenotypes\"=>[]}]}, {\"id\"=>216, \"entrez_name\"=>\"SMAD4\", \"entrez_id\"=>4089, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>77, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}, {\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>25.0, \"coordinates\"=>{\"chromosome\"=>\"18\", \"start\"=>48556583, \"stop\"=>48611409, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000342988.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/77/summary/variants/216/summary#variant\", \"evidence_items\"=>[{\"id\"=>719, \"name\"=>\"EID719\", \"description\"=>\"In a retrospective analysis of 65 patients with metastatic colorectal cancer, SMAD4 mutations were more common among patients with no benefit from EGFR-inhibition (cetuximab or panitumumab) (4 patients) than with patients with response to EGFR directed treatment (1 patient).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}, {\"id\"=>28, \"name\"=>\"Panitumumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>473, \"name\"=>\"Prediction of response to anti-EGFR antibody-based therapies by multigene sequencing in colorectal cancer patients.\", \"citation\"=>\"Lupini et al., 2015, BMC Cancer\", \"pubmed_id\"=>\"26508446\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26508446\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4624582\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"BMC Cancer\", \"full_journal_title\"=>\"BMC cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>216, \"phenotypes\"=>[]}, {\"id\"=>511, \"name\"=>\"EID511\", \"description\"=>\"In a study of 89 patients with adenocarcinoma of the pancreas who underwent a pancreaticoduodenectomy, those with mutations in SMAD4 had shorter overall survival than wild-type patients.\", \"disease\"=>{\"id\"=>158, \"name\"=>\"Pancreatic Adenocarcinoma\", \"display_name\"=>\"Pancreatic Adenocarcinoma\", \"doid\"=>\"4074\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4074\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>315, \"name\"=>\"SMAD4 gene mutations are associated with poor prognosis in pancreatic cancer.\", \"citation\"=>\"Blackford et al., 2009, Clin. Cancer Res.\", \"pubmed_id\"=>\"19584151\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19584151\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2819274\", \"publication_date\"=>{\"year\"=>2009, \"month\"=>7, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>216, \"phenotypes\"=>[]}]}, {\"id\"=>217, \"entrez_name\"=>\"SMARCA4\", \"entrez_id\"=>6597, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>78, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>30, \"name\"=>\"gene_variant\", \"display_name\"=>\"Gene Variant\", \"so_id\"=>\"SO:0001564\", \"description\"=>\"A sequence variant where the structure of the gene is changed.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001564\"}, {\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>25.0, \"coordinates\"=>{\"chromosome\"=>\"19\", \"start\"=>11071598, \"stop\"=>11172958, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000358026.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/78/summary/variants/217/summary#variant\", \"evidence_items\"=>[{\"id\"=>512, \"name\"=>\"EID512\", \"description\"=>\"Nonsense mutations in SMARCA4 were associated with small cell carcinoma of the ovary, hypercalcemic type in 10/10 tumor samples, and were rarely found in other solid tumor types catalogued by TCGA.\", \"disease\"=>{\"id\"=>253, \"name\"=>\"Small Cell Carcinoma Of The Ovary Hypercalcemic Type\", \"display_name\"=>\"Small Cell Carcinoma Of The Ovary Hypercalcemic Type\", \"doid\"=>\"7651\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:7651\"}, \"drugs\"=>[], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Diagnostic\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>316, \"name\"=>\"Recurrent SMARCA4 mutations in small cell carcinoma of the ovary.\", \"citation\"=>\"Jelinic et al., 2014, Nat. Genet.\", \"pubmed_id\"=>\"24658004\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24658004\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>5}, \"journal\"=>\"Nat. Genet.\", \"full_journal_title\"=>\"Nature genetics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>217, \"phenotypes\"=>[]}]}, {\"id\"=>218, \"entrez_name\"=>\"SMARCA4\", \"entrez_id\"=>6597, \"name\"=>\"INACTIVATING MUTATION\", \"description\"=>\"\", \"gene_id\"=>78, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"19\", \"start\"=>11071598, \"stop\"=>11172958, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000358026.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/78/summary/variants/218/summary#variant\", \"evidence_items\"=>[{\"id\"=>5928, \"name\"=>\"EID5928\", \"description\"=>\"AURKA activity as essential in NSCLC cells lacking SMARCA4/BRG1. In these cells, RNAi-mediated depletion or chemical inhibition of AURKA induces apoptosis and cell death in vitro and in xenograft mouse models.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>554, \"name\"=>\"Tozasertib(MK-0457,VX-680)\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2400, \"name\"=>\"SMARCA4-inactivating mutations increase sensitivity to Aurora kinase A inhibitor VX-680 in non-small cell lung cancers.\", \"citation\"=>\"Tagal et al., 2017, Nat Commun\", \"pubmed_id\"=>\"28102363\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28102363\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5253647\", \"publication_date\"=>{\"year\"=>2017, \"month\"=>1, \"day\"=>19}, \"journal\"=>\"Nat Commun\", \"full_journal_title\"=>\"Nature communications\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>218, \"phenotypes\"=>[]}, {\"id\"=>513, \"name\"=>\"EID513\", \"description\"=>\"In patients with lung adenocarcinoma, those with loss-of-function mutations in SMARCA4 had worse overall survival than those with only missense or no mutations.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>316, \"name\"=>\"Recurrent SMARCA4 mutations in small cell carcinoma of the ovary.\", \"citation\"=>\"Jelinic et al., 2014, Nat. Genet.\", \"pubmed_id\"=>\"24658004\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24658004\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>5}, \"journal\"=>\"Nat. Genet.\", \"full_journal_title\"=>\"Nature genetics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>218, \"phenotypes\"=>[]}]}, {\"id\"=>230, \"entrez_name\"=>\"CHEK2\", \"entrez_id\"=>11200, \"name\"=>\"LOSS-OF-FUNCTION\", \"description\"=>\"\", \"gene_id\"=>8950, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>25.0, \"coordinates\"=>{\"chromosome\"=>\"22\", \"start\"=>29083732, \"stop\"=>29137832, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000328354.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/8950/summary/variants/230/summary#variant\", \"evidence_items\"=>[{\"id\"=>531, \"name\"=>\"EID531\", \"description\"=>\"Heterozygous germline truncating (frameshift) and missense mutations confer highly penetrant predisposition to multiple cancer types including sarcoma, breast cancer, and brain tumors. Screening of individuals with Li-Fraumeni syndrome but lacking TP53 mutations identified 1/4 classical and 2/18 LFS variant presentation. This included 1 large family pedigree with strong co-segregation of the mutation with cancer development.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>331, \"name\"=>\"Heterozygous germ line hCHK2 mutations in Li-Fraumeni syndrome.\", \"citation\"=>\"Bell et al., 1999, Science\", \"pubmed_id\"=>\"10617473\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/10617473\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1999, \"month\"=>12, \"day\"=>24}, \"journal\"=>\"Science\", \"full_journal_title\"=>\"Science (New York, N.Y.)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>230, \"phenotypes\"=>[]}]}, {\"id\"=>236, \"entrez_name\"=>\"CDK12\", \"entrez_id\"=>51755, \"name\"=>\"LOSS-OF-FUNCTION\", \"description\"=>\"\", \"gene_id\"=>12112, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>4.0, \"coordinates\"=>{\"chromosome\"=>\"17\", \"start\"=>37618292, \"stop\"=>37691399, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000447079.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/12112/summary/variants/236/summary#variant\", \"evidence_items\"=>[{\"id\"=>623, \"name\"=>\"EID623\", \"description\"=>\"In models of high-grade serous ovarian cancer (HGS-OVCa), CDK12 attenuation was sufficient to confer sensitivity to PARP1/2 inhibition, suppression of DNA repair via homologous recombination, and reduced expression of BRCA1.\", \"disease\"=>{\"id\"=>87, \"name\"=>\"Ovarian Serous Carcinoma\", \"display_name\"=>\"Ovarian Serous Carcinoma\", \"doid\"=>\"0050933\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050933\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>386, \"name\"=>\"Genome-wide profiling of genetic synthetic lethality identifies CDK12 as a novel determinant of PARP1/2 inhibitor sensitivity.\", \"citation\"=>\"Bajrami et al., 2014, Cancer Res.\", \"pubmed_id\"=>\"24240700\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24240700\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4886090\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>1, \"day\"=>1}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>236, \"phenotypes\"=>[]}]}, {\"id\"=>281, \"entrez_name\"=>\"FBXW7\", \"entrez_id\"=>55294, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>12903, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}, {\"id\"=>103, \"name\"=>\"protein_altering_variant\", \"display_name\"=>\"Protein Altering Variant\", \"so_id\"=>\"SO:0001818\", \"description\"=>\"A sequence_variant which is predicted to change the protein encoded in the coding sequence.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001818\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>153245446, \"stop\"=>153332714, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000281708.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/12903/summary/variants/281/summary#variant\", \"evidence_items\"=>[{\"id\"=>1631, \"name\"=>\"EID1631\", \"description\"=>\"418 patients with advanced cancer were screened for FBXW7 mutations using a multiplex gene panel. 17 FBXW7 mutations were detected (isolated mutation in 2 patients).\\nTen patients were treated with an mTOR inhibitor with 7 patients having stable disease as best responses and a median time to treatment failure of 2.8 months, two patients had prolonged stable disease of >16 weeks.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>67, \"name\"=>\"MTOR Inhibitors\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1079, \"name\"=>\"FBXW7 mutations in patients with advanced cancers: clinical and molecular characteristics and outcomes with mTOR inhibitors.\", \"citation\"=>\"Jardim et al., 2014, PLoS ONE\", \"pubmed_id\"=>\"24586741\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24586741\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3929689\", \"publication_date\"=>{\"year\"=>2014}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>281, \"phenotypes\"=>[]}, {\"id\"=>718, \"name\"=>\"EID718\", \"description\"=>\"65 patients were retrospectively analyzed for mutational profiles predictive of response to EGFR-inhibitors (cetuximab or panitumumab). FBXW7 mutation was found to be more prevalent in non-responders (5 mutations) than responders (1 mutation).\", \"disease\"=>{\"id\"=>11, \"name\"=>\"Colorectal Cancer\", \"display_name\"=>\"Colorectal Cancer\", \"doid\"=>\"9256\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9256\"}, \"drugs\"=>[{\"id\"=>16, \"name\"=>\"Cetuximab\", \"pubchem_id\"=>nil}, {\"id\"=>28, \"name\"=>\"Panitumumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>473, \"name\"=>\"Prediction of response to anti-EGFR antibody-based therapies by multigene sequencing in colorectal cancer patients.\", \"citation\"=>\"Lupini et al., 2015, BMC Cancer\", \"pubmed_id\"=>\"26508446\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26508446\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4624582\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"BMC Cancer\", \"full_journal_title\"=>\"BMC cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>281, \"phenotypes\"=>[]}]}, {\"id\"=>301, \"entrez_name\"=>\"PTCH1\", \"entrez_id\"=>5727, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>4645, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>30, \"name\"=>\"gene_variant\", \"display_name\"=>\"Gene Variant\", \"so_id\"=>\"SO:0001564\", \"description\"=>\"A sequence variant where the structure of the gene is changed.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001564\"}, {\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>25.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>98205262, \"stop\"=>98270943, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000331920.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4645/summary/variants/301/summary#variant\", \"evidence_items\"=>[{\"id\"=>5978, \"name\"=>\"EID5978\", \"description\"=>\"The phase 2a MyPathway study assigned patients with HER2, EGFR, BRAF or SHH alterations to treatment with pertuzumab plus trastuzumab, erlotinib, vemurafenib, or vismodegib, respectively.\\nTwenty-one patients had mutations in the Hedgehog pathway (PTCH-1, n = 18; SMO, n = 3). Three patients with PTCH1 mutations had PRs to vismodegib (17%; unknown primary cancer, n = 1; squamous skin cancer, n = 1; salivary gland cancer, n = 1).\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>190, \"name\"=>\"Vismodegib\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2414, \"name\"=>\"Targeted Therapy for Advanced Solid Tumors on the Basis of Molecular Profiles: Results From MyPathway, an Open-Label, Phase IIa Multiple Basket Study.\", \"citation\"=>\"Hainsworth et al., 2018, J. Clin. Oncol.\", \"pubmed_id\"=>\"29320312\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/29320312\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2018, \"month\"=>1, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[{\"nct_id\"=>\"NCT02091141\", \"name\"=>\"My Pathway: A Study Evaluating Herceptin/Perjeta, Tarceva, Zelboraf/Cotellic, Erivedge, Alecensa, and Tecentriq Treatment Targeted Against Certain Molecular Alterations in Participants With Advanced Solid Tumors\", \"description\"=>\"This multicenter, non-randomized, open-label study will evaluate the efficacy and safety of six treatment regimens in participants with advanced solid tumors for whom therapies that will convey clinical benefit are not available and/or are not suitable options per the treating physician's judgment.\", \"clinical_trial_url\"=>\"https://clinicaltrials.gov/show/NCT02091141\"}]}, \"variant_id\"=>301, \"phenotypes\"=>[]}, {\"id\"=>748, \"name\"=>\"EID748\", \"description\"=>\"133 sonic-hedghog driven medulloblastomas were sequenced for alterations (WGS or WES). 60 cases exhibited PTCH1 mutations, most of which were likely loss-of-function (spread across the gene and of types including: stop gain, splice site, frameshift, etc.). Functional models also showed PTCH1 mutant xenografts to be sensitive to SHH inhibition with vismodegib.\", \"disease\"=>{\"id\"=>97, \"name\"=>\"Brain Medulloblastoma\", \"display_name\"=>\"Brain Medulloblastoma\", \"doid\"=>\"0060105\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060105\"}, \"drugs\"=>[{\"id\"=>190, \"name\"=>\"Vismodegib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>494, \"name\"=>\"Genome sequencing of SHH medulloblastoma predicts genotype-related response to smoothened inhibition.\", \"citation\"=>\"Kool et al., 2014, Cancer Cell\", \"pubmed_id\"=>\"24651015\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24651015\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4493053\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>3, \"day\"=>17}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>301, \"phenotypes\"=>[]}]}, {\"id\"=>327, \"entrez_name\"=>\"PTPRD\", \"entrez_id\"=>5789, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>4692, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}, {\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>8314246, \"stop\"=>10033790, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000381196.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4692/summary/variants/327/summary#variant\", \"evidence_items\"=>[{\"id\"=>807, \"name\"=>\"EID807\", \"description\"=>\"In this preclinical study, loss-of-function of PTPRD was mainly caused by mutation in head and neck cancer, as identified in TCGA data. PTPRD mutated cell-lines were more sensitive to STAT3 inhibition (JSI-124) than wild-type cells. The authors generated several representative HNSCC-derived PTPRD mutants by site-directed mutagenesis (S384R, K1502M, T1100M and L1147F).\", \"disease\"=>{\"id\"=>109, \"name\"=>\"Head And Neck Carcinoma\", \"display_name\"=>\"Head And Neck Carcinoma\", \"doid\"=>\"1542\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1542\"}, \"drugs\"=>[{\"id\"=>154, \"name\"=>\"JSI-124\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>541, \"name\"=>\"Loss-of-Function PTPRD Mutations Lead to Increased STAT3 Activation and Sensitivity to STAT3 Inhibition in Head and Neck Cancer.\", \"citation\"=>\"Peyser et al., 2015, PLoS ONE\", \"pubmed_id\"=>\"26267899\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26267899\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4534317\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>327, \"phenotypes\"=>[]}]}, {\"id\"=>370, \"entrez_name\"=>\"MRE11\", \"entrez_id\"=>4361, \"name\"=>\"LOSS\", \"description\"=>\"\", \"gene_id\"=>3586, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"11\", \"start\"=>94152895, \"stop\"=>94227074, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000323929.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3586/summary/variants/370/summary#variant\", \"evidence_items\"=>[{\"id\"=>877, \"name\"=>\"EID877\", \"description\"=>\"One patient in a cohort of 50 patients with metastatic prostate cancer had a frameshift mutation in the MRE11 gene and showed clinical response to PARP inhibitor olaparib.\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"rejected\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>409, \"name\"=>\"DNA-Repair Defects and Olaparib in Metastatic Prostate Cancer.\", \"citation\"=>\"Mateo et al., 2015, N. Engl. J. Med.\", \"pubmed_id\"=>\"26510020\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26510020\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>10, \"day\"=>29}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>370, \"phenotypes\"=>[]}, {\"id\"=>876, \"name\"=>\"EID876\", \"description\"=>\"One endometrial cell line with decreased MRE11 expression caused by a homozygous poly(T) mutation exhibited increased sensitivity to the PARP inhibitor BMN-673. Loss of MRE11 was identified in 30.7% of endometrial tumors (N=521).\", \"disease\"=>{\"id\"=>108, \"name\"=>\"Endometrial Cancer\", \"display_name\"=>\"Endometrial Cancer\", \"doid\"=>\"1380\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1380\"}, \"drugs\"=>[{\"id\"=>168, \"name\"=>\"BMN-673\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>590, \"name\"=>\"Effect of MRE11 loss on PARP-inhibitor sensitivity in endometrial cancer in vitro.\", \"citation\"=>\"Koppensteiner et al., 2014, PLoS ONE\", \"pubmed_id\"=>\"24927325\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24927325\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4057395\", \"publication_date\"=>{\"year\"=>2014}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>370, \"phenotypes\"=>[]}]}, {\"id\"=>436, \"entrez_name\"=>\"VHL\", \"entrez_id\"=>7428, \"name\"=>\"LOSS\", \"description\"=>\"\", \"gene_id\"=>58, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>23.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>10182692, \"stop\"=>10193904, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000256474.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/58/summary/variants/436/summary#variant\", \"evidence_items\"=>[{\"id\"=>4830, \"name\"=>\"EID4830\", \"description\"=>\"A meta-analysis to evaluate the association between the VHL alteration and clinical outcomes in patients with RCC. Six studies (total of 633 patients) were included. 61,8% had a VHL mutation. The meta-analysis showed no association between the VHL gene alteration and overall response rate or progression free survival. There was also no correlation to overall survival. VEGFR inhibition included: Sunitinib, Sorafenib, Axitinib, Pazopanib, Bevacizumab.\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[{\"id\"=>521, \"name\"=>\"VEGF Inhibition\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2139, \"name\"=>\"Prognostic and predictive value of VHL gene alteration in renal cell carcinoma: a meta-analysis and review.\", \"citation\"=>\"Kim et al., 2017, Oncotarget\", \"pubmed_id\"=>\"28103578\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28103578\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5355155\", \"publication_date\"=>{\"year\"=>2017, \"month\"=>2, \"day\"=>21}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>436, \"phenotypes\"=>[]}, {\"id\"=>1035, \"name\"=>\"EID1035\", \"description\"=>\"Preclinical studies linking the efficacy of mTOR inhibitor CCI-779 to VHL loss in vitro and in mouse models of kidney cancer. VHL-deficient tumors were more sensitive to mTOR inhibition than VHL-proficient tumors. Growth arrest caused by CCI-779 correlated with a block in translation of HIF1A mRNA. Expression of a VHL-resistant HIF1A cDNA lacking the 5' untranslated region rescued tumor growth.\", \"disease\"=>{\"id\"=>161, \"name\"=>\"Renal Carcinoma\", \"display_name\"=>\"Renal Carcinoma\", \"doid\"=>\"4451\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4451\"}, \"drugs\"=>[{\"id\"=>83, \"name\"=>\"Temsirolimus\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>719, \"name\"=>\"Hypoxia-inducible factor determines sensitivity to inhibitors of mTOR in kidney cancer.\", \"citation\"=>\"Thomas et al., 2006, Nat. Med.\", \"pubmed_id\"=>\"16341243\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16341243\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2006, \"month\"=>1}, \"journal\"=>\"Nat. Med.\", \"full_journal_title\"=>\"Nature medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>436, \"phenotypes\"=>[]}]}, {\"id\"=>485, \"entrez_name\"=>\"STK11\", \"entrez_id\"=>6794, \"name\"=>\"LOSS\", \"description\"=>\"\", \"gene_id\"=>5534, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>23.0, \"coordinates\"=>{\"chromosome\"=>\"19\", \"start\"=>1205740, \"stop\"=>1228428, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000326873.7\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5534/summary/variants/485/summary#variant\", \"evidence_items\"=>[{\"id\"=>1618, \"name\"=>\"EID1618\", \"description\"=>\"Preclinical study in a mouse model of Peutz-Jeghers syndrome. Heterozygous Lkb1 (STK11) knockout mice were generated and treated with rapamycin after the onset of polyposis at 9 months of age. Rapamycin treatment significantly suppressed polyposis (p=0.017) and pS6 staining (IHC, p=0.011).\", \"disease\"=>{\"id\"=>2138, \"name\"=>\"Peutz-Jeghers Syndrome\", \"display_name\"=>\"Peutz-Jeghers Syndrome\", \"doid\"=>\"3852\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3852\"}, \"drugs\"=>[{\"id\"=>352, \"name\"=>\"Rapamycin (Sirolimus)\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1070, \"name\"=>\"Suppression of Peutz-Jeghers polyposis by targeting mammalian target of rapamycin signaling.\", \"citation\"=>\"Wei et al., 2008, Clin. Cancer Res.\", \"pubmed_id\"=>\"18281551\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18281551\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2008, \"month\"=>2, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>485, \"phenotypes\"=>[]}, {\"id\"=>1620, \"name\"=>\"EID1620\", \"description\"=>\"Preclinical study in 5 NSCLC cell lines. Two LKB1 (STK11) mutant cell lines had higher basal levels of mTOR activity (pS6K and p4EBP1 levels) but were not more sensitive to mTOR inhibitors (everolimus or rapamycin) than two wild type cell lines. Knockdown of LKB1 in one additional cell line did not lead to an increase in sensitivity to mTOR inhibitors.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}, {\"id\"=>352, \"name\"=>\"Rapamycin (Sirolimus)\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1072, \"name\"=>\"LKB1 gene inactivation does not sensitize non-small cell lung cancer cells to mTOR inhibitors in vitro.\", \"citation\"=>\"Xiao et al., 2015, Acta Pharmacol. Sin.\", \"pubmed_id\"=>\"26027660\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26027660\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4561965\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>9}, \"journal\"=>\"Acta Pharmacol. Sin.\", \"full_journal_title\"=>\"Acta pharmacologica Sinica\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>485, \"phenotypes\"=>[]}, {\"id\"=>1619, \"name\"=>\"EID1619\", \"description\"=>\"Treatment of mice with a heterozygous loss of Lkb1 (STK11) with rapamycin led to reduction in polyp size (p < 0.0001) and number of polyps (p=0.00022). pS6 staining was also reduced after rapamycin treatment. Rapamycin treatment abolished FDG-PET signal on imaging.\", \"disease\"=>{\"id\"=>2138, \"name\"=>\"Peutz-Jeghers Syndrome\", \"display_name\"=>\"Peutz-Jeghers Syndrome\", \"doid\"=>\"3852\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3852\"}, \"drugs\"=>[{\"id\"=>352, \"name\"=>\"Rapamycin (Sirolimus)\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1071, \"name\"=>\"mTOR and HIF-1alpha-mediated tumor metabolism in an LKB1 mouse model of Peutz-Jeghers syndrome.\", \"citation\"=>\"Shackelford et al., 2009, Proc. Natl. Acad. Sci. U.S.A.\", \"pubmed_id\"=>\"19541609\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19541609\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2708689\", \"publication_date\"=>{\"year\"=>2009, \"month\"=>7, \"day\"=>7}, \"journal\"=>\"Proc. Natl. Acad. Sci. U.S.A.\", \"full_journal_title\"=>\"Proceedings of the National Academy of Sciences of the United States of America\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>485, \"phenotypes\"=>[]}, {\"id\"=>1144, \"name\"=>\"EID1144\", \"description\"=>\"A clinical trial comparing selumetinib and docetaxel vs. docetaxel and placebo in KRAS mutant NSCLC was recapitulated in mice. Tumors were induced in lung epithelium by nasal instillation of CRE-bearing adenovirus in conditionally targeted mice. Kras(G12D) and Lkb1 (STK11) knockout mutant mice showed only mild response to combined treatment and very low response rates to docetaxel monotherapy compared to mice with Kras(G12D) alone or Kras(G12D) and Trp53 knockout. In a small number of human NSCLC patients with these genotypes FDG-PET signal intensity changes and pERK IHC staining correlated with mouse data.\", \"disease\"=>{\"id\"=>8, \"name\"=>\"Non-small Cell Lung Carcinoma\", \"display_name\"=>\"Non-small Cell Lung Carcinoma\", \"doid\"=>\"3908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3908\"}, \"drugs\"=>[{\"id\"=>73, \"name\"=>\"Docetaxel\", \"pubchem_id\"=>nil}, {\"id\"=>122, \"name\"=>\"Selumetinib (AZD6244)\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>792, \"name\"=>\"A murine lung cancer co-clinical trial identifies genetic modifiers of therapeutic response.\", \"citation\"=>\"Chen et al., 2012, Nature\", \"pubmed_id\"=>\"22425996\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22425996\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3385933\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>3, \"day\"=>29}, \"journal\"=>\"Nature\", \"full_journal_title\"=>\"Nature\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>485, \"phenotypes\"=>[]}, {\"id\"=>1737, \"name\"=>\"EID1737\", \"description\"=>\"In a series of 33 patients with Peutz-Jeghers Syndrome, 17 mutations in STK11/LKB1 were observed in exons 1-8 (none in exon 9). 11 of these mutations were predicted to be truncating. These mutations occurred in 10/13 patients with familial PJS and 7/20 patients with sporadic PJS.\", \"disease\"=>{\"id\"=>2138, \"name\"=>\"Peutz-Jeghers Syndrome\", \"display_name\"=>\"Peutz-Jeghers Syndrome\", \"doid\"=>\"3852\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3852\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Positive\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1181, \"name\"=>\"Further observations on LKB1/STK11 status and cancer risk in Peutz-Jeghers syndrome.\", \"citation\"=>\"Lim et al., 2003, Br. J. Cancer\", \"pubmed_id\"=>\"12865922\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/12865922\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2394252\", \"publication_date\"=>{\"year\"=>2003, \"month\"=>7, \"day\"=>21}, \"journal\"=>\"Br. J. Cancer\", \"full_journal_title\"=>\"British journal of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>485, \"phenotypes\"=>[]}]}, {\"id\"=>532, \"entrez_name\"=>\"PALB2\", \"entrez_id\"=>79728, \"name\"=>\"BIALLELIC INACTIVATION\", \"description\"=>\"\", \"gene_id\"=>15013, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>17.5, \"coordinates\"=>{\"chromosome\"=>\"16\", \"start\"=>23614491, \"stop\"=>23652631, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000261584.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/15013/summary/variants/532/summary#variant\", \"evidence_items\"=>[{\"id\"=>1963, \"name\"=>\"EID1963\", \"description\"=>\"From 14 prosatate cancer patients (88%) who had a response to olaparib, 7 patients had BRCA2 loss (4 with biallelic somatic loss, and 3 with germline mutations), 4 of 5 with had ATM aberrations and one had a biallelic inactivation of PALB2 (frameshift mutation and deletion).\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>409, \"name\"=>\"DNA-Repair Defects and Olaparib in Metastatic Prostate Cancer.\", \"citation\"=>\"Mateo et al., 2015, N. Engl. J. Med.\", \"pubmed_id\"=>\"26510020\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26510020\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>10, \"day\"=>29}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>532, \"phenotypes\"=>[]}, {\"id\"=>1305, \"name\"=>\"EID1305\", \"description\"=>\"In a patient with gemcitabine-refractory pancreatic cancer, a tumor-derived xenograft was found to be responsive to Mitomycin C. After disease progression, the patient was started on a regimen of mitomycin C, and the patient was shown to be asymptomatic after 3 years of follow-up.\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>325, \"name\"=>\"Mitomycin C\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>898, \"name\"=>\"Personalizing cancer treatment in the age of global genomic analyses: PALB2 gene mutations and the response to DNA damaging agents in pancreatic cancer.\", \"citation\"=>\"Villarroel et al., 2011, Mol. Cancer Ther.\", \"pubmed_id\"=>\"21135251\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21135251\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3307340\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>1}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>532, \"phenotypes\"=>[]}]}, {\"id\"=>534, \"entrez_name\"=>\"FANCC\", \"entrez_id\"=>2176, \"name\"=>\"LOSS-OF-FUNCTION\", \"description\"=>\"\", \"gene_id\"=>1811, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>97861336, \"stop\"=>98079984, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000289081.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1811/summary/variants/534/summary#variant\", \"evidence_items\"=>[{\"id\"=>1307, \"name\"=>\"EID1307\", \"description\"=>\"In the pancreatic cell lines PL11 (FANCC homozygous deletion) and Hs766T (FANCG-mutated), sensitivity to DNA cross-linking agents was observed. PL11 cells (empty vector transduced) were more sensitive to mitomycin C (MMC), cisplatin, chlorambucil, and melphalan than PL11 cells transduced with a vector wildtype FANCC. Similarly, mouse xenografts of PL11 cells (empty vector) were more sensitive to MMC or gemcitabine than PL11 cells expressing a wildtype FANCC.\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[{\"id\"=>85, \"name\"=>\"Gemcitabine\", \"pubchem_id\"=>nil}, {\"id\"=>325, \"name\"=>\"Mitomycin C\", \"pubchem_id\"=>nil}, {\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}, {\"id\"=>327, \"name\"=>\"Chlorambucil\", \"pubchem_id\"=>nil}, {\"id\"=>328, \"name\"=>\"Melphalan\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Unknown\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>900, \"name\"=>\"In vivo therapeutic responses contingent on Fanconi anemia/BRCA2 status of the tumor.\", \"citation\"=>\"van der Heijden et al., 2005, Clin. Cancer Res.\", \"pubmed_id\"=>\"16243825\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/16243825\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2005, \"month\"=>10, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>534, \"phenotypes\"=>[]}]}, {\"id\"=>554, \"entrez_name\"=>\"CDKN2A\", \"entrez_id\"=>1029, \"name\"=>\"LOSS\", \"description\"=>\"\", \"gene_id\"=>14, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>22.5, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>21967752, \"stop\"=>21994623, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000579755.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/14/summary/variants/554/summary#variant\", \"evidence_items\"=>[{\"id\"=>1881, \"name\"=>\"EID1881\", \"description\"=>\"An imatinib-resistant cell line, DFSP105, was established from a dermatofibrosarcoma protuberans which had progressed to a high-grade fibrosarcomatous DFSP. The disease-associated PDGFB-COL1A1 fusion was consistent with the most common form, a supernumerary ring chromosome. Genetic characterization of the cell line with a SNP array identified a homozygous CDKN2A/B deletion. Treatment of this cell line with CDK4/6 inhibitor palbociclib led to growth inhibition and reduced RB1 phosphorylation in vitro and in-vivo.\", \"disease\"=>{\"id\"=>142, \"name\"=>\"Dermatofibrosarcoma Protuberans\", \"display_name\"=>\"Dermatofibrosarcoma Protuberans\", \"doid\"=>\"3507\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3507\"}, \"drugs\"=>[{\"id\"=>429, \"name\"=>\"Palbociclib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Unknown\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1306, \"name\"=>\"CDKN2A/p16 Loss Implicates CDK4 as a Therapeutic Target in Imatinib-Resistant Dermatofibrosarcoma Protuberans.\", \"citation\"=>\"Eilers et al., 2015, Mol. Cancer Ther.\", \"pubmed_id\"=>\"25852058\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25852058\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4458458\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>6}, \"journal\"=>\"Mol. Cancer Ther.\", \"full_journal_title\"=>\"Molecular cancer therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>554, \"phenotypes\"=>[]}, {\"id\"=>1559, \"name\"=>\"EID1559\", \"description\"=>\"Short term explant cultures from 20 glioblastoma multiforme (GBM) tumor xenograft lines were evaluated for CDKN2A (p16 aka INK4A) expression by western blot and RT-PCR as well as deletion by aCGH and Sanger sequencing. Two lines with intact CDKN2A showed resistance to the CDK4/6 inhibitor palbociclib (PD0332991). The two resistant lines had additional mutations (RB1 mutation or CDK4 amplification).  In contrast, three CDKN2A mutant (homozygous deletion) lines showed significant growth inhibition in comparison to vehicle/DMSO controls.  Screening of another 20 lines identified one with CDKN2A deletion and wildtype CDK4 which was tested in vitro and in vivo, and found to be sensitive to palbociclib.\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1030, \"name\"=>\"p16-Cdk4-Rb axis controls sensitivity to a cyclin-dependent kinase inhibitor PD0332991 in glioblastoma xenograft cells.\", \"citation\"=>\"Cen et al., 2012, Neuro-oncology\", \"pubmed_id\"=>\"22711607\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22711607\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3379801\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>7}, \"journal\"=>\"Neuro-oncology\", \"full_journal_title\"=>\"Neuro-oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>554, \"phenotypes\"=>[]}, {\"id\"=>1879, \"name\"=>\"EID1879\", \"description\"=>\"A patient-derived xenograft (PDX) from a patient with cyclin-dependent kinase inhibitor 2A/B (CDKN2A/B) loss and FUS-ERG fusion positive Ewing's sarcoma was created. The PDX was treated in four cohorts (each N=6) with control, doxorubicin, CDK4/6 inhibitor palbociclib or IGF-1R inhibitor linsitinib. Both linsitinib and palbociclib cohorts were significantly more effective than control and doxorubicin. Doxorubicin was part of the patient's treatment (Neoadjuvant chemotherapy with DOX, vincristine, and cyclophosphamide) and was noted by authors as failing to control tumor growth.\", \"disease\"=>{\"id\"=>50, \"name\"=>\"Ewing Sarcoma\", \"display_name\"=>\"Ewing Sarcoma\", \"doid\"=>\"3369\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3369\"}, \"drugs\"=>[{\"id\"=>429, \"name\"=>\"Palbociclib\", \"pubchem_id\"=>nil}, {\"id\"=>430, \"name\"=>\"Linsitinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1305, \"name\"=>\"Effective molecular targeting of CDK4/6 and IGF-1R in a rare FUS-ERG fusion CDKN2A-deletion doxorubicin-resistant Ewing's sarcoma patient-derived orthotopic xenograft (PDOX) nude-mouse model.\", \"citation\"=>\"Murakami et al., 2016, Oncotarget\", \"pubmed_id\"=>\"27286459\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27286459\", \"open_access\"=>true, \"pmc_id\"=>\"PMC5216960\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>7, \"day\"=>26}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>554, \"phenotypes\"=>[]}, {\"id\"=>1765, \"name\"=>\"EID1765\", \"description\"=>\"A patient with metastatic ER-positive, her2-negative breast cancer with CDKN2A loss experienced a short-lived (~3-month) clinical response to palbociclib when given in combination with letrozole.\", \"disease\"=>{\"id\"=>369, \"name\"=>\"Her2-receptor Negative Breast Cancer\", \"display_name\"=>\"Her2-receptor Negative Breast Cancer\", \"doid\"=>\"0060080\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060080\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}, {\"id\"=>375, \"name\"=>\"Letrozole\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1231, \"name\"=>\"Does CDKN2A loss predict palbociclib benefit?\", \"citation\"=>\"Gao et al., 2015, Curr Oncol\", \"pubmed_id\"=>\"26715889\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26715889\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4687677\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>12}, \"journal\"=>\"Curr Oncol\", \"full_journal_title\"=>\"Current oncology (Toronto, Ont.)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>554, \"phenotypes\"=>[]}, {\"id\"=>1372, \"name\"=>\"EID1372\", \"description\"=>\"Preclinical study in 9 melanoma cell lines. Three cell lines did not have CDKN2A (p16) mutations and showed expression of the protein on western blot analysis. Two of the other six cell lines had stop codons leading to a truncated protein, and three had missense mutations with evidence of protein expression on western blot (two cell lines with a P114L mutation and one with an E88K mutation). Mean IC50s of flavopiridol were significantly lower in cell lines with a mutated or non-expressed p16 (p16-) (mean 50% inhibitory concentration [IC(50)] = 12.5 nM), than in cell lines with intact p16 (p16+) (mean IC(50) = 25 nM).\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>335, \"name\"=>\"Flavopiridol\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>931, \"name\"=>\"The effect of flavopiridol on the growth of p16+ and p16- melanoma cell lines.\", \"citation\"=>\"Robinson et al., 2003, Melanoma Res.\", \"pubmed_id\"=>\"12777976\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/12777976\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2003, \"month\"=>6}, \"journal\"=>\"Melanoma Res.\", \"full_journal_title\"=>\"Melanoma research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>554, \"phenotypes\"=>[]}, {\"id\"=>1373, \"name\"=>\"EID1373\", \"description\"=>\"Preclinical study of CDK4/6 inhibitor PD-0332991 in 28 renal carcinoma cell lines (25 malignant RCC and three non-malignant, transformed cell lines derived from kidney epithelium). Both p15 and p16 loss (CGH) were tested for association with response to PD-0332991 using Fisher's exact test. Cell lines with p16 and p15 losses were significantly more likely to be classified as responders (defined by pharmacokinetic data-based concentration of <=400nM) to PD-0332991 (p=0.027, relative risk (RR)=2.06 and p=0.047, RR=1.71, respectively). Four cell lines had homozygous deletions of CDKN2A but not CDKN2B, and three of these lines were classified as being sensitive to PD-0332991. There were no cell lines with only CDKN2B deletion.\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>212, \"name\"=>\"PD-0332991, a potent and selective inhibitor of cyclin-dependent kinase 4/6, demonstrates inhibition of proliferation in renal cell carcinoma at nanomolar concentrations and molecular markers predict for sensitivity.\", \"citation\"=>\"Logan et al., 2013, Anticancer Res.\", \"pubmed_id\"=>\"23898052\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23898052\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>8}, \"journal\"=>\"Anticancer Res.\", \"full_journal_title\"=>\"Anticancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>554, \"phenotypes\"=>[]}, {\"id\"=>1375, \"name\"=>\"EID1375\", \"description\"=>\"In a cohort of 143 patients with primary invasive melanoma hemizygous or homozygous loss of CDKN2A was observed (FISH) in 56% of cases. PD0332991 efficacy (proliferation assay) was assessed in a panel of 47 melanoma cell lines. Loss of functional p16INK4A (CDKN2A) due to gene deletion (49% of cells), methylation (4% of cells), or mutation (19% of cells) was the most prevalent alteration occurring in 71% of cell lines, and significantly correlated with PD0332991 sensitivity (Fisher's exact test P < 0.02).\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>932, \"name\"=>\"Loss of CDKN2A expression is a frequent event in primary invasive melanoma and correlates with sensitivity to the CDK4/6 inhibitor PD0332991 in melanoma cell lines.\", \"citation\"=>\"Young et al., 2014, Pigment Cell Melanoma Res\", \"pubmed_id\"=>\"24495407\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24495407\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>7}, \"journal\"=>\"Pigment Cell Melanoma Res\", \"full_journal_title\"=>\"Pigment cell & melanoma research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>554, \"phenotypes\"=>[]}, {\"id\"=>1377, \"name\"=>\"EID1377\", \"description\"=>\"Preclinical study of the effects of PD-0332991 on proliferation, cell-cycle, apoptosis, and Rb phosphorylation in a panel of 40 human ovarian cancer cell lines. Concentration-dependent antiproliferative effects of PD-0332991 were seen in all ovarian cancer cell lines, but varied significantly between individual lines. Rb-proficient cell lines with low p16 expression were significantly more sensitive to PD-0332991 (as defined by IC50 < mean) in mRNA expression (Agilent), protein level (Western Blot) and copy number (Array CGH) analyses.\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>933, \"name\"=>\"Expression of p16 and retinoblastoma determines response to CDK4/6 inhibition in ovarian cancer.\", \"citation\"=>\"Konecny et al., 2011, Clin. Cancer Res.\", \"pubmed_id\"=>\"21278246\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21278246\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4598646\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>3, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>554, \"phenotypes\"=>[]}]}, {\"id\"=>564, \"entrez_name\"=>\"SMAD4\", \"entrez_id\"=>4089, \"name\"=>\"LOSS\", \"description\"=>\"\", \"gene_id\"=>77, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"18\", \"start\"=>48556583, \"stop\"=>48611409, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000342988.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/77/summary/variants/564/summary#variant\", \"evidence_items\"=>[{\"id\"=>1412, \"name\"=>\"EID1412\", \"description\"=>\"Study employed a rapid autopsy technique on a cohort of 76 patients with pancreatic cancer and explored the status of several genes. 63% (41/65) of carcinomas analyzed showed inactivation of SMAD4 (DPC4) via immunolabeling. Among the patterns of failure \\\"metastatic and locally destructive\\\", It was discovered that DPC4 loss was significantly associated with metastatic disease (p=.007).\", \"disease\"=>{\"id\"=>556, \"name\"=>\"Pancreatic Cancer\", \"display_name\"=>\"Pancreatic Cancer\", \"doid\"=>\"1793\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1793\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>956, \"name\"=>\"DPC4 gene status of the primary carcinoma correlates with patterns of failure in patients with pancreatic cancer.\", \"citation\"=>\"Iacobuzio-Donahue et al., 2009, J. Clin. Oncol.\", \"pubmed_id\"=>\"19273710\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19273710\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2668706\", \"publication_date\"=>{\"year\"=>2009, \"month\"=>4, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>564, \"phenotypes\"=>[]}]}, {\"id\"=>581, \"entrez_name\"=>\"BRAF\", \"entrez_id\"=>673, \"name\"=>\"K483M\", \"description\"=>\"The K483M variant is a kinase dead loss-of-function variant in the catalytic site of BRAF. This engineered variant is often used as a control when evaluating other BRAF variants.\", \"gene_id\"=>5, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>0.75, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>140477859, \"stop\"=>140477860, \"reference_bases\"=>\"TT\", \"variant_bases\"=>\"CA\", \"representative_transcript\"=>\"ENST00000288602.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5/summary/variants/581/summary#variant\", \"evidence_items\"=>[{\"id\"=>1457, \"name\"=>\"EID1457\", \"description\"=>\"Preclinical study in melanoma cell lines. Inactivity of BRAF as mediated by specific mutation (D594A, D594V or K483M) or selective pharmacological inhibition leads to MEK hyperactivation through CRAF binding in NRAS mutant D04 cells. Either pan-RAF (e.g. sorafenib in the abscence of CRAF T421N gatekeeper mutation) or MEK inhibitors (e.g., trametinib) could instead block ERK pathway hyperactivation in RAS mutated cancers.\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>6, \"name\"=>\"Sorafenib\", \"pubchem_id\"=>nil}, {\"id\"=>110, \"name\"=>\"MEK Inhibitor\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"E\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>982, \"name\"=>\"Kinase-dead BRAF and oncogenic RAS cooperate to drive tumor progression through CRAF.\", \"citation\"=>\"Heidorn et al., 2010, Cell\", \"pubmed_id\"=>\"20141835\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20141835\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2872605\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>1, \"day\"=>22}, \"journal\"=>\"Cell\", \"full_journal_title\"=>\"Cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>581, \"phenotypes\"=>[]}]}, {\"id\"=>587, \"entrez_name\"=>\"NF1\", \"entrez_id\"=>4763, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>3867, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}, {\"id\"=>103, \"name\"=>\"protein_altering_variant\", \"display_name\"=>\"Protein Altering Variant\", \"so_id\"=>\"SO:0001818\", \"description\"=>\"A sequence_variant which is predicted to change the protein encoded in the coding sequence.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001818\"}], \"civic_actionability_score\"=>32.0, \"coordinates\"=>{\"chromosome\"=>\"17\", \"start\"=>29421945, \"stop\"=>29704695, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000358273.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3867/summary/variants/587/summary#variant\", \"evidence_items\"=>[{\"id\"=>1958, \"name\"=>\"EID1958\", \"description\"=>\"Treatment with selumetinib (NCT01362803) resulted in confirmed partial, long term responses in 17 of the 24 children with NF1 associated plexiform fibromas (71%) and decreases from baseline in neurofibroma volume in 12 of 18 mice (67%).\", \"disease\"=>{\"id\"=>1215, \"name\"=>\"Plexiform Neurofibroma\", \"display_name\"=>\"Plexiform Neurofibroma\", \"doid\"=>\"5151\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5151\"}, \"drugs\"=>[{\"id\"=>122, \"name\"=>\"Selumetinib (AZD6244)\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1377, \"name\"=>\"Activity of Selumetinib in Neurofibromatosis Type 1-Related Plexiform Neurofibromas.\", \"citation\"=>\"Dombi et al., 2016, N. Engl. J. Med.\", \"pubmed_id\"=>\"28029918\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28029918\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"day\"=>29}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>587, \"phenotypes\"=>[]}, {\"id\"=>1469, \"name\"=>\"EID1469\", \"description\"=>\"Preclinical study of mouse melanoma both in-vitro and in-vivo. Braf/Nf1-mutant melanomas were less sensitive to vemurafenib than the MEK inhibitor PD0325901. mTOR inhibition with rapamycin had a more pronounced effect than GDC-0941 on mouse melanoma, but synergized with PD0325901(but not vemurafenib) to promote tumor regression.\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>29, \"name\"=>\"PD0325901\", \"pubchem_id\"=>nil}, {\"id\"=>352, \"name\"=>\"Rapamycin (Sirolimus)\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>985, \"name\"=>\"Elucidating distinct roles for NF1 in melanomagenesis.\", \"citation\"=>\"Maertens et al., 2013, Cancer Discov\", \"pubmed_id\"=>\"23171796\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23171796\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3595355\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>3}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>587, \"phenotypes\"=>[]}, {\"id\"=>1468, \"name\"=>\"EID1468\", \"description\"=>\"Preclinical study of mouse melanoma in-vitro and in-vivo. In-vitro treament of Braf-mutant melanomas were 10-fold more sensitive to vermurafenib (PLX4032) treatment than Braf/Nf1-mutant melanomas. In vivo treatment of mice with Braf/Nf1-mutant allografts showed less sensitivity to vemurafenib (PLX4032) than the MEK inhibitor, PD0325901.\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>4, \"name\"=>\"Vemurafenib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>985, \"name\"=>\"Elucidating distinct roles for NF1 in melanomagenesis.\", \"citation\"=>\"Maertens et al., 2013, Cancer Discov\", \"pubmed_id\"=>\"23171796\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23171796\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3595355\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>3}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>587, \"phenotypes\"=>[]}, {\"id\"=>1471, \"name\"=>\"EID1471\", \"description\"=>\"NF1 was identified in an unbiased RNAi screen as mediator of resistance to BRAF inhibition in BRAF mutant melanoma. BRAF V600E/NF1 mutant cell lines were sensitive to ERK inhibition with VTX-11e, modest resistance was observed against RAF inhibitor AZ628. Combined PLX4720 and AZD6244 (selumetinib) achieved greater efficacy than either agent alone but NF1 knockdown was still associated with residual resistance to combined RAF/MEK inhibition in vitro.\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>353, \"name\"=>\"VTX-11e\", \"pubchem_id\"=>nil}, {\"id\"=>354, \"name\"=>\"AZ628\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>986, \"name\"=>\"A genome-scale RNA interference screen implicates NF1 loss in resistance to RAF inhibition.\", \"citation\"=>\"Whittaker et al., 2013, Cancer Discov\", \"pubmed_id\"=>\"23288408\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23288408\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3606893\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>3}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>587, \"phenotypes\"=>[]}, {\"id\"=>1470, \"name\"=>\"EID1470\", \"description\"=>\"NF1 was identified in an unbiased RNAi screen as mediator of resistance to BRAF inhibition in BRAF mutant melanoma. Knockdown of NF1 led to PLX4720 resistance in-vitro. MEK inhibition with AZD6244 (selumetinib) had some efficacy but was also less active than in NF1 proficient cells.  4 patients with mutant alleles of NF1 were identified among patients relapsed after vemurafenib. Of these, one was a nonsense mutation (p.R2450*) present in both pretreatment and postrelapse biopsies. The patient had a progression-free survival (PFS) of about 10 weeks. The other NF1 mutations were silent events (patient 15, c.135T>C, patient 45 c.4023G>A, and patient 50 c3018C>T) located within candidate splicing motifs. Patients 15 and 50 exhibited response rates of relatively short duration (10 and 12 weeks, respectively) and the mutations were observed in both the pretreatment and postrelapse biopsies. Patient 45 experienced a PFS of 5 months, and the NF1 mutation was only present in the postrelapse biopsy indicative of acquired resistance.\", \"disease\"=>{\"id\"=>206, \"name\"=>\"Skin Melanoma\", \"display_name\"=>\"Skin Melanoma\", \"doid\"=>\"8923\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:8923\"}, \"drugs\"=>[{\"id\"=>4, \"name\"=>\"Vemurafenib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>986, \"name\"=>\"A genome-scale RNA interference screen implicates NF1 loss in resistance to RAF inhibition.\", \"citation\"=>\"Whittaker et al., 2013, Cancer Discov\", \"pubmed_id\"=>\"23288408\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23288408\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3606893\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>3}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>587, \"phenotypes\"=>[]}]}, {\"id\"=>594, \"entrez_name\"=>\"CEBPA\", \"entrez_id\"=>1050, \"name\"=>\"BIALLELIC INACTIVATION\", \"description\"=>\"\", \"gene_id\"=>15, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>25.0, \"coordinates\"=>{\"chromosome\"=>\"19\", \"start\"=>33790840, \"stop\"=>33793470, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000498907.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/15/summary/variants/594/summary#variant\", \"evidence_items\"=>[{\"id\"=>1512, \"name\"=>\"EID1512\", \"description\"=>\"In a study of 1,540 patients with Acute Myeloid Leukemia, 66 patients (4%) had a CEBPA (biallelic) mutation.  In the absence of gene fusion products, patients with biallelic CEBPA mutations showed the highest rate of survival (~78%) when compared to IDH2 (R172) mutations, NPM1(W288fs) mutations, chromatin-spliceosome mutations and TP53–aneuploidy.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>5, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1009, \"name\"=>\"Genomic Classification and Prognosis in Acute Myeloid Leukemia.\", \"citation\"=>\"Papaemmanuil et al., 2016, N. Engl. J. Med.\", \"pubmed_id\"=>\"27276561\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27276561\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4979995\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>6, \"day\"=>9}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>594, \"phenotypes\"=>[]}]}, {\"id\"=>601, \"entrez_name\"=>\"NOTCH1\", \"entrez_id\"=>4851, \"name\"=>\"LOSS-OF-FUNCTION\", \"description\"=>\"\", \"gene_id\"=>50, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>2.0, \"coordinates\"=>{\"chromosome\"=>\"9\", \"start\"=>139388896, \"stop\"=>139440314, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000277541.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/50/summary/variants/601/summary#variant\", \"evidence_items\"=>[{\"id\"=>1525, \"name\"=>\"EID1525\", \"description\"=>\"In an evaluation of 96 head and neck squamous cell carcinoma (HNSCC) cell lines, 31 were responsive to LGK974 (PORCN inhibitor). Of 40 cell lines (25 responsive, 15 non-responsive) that were exome sequenced, 6 cell lines had NOTCH1 loss-of-function mutations (N-terminal frameshift/nonsense) and 5/6 cell lines that had NOTCH1 mutations were susceptible to LGK974 (PORCN inhibitor).\", \"disease\"=>{\"id\"=>37, \"name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"display_name\"=>\"Head And Neck Squamous Cell Carcinoma\", \"doid\"=>\"5520\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5520\"}, \"drugs\"=>[{\"id\"=>370, \"name\"=>\"LGK974\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1017, \"name\"=>\"Targeting Wnt-driven cancer through the inhibition of Porcupine by LGK974.\", \"citation\"=>\"Liu et al., 2013, Proc. Natl. Acad. Sci. U.S.A.\", \"pubmed_id\"=>\"24277854\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24277854\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3864356\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>12, \"day\"=>10}, \"journal\"=>\"Proc. Natl. Acad. Sci. U.S.A.\", \"full_journal_title\"=>\"Proceedings of the National Academy of Sciences of the United States of America\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>601, \"phenotypes\"=>[]}]}, {\"id\"=>606, \"entrez_name\"=>\"RB1\", \"entrez_id\"=>5925, \"name\"=>\"LOSS-OF-FUNCTION\", \"description\"=>\"\", \"gene_id\"=>4795, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>6.0, \"coordinates\"=>{\"chromosome\"=>\"13\", \"start\"=>48877911, \"stop\"=>49056122, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000267163.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4795/summary/variants/606/summary#variant\", \"evidence_items\"=>[{\"id\"=>1535, \"name\"=>\"EID1535\", \"description\"=>\"In 3 ER positive and 2 ER negative breast cancer cell lines with intact Rb, significant growth inhibition, as measured by BrdU incorporation, was seen with incubation in CDK4/6 inhibitor palbociclib (PD-0332991). In contrast, two ER negative cell lines lacking detectable Rb were fully resistant to palbociclib. Knockdown of Rb in 3 of the tested Rb positive cell lines led to moderate palbociclib resistance in short term incubation experiments. \\nIn longer term (15 day) palbociclib incubations of ER negative MDA-MB-231 cells and ER positive MCF-7 cells, significant resistance to palbociclib was seen in Rb knockdown conditions while wild-type Rb cells maintained sensitivity.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1022, \"name\"=>\"Therapeutic CDK4/6 inhibition in breast cancer: key mechanisms of response and failure.\", \"citation\"=>\"Dean et al., 2010, Oncogene\", \"pubmed_id\"=>\"20473330\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20473330\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>7, \"day\"=>15}, \"journal\"=>\"Oncogene\", \"full_journal_title\"=>\"Oncogene\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>606, \"phenotypes\"=>[]}, {\"id\"=>1595, \"name\"=>\"EID1595\", \"description\"=>\"A panel of 21 glioblastoma cell lines harboring various mutations was treated with palbociclib. 16 of the cell lines with intact Rb showed decreased proliferation (bromodeoxyuridine incorporation, BrdU flow cytometry) and growth arrest characteristic of senescence, while the 5 cell lines that did not show a palbociclib response had homozygous deletion of Rb. Rb positive and palbociclib responsive GBM cell line U87MG was treated with shRNA against Rb. shRb treated cells showed restoration of cell proliferation (BrdU incorporation) under palbociclib treatment, indicating resistance.\", \"disease\"=>{\"id\"=>19, \"name\"=>\"Glioblastoma Multiforme\", \"display_name\"=>\"Glioblastoma Multiforme\", \"doid\"=>\"3068\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3068\"}, \"drugs\"=>[{\"id\"=>48, \"name\"=>\"Palbociclib (PD0332991)\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1057, \"name\"=>\"Pharmacologic inhibition of cyclin-dependent kinases 4 and 6 arrests the growth of glioblastoma multiforme intracranial xenografts.\", \"citation\"=>\"Michaud et al., 2010, Cancer Res.\", \"pubmed_id\"=>\"20354191\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20354191\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2855904\", \"publication_date\"=>{\"year\"=>2010, \"month\"=>4, \"day\"=>15}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>606, \"phenotypes\"=>[]}]}, {\"id\"=>623, \"entrez_name\"=>\"POT1\", \"entrez_id\"=>25913, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>9935, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}, {\"id\"=>103, \"name\"=>\"protein_altering_variant\", \"display_name\"=>\"Protein Altering Variant\", \"so_id\"=>\"SO:0001818\", \"description\"=>\"A sequence_variant which is predicted to change the protein encoded in the coding sequence.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001818\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>124462440, \"stop\"=>124570035, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000357628.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/9935/summary/variants/623/summary#variant\", \"evidence_items\"=>[{\"id\"=>1590, \"name\"=>\"EID1590\", \"description\"=>\"In previously untreated patients with chronic lymphocytic leukemia, POT1 mutations (N=13) were associated with shorter overall survival than wildtype POT1 (N=148) in univariate and multivariate analysis (HR, 4.0; 95% CI, 1.6-10.1; P = .003) independent of IGHV mutation status, Binet stage, and serum beta-2-microglobulin.\", \"disease\"=>{\"id\"=>52, \"name\"=>\"Chronic Lymphocytic Leukemia\", \"display_name\"=>\"Chronic Lymphocytic Leukemia\", \"doid\"=>\"1040\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1040\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1051, \"name\"=>\"Complex karyotypes and KRAS and POT1 mutations impact outcome in CLL after chlorambucil-based chemotherapy or chemoimmunotherapy.\", \"citation\"=>\"Herling et al., 2016, Blood\", \"pubmed_id\"=>\"27226433\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27226433\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>7, \"day\"=>21}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>623, \"phenotypes\"=>[]}]}, {\"id\"=>637, \"entrez_name\"=>\"FBXW7\", \"entrez_id\"=>55294, \"name\"=>\"LOSS-OF-FUNCTION\", \"description\"=>\"\", \"gene_id\"=>12903, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>4.5, \"coordinates\"=>{\"chromosome\"=>\"4\", \"start\"=>153242410, \"stop\"=>153457253, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000281708.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/12903/summary/variants/637/summary#variant\", \"evidence_items\"=>[{\"id\"=>1632, \"name\"=>\"EID1632\", \"description\"=>\"FBXW7 was shown to target mTOR for ubiquitination and degradation.  A reciprocal relationship between FBXW7 or PTEN loss was identified in breast cancer tumors and cell lines (mutual exclusivity of these two events).  In total only 4 out of 450 tumor and cell line samples had concomitant loss of both genes (p=4.9x10^-7). Cell lines (n=5) with FBXW7 mutations or deletions were significantly sensitive to rapamycin and downregulation of FBXW7 (shRNA) increased the sensitivity to rapamycin. In summary, this evidence suggests that FBXW7 may be a biomarker for cancers sensitivity to inhibitors of the mTOR pathway.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>352, \"name\"=>\"Rapamycin (Sirolimus)\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1080, \"name\"=>\"FBXW7 targets mTOR for degradation and cooperates with PTEN in tumor suppression.\", \"citation\"=>\"Mao et al., 2008, Science\", \"pubmed_id\"=>\"18787170\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/18787170\", \"open_access\"=>true, \"pmc_id\"=>\"PMC2849753\", \"publication_date\"=>{\"year\"=>2008, \"month\"=>9, \"day\"=>12}, \"journal\"=>\"Science\", \"full_journal_title\"=>\"Science (New York, N.Y.)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>637, \"phenotypes\"=>[]}, {\"id\"=>1628, \"name\"=>\"EID1628\", \"description\"=>\"mTOR protein levels were found to underly a circadian 24-hour rhythm regulated by time-dependent Fbxw7-mediated ubiquitination. Survival rates of tumor-bearing mice with everolimus showed time-dependency consistent with mTOR protein levels. It could be hypothesized that FBXW7 loss leads to an increase in mTOR protein levels independent of circadian regulation and higher sensitivity to mTOR inhibitors.\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[{\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"E\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1077, \"name\"=>\"Circadian regulation of mTOR by the ubiquitin pathway in renal cell carcinoma.\", \"citation\"=>\"Okazaki et al., 2014, Cancer Res.\", \"pubmed_id\"=>\"24253377\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24253377\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>1, \"day\"=>15}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>637, \"phenotypes\"=>[]}]}, {\"id\"=>644, \"entrez_name\"=>\"PRDM1\", \"entrez_id\"=>639, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>593, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}, {\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"6\", \"start\"=>106534195, \"stop\"=>106557813, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000369096.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/593/summary/variants/644/summary#variant\", \"evidence_items\"=>[{\"id\"=>1717, \"name\"=>\"EID1717\", \"description\"=>\"In 177 patients with activated B-cell diffuse large B-cell lymphoma (ABC-DLBCL), PRDM1 mutations (N=46) were associated with worse overall (P=0.016) and progression-free (P=0.023) survival compared to wildtype patients. This may have been driven by patients with exon1-2 mutations (N=15; P=0.021 for OS; P=0.024 for PFS) versus those with exon 3-7 mutations (N=31; P=0.16 for OS; P=0.24 for PFS) when compared to wildtype patients. Multivariate analysis did not identify PRDM1 mutations as an independent prognostic factor.\", \"disease\"=>{\"id\"=>45, \"name\"=>\"Diffuse Large B-cell Lymphoma\", \"display_name\"=>\"Diffuse Large B-cell Lymphoma\", \"doid\"=>\"0050745\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050745\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>nil, \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1084, \"name\"=>\"Loss of PRDM1/BLIMP-1 function contributes to poor prognosis of activated B-cell-like diffuse large B-cell lymphoma.\", \"citation\"=>\"Xia et al., 2016, Leukemia\", \"pubmed_id\"=>\"27568520\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27568520\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>9, \"day\"=>30}, \"journal\"=>\"Leukemia\", \"full_journal_title\"=>\"Leukemia\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>644, \"phenotypes\"=>[]}, {\"id\"=>1638, \"name\"=>\"EID1638\", \"description\"=>\"Biopsies from 520 DLBCL patients treated with R-CHOP were evaluated for PRDM1 mutations or protein (BLIMP-1) expression. Patients with a homozygous deletion of PRDM1 (7%, N=19) had a significantly shorter overall survival and poorer progression free survival compared to heterozygous or wildtype PRDM1.\", \"disease\"=>{\"id\"=>45, \"name\"=>\"Diffuse Large B-cell Lymphoma\", \"display_name\"=>\"Diffuse Large B-cell Lymphoma\", \"doid\"=>\"0050745\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050745\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1084, \"name\"=>\"Loss of PRDM1/BLIMP-1 function contributes to poor prognosis of activated B-cell-like diffuse large B-cell lymphoma.\", \"citation\"=>\"Xia et al., 2016, Leukemia\", \"pubmed_id\"=>\"27568520\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27568520\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>9, \"day\"=>30}, \"journal\"=>\"Leukemia\", \"full_journal_title\"=>\"Leukemia\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>644, \"phenotypes\"=>[]}]}, {\"id\"=>672, \"entrez_name\"=>\"SOX10\", \"entrez_id\"=>6663, \"name\"=>\"LOSS\", \"description\"=>\"\", \"gene_id\"=>5418, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>4.0, \"coordinates\"=>{\"chromosome\"=>\"22\", \"start\"=>38368307, \"stop\"=>38380544, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000396884.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5418/summary/variants/672/summary#variant\", \"evidence_items\"=>[{\"id\"=>1710, \"name\"=>\"EID1710\", \"description\"=>\"FACS-assisted shRNA genetic screen identified SOX10 as a determinant of vemurafenib resistance. Suppression of SOX10 in melanoma cells confers resistance to BRAF and MEK inhibitors through activation of TGF-β signalling and upregulation of EGFR and PDGFRB. EGFR expression or exposure to TGF-β becomes beneficial for proliferation of melanoma cells in the presence of BRAF or MEK inhibitors. Melanoma cells with low SOX10 and consequently high EGFR expression are enriched in the presence of vemurafenib. This study finds evidence for SOX10 loss (N=2) and/or activation of TGF-β signalling (N=2) in 4 of the 6 EGFR-positive drug-resistant melanoma patient samples.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>4, \"name\"=>\"Vemurafenib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"N/A\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1153, \"name\"=>\"Reversible and adaptive resistance to BRAF(V600E) inhibition in melanoma.\", \"citation\"=>\"Sun et al., 2014, Nature\", \"pubmed_id\"=>\"24670642\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24670642\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>4, \"day\"=>3}, \"journal\"=>\"Nature\", \"full_journal_title\"=>\"Nature\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>672, \"phenotypes\"=>[]}]}, {\"id\"=>691, \"entrez_name\"=>\"ERRFI1\", \"entrez_id\"=>54206, \"name\"=>\"E384*\", \"description\"=>\"\", \"gene_id\"=>12271, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>50, \"name\"=>\"stop_gained\", \"display_name\"=>\"Stop Gained\", \"so_id\"=>\"SO:0001587\", \"description\"=>\"A sequence variant whereby at least one base of a codon is changed, resulting in a premature stop codon, leading to a shortened polypeptide.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001587\"}, {\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>8073509, \"stop\"=>8073509, \"reference_bases\"=>\"C\", \"variant_bases\"=>\"A\", \"representative_transcript\"=>\"ENST00000377482.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/12271/summary/variants/691/summary#variant\", \"evidence_items\"=>[{\"id\"=>5869, \"name\"=>\"EID5869\", \"description\"=>\"A patient with cholangiocarcinoma who has initially undergone surgical resection progressed on gemcitabine and cisplatin. He subsequently received pegylated hyaluroniade (PEGPH20) in an investigational study and had no response. Finally, he underwent whole genome sequencing of a lymph node. A novel nonsense mutation in ERRFI1, a direct negative regulator of EGFR (E384TER) was detected. The mutation was present in large fraction of tumor RNAseq reads, which the authors suggest indicated nearly complete loss of wildtype function. The patient received erlotinib and after 3 months had a partial response evidenced by tumor shrinkage.\", \"disease\"=>{\"id\"=>29, \"name\"=>\"Cholangiocarcinoma\", \"display_name\"=>\"Cholangiocarcinoma\", \"doid\"=>\"4947\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4947\"}, \"drugs\"=>[{\"id\"=>15, \"name\"=>\"Erlotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"rejected\", \"open_change_count\"=>2, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>186, \"name\"=>\"Integrated genomic characterization reveals novel, therapeutically relevant drug targets in FGFR and EGFR pathways in sporadic intrahepatic cholangiocarcinoma.\", \"citation\"=>\"Borad et al., 2014, PLoS Genet.\", \"pubmed_id\"=>\"24550739\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24550739\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3923676\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>2}, \"journal\"=>\"PLoS Genet.\", \"full_journal_title\"=>\"PLoS genetics\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>691, \"phenotypes\"=>[]}, {\"id\"=>1724, \"name\"=>\"EID1724\", \"description\"=>\"An ERRFI1 (E384X) was detected in a patient with metastatic, recurrent/refractory cholangiocarcinoma (allelic fraction 11%) without any mutations or amplifications in other EGFR signaling members such as EGFR and BRAF. The patient was treated with erlotinib 150 mg orally/daily. After 3 months, RECIST v1.1 partial response evidenced by a decrease of 58% in the sum of largest diameters was observed.\", \"disease\"=>{\"id\"=>29, \"name\"=>\"Cholangiocarcinoma\", \"display_name\"=>\"Cholangiocarcinoma\", \"doid\"=>\"4947\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4947\"}, \"drugs\"=>[{\"id\"=>15, \"name\"=>\"Erlotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>186, \"name\"=>\"Integrated genomic characterization reveals novel, therapeutically relevant drug targets in FGFR and EGFR pathways in sporadic intrahepatic cholangiocarcinoma.\", \"citation\"=>\"Borad et al., 2014, PLoS Genet.\", \"pubmed_id\"=>\"24550739\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24550739\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3923676\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>2}, \"journal\"=>\"PLoS Genet.\", \"full_journal_title\"=>\"PLoS genetics\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>691, \"phenotypes\"=>[]}]}, {\"id\"=>697, \"entrez_name\"=>\"NF2\", \"entrez_id\"=>4771, \"name\"=>\"LOSS\", \"description\"=>\"\", \"gene_id\"=>3870, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>4.0, \"coordinates\"=>{\"chromosome\"=>\"22\", \"start\"=>29999547, \"stop\"=>30094587, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000338641.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3870/summary/variants/697/summary#variant\", \"evidence_items\"=>[{\"id\"=>1742, \"name\"=>\"EID1742\", \"description\"=>\"NF2 loss promotes oncogenic RAS-signaling via YAP-dependent transactivation of RAS and sensitizes to MEK inhibition. The authors demonstrate this in (A) isogenic lines derived from C643 cells (HRASG13R, NF2-WT) modified to stably express shNF2, (B) in set of 7 RAS-mutant thyroid cancer cell lines that differ in their NF2 expression levels, and (C) in a Hras-G12V/Nf2-null mouse model.\", \"disease\"=>{\"id\"=>155, \"name\"=>\"Thyroid Carcinoma\", \"display_name\"=>\"Thyroid Carcinoma\", \"doid\"=>\"3963\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3963\"}, \"drugs\"=>[{\"id\"=>122, \"name\"=>\"Selumetinib (AZD6244)\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1191, \"name\"=>\"NF2 Loss Promotes Oncogenic RAS-Induced Thyroid Cancers via YAP-Dependent Transactivation of RAS Proteins and Sensitizes Them to MEK Inhibition.\", \"citation\"=>\"Garcia-Rendueles et al., 2015, Cancer Discov\", \"pubmed_id\"=>\"26359368\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26359368\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4642441\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>11}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>697, \"phenotypes\"=>[]}]}, {\"id\"=>698, \"entrez_name\"=>\"NF1\", \"entrez_id\"=>4763, \"name\"=>\"LOSS\", \"description\"=>\"\", \"gene_id\"=>3867, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"17\", \"start\"=>29421945, \"stop\"=>29704695, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000358273.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3867/summary/variants/698/summary#variant\", \"evidence_items\"=>[{\"id\"=>1957, \"name\"=>\"EID1957\", \"description\"=>\"Taken together with preclinical studies in the mouse model and genetic studies in human melanoma cell lines, these observations further support the hypothesis that NF1/neurofibromin suppression may play an important role in mediating resistance to BRAF inhibitors.\", \"disease\"=>{\"id\"=>7, \"name\"=>\"Melanoma\", \"display_name\"=>\"Melanoma\", \"doid\"=>\"1909\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1909\"}, \"drugs\"=>[{\"id\"=>22, \"name\"=>\"Dabrafenib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>985, \"name\"=>\"Elucidating distinct roles for NF1 in melanomagenesis.\", \"citation\"=>\"Maertens et al., 2013, Cancer Discov\", \"pubmed_id\"=>\"23171796\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23171796\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3595355\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>3}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>698, \"phenotypes\"=>[]}, {\"id\"=>1956, \"name\"=>\"EID1956\", \"description\"=>\"Low NF1 protein expression correlated positively with responses to binimetinib in neuroblastoma cell lines.\", \"disease\"=>{\"id\"=>13, \"name\"=>\"Neuroblastoma\", \"display_name\"=>\"Neuroblastoma\", \"doid\"=>\"769\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:769\"}, \"drugs\"=>[{\"id\"=>355, \"name\"=>\"Binimetinib (MEK162)\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1376, \"name\"=>\"Binimetinib inhibits MEK and is effective against neuroblastoma tumor cells with low NF1 expression.\", \"citation\"=>\"Woodfield et al., 2016, BMC Cancer\", \"pubmed_id\"=>\"26925841\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26925841\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4772351\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>3, \"day\"=>1}, \"journal\"=>\"BMC Cancer\", \"full_journal_title\"=>\"BMC cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>698, \"phenotypes\"=>[]}, {\"id\"=>1743, \"name\"=>\"EID1743\", \"description\"=>\"BRD4 inhibition supressess growth and tumorigenesis of NF1-/-, TP53-/- MPNST cells in mouse xenografts\", \"disease\"=>{\"id\"=>326, \"name\"=>\"Malignant Peripheral Nerve Sheath Tumor\", \"display_name\"=>\"Malignant Peripheral Nerve Sheath Tumor\", \"doid\"=>\"5940\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5940\"}, \"drugs\"=>[{\"id\"=>414, \"name\"=>\"JQ1 Compound\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1192, \"name\"=>\"BET bromodomain inhibition triggers apoptosis of NF1-associated malignant peripheral nerve sheath tumors through Bim induction.\", \"citation\"=>\"Patel et al., 2014, Cell Rep\", \"pubmed_id\"=>\"24373973\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24373973\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3904298\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>1, \"day\"=>16}, \"journal\"=>\"Cell Rep\", \"full_journal_title\"=>\"Cell reports\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>698, \"phenotypes\"=>[]}]}, {\"id\"=>721, \"entrez_name\"=>\"CYP2D6\", \"entrez_id\"=>1565, \"name\"=>\"LOSS-OF-FUNCTION\", \"description\"=>\"\", \"gene_id\"=>1342, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"22\", \"start\"=>42522501, \"stop\"=>42526908, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000360608.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1342/summary/variants/721/summary#variant\", \"evidence_items\"=>[{\"id\"=>1784, \"name\"=>\"EID1784\", \"description\"=>\"Meta-analysis of 25 studies involving 13,629 women with tamoxifen treated breast cancer. The meta-analysis investigated the association between CYP2D6 genotypes and response to tamoxifen treatment (loss/reduced function vs. unchanged/increased function). The authors examine a prevailing hypothesis that individuals carrying a reduced function CYP2D6 allele have poorer outcomes due to less metabolism of tamoxifen to active metabolites. The study concludes that there is insufficient evidence to support use of CYP2D6 genotyping as a guide for tamoxifen use in breast cancer.\", \"disease\"=>{\"id\"=>22, \"name\"=>\"Breast Cancer\", \"display_name\"=>\"Breast Cancer\", \"doid\"=>\"1612\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:1612\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Germline Polymorphism\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1248, \"name\"=>\"CYP2D6 genotype and tamoxifen response for breast cancer: a systematic review and meta-analysis.\", \"citation\"=>\"Lum et al., 2013, PLoS ONE\", \"pubmed_id\"=>\"24098545\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24098545\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3788742\", \"publication_date\"=>{\"year\"=>2013}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>721, \"phenotypes\"=>[]}]}, {\"id\"=>806, \"entrez_name\"=>\"RUNX1\", \"entrez_id\"=>861, \"name\"=>\"D198Y\", \"description\"=>\"D198Y was observed in a pedigree with a history of Familial platelet disorder with predisposition to acute myeloid leukemia. It occurs within the Runt domain of RUNX1, known to be associated with the disease.\", \"gene_id\"=>43, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}, {\"id\"=>47, \"name\"=>\"missense_variant\", \"display_name\"=>\"Missense Variant\", \"so_id\"=>\"SO:0001583\", \"description\"=>\"A sequence variant, that changes one or more bases, resulting in a different amino acid sequence but where the length is preserved.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001583\"}], \"civic_actionability_score\"=>7.5, \"coordinates\"=>{\"chromosome\"=>\"21\", \"start\"=>36231792, \"stop\"=>36231792, \"reference_bases\"=>\"C\", \"variant_bases\"=>\"A\", \"representative_transcript\"=>\"ENST00000300305.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/43/summary/variants/806/summary#variant\", \"evidence_items\"=>[{\"id\"=>1875, \"name\"=>\"EID1875\", \"description\"=>\"In a pedigree with familial platelet disorder with predisposition to acute myeloid leukemia (FPD/AML) spanning three generations, five members of generation II were sequenced at exons 3, 4 and 5 of the RUNX1 (or CBFA2) gene and the D198Y mutation (in the DNA binding RUNT domain) was found in 3/5 of the individuals with FPD/AML but not in 2/5 without FPD/AML. The mutation was not found in 108 control subjects.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1292, \"name\"=>\"A novel CBFA2 single-nucleotide mutation in familial platelet disorder with propensity to develop myeloid malignancies.\", \"citation\"=>\"Buijs et al., 2001, Blood\", \"pubmed_id\"=>\"11675361\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/11675361\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2001, \"month\"=>11, \"day\"=>1}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>806, \"phenotypes\"=>[]}]}, {\"id\"=>808, \"entrez_name\"=>\"MSH2\", \"entrez_id\"=>4436, \"name\"=>\"LOSS\", \"description\"=>\"\", \"gene_id\"=>3628, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>7.5, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>47630108, \"stop\"=>47710367, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000233146.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3628/summary/variants/808/summary#variant\", \"evidence_items\"=>[{\"id\"=>1877, \"name\"=>\"EID1877\", \"description\"=>\"Case report of a 45-year old woman of Japanese descent with recurrent urothelial carcinoma. The patient was identified to harbor mutations in MSH2 (A913fs*2, E226*, E580*) and in MSH6 (R361H) by panel sequencing as well as loss of expression of these genes as confirmed by IHC. Testing also revealed a high mutational load as well as 2+ PD-L1 expression in 80% of the cells.  The patient was treated with anti-PD-L1 inhibitor MEDI4736 and MEDI0680 in a phase 1 trial and had a complete remission within 2 months of treatment that was ongoing after 11 months.\", \"disease\"=>{\"id\"=>2132, \"name\"=>\"Urothelial Carcinoma\", \"display_name\"=>\"Urothelial Carcinoma\", \"doid\"=>\"2671\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2671\"}, \"drugs\"=>[{\"id\"=>427, \"name\"=>\"MEDI4736\", \"pubchem_id\"=>nil}, {\"id\"=>428, \"name\"=>\"MEDI0680\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1304, \"name\"=>\"Mismatch repair deficiency associated with complete remission to combination programmed cell death ligand immune therapy in a patient with sporadic urothelial carcinoma: immunotheranostic considerations.\", \"citation\"=>\"Castro et al., 2015, J Immunother Cancer\", \"pubmed_id\"=>\"26674132\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26674132\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4678532\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"J Immunother Cancer\", \"full_journal_title\"=>\"Journal for immunotherapy of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>808, \"phenotypes\"=>[]}]}, {\"id\"=>809, \"entrez_name\"=>\"MSH6\", \"entrez_id\"=>2956, \"name\"=>\"LOSS\", \"description\"=>\"\", \"gene_id\"=>2478, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>7.5, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>48010221, \"stop\"=>48037240, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000234420.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/2478/summary/variants/809/summary#variant\", \"evidence_items\"=>[{\"id\"=>1878, \"name\"=>\"EID1878\", \"description\"=>\"Case report of a 45-year old woman of Japanese descent with recurrent urothelial carcinoma. The patient was identified to harbor mutations in MSH2 (A913fs*2, E226*, E580*) and in MSH6 (R361H) by panel sequencing as well as loss of expression of these genes as confirmed by IHC. Testing also revealed a high mutational load as well as 2+ PD-L1 expression in 80% of the cells. The patient was treated with anti-PD-L1 inhibitor MEDI4736 and MEDI0680 in a phase 1 trial and had a complete remission within 2 months of treatment that was ongoing after 11 months.\", \"disease\"=>{\"id\"=>2132, \"name\"=>\"Urothelial Carcinoma\", \"display_name\"=>\"Urothelial Carcinoma\", \"doid\"=>\"2671\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2671\"}, \"drugs\"=>[{\"id\"=>427, \"name\"=>\"MEDI4736\", \"pubchem_id\"=>nil}, {\"id\"=>428, \"name\"=>\"MEDI0680\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>nil, \"drug_interaction_type\"=>\"Combination\", \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1304, \"name\"=>\"Mismatch repair deficiency associated with complete remission to combination programmed cell death ligand immune therapy in a patient with sporadic urothelial carcinoma: immunotheranostic considerations.\", \"citation\"=>\"Castro et al., 2015, J Immunother Cancer\", \"pubmed_id\"=>\"26674132\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26674132\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4678532\", \"publication_date\"=>{\"year\"=>2015}, \"journal\"=>\"J Immunother Cancer\", \"full_journal_title\"=>\"Journal for immunotherapy of cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>809, \"phenotypes\"=>[]}]}, {\"id\"=>819, \"entrez_name\"=>\"PTPRB\", \"entrez_id\"=>5787, \"name\"=>\"LOSS-OF-FUNCTION\", \"description\"=>\"\", \"gene_id\"=>4690, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>161, \"name\"=>\"loss_of_function_variant\", \"display_name\"=>\"Loss Of Function Variant\", \"so_id\"=>\"SO:0002054\", \"description\"=>\"A sequence variant whereby the gene product has diminished or abolished function.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002054\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"12\", \"start\"=>70910630, \"stop\"=>71031220, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000334414.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4690/summary/variants/819/summary#variant\", \"evidence_items\"=>[{\"id\"=>1895, \"name\"=>\"EID1895\", \"description\"=>\"The endothelial phosphatase PTPRB, a negative regulator of vascular growth factor tyrosine kinases, harbored truncating mutations in 10 of 39 angiosarcomas (26%) which correlated with MYC amplifications. In HUVEC cell line model, silencing of PTPRB  induced features of angiogenesis which were abolished in the presence of sunitinib or vatalinib, inhibitors of VEGFR.\", \"disease\"=>{\"id\"=>330, \"name\"=>\"Angiosarcoma\", \"display_name\"=>\"Angiosarcoma\", \"doid\"=>\"0001816\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0001816\"}, \"drugs\"=>[{\"id\"=>157, \"name\"=>\"Sunitinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1324, \"name\"=>\"Recurrent PTPRB and PLCG1 mutations in angiosarcoma.\", \"citation\"=>\"Behjati et al., 2014, Nat. Genet.\", \"pubmed_id\"=>\"24633157\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24633157\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4032873\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>4}, \"journal\"=>\"Nat. Genet.\", \"full_journal_title\"=>\"Nature genetics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>819, \"phenotypes\"=>[]}]}, {\"id\"=>1974, \"entrez_name\"=>\"VHL\", \"entrez_id\"=>7428, \"name\"=>\"Splicing alteration (c.464-1G>A)\", \"description\"=>\"\", \"gene_id\"=>58, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>40, \"name\"=>\"splice_acceptor_variant\", \"display_name\"=>\"Splice Acceptor Variant\", \"so_id\"=>\"SO:0001574\", \"description\"=>\"A splice variant that changes the 2 base region at the 3' end of an intron.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001574\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>10191470, \"stop\"=>10191470, \"reference_bases\"=>\"G\", \"variant_bases\"=>\"A\", \"representative_transcript\"=>\"ENST00000256474.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/58/summary/variants/1974/summary#variant\", \"evidence_items\"=>[{\"id\"=>5612, \"name\"=>\"EID5612\", \"description\"=>\"64 VHL patients with renal involvement were analyzed. 61 of the 64 patients had a known germline mutation. The above mutation was found in 2 patients. Clinical manifestations included renal cell carcinoma; other manifestations are possible, but not discussed. ACMG evidence codes: 'PSV1' because they observe a canonical -1 splice site variant in a gene where loss of function is a known mechanism of disease. Relevant HPO terms: Renal cell carcinoma.\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2286, \"name\"=>\"Growth kinetics in von Hippel-Lindau-associated renal cell carcinoma.\", \"citation\"=>\"Jilg et al., 2012, Urol. Int.\", \"pubmed_id\"=>\"22156657\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22156657\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012}, \"journal\"=>\"Urol. Int.\", \"full_journal_title\"=>\"Urologia internationalis\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1974, \"phenotypes\"=>[]}, {\"id\"=>5898, \"name\"=>\"EID5898\", \"description\"=>\"In a study of 426 unrelated VHL patients, 111 were discovered to have alterations in the VHL gene. 18 novel variants were identified in VHL patients, but none were present in 200 unaffected control individuals. This splice site variant was found in a 15 year-old VHL patient with retinal hemangioblastomas (index case 227). This study contains very strong evidence for pathogenicity in the form of a splice site variant in a gene where loss-of-function is a known mechanism of disease (ACMG code: PVS1).\\nHPO terms: retinal hemangioblastoma.\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2174, \"name\"=>\"Identification and in silico analysis of novel von Hippel-Lindau (VHL) gene variants from a large population.\", \"citation\"=>\"Leonardi et al., 2011, Ann. Hum. Genet.\", \"pubmed_id\"=>\"21463266\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21463266\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>7}, \"journal\"=>\"Ann. Hum. Genet.\", \"full_journal_title\"=>\"Annals of human genetics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1974, \"phenotypes\"=>[]}, {\"id\"=>5387, \"name\"=>\"EID5387\", \"description\"=>\"A previous study of 26 Japanese, VHL families was extended to 41 additional families. Germline mutations were detected in 55 of 77 Japanese families. Renal cell carcinoma occurred predominantly in non-missense mutations predicted to lead to gross alteration or complete loss of the VHL protein. This mutation was found in a VHL type 1 patient with retinal angiomas, pancreatic cysts, and renal cell carcinoma (family ID 68). This study contains very strong evidence for pathogenicity in the form of a -1 splice-site variant in a gene where loss-of-function is a known mechanism of disease (ACMG code: PVS1).\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2240, \"name\"=>\"Germ-line mutation analysis in patients with von Hippel-Lindau disease in Japan: an extended study of 77 families.\", \"citation\"=>\"Yoshida et al., 2000, Jpn. J. Cancer Res.\", \"pubmed_id\"=>\"10761708\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/10761708\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2000, \"month\"=>2}, \"journal\"=>\"Jpn. J. Cancer Res.\", \"full_journal_title\"=>\"Japanese journal of cancer research : Gann\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1974, \"phenotypes\"=>[]}, {\"id\"=>5357, \"name\"=>\"EID5357\", \"description\"=>\"Of 65 VHL families from central Europe, 53 were identified with germline mutations. This splice mutation was found in a German, VHL type 1 patient with hemangioblastomas of the central nervous system (family VHL 48). This study contains very strong evidence of pathogenicity in the form of a canonical -1 splice site variant in a gene where loss-of-function is a known mechanism of disease (ACMG code: PVS1). \\nHPO terms: hemangioblastoma.\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2239, \"name\"=>\"Mutations in the VHL tumor suppressor gene and associated lesions in families with von Hippel-Lindau disease from central Europe.\", \"citation\"=>\"Glavac et al., 1996, Hum. Genet.\", \"pubmed_id\"=>\"8707293\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/8707293\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1996, \"month\"=>9}, \"journal\"=>\"Hum. Genet.\", \"full_journal_title\"=>\"Human genetics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1974, \"phenotypes\"=>[]}, {\"id\"=>5309, \"name\"=>\"EID5309\", \"description\"=>\"43 Italian patients were molecularly analyzed for Von Hippel-Lindau disease based on clinical suspicion. 1 of the 43 patients was found to have the above mutation. Clinical manifestations associated with this mutation include: CNS hemangioblastoma, renal cell carcinoma, and epididymal cysts. ACMG evidence codes: 'PSV1' because they observe a canonical -1 splice site variant in a gene where loss of function is a known mechanism of disease. Relevant HPO terms: Hemangioblastoma, Renal cell carcinoma, Epididymal cysts.\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2216, \"name\"=>\"Germline mutations in the von Hippel-Lindau gene in Italian patients.\", \"citation\"=>\"Ciotti et al., Eur J Med Genet\", \"pubmed_id\"=>\"19464396\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/19464396\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{}, \"journal\"=>\"Eur J Med Genet\", \"full_journal_title\"=>\"European journal of medical genetics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1974, \"phenotypes\"=>[]}]}, {\"id\"=>1990, \"entrez_name\"=>\"VHL\", \"entrez_id\"=>7428, \"name\"=>\"Splicing alteration (c.464-1G>T)\", \"description\"=>\"\", \"gene_id\"=>58, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>40, \"name\"=>\"splice_acceptor_variant\", \"display_name\"=>\"Splice Acceptor Variant\", \"so_id\"=>\"SO:0001574\", \"description\"=>\"A splice variant that changes the 2 base region at the 3' end of an intron.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001574\"}], \"civic_actionability_score\"=>12.5, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>10191470, \"stop\"=>10191470, \"reference_bases\"=>\"G\", \"variant_bases\"=>\"T\", \"representative_transcript\"=>\"ENST00000256474.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/58/summary/variants/1990/summary#variant\", \"evidence_items\"=>[{\"id\"=>5642, \"name\"=>\"EID5642\", \"description\"=>\"An investigation of 94 VHL patients without large deletions for intragenic mutations revealed 40 different mutations in 55 unrelated individuals. In all patients with multiple affected family members the missense mutation segregated with the disease. Large deletions or intragenic mutations predicted to cause a truncated protein were found in 36 of 53 families without pheochromocytoma (VHL type 1) and 2 of 12 families with pheochromocytoma (VHL type 2). This splice mutation was found in a VHL type 1 patient (kindred no. 143). This study contains very strong evidence for pathogenicity in the form of a canonical -1 splice site mutation in a gene where loss-of-function is a known mechanism of disease (ACMG code: PVS1).\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1329, \"name\"=>\"Identification of intragenic mutations in the von Hippel-Lindau disease tumour suppressor gene and correlation with disease phenotype.\", \"citation\"=>\"Crossey et al., 1994, Hum. Mol. Genet.\", \"pubmed_id\"=>\"7987306\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/7987306\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1994, \"month\"=>8}, \"journal\"=>\"Hum. Mol. Genet.\", \"full_journal_title\"=>\"Human molecular genetics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1990, \"phenotypes\"=>[]}, {\"id\"=>5614, \"name\"=>\"EID5614\", \"description\"=>\"64 VHL patients with renal involvement were analyzed. 61 of the 64 patients had a known germline mutation. The above mutation was found in 1 patient. Clinical manifestations included renal cell carcinoma; other manifestations are possible, but not discussed. ACMG evidence codes: 'PSV1' because they observe a canonical -1 splice site variant in a gene where loss of function is a known mechanism of disease. Relevant HPO terms: Renal cell carcinoma.\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2286, \"name\"=>\"Growth kinetics in von Hippel-Lindau-associated renal cell carcinoma.\", \"citation\"=>\"Jilg et al., 2012, Urol. Int.\", \"pubmed_id\"=>\"22156657\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22156657\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012}, \"journal\"=>\"Urol. Int.\", \"full_journal_title\"=>\"Urologia internationalis\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1990, \"phenotypes\"=>[]}, {\"id\"=>5388, \"name\"=>\"EID5388\", \"description\"=>\"A previous study of 26 Japanese, VHL families was extended to 41 additional families. Germline mutations were detected in 55 of 77 Japanese families. Renal cell carcinoma occurred predominantly in non-missense mutations predicted to lead to gross alteration or complete loss of the VHL protein. This mutation was found in a VHL type 1 family of 7. All seven had hemangioblastomas of the central nervous system, 3 had retinal angiomas, one had pancreatic cysts, and 3 had renal cell carcinoma (family ID 3) ACMG codes as follows:. A canonical -1 splice-site variant in a gene where loss-of-function is a known mechanism of disease (PVS1). Cosegregation with disease in multiple affected family members in a gene definitively known to cause the disease (PP1).\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2240, \"name\"=>\"Germ-line mutation analysis in patients with von Hippel-Lindau disease in Japan: an extended study of 77 families.\", \"citation\"=>\"Yoshida et al., 2000, Jpn. J. Cancer Res.\", \"pubmed_id\"=>\"10761708\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/10761708\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2000, \"month\"=>2}, \"journal\"=>\"Jpn. J. Cancer Res.\", \"full_journal_title\"=>\"Japanese journal of cancer research : Gann\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1990, \"phenotypes\"=>[]}, {\"id\"=>5343, \"name\"=>\"EID5343\", \"description\"=>\"Blood samples from 5 Japanese families revealed 4 germline mutations. This mutation was found in a family of 10 individuals. Three VHL patients were identified with hemangioblastomas of the central nervous system and retinal angiomas (family C). The eldest patient also had renal cell carcinoma (listed as C-1 in publication). This study contains very strong evidence of pathogenicity in the form of a canonical -1 splice site mutation in a gene where loss-of-function is a known mechanism of disease (ACMG code: PVS1).\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2235, \"name\"=>\"Molecular genetic diagnosis of von Hippel-Lindau disease: analysis of five Japanese families.\", \"citation\"=>\"Kanno et al., 1996, Jpn. J. Cancer Res.\", \"pubmed_id\"=>\"8641976\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/8641976\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1996, \"month\"=>5}, \"journal\"=>\"Jpn. J. Cancer Res.\", \"full_journal_title\"=>\"Japanese journal of cancer research : Gann\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1990, \"phenotypes\"=>[]}]}, {\"id\"=>1992, \"entrez_name\"=>\"VHL\", \"entrez_id\"=>7428, \"name\"=>\"Splicing alteration (c.464-2A>T)\", \"description\"=>\"\", \"gene_id\"=>58, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>40, \"name\"=>\"splice_acceptor_variant\", \"display_name\"=>\"Splice Acceptor Variant\", \"so_id\"=>\"SO:0001574\", \"description\"=>\"A splice variant that changes the 2 base region at the 3' end of an intron.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001574\"}], \"civic_actionability_score\"=>5.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>10191469, \"stop\"=>10191469, \"reference_bases\"=>\"A\", \"variant_bases\"=>\"T\", \"representative_transcript\"=>\"ENST00000256474.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/58/summary/variants/1992/summary#variant\", \"evidence_items\"=>[{\"id\"=>5351, \"name\"=>\"EID5351\", \"description\"=>\"Of 65 VHL families from central Europe, 53 were identified with germline mutations. This splice mutation was found in a German, VHL type 1 family. Only one affected patient was identified with hemangioblastomas of the central nervous system, renal cell carcinoma, and pancreatic cysts. (family VHL 57). This study contains very strong evidence of pathogenicity in the form of a canonical -2 splice variant in a gene where loss-of-function is a known mechanism of disease (ACMG code: PVS1).\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2239, \"name\"=>\"Mutations in the VHL tumor suppressor gene and associated lesions in families with von Hippel-Lindau disease from central Europe.\", \"citation\"=>\"Glavac et al., 1996, Hum. Genet.\", \"pubmed_id\"=>\"8707293\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/8707293\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1996, \"month\"=>9}, \"journal\"=>\"Hum. Genet.\", \"full_journal_title\"=>\"Human genetics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>1992, \"phenotypes\"=>[]}]}, {\"id\"=>2077, \"entrez_name\"=>\"VHL\", \"entrez_id\"=>7428, \"name\"=>\"Splicing alteration (c.464-1G>C)\", \"description\"=>\"\", \"gene_id\"=>58, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>40, \"name\"=>\"splice_acceptor_variant\", \"display_name\"=>\"Splice Acceptor Variant\", \"so_id\"=>\"SO:0001574\", \"description\"=>\"A splice variant that changes the 2 base region at the 3' end of an intron.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001574\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>10191470, \"stop\"=>10191470, \"reference_bases\"=>\"G\", \"variant_bases\"=>\"C\", \"representative_transcript\"=>\"ENST00000256474.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/58/summary/variants/2077/summary#variant\", \"evidence_items\"=>[{\"id\"=>5745, \"name\"=>\"EID5745\", \"description\"=>\"Genotype-phenotype correlations of 573 VHL patients were analyzed and confirmed that higher risk of pheochromocytoma is associated with missense mutations that result in substitution of a surface amino acid or that disrupt the structural integrity of the VHL gene product (pVHL). Age-related risks of retinal angiomas and renal cell carcinoma were higher in patients with nonsense or frameshift mutations. This mutation was found in 2 unrelated VHL families with 4 individuals total. Three had hemangioblastomas of the central nervous system, 4 had retinal angiomas, and 2 had renal cell carcinoma. There is very strong evidence for pathogenicity in the form of a splice-site variant in a gene where loss-of-function is known mechanism of disease (ACMG code: PSV1).\\nHPO terms: hemangioblastoma, retinal hemangioblastoma, renal cell carcinoma.\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>2, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1264, \"name\"=>\"Genotype-phenotype correlations in von Hippel-Lindau disease.\", \"citation\"=>\"Ong et al., 2007, Hum. Mutat.\", \"pubmed_id\"=>\"17024664\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17024664\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>2}, \"journal\"=>\"Hum. Mutat.\", \"full_journal_title\"=>\"Human mutation\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2077, \"phenotypes\"=>[]}, {\"id\"=>5613, \"name\"=>\"EID5613\", \"description\"=>\"64 VHL patients with renal involvement were analyzed. 61 of the 64 patients had a known germline mutation. The above mutation was found in 2 patients. Clinical manifestations included renal cell carcinoma; other manifestations are possible, but not discussed. ACMG evidence codes: 'PSV1' because they observe a canonical -1 splice site variant in a gene where loss of function is a known mechanism of disease. Relevant HPO terms: Renal cell carcinoma.\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2286, \"name\"=>\"Growth kinetics in von Hippel-Lindau-associated renal cell carcinoma.\", \"citation\"=>\"Jilg et al., 2012, Urol. Int.\", \"pubmed_id\"=>\"22156657\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22156657\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012}, \"journal\"=>\"Urol. Int.\", \"full_journal_title\"=>\"Urologia internationalis\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2077, \"phenotypes\"=>[]}]}, {\"id\"=>2078, \"entrez_name\"=>\"VHL\", \"entrez_id\"=>7428, \"name\"=>\"Splicing alteration (c.464-2A>G)\", \"description\"=>\"\", \"gene_id\"=>58, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>40, \"name\"=>\"splice_acceptor_variant\", \"display_name\"=>\"Splice Acceptor Variant\", \"so_id\"=>\"SO:0001574\", \"description\"=>\"A splice variant that changes the 2 base region at the 3' end of an intron.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001574\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>10191469, \"stop\"=>10191469, \"reference_bases\"=>\"A\", \"variant_bases\"=>\"G\", \"representative_transcript\"=>\"ENST00000256474.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/58/summary/variants/2078/summary#variant\", \"evidence_items\"=>[{\"id\"=>5724, \"name\"=>\"EID5724\", \"description\"=>\"Genotype-phenotype correlations of 573 VHL patients were analyzed and confirmed that higher risk of pheochromocytoma is associated with missense mutations that result in substitution of a surface amino acid or that disrupt the structural integrity of the VHL gene product (pVHL). Age-related risks of retinal angiomas and renal cell carcinoma were higher in patients with nonsense or frameshift mutations. This splice mutation was found in a VHL type 1 family of 3 individuals. One family member had retinal angiomas and 2 had hemangioblastomas of the central nervous system. There is very strong evidence for pathogenicity in the form of a splice-site variant in a gene where loss-of-function is known mechanism of disease (ACMG code: PSV1).\\nHPO terms: retinal hemangioblastoma, hemangioblastoma.\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1264, \"name\"=>\"Genotype-phenotype correlations in von Hippel-Lindau disease.\", \"citation\"=>\"Ong et al., 2007, Hum. Mutat.\", \"pubmed_id\"=>\"17024664\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17024664\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>2}, \"journal\"=>\"Hum. Mutat.\", \"full_journal_title\"=>\"Human mutation\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2078, \"phenotypes\"=>[]}, {\"id\"=>5615, \"name\"=>\"EID5615\", \"description\"=>\"64 VHL patients with renal involvement were analyzed. 61 of the 64 patients had a known germline mutation. The above mutation was found in 1 patient. Clinical manifestations included renal cell carcinoma; other manifestations are possible, but not discussed. ACMG evidence codes: 'PSV1' because they observe a canonical -2 splice site variant in a gene where loss of function is a known mechanism of disease. Relevant HPO terms: Renal cell carcinoma.\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2286, \"name\"=>\"Growth kinetics in von Hippel-Lindau-associated renal cell carcinoma.\", \"citation\"=>\"Jilg et al., 2012, Urol. Int.\", \"pubmed_id\"=>\"22156657\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22156657\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012}, \"journal\"=>\"Urol. Int.\", \"full_journal_title\"=>\"Urologia internationalis\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2078, \"phenotypes\"=>[]}, {\"id\"=>5851, \"name\"=>\"EID5851\", \"description\"=>\"Of 65 VHL families from central Europe, 53 were identified with germline mutations. This splice mutation was found in a German, VHL type 1 family. Only one affected patient was identified with retinal angiomas, hemangioblastomas of the central nervous system, renal cell carcinoma, pancreatic cysts and cystadenomas\\nof the epididymis. (family VHL 31).  HPO terms: hemangioblastomas, retinal hemangioblastomas, renal cell carcinoma, multiple pancreatic cysts, papillary cystadenomas of the broad ligament.\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2239, \"name\"=>\"Mutations in the VHL tumor suppressor gene and associated lesions in families with von Hippel-Lindau disease from central Europe.\", \"citation\"=>\"Glavac et al., 1996, Hum. Genet.\", \"pubmed_id\"=>\"8707293\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/8707293\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>1996, \"month\"=>9}, \"journal\"=>\"Hum. Genet.\", \"full_journal_title\"=>\"Human genetics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2078, \"phenotypes\"=>[]}]}, {\"id\"=>2133, \"entrez_name\"=>\"VHL\", \"entrez_id\"=>7428, \"name\"=>\"Splicing alteration (c.341-2A>C)\", \"description\"=>\"\", \"gene_id\"=>58, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>40, \"name\"=>\"splice_acceptor_variant\", \"display_name\"=>\"Splice Acceptor Variant\", \"so_id\"=>\"SO:0001574\", \"description\"=>\"A splice variant that changes the 2 base region at the 3' end of an intron.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001574\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>10188196, \"stop\"=>10188196, \"reference_bases\"=>\"A\", \"variant_bases\"=>\"C\", \"representative_transcript\"=>\"ENST00000256474.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/58/summary/variants/2133/summary#variant\", \"evidence_items\"=>[{\"id\"=>5748, \"name\"=>\"EID5748\", \"description\"=>\"Molecular analysis of VHL gene in 146 probands, 103 with and 43 without a positive family history, resulted in the detection of 43 germline VHL mutations. Majority of mutations were found in patients with classic or multi-organ manifestations of VHL disease, but mutations were also found in patients with limited or single-organ manifestations of VHL disease. Follow-up of families with germline mutations and limited VHL manifestations was recommended. This splice mutation was found in VHL patient with hemangioblastomas of the retina and central nervous system, renal cell carcinoma, and pancreatic cysts (family no. 32). There is very strong evidence for pathogenicity in the form of a splice-site variant in a gene where loss-of-function is known mechanism of disease (ACMG code: PSV1).\\nHPO terms, retinal hemangioblastoma, hemangioblastoma, renal cell carcinoma, multiple pancreatic cysts.\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2192, \"name\"=>\"Frequency of Von Hippel-Lindau germline mutations in classic and non-classic Von Hippel-Lindau disease identified by DNA sequencing, Southern blot analysis and multiplex ligation-dependent probe amplification.\", \"citation\"=>\"Hes et al., 2007, Clin. Genet.\", \"pubmed_id\"=>\"17661816\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17661816\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>8}, \"journal\"=>\"Clin. Genet.\", \"full_journal_title\"=>\"Clinical genetics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2133, \"phenotypes\"=>[]}, {\"id\"=>5730, \"name\"=>\"EID5730\", \"description\"=>\"Genotype-phenotype correlations of 573 VHL patients were analyzed and confirmed that higher risk of pheochromocytoma is associated with missense mutations that result in substitution of a surface amino acid or that disrupt the structural integrity of the VHL gene product (pVHL). Age-related risks of retinal angiomas and renal cell carcinoma were higher in patients with nonsense or frameshift mutations. This mutation was found in a VHL type 2 patient with retinal angiomas, hemangioblastomas of the central nervous system, renal cell carcinoma, and pheochromocytoma. There is very strong evidence for pathogenicity in the form of a splice-site variant in a gene where loss-of-function is known mechanism of disease (ACMG code: PSV1).\", \"disease\"=>{\"id\"=>160, \"name\"=>\"Renal Cell Carcinoma\", \"display_name\"=>\"Renal Cell Carcinoma\", \"doid\"=>\"4450\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:4450\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1264, \"name\"=>\"Genotype-phenotype correlations in von Hippel-Lindau disease.\", \"citation\"=>\"Ong et al., 2007, Hum. Mutat.\", \"pubmed_id\"=>\"17024664\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/17024664\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2007, \"month\"=>2}, \"journal\"=>\"Hum. Mutat.\", \"full_journal_title\"=>\"Human mutation\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>2133, \"phenotypes\"=>[]}]}, {\"id\"=>275, \"entrez_name\"=>\"HRAS\", \"entrez_id\"=>3265, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>2747, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>137, \"name\"=>\"coding_transcript_variant\", \"display_name\"=>\"Coding Transcript Variant\", \"so_id\"=>\"SO:0001968\", \"description\"=>\"A transcript variant of a protein coding gene.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001968\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"11\", \"start\"=>533873, \"stop\"=>534289, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000451590.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/2747/summary/variants/275/summary#variant\", \"evidence_items\"=>[{\"id\"=>700, \"name\"=>\"EID700\", \"description\"=>\"A pre-clinical study in 10 solid cancer cell lines. HRAS mutations (Q61L, Q61R and G12V) sensitized cell lines toward MEK inhibitors (AZD6244/selumetinib, MEK162 and PD0325901) and mTOR inhibitors (Everolimus and AZD8055).\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>29, \"name\"=>\"PD0325901\", \"pubchem_id\"=>nil}, {\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}, {\"id\"=>122, \"name\"=>\"Selumetinib (AZD6244)\", \"pubchem_id\"=>nil}, {\"id\"=>124, \"name\"=>\"AZD8055\", \"pubchem_id\"=>nil}, {\"id\"=>355, \"name\"=>\"Binimetinib (MEK162)\", \"pubchem_id\"=>\"\"}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>458, \"name\"=>\"Mutant HRAS as novel target for MEK and mTOR inhibitors.\", \"citation\"=>\"Kiessling et al., 2015, Oncotarget\", \"pubmed_id\"=>\"26544513\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26544513\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4747218\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>12, \"day\"=>8}, \"journal\"=>\"Oncotarget\", \"full_journal_title\"=>\"Oncotarget\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>275, \"phenotypes\"=>[]}]}, {\"id\"=>300, \"entrez_name\"=>\"SMO\", \"entrez_id\"=>6608, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>5365, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>137, \"name\"=>\"coding_transcript_variant\", \"display_name\"=>\"Coding Transcript Variant\", \"so_id\"=>\"SO:0001968\", \"description\"=>\"A transcript variant of a protein coding gene.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001968\"}], \"civic_actionability_score\"=>44.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>128828713, \"stop\"=>128853386, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000249373.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/5365/summary/variants/300/summary#variant\", \"evidence_items\"=>[{\"id\"=>5979, \"name\"=>\"EID5979\", \"description\"=>\"The phase 2a MyPathway study assigned patients with HER2, EGFR, BRAF or SHH alterations to treatment with pertuzumab plus trastuzumab, erlotinib, vemurafenib, or vismodegib, respectively. Twenty-one patients had mutations in the Hedgehog pathway (PTCH-1, n = 18; SMO, n = 3). None of the patients with SMO mutations responded to SHH-inhibition with vismodegib.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>190, \"name\"=>\"Vismodegib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Does Not Support\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2414, \"name\"=>\"Targeted Therapy for Advanced Solid Tumors on the Basis of Molecular Profiles: Results From MyPathway, an Open-Label, Phase IIa Multiple Basket Study.\", \"citation\"=>\"Hainsworth et al., 2018, J. Clin. Oncol.\", \"pubmed_id\"=>\"29320312\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/29320312\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2018, \"month\"=>1, \"day\"=>10}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[{\"nct_id\"=>\"NCT02091141\", \"name\"=>\"My Pathway: A Study Evaluating Herceptin/Perjeta, Tarceva, Zelboraf/Cotellic, Erivedge, Alecensa, and Tecentriq Treatment Targeted Against Certain Molecular Alterations in Participants With Advanced Solid Tumors\", \"description\"=>\"This multicenter, non-randomized, open-label study will evaluate the efficacy and safety of six treatment regimens in participants with advanced solid tumors for whom therapies that will convey clinical benefit are not available and/or are not suitable options per the treating physician's judgment.\", \"clinical_trial_url\"=>\"https://clinicaltrials.gov/show/NCT02091141\"}]}, \"variant_id\"=>300, \"phenotypes\"=>[]}, {\"id\"=>1477, \"name\"=>\"EID1477\", \"description\"=>\"Study explored the response to the SMO inhibitor sonidegib (LDE225) in a cohort of 9 patients with advanced basal cell carcinoma  that exhibited treatment resistance with vismodegib (GDC0449). Sequencing revealed that 5 of these patients harbored SMO mutations known to confer resistance to the Smoothened inhibitors (D473G, D473H, Q477E, S533N, W535L) an additional patient harbored a SMO mutation of unknown consequence (R168H). Of these patients, 4 with a SMO mutation displayed progressive disease and 1 stable disease for 58 weeks according to RECIST v1.1 criteria.\", \"disease\"=>{\"id\"=>314, \"name\"=>\"Basal Cell Carcinoma\", \"display_name\"=>\"Basal Cell Carcinoma\", \"doid\"=>\"2513\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2513\"}, \"drugs\"=>[{\"id\"=>190, \"name\"=>\"Vismodegib\", \"pubchem_id\"=>nil}, {\"id\"=>357, \"name\"=>\"LDE225\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Sequential\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>990, \"name\"=>\"An Investigator-Initiated Open-Label Trial of Sonidegib in Advanced Basal Cell Carcinoma Patients Resistant to Vismodegib.\", \"citation\"=>\"Danial et al., 2016, Clin. Cancer Res.\", \"pubmed_id\"=>\"26546616\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26546616\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4794361\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>3, \"day\"=>15}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>300, \"phenotypes\"=>[]}, {\"id\"=>747, \"name\"=>\"EID747\", \"description\"=>\"Basal cell carcinomas with ligand-binding pocket or constitutively active mutations in the SMO gene were sensitive to inhibition of downstream hedgehog signaling with PSI (aPKC-ι/λ/GLI inhibitor) or arsenic trioxide.\", \"disease\"=>{\"id\"=>314, \"name\"=>\"Basal Cell Carcinoma\", \"display_name\"=>\"Basal Cell Carcinoma\", \"doid\"=>\"2513\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2513\"}, \"drugs\"=>[{\"id\"=>47, \"name\"=>\"Arsenic Trioxide\", \"pubchem_id\"=>nil}, {\"id\"=>271, \"name\"=>\"PSI\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>493, \"name\"=>\"Smoothened variants explain the majority of drug resistance in basal cell carcinoma.\", \"citation\"=>\"Atwood et al., 2015, Cancer Cell\", \"pubmed_id\"=>\"25759020\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25759020\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4357167\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>3, \"day\"=>9}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>300, \"phenotypes\"=>[]}, {\"id\"=>746, \"name\"=>\"EID746\", \"description\"=>\"In basal cell carcinoma, H231R, D473G, W281C and Q477E mutations conferred resistance to SMO-inhibitor vismodegib. SMO mutations were identified in 22 of 44 resistant basal cell carcinomas in 15 patients.\", \"disease\"=>{\"id\"=>314, \"name\"=>\"Basal Cell Carcinoma\", \"display_name\"=>\"Basal Cell Carcinoma\", \"doid\"=>\"2513\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2513\"}, \"drugs\"=>[{\"id\"=>190, \"name\"=>\"Vismodegib\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>493, \"name\"=>\"Smoothened variants explain the majority of drug resistance in basal cell carcinoma.\", \"citation\"=>\"Atwood et al., 2015, Cancer Cell\", \"pubmed_id\"=>\"25759020\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25759020\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4357167\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>3, \"day\"=>9}, \"journal\"=>\"Cancer Cell\", \"full_journal_title\"=>\"Cancer cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>300, \"phenotypes\"=>[]}]}, {\"id\"=>642, \"entrez_name\"=>\"MDM2\", \"entrez_id\"=>4193, \"name\"=>\"RS34886328\", \"description\"=>\"\", \"gene_id\"=>3465, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>154, \"name\"=>\"3_prime_UTR_truncation\", \"display_name\"=>\"3 Prime UTR Truncation\", \"so_id\"=>\"SO:0002015\", \"description\"=>\"A sequence variant that causes the reduction of a the 3' UTR  with regard to the reference sequence.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0002015\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"12\", \"start\"=>69234988, \"stop\"=>69234988, \"reference_bases\"=>nil, \"variant_bases\"=>\"\", \"representative_transcript\"=>\"ENST00000462284.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3465/summary/variants/642/summary#variant\", \"evidence_items\"=>[{\"id\"=>1636, \"name\"=>\"EID1636\", \"description\"=>\"124 esophageal squamous cell carcinoma patients with radical resection were genotyped using MassArray.  Patients with the DEL/A +AA genotype of MDM2 rs34886328 had a notably increased overall survival (HR = 0.27, 95% CI: 0.13–0.56, P = 4.7×10−4, Pc = 0.003) and increased disease free survival (HR = 0.22, 95% CI: 0.11–0.43, P < .0001) when compared to patients without this mutation.\", \"disease\"=>{\"id\"=>149, \"name\"=>\"Esophagus Squamous Cell Carcinoma\", \"display_name\"=>\"Esophagus Squamous Cell Carcinoma\", \"doid\"=>\"3748\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3748\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1082, \"name\"=>\"Genetic Variants in the p14ARF/MDM2/TP53 Pathway Are Associated with the Prognosis of Esophageal Squamous Cell Carcinoma Patients Treated with Radical Resection.\", \"citation\"=>\"Li et al., 2016, PLoS ONE\", \"pubmed_id\"=>\"27414035\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27414035\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4944974\", \"publication_date\"=>{\"year\"=>2016}, \"journal\"=>\"PLoS ONE\", \"full_journal_title\"=>\"PloS one\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>642, \"phenotypes\"=>[]}]}, {\"id\"=>189, \"entrez_name\"=>\"DNMT3A\", \"entrez_id\"=>1788, \"name\"=>\"MUTATION\", \"description\"=>\"DNMT3A mutations occur in a variety of myeloid malignancies. These mutations are associated with adverse prognosis in acute myeloid leukemia (AML), particularly in the context of NPM1 and FLT3-ITD mutations, which often co-occur. Early results indicate AML patients with DNMT3A mutations may respond better to decitabine than those with wild-type. Shorter survival has been indicated in patients with T-cell acute lymphoblastic leukemia and myelodysplastic syndrome, however, larger cohorts are required to verify this finding. DNMT3A mutations have been suggested as a pre-leukemic lesion and found in older individuals with clonal hematopoiesis but no overt malignancy.\", \"gene_id\"=>18, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>115.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>25455845, \"stop\"=>25565459, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000264709.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/18/summary/variants/189/summary#variant\", \"evidence_items\"=>[{\"id\"=>2926, \"name\"=>\"EID2926\", \"description\"=>\"A subset of patients receiving PD1/PD-L1 inhibition appear to be \\\"hyper-progressors,\\\" with a greatly accelerated rate of tumor growth and clinical deterioration compared to pre-therapy. The current study investigated potential genomic markers associated with \\\"hyper-progression\\\". Consecutive stage IV cancer patients who received immunotherapies (CTLA-4, PD-1/PD-L1 inhibitors or other [investigational] agents) and had their tumor evaluated by next-generation sequencing Amongst 155 patients, TTF (time to treatment failure) <2 months was seen in 4/5 individuals harbouring DNMT3A alteration.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>137, \"name\"=>\"Nivolumab\", \"pubchem_id\"=>nil}, {\"id\"=>138, \"name\"=>\"Pembrolizumab\", \"pubchem_id\"=>nil}, {\"id\"=>261, \"name\"=>\"Atezolizumab\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1679, \"name\"=>\"Hyper-progressors after Immunotherapy: Analysis of Genomic Alterations Associated with Accelerated Growth Rate.\", \"citation\"=>\"Kato et al., 2017, Clin. Cancer Res.\", \"pubmed_id\"=>\"28351930\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28351930\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>3, \"day\"=>28}, \"journal\"=>\"Clin. Cancer Res.\", \"full_journal_title\"=>\"Clinical cancer research : an official journal of the American Association for Cancer Research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>189, \"phenotypes\"=>[]}, {\"id\"=>1587, \"name\"=>\"EID1587\", \"description\"=>\"Among 46 AML patients treated with decitabine, the response rate was 75% (6/8) among DNMT3A mutated and 34% (13/38) among DNMT3A wild-type patients (P=.008)\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[{\"id\"=>155, \"name\"=>\"Decitabine\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1049, \"name\"=>\"DNMT3A mutations and response to the hypomethylating agent decitabine in acute myeloid leukemia.\", \"citation\"=>\"Metzeler et al., 2012, Leukemia\", \"pubmed_id\"=>\"22124213\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22124213\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3696987\", \"publication_date\"=>{\"year\"=>2012, \"month\"=>5}, \"journal\"=>\"Leukemia\", \"full_journal_title\"=>\"Leukemia\", \"status\"=>\"partially curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>189, \"phenotypes\"=>[]}, {\"id\"=>1586, \"name\"=>\"EID1586\", \"description\"=>\"DNMT3A mutations associate with adverse overall survival in acute myeloid leukemia patients if they occur in the context of mutated NPM1 and  FLT3-ITD (NPM1-FLT3ITD-DNMT3A phenotype), and in the context of IDH2 codon R140 mutations.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1009, \"name\"=>\"Genomic Classification and Prognosis in Acute Myeloid Leukemia.\", \"citation\"=>\"Papaemmanuil et al., 2016, N. Engl. J. Med.\", \"pubmed_id\"=>\"27276561\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27276561\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4979995\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>6, \"day\"=>9}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>189, \"phenotypes\"=>[]}, {\"id\"=>1585, \"name\"=>\"EID1585\", \"description\"=>\"In AML patients aged <60years, presence of a DNMT3A mutation associated with inferior CR rate (HR, 0.54; P=.05) and shorter OS (HR, 1.84; P<.001) in multivariate analyses. Mutated DNMT3A associated with shorter RFS (HR 1.38; P=.034) irrespective of patient age.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1048, \"name\"=>\"Spectrum and prognostic relevance of driver gene mutations in acute myeloid leukemia.\", \"citation\"=>\"Metzeler et al., 2016, Blood\", \"pubmed_id\"=>\"27288520\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27288520\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>8, \"day\"=>4}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>189, \"phenotypes\"=>[]}, {\"id\"=>1238, \"name\"=>\"EID1238\", \"description\"=>\"12/150 patients with de novo myelodysplastic syndrome screened for DNMT3A mutations (bone marrow and paired normal skin) had heterozygous mutations with predicted translational consequences. 1 patient was identified with 2 nonsynonymous coding mutations. R882 mutations accounted for 4/13 reported mutations, P904L for 2/13 mutations and the remaining had not been previously reported at the time of this report. Overall and event-free survival were worse for DNMT3A mutant patients although transplantation status was not considered (log-rank p=0.005 and p=0.009). Transformation to AML was increased for mutant 7/12 (58%) versus wildtype 39/138 (28%) patients (p=0.007). The authors note larger cohorts will be required to confirm these results.\", \"disease\"=>{\"id\"=>34, \"name\"=>\"Myelodysplastic Syndrome\", \"display_name\"=>\"Myelodysplastic Syndrome\", \"doid\"=>\"0050908\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0050908\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>785, \"name\"=>\"Recurrent DNMT3A mutations in patients with myelodysplastic syndromes.\", \"citation\"=>\"Walter et al., 2011, Leukemia\", \"pubmed_id\"=>\"21415852\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21415852\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3202965\", \"publication_date\"=>{\"year\"=>2011, \"month\"=>7}, \"journal\"=>\"Leukemia\", \"full_journal_title\"=>\"Leukemia\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>189, \"phenotypes\"=>[]}, {\"id\"=>469, \"name\"=>\"EID469\", \"description\"=>\"In adult patients with early (n = 30), cortical (n = 30), mature (n = 3) or unknown (n = 8) T-cell acute lymphoblastic leukemia, those with mutations in DNMT3A (n=12) had shorter overall survival than wild-type patients (n=59; p=0.011). This was also true for the subset of patients with early T-ALL DNMT3A-mutated (n = 8) compared to wild-type patients (n = 22; p=0.005).\", \"disease\"=>{\"id\"=>227, \"name\"=>\"T-cell Acute Lymphoblastic Leukemia\", \"display_name\"=>\"T-cell Acute Lymphoblastic Leukemia\", \"doid\"=>\"5603\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:5603\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>283, \"name\"=>\"The molecular profile of adult T-cell acute lymphoblastic leukemia: mutations in RUNX1 and DNMT3A are associated with poor prognosis in T-ALL.\", \"citation\"=>\"Grossmann et al., 2013, Genes Chromosomes Cancer\", \"pubmed_id\"=>\"23341344\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23341344\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>4}, \"journal\"=>\"Genes Chromosomes Cancer\", \"full_journal_title\"=>\"Genes, chromosomes & cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>189, \"phenotypes\"=>[]}, {\"id\"=>188, \"name\"=>\"EID188\", \"description\"=>\"Young AML patients (415 patients; <60 years old) with any DNMT3A mutation (n=96) have shorter overall survival and relapse-free survival than patients with wildtype DNMT3A.\", \"disease\"=>{\"id\"=>3, \"name\"=>\"Acute Myeloid Leukemia\", \"display_name\"=>\"Acute Myeloid Leukemia\", \"doid\"=>\"9119\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:9119\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Poor Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>54, \"name\"=>\"Mutant DNMT3A: a marker of poor prognosis in acute myeloid leukemia.\", \"citation\"=>\"Ribeiro et al., 2012, Blood\", \"pubmed_id\"=>\"22490330\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22490330\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>6, \"day\"=>14}, \"journal\"=>\"Blood\", \"full_journal_title\"=>\"Blood\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>189, \"phenotypes\"=>[]}]}, {\"id\"=>297, \"entrez_name\"=>\"RIT1\", \"entrez_id\"=>6016, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>4875, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>155870065, \"stop\"=>155880706, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000368322.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/4875/summary/variants/297/summary#variant\", \"evidence_items\"=>[{\"id\"=>742, \"name\"=>\"EID742\", \"description\"=>\"RIT1 mutations were identified in approx. 2% of lung adenocarcinomas (5/87 in the \\\"oncogene negative\\\" samples of TCGA - no RIT1 mutations were observed in 143 samples with a known driver mutation). GDC-0941 impaired tumor growth of NCI-H2110 (RIT1 M90I mutated) xenografts in nude mice. GDC-0941 and AZD-6244 (selumetinib) impaired colony formation in-vitro. Combined PI3K/MEK inhibition was more effective than single agent treatment in-vitro.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>122, \"name\"=>\"Selumetinib (AZD6244)\", \"pubchem_id\"=>nil}, {\"id\"=>141, \"name\"=>\"Pictilisib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>490, \"name\"=>\"Oncogenic RIT1 mutations in lung adenocarcinoma.\", \"citation\"=>\"Berger et al., 2014, Oncogene\", \"pubmed_id\"=>\"24469055\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/24469055\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4150988\", \"publication_date\"=>{\"year\"=>2014, \"month\"=>8, \"day\"=>28}, \"journal\"=>\"Oncogene\", \"full_journal_title\"=>\"Oncogene\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>297, \"phenotypes\"=>[]}]}, {\"id\"=>600, \"entrez_name\"=>\"AR\", \"entrez_id\"=>367, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>67, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>20.0, \"coordinates\"=>{\"chromosome\"=>\"X\", \"start\"=>66764465, \"stop\"=>66950461, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000374690.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/67/summary/variants/600/summary#variant\", \"evidence_items\"=>[{\"id\"=>1521, \"name\"=>\"EID1521\", \"description\"=>\"In a study of 150 patients with metastatic castration resistant prostate cancer (mCRPC), 71.3% of cases (n=107) harbored androgen receptor (AR) pathway aberrations.   Within the AR mutation cohort, 78 patients had AR amplification mutations, 16 patients had AR missense mutations, and 13 patients had mutations in ZBTB16, NCOR1, FOXA1, and SPOP, which are all AR regulators or downstream effectors.  All AR mutations were exclusive to mCRPC neoplasms and were not found in primary prostate cancer.\", \"disease\"=>{\"id\"=>46, \"name\"=>\"Prostate Cancer\", \"display_name\"=>\"Prostate Cancer\", \"doid\"=>\"10283\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10283\"}, \"drugs\"=>[{\"id\"=>78, \"name\"=>\"Bicalutamide\", \"pubchem_id\"=>nil}, {\"id\"=>366, \"name\"=>\"Flutamide\", \"pubchem_id\"=>nil}, {\"id\"=>367, \"name\"=>\"Cyproterone Acetate\", \"pubchem_id\"=>\"\"}, {\"id\"=>368, \"name\"=>\"Nilutamide\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1014, \"name\"=>\"Integrative clinical genomics of advanced prostate cancer.\", \"citation\"=>\"Robinson et al., 2015, Cell\", \"pubmed_id\"=>\"26000489\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26000489\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4484602\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>5, \"day\"=>21}, \"journal\"=>\"Cell\", \"full_journal_title\"=>\"Cell\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>600, \"phenotypes\"=>[]}]}, {\"id\"=>645, \"entrez_name\"=>\"IDH1\", \"entrez_id\"=>3417, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>26, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>18.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>209101803, \"stop\"=>209116275, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000415913.1\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/26/summary/variants/645/summary#variant\", \"evidence_items\"=>[{\"id\"=>2933, \"name\"=>\"EID2933\", \"description\"=>\"Mutations in isocitrate dehydrogenase 1 and 2, which result in overproduction of 2-hydroxyglutarate (2HG). 2HG is considered to be an oncometabolite, or a metabolite that promotes carcinogenesis, and inhibitors of mutant isocitrate dehydrogenase are in development to target this process. The authors discovered that 2HG overproduction impairs homologous recombination used in DNA repair and sensitizes cancer cells to treatment with PARP inhibitors, another class of cancer drugs that are already in clinical use.\", \"disease\"=>{\"id\"=>98, \"name\"=>\"Brain Glioma\", \"display_name\"=>\"Brain Glioma\", \"doid\"=>\"0060108\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060108\"}, \"drugs\"=>[{\"id\"=>42, \"name\"=>\"Olaparib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1683, \"name\"=>\"2-Hydroxyglutarate produced by neomorphic IDH mutations suppresses homologous recombination and induces PARP inhibitor sensitivity.\", \"citation\"=>\"Sulkowski et al., 2017, Sci Transl Med\", \"pubmed_id\"=>\"28148839\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/28148839\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2017, \"month\"=>2, \"day\"=>1}, \"journal\"=>\"Sci Transl Med\", \"full_journal_title\"=>\"Science translational medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>645, \"phenotypes\"=>[]}, {\"id\"=>1639, \"name\"=>\"EID1639\", \"description\"=>\"Genomewide analysis of 250 low-grade adult gliomas revealed that patients with a wildtype IDH1/2 had significantly shorter overall survival than those with a mutated IDH gene (HR 7.4; 95%CI: 4.0-13.8). The median survival for patients with low grade gliomas was 1.7 years for patients with wild-type IDH (N=55); 6.3 years for patients with mutated IDH and no 1p/19q co-deletion (N=139); and 8.0 years for patients with IDH mutation and a 1p/19q co-deletion (N=84).\", \"disease\"=>{\"id\"=>98, \"name\"=>\"Brain Glioma\", \"display_name\"=>\"Brain Glioma\", \"doid\"=>\"0060108\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:0060108\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1085, \"name\"=>\"Comprehensive, Integrative Genomic Analysis of Diffuse Lower-Grade Gliomas.\", \"citation\"=>\"2015, N. Engl. J. Med.\", \"pubmed_id\"=>\"26061751\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26061751\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4530011\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>6, \"day\"=>25}, \"journal\"=>\"N. Engl. J. Med.\", \"full_journal_title\"=>\"The New England journal of medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>645, \"phenotypes\"=>[]}]}, {\"id\"=>659, \"entrez_name\"=>\"NF2\", \"entrez_id\"=>4771, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>3870, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"22\", \"start\"=>29999547, \"stop\"=>30094587, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000338641.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3870/summary/variants/659/summary#variant\", \"evidence_items\"=>[{\"id\"=>1671, \"name\"=>\"EID1671\", \"description\"=>\"Ten patients with Neurofibromatosis 2 were enrolled for treatment with Everolimus.  At 12 months, no patients had reduction in tumor volume >20%, however, five patients had stable disease with a reduction in annual growth rate from 67% growth/year to 0.5% growth/year.  Growth of tumors ensued after discontinuation of Everolimus.\", \"disease\"=>{\"id\"=>2140, \"name\"=>\"Acoustic Neuroma\", \"display_name\"=>\"Acoustic Neuroma\", \"doid\"=>\"12689\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:12689\"}, \"drugs\"=>[{\"id\"=>41, \"name\"=>\"Everolimus\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"N/A\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1131, \"name\"=>\"Phase II study of mTORC1 inhibition by everolimus in neurofibromatosis type 2 patients with growing vestibular schwannomas.\", \"citation\"=>\"Goutagny et al., 2015, J. Neurooncol.\", \"pubmed_id\"=>\"25567352\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25567352\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>4}, \"journal\"=>\"J. Neurooncol.\", \"full_journal_title\"=>\"Journal of neuro-oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>659, \"phenotypes\"=>[]}]}, {\"id\"=>663, \"entrez_name\"=>\"BCOR\", \"entrez_id\"=>54880, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>12555, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>15.0, \"coordinates\"=>{\"chromosome\"=>\"X\", \"start\"=>39910501, \"stop\"=>40036582, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000342274.4\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/12555/summary/variants/663/summary#variant\", \"evidence_items\"=>[{\"id\"=>1681, \"name\"=>\"EID1681\", \"description\"=>\"46 patients with poorly cohesive, signet-cell ring, gastric carcinoma were evaluated for genetic mutations.  Overall survival rate of BCOR positive cases were significantly better than BCOR negative cases cases (p=0.038).\", \"disease\"=>{\"id\"=>21, \"name\"=>\"Stomach Cancer\", \"display_name\"=>\"Stomach Cancer\", \"doid\"=>\"10534\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:10534\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"B\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1136, \"name\"=>\"Comprehensive mutation profiling of mucinous gastric carcinoma.\", \"citation\"=>\"Rokutan et al., 2016, J. Pathol.\", \"pubmed_id\"=>\"27313181\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/27313181\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>10}, \"journal\"=>\"J. Pathol.\", \"full_journal_title\"=>\"The Journal of pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>663, \"phenotypes\"=>[]}]}, {\"id\"=>674, \"entrez_name\"=>\"EIF1AX\", \"entrez_id\"=>1964, \"name\"=>\"MUTATION\", \"description\"=>\"\", \"gene_id\"=>1646, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>42, \"name\"=>\"transcript_variant\", \"display_name\"=>\"Transcript Variant\", \"so_id\"=>\"SO:0001576\", \"description\"=>\"A sequence variant that changes the structure of the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001576\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"X\", \"start\"=>20156713, \"stop\"=>20159754, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000379607.5\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1646/summary/variants/674/summary#variant\", \"evidence_items\"=>[{\"id\"=>1712, \"name\"=>\"EID1712\", \"description\"=>\"EIF1AX was identified as a uveal melanoma (UM) driver gene by whole-exome sequencing. Approximately 14%–20% of UM carry a mutation in EIF1AX, with most mutations found in exons 1 and 2. EIF1AX mutations are associated with a good prognosis, and are inversely associated with metastasis. This assertion is based on observed survival for 19 EIF1AX mutant UM cases compared to 69 EIF1AX wildtype cases.\", \"disease\"=>{\"id\"=>188, \"name\"=>\"Uveal Melanoma\", \"display_name\"=>\"Uveal Melanoma\", \"doid\"=>\"6039\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:6039\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Prognostic\", \"clinical_significance\"=>\"Better Outcome\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1155, \"name\"=>\"Exome sequencing identifies recurrent somatic mutations in EIF1AX and SF3B1 in uveal melanoma with disomy 3.\", \"citation\"=>\"Martin et al., 2013, Nat. Genet.\", \"pubmed_id\"=>\"23793026\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23793026\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4307600\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>8}, \"journal\"=>\"Nat. Genet.\", \"full_journal_title\"=>\"Nature genetics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>674, \"phenotypes\"=>[]}]}, {\"id\"=>252, \"entrez_name\"=>\"EGFR\", \"entrez_id\"=>1956, \"name\"=>\"EXON 4 DELETION\", \"description\"=>\"\", \"gene_id\"=>19, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>38, \"name\"=>\"exon_loss_variant\", \"display_name\"=>\"Exon Loss Variant\", \"so_id\"=>\"SO:0001572\", \"description\"=>\"A sequence variant whereby an exon is lost from the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001572\"}], \"civic_actionability_score\"=>4.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>55214299, \"stop\"=>55214433, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000275493.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/19/summary/variants/252/summary#variant\", \"evidence_items\"=>[{\"id\"=>662, \"name\"=>\"EID662\", \"description\"=>\"A variant of EGFR with exon 4 deletion (de4 EGFR) enhanced the invasiveness and cisplatin resistance of epithelial ovarian cancer cell lines (SKOV3, CAOV3, ES2) in vitro when compared with wild-type EGFR. The three cell lines were infected with lentivirus for either EGFR or de4 EGFR, and lentivirus-infected cells were used to perform transwell migration assays, invasion assays, and cisplatin sensitivity assays.\", \"disease\"=>{\"id\"=>225, \"name\"=>\"Epithelial Ovarian Cancer\", \"display_name\"=>\"Epithelial Ovarian Cancer\", \"doid\"=>\"2152\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2152\"}, \"drugs\"=>[{\"id\"=>326, \"name\"=>\"Cisplatin\", \"pubchem_id\"=>nil}], \"rating\"=>4, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>419, \"name\"=>\"Exon 4 deletion variant of epidermal growth factor receptor enhances invasiveness and cisplatin resistance in epithelial ovarian cancer.\", \"citation\"=>\"Zhang et al., 2013, Carcinogenesis\", \"pubmed_id\"=>\"23764753\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23764753\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2013, \"month\"=>11}, \"journal\"=>\"Carcinogenesis\", \"full_journal_title\"=>\"Carcinogenesis\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>252, \"phenotypes\"=>[]}]}, {\"id\"=>266, \"entrez_name\"=>\"LRP1B\", \"entrez_id\"=>53353, \"name\"=>\"EXON 12–22 DELETION\", \"description\"=>\"\", \"gene_id\"=>12146, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>38, \"name\"=>\"exon_loss_variant\", \"display_name\"=>\"Exon Loss Variant\", \"so_id\"=>\"SO:0001572\", \"description\"=>\"A sequence variant whereby an exon is lost from the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001572\"}], \"civic_actionability_score\"=>3.0, \"coordinates\"=>{\"chromosome\"=>\"2\", \"start\"=>141665446, \"stop\"=>141777671, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000389484.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/12146/summary/variants/266/summary#variant\", \"evidence_items\"=>[{\"id\"=>679, \"name\"=>\"EID679\", \"description\"=>\"A minimum common region of deletion of 0.19Mb, encompassing exons 12–22 of LRP1B4 was observed in 4 patients exhibiting acquired treatment resistance. Transfection of mLRP1B4, a fully functional minigene of LRP1B, in IGROV1 and OVCAR4 ovarian cell lines with low expression of LRP1B4 increased sensitivity to liposomal doxorubicin but not doxorubicin. siRNA knockdown of LRP1B4 in JHOS3 and Kuramochi ovarian cancer cell lines decreased sensitivity to liposomal doxorubicin but not doxorubicin.\", \"disease\"=>{\"id\"=>20, \"name\"=>\"Ovarian Cancer\", \"display_name\"=>\"Ovarian Cancer\", \"doid\"=>\"2394\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:2394\"}, \"drugs\"=>[{\"id\"=>68, \"name\"=>\"Doxorubicin\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Resistance or Non-Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>433, \"name\"=>\"LRP1B deletion in high-grade serous ovarian cancers is associated with acquired chemotherapy resistance to liposomal doxorubicin.\", \"citation\"=>\"Cowin et al., 2012, Cancer Res.\", \"pubmed_id\"=>\"22896685\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/22896685\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>8, \"day\"=>15}, \"journal\"=>\"Cancer Res.\", \"full_journal_title\"=>\"Cancer research\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>266, \"phenotypes\"=>[]}]}, {\"id\"=>324, \"entrez_name\"=>\"MET\", \"entrez_id\"=>4233, \"name\"=>\"EXON 14 SKIPPING MUTATION\", \"description\"=>\"Exon 14 mutations have been observed in ~3% of patients with NSCLC (28/933 reported by Awad et al J Clin Onc 2016), confirmed to cause exon skipping when available, and associated with concurrent MET amplification. Responses to the c-MET inhibitor crizotinib have been reported in preclinical models and in individual case reports as well as a small-scale clinical trial (17 patients treated, Drilon et al., 2016 (suppl; abstr 108; ASCO ID 167889-176)). Larger studies are needed but enrollment of patients with MET exon 14 skipping mutation in trials with MET inhibitors or off-label treatment is encouraged (Paik et al., 2015 (suppl; abstr 8021)).\", \"gene_id\"=>52, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>38, \"name\"=>\"exon_loss_variant\", \"display_name\"=>\"Exon Loss Variant\", \"so_id\"=>\"SO:0001572\", \"description\"=>\"A sequence variant whereby an exon is lost from the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001572\"}], \"civic_actionability_score\"=>33.0, \"coordinates\"=>{\"chromosome\"=>\"7\", \"start\"=>116411903, \"stop\"=>116412043, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000318493.6\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/52/summary/variants/324/summary#variant\", \"evidence_items\"=>[{\"id\"=>4786, \"name\"=>\"EID4786\", \"description\"=>\"In a never-smoking metastatic lung adenocarcinoma patient harboring a MET splice site mutation (c.2887-18_2887-7del12)  treated with crizotinib monotherapy, was  associated with a response as evident by the reduced size of tumors. Prior to identification of the MET splice site mutation, the patient was treated with stereotactic radiotherapy, palliative radiation, and a single cycle of pemetrexed; subsequently, the patient was treated with crizotinib, but progressed at 8 weeks of treatment.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>1, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2127, \"name\"=>\"Response to Crizotinib in a Patient With Lung Adenocarcinoma Harboring a MET Splice Site Mutation.\", \"citation\"=>\"Jenkins et al., 2015, Clin Lung Cancer\", \"pubmed_id\"=>\"25769807\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25769807\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4529392\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>9}, \"journal\"=>\"Clin Lung Cancer\", \"full_journal_title\"=>\"Clinical lung cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>324, \"phenotypes\"=>[]}, {\"id\"=>1651, \"name\"=>\"EID1651\", \"description\"=>\"Case report of a patient with lung adenocarcinoma harboring a MET exon 14 skipping mutation and major response to crizotinib.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1115, \"name\"=>\"Lazarus-Type Response to Crizotinib in a Patient with Poor Performance Status and Advanced MET Exon 14 Skipping Mutation-Positive Lung Adenocarcinoma.\", \"citation\"=>\"Shea et al., 2016, J Thorac Oncol\", \"pubmed_id\"=>\"26845194\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26845194\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4920699\", \"publication_date\"=>{\"year\"=>2016, \"month\"=>7}, \"journal\"=>\"J Thorac Oncol\", \"full_journal_title\"=>\"Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>324, \"phenotypes\"=>[]}, {\"id\"=>1186, \"name\"=>\"EID1186\", \"description\"=>\"A patient with MET c.3028+1G>T mutation (METex14) had partial response (tumor reduction of 61%) after 13 months capmatinib. MET gene copy number was four, MET FISH 13.8 copy number, and IHC 3+.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>300, \"name\"=>\"Capmatinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>818, \"name\"=>\"Activation of MET via diverse exon 14 splicing alterations occurs in multiple tumor types and confers clinical sensitivity to MET inhibitors.\", \"citation\"=>\"Frampton et al., 2015, Cancer Discov\", \"pubmed_id\"=>\"25971938\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25971938\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>8}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>324, \"phenotypes\"=>[]}, {\"id\"=>1185, \"name\"=>\"EID1185\", \"description\"=>\"A single patient with a MET c.3028G>A (splice donor site of exon 14) mutation exhibited clinical sensitivity to the c-Met inhibitor crizotinib. Exon 14 skipping was detected using qRT-PCR. Response to the therapy was demonstrated by comparing CAT scans obtained before treatment to those obtained during treatment (at 2 months of crizotinib therapy).\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>754, \"name\"=>\"MET Exon 14 Mutations in Non-Small-Cell Lung Cancer Are Associated With Advanced Age and Stage-Dependent MET Genomic Amplification and c-Met Overexpression.\", \"citation\"=>\"Awad et al., 2016, J. Clin. Oncol.\", \"pubmed_id\"=>\"26729443\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26729443\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2016, \"month\"=>3, \"day\"=>1}, \"journal\"=>\"J. Clin. Oncol.\", \"full_journal_title\"=>\"Journal of clinical oncology : official journal of the American Society of Clinical Oncology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>324, \"phenotypes\"=>[]}, {\"id\"=>736, \"name\"=>\"EID736\", \"description\"=>\"The creation of a cell line model (HEK293) with MET exon 14 deletion (using the CRISPR/Cas9 system) led to enhanced cellular growth and sensitivity to the MET inhibitor crizotinib. MET exon 14 deletion results in an in-frame deletion in the MET protein. Recurrent splice site mutations that result in exon 14 skipping have been observed in ~3% of all lung cancers.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"D\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>484, \"name\"=>\"MET gene exon 14 deletion created using the CRISPR/Cas9 system enhances cellular growth and sensitivity to a MET inhibitor.\", \"citation\"=>\"Togashi et al., 2015, Lung Cancer\", \"pubmed_id\"=>\"26547802\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/26547802\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2015, \"month\"=>12}, \"journal\"=>\"Lung Cancer\", \"full_journal_title\"=>\"Lung cancer (Amsterdam, Netherlands)\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>324, \"phenotypes\"=>[]}, {\"id\"=>797, \"name\"=>\"EID797\", \"description\"=>\"We now report responses to the MET inhibitors crizotinib\\nand cabozantinib in four patients with stage IV lung adenocarcinomas harboring mutations leading\\nto MET exon 14 skipping.\", \"disease\"=>{\"id\"=>26, \"name\"=>\"Lung Adenocarcinoma\", \"display_name\"=>\"Lung Adenocarcinoma\", \"doid\"=>\"3910\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3910\"}, \"drugs\"=>[{\"id\"=>12, \"name\"=>\"Crizotinib\", \"pubchem_id\"=>nil}], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Sensitivity\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>531, \"name\"=>\"Response to MET inhibitors in patients with stage IV lung adenocarcinomas harboring MET mutations causing exon 14 skipping.\", \"citation\"=>\"Paik et al., 2015, Cancer Discov\", \"pubmed_id\"=>\"25971939\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25971939\", \"open_access\"=>true, \"pmc_id\"=>\"PMC4658654\", \"publication_date\"=>{\"year\"=>2015, \"month\"=>8}, \"journal\"=>\"Cancer Discov\", \"full_journal_title\"=>\"Cancer discovery\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>324, \"phenotypes\"=>[]}]}, {\"id\"=>843, \"entrez_name\"=>\"VHL\", \"entrez_id\"=>7428, \"name\"=>\"EXON 1-3 DELETION\", \"description\"=>\"\", \"gene_id\"=>58, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>38, \"name\"=>\"exon_loss_variant\", \"display_name\"=>\"Exon Loss Variant\", \"so_id\"=>\"SO:0001572\", \"description\"=>\"A sequence variant whereby an exon is lost from the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001572\"}], \"civic_actionability_score\"=>10.0, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>10183532, \"stop\"=>10191649, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000256474.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/58/summary/variants/843/summary#variant\", \"evidence_items\"=>[{\"id\"=>5227, \"name\"=>\"EID5227\", \"description\"=>\"In a study of 426 unrelated VHL patients, 111 were discovered to have alterations in the VHL gene. 18 novel variants were identified in VHL patients, but none were present in 200 unaffected control individuals. This mutation was found in 14 separate VHL patients. Twelve of 14 cases show hemangioblastomas of the central nervous system. Of those 12 patients, others had retinal hemangioblastomas, renal cell carcinoma, multiple pancreatic cysts, and multiple renal cysts (index cases 51, 65, 67, 108, 116, 124, 147, 283, 302, 339, 356, 204). The other 2 patients had pancreatic cysts and renal cell carcinoma (index cases 226, 384). This study contains very strong evidence of pathogenicity for a null variant in a gene where loss-of-function is a known mechanism of disease (ACMG code: PVS1). Relevant HPO terms: hemangioblastoma, renal cell carcinoma, multiple Renal cysts, multiple pancreatic cysts.\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2174, \"name\"=>\"Identification and in silico analysis of novel von Hippel-Lindau (VHL) gene variants from a large population.\", \"citation\"=>\"Leonardi et al., 2011, Ann. Hum. Genet.\", \"pubmed_id\"=>\"21463266\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21463266\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>7}, \"journal\"=>\"Ann. Hum. Genet.\", \"full_journal_title\"=>\"Annals of human genetics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>843, \"phenotypes\"=>[]}, {\"id\"=>1942, \"name\"=>\"EID1942\", \"description\"=>\"Mutation detected in 4 separate cases. Two cases show CNS hemangioblastoma, pancreatic/hepatic/renal cysts, and renal cell carcinoma. One case shows CNS hemangioblastoma, pancreatic cysts, and Grawitz. ACMG evidence codes: 'PVS1' based on deletion of all exons, 'PP1' based on segregation results, 'PP4' because the patients phenotypes are highly specific for a disease with single genetic etiology.\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>4, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Pathogenic\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1368, \"name\"=>\"Genetic analysis of von Hippel-Lindau disease.\", \"citation\"=>\"Nordstrom-O'Brien et al., 2010, Hum. Mutat.\", \"pubmed_id\"=>\"20151405\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20151405\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>5}, \"journal\"=>\"Hum. Mutat.\", \"full_journal_title\"=>\"Human mutation\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>843, \"phenotypes\"=>[]}]}, {\"id\"=>844, \"entrez_name\"=>\"VHL\", \"entrez_id\"=>7428, \"name\"=>\"EXON 1 DELETION\", \"description\"=>\"\", \"gene_id\"=>58, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>38, \"name\"=>\"exon_loss_variant\", \"display_name\"=>\"Exon Loss Variant\", \"so_id\"=>\"SO:0001572\", \"description\"=>\"A sequence variant whereby an exon is lost from the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001572\"}], \"civic_actionability_score\"=>7.5, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>10183532, \"stop\"=>10183871, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000256474.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/58/summary/variants/844/summary#variant\", \"evidence_items\"=>[{\"id\"=>5222, \"name\"=>\"EID5222\", \"description\"=>\"In a study of 426 unrelated VHL patients, 111 were discovered to have alterations in the VHL gene. 18 novel variants were identified in VHL patients, but none were present in 200 unaffected control individuals. This mutation was found in two VHL patients with hemangioblastomas of the central nervous system. The 48 yr old patient had retinal hemangioblastomas and the 30 yr old patient had renal cyst lesions(index case 437, 347). This study has very strong evidence of pathogenicity for this single exon deletion in a gene where loss-of-function is a known mechanism of disease (ACMG code: PVS1).\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>1, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2174, \"name\"=>\"Identification and in silico analysis of novel von Hippel-Lindau (VHL) gene variants from a large population.\", \"citation\"=>\"Leonardi et al., 2011, Ann. Hum. Genet.\", \"pubmed_id\"=>\"21463266\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21463266\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>7}, \"journal\"=>\"Ann. Hum. Genet.\", \"full_journal_title\"=>\"Annals of human genetics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>844, \"phenotypes\"=>[]}, {\"id\"=>5249, \"name\"=>\"EID5249\", \"description\"=>\"Germline mutation analysis of seven east-Chinese families revealed 6 mutations. This mutation was found two separate VHL families. One family contained a male VHL type 1 patient with hemangioblastomas of the central nervous system and a male asymptomatic patient (family A). The other family contained two male VHL type patients with hemangioblastomas of the central nervous system and 4 asymptomatic family members (family E). This study contains very strong evidence for pathogenicity in the form of a single exon deletion variant in a gene where loss-of-function is a known mechanism of disease (ACMG code: PVS1).\\nHPO terms: hemangioblastoma.\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2215, \"name\"=>\"Germline mutations of the VHL gene in seven Chinese families with von Hippel-Lindau disease.\", \"citation\"=>\"Huang et al., 2012, Int. J. Mol. Med.\", \"pubmed_id\"=>\"21972040\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21972040\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2012, \"month\"=>1}, \"journal\"=>\"Int. J. Mol. Med.\", \"full_journal_title\"=>\"International journal of molecular medicine\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>844, \"phenotypes\"=>[]}, {\"id\"=>1943, \"name\"=>\"EID1943\", \"description\"=>\"Mutation detected in one case. Case shows CNS hemangioblastoma. ACMG evidence codes: 'PVS1' for a single exon deletion, 'PP1' based on segregation results, 'PP4' because patients phenotypes are highly specific for a disease with single genetic etiology.\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Pathogenic\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1368, \"name\"=>\"Genetic analysis of von Hippel-Lindau disease.\", \"citation\"=>\"Nordstrom-O'Brien et al., 2010, Hum. Mutat.\", \"pubmed_id\"=>\"20151405\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20151405\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>5}, \"journal\"=>\"Hum. Mutat.\", \"full_journal_title\"=>\"Human mutation\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>844, \"phenotypes\"=>[]}]}, {\"id\"=>845, \"entrez_name\"=>\"VHL\", \"entrez_id\"=>7428, \"name\"=>\"EXON 1-2 DELETION\", \"description\"=>\"\", \"gene_id\"=>58, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>38, \"name\"=>\"exon_loss_variant\", \"display_name\"=>\"Exon Loss Variant\", \"so_id\"=>\"SO:0001572\", \"description\"=>\"A sequence variant whereby an exon is lost from the transcript.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001572\"}], \"civic_actionability_score\"=>7.5, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>10183532, \"stop\"=>10188320, \"reference_bases\"=>nil, \"variant_bases\"=>nil, \"representative_transcript\"=>\"ENST00000256474.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/58/summary/variants/845/summary#variant\", \"evidence_items\"=>[{\"id\"=>5225, \"name\"=>\"EID5225\", \"description\"=>\"In a study of 426 unrelated VHL patients, 111 were discovered to have alterations in the VHL gene. 18 novel variants were identified in VHL patients, but none were present in 200 unaffected control individuals. This mutation was found in a 52 yr old VHL patient with hemangioblastomas of the central nervous system (index case 378). This study contains very strong evidence of pathogenicity for this null variant in a gene where loss-of-function is a known mechanism of disease (ACMG code: PVS1). Relevant HPO terms: Hemangioblastoma.\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"submitted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>2174, \"name\"=>\"Identification and in silico analysis of novel von Hippel-Lindau (VHL) gene variants from a large population.\", \"citation\"=>\"Leonardi et al., 2011, Ann. Hum. Genet.\", \"pubmed_id\"=>\"21463266\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/21463266\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2011, \"month\"=>7}, \"journal\"=>\"Ann. Hum. Genet.\", \"full_journal_title\"=>\"Annals of human genetics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>845, \"phenotypes\"=>[]}, {\"id\"=>1944, \"name\"=>\"EID1944\", \"description\"=>\"Mutation detected in one case. Case shows renal cell carcinoma. ACMG evidence codes: 'PP1' based on segregation results, 'PP4' the patients phenotypes are highly specific for a disease with single genetic etiology.\", \"disease\"=>{\"id\"=>2198, \"name\"=>\"Von Hippel-Lindau Disease\", \"display_name\"=>\"Von Hippel-Lindau Disease\", \"doid\"=>\"14175\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:14175\"}, \"drugs\"=>[], \"rating\"=>3, \"evidence_level\"=>\"C\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>2, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1368, \"name\"=>\"Genetic analysis of von Hippel-Lindau disease.\", \"citation\"=>\"Nordstrom-O'Brien et al., 2010, Hum. Mutat.\", \"pubmed_id\"=>\"20151405\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/20151405\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2010, \"month\"=>5}, \"journal\"=>\"Hum. Mutat.\", \"full_journal_title\"=>\"Human mutation\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>845, \"phenotypes\"=>[]}]}, {\"id\"=>731, \"entrez_name\"=>\"MLH1\", \"entrez_id\"=>4292, \"name\"=>\"C.790+1G>A\", \"description\"=>\"\", \"gene_id\"=>3532, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>41, \"name\"=>\"splice_donor_variant\", \"display_name\"=>\"Splice Donor Variant\", \"so_id\"=>\"SO:0001575\", \"description\"=>\"A splice variant that changes the 2 base pair region at the 5' end of an intron.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001575\"}], \"civic_actionability_score\"=>0.5, \"coordinates\"=>{\"chromosome\"=>\"3\", \"start\"=>37056036, \"stop\"=>37056036, \"reference_bases\"=>\"G\", \"variant_bases\"=>\"A\", \"representative_transcript\"=>\"ENST0000023170.2\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/3532/summary/variants/731/summary#variant\", \"evidence_items\"=>[{\"id\"=>1794, \"name\"=>\"EID1794\", \"description\"=>\"This variant, identified in a case of microsatellite-unstable colorectal cancer was confirmed to be somatic in a patient with suspected Lynch Syndrome (LS). The patient was therefore negative for LS. However, based on this study we can infer that the variant would predispose to LS in a germline setting. This patient also harbored a p.K618del mutation.\", \"disease\"=>{\"id\"=>2194, \"name\"=>\"Lynch Syndrome\", \"display_name\"=>\"Lynch Syndrome\", \"doid\"=>\"3883\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:3883\"}, \"drugs\"=>[], \"rating\"=>2, \"evidence_level\"=>\"E\", \"evidence_type\"=>\"Predisposing\", \"clinical_significance\"=>\"Uncertain Significance\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Somatic Mutation\", \"drug_interaction_type\"=>nil, \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1251, \"name\"=>\"Somatic aberrations of mismatch repair genes as a cause of microsatellite-unstable cancers.\", \"citation\"=>\"Geurts-Giele et al., 2014, J. Pathol.\", \"pubmed_id\"=>\"25111426\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/25111426\", \"open_access\"=>nil, \"pmc_id\"=>nil, \"publication_date\"=>{\"year\"=>2014, \"month\"=>12}, \"journal\"=>\"J. Pathol.\", \"full_journal_title\"=>\"The Journal of pathology\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>731, \"phenotypes\"=>[]}]}, {\"id\"=>737, \"entrez_name\"=>\"DPYD\", \"entrez_id\"=>1806, \"name\"=>\"DPYD*2A HOMOZYGOSITY\", \"description\"=>\"\", \"gene_id\"=>1526, \"type\"=>\"variant\", \"variant_types\"=>[{\"id\"=>41, \"name\"=>\"splice_donor_variant\", \"display_name\"=>\"Splice Donor Variant\", \"so_id\"=>\"SO:0001575\", \"description\"=>\"A splice variant that changes the 2 base pair region at the 5' end of an intron.\", \"url\"=>\"http://www.sequenceontology.org/browser/current_svn/term/SO:0001575\"}], \"civic_actionability_score\"=>50.0, \"coordinates\"=>{\"chromosome\"=>\"1\", \"start\"=>97915614, \"stop\"=>97915614, \"reference_bases\"=>\"C\", \"variant_bases\"=>\"T\", \"representative_transcript\"=>\"ENST00000370192.3\", \"chromosome2\"=>nil, \"start2\"=>nil, \"stop2\"=>nil, \"representative_transcript2\"=>nil, \"ensembl_version\"=>75, \"reference_build\"=>\"GRCh37\"}, \"variant_url\"=>\"/events/genes/1526/summary/variants/737/summary#variant\", \"evidence_items\"=>[{\"id\"=>1800, \"name\"=>\"EID1800\", \"description\"=>\"The Clinical Pharmacogenomics Implementation Consortium Guidelines for DPYD genotype and 5-FU dosing does not recommend the use of 5-FU or capecitabine in patients with homozygous DPYD*2A, *13 or rs67376798 and the FDA drug labels warn to use precautions in this patient population.\", \"disease\"=>{\"id\"=>216, \"name\"=>\"Cancer\", \"display_name\"=>\"Cancer\", \"doid\"=>\"162\", \"url\"=>\"http://www.disease-ontology.org/?id=DOID:162\"}, \"drugs\"=>[{\"id\"=>32, \"name\"=>\"Capecitabine\", \"pubchem_id\"=>nil}, {\"id\"=>420, \"name\"=>\"5-FU\", \"pubchem_id\"=>nil}, {\"id\"=>421, \"name\"=>\"Tegafur\", \"pubchem_id\"=>nil}], \"rating\"=>5, \"evidence_level\"=>\"A\", \"evidence_type\"=>\"Predictive\", \"clinical_significance\"=>\"Adverse Response\", \"evidence_direction\"=>\"Supports\", \"variant_origin\"=>\"Germline Polymorphism\", \"drug_interaction_type\"=>\"Substitutes\", \"status\"=>\"accepted\", \"open_change_count\"=>0, \"type\"=>\"evidence\", \"source\"=>{\"id\"=>1253, \"name\"=>\"Clinical Pharmacogenetics Implementation Consortium guidelines for dihydropyrimidine dehydrogenase genotype and fluoropyrimidine dosing.\", \"citation\"=>\"Caudle et al., 2013, Clin. Pharmacol. Ther.\", \"pubmed_id\"=>\"23988873\", \"source_url\"=>\"http://www.ncbi.nlm.nih.gov/pubmed/23988873\", \"open_access\"=>true, \"pmc_id\"=>\"PMC3831181\", \"publication_date\"=>{\"year\"=>2013, \"month\"=>12}, \"journal\"=>\"Clin. Pharmacol. Ther.\", \"full_journal_title\"=>\"Clinical pharmacology and therapeutics\", \"status\"=>\"fully curated\", \"is_review\"=>false, \"clinical_trials\"=>[]}, \"variant_id\"=>737, \"phenotypes\"=>[]}]}]"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#pull in CIViC API for RNA-based variants \n",
    "url_RNA = 'https://civic.genome.wustl.edu/api/panels/RNA-based/qualifying_variants?minimum_score=0'\n",
    "resp_RNA = Net::HTTP.get_response(URI.parse(url_RNA))\n",
    "variants_RNA = JSON.parse(resp_RNA.body)['records']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[[\"chrom\", \"start\", \"stop\", \"chrom2\", \"start2\", \"stop2\", \"gene\", \"variant\", \"pipeline\"]]"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#set list for RNA-based capture\n",
    "capture_RNA = []\n",
    "capture_RNA << [\"chrom\", \"start\", \"stop\", \"chrom2\", \"start2\", \"stop2\", \"gene\", \"variant\", \"pipeline\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[[\"chrom\", \"start\", \"stop\", \"chrom2\", \"start2\", \"stop2\", \"gene\", \"variant\", \"pipeline\"], [\"11\", 69455855, 69469242, \" \", \" \", \" \", \"CCND1\", \"EXPRESSION\", \"RNA-based\"], [\"11\", 69455855, 69469242, \" \", \" \", \" \", \"CCND1\", \"OVEREXPRESSION\", \"RNA-based\"], [\"12\", 4382938, 4414516, \" \", \" \", \" \", \"CCND2\", \"OVEREXPRESSION\", \"RNA-based\"], [\"12\", 4381437, 4382937, \" \", \" \", \" \", \"CCND2\", \"PROMOTER DEMETHYLATION\", \"RNA-based\"], [\"19\", 30302805, 30315215, \" \", \" \", \" \", \"CCNE1\", \"OVEREXPRESSION\", \"RNA-based\"], [\"12\", 58141510, 58146304, \" \", \" \", \" \", \"CDK4\", \"EXPRESSION\", \"RNA-based\"], [\"9\", 21974827, 21994591, \" \", \" \", \" \", \"CDKN2A\", \"PROMOTER HYPERMETHYLATION\", \"RNA-based\"], [\"3\", 128198270, 128212028, \" \", \" \", \" \", \"GATA2\", \"EXPRESSION\", \"RNA-based\"], [\"10\", 131264495, 131265656, \" \", \" \", \" \", \"MGMT\", \"PROMOTER METHYLATION\", \"RNA-based\"], [\"18\", 60790579, 60987361, \" \", \" \", \" \", \"BCL2\", \"EXPRESSION\", \"RNA-based\"], [\"11\", 2150348, 2170833, \" \", \" \", \" \", \"IGF2\", \"OVEREXPRESSION\", \"RNA-based\"], [\"20\", 54944446, 54967393, \" \", \" \", \" \", \"AURKA\", \"EXPRESSION\", \"RNA-based\"], [\"11\", 108093559, 108239826, \" \", \" \", \" \", \"ATM\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"20\", 54944446, 54967393, \" \", \" \", \" \", \"AURKA\", \"OVEREXPRESSION\", \"RNA-based\"], [\"3\", 52435029, 52444366, \" \", \" \", \" \", \"BAP1\", \"EXPRESSION\", \"RNA-based\"], [\"7\", 55086794, 55279321, \" \", \" \", \" \", \"EGFR\", \"OVEREXPRESSION\", \"RNA-based\"], [\"20\", 46130646, 46285621, \" \", \" \", \" \", \"NCOA3\", \"OVEREXPRESSION\", \"RNA-based\"], [\"11\", 100900355, 101001255, \" \", \" \", \" \", \"PGR\", \"EXPRESSION\", \"RNA-based\"], [\"8\", 38268656, 38325363, \" \", \" \", \" \", \"FGFR1\", \"EXPRESSION\", \"RNA-based\"], [\"9\", 21968055, 21974865, \" \", \" \", \" \", \"CDKN2A\", \"p16 EXPRESSION\", \"RNA-based\"], [\"9\", 5450525, 5470547, \" \", \" \", \" \", \"CD274\", \"EXPRESSION\", \"RNA-based\"], [\"19\", 40736224, 40791302, \" \", \" \", \" \", \"AKT2\", \"EXPRESSION\", \"RNA-based\"], [\"18\", 48556583, 48611409, \" \", \" \", \" \", \"SMAD4\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"12\", 56473645, 56497289, \" \", \" \", \" \", \"ERBB3\", \"OVEREXPRESSION\", \"RNA-based\"], [\"6\", 86159809, 86205496, \" \", \" \", \" \", \"NT5E\", \"OVEREXPRESSION\", \"RNA-based\"], [\"7\", 148504477, 148581370, \" \", \" \", \" \", \"EZH2\", \"OVEREXPRESSION\", \"RNA-based\"], [\"3\", 38179969, 38184510, \" \", \" \", \" \", \"MYD88\", \"OVEREXPRESSION\", \"RNA-based\"], [\"10\", 43572475, 43625799, \" \", \" \", \" \", \"RET\", \"EXPRESSION\", \"RNA-based\"], [\"1\", 155870065, 155880706, \" \", \" \", \" \", \"RIT1\", \"OVEREXPRESSION\", \"RNA-based\"], [\"19\", 1205740, 1228428, \" \", \" \", \" \", \"STK11\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"9\", 93564069, 93660831, \" \", \" \", \" \", \"SYK\", \"OVEREXPRESSION\", \"RNA-based\"], [\"10\", 89622870, 89731687, \" \", \" \", \" \", \"PTEN\", \"EXPRESSION\", \"RNA-based\"], [\"8\", 31497942, 32622294, \" \", \" \", \" \", \"NRG1\", \"EXPRESSION\", \"RNA-based\"], [\"4\", 75310851, 75320726, \" \", \" \", \" \", \"AREG\", \"EXPRESSION\", \"RNA-based\"], [\"2\", 70674412, 70781147, \" \", \" \", \" \", \"TGFA\", \"EXPRESSION\", \"RNA-based\"], [\"4\", 75230860, 75254468, \" \", \" \", \" \", \"EREG\", \"EXPRESSION\", \"RNA-based\"], [\"4\", 1795039, 1810599, \" \", \" \", \" \", \"FGFR3\", \"EXPRESSION\", \"RNA-based\"], [\"19\", 45281126, 45303891, \" \", \" \", \" \", \"CBLC\", \"EXPRESSION\", \"RNA-based\"], [\"8\", 6835172, 6837602, \" \", \" \", \" \", \"DEFA1\", \"EXPRESSION\", \"RNA-based\"], [\"19\", 45916692, 45926824, \" \", \" \", \" \", \"ERCC1\", \"EXPRESSION\", \"RNA-based\"], [\"6\", 43738444, 43752346, \" \", \" \", \" \", \"VEGFA\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"19\", 10244022, 10305811, \" \", \" \", \" \", \"DNMT1\", \"EXPRESSION\", \"RNA-based\"], [\"10\", 112631596, 112659763, \" \", \" \", \" \", \"PDCD4\", \"EXPRESSION\", \"RNA-based\"], [\"21\", 43731777, 43735761, \" \", \" \", \" \", \"TFF3\", \"EXPRESSION\", \"RNA-based\"], [\"10\", 54074056, 54077417, \" \", \" \", \" \", \"DKK1\", \"NUCLEAR EXPRESSION\", \"RNA-based\"], [\"11\", 35160728, 35251574, \" \", \" \", \" \", \"CD44\", \"ISOFORM EXPRESSION\", \"RNA-based\"], [\"4\", 110834047, 110933422, \" \", \" \", \" \", \"EGF\", \"EXPRESSION\", \"RNA-based\"], [\"12\", 69201956, 69239214, \" \", \" \", \" \", \"MDM2\", \"EXPRESSION\", \"RNA-based\"], [\"22\", 35776828, 35790207, \" \", \" \", \" \", \"HMOX1\", \"EXPRESSION\", \"RNA-based\"], [\"17\", 74380731, 74383936, \" \", \" \", \" \", \"SPHK1\", \"OVEREXPRESSION\", \"RNA-based\"], [\"2\", 202122759, 202152434, \" \", \" \", \" \", \"CASP8\", \"EXPRESSION\", \"RNA-based\"], [\"18\", 657604, 673578, \" \", \" \", \" \", \"TYMS\", \"EXPRESSION\", \"RNA-based\"], [\"8\", 27667137, 27695612, \" \", \" \", \" \", \"PBK\", \"OVEREXPRESSION\", \"RNA-based\"], [\"19\", 11071598, 11172958, \" \", \" \", \" \", \"SMARCA4\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"7\", 55086794, 55279321, \" \", \" \", \" \", \"EGFR\", \"EXPRESSION\", \"RNA-based\"], [\"6\", 36646487, 36655108, \" \", \" \", \" \", \"CDKN1A\", \"EXPRESSION\", \"RNA-based\"], [\"3\", 105085753, 105295744, \" \", \" \", \" \", \"ALCAM\", \"EXPRESSION\", \"RNA-based\"], [\"1\", 26211931, 26232957, \" \", \" \", \" \", \"STMN1\", \"EXPRESSION\", \"RNA-based\"], [\"X\", 66764465, 66950461, \" \", \" \", \" \", \"AR\", \"OVEREXPRESSION\", \"RNA-based\"], [\"12\", 12870058, 12875305, \" \", \" \", \" \", \"CDKN1B\", \"CYTOPLASMIC MISLOCALIZATION\", \"RNA-based\"], [\"12\", 56473645, 56497289, \" \", \" \", \" \", \"ERBB3\", \"EXPRESSION\", \"RNA-based\"], [\"2\", 234526291, 234681956, \" \", \" \", \" \", \"UGT1A\", \"EXPRESSION\", \"RNA-based\"], [\"2\", 212240446, 213403565, \" \", \" \", \" \", \"ERBB4\", \"NUCLEAR TRANSLOCATION\", \"RNA-based\"], [\"2\", 212240446, 213403565, \" \", \" \", \" \", \"ERBB4\", \"EXPRESSION\", \"RNA-based\"], [\"17\", 33677324, 33700639, \" \", \" \", \" \", \"SLFN11\", \"EXPRESSION\", \"RNA-based\"], [\"17\", 76210267, 76221717, \" \", \" \", \" \", \"BIRC5\", \"NUCLEAR EXPRESSION\", \"RNA-based\"], [\"12\", 12870058, 12875305, \" \", \" \", \" \", \"CDKN1B\", \"EXPRESSION\", \"RNA-based\"], [\"7\", 16831435, 16844704, \" \", \" \", \" \", \"AGR2\", \"EXPRESSION\", \"RNA-based\"], [\"10\", 31608101, 31818741, \" \", \" \", \" \", \"ZEB1\", \"EXPRESSION\", \"RNA-based\"], [\"X\", 47441712, 47446188, \" \", \" \", \" \", \"TIMP1\", \"OVEREXPRESSION\", \"RNA-based\"], [\"8\", 142402093, 142441394, \" \", \" \", \" \", \"PTP4A3\", \"OVEREXPRESSION\", \"RNA-based\"], [\"15\", 99192200, 99507759, \" \", \" \", \" \", \"IGF1R\", \"EXPRESSION\", \"RNA-based\"], [\"7\", 100400187, 100425121, \" \", \" \", \" \", \"EPHB4\", \"EXPRESSION\", \"RNA-based\"], [\"18\", 657604, 673578, \" \", \" \", \" \", \"TYMS\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"17\", 38544768, 38574202, \" \", \" \", \" \", \"TOP2A\", \"EXPRESSION\", \"RNA-based\"], [\"20\", 39657458, 39753127, \" \", \" \", \" \", \"TOP1\", \"EXPRESSION\", \"RNA-based\"], [\"1\", 57159476, 57159478, \" \", \" \", \" \", \"PRKAA2\", \"T172 PHOSPHORYLATION\", \"RNA-based\"], [\"9\", 127997132, 128003609, \" \", \" \", \" \", \"HSPA5\", \"EXPRESSION\", \"RNA-based\"], [\"8\", 37888200, 37914663, \" \", \" \", \" \", \"EIF4EBP1\", \"PHOSPHORYLATION\", \"RNA-based\"], [\"16\", 89989687, 90002505, \" \", \" \", \" \", \"TUBB3\", \"EXPRESSION\", \"RNA-based\"], [\"7\", 55272951, 55272953, \" \", \" \", \" \", \"EGFR\", \"Y1092 PHOSPHORYLATION\", \"RNA-based\"], [\"15\", 99192200, 99507759, \" \", \" \", \" \", \"IGF1R\", \"NUCLEAR EXPRESSION\", \"RNA-based\"], [\"2\", 201980827, 202041410, \" \", \" \", \" \", \"CFLAR\", \"EXPRESSION\", \"RNA-based\"], [\"4\", 15969857, 16085324, \" \", \" \", \" \", \"PROM1\", \"EXPRESSION\", \"RNA-based\"], [\"X\", 49106897, 49121288, \" \", \" \", \" \", \"FOXP3\", \"EXPRESSION\", \"RNA-based\"], [\"9\", 19376309, 19376323, \" \", \" \", \" \", \"RPS6\", \"PHOSPHORYLATION\", \"RNA-based\"], [\"1\", 65298912, 65432187, \" \", \" \", \" \", \"JAK1\", \"OVEREXPRESSION\", \"RNA-based\"], [\"17\", 41196312, 41277387, \" \", \" \", \" \", \"BRCA1\", \"EXPRESSION\", \"RNA-based\"], [\"12\", 89741009, 89747048, \" \", \" \", \" \", \"DUSP6\", \"EXPRESSION\", \"RNA-based\"], [\"7\", 75931861, 75933612, \" \", \" \", \" \", \"HSPB1\", \"EXPRESSION\", \"RNA-based\"], [\"16\", 69743304, 69760854, \" \", \" \", \" \", \"NQO1\", \"EXPRESSION\", \"RNA-based\"], [\"17\", 41196312, 41277387, \" \", \" \", \" \", \"BRCA1\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"7\", 87133175, 87342564, \" \", \" \", \" \", \"ABCB1\", \"EXPRESSION\", \"RNA-based\"], [\"2\", 136871919, 136873813, \" \", \" \", \" \", \"CXCR4\", \"EXPRESSION\", \"RNA-based\"], [\"X\", 55478538, 55479998, \" \", \" \", \" \", \"MAGEH1\", \"EXPRESSION\", \"RNA-based\"], [\"6\", 43395292, 43418168, \" \", \" \", \" \", \"ABCC10\", \"EXPRESSION\", \"RNA-based\"], [\"13\", 28536274, 28545276, \" \", \" \", \" \", \"CDX2\", \"EXPRESSION\", \"RNA-based\"], [\"6\", 31132120, 31138470, \" \", \" \", \" \", \"POU5F1\", \"EXPRESSION\", \"RNA-based\"], [\"17\", 37856333, 37884915, \" \", \" \", \" \", \"ERBB2\", \"SERUM LEVELS\", \"RNA-based\"], [\"6\", 169615875, 169654139, \" \", \" \", \" \", \"THBS2\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"X\", 123095568, 123236506, \" \", \" \", \" \", \"STAG2\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"6\", 32407619, 32412823, \" \", \" \", \" \", \"HLA-DRA\", \"EXPRESSION\", \"RNA-based\"], [\"4\", 55524085, 55606881, \" \", \" \", \" \", \"KIT\", \"EXPRESSION\", \"RNA-based\"], [\"14\", 62162258, 62214976, \" \", \" \", \" \", \"HIF1A\", \"OVEREXPRESSION\", \"RNA-based\"], [\"2\", 46524541, 46613836, \" \", \" \", \" \", \"EPAS1\", \"OVEREXPRESSION\", \"RNA-based\"], [\"6\", 32808494, 32812480, \" \", \" \", \" \", \"PSMB8\", \"NUCLEAR EXPRESSION\", \"RNA-based\"], [\"8\", 27667137, 27695612, \" \", \" \", \" \", \"PBK\", \"NUCLEAR EXPRESSION\", \"RNA-based\"], [\"5\", 159848829, 159855748, \" \", \" \", \" \", \"PTTG1\", \"OVEREXPRESSION\", \"RNA-based\"], [\"1\", 65886335, 66107242, \" \", \" \", \" \", \"LEPR\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"6\", 11185051, 11382581, \" \", \" \", \" \", \"NEDD9\", \"EXPRESSION\", \"RNA-based\"], [\"12\", 53342655, 53346684, \" \", \" \", \" \", \"KRT18\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"X\", 137715011, 138286269, \" \", \" \", \" \", \"FGF13\", \"CYTOPLASMIC EXPRESSION\", \"RNA-based\"], [\"15\", 58245622, 58358616, \" \", \" \", \" \", \"ALDH1A2\", \"EXPRESSION\", \"RNA-based\"], [\"15\", 58245622, 58358616, \" \", \" \", \" \", \"ALDH1A2\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"3\", 108268716, 108308299, \" \", \" \", \" \", \"KIAA1524\", \"EXPRESSION\", \"RNA-based\"], [\"3\", 108268716, 108308299, \" \", \" \", \" \", \"KIAA1524\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"4\", 20529898, 20530007, \" \", \" \", \" \", \"MIR218-1\", \"EXPRESSION\", \"RNA-based\"], [\"6\", 74104471, 74127292, \" \", \" \", \" \", \"DDX43\", \"OVEREXPRESSION\", \"RNA-based\"], [\"12\", 64845660, 64895888, \" \", \" \", \" \", \"TBK1\", \"OVEREXPRESSION\", \"RNA-based\"], [\"5\", 156512843, 156536725, \" \", \" \", \" \", \"HAVCR2\", \"OVEREXPRESSION\", \"RNA-based\"], [\"16\", 55512883, 55540603, \" \", \" \", \" \", \"MMP2\", \"SERUM LEVELS\", \"RNA-based\"], [\"20\", 44637547, 44645200, \" \", \" \", \" \", \"MMP9\", \"SERUM LEVELS\", \"RNA-based\"], [\"13\", 31710765, 31736064, \" \", \" \", \" \", \"HSPH1\", \"NUCLEAR EXPRESSION\", \"RNA-based\"], [\"11\", 124753587, 124768177, \" \", \" \", \" \", \"ROBO4\", \"EXPRESSION\", \"RNA-based\"], [\"6\", 37137979, 37143202, \" \", \" \", \" \", \"PIM1\", \"NUCLEAR EXPRESSION\", \"RNA-based\"], [\"4\", 20529898, 20530007, \" \", \" \", \" \", \"MIR218-1\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"9\", 33110635, 33167354, \" \", \" \", \" \", \"B4GALT1\", \"EXPRESSION\", \"RNA-based\"], [\"10\", 69644427, 69678147, \" \", \" \", \" \", \"SIRT1\", \"OVEREXPRESSION\", \"RNA-based\"], [\"18\", 48556583, 48611409, \" \", \" \", \" \", \"SMAD4\", \"EXPRESSION\", \"RNA-based\"], [\"13\", 48877911, 49056122, \" \", \" \", \" \", \"RB1\", \"EXPRESSION\", \"RNA-based\"], [\"12\", 112856718, 112947717, \" \", \" \", \" \", \"PTPN11\", \"EXPRESSION\", \"RNA-based\"], [\"1\", 229566992, 229569841, \" \", \" \", \" \", \"ACTA1\", \"EXPRESSION\", \"RNA-based\"], [\"13\", 48877911, 49056122, \" \", \" \", \" \", \"RB1\", \"OVEREXPRESSION\", \"RNA-based\"], [\"17\", 76210267, 76221717, \" \", \" \", \" \", \"BIRC5\", \"OVEREXPRESSION\", \"RNA-based\"], [\"10\", 43572475, 43625799, \" \", \" \", \" \", \"RET\", \"OVEREXPRESSION\", \"RNA-based\"], [\"7\", 92234235, 92463231, \" \", \" \", \" \", \"CDK6\", \"OVEREXPRESSION\", \"RNA-based\"], [\"13\", 28577411, 28674729, \" \", \" \", \" \", \"FLT3\", \"OVEREXPRESSION\", \"RNA-based\"], [\"6\", 152128686, 152424406, \" \", \" \", \" \", \"ESR1\", \"OVEREXPRESSION\", \"RNA-based\"], [\"1\", 186640923, 186649559, \" \", \" \", \" \", \"PTGS2\", \"OVEREXPRESSION\", \"RNA-based\"], [\"7\", 116312459, 116436396, \" \", \" \", \" \", \"MET\", \"OVEREXPRESSION\", \"RNA-based\"], [\"10\", 129894923, 129924649, \" \", \" \", \" \", \"MKI67\", \"EXPRESSION\", \"RNA-based\"], [\"9\", 110045560, 110094475, \" \", \" \", \" \", \"RAD23B\", \"EXPRESSION\", \"RNA-based\"], [\"13\", 48877911, 49056122, \" \", \" \", \" \", \"RB1\", \"PHOSPHORYLATION\", \"RNA-based\"], [\"1\", 59246465, 59249785, \" \", \" \", \" \", \"JUN\", \"OVEREXPRESSION\", \"RNA-based\"], [\"14\", 75745477, 75748933, \" \", \" \", \" \", \"FOS\", \"OVEREXPRESSION\", \"RNA-based\"], [\"X\", 76760356, 77041702, \" \", \" \", \" \", \"ATRX\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"19\", 41725108, 41767671, \" \", \" \", \" \", \"AXL\", \"EXPRESSION\", \"RNA-based\"], [\"13\", 114523524, 114567046, \" \", \" \", \" \", \"GAS6\", \"EXPRESSION\", \"RNA-based\"], [\"17\", 41605212, 41623800, \" \", \" \", \" \", \"ETV4\", \"OVEREXPRESSION\", \"RNA-based\"], [\"7\", 99775186, 99812003, \" \", \" \", \" \", \"STAG3\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"14\", 62164340, 62214489, \" \", \" \", \" \", \"HIF1A\", \"EXPRESSION\", \"RNA-based\"], [\"3\", 37034823, 37092409, \" \", \" \", \" \", \"MLH1\", \"EXPRESSION\", \"RNA-based\"], [\"1\", 40361098, 40367685, \" \", \" \", \" \", \"MYCL\", \"EXPRESSION\", \"RNA-based\"], [\"7\", 81328322, 81399514, \" \", \" \", \" \", \"HGF\", \"EXPRESSION\", \"RNA-based\"], [\"4\", 123747863, 123819391, \" \", \" \", \" \", \"FGF2\", \"EXPRESSION\", \"RNA-based\"], [\"2\", 113973574, 114036527, \" \", \" \", \" \", \"PAX8\", \"EXPRESSION\", \"RNA-based\"], [\"6\", 134490384, 134639196, \" \", \" \", \" \", \"SGK1\", \"OVEREXPRESSION\", \"RNA-based\"], [\"22\", 24129150, 24176703, \" \", \" \", \" \", \"SMARCB1\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"15\", 69706688, 69740756, \" \", \" \", \" \", \"KIF23\", \"EXPRESSION\", \"RNA-based\"], [\"9\", 22009121, 22009172, \" \", \" \", \" \", \"CDKN2B\", \"METHYLATION\", \"RNA-based\"], [\"11\", 4116039, 4160106, \" \", \" \", \" \", \"RRM1\", \"OVEREXPRESSION\", \"RNA-based\"], [\"2\", 10262455, 10271545, \" \", \" \", \" \", \"RRM2\", \"OVEREXPRESSION\", \"RNA-based\"], [\"1\", 156830686, 156851434, \" \", \" \", \" \", \"NTRK1\", \"EXPRESSION\", \"RNA-based\"], [\"10\", 131265448, 131566271, \" \", \" \", \" \", \"MGMT\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"11\", 4116039, 4160106, \" \", \" \", \" \", \"RRM1\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"2\", 112656056, 112787138, \" \", \" \", \" \", \"MERTK\", \"OVEREXPRESSION\", \"RNA-based\"], [\"1\", 27022524, 27108595, \" \", \" \", \" \", \"ARID1A\", \"UNDEREXPRESSION\", \"RNA-based\"], [\"22\", 23522397, 23632600, \"9\", 133729451, 133763063, \"ABL1\", \"BCR-ABL\", \"RNA-based\"], [\"9\", 133748283, 133748283, \" \", \" \", \" \", \"ABL1\", \"BCR-ABL T334I\", \"RNA-based\"], [\"9\", 133738363, 133738363, \" \", \" \", \" \", \"ABL1\", \"BCR-ABL E255K\", \"RNA-based\"], [\"2\", 42396490, 42522656, \"2\", 29415640, 29446394, \"ALK\", \"EML4-ALK\", \"RNA-based\"], [\"2\", 29445258, 29445258, \" \", \" \", \" \", \"ALK\", \"EML4-ALK C1156Y\", \"RNA-based\"], [\"2\", 29443631, 29443631, \" \", \" \", \" \", \"ALK\", \"EML4-ALK L1196M\", \"RNA-based\"], [\"1\", 156446804, 156470620, \"5\", 149433787, 149441412, \"CSF1R\", \"MEF2D-CSF1R\", \"RNA-based\"], [\"19\", 14628951, 14629232, \"19\", 14202500, 14218221, \"PRKACA\", \"DNAJB1-PRKACA\", \"RNA-based\"], [\"10\", 123243212, 123357917, \"10\", 103544209, 103552700, \"FGFR2\", \"FGFR2-MGEA5\", \"RNA-based\"], [\"4\", 1795039, 1810599, \"7\", 97991744, 97920963, \"FGFR3\", \"FGFR3-BAIAP2L1\", \"RNA-based\"], [\"4\", 123243212, 123357598, \"4\", 1723266, 1741505, \"FGFR2\", \"FGFR2-TACC3\", \"RNA-based\"], [\"15\", 74287058, 74325755, \"17\", 38504568, 38513048, \"PML\", \"PML-RARA\", \"RNA-based\"], [\"22\", 29664007, 29683123, \"11\", 128675261, 128683162, \"FLI1\", \"EWSR1-FLI1 Type 1\", \"RNA-based\"], [\"2\", 42396490, 42528380, \"2\", 29415640, 29446394, \"ALK\", \"EML4-ALK AMPLIFICATION\", \"RNA-based\"], [\"2\", 29443613, 29443613, \" \", \" \", \" \", \"ALK\", \"ALK FUSION G1202R\", \"RNA-based\"], [\"2\", 29443600, 29443600, \" \", \" \", \" \", \"ALK\", \"EML4-ALK S1206Y\", \"RNA-based\"], [\"2\", 29445271, 29445272, \" \", \" \", \" \", \"ALK\", \"EML4-ALK T1151INST\", \"RNA-based\"], [\"7\", 91570181, 91625114, \"7\", 140434279, 140487384, \"BRAF\", \"AKAP9-BRAF\", \"RNA-based\"], [\"22\", 29664007, 29683123, \"21\", 39751949, 39755845, \"ERG\", \"EWSR1-ERG\", \"RNA-based\"], [\"21\", 42879877, 42879992, \"21\", 39751949, 39817544, \"ERG\", \"TMPRSS2-ERG\", \"RNA-based\"], [\"22\", 29664007, 29683123, \"11\", 128651853, 128683162, \"FLI1\", \"EWSR1-FLI1\", \"RNA-based\"], [\"2\", 114036521, 113992971, \"3\", 12329400, 12475810, \"PAX8\", \"PAX8-PPARG\", \"RNA-based\"], [\"9\", 133748288, 133748288, \" \", \" \", \" \", \"ABL1\", \"BCR-ABL F317L\", \"RNA-based\"], [\"6\", 117609463, 117747018, \" \", \" \", \" \", \"ROS1\", \"REARRANGEMENT\", \"RNA-based\"], [\"10\", 32306071, 32345359, \"10\", 43609928, 43625799, \"RET\", \"KIF5B-RET\", \"RNA-based\"], [\"7\", 141250989, 141255367, \"7\", 140434279, 140494267, \"BRAF\", \"AGK-BRAF\", \"RNA-based\"], [\"4\", 108603171, 108641608, \"7\", 140434279, 140487384, \"BRAF\", \"PAPSS1-BRAF\", \"RNA-based\"], [\"7\", 138145079, 138239711, \"7\", 140434279, 140487384, \"BRAF\", \"TRIM24-BRAF\", \"RNA-based\"], [\"2\", 29445270, 29445270, \" \", \" \", \" \", \"ALK\", \"EML4-ALK L1152R\", \"RNA-based\"], [\"2\", 29432682, 29432682, \" \", \" \", \" \", \"ALK\", \"EML4-ALK G1269A\", \"RNA-based\"], [\"2\", 29443625, 29445258, \" \", \" \", \" \", \"ALK\", \"EML4-ALK C1156Y–L1198F\", \"RNA-based\"], [\"1\", 156844175, 156851434, \" \", \" \", \" \", \"NTRK1\", \"NTRK1 FUSIONS\", \"RNA-based\"], [\"16\", 67063148, 67116211, \"16\", 15797029, 15820911, \"CBFB\", \"CBFB-MYH11\", \"RNA-based\"], [\"18\", 23612363, 23670589, \"X\", 48261524, 48265521, \"SSX4\", \"SS18-SSX4\", \"RNA-based\"], [\"6\", 117638347, 117638347, \" \", \" \", \" \", \"ROS1\", \"CD74-ROS1 G2032R\", \"RNA-based\"], [\"2\", 29415640, 29446394, \" \", \" \", \" \", \"ALK\", \"ALK FUSIONS\", \"RNA-based\"], [\"2\", 42396490, 42552694, \"2\", 29415640, 29446394, \"ALK\", \"EML4-ALK E20;A20\", \"RNA-based\"], [\"2\", 42396490, 42472827, \"2\", 29415640, 29446394, \"ALK\", \"EML4-ALK E2;A20\", \"RNA-based\"], [\"2\", 42396490, 42491871, \"2\", 29415640, 29446394, \"ALK\", \"EML4-ALK E6;A20\", \"RNA-based\"], [\"5\", 170814120, 170818803, \"2\", 29415640, 29446394, \"ALK\", \"NPM-ALK\", \"RNA-based\"], [\"2\", 109335937, 109375004, \"2\", 29415640, 29446394, \"ALK\", \"RANBP2-ALK\", \"RNA-based\"], [\"6\", 117638365, 117638365, \" \", \" \", \" \", \"ROS1\", \"CD74-ROS1 L2026M\", \"RNA-based\"], [\"6\", 117630062, 117630062, \" \", \" \", \" \", \"ROS1\", \"CD74-ROS1 L2155S\", \"RNA-based\"], [\"6\", 117631376, 117631376, \" \", \" \", \" \", \"ROS1\", \"CD74-ROS1 G2101A\", \"RNA-based\"], [\"17\", 57697219, 57768072, \"2\", 29415640, 29446394, \"ALK\", \"CLTC-ALK\", \"RNA-based\"], [\"2\", 29445213, 29445213, \" \", \" \", \" \", \"ALK\", \"ALK FUSION I1171\", \"RNA-based\"], [\"2\", 29443679, 29443679, \" \", \" \", \" \", \"ALK\", \"EML4-ALK  V1180L\", \"RNA-based\"], [\"2\", 29436859, 29436859, \" \", \" \", \" \", \"ALK\", \"ALK FUSION F1245C\", \"RNA-based\"], [\"2\", 29432682, 29432682, \" \", \" \", \" \", \"ALK\", \"ALK FUSION G1269A\", \"RNA-based\"], [\"8\", 38268656, 38325363, \" \", \" \", \" \", \"FGFR1\", \"FGFR1 FUSIONS\", \"RNA-based\"], [\"4\", 54243812, 54294350, \"4\", 55141008, 55164414, \"PDGFRA\", \"FIP1L1-PDGFRA\", \"RNA-based\"], [\"3\", 100428269, 100447702, \"6\", 117609463, 117642557, \"ROS1\", \"TFG-ROS1\", \"RNA-based\"], [\"4\", 55144547, 55144547, \" \", \" \", \" \", \"PDGFRA\", \"FIP1L1-PDGFRA T674I\", \"RNA-based\"], [\"2\", 29445213, 29445213, \" \", \" \", \" \", \"ALK\", \"HIP1-ALK I1171N\", \"RNA-based\"], [\"2\", 29445213, 29445213, \" \", \" \", \" \", \"ALK\", \"EML4-ALK I1171S\", \"RNA-based\"], [\"2\", 204732509, 204736210, \"2\", 204599507, 204603635, \"CTLA4\", \"CTLA4-CD28\", \"RNA-based\"], [\"2\", 29443631, 29445258, \" \", \" \", \" \", \"ALK\", \"EML4-ALK C1156Y–L1196M\", \"RNA-based\"], [\"11\", 7535001, 7586998, \"7\", 140434279, 140481493, \"BRAF\", \"PPFIBP2-BRAF\", \"RNA-based\"], [\"7\", 138545885, 138666064, \"7\", 140434279, 140487384, \"BRAF\", \"KIAA1549-BRAF\", \"RNA-based\"], [\"7\", 140434279, 140487384, \"7\", 148395737, 148457588, \"BRAF\", \"BRAF-CUL1\", \"RNA-based\"], [\"7\", 99613222, 99627575, \"7\", 140434279, 140482957, \"BRAF\", \"ZKSCAN1-BRAF\", \"RNA-based\"], [\"22\", 29664307, 29688158, \"11\", 128638013, 128683162, \"EWSR1\", \"EWSR1-FLI1\", \"RNA-based\"], [\"22\", 29664307, 29687588, \"12\", 57910376, 57911536, \"DDIT3\", \"EWSR1-DDIT3\", \"RNA-based\"], [\"14\", 75745477, 75748127, \" \", \" \", \" \", \"FOS\", \"TRUNCATING FUSION\", \"RNA-based\"], [\"3\", 149235022, 149290787, \"1\", 7721786, 7829766, \"WWTR1\", \"WWTR1-CAMTA1\", \"RNA-based\"], [\"10\", 93558069, 93619405, \"4\", 55141008, 55164414, \"PDGFRA\", \"TNKS2-PDGFRA\", \"RNA-based\"], [\"10\", 123239371, 123357917, \"10\", 60461834, 60591195, \"FGFR2\", \"FGFR2-BICC1\", \"RNA-based\"], [\"10\", 123239371, 123357917, \"1\", 110554984, 110566357, \"FGFR2\", \"FGFR2-AHCYL1\", \"RNA-based\"], [\"12\", 11802788, 12022903, \"15\", 88483984, 88418230, \"NTRK3\", \"ETV6-NTRK3\", \"RNA-based\"], [\"15\", 88476265, 88476265, \" \", \" \", \" \", \"NTRK3\", \"ETV6-NTRK3 G623R\", \"RNA-based\"], [\"9\", 133755488, 133755488, \" \", \" \", \" \", \"ABL1\", \"BCR-ABL F486S\", \"RNA-based\"], [\"9\", 133750361, 133750361, \" \", \" \", \" \", \"ABL1\", \"BCR-ABL G398R\", \"RNA-based\"], [\"1\", 156084498, 156108548, \"1\", 156844363, 156851434, \"NTRK1\", \"LMNA-NTRK1\", \"RNA-based\"], [\"14\", 106032614, 107288051, \"18\", 60794268, 60987019, \"BCL2\", \"IGH-BCL2\", \"RNA-based\"], [\"7\", 140434279, 140624564, \" \", \" \", \" \", \"BRAF\", \"V600E AMPLIFICATION\", \"RNA-based\"], [\"11\", 69455855, 69469242, \" \", \" \", \" \", \"CCND1\", \"AMPLIFICATION\", \"RNA-based\"], [\"3\", 71003844, 71633140, \" \", \" \", \" \", \"FOXP1\", \"AMPLIFICATION\", \"RNA-based\"], [\"2\", 61108709, 61149800, \" \", \" \", \" \", \"REL\", \"AMPLIFICATION\", \"RNA-based\"], [\"20\", 54944446, 54967393, \" \", \" \", \" \", \"AURKA\", \"AMPLIFICATION\", \"RNA-based\"], [\"19\", 30302805, 30315215, \" \", \" \", \" \", \"CCNE1\", \"AMPLIFICATION\", \"RNA-based\"], [\"7\", 55086794, 55279321, \" \", \" \", \" \", \"EGFR\", \"AMPLIFICATION\", \"RNA-based\"], [\"20\", 46130646, 46285621, \" \", \" \", \" \", \"NCOA3\", \"AMPLIFICATION\", \"RNA-based\"], [\"3\", 178866311, 178957881, \" \", \" \", \" \", \"PIK3CA\", \"AMPLIFICATION\", \"RNA-based\"], [\"5\", 1253282, 1295162, \" \", \" \", \" \", \"TERT\", \"AMPLIFICATION\", \"RNA-based\"], [\"9\", 135251008, 135282209, \" \", \" \", \" \", \"TTF1\", \"AMPLIFICATION\", \"RNA-based\"], [\"20\", 61867235, 61871859, \" \", \" \", \" \", \"BIRC7\", \"AMPLIFICATION\", \"RNA-based\"], [\"8\", 38268656, 38325363, \" \", \" \", \" \", \"FGFR1\", \"AMPLIFICATION\", \"RNA-based\"], [\"7\", 116312459, 116436396, \" \", \" \", \" \", \"MET\", \"AMPLIFICATION\", \"RNA-based\"], [\"2\", 16080686, 16087129, \" \", \" \", \" \", \"MYCN\", \"AMPLIFICATION\", \"RNA-based\"], [\"17\", 37856333, 37884915, \" \", \" \", \" \", \"ERBB2\", \"AMPLIFICATION\", \"RNA-based\"], [\"22\", 22108789, 22221919, \" \", \" \", \" \", \"MAPK1\", \"AMPLIFICATION\", \"RNA-based\"], [\"9\", 139388896, 139440314, \" \", \" \", \" \", \"NOTCH1\", \"AMPLIFICATION\", \"RNA-based\"], [\"11\", 77371041, 77532063, \" \", \" \", \" \", \"RSF1\", \"AMPLIFICATION\", \"RNA-based\"], [\"20\", 39657458, 39753127, \" \", \" \", \" \", \"TOP1\", \"AMPLIFICATION\", \"RNA-based\"], [\"18\", 657604, 673578, \" \", \" \", \" \", \"TYMS\", \"AMPLIFICATION\", \"RNA-based\"], [\"17\", 48712218, 48769613, \" \", \" \", \" \", \"ABCC3\", \"AMPLIFICATION\", \"RNA-based\"], [\"7\", 97481430, 97501854, \" \", \" \", \" \", \"ASNS\", \"AMPLIFICATION\", \"RNA-based\"], [\"7\", 116312459, 116436396, \" \", \" \", \" \", \"MET\", \"EXON 14 MUTATION + AMPLIFICATION\", \"RNA-based\"], [\"12\", 58141510, 58146304, \" \", \" \", \" \", \"CDK4\", \"AMPLIFICATION\", \"RNA-based\"], [\"5\", 38938021, 39074510, \" \", \" \", \" \", \"RICTOR\", \"AMPLIFICATION\", \"RNA-based\"], [\"4\", 55524085, 55606881, \" \", \" \", \" \", \"KIT\", \"AMPLIFICATION\", \"RNA-based\"], [\"3\", 12625100, 12705725, \" \", \" \", \" \", \"RAF1\", \"AMPLIFICATION\", \"RNA-based\"], [\"12\", 25362365, 25403737, \" \", \" \", \" \", \"KRAS\", \"AMPLIFICATION\", \"RNA-based\"], [\"10\", 123239371, 123357917, \" \", \" \", \" \", \"FGFR2\", \"AMPLIFICATION\", \"RNA-based\"], [\"11\", 69624992, 69633792, \" \", \" \", \" \", \"FGF3\", \"AMPLIFICATION\", \"RNA-based\"], [\"19\", 40736224, 40791302, \" \", \" \", \" \", \"AKT2\", \"AMPLIFICATION\", \"RNA-based\"], [\"4\", 55095264, 55164414, \" \", \" \", \" \", \"PDGFRA\", \"AMPLIFICATION\", \"RNA-based\"], [\"7\", 140434279, 140624564, \" \", \" \", \" \", \"BRAF\", \"AMPLIFICATION\", \"RNA-based\"], [\"7\", 50344378, 50472799, \" \", \" \", \" \", \"IKZF1\", \"DELETION\", \"RNA-based\"], [\"10\", 89622870, 89731687, \" \", \" \", \" \", \"PTEN\", \"DELETION\", \"RNA-based\"], [\"11\", 67351066, 67354131, \" \", \" \", \" \", \"GSTP1\", \"DELETION\", \"RNA-based\"], [\"9\", 22002902, 22009362, \" \", \" \", \" \", \"CDKN2B\", \"LOSS\", \"RNA-based\"], [\"18\", 48556583, 48611409, \" \", \" \", \" \", \"SMAD4\", \"DELETION\", \"RNA-based\"], [\"2\", 140988992, 142889270, \" \", \" \", \" \", \"LRP1B\", \"DELETION\", \"RNA-based\"], [\"22\", 24129150, 24176703, \" \", \" \", \" \", \"SMARCB1\", \"DELETION\", \"RNA-based\"], [\"7\", 55086794, 55279321, \" \", \" \", \" \", \"EGFR\", \"COPY NUMBER VARIATION\", \"RNA-based\"], [\"5\", 1295161, 1295373, \" \", \" \", \" \", \"TERT\", \"PROMOTER MUTATION\", \"RNA-based\"], [\"5\", 1295228, 1295228, \" \", \" \", \" \", \"TERT\", \"C228T\", \"RNA-based\"], [\"14\", 65453063, 65453063, \" \", \" \", \" \", \"FNTB\", \"RS11623866\", \"RNA-based\"], [\"3\", 52581857, 52719852, \" \", \" \", \" \", \"PBRM1\", \"MUTATION\", \"RNA-based\"], [\"11\", 108093559, 108239826, \" \", \" \", \" \", \"ATM\", \"MUTATION\", \"RNA-based\"], [\"7\", 116312459, 116436396, \" \", \" \", \" \", \"MET\", \"MUTATION\", \"RNA-based\"], [\"16\", 68771128, 68869451, \" \", \" \", \" \", \"CDH1\", \"MUTATION\", \"RNA-based\"], [\"3\", 49412898, 49405977, \" \", \" \", \" \", \"RHOA\", \"MUTATION\", \"RNA-based\"], [\"7\", 151832010, 152133090, \" \", \" \", \" \", \"KMT2C\", \"MUTATION\", \"RNA-based\"], [\"9\", 5073770, 5073770, \" \", \" \", \" \", \"JAK2\", \"V617F\", \"RNA-based\"], [\"1\", 115247090, 115259515, \" \", \" \", \" \", \"NRAS\", \"MUTATION\", \"RNA-based\"], [\"1\", 11188519, 11188519, \" \", \" \", \" \", \"MTOR\", \"H1968Y\", \"RNA-based\"], [\"1\", 11184580, 11184580, \" \", \" \", \" \", \"MTOR\", \"P2213S\", \"RNA-based\"], [\"2\", 212295697, 212578308, \" \", \" \", \" \", \"ERBB4\", \"MUTATION\", \"RNA-based\"], [\"3\", 178866311, 178957881, \" \", \" \", \" \", \"PIK3CA\", \"MUTATION\", \"RNA-based\"], [\"22\", 22127164, 22127164, \" \", \" \", \" \", \"MAPK1\", \"E322K\", \"RNA-based\"], [\"4\", 55524085, 55606881, \" \", \" \", \" \", \"KIT\", \"MUTATION\", \"RNA-based\"], [\"7\", 140453136, 140481403, \" \", \" \", \" \", \"BRAF\", \"MUTATION\", \"RNA-based\"], [\"3\", 38182641, 38182641, \" \", \" \", \" \", \"MYD88\", \"L265P\", \"RNA-based\"], [\"10\", 123239371, 123357917, \" \", \" \", \" \", \"FGFR2\", \"MUTATION\", \"RNA-based\"], [\"2\", 29415640, 30144432, \" \", \" \", \" \", \"ALK\", \"MUTATION\", \"RNA-based\"], [\"13\", 28577411, 28674729, \" \", \" \", \" \", \"FLT3\", \"MUTATION\", \"RNA-based\"], [\"10\", 123258034, 123258034, \" \", \" \", \" \", \"FGFR2\", \"N550K\", \"RNA-based\"], [\"5\", 67511548, 67597649, \" \", \" \", \" \", \"PIK3R1\", \"MUTATION\", \"RNA-based\"], [\"3\", 41240930, 41281936, \" \", \" \", \" \", \"CTNNB1\", \"ACTIVATING MUTATION\", \"RNA-based\"], [\"6\", 41902671, 41909586, \" \", \" \", \" \", \"CCND3\", \"LOSS\", \"RNA-based\"], [\"19\", 33790840, 33793470, \" \", \" \", \" \", \"CEBPA\", \"MUTATION\", \"RNA-based\"], [\"10\", 89717672, 89717672, \" \", \" \", \" \", \"PTEN\", \"R233*\", \"RNA-based\"], [\"9\", 135766735, 135820008, \" \", \" \", \" \", \"TSC1\", \"LOSS-OF-FUNCTION\", \"RNA-based\"], [\"17\", 41196312, 41277387, \" \", \" \", \" \", \"BRCA1\", \"LOSS-OF-FUNCTION\", \"RNA-based\"], [\"13\", 32889611, 32973347, \" \", \" \", \" \", \"BRCA2\", \"LOSS-OF-FUNCTION\", \"RNA-based\"], [\"21\", 36160098, 36421641, \" \", \" \", \" \", \"RUNX1\", \"MUTATION\", \"RNA-based\"], [\"4\", 106067450, 106200973, \" \", \" \", \" \", \"TET2\", \"MUTATION\", \"RNA-based\"], [\"3\", 10182692, 10193904, \" \", \" \", \" \", \"VHL\", \"LOSS-OF-FUNCTION\", \"RNA-based\"], [\"3\", 10182692, 10193904, \" \", \" \", \" \", \"VHL\", \"MUTATION\", \"RNA-based\"], [\"12\", 49412758, 49449107, \" \", \" \", \" \", \"KMT2D\", \"LOSS-OF-FUNCTION\", \"RNA-based\"], [\"3\", 52435029, 52444366, \" \", \" \", \" \", \"BAP1\", \"MUTATION\", \"RNA-based\"], [\"17\", 41196312, 41277387, \" \", \" \", \" \", \"BRCA1\", \"MUTATION\", \"RNA-based\"], [\"13\", 32889611, 32973347, \" \", \" \", \" \", \"BRCA2\", \"MUTATION\", \"RNA-based\"], [\"10\", 89622870, 89731687, \" \", \" \", \" \", \"PTEN\", \"LOSS\", \"RNA-based\"], [\"18\", 48556583, 48611409, \" \", \" \", \" \", \"SMAD4\", \"MUTATION\", \"RNA-based\"], [\"19\", 11071598, 11172958, \" \", \" \", \" \", \"SMARCA4\", \"MUTATION\", \"RNA-based\"], [\"19\", 11071598, 11172958, \" \", \" \", \" \", \"SMARCA4\", \"INACTIVATING MUTATION\", \"RNA-based\"], [\"22\", 29083732, 29137832, \" \", \" \", \" \", \"CHEK2\", \"LOSS-OF-FUNCTION\", \"RNA-based\"], [\"17\", 37618292, 37691399, \" \", \" \", \" \", \"CDK12\", \"LOSS-OF-FUNCTION\", \"RNA-based\"], [\"4\", 153245446, 153332714, \" \", \" \", \" \", \"FBXW7\", \"MUTATION\", \"RNA-based\"], [\"9\", 98205262, 98270943, \" \", \" \", \" \", \"PTCH1\", \"MUTATION\", \"RNA-based\"], [\"9\", 8314246, 10033790, \" \", \" \", \" \", \"PTPRD\", \"MUTATION\", \"RNA-based\"], [\"11\", 94152895, 94227074, \" \", \" \", \" \", \"MRE11\", \"LOSS\", \"RNA-based\"], [\"3\", 10182692, 10193904, \" \", \" \", \" \", \"VHL\", \"LOSS\", \"RNA-based\"], [\"19\", 1205740, 1228428, \" \", \" \", \" \", \"STK11\", \"LOSS\", \"RNA-based\"], [\"16\", 23614491, 23652631, \" \", \" \", \" \", \"PALB2\", \"BIALLELIC INACTIVATION\", \"RNA-based\"], [\"9\", 97861336, 98079984, \" \", \" \", \" \", \"FANCC\", \"LOSS-OF-FUNCTION\", \"RNA-based\"], [\"9\", 21967752, 21994623, \" \", \" \", \" \", \"CDKN2A\", \"LOSS\", \"RNA-based\"], [\"18\", 48556583, 48611409, \" \", \" \", \" \", \"SMAD4\", \"LOSS\", \"RNA-based\"], [\"7\", 140477859, 140477860, \" \", \" \", \" \", \"BRAF\", \"K483M\", \"RNA-based\"], [\"17\", 29421945, 29704695, \" \", \" \", \" \", \"NF1\", \"MUTATION\", \"RNA-based\"], [\"19\", 33790840, 33793470, \" \", \" \", \" \", \"CEBPA\", \"BIALLELIC INACTIVATION\", \"RNA-based\"], [\"9\", 139388896, 139440314, \" \", \" \", \" \", \"NOTCH1\", \"LOSS-OF-FUNCTION\", \"RNA-based\"], [\"13\", 48877911, 49056122, \" \", \" \", \" \", \"RB1\", \"LOSS-OF-FUNCTION\", \"RNA-based\"], [\"7\", 124462440, 124570035, \" \", \" \", \" \", \"POT1\", \"MUTATION\", \"RNA-based\"], [\"4\", 153242410, 153457253, \" \", \" \", \" \", \"FBXW7\", \"LOSS-OF-FUNCTION\", \"RNA-based\"], [\"6\", 106534195, 106557813, \" \", \" \", \" \", \"PRDM1\", \"MUTATION\", \"RNA-based\"], [\"22\", 38368307, 38380544, \" \", \" \", \" \", \"SOX10\", \"LOSS\", \"RNA-based\"], [\"1\", 8073509, 8073509, \" \", \" \", \" \", \"ERRFI1\", \"E384*\", \"RNA-based\"], [\"22\", 29999547, 30094587, \" \", \" \", \" \", \"NF2\", \"LOSS\", \"RNA-based\"], [\"17\", 29421945, 29704695, \" \", \" \", \" \", \"NF1\", \"LOSS\", \"RNA-based\"], [\"22\", 42522501, 42526908, \" \", \" \", \" \", \"CYP2D6\", \"LOSS-OF-FUNCTION\", \"RNA-based\"], [\"21\", 36231792, 36231792, \" \", \" \", \" \", \"RUNX1\", \"D198Y\", \"RNA-based\"], [\"2\", 47630108, 47710367, \" \", \" \", \" \", \"MSH2\", \"LOSS\", \"RNA-based\"], [\"2\", 48010221, 48037240, \" \", \" \", \" \", \"MSH6\", \"LOSS\", \"RNA-based\"], [\"12\", 70910630, 71031220, \" \", \" \", \" \", \"PTPRB\", \"LOSS-OF-FUNCTION\", \"RNA-based\"], [\"3\", 10191470, 10191470, \" \", \" \", \" \", \"VHL\", \"Splicing alteration (c.464-1G>A)\", \"RNA-based\"], [\"3\", 10191470, 10191470, \" \", \" \", \" \", \"VHL\", \"Splicing alteration (c.464-1G>T)\", \"RNA-based\"], [\"3\", 10191469, 10191469, \" \", \" \", \" \", \"VHL\", \"Splicing alteration (c.464-2A>T)\", \"RNA-based\"], [\"3\", 10191470, 10191470, \" \", \" \", \" \", \"VHL\", \"Splicing alteration (c.464-1G>C)\", \"RNA-based\"], [\"3\", 10191469, 10191469, \" \", \" \", \" \", \"VHL\", \"Splicing alteration (c.464-2A>G)\", \"RNA-based\"], [\"3\", 10188196, 10188196, \" \", \" \", \" \", \"VHL\", \"Splicing alteration (c.341-2A>C)\", \"RNA-based\"], [\"11\", 533873, 534289, \" \", \" \", \" \", \"HRAS\", \"MUTATION\", \"RNA-based\"], [\"7\", 128828713, 128853386, \" \", \" \", \" \", \"SMO\", \"MUTATION\", \"RNA-based\"], [\"12\", 69234988, 69234988, \" \", \" \", \" \", \"MDM2\", \"RS34886328\", \"RNA-based\"], [\"2\", 25455845, 25565459, \" \", \" \", \" \", \"DNMT3A\", \"MUTATION\", \"RNA-based\"], [\"1\", 155870065, 155880706, \" \", \" \", \" \", \"RIT1\", \"MUTATION\", \"RNA-based\"], [\"X\", 66764465, 66950461, \" \", \" \", \" \", \"AR\", \"MUTATION\", \"RNA-based\"], [\"2\", 209101803, 209116275, \" \", \" \", \" \", \"IDH1\", \"MUTATION\", \"RNA-based\"], [\"22\", 29999547, 30094587, \" \", \" \", \" \", \"NF2\", \"MUTATION\", \"RNA-based\"], [\"X\", 39910501, 40036582, \" \", \" \", \" \", \"BCOR\", \"MUTATION\", \"RNA-based\"], [\"X\", 20156713, 20159754, \" \", \" \", \" \", \"EIF1AX\", \"MUTATION\", \"RNA-based\"], [\"7\", 55214299, 55214433, \" \", \" \", \" \", \"EGFR\", \"EXON 4 DELETION\", \"RNA-based\"], [\"2\", 141665446, 141777671, \" \", \" \", \" \", \"LRP1B\", \"EXON 12–22 DELETION\", \"RNA-based\"], [\"7\", 116411903, 116412043, \" \", \" \", \" \", \"MET\", \"EXON 14 SKIPPING MUTATION\", \"RNA-based\"], [\"3\", 10183532, 10191649, \" \", \" \", \" \", \"VHL\", \"EXON 1-3 DELETION\", \"RNA-based\"], [\"3\", 10183532, 10183871, \" \", \" \", \" \", \"VHL\", \"EXON 1 DELETION\", \"RNA-based\"], [\"3\", 10183532, 10188320, \" \", \" \", \" \", \"VHL\", \"EXON 1-2 DELETION\", \"RNA-based\"], [\"3\", 37056036, 37056036, \" \", \" \", \" \", \"MLH1\", \"C.790+1G>A\", \"RNA-based\"], [\"1\", 97915614, 97915614, \" \", \" \", \" \", \"DPYD\", \"DPYD*2A HOMOZYGOSITY\", \"RNA-based\"]]\n"
     ]
    }
   ],
   "source": [
    "## For variants listed in the RNA-based API, create bed-like files for capture design\n",
    "pipeline = 'RNA-based' #set pipeline\n",
    "blank = ' ' #set blanks\n",
    "\n",
    "#iterate through evidence items\n",
    "for item in variants_RNA\n",
    "  flag = false #set flag for coordinates; false = chr1,start1,stop1 only\n",
    "  chrom = item['coordinates']['chromosome'] #call chrom\n",
    "  start = item['coordinates']['start'] #call start\n",
    "  stop = item['coordinates']['stop']\n",
    "  if item['coordinates']['chromosome2'] and item['coordinates']['start2'] and item['coordinates']['stop2'] #determine if there is a second set of coordinates\n",
    "    chrom2 = item['coordinates']['chromosome2']  # call chrom2\n",
    "    start2 = item['coordinates']['start2']  # call start2\n",
    "    stop2 = item['coordinates']['stop2']  # call stop2\n",
    "    flag = true #set flag for coordinates; true = second set of coordinates available\n",
    "  end\n",
    "  gene = item['entrez_name']  #Call Gene name\n",
    "  variant = item['name'] #call variant\n",
    "  if flag #if there is only one set of coordinates\n",
    "    capture_RNA << [chrom, start, stop, chrom2, start2, stop2, gene, variant, pipeline] #append to list\n",
    "  end\n",
    "  if not flag #if there are two sets of coordinates\n",
    "    capture_RNA << [chrom, start, stop, blank, blank, blank, gene, variant, pipeline] #append to list\n",
    "  end\n",
    "end\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Ruby 2.4.1",
   "language": "ruby",
   "name": "ruby"
  },
  "language_info": {
   "file_extension": ".rb",
   "mimetype": "application/x-ruby",
   "name": "ruby",
   "version": "2.4.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
