{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Create Required Coverage for smMIPs Panel\n",
    "\n",
    "#### Usage\n",
    "\n",
    "probe_selection.py will pull existing variants from the CIViC Knowledgebase, iterate through all pulled \n",
    "variants, and pull variants based on the provided Actionability Score threshold. The threshold (see input) is the lower limit required for a variant to be considered extensively curated and therefore eligible for probe design. It will create output for DNA-based and RNA-based variants to detail the coverage required to evaluate all extensively curated variants (i.e. variants that have an actionability score > threshold)\n",
    "\n",
    "\n",
    "#### Inputs:\n",
    "\n",
    "1) Threshold = minimum requirement for actionability score to be included in output files; default is 20 points.\n",
    "\n",
    "2) tile_classificaiton = an input file that designates if a variant must be assessed by tiling all protein coding exons or if the variant can be assessed with a few number of probes.\n",
    "\n",
    "#### Output Files: \n",
    "\n",
    "1) capture_sequence_probes.tsv = list of DNA-based variants\n",
    "\n",
    "2) nanoString_probes.tsv = list of RNA-based variants\n",
    "\n",
    "3) tile_classification.txt =updated curated list of variants that need to be tiled or not\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "##SET INPUT VALUES\n",
    "\n",
    "threshold = str(20)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#!/usr/bin/env python3\n",
    "import json\n",
    "import numpy as np\n",
    "import requests\n",
    "import sys"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "##Pull in Data from JSON\n",
    "variants_capture = requests.get('https://civic.genome.wustl.edu/api/panels/captureseq/qualifying_variants?minimum_score=',threshold).json()['records'] #Call eligible variants\n",
    "variants_nanostring = requests.get('https://civic.genome.wustl.edu/api/panels/nanostring/qualifying_variants?minimum_score=',threshold).json()['records'] #Call eligible variants"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total Number of Eligible Variants:  988\n",
      "Total Number of Eligible Genes:  275\n"
     ]
    }
   ],
   "source": [
    "##Use API to determine the total number of eligible variants and the total number of eligible genes\n",
    "\n",
    "total_variant_count = 0 #Start count to determine the total number of eligible variants\n",
    "variant_list = [] #Create a Variant List for all genes\n",
    "\n",
    "for k in range(0, len(variants_nanostring)): #iterate through API and pull all eligible variants\n",
    "    if variants_nanostring[k]['entrez_name'] not in variant_list: #If the gene is not in the list already\n",
    "        variant_list.append(variants_nanostring[k]['entrez_name']) #add to the list\n",
    "    total_variant_count += 1 #Count all of the eligible variants\n",
    "    \n",
    "for k in range(0, len(variants_capture)): #iterate through API and pull all eligible variants\n",
    "    if variants_capture[k]['entrez_name'] not in variant_list: #If the gene is not in the list already\n",
    "        variant_list.append(variants_capture[k]['entrez_name']) #add to the list\n",
    "    total_variant_count += 1 #Count all of the eligible variants\n",
    "    \n",
    "print('Total Number of Eligible Variants: ', total_variant_count) #Print out all variants\n",
    "print('Total Number of Eligible Genes: ', len(variant_list)) #Print out all Genes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "## For variants listed in the CaptureSeq API, create bed-like files for capture design\n",
    "\n",
    "#make dictionary for evidence types\n",
    "score = {'A':5, 'B':4, 'C':3, 'D':2, 'E':1}\n",
    "\n",
    "capture_sequence_probes = [] #create empty list for capture sequence probes\n",
    "capture_sequence_probes.append(['gene', 'so_id', 'variant_type', 'variant_name', 'representative_transcript', 'top_evidence_level', 'diseases', 'evidence_types', 'number_of_evidence_statements', 'evidence_score', 'chrom', 'start', 'stop'])\n",
    "for k in range(0, len(variants_capture)): #iterate through API and pull all eligible variants\n",
    "    gene = variants_capture[k]['entrez_name']  #Call Gene name\n",
    "    variant = variants_capture[k]['name'] #call variant\n",
    "    soid = variants_capture[k]['variant_types'][0]['so_id'] #call soid\n",
    "    variant_type = variants_capture[k]['variant_types'][0]['name'] #call variant type\n",
    "    transcript = variants_capture[k]['coordinates']['representative_transcript'] #call transcript\n",
    "    evidence = variants_capture[k]['evidence_items'] #pull evidence items\n",
    "    evidence_statements = len(variants_capture[k]['evidence_items']) #pull number of evidence statements\n",
    "    chrom = variants_capture[k]['coordinates']['chromosome'] #call chrom\n",
    "    start = variants_capture[k]['coordinates']['start'] #call start\n",
    "    stop = variants_capture[k]['coordinates']['stop'] #call stop\n",
    "    diseases = [] #set list for all of the diseases for this varinat\n",
    "    evidence_type = [] #set list for evidence types\n",
    "    evidence_scores = [] #set list for evidence scores\n",
    "    top_evidences = [] #set list for top evidence level\n",
    "    for item in evidence: #iterate through the evidence items\n",
    "        if item['disease']['name'] not in diseases: #see if disease is already there\n",
    "            if 'Walden' in item['disease']['name']: #Change waldenstroms issues (the A is not accepted by R code)\n",
    "                if 'Waldenstroms Macroglobulinemia' not in diseases: #check if it is already there\n",
    "                    diseases.append('Waldenstroms Macroglobulinemia') #if it is not append to diseases\n",
    "            else: #if its not a weird name\n",
    "                diseases.append(item['disease']['name']) #append the disease\n",
    "        if item['evidence_type'] not in evidence_type: #see if the evidence type is already there\n",
    "            evidence_type.append(item['evidence_type']) #if it is not append it\n",
    "        trust_rating = int(item['rating'] or 0) #make the trust rating either what is listed or 0\n",
    "        evidence_level = int(score[item['evidence_level']]) #make the evidence level the value from the score dictionary\n",
    "        evidence_scores.append(evidence_level * trust_rating) #calculate the Evidence Score\n",
    "        if item['evidence_level'] != '[]': #find the evidence levels that are not blank\n",
    "            top_evidences.append(item['evidence_level'].strip()) #add to the list\n",
    "    #pull the maximum evidence level\n",
    "    if 'A' in top_evidences:\n",
    "        top_evidence = 'A'\n",
    "    elif 'B' in top_evidences:\n",
    "        top_evidence = 'B'\n",
    "    elif 'C' in top_evidences:\n",
    "        top_evidence = 'C'\n",
    "    elif 'D' in top_evidences:\n",
    "        top_evidence = 'D'\n",
    "    else:\n",
    "        top_evidence = 'E'\n",
    "    evidence_score = sum(evidence_scores) #sum the evidence scores to get a CIVic Score\n",
    "    disease = ', '.join(diseases) #format the diseases\n",
    "    evidence_types = ', '.join(evidence_type) #format the evidence types\n",
    "\n",
    "    #Append coordinates to the capture_sequence_probes list\n",
    "    if variants_capture[k]['coordinates']['chromosome2'] is not None and variants_capture[k]['coordinates']['start2'] is not None and variants_capture[k]['coordinates']['stop2'] is not None: #if there are two chromosomes for the variant\n",
    "        transcript2 = variants_capture[k]['coordinates']['representative_transcript'] #call transcript\n",
    "        chrom2 = variants_capture[k]['coordinates']['chromosome2'] #call chrom2\n",
    "        start2 = variants_capture[k]['coordinates']['start2'] #call start2\n",
    "        stop2 = variants_capture[k]['coordinates']['stop2'] #call stop2\n",
    "        capture_sequence_probes.append([gene, soid, variant_type, variant, transcript, top_evidence, disease, evidence_types, evidence_statements, evidence_score, chrom, start, stop, transcript2, chrom2, start2, stop2]) #append new list with bed informaiton\n",
    "    else: #if there is only 1 chromosome for the variant\n",
    "        capture_sequence_probes.append([gene, soid, variant_type, variant, transcript, top_evidence, disease, evidence_types, evidence_statements, evidence_score, chrom, start, stop]) #append new list with bed information\n",
    "\n",
    "##Create output files for capture     \n",
    "capture = open('../../smMIPs_panel/output_files/capture_sequence_probes.tsv', 'w') #create empy file for capture sequence coordinates\n",
    "for item in capture_sequence_probes: #iterate through capture list\n",
    "    for k in item:\n",
    "        if k is item[-1]:\n",
    "            capture.write(str(k))\n",
    "        else:\n",
    "            capture.write(str(k) + '\\t')\n",
    "    capture.write('\\n')\n",
    "capture.close() #close file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "## For variants listed in the NanoString API, create bed-like files for capture design\n",
    "\n",
    "nanoString_probes = []  # create empty list for nanostring probes\n",
    "nanoString_probes.append(['gene', 'soid', 'variant_type', 'variant_name', 'representative_transcript', 'top_evidence_level', 'diseases','evidence_types','number_of_evidence_statements', 'chrom', 'start', 'stop', 'transcript2', 'chrom2', 'start2', 'stop2'])\n",
    "for k in range(0, len(variants_nanostring)):  # iterate through API and pull all eligible variants\n",
    "    gene = variants_nanostring[k]['entrez_name']  #Call Gene name\n",
    "    variant = variants_nanostring[k]['name'] #call variant\n",
    "    soid = variants_nanostring[k]['variant_types'][0]['so_id'] #call soid\n",
    "    variant_type = variants_nanostring[k]['variant_types'][0]['name'] #call variant type\n",
    "    transcript = variants_nanostring[k]['coordinates']['representative_transcript'] #call transcript\n",
    "    top_evidence = variants_nanostring[k]\n",
    "    diseases = variants_nanostring[k]\n",
    "    chrom = variants_nanostring[k]['coordinates']['chromosome'] #call chrom\n",
    "    start = variants_nanostring[k]['coordinates']['start'] #call start\n",
    "    stop = variants_nanostring[k]['coordinates']['stop'] #call stop\n",
    "    evidence = variants_nanostring[k]['evidence_items']\n",
    "    evidence_statements = len(variants_nanostring[k]['evidence_items'])\n",
    "    diseases = []  # set list for all of the diseases for this varinat\n",
    "    evidence_type = []  # set list for evidence types\n",
    "    evidence_scores = []  # set list for evidence scores\n",
    "    top_evidences = []  # set list for top evidence level\n",
    "    for item in evidence:  # iterate through the evidence items\n",
    "        if item['disease']['name'] not in diseases:  # see if disease is already there\n",
    "            if 'Walden' in item['disease']['name']:  # Change waldenstroms issues (the A is not accepted by R code)\n",
    "                if 'Waldenstroms Macroglobulinemia' not in diseases:  # check if it is already there\n",
    "                    diseases.append('Waldenstroms Macroglobulinemia')  # if it is not append to diseases\n",
    "            else:  # if its not a weird name\n",
    "                diseases.append(item['disease']['name'])  # append the disease\n",
    "        if item['evidence_type'] not in evidence_type:  # see if the evidence type is already there\n",
    "            evidence_type.append(item['evidence_type'])  # if it is not append it\n",
    "        trust_rating = int(item['rating'] or 0)  # make the trust rating either what is listed or 0\n",
    "        evidence_level = int(\n",
    "            score[item['evidence_level']])  # make the evidence level the value from the score dictionary\n",
    "        evidence_scores.append(evidence_level * trust_rating)  # calculate the Evidence Score\n",
    "        if item['evidence_level'] != '[]':  # find the evidence levels that are not blank\n",
    "            top_evidences.append(item['evidence_level'].strip())  # add to the list\n",
    "    # pull the maximum evidence level\n",
    "    if 'A' in top_evidences:\n",
    "        top_evidence = 'A'\n",
    "    elif 'B' in top_evidences:\n",
    "        top_evidence = 'B'\n",
    "    elif 'C' in top_evidences:\n",
    "        top_evidence = 'C'\n",
    "    elif 'D' in top_evidences:\n",
    "        top_evidence = 'D'\n",
    "    else:\n",
    "        top_evidence = 'E'\n",
    "    evidence_score = sum(evidence_scores)  # sum the evidence scores to get a CIVic Score\n",
    "    disease = ', '.join(diseases)  # format the diseases\n",
    "    evidence_types = ', '.join(evidence_type)  # format the evidence types\n",
    "\n",
    "    if variants_nanostring[k]['coordinates']['chromosome2'] is not None and variants_nanostring[k]['coordinates']['start2'] is not None and variants_nanostring[k]['coordinates']['stop2'] is not None:  # if there are two chromosomes for the variant\n",
    "        chrom2 = variants_nanostring[k]['coordinates']['chromosome2']  # call chrom2\n",
    "        start2 = variants_nanostring[k]['coordinates']['start2']  # call start2\n",
    "        stop2 = variants_nanostring[k]['coordinates']['stop2']  # call stop2\n",
    "        nanoString_probes.append([gene, soid, variant_type, variant, transcript, top_evidence, disease, evidence_types, evidence_statements, chrom, start, stop, chrom2, start2, stop2])  # append new list with bed information\n",
    "    else:  # if there is only 1 chromosome for the variant\n",
    "        nanoString_probes.append([gene, soid, variant_type, variant, transcript, top_evidence, disease, evidence_types, evidence_statements, chrom, start, stop])  # append new list with bed information\n",
    "\n",
    "##Create output files for nanostring\n",
    "nanostring = open('../../smMIPs_panel/output_files/nanoString_probes.tsv', 'w')  #create empy file for nanostring coordinates\n",
    "for item in nanoString_probes: #iterate through nanostring list\n",
    "    for k in item:\n",
    "        if k is item[-1]:\n",
    "            nanostring.write(str(k))\n",
    "        else:\n",
    "            nanostring.write(str(k) + '\\t')\n",
    "    nanostring.write('\\n')\n",
    "nanostring.close() #close file"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [],
   "source": [
    "## Evaluate and Update Tiling File\n",
    "\n",
    "tiling_output = [] #create empty list for output file\n",
    "tiling_file = [] #create dictionary for keys as the gene, variant and values as tile notes\n",
    "tile_panel = open('../../smMIPs_panel/input_files/tile_classification.txt', 'r') #open tiled genes input file\n",
    "for line in tile_panel: #iterate through the tile panel\n",
    "    line = line.strip('\\n')  # strip the new line\n",
    "    line = line.split('\\t')  # split by tabs\n",
    "    if len(line) == 2:\n",
    "        print(line)\n",
    "        print('YOU NEED TO UPDATE THE TILE_CLASSIFICATION.TXT!!!')\n",
    "        break\n",
    "    gene = line[0]  # pull gene\n",
    "    variant = line[1] #pull variant\n",
    "    tile = line[2] #pull tile value\n",
    "    tiling_file.append([gene, variant])\n",
    "    tiling_output.append([gene, variant, tile])\n",
    "\n",
    "#check and see if there are new items that need to be added to tiling list    \n",
    "for item in capture_sequence_probes: #iterate through the capture_sequencing_list\n",
    "    if [item[0], item[3]] not in tiling_file: #see if the gene, variant matches with the keys in tiling dictionary\n",
    "        tiling_output.append([item[0], item[3]]) #if it is not there, append the gene and variant to the tiling list for curation\n",
    "    \n",
    "#update tiling folder\n",
    "tiling = open('../../smMIPs_panel/input_files/tile_classification.txt', 'w')\n",
    "for item in tiling_output:\n",
    "    for k in item:\n",
    "        if k is item[-1]:\n",
    "            tiling.write(str(k))\n",
    "        else:\n",
    "            tiling.write(str(k) + '\\t')\n",
    "    tiling.write('\\n')\n",
    "tiling.close()\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
