{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Results - smMIPs Sequencing and Data Analysis"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Tools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#!/usr/bin/env python3\n",
    "import json\n",
    "import numpy as np\n",
    "import requests\n",
    "import sys\n",
    "import pandas as pd\n",
    "import seaborn as sns\n",
    "sns.set(style='white')\n",
    "sns.set_context(\"talk\")\n",
    "import matplotlib.pyplot as plt\n",
    "import glob"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "from pyliftover import LiftOver\n",
    "lo = LiftOver('hg19', 'hg38')\n",
    "li = LiftOver('hg38', 'hg19')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Pull in Input Files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "samples = pd.read_csv('../data/validation_samples/sample_dataframe.txt', sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "samples_QC = pd.read_csv('../data/validation_samples/sequencing_quality_check.txt', sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "overlap_with_smmips = pd.read_csv('../output/supplementary_table_3-variant_overlap.tsv', sep='\\t')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Initial Quality Check"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Mean:  5369817.166666667\n",
      "Standard Deviation:  3334506.223398502\n"
     ]
    }
   ],
   "source": [
    "print('Mean: ', samples_QC['total tags weighted'].mean())\n",
    "mean = samples_QC['total tags weighted'].mean()\n",
    "print('Standard Deviation: ', samples_QC['total tags weighted'].std())\n",
    "std = samples_QC['total tags weighted'].std()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "H_ML-08-0075-248_normal\n",
      "Merck016_tumor\n",
      "Merck012_tumor\n",
      "HOC71_normal\n",
      "SCLC19_normal\n",
      "SCLC31_normal\n"
     ]
    }
   ],
   "source": [
    "ineligible_samples = []\n",
    "for i,row in samples_QC.iterrows():\n",
    "    if row['total tags weighted'] < mean - 1*std:\n",
    "        print(row['count'] + '_' +  row['Type'])\n",
    "        ineligible_samples.append(row['count'] + '_' +  row['Type'])\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of elgible Samples:  30\n"
     ]
    }
   ],
   "source": [
    "print('Number of elgible Samples: ', len(samples_QC) - len(ineligible_samples))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# If tumor is ineligible, eliminate variants for subsequent analysis\n",
    "overlap_with_smmips['Passed QC'] = ''\n",
    "for i,row in overlap_with_smmips.iterrows():\n",
    "    if row['sample'] + '_' +  'tumor' in ineligible_samples:\n",
    "        overlap_with_smmips.loc[[i], 'Passed QC'] = 'no'\n",
    "    else:\n",
    "        overlap_with_smmips.loc[[i], 'Passed QC'] = 'yes'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of Eligible Individuals:  17\n"
     ]
    }
   ],
   "source": [
    "print('Number of Eligible Individuals: ', len(overlap_with_smmips[overlap_with_smmips['Passed QC'] == 'yes'][['sample']].drop_duplicates()))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of Eligible Variants:  74\n"
     ]
    }
   ],
   "source": [
    "print('Number of Eligible Variants: ',overlap_with_smmips[overlap_with_smmips['Passed QC'] == 'yes'].groupby('sample').size().sum())"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Accuracy profile of smMIPs CIViC panel when compared to WEX"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    " ### Pull in VCF Files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "smmips_variants = pd.DataFrame()\n",
    "for item in glob.glob('../data/smmips_sequencing/*T*.vcf'):\n",
    "    if item.split('/')[3].startswith('H_'):\n",
    "        name = item.split('/')[3].split('_')[0] +  '_' +item.split('/')[3].split('_')[1]\n",
    "    else:\n",
    "        name = item.split('/')[3].split('_')[0]\n",
    "    current = pd.read_csv(item, sep='\\t', comment='#', header=None).filter(items=[0,1,1,3,4,9])\n",
    "    current['sample'] = name\n",
    "    smmips_variants = smmips_variants.append(current)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "smmips_variants.columns = ['chrom', 'start', 'stop', 'reference', 'variant', 'CIViC Panel VAF', 'sample']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "for i,row in smmips_variants.iterrows():\n",
    "    VAF = float(row['CIViC Panel VAF'].split(':')[-1])*100\n",
    "    smmips_variants.loc[[i], 'CIViC Panel VAF'] = VAF"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "overlap_with_smmips = overlap_with_smmips.merge(smmips_variants, on=['chrom', 'start', 'stop','reference', 'variant', 'sample'], how='left')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "overlap_with_smmips = overlap_with_smmips.drop(['Unnamed: 0'], axis=1)\n",
    "overlap_with_smmips['CIViC Panel VAF'] = overlap_with_smmips['CIViC Panel VAF'].replace(np.nan, 0)\n",
    "overlap_with_smmips['CIViC Panel VAF'] = overlap_with_smmips['CIViC Panel VAF'].astype('float')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['chrom', 'start', 'stop', 'reference', 'variant', 'sample', 'type',\n",
       "       'gene_name', 'genome', 'strand', 'trv_type', 'c_position',\n",
       "       'amino_acid', 'tier', 'NORMAL_EXOME_ref_count',\n",
       "       'NORMAL_EXOME_var_count', 'NORMAL_EXOME_VAF',\n",
       "       'TUMOR_EXOME_ref_count', 'TUMOR_EXOME_var_count', 'Coverage', 'VAF',\n",
       "       'Passed QC', 'CIViC Panel VAF'], dtype=object)"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "overlap_with_smmips.columns.values"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total eligible variants:  74\n",
      "Total overlap with smMIPs:  52\n",
      "Non-overlapping variants:  22\n"
     ]
    }
   ],
   "source": [
    "print('Total eligible variants: ', len(overlap_with_smmips[overlap_with_smmips['Passed QC'] == 'yes']))\n",
    "print('Total overlap with smMIPs: ', len(overlap_with_smmips[overlap_with_smmips['CIViC Panel VAF'] > 0]))\n",
    "print('Non-overlapping variants: ', len(overlap_with_smmips[overlap_with_smmips['Passed QC'] == 'yes']) - len(overlap_with_smmips[overlap_with_smmips['CIViC Panel VAF'] > 0]))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>chrom</th>\n",
       "      <th>start</th>\n",
       "      <th>stop</th>\n",
       "      <th>reference</th>\n",
       "      <th>variant</th>\n",
       "      <th>sample</th>\n",
       "      <th>type</th>\n",
       "      <th>gene_name</th>\n",
       "      <th>genome</th>\n",
       "      <th>strand</th>\n",
       "      <th>...</th>\n",
       "      <th>tier</th>\n",
       "      <th>NORMAL_EXOME_ref_count</th>\n",
       "      <th>NORMAL_EXOME_var_count</th>\n",
       "      <th>NORMAL_EXOME_VAF</th>\n",
       "      <th>TUMOR_EXOME_ref_count</th>\n",
       "      <th>TUMOR_EXOME_var_count</th>\n",
       "      <th>Coverage</th>\n",
       "      <th>VAF</th>\n",
       "      <th>Passed QC</th>\n",
       "      <th>CIViC Panel VAF</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>chr11</td>\n",
       "      <td>108196129</td>\n",
       "      <td>108196129</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>H_ML-08-0075-513</td>\n",
       "      <td>SNP</td>\n",
       "      <td>ATM</td>\n",
       "      <td>37</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>546.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.18</td>\n",
       "      <td>377</td>\n",
       "      <td>18</td>\n",
       "      <td>942</td>\n",
       "      <td>4.550000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>chr12</td>\n",
       "      <td>25380347</td>\n",
       "      <td>25380347</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>H_ML-08-0075-513</td>\n",
       "      <td>SNP</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>37</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>597.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>496</td>\n",
       "      <td>26</td>\n",
       "      <td>1119</td>\n",
       "      <td>4.980000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>chr13</td>\n",
       "      <td>32913614</td>\n",
       "      <td>32913614</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>H_ML-08-0075-513</td>\n",
       "      <td>SNP</td>\n",
       "      <td>BRCA2</td>\n",
       "      <td>37</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>349.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>273</td>\n",
       "      <td>20</td>\n",
       "      <td>642</td>\n",
       "      <td>6.830000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>chr17</td>\n",
       "      <td>41209104</td>\n",
       "      <td>41209104</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>H_ML-08-0075-513</td>\n",
       "      <td>SNP</td>\n",
       "      <td>BRCA1</td>\n",
       "      <td>37</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>469.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>478</td>\n",
       "      <td>41</td>\n",
       "      <td>988</td>\n",
       "      <td>7.900000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>chr1</td>\n",
       "      <td>11187201</td>\n",
       "      <td>11187201</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>H_ML-08-0075-513</td>\n",
       "      <td>SNP</td>\n",
       "      <td>MTOR</td>\n",
       "      <td>37</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>764.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.13</td>\n",
       "      <td>866</td>\n",
       "      <td>40</td>\n",
       "      <td>1671</td>\n",
       "      <td>4.420000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>chr1</td>\n",
       "      <td>11856310</td>\n",
       "      <td>11856310</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>H_ML-08-0075-513</td>\n",
       "      <td>SNP</td>\n",
       "      <td>MTHFR</td>\n",
       "      <td>37</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>627.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>660</td>\n",
       "      <td>50</td>\n",
       "      <td>1337</td>\n",
       "      <td>7.040000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>chr2</td>\n",
       "      <td>198274698</td>\n",
       "      <td>198274698</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>H_ML-08-0075-513</td>\n",
       "      <td>SNP</td>\n",
       "      <td>SF3B1</td>\n",
       "      <td>37</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>611.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>580</td>\n",
       "      <td>63</td>\n",
       "      <td>1254</td>\n",
       "      <td>9.800000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>chr7</td>\n",
       "      <td>140453161</td>\n",
       "      <td>140453161</td>\n",
       "      <td>T</td>\n",
       "      <td>-</td>\n",
       "      <td>SCLC19</td>\n",
       "      <td>DEL</td>\n",
       "      <td>BRAF</td>\n",
       "      <td>37</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>102.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>52</td>\n",
       "      <td>21</td>\n",
       "      <td>175</td>\n",
       "      <td>28.770000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>chr17</td>\n",
       "      <td>7578458</td>\n",
       "      <td>7578458</td>\n",
       "      <td>G</td>\n",
       "      <td>-</td>\n",
       "      <td>SCLC19</td>\n",
       "      <td>DEL</td>\n",
       "      <td>TP53</td>\n",
       "      <td>37</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>49.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>8</td>\n",
       "      <td>33</td>\n",
       "      <td>90</td>\n",
       "      <td>80.490000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>chr10</td>\n",
       "      <td>89690805</td>\n",
       "      <td>89690805</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>SCLC25</td>\n",
       "      <td>SNP</td>\n",
       "      <td>PTEN</td>\n",
       "      <td>37</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>106.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>3</td>\n",
       "      <td>49</td>\n",
       "      <td>158</td>\n",
       "      <td>94.230000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>chr9</td>\n",
       "      <td>139396724</td>\n",
       "      <td>139396736</td>\n",
       "      <td>TTGAGGCCCACGG</td>\n",
       "      <td>-</td>\n",
       "      <td>SCLC31</td>\n",
       "      <td>DEL</td>\n",
       "      <td>NOTCH1</td>\n",
       "      <td>37</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>109.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>52</td>\n",
       "      <td>35</td>\n",
       "      <td>196</td>\n",
       "      <td>40.230000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>chr17</td>\n",
       "      <td>41243482</td>\n",
       "      <td>41243482</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>Merck002</td>\n",
       "      <td>SNP</td>\n",
       "      <td>BRCA1</td>\n",
       "      <td>37</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>75.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>47</td>\n",
       "      <td>30</td>\n",
       "      <td>152</td>\n",
       "      <td>38.460000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>57</th>\n",
       "      <td>chr17</td>\n",
       "      <td>7577121</td>\n",
       "      <td>7577121</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>Merck017</td>\n",
       "      <td>SNP</td>\n",
       "      <td>TP53</td>\n",
       "      <td>37</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>107.0</td>\n",
       "      <td>3.0</td>\n",
       "      <td>2.73</td>\n",
       "      <td>78</td>\n",
       "      <td>22</td>\n",
       "      <td>210</td>\n",
       "      <td>21.780000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>59</th>\n",
       "      <td>chr17</td>\n",
       "      <td>7578395</td>\n",
       "      <td>7578395</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>Merck002</td>\n",
       "      <td>SNP</td>\n",
       "      <td>TP53</td>\n",
       "      <td>37</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>149.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>57</td>\n",
       "      <td>59</td>\n",
       "      <td>265</td>\n",
       "      <td>50.860000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>60</th>\n",
       "      <td>chr17</td>\n",
       "      <td>7578469</td>\n",
       "      <td>7578469</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>Merck017</td>\n",
       "      <td>SNP</td>\n",
       "      <td>TP53</td>\n",
       "      <td>37</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>88.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>69</td>\n",
       "      <td>40</td>\n",
       "      <td>197</td>\n",
       "      <td>36.700000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>chr17</td>\n",
       "      <td>7579528</td>\n",
       "      <td>7579528</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>Merck002</td>\n",
       "      <td>SNP</td>\n",
       "      <td>TP53</td>\n",
       "      <td>37</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>181.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>91</td>\n",
       "      <td>14</td>\n",
       "      <td>286</td>\n",
       "      <td>13.330000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>chr20</td>\n",
       "      <td>31022441</td>\n",
       "      <td>31022442</td>\n",
       "      <td>-</td>\n",
       "      <td>G</td>\n",
       "      <td>Merck002</td>\n",
       "      <td>INS</td>\n",
       "      <td>ASXL1</td>\n",
       "      <td>37</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>175.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.57</td>\n",
       "      <td>106</td>\n",
       "      <td>51</td>\n",
       "      <td>333</td>\n",
       "      <td>32.480000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66</th>\n",
       "      <td>chr3</td>\n",
       "      <td>178936082</td>\n",
       "      <td>178936082</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>Merck017</td>\n",
       "      <td>SNP</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>37</td>\n",
       "      <td>1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>584.0</td>\n",
       "      <td>1.0</td>\n",
       "      <td>0.17</td>\n",
       "      <td>373</td>\n",
       "      <td>259</td>\n",
       "      <td>1217</td>\n",
       "      <td>40.920000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>chr8</td>\n",
       "      <td>38277223</td>\n",
       "      <td>38277223</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>Merck002</td>\n",
       "      <td>SNP</td>\n",
       "      <td>FGFR1</td>\n",
       "      <td>37</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>128.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>20</td>\n",
       "      <td>69</td>\n",
       "      <td>217</td>\n",
       "      <td>77.530000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69</th>\n",
       "      <td>chr9</td>\n",
       "      <td>21971120</td>\n",
       "      <td>21971120</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>Merck002</td>\n",
       "      <td>SNP</td>\n",
       "      <td>CDKN2A</td>\n",
       "      <td>37</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>295.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>65</td>\n",
       "      <td>240</td>\n",
       "      <td>600</td>\n",
       "      <td>78.690000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76</th>\n",
       "      <td>chr17</td>\n",
       "      <td>7579541</td>\n",
       "      <td>7579542</td>\n",
       "      <td>TC</td>\n",
       "      <td>-</td>\n",
       "      <td>SCLC8</td>\n",
       "      <td>DEL</td>\n",
       "      <td>TP53</td>\n",
       "      <td>37</td>\n",
       "      <td>-1.0</td>\n",
       "      <td>...</td>\n",
       "      <td>tier1</td>\n",
       "      <td>33.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.00</td>\n",
       "      <td>18</td>\n",
       "      <td>14</td>\n",
       "      <td>65</td>\n",
       "      <td>43.750000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>77</th>\n",
       "      <td>chr9</td>\n",
       "      <td>21970952</td>\n",
       "      <td>21970952</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>colon_2</td>\n",
       "      <td>SNP</td>\n",
       "      <td>CDKN2A</td>\n",
       "      <td>37</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>27</td>\n",
       "      <td>27</td>\n",
       "      <td>54</td>\n",
       "      <td>0.500000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78</th>\n",
       "      <td>chr9</td>\n",
       "      <td>21971160</td>\n",
       "      <td>21971160</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>colon_2</td>\n",
       "      <td>SNP</td>\n",
       "      <td>CDKN2A</td>\n",
       "      <td>37</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>36</td>\n",
       "      <td>36</td>\n",
       "      <td>72</td>\n",
       "      <td>0.500000</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>79</th>\n",
       "      <td>chr9</td>\n",
       "      <td>21970929</td>\n",
       "      <td>21970929</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>colon_2</td>\n",
       "      <td>SNP</td>\n",
       "      <td>CDKN2A</td>\n",
       "      <td>37</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>33</td>\n",
       "      <td>31</td>\n",
       "      <td>64</td>\n",
       "      <td>0.484375</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80</th>\n",
       "      <td>chr19</td>\n",
       "      <td>1223045</td>\n",
       "      <td>1223045</td>\n",
       "      <td>C</td>\n",
       "      <td>A</td>\n",
       "      <td>colon_5</td>\n",
       "      <td>SNP</td>\n",
       "      <td>STK11</td>\n",
       "      <td>37</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>75</td>\n",
       "      <td>36</td>\n",
       "      <td>111</td>\n",
       "      <td>0.324324</td>\n",
       "      <td>yes</td>\n",
       "      <td>0.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>25 rows × 23 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    chrom      start       stop      reference variant            sample type  \\\n",
       "8   chr11  108196129  108196129              C       A  H_ML-08-0075-513  SNP   \n",
       "9   chr12   25380347   25380347              C       A  H_ML-08-0075-513  SNP   \n",
       "10  chr13   32913614   32913614              G       A  H_ML-08-0075-513  SNP   \n",
       "11  chr17   41209104   41209104              C       A  H_ML-08-0075-513  SNP   \n",
       "14   chr1   11187201   11187201              C       T  H_ML-08-0075-513  SNP   \n",
       "15   chr1   11856310   11856310              C       T  H_ML-08-0075-513  SNP   \n",
       "18   chr2  198274698  198274698              C       T  H_ML-08-0075-513  SNP   \n",
       "34   chr7  140453161  140453161              T       -            SCLC19  DEL   \n",
       "38  chr17    7578458    7578458              G       -            SCLC19  DEL   \n",
       "39  chr10   89690805   89690805              G       A            SCLC25  SNP   \n",
       "47   chr9  139396724  139396736  TTGAGGCCCACGG       -            SCLC31  DEL   \n",
       "55  chr17   41243482   41243482              G       C          Merck002  SNP   \n",
       "57  chr17    7577121    7577121              G       A          Merck017  SNP   \n",
       "59  chr17    7578395    7578395              G       A          Merck002  SNP   \n",
       "60  chr17    7578469    7578469              C       A          Merck017  SNP   \n",
       "62  chr17    7579528    7579528              C       T          Merck002  SNP   \n",
       "64  chr20   31022441   31022442              -       G          Merck002  INS   \n",
       "66   chr3  178936082  178936082              G       A          Merck017  SNP   \n",
       "67   chr8   38277223   38277223              G       A          Merck002  SNP   \n",
       "69   chr9   21971120   21971120              G       A          Merck002  SNP   \n",
       "76  chr17    7579541    7579542             TC       -             SCLC8  DEL   \n",
       "77   chr9   21970952   21970952              C       T           colon_2  SNP   \n",
       "78   chr9   21971160   21971160              T       C           colon_2  SNP   \n",
       "79   chr9   21970929   21970929              G       C           colon_2  SNP   \n",
       "80  chr19    1223045    1223045              C       A           colon_5  SNP   \n",
       "\n",
       "   gene_name  genome  strand       ...          tier NORMAL_EXOME_ref_count  \\\n",
       "8        ATM      37     1.0       ...         tier1                  546.0   \n",
       "9       KRAS      37    -1.0       ...         tier1                  597.0   \n",
       "10     BRCA2      37     1.0       ...         tier1                  349.0   \n",
       "11     BRCA1      37    -1.0       ...         tier1                  469.0   \n",
       "14      MTOR      37    -1.0       ...         tier1                  764.0   \n",
       "15     MTHFR      37    -1.0       ...         tier1                  627.0   \n",
       "18     SF3B1      37    -1.0       ...         tier1                  611.0   \n",
       "34      BRAF      37    -1.0       ...         tier1                  102.0   \n",
       "38      TP53      37    -1.0       ...         tier1                   49.0   \n",
       "39      PTEN      37     1.0       ...         tier1                  106.0   \n",
       "47    NOTCH1      37    -1.0       ...         tier1                  109.0   \n",
       "55     BRCA1      37    -1.0       ...         tier1                   75.0   \n",
       "57      TP53      37    -1.0       ...         tier1                  107.0   \n",
       "59      TP53      37    -1.0       ...         tier1                  149.0   \n",
       "60      TP53      37    -1.0       ...         tier1                   88.0   \n",
       "62      TP53      37    -1.0       ...         tier1                  181.0   \n",
       "64     ASXL1      37     1.0       ...         tier1                  175.0   \n",
       "66    PIK3CA      37     1.0       ...         tier1                  584.0   \n",
       "67     FGFR1      37    -1.0       ...         tier1                  128.0   \n",
       "69    CDKN2A      37    -1.0       ...         tier1                  295.0   \n",
       "76      TP53      37    -1.0       ...         tier1                   33.0   \n",
       "77    CDKN2A      37     NaN       ...           NaN                    NaN   \n",
       "78    CDKN2A      37     NaN       ...           NaN                    NaN   \n",
       "79    CDKN2A      37     NaN       ...           NaN                    NaN   \n",
       "80     STK11      37     NaN       ...           NaN                    NaN   \n",
       "\n",
       "   NORMAL_EXOME_var_count NORMAL_EXOME_VAF  TUMOR_EXOME_ref_count  \\\n",
       "8                     1.0             0.18                    377   \n",
       "9                     0.0             0.00                    496   \n",
       "10                    0.0             0.00                    273   \n",
       "11                    0.0             0.00                    478   \n",
       "14                    1.0             0.13                    866   \n",
       "15                    0.0             0.00                    660   \n",
       "18                    0.0             0.00                    580   \n",
       "34                    0.0             0.00                     52   \n",
       "38                    0.0             0.00                      8   \n",
       "39                    0.0             0.00                      3   \n",
       "47                    0.0             0.00                     52   \n",
       "55                    0.0             0.00                     47   \n",
       "57                    3.0             2.73                     78   \n",
       "59                    0.0             0.00                     57   \n",
       "60                    0.0             0.00                     69   \n",
       "62                    0.0             0.00                     91   \n",
       "64                    1.0             0.57                    106   \n",
       "66                    1.0             0.17                    373   \n",
       "67                    0.0             0.00                     20   \n",
       "69                    0.0             0.00                     65   \n",
       "76                    0.0             0.00                     18   \n",
       "77                    NaN              NaN                     27   \n",
       "78                    NaN              NaN                     36   \n",
       "79                    NaN              NaN                     33   \n",
       "80                    NaN              NaN                     75   \n",
       "\n",
       "    TUMOR_EXOME_var_count  Coverage        VAF  Passed QC  CIViC Panel VAF  \n",
       "8                      18       942   4.550000        yes              0.0  \n",
       "9                      26      1119   4.980000        yes              0.0  \n",
       "10                     20       642   6.830000        yes              0.0  \n",
       "11                     41       988   7.900000        yes              0.0  \n",
       "14                     40      1671   4.420000        yes              0.0  \n",
       "15                     50      1337   7.040000        yes              0.0  \n",
       "18                     63      1254   9.800000        yes              0.0  \n",
       "34                     21       175  28.770000        yes              0.0  \n",
       "38                     33        90  80.490000        yes              0.0  \n",
       "39                     49       158  94.230000        yes              0.0  \n",
       "47                     35       196  40.230000        yes              0.0  \n",
       "55                     30       152  38.460000        yes              0.0  \n",
       "57                     22       210  21.780000        yes              0.0  \n",
       "59                     59       265  50.860000        yes              0.0  \n",
       "60                     40       197  36.700000        yes              0.0  \n",
       "62                     14       286  13.330000        yes              0.0  \n",
       "64                     51       333  32.480000        yes              0.0  \n",
       "66                    259      1217  40.920000        yes              0.0  \n",
       "67                     69       217  77.530000        yes              0.0  \n",
       "69                    240       600  78.690000        yes              0.0  \n",
       "76                     14        65  43.750000        yes              0.0  \n",
       "77                     27        54   0.500000        yes              0.0  \n",
       "78                     36        72   0.500000        yes              0.0  \n",
       "79                     31        64   0.484375        yes              0.0  \n",
       "80                     36       111   0.324324        yes              0.0  \n",
       "\n",
       "[25 rows x 23 columns]"
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "overlap_with_smmips[(overlap_with_smmips['Passed QC'] == 'yes') & (overlap_with_smmips['CIViC Panel VAF'] == 0)]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Build Waterfall Dataframe "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "samples_waterfall = overlap_with_smmips[overlap_with_smmips['Passed QC'] == 'yes']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# Make dataframe for WaterFall plot (Figure 1)\n",
    "samples_waterfall = samples_waterfall.filter(items=['sample', 'gene_name', 'amino_acid', 'VAF', 'CIViC Panel VAF'])\n",
    "samples_waterfall['Validated'] = (samples_waterfall['CIViC Panel VAF'] > 0).astype('int')\n",
    "\n",
    "samples_waterfall.to_csv('../data/validation_samples/waterfall_dataframe.tsv', sep='\\t')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Variant allele frequency correlation between  smMIPs CIViC panel and exome/genome sequencing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "pd.DataFrame.corr(overlap_with_smmips.filter(items=['CIViC Panel VAF', 'VAF']),method='pearson')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "fig, ax = plt.subplots( nrows=1, ncols=1 )\n",
    "plt.figure(figsize=(5,5))\n",
    "sns.scatterplot(x=\"Exome/Genome VAF\", y=\"CIViC Panel VAF\", data=samples, alpha=0.8, hue='Tumor Type', palette=\"deep\")\n",
    "plt.legend(bbox_to_anchor=(1.05, 1), loc=2, borderaxespad=0.)\n",
    "plt.savefig('../data/Figures/VAF_correlation_TumorType.png', bbox_inches='tight', dpi=400)\n",
    "plt.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "fig, ax = plt.subplots( nrows=1, ncols=1 )\n",
    "plt.figure(figsize=(5,5))\n",
    "sns.scatterplot(x=\"Exome/Genome VAF\", y=\"CIViC Panel VAF\", data=samples, alpha=0.8, hue='Matched Normal', palette=\"deep\")\n",
    "plt.legend(bbox_to_anchor=(1.05, 1), loc=2, borderaxespad=0.)\n",
    "plt.savefig('../data/Figures/VAF_correlation_MatchedNormal.png', bbox_inches='tight', dpi=400)\n",
    "plt.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "fig, ax = plt.subplots( nrows=1, ncols=1 )\n",
    "plt.figure(figsize=(5,5))\n",
    "sns.scatterplot(x=\"Exome/Genome VAF\", y=\"CIViC Panel VAF\", data=samples, alpha=0.8, hue='Passed QC', palette=\"deep\")\n",
    "plt.legend(bbox_to_anchor=(1.05, 1), loc=2, borderaxespad=0.)\n",
    "plt.savefig('../data/Figures/VAF_correlation_PassedQC.png', bbox_inches='tight', dpi=400)\n",
    "plt.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "fig, ax = plt.subplots( nrows=1, ncols=1 )\n",
    "plt.figure(figsize=(5,5))\n",
    "sns.scatterplot(x=\"Exome/Genome VAF\", y=\"CIViC Panel VAF\", data=samples, alpha=0.8, hue='Status', palette=\"deep\")\n",
    "plt.legend(bbox_to_anchor=(1.05, 1), loc=2, borderaxespad=0.)\n",
    "plt.savefig('../data/Figures/VAF_correlation_Status.png', bbox_inches='tight', dpi=400)\n",
    "plt.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
