{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Retrospective capture of clinically relevant variants"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Tools"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "#!/usr/bin/env python3\n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "import matplotlib as plt\n",
    "import glob"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Pull in Input Files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "overlap_with_smmips = pd.read_csv('../output/supplementary_table_3-variant_overlap.tsv', sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "samples_QC = pd.read_csv('../data/validation_samples/sequencing_quality_check.txt', sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "sample_info = pd.read_csv('../data/validation_samples/sample_dataframe.txt', sep='\\t')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## smMIPs variant rescue for samples with both tumor and matched normal"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "overlap_with_smmips_data = overlap_with_smmips.merge(sample_info, right_on='Sample', left_on='sample')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "tumor_normal_samples = []\n",
    "for item in overlap_with_smmips_data[(overlap_with_smmips_data['Passed QC'] == 'yes') & (overlap_with_smmips_data['Matched Normal'] == 'yes')]['sample'].drop_duplicates():\n",
    "    tumor_normal_samples.append(item)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "def find_somatic_variants(name):\n",
    "    print(name)\n",
    "    \n",
    "    to_iterate = []\n",
    "    for item in glob.glob('../data/smmips_sequencing/*.vcf'):\n",
    "        file_name = item.split('/')[-1].split('_')[0] +'_' + item.split('/')[-1].split('_')[1]\n",
    "        if name == file_name:\n",
    "            to_iterate.append(item)\n",
    "    if len(to_iterate) != 4:\n",
    "        print(\"Normal sequencing Failed for \" + name + \" ... skiping this sample\")\n",
    "        print()\n",
    "        return 0,0\n",
    "    else:\n",
    "        tumor_variants = pd.DataFrame()\n",
    "        normal_variants = pd.DataFrame()\n",
    "        for item in to_iterate:\n",
    "            current = pd.read_csv(item, sep='\\t', comment='#', header=None).filter(items=[0,1,1,3,4])\n",
    "            if len(current) > 1:\n",
    "                if item.split('_')[3].split('.')[0] == 'T':\n",
    "                    tumor_variants = tumor_variants.append(current)\n",
    "                if item.split('_')[3].split('.')[0] == 'N':  \n",
    "                    normal_variants = normal_variants.append(current)\n",
    "        tumor_variants.columns = ['chrom', 'start', 'stop', 'reference', 'variant']\n",
    "        normal_variants.columns = ['chrom', 'start', 'stop', 'reference', 'variant']\n",
    "        \n",
    "        print('Total tumor varinats: ', len(tumor_variants))\n",
    "        \n",
    "        somatic = tumor_variants.merge(normal_variants, how='outer', indicator=True)\n",
    "        somatic = somatic[somatic['_merge'] == 'left_only']\n",
    "\n",
    "#         somatic.to_csv('/Users/ebarnell/Desktop/' + name +'.bed.txt',sep='\\t',index=False)\n",
    "        print('Total somatic varinats: ', len(somatic))\n",
    "        return 1, int(len(somatic))\n",
    "    print()\n",
    "    \n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "OSCC_1\n",
      "Normal sequencing Failed for OSCC_1 ... skiping this sample\n",
      "\n",
      "SCLC_7\n",
      "Total tumor varinats:  764\n",
      "Total somatic varinats:  681\n",
      "SCLC_1\n",
      "Total tumor varinats:  120\n",
      "Total somatic varinats:  82\n",
      "SCLC_4\n",
      "Total tumor varinats:  131\n",
      "Total somatic varinats:  81\n",
      "SCLC_2\n",
      "Total tumor varinats:  149\n",
      "Total somatic varinats:  101\n",
      "OSCC_4\n",
      "Total tumor varinats:  193\n",
      "Total somatic varinats:  135\n",
      "SCLC_5\n",
      "Total tumor varinats:  413\n",
      "Total somatic varinats:  344\n",
      "SCLC_6\n",
      "Total tumor varinats:  257\n",
      "Total somatic varinats:  200\n",
      "SCLC_9\n",
      "Total tumor varinats:  242\n",
      "Total somatic varinats:  192\n",
      "HL_1\n",
      "Total tumor varinats:  100\n",
      "Total somatic varinats:  65\n",
      "SCLC_3\n",
      "Total tumor varinats:  147\n",
      "Total somatic varinats:  94\n",
      "OSCC_5\n",
      "Total tumor varinats:  94\n",
      "Total somatic varinats:  39\n",
      "SCLC_8\n",
      "Total tumor varinats:  122\n",
      "Total somatic varinats:  63\n"
     ]
    }
   ],
   "source": [
    "eligible_samples = 0\n",
    "total_variants = 0\n",
    "for item in tumor_normal_samples:\n",
    "    sample, somatic_variants = find_somatic_variants(item)\n",
    "    eligible_samples += sample\n",
    "    total_variants +=somatic_variants"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total samples with paired tumor/normal sequencing:  12\n",
      "Total stomatic variants identified exclusively on smMIPs:  2077\n"
     ]
    }
   ],
   "source": [
    "print('Total samples with paired tumor/normal sequencing: ', eligible_samples)\n",
    "print('Total stomatic variants identified exclusively on smMIPs: ', total_variants)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Determine Cause of Missed Variants"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 122,
   "metadata": {},
   "outputs": [],
   "source": [
    "manual_review_dataframe = pd.DataFrame()\n",
    "for item in glob.glob('../data/manual_review/*'):\n",
    "    name = item.split('/')[3].split('.')[0]\n",
    "    current_df = pd.read_csv(item,sep='\\t')\n",
    "    current_df['Sample'] = name\n",
    "    manual_review_dataframe = manual_review_dataframe.append(current_df)\n",
    "    \n",
    "    "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Analysis of germline variants"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 123,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total Germline Variants:  25\n",
      "13.661202185792352 %\n"
     ]
    }
   ],
   "source": [
    "#GERMLINE VARIANTS\n",
    "\n",
    "print('Total Germline Variants: ',len(manual_review_dataframe[manual_review_dataframe['Notes'] == 'germline']))\n",
    "\n",
    "print(len(manual_review_dataframe[manual_review_dataframe['Notes'] == 'germline'])/len(manual_review_dataframe)*100,'%')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Analysis of variants with sequencing artifacts "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 124,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total Sequencing Artifact Variants:  29\n",
      "15.846994535519126 %\n"
     ]
    }
   ],
   "source": [
    "#SEQUENCING ARTIFACTS\n",
    "\n",
    "print('Total Sequencing Artifact Variants: ',len(manual_review_dataframe[manual_review_dataframe['Notes'] == 'Sequencing artifact']))\n",
    "\n",
    "print(len(manual_review_dataframe[manual_review_dataframe['Notes'] == 'Sequencing artifact'])/len(manual_review_dataframe)*100,'%')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Analysis of variants not observed on original sequencing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total Variants not observed on original sequencing:  106\n",
      "57.92349726775956 %\n"
     ]
    }
   ],
   "source": [
    "print('Total Variants not observed on original sequencing: ',len(manual_review_dataframe[manual_review_dataframe['original_VAF'] == '0']))\n",
    "print(len(manual_review_dataframe[manual_review_dataframe['original_VAF'] == '0'])/len(manual_review_dataframe)*100,'%')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "metadata": {},
   "outputs": [],
   "source": [
    "smMIPs_somatic = pd.DataFrame()\n",
    "for name in tumor_normal_samples:\n",
    "    to_iterate = []\n",
    "    for item in glob.glob('../data/smmips_sequencing/*.vcf'):\n",
    "\n",
    "        \n",
    "        file_name = item.split('/')[-1].split('_')[0] +'_' + item.split('/')[-1].split('_')[1]\n",
    "\n",
    "        if name == file_name:\n",
    "            to_iterate.append(item)\n",
    "\n",
    "    if len(to_iterate) != 4:\n",
    "        continue\n",
    "\n",
    "    else:\n",
    "        tumor_variants = pd.DataFrame()\n",
    "        normal_variants = pd.DataFrame()\n",
    "        for item in to_iterate:\n",
    "            current = pd.read_csv(item, sep='\\t', comment='#', header=None).filter(items=[0,1,1,3,4,9])\n",
    "            if len(current) > 1:\n",
    "                if item.split('_')[3].split('.')[0] == 'T':\n",
    "                    tumor_variants = tumor_variants.append(current)\n",
    "                if item.split('_')[3].split('.')[0] == 'N':  \n",
    "                    normal_variants = normal_variants.append(current)\n",
    "        tumor_variants.columns = ['chrom', 'start', 'stop', 'reference', 'variant','GT:FALT:FTOT:RALT:RTOT:ALT:TOT:FRAC']\n",
    "        normal_variants.columns = ['chrom', 'start', 'stop', 'reference', 'variant','GT:FALT:FTOT:RALT:RTOT:ALT:TOT:FRAC']\n",
    "\n",
    "        somatic = tumor_variants.merge(normal_variants, how='outer', indicator=True)\n",
    "        somatic = somatic[somatic['_merge'] == 'left_only']\n",
    "\n",
    "        somatic['Sample'] = name\n",
    "\n",
    "        smMIPs_somatic = smMIPs_somatic.append(somatic)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "metadata": {},
   "outputs": [],
   "source": [
    "smMIPs_somatic['GT'], smMIPs_somatic['FALT'], smMIPs_somatic['FTOT'], smMIPs_somatic['RALT'], smMIPs_somatic['RTOT'], smMIPs_somatic['ALT'],smMIPs_somatic['TOT'],smMIPs_somatic['FRAC']  = smMIPs_somatic['GT:FALT:FTOT:RALT:RTOT:ALT:TOT:FRAC'].str.split(':', 8).str"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 128,
   "metadata": {},
   "outputs": [],
   "source": [
    "smMIPs_somatic = smMIPs_somatic.drop(['GT:FALT:FTOT:RALT:RTOT:ALT:TOT:FRAC', '_merge'], axis=1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 129,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "not_observed = manual_review_dataframe[manual_review_dataframe['original_VAF'] == '0']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 145,
   "metadata": {},
   "outputs": [],
   "source": [
    "not_observed_merge = not_observed.merge(smMIPs_somatic, how='outer', left_on=['Chromosome', 'Start', 'Stop', 'Reference', 'Variant', 'Sample'], right_on=['chrom', 'start', 'stop', 'reference', 'variant', 'Sample'],indicator=True)\n",
    "not_observed_merge = not_observed_merge[not_observed_merge['_merge'] == 'both']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 154,
   "metadata": {},
   "outputs": [],
   "source": [
    "insufficient_coverage = 0\n",
    "total = 0\n",
    "for i,row in not_observed_merge.iterrows():\n",
    "    total +=1\n",
    "    smMIPs_VAF = float(row['FRAC'])\n",
    "    original_tumor_coverage = float(row['tumor coverage'])\n",
    "    if original_tumor_coverage == 0:\n",
    "        insufficient_coverage +=1\n",
    "    else:\n",
    "        if 1/original_tumor_coverage > smMIPs_VAF:\n",
    "            insufficient_coverage +=1\n",
    "        \n",
    "    \n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 155,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Variants with insufficient coverage: 0.9905660377358491\n"
     ]
    }
   ],
   "source": [
    "print('Variants with insufficient coverage: ' + str(insufficient_coverage/total))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Analysis of  variants with support in the original sequencing data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 166,
   "metadata": {},
   "outputs": [],
   "source": [
    "original_support = manual_review_dataframe[(manual_review_dataframe['original_VAF'] != '0') & (manual_review_dataframe['original_VAF'] != 'None')]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 167,
   "metadata": {},
   "outputs": [],
   "source": [
    "original_support_merge = original_support.merge(smMIPs_somatic, how='outer', left_on=['Chromosome', 'Start', 'Stop', 'Reference', 'Variant', 'Sample'], right_on=['chrom', 'start', 'stop', 'reference', 'variant', 'Sample'],indicator=True)\n",
    "original_support_merge = original_support_merge[original_support_merge['_merge'] == 'both']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 170,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style>\n",
       "    .dataframe thead tr:only-child th {\n",
       "        text-align: right;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: left;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Chromosome</th>\n",
       "      <th>Start</th>\n",
       "      <th>Stop</th>\n",
       "      <th>Reference</th>\n",
       "      <th>Variant</th>\n",
       "      <th>Call</th>\n",
       "      <th>Tags</th>\n",
       "      <th>tumor coverage</th>\n",
       "      <th>normal coverage</th>\n",
       "      <th>original_VAF</th>\n",
       "      <th>...</th>\n",
       "      <th>variant</th>\n",
       "      <th>GT</th>\n",
       "      <th>FALT</th>\n",
       "      <th>FTOT</th>\n",
       "      <th>RALT</th>\n",
       "      <th>RTOT</th>\n",
       "      <th>ALT</th>\n",
       "      <th>TOT</th>\n",
       "      <th>FRAC</th>\n",
       "      <th>_merge</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>chr9</td>\n",
       "      <td>21994208.0</td>\n",
       "      <td>21994208.0</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>225</td>\n",
       "      <td>110</td>\n",
       "      <td>1%</td>\n",
       "      <td>...</td>\n",
       "      <td>A</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>3876</td>\n",
       "      <td>1</td>\n",
       "      <td>6866</td>\n",
       "      <td>2</td>\n",
       "      <td>10742</td>\n",
       "      <td>0.00018618506795755</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>chr9</td>\n",
       "      <td>139400268.0</td>\n",
       "      <td>139400268.0</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>72</td>\n",
       "      <td>26</td>\n",
       "      <td>1%</td>\n",
       "      <td>...</td>\n",
       "      <td>A</td>\n",
       "      <td>1</td>\n",
       "      <td>3</td>\n",
       "      <td>854</td>\n",
       "      <td>1</td>\n",
       "      <td>1418</td>\n",
       "      <td>4</td>\n",
       "      <td>2272</td>\n",
       "      <td>0.00176056338028169</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>chr4</td>\n",
       "      <td>106197116.0</td>\n",
       "      <td>106197116.0</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>58</td>\n",
       "      <td>42</td>\n",
       "      <td>2%</td>\n",
       "      <td>...</td>\n",
       "      <td>T</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>255</td>\n",
       "      <td>1</td>\n",
       "      <td>244</td>\n",
       "      <td>2</td>\n",
       "      <td>499</td>\n",
       "      <td>0.00400801603206413</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>chr17</td>\n",
       "      <td>7578202.0</td>\n",
       "      <td>7578202.0</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>98</td>\n",
       "      <td>113</td>\n",
       "      <td>17%</td>\n",
       "      <td>...</td>\n",
       "      <td>G</td>\n",
       "      <td>1</td>\n",
       "      <td>136</td>\n",
       "      <td>560</td>\n",
       "      <td>129</td>\n",
       "      <td>556</td>\n",
       "      <td>265</td>\n",
       "      <td>1116</td>\n",
       "      <td>0.237455197132616</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>chr17</td>\n",
       "      <td>7578457.0</td>\n",
       "      <td>7578457.0</td>\n",
       "      <td>CG</td>\n",
       "      <td>C</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>41</td>\n",
       "      <td>49</td>\n",
       "      <td>20%</td>\n",
       "      <td>...</td>\n",
       "      <td>C</td>\n",
       "      <td>1</td>\n",
       "      <td>172</td>\n",
       "      <td>238</td>\n",
       "      <td>765</td>\n",
       "      <td>967</td>\n",
       "      <td>937</td>\n",
       "      <td>1205</td>\n",
       "      <td>0.777593360995851</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>chr7</td>\n",
       "      <td>140453160.0</td>\n",
       "      <td>140453160.0</td>\n",
       "      <td>AT</td>\n",
       "      <td>A</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>73</td>\n",
       "      <td>102</td>\n",
       "      <td>29%</td>\n",
       "      <td>...</td>\n",
       "      <td>A</td>\n",
       "      <td>1</td>\n",
       "      <td>106</td>\n",
       "      <td>242</td>\n",
       "      <td>404</td>\n",
       "      <td>1058</td>\n",
       "      <td>510</td>\n",
       "      <td>1300</td>\n",
       "      <td>0.392307692307692</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>chr13</td>\n",
       "      <td>32912785.0</td>\n",
       "      <td>32912785.0</td>\n",
       "      <td>A</td>\n",
       "      <td>T</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>45</td>\n",
       "      <td>108</td>\n",
       "      <td>33%</td>\n",
       "      <td>...</td>\n",
       "      <td>T</td>\n",
       "      <td>1</td>\n",
       "      <td>52</td>\n",
       "      <td>137</td>\n",
       "      <td>65</td>\n",
       "      <td>196</td>\n",
       "      <td>117</td>\n",
       "      <td>333</td>\n",
       "      <td>0.351351351351351</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>chr9</td>\n",
       "      <td>139401330.0</td>\n",
       "      <td>139401330.0</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>215</td>\n",
       "      <td>111</td>\n",
       "      <td>85%</td>\n",
       "      <td>...</td>\n",
       "      <td>A</td>\n",
       "      <td>1</td>\n",
       "      <td>836</td>\n",
       "      <td>953</td>\n",
       "      <td>806</td>\n",
       "      <td>920</td>\n",
       "      <td>1642</td>\n",
       "      <td>1873</td>\n",
       "      <td>0.876668446342766</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>chr17</td>\n",
       "      <td>7578518.0</td>\n",
       "      <td>7578518.0</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>60</td>\n",
       "      <td>86</td>\n",
       "      <td>72%</td>\n",
       "      <td>...</td>\n",
       "      <td>G</td>\n",
       "      <td>1</td>\n",
       "      <td>157</td>\n",
       "      <td>233</td>\n",
       "      <td>285</td>\n",
       "      <td>432</td>\n",
       "      <td>442</td>\n",
       "      <td>665</td>\n",
       "      <td>0.664661654135338</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>chr2</td>\n",
       "      <td>25459828.0</td>\n",
       "      <td>25459828.0</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>130</td>\n",
       "      <td>133</td>\n",
       "      <td>31%</td>\n",
       "      <td>...</td>\n",
       "      <td>T</td>\n",
       "      <td>1</td>\n",
       "      <td>29</td>\n",
       "      <td>121</td>\n",
       "      <td>57</td>\n",
       "      <td>212</td>\n",
       "      <td>86</td>\n",
       "      <td>333</td>\n",
       "      <td>0.258258258258258</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>chr13</td>\n",
       "      <td>32915105.0</td>\n",
       "      <td>32915105.0</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>138</td>\n",
       "      <td>108</td>\n",
       "      <td>31%</td>\n",
       "      <td>...</td>\n",
       "      <td>T</td>\n",
       "      <td>1</td>\n",
       "      <td>23</td>\n",
       "      <td>76</td>\n",
       "      <td>104</td>\n",
       "      <td>310</td>\n",
       "      <td>127</td>\n",
       "      <td>386</td>\n",
       "      <td>0.329015544041451</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>chr2</td>\n",
       "      <td>25463176.0</td>\n",
       "      <td>25463176.0</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>93</td>\n",
       "      <td>105</td>\n",
       "      <td>26%</td>\n",
       "      <td>...</td>\n",
       "      <td>C</td>\n",
       "      <td>1</td>\n",
       "      <td>237</td>\n",
       "      <td>929</td>\n",
       "      <td>100</td>\n",
       "      <td>441</td>\n",
       "      <td>337</td>\n",
       "      <td>1370</td>\n",
       "      <td>0.245985401459854</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>chr9</td>\n",
       "      <td>139412606.0</td>\n",
       "      <td>139412606.0</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>128</td>\n",
       "      <td>136</td>\n",
       "      <td>1%</td>\n",
       "      <td>...</td>\n",
       "      <td>C</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>5444</td>\n",
       "      <td>1</td>\n",
       "      <td>2608</td>\n",
       "      <td>2</td>\n",
       "      <td>8052</td>\n",
       "      <td>0.000248385494287134</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>chr9</td>\n",
       "      <td>139412675.0</td>\n",
       "      <td>139412675.0</td>\n",
       "      <td>T</td>\n",
       "      <td>G</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>134</td>\n",
       "      <td>143</td>\n",
       "      <td>1%</td>\n",
       "      <td>...</td>\n",
       "      <td>G</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>5444</td>\n",
       "      <td>1</td>\n",
       "      <td>2608</td>\n",
       "      <td>2</td>\n",
       "      <td>8052</td>\n",
       "      <td>0.000248385494287134</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>chr9</td>\n",
       "      <td>139409805.0</td>\n",
       "      <td>139409805.0</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>136</td>\n",
       "      <td>143</td>\n",
       "      <td>1%</td>\n",
       "      <td>...</td>\n",
       "      <td>T</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>1462</td>\n",
       "      <td>1</td>\n",
       "      <td>2291</td>\n",
       "      <td>2</td>\n",
       "      <td>3753</td>\n",
       "      <td>0.000532907007727152</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>chr18</td>\n",
       "      <td>48584827.0</td>\n",
       "      <td>48584827.0</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>156</td>\n",
       "      <td>188</td>\n",
       "      <td>1%</td>\n",
       "      <td>...</td>\n",
       "      <td>T</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>180</td>\n",
       "      <td>17</td>\n",
       "      <td>939</td>\n",
       "      <td>18</td>\n",
       "      <td>1119</td>\n",
       "      <td>0.0160857908847185</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>chr18</td>\n",
       "      <td>48584804.0</td>\n",
       "      <td>48584804.0</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>163</td>\n",
       "      <td>187</td>\n",
       "      <td>1%</td>\n",
       "      <td>...</td>\n",
       "      <td>T</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>180</td>\n",
       "      <td>8</td>\n",
       "      <td>939</td>\n",
       "      <td>9</td>\n",
       "      <td>1119</td>\n",
       "      <td>0.00804289544235925</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>chr2</td>\n",
       "      <td>25536880.0</td>\n",
       "      <td>25536880.0</td>\n",
       "      <td>T</td>\n",
       "      <td>G</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>181</td>\n",
       "      <td>136</td>\n",
       "      <td>1%</td>\n",
       "      <td>...</td>\n",
       "      <td>G</td>\n",
       "      <td>1</td>\n",
       "      <td>4</td>\n",
       "      <td>543</td>\n",
       "      <td>3</td>\n",
       "      <td>716</td>\n",
       "      <td>7</td>\n",
       "      <td>1259</td>\n",
       "      <td>0.00555996822875298</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>chr2</td>\n",
       "      <td>25523037.0</td>\n",
       "      <td>25523037.0</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>203</td>\n",
       "      <td>192</td>\n",
       "      <td>1%</td>\n",
       "      <td>...</td>\n",
       "      <td>C</td>\n",
       "      <td>1</td>\n",
       "      <td>2</td>\n",
       "      <td>731</td>\n",
       "      <td>1</td>\n",
       "      <td>267</td>\n",
       "      <td>3</td>\n",
       "      <td>998</td>\n",
       "      <td>0.0030060120240481</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>chr9</td>\n",
       "      <td>139409121.0</td>\n",
       "      <td>139409121.0</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>449</td>\n",
       "      <td>416</td>\n",
       "      <td>1%</td>\n",
       "      <td>...</td>\n",
       "      <td>A</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>2534</td>\n",
       "      <td>1</td>\n",
       "      <td>5228</td>\n",
       "      <td>2</td>\n",
       "      <td>7762</td>\n",
       "      <td>0.00025766555011595</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>chr9</td>\n",
       "      <td>139401136.0</td>\n",
       "      <td>139401136.0</td>\n",
       "      <td>A</td>\n",
       "      <td>C</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>200</td>\n",
       "      <td>206</td>\n",
       "      <td>0.50%</td>\n",
       "      <td>...</td>\n",
       "      <td>C</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>2444</td>\n",
       "      <td>1</td>\n",
       "      <td>1784</td>\n",
       "      <td>2</td>\n",
       "      <td>4228</td>\n",
       "      <td>0.000473036896877956</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>chr9</td>\n",
       "      <td>139413142.0</td>\n",
       "      <td>139413142.0</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>263</td>\n",
       "      <td>266</td>\n",
       "      <td>0.38%</td>\n",
       "      <td>...</td>\n",
       "      <td>T</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>2233</td>\n",
       "      <td>2</td>\n",
       "      <td>10495</td>\n",
       "      <td>3</td>\n",
       "      <td>12728</td>\n",
       "      <td>0.000235700817096166</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>chr9</td>\n",
       "      <td>139409052.0</td>\n",
       "      <td>139409052.0</td>\n",
       "      <td>T</td>\n",
       "      <td>G</td>\n",
       "      <td>S</td>\n",
       "      <td>NaN</td>\n",
       "      <td>528</td>\n",
       "      <td>459</td>\n",
       "      <td>0.19%</td>\n",
       "      <td>...</td>\n",
       "      <td>G</td>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>2240</td>\n",
       "      <td>1</td>\n",
       "      <td>5228</td>\n",
       "      <td>2</td>\n",
       "      <td>7468</td>\n",
       "      <td>0.000267809319764328</td>\n",
       "      <td>both</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>23 rows × 26 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "   Chromosome        Start         Stop Reference Variant Call Tags  \\\n",
       "0        chr9   21994208.0   21994208.0         G       A    S  NaN   \n",
       "1        chr9  139400268.0  139400268.0         G       A    S  NaN   \n",
       "2        chr4  106197116.0  106197116.0         C       T    S  NaN   \n",
       "3       chr17    7578202.0    7578202.0         A       G    S  NaN   \n",
       "4       chr17    7578457.0    7578457.0        CG       C    S  NaN   \n",
       "5        chr7  140453160.0  140453160.0        AT       A    S  NaN   \n",
       "6       chr13   32912785.0   32912785.0         A       T    S  NaN   \n",
       "7        chr9  139401330.0  139401330.0         G       A    S  NaN   \n",
       "8       chr17    7578518.0    7578518.0         C       G    S  NaN   \n",
       "9        chr2   25459828.0   25459828.0         G       T    S  NaN   \n",
       "10      chr13   32915105.0   32915105.0         G       T    S  NaN   \n",
       "11       chr2   25463176.0   25463176.0         G       C    S  NaN   \n",
       "12       chr9  139412606.0  139412606.0         A       C    S  NaN   \n",
       "13       chr9  139412675.0  139412675.0         T       G    S  NaN   \n",
       "14       chr9  139409805.0  139409805.0         C       T    S  NaN   \n",
       "15      chr18   48584827.0   48584827.0         G       T    S  NaN   \n",
       "16      chr18   48584804.0   48584804.0         G       T    S  NaN   \n",
       "17       chr2   25536880.0   25536880.0         T       G    S  NaN   \n",
       "18       chr2   25523037.0   25523037.0         G       C    S  NaN   \n",
       "19       chr9  139409121.0  139409121.0         G       A    S  NaN   \n",
       "20       chr9  139401136.0  139401136.0         A       C    S  NaN   \n",
       "21       chr9  139413142.0  139413142.0         C       T    S  NaN   \n",
       "22       chr9  139409052.0  139409052.0         T       G    S  NaN   \n",
       "\n",
       "   tumor coverage normal coverage original_VAF  ...   variant GT FALT  FTOT  \\\n",
       "0             225             110           1%  ...         A  1    1  3876   \n",
       "1              72              26           1%  ...         A  1    3   854   \n",
       "2              58              42           2%  ...         T  1    1   255   \n",
       "3              98             113          17%  ...         G  1  136   560   \n",
       "4              41              49          20%  ...         C  1  172   238   \n",
       "5              73             102          29%  ...         A  1  106   242   \n",
       "6              45             108          33%  ...         T  1   52   137   \n",
       "7             215             111          85%  ...         A  1  836   953   \n",
       "8              60              86          72%  ...         G  1  157   233   \n",
       "9             130             133          31%  ...         T  1   29   121   \n",
       "10            138             108          31%  ...         T  1   23    76   \n",
       "11             93             105          26%  ...         C  1  237   929   \n",
       "12            128             136           1%  ...         C  1    1  5444   \n",
       "13            134             143           1%  ...         G  1    1  5444   \n",
       "14            136             143           1%  ...         T  1    1  1462   \n",
       "15            156             188           1%  ...         T  1    1   180   \n",
       "16            163             187           1%  ...         T  1    1   180   \n",
       "17            181             136           1%  ...         G  1    4   543   \n",
       "18            203             192           1%  ...         C  1    2   731   \n",
       "19            449             416           1%  ...         A  1    1  2534   \n",
       "20            200             206        0.50%  ...         C  1    1  2444   \n",
       "21            263             266        0.38%  ...         T  1    1  2233   \n",
       "22            528             459        0.19%  ...         G  1    1  2240   \n",
       "\n",
       "    RALT   RTOT   ALT    TOT                  FRAC _merge  \n",
       "0      1   6866     2  10742   0.00018618506795755   both  \n",
       "1      1   1418     4   2272   0.00176056338028169   both  \n",
       "2      1    244     2    499   0.00400801603206413   both  \n",
       "3    129    556   265   1116     0.237455197132616   both  \n",
       "4    765    967   937   1205     0.777593360995851   both  \n",
       "5    404   1058   510   1300     0.392307692307692   both  \n",
       "6     65    196   117    333     0.351351351351351   both  \n",
       "7    806    920  1642   1873     0.876668446342766   both  \n",
       "8    285    432   442    665     0.664661654135338   both  \n",
       "9     57    212    86    333     0.258258258258258   both  \n",
       "10   104    310   127    386     0.329015544041451   both  \n",
       "11   100    441   337   1370     0.245985401459854   both  \n",
       "12     1   2608     2   8052  0.000248385494287134   both  \n",
       "13     1   2608     2   8052  0.000248385494287134   both  \n",
       "14     1   2291     2   3753  0.000532907007727152   both  \n",
       "15    17    939    18   1119    0.0160857908847185   both  \n",
       "16     8    939     9   1119   0.00804289544235925   both  \n",
       "17     3    716     7   1259   0.00555996822875298   both  \n",
       "18     1    267     3    998    0.0030060120240481   both  \n",
       "19     1   5228     2   7762   0.00025766555011595   both  \n",
       "20     1   1784     2   4228  0.000473036896877956   both  \n",
       "21     2  10495     3  12728  0.000235700817096166   both  \n",
       "22     1   5228     2   7468  0.000267809319764328   both  \n",
       "\n",
       "[23 rows x 26 columns]"
      ]
     },
     "execution_count": 170,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sns.original_support_merge"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
